---
figid: PMC1797292__zjb0030764020001
figlink: /pmc/articles/PMC1797292/figure/f1/
number: FIG. 1
caption: The activated methyl cycle, biosynthesis of AI-2, and directly coupled metabolic
  pathways in gram-positive bacteria. AI-2 is produced from SAM, which is essential
  for a large number of methylation processes and is used for polyamine synthesis,
  in three enzymatic steps. During various methyltransferase reactions, SAM is converted
  to SAH. SAH is detoxified by the Pfs enzyme to yield adenine and SRH. LuxS then
  catalyzes the conversion of SRH, yielding DPD and homocysteine. DPD undergoes spontaneous
  rearrangements to form AI-2. Homocysteine is used to recycle methionine. Methionine
  synthase is required by all organisms to ensure the regeneration of the methyl group
  of SAM. Most gram-positive bacteria contain a cobalamin-independent methionine synthase,
  MetE, and a YxjH enzyme, which is suggested to be an alternative cobalamin-independent
  methionine synthase that is mainly involved in the SAM recycling pathway. Methionine
  is then converted to SAM in a reaction catalyzed by SAM synthetase (MetK). Homocysteine
  can also be synthesized de novo from oxaloacetate in seven steps as part of the
  aspartate family. Notably, some organisms, such as L. plantarum and L. salivarius,
  can synthesize homocysteine from cysteine by direct sulfhydrylation involving the
  CysK enzyme. Homocysteine can also be converted to cysteine via the reverse transsulfuration
  pathway. The enzymes cystathionine β-lyase and cystathionine γ-synthase (encoded
  by metI and metC homologues) require pyridoxal-phosphate as cofactor. These complex
  branching pathways are coordinated by a variety of regulatory mechanisms, including
  multivalent feedback inhibition of enzyme activity by more than one end product,
  “metabolite-sensing” controllers of gene expression such as riboswitches and amino-acid-specific
  T boxes, and regulatory proteins. Moreover, the scheme presented here is simplified.
  For example, some gram-positive organisms can use alternative organic and inorganic
  sulfur sources, and some bacteria use O-succinylhomoserine instead of O-acetylhomoserine
  in the MetB-catalyzed reaction. Additionally, not all gram-positive bacteria contain
  the complete set of pathways. The common gene names for B. subtilis and E. coli
  are used. The pathway was constructed based on previously described data (, ), the
  KEGG PATHWAY website (http://www.genome.jp/kegg/pathway/map/map00271.html), and
  the LacplantCyc pathway genome database (). TCA, tricarboxylic acid; MTA, methylthioadenosine;
  MTR, methylthioribose; THF, tetrahydrofolate; THPG, tetrahydropteroyltriglutamate;
  dcSAM, decarboxylated SAM.
pmcid: PMC1797292
papertitle: Functional Analysis of luxS in the Probiotic Strain Lactobacillus rhamnosus
  GG Reveals a Central Metabolic Role Important for Growth and Biofilm Formation .
reftext: Sarah Lebeer, et al. J Bacteriol. 2007 Feb;189(3):860-871.
pmc_ranked_result_index: '17869'
pathway_score: 0.9425957
filename: zjb0030764020001.jpg
figtitle: The activated methyl cycle, biosynthesis of AI-2, and directly coupled metabolic
  pathways in gram-positive bacteria
year: '2007'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC1797292__zjb0030764020001.html
  '@type': Dataset
  description: The activated methyl cycle, biosynthesis of AI-2, and directly coupled
    metabolic pathways in gram-positive bacteria. AI-2 is produced from SAM, which
    is essential for a large number of methylation processes and is used for polyamine
    synthesis, in three enzymatic steps. During various methyltransferase reactions,
    SAM is converted to SAH. SAH is detoxified by the Pfs enzyme to yield adenine
    and SRH. LuxS then catalyzes the conversion of SRH, yielding DPD and homocysteine.
    DPD undergoes spontaneous rearrangements to form AI-2. Homocysteine is used to
    recycle methionine. Methionine synthase is required by all organisms to ensure
    the regeneration of the methyl group of SAM. Most gram-positive bacteria contain
    a cobalamin-independent methionine synthase, MetE, and a YxjH enzyme, which is
    suggested to be an alternative cobalamin-independent methionine synthase that
    is mainly involved in the SAM recycling pathway. Methionine is then converted
    to SAM in a reaction catalyzed by SAM synthetase (MetK). Homocysteine can also
    be synthesized de novo from oxaloacetate in seven steps as part of the aspartate
    family. Notably, some organisms, such as L. plantarum and L. salivarius, can synthesize
    homocysteine from cysteine by direct sulfhydrylation involving the CysK enzyme.
    Homocysteine can also be converted to cysteine via the reverse transsulfuration
    pathway. The enzymes cystathionine β-lyase and cystathionine γ-synthase (encoded
    by metI and metC homologues) require pyridoxal-phosphate as cofactor. These complex
    branching pathways are coordinated by a variety of regulatory mechanisms, including
    multivalent feedback inhibition of enzyme activity by more than one end product,
    “metabolite-sensing” controllers of gene expression such as riboswitches and amino-acid-specific
    T boxes, and regulatory proteins. Moreover, the scheme presented here is simplified.
    For example, some gram-positive organisms can use alternative organic and inorganic
    sulfur sources, and some bacteria use O-succinylhomoserine instead of O-acetylhomoserine
    in the MetB-catalyzed reaction. Additionally, not all gram-positive bacteria contain
    the complete set of pathways. The common gene names for B. subtilis and E. coli
    are used. The pathway was constructed based on previously described data (, ),
    the KEGG PATHWAY website (http://www.genome.jp/kegg/pathway/map/map00271.html),
    and the LacplantCyc pathway genome database (). TCA, tricarboxylic acid; MTA,
    methylthioadenosine; MTR, methylthioribose; THF, tetrahydrofolate; THPG, tetrahydropteroyltriglutamate;
    dcSAM, decarboxylated SAM.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - METTL2B
  - ACSM3
  - MTR
  - DPYD
  - adenine
  - Aspartate
  - Aspartate semialdehyde
  - Cystathionine
  - Cysteine
  - cysteine
  - Glutathione
  - Glycine
  - Homocysteine
  - Isoleucine
  - Methionine
  - Oxaloacetate
  - putrescine
  - spermidine
  - Sulfate
  - Threonine
  - glutamate
genes:
- word: metl
  symbol: METL
  source: hgnc_alias_symbol
  hgnc_symbol: METTL2B
  entrez: '55798'
- word: SAH
  symbol: SAH
  source: hgnc_prev_symbol
  hgnc_symbol: ACSM3
  entrez: '6296'
- word: MTR
  symbol: MTR
  source: hgnc_symbol
  hgnc_symbol: MTR
  entrez: '4548'
- word: DPD
  symbol: DPD
  source: hgnc_alias_symbol
  hgnc_symbol: DPYD
  entrez: '1806'
chemicals:
- word: adenine
  source: MESH
  identifier: D000225
- word: Aspartate
  source: MESH
  identifier: D001224
- word: Aspartate semialdehyde
  source: MESH
  identifier: D001224
- word: Cystathionine
  source: MESH
  identifier: D003540
- word: Cysteine
  source: MESH
  identifier: D003545
- word: cysteine
  source: MESH
  identifier: D003545
- word: Glutathione
  source: MESH
  identifier: D005978
- word: Glycine
  source: MESH
  identifier: D005998
- word: Homocysteine
  source: MESH
  identifier: D006710
- word: Isoleucine
  source: MESH
  identifier: D007532
- word: Methionine
  source: MESH
  identifier: D008715
- word: Oxaloacetate
  source: MESH
  identifier: D010071
- word: putrescine
  source: MESH
  identifier: D011700
- word: spermidine
  source: MESH
  identifier: D013095
- word: Sulfate
  source: MESH
  identifier: D013431
- word: Threonine
  source: MESH
  identifier: D013912
- word: glutamate
  source: MESH
  identifier: D018698
diseases: []
---
