---
figid: PMC2263022__1471-2105-9-110-6
figlink: /pmc/articles/PMC2263022/figure/F6/
number: Figure 6
caption: 'Superposition of ''omics'' data on maps. By allowing the user to superimpose
  functional genomic or proteomic data on pathway maps, SPIKE facilitates improved
  interpretation of such datasets. The color of the bar above each gene''s node indicates
  its response. Upregulated genes are shades of red: the darker the red the greater
  the fold of induction. Similarly, downregulated genes are in shades of green. Genes
  whose expression was not changed are yellow, and genes for which data are not available
  (e.g., genes not present on the microarray) are grey. In this example, superimposing
  gene expression data measured in human lymphocyte cells exposed to ionizing radiation
  on the signaling map shows the activation of the p53 network (top-left part of the
  network) and the down-regulation of genes that function in the cell cycle G2-M phase
  transition (bottom-right part of the network). Superimposing the data on this map
  clearly shows that a large fraction of the p53 network was activated in the analyzed
  condition. SPIKE also allows superimposition of clustering data: any partition of
  the genes into groups (e.g., according to GO annotation, cellular compartment, clustering,
  or user-specific definitions) can be viewed on the pathway map. The color scheme
  can be adjusted by the user.'
pmcid: PMC2263022
papertitle: SPIKE â€“ a database, visualization and analysis tool of cellular signaling
  pathways.
reftext: Ran Elkon, et al. BMC Bioinformatics. 2008;9:110-110.
pmc_ranked_result_index: '14676'
pathway_score: 0.7718888
filename: 1471-2105-9-110-6.jpg
figtitle: Superposition of 'omics' data on maps
year: '2008'
organisms: Homo sapiens
ndex: e924a918-df21-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2263022__1471-2105-9-110-6.html
  '@type': Dataset
  description: 'Superposition of ''omics'' data on maps. By allowing the user to superimpose
    functional genomic or proteomic data on pathway maps, SPIKE facilitates improved
    interpretation of such datasets. The color of the bar above each gene''s node
    indicates its response. Upregulated genes are shades of red: the darker the red
    the greater the fold of induction. Similarly, downregulated genes are in shades
    of green. Genes whose expression was not changed are yellow, and genes for which
    data are not available (e.g., genes not present on the microarray) are grey. In
    this example, superimposing gene expression data measured in human lymphocyte
    cells exposed to ionizing radiation on the signaling map shows the activation
    of the p53 network (top-left part of the network) and the down-regulation of genes
    that function in the cell cycle G2-M phase transition (bottom-right part of the
    network). Superimposing the data on this map clearly shows that a large fraction
    of the p53 network was activated in the analyzed condition. SPIKE also allows
    superimposition of clustering data: any partition of the genes into groups (e.g.,
    according to GO annotation, cellular compartment, clustering, or user-specific
    definitions) can be viewed on the pathway map. The color scheme can be adjusted
    by the user.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MAP2K1
  - APC
  - CFH
  - RFC2
  - BAX
  - RFC4
  - ATM
  - SMC3
  - PCNA
  - RFC3
  - CENPE
  - CYCS
  - RFC5
  - RAD21
  - XPC
  - TRAF4
  - RFC1
  - CENPF
  - CENPA
  - MAP2K2
  - TTK
genes:
- word: MEK
  symbol: MEK
  source: bioentities_symbol
  hgnc_symbol: MAP2K1
  entrez: '5604'
- word: APC
  symbol: APC
  source: hgnc_symbol
  hgnc_symbol: APC
  entrez: '324'
- word: HUS
  symbol: HUS
  source: hgnc_alias_symbol
  hgnc_symbol: CFH
  entrez: '3075'
- word: RFC
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC2
  entrez: '5982'
- word: BAX
  symbol: BAX
  source: hgnc_symbol
  hgnc_symbol: BAX
  entrez: '581'
- word: RFC
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC4
  entrez: '5984'
- word: '"""ATM"'
  symbol: ATM
  source: hgnc_symbol
  hgnc_symbol: ATM
  entrez: '472'
- word: Cohesin
  symbol: Cohesin
  source: bioentities_symbol
  hgnc_symbol: SMC3
  entrez: '9126'
- word: '"""PCNA"'
  symbol: PCNA
  source: hgnc_symbol
  hgnc_symbol: PCNA
  entrez: '5111'
- word: RFC
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC3
  entrez: '5983'
- word: CENPE
  symbol: CENPE
  source: hgnc_symbol
  hgnc_symbol: CENPE
  entrez: '1062'
- word: Cyc
  symbol: CYC
  source: hgnc_alias_symbol
  hgnc_symbol: CYCS
  entrez: '54205'
- word: RFC
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC5
  entrez: '5985'
- word: Cohesin
  symbol: Cohesin
  source: bioentities_symbol
  hgnc_symbol: RAD21
  entrez: '5885'
- word: XPC
  symbol: XPC
  source: hgnc_symbol
  hgnc_symbol: XPC
  entrez: '7508'
- word: ',TRAF4'
  symbol: TRAF4
  source: hgnc_symbol
  hgnc_symbol: TRAF4
  entrez: '9618'
- word: RFC
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC1
  entrez: '5981'
- word: CENPF
  symbol: CENPF
  source: hgnc_symbol
  hgnc_symbol: CENPF
  entrez: '1063'
- word: CENPA
  symbol: CENPA
  source: hgnc_symbol
  hgnc_symbol: CENPA
  entrez: '1058'
- word: MEK
  symbol: MEK
  source: bioentities_symbol
  hgnc_symbol: MAP2K2
  entrez: '5605'
- word: TTK
  symbol: TTK
  source: hgnc_symbol
  hgnc_symbol: TTK
  entrez: '7272'
chemicals: []
diseases: []
---
