---
figid: PMC2882257__jexboterq089f03_ht
figlink: /pmc/articles/PMC2882257/figure/fig3/
number: F3
caption: Induction of the oxylipin biosynthethic pathway in arbuscular mycorrhizal
  symbiosis. Metabolic scheme of the oxylipin pathway including the 9-LOX and 13-LOX
  branches (modified after ). Shaded boxes show the metabolites analysed by LC-MS/MS
  in the present study. Thick arrows indicate the direction of the changes in expression
  levels compared with non-mycorrhizal control roots (up- or down-regulated) of the
  genes coding for the enzyme cited, as determined by the transcriptomic analysis;
  = indicates no changes in gene expression. LOX, lipoxygenase; AOS, allene oxide
  synthase; AOC, allene oxide cyclase; OPR3, oxo-phytodienoic acid reductase; JMT,
  jasmonic acid carboxyl methyltransferase; JAME, methyl jasmonate esterase; JAR1,
  jasmonate-amino synthetase; HPL, hydroperoxide lyase; and DES, divinyl ether synthase.
pmcid: PMC2882257
papertitle: Hormonal and transcriptional profiles highlight common and differential
  host responses to arbuscular mycorrhizal fungi and the regulation of the oxylipin
  pathway.
reftext: Juan A. López-Ráez, et al. J Exp Bot. 2010 Jun;61(10):2589-2601.
pmc_ranked_result_index: '89146'
pathway_score: 0.9283642
filename: jexboterq089f03_ht.jpg
figtitle: Induction of the oxylipin biosynthethic pathway in arbuscular mycorrhizal
  symbiosis
year: '2010'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2882257__jexboterq089f03_ht.html
  '@type': Dataset
  description: Induction of the oxylipin biosynthethic pathway in arbuscular mycorrhizal
    symbiosis. Metabolic scheme of the oxylipin pathway including the 9-LOX and 13-LOX
    branches (modified after ). Shaded boxes show the metabolites analysed by LC-MS/MS
    in the present study. Thick arrows indicate the direction of the changes in expression
    levels compared with non-mycorrhizal control roots (up- or down-regulated) of
    the genes coding for the enzyme cited, as determined by the transcriptomic analysis;
    = indicates no changes in gene expression. LOX, lipoxygenase; AOS, allene oxide
    synthase; AOC, allene oxide cyclase; OPR3, oxo-phytodienoic acid reductase; JMT,
    jasmonic acid carboxyl methyltransferase; JAME, methyl jasmonate esterase; JAR1,
    jasmonate-amino synthetase; HPL, hydroperoxide lyase; and DES, divinyl ether synthase.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - GNAI3
  - GNAI2
  - GNAI1
  - GNAT1
  - GNAZ
  - GNAT3
  - GNAT2
  - DES
  - GNAO1
  - Divinyl ether
  - colnelenic acid
  - (3Z)-aldehydes
  - oxo-fatty acids
  - linolenic acid
genes:
- word: Gi
  symbol: Gi
  source: bioentities_symbol
  hgnc_symbol: GNAI3
  entrez: '2773'
- word: Gi
  symbol: Gi
  source: bioentities_symbol
  hgnc_symbol: GNAI2
  entrez: '2771'
- word: Gi
  symbol: Gi
  source: bioentities_symbol
  hgnc_symbol: GNAI1
  entrez: '2770'
- word: Gi
  symbol: Gi
  source: bioentities_symbol
  hgnc_symbol: GNAT1
  entrez: '2779'
- word: Gi
  symbol: Gi
  source: bioentities_symbol
  hgnc_symbol: GNAZ
  entrez: '2781'
- word: Gi
  symbol: Gi
  source: bioentities_symbol
  hgnc_symbol: GNAT3
  entrez: '346562'
- word: Gi
  symbol: Gi
  source: bioentities_symbol
  hgnc_symbol: GNAT2
  entrez: '2780'
- word: DES
  symbol: DES
  source: hgnc_symbol
  hgnc_symbol: DES
  entrez: '1674'
- word: Gi
  symbol: Gi
  source: bioentities_symbol
  hgnc_symbol: GNAO1
  entrez: '2775'
chemicals:
- word: Divinyl ether
  source: MESH
  identifier: C100195
- word: colnelenic acid
  source: MESH
  identifier: C118818
- word: (3Z)-aldehydes
  source: MESH
  identifier: D000447
- word: oxo-fatty acids
  source: MESH
  identifier: D005227
- word: linolenic acid
  source: MESH
  identifier: D008042
diseases: []
figid_alias: PMC2882257__F3
redirect_from: /figures/PMC2882257__F3
figtype: Figure
---
