---
figid: PMC2694679__msb200929-f1
figlink: /pmc/articles/PMC2694679/figure/f1/
number: F1
caption: The central enzymes of the Asp-derived amino-acid pathway in chloroplasts
  and allosteric regulation. (A) The amino acids Lys, Met, Thr and Ile, and the methylating
  agent AdoMet are synthesized from aspartate. Simple and double arrows indicate reactions
  treated as irreversible and reversible, respectively. Enzyme names are indicated
  in italics. (B) Regulatory map of the Asp-derived amino-acid pathway in Arabidopsis
  leaf mesophyll cell. The 13 enzymes constituting the core of the system are co-expressed
  in Arabidopsis mesophyll cell chloroplasts. Isoforms exist for monofunctional AKs
  (AK1, AK2), bifunctional AKs (AKI–HSDH I and AKII–HSDH II) and DHDPS (DHDPS1 and
  2). AK3 and TS2 are not expressed in mesophyll cell chloroplasts (see ). Genes for
  ASADH, HSK, CGS and allosteric TD are present as single copies in the Arabidopsis
  genome. Enzymes shown against a white background are non-allosteric, and those with
  a yellow background are allosteric. Bifunctional AK–HSDH proteins are symbolized
  with a linker between the two domains. Continuous lines symbolize interactions of
  high apparent affinity, and broken lines symbolize interactions of low apparent
  affinity. Boxed metabolites appear explicitly in the enzyme equation models. Other
  metabolite concentrations were taken into account but are subsumed in the apparent
  rate constants (see ).
pmcid: PMC2694679
papertitle: Understanding the regulation of aspartate metabolism using a model based
  on measured kinetic parameters.
reftext: Gilles Curien, et al. Mol Syst Biol. 2009;5:271-271.
pmc_ranked_result_index: '149043'
pathway_score: 0.9251071
filename: msb200929-f1.jpg
figtitle: The central enzymes of the Asp-derived amino-acid pathway in chloroplasts
  and allosteric regulation
year: '2009'
organisms:
- Escherichia coli IS5
- Escherichia coli IS1
- Spinacia oleracea
- Arabidopsis thaliana
- Glycine max
- Solanum lycopersicum
- Nicotiana tabacum
- Solanum tuberosum
- Hordeum vulgare
- Saccharum officinarum
- Zea mays
- Saccharomyces cerevisiae
- Escherichia coli
- Trypanosoma brucei
- Lemna aequinoctialis
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2694679__msb200929-f1.html
  '@type': Dataset
  description: The central enzymes of the Asp-derived amino-acid pathway in chloroplasts
    and allosteric regulation. (A) The amino acids Lys, Met, Thr and Ile, and the
    methylating agent AdoMet are synthesized from aspartate. Simple and double arrows
    indicate reactions treated as irreversible and reversible, respectively. Enzyme
    names are indicated in italics. (B) Regulatory map of the Asp-derived amino-acid
    pathway in Arabidopsis leaf mesophyll cell. The 13 enzymes constituting the core
    of the system are co-expressed in Arabidopsis mesophyll cell chloroplasts. Isoforms
    exist for monofunctional AKs (AK1, AK2), bifunctional AKs (AKI–HSDH I and AKII–HSDH
    II) and DHDPS (DHDPS1 and 2). AK3 and TS2 are not expressed in mesophyll cell
    chloroplasts (see ). Genes for ASADH, HSK, CGS and allosteric TD are present as
    single copies in the Arabidopsis genome. Enzymes shown against a white background
    are non-allosteric, and those with a yellow background are allosteric. Bifunctional
    AK–HSDH proteins are symbolized with a linker between the two domains. Continuous
    lines symbolize interactions of high apparent affinity, and broken lines symbolize
    interactions of low apparent affinity. Boxed metabolites appear explicitly in
    the enzyme equation models. Other metabolite concentrations were taken into account
    but are subsumed in the apparent rate constants (see ).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - AKT3
  - AKT1
  - TST
  - THRB
  - THRA
  - WDTC1
  - MET
  - HOGA1
  - AKT2
  - Oxobutyrate
  - Ala
  - Aspartate
  - Pyruvate Aspartate
  - Asp
  - Cysteine
  - Cys
  - Lysine
  - Lys
  - Methionine
  - methionine
  - Met
  - Ser
  - Threonine
  - Thr
  - Val
genes:
- word: AKT
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT3
  entrez: '10000'
- word: AKT
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT1
  entrez: '207'
- word: TST
  symbol: TST
  source: hgnc_symbol
  hgnc_symbol: TST
  entrez: '7263'
- word: Thr
  symbol: THR
  source: bioentities_symbol
  hgnc_symbol: THRB
  entrez: '7068'
- word: Thr
  symbol: THR
  source: bioentities_symbol
  hgnc_symbol: THRA
  entrez: '7067'
- word: ADP
  symbol: ADP
  source: hgnc_alias_symbol
  hgnc_symbol: WDTC1
  entrez: '23038'
- word: (Met)
  symbol: MET
  source: hgnc_symbol
  hgnc_symbol: MET
  entrez: '4233'
- word: DHDPS2
  symbol: DHDPS2
  source: hgnc_alias_symbol
  hgnc_symbol: HOGA1
  entrez: '112817'
- word: AKT
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT2
  entrez: '208'
chemicals:
- word: Oxobutyrate
  source: MESH
  identifier: C016635
- word: Ala
  source: MESH
  identifier: D000409
- word: Aspartate
  source: MESH
  identifier: D001224
- word: Pyruvate Aspartate
  source: MESH
  identifier: D001224
- word: Asp
  source: MESH
  identifier: D001224
- word: Cysteine
  source: MESH
  identifier: D003545
- word: Cys
  source: MESH
  identifier: D003545
- word: Lysine
  source: MESH
  identifier: D008239
- word: Lys
  source: MESH
  identifier: D008239
- word: Methionine
  source: MESH
  identifier: D008715
- word: methionine
  source: MESH
  identifier: D008715
- word: Met
  source: MESH
  identifier: D008715
- word: Ser
  source: MESH
  identifier: D012694
- word: Threonine
  source: MESH
  identifier: D013912
- word: Thr
  source: MESH
  identifier: D013912
- word: Val
  source: MESH
  identifier: D014633
diseases: []
figid_alias: PMC2694679__F1
redirect_from: /figures/PMC2694679__F1
figtype: Figure
---
