---
figid: PMC2912833__1471-2172-11-39-4
figlink: /pmc/articles/PMC2912833/figure/F4/
number: F4
caption: Generation of a biological network of genes of the most significant Canonical
  Pathways for the Il10-/- (EF·CIF vs. C) comparison. The network was generated by
  IPA using all molecules from significantly affected Canonical Pathways (Oxidative
  Phosphorylation; Antigen Presentation Pathway; IL-10 Signaling; Interferon Signaling;
  LPS/IL-1 mediated inhibition of RXR function; Circadian Rhythm Signaling). Connections
  were applied based on known interactions between these genes within the Ingenuity
  Pathways Knowledge Base. Central genes and their direct interactions were identified
  and were supported by published information. *Genes that are detected 2 or more
  times on the array. Genes or gene products are represented as nodes, and the biological
  relationship between two nodes is represented as a line (i.e. an edge). All edges
  are supported by at least 1 reference from the literature. Red and green coloured
  nodes indicate genes with up- and down-regulated expression, respectively. The intensity
  of the colours specifies the degree of up- or down-regulation. Greater intensity
  represents a higher level of differential expression. Nodes and edges are displayed
  with various shapes and labels that present the functional class of genes and the
  nature of the relationships between the nodes, as shown in the legend to Figure
  3.
pmcid: PMC2912833
papertitle: Changes in colon gene expression associated with increased colon inflammation
  in interleukin-10 gene-deficient mice inoculated with Enterococcus species.
reftext: Matthew PG Barnett, et al. BMC Immunol. 2010;11:39-39.
pmc_ranked_result_index: '148335'
pathway_score: 0.7855425
filename: 1471-2172-11-39-4.jpg
figtitle: Generation of a biological network of genes of the most significant Canonical
  Pathways for the Il10-/- (EF·CIF vs
year: '2010'
organisms:
- Mus musculus
- Rattus norvegicus
- Enterococcus faecalis
- Enterococcus faecium
- Sequoia sempervirens
- Armoracia rusticana
- Matthiola incana
- bacterium 1-7A
- Escherichia coli
- Salmonella enterica subsp. enterica serovar Typhimurium
- Homo sapiens
- Bos taurus
ndex: b91a8f6f-df29-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2912833__1471-2172-11-39-4.html
  '@type': Dataset
  description: Generation of a biological network of genes of the most significant
    Canonical Pathways for the Il10-/- (EF·CIF vs. C) comparison. The network was
    generated by IPA using all molecules from significantly affected Canonical Pathways
    (Oxidative Phosphorylation; Antigen Presentation Pathway; IL-10 Signaling; Interferon
    Signaling; LPS/IL-1 mediated inhibition of RXR function; Circadian Rhythm Signaling).
    Connections were applied based on known interactions between these genes within
    the Ingenuity Pathways Knowledge Base. Central genes and their direct interactions
    were identified and were supported by published information. *Genes that are detected
    2 or more times on the array. Genes or gene products are represented as nodes,
    and the biological relationship between two nodes is represented as a line (i.e.
    an edge). All edges are supported by at least 1 reference from the literature.
    Red and green coloured nodes indicate genes with up- and down-regulated expression,
    respectively. The intensity of the colours specifies the degree of up- or down-regulation.
    Greater intensity represents a higher level of differential expression. Nodes
    and edges are displayed with various shapes and labels that present the functional
    class of genes and the nature of the relationships between the nodes, as shown
    in the legend to Figure 3.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TAP2
  - CD14
  - B2M
  - IFNAR2
  - HLA-DRB1
  - STAT1
  - ATF4
  - SLC27A1
  - HLA-DMB
  - HLA-G
  - TNF
  - NR5A2
  - FCGR2A
  - SOCS3
  - NFKBIE
  - IL18
  - ALA
genes:
- word: TAP2
  symbol: TAP2
  source: hgnc_symbol
  hgnc_symbol: TAP2
  entrez: '6891'
- word: CD14
  symbol: CD14
  source: hgnc_symbol
  hgnc_symbol: CD14
  entrez: '929'
- word: B2M
  symbol: B2M
  source: hgnc_symbol
  hgnc_symbol: B2M
  entrez: '567'
- word: IFNAR2
  symbol: IFNAR2
  source: hgnc_symbol
  hgnc_symbol: IFNAR2
  entrez: '3455'
- word: HLA-DRB1
  symbol: HLA-DRB1
  source: hgnc_symbol
  hgnc_symbol: HLA-DRB1
  entrez: '3123'
- word: STAT1
  symbol: STAT1
  source: hgnc_symbol
  hgnc_symbol: STAT1
  entrez: '6772'
- word: ATF4
  symbol: ATF4
  source: hgnc_symbol
  hgnc_symbol: ATF4
  entrez: '468'
- word: SLC27A1
  symbol: SLC27A1
  source: hgnc_symbol
  hgnc_symbol: SLC27A1
  entrez: '376497'
- word: HLA-DMB
  symbol: HLA-DMB
  source: hgnc_symbol
  hgnc_symbol: HLA-DMB
  entrez: '3109'
- word: HLA-G
  symbol: HLA-G
  source: hgnc_symbol
  hgnc_symbol: HLA-G
  entrez: '3135'
- word: TNF
  symbol: TNF
  source: hgnc_symbol
  hgnc_symbol: TNF
  entrez: '7124'
- word: NR5A2
  symbol: NR5A2
  source: hgnc_symbol
  hgnc_symbol: NR5A2
  entrez: '2494'
- word: FCGR2A
  symbol: FCGR2A
  source: hgnc_symbol
  hgnc_symbol: FCGR2A
  entrez: '2212'
- word: Socs3
  symbol: SOCS3
  source: hgnc_symbol
  hgnc_symbol: SOCS3
  entrez: '9021'
- word: NFKBIE
  symbol: NFKBIE
  source: hgnc_symbol
  hgnc_symbol: NFKBIE
  entrez: '4794'
- word: IL18
  symbol: IL18
  source: hgnc_symbol
  hgnc_symbol: IL18
  entrez: '3606'
chemicals:
- word: ALA
  source: MESH
  identifier: D000409
diseases: []
figid_alias: PMC2912833__F4
redirect_from: /figures/PMC2912833__F4
figtype: Figure
---
