---
figid: PMC2430757__nihms-51003-f0002
figlink: /pmc/articles/PMC2430757/figure/F2/
number: F2
caption: A theoretical example of what the informatics tools may look like that will
  make nutrigenomics and metabolomics accessible to nutrition scientists and clinicians.
  After appropriate statistical tests and filtering, data from nutrigenomic, epigenetic,
  and metabolomic platforms can be overlaid on the metabolic map that we are familiar
  with. By selecting an area of the metabolic map, the user can zoom in on the region
  and see a presentation of changes in metabolite concentrations, enzyme activities,
  and perhaps flux rates with color coding to indicate direction of change. Information
  on changes in DNA expression (E), gene methylation (M), and gene SNPs also can be
  presented by using similar color coding. By selecting the metabolite, enzyme, or
  gene box, an investigator would be able to examine the exact values and variances
  and perhaps even to examine the raw data. Another level of the informatics program
  would summarize all of the related changes by pathway and present information on
  false discovery and significant difference statistics. Finally, a summary of appropriate
  diet recommendations would be presented.
pmcid: PMC2430757
papertitle: 'Nutrigenomics and metabolomics will change clinical nutrition and public
  health practice: insights from studies on dietary requirements for choline.'
reftext: Steven H Zeisel. Am J Clin Nutr. ;86(3):542-548.
pmc_ranked_result_index: '108426'
pathway_score: 0.7663347
filename: nihms-51003-f0002.jpg
figtitle: A theoretical example of what the informatics tools may look like that will
  make nutrigenomics and metabolomics accessible to nutrition scientists and clinicians
year: ''
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2430757__nihms-51003-f0002.html
  '@type': Dataset
  description: A theoretical example of what the informatics tools may look like that
    will make nutrigenomics and metabolomics accessible to nutrition scientists and
    clinicians. After appropriate statistical tests and filtering, data from nutrigenomic,
    epigenetic, and metabolomic platforms can be overlaid on the metabolic map that
    we are familiar with. By selecting an area of the metabolic map, the user can
    zoom in on the region and see a presentation of changes in metabolite concentrations,
    enzyme activities, and perhaps flux rates with color coding to indicate direction
    of change. Information on changes in DNA expression (E), gene methylation (M),
    and gene SNPs also can be presented by using similar color coding. By selecting
    the metabolite, enzyme, or gene box, an investigator would be able to examine
    the exact values and variances and perhaps even to examine the raw data. Another
    level of the informatics program would summarize all of the related changes by
    pathway and present information on false discovery and significant difference
    statistics. Finally, a summary of appropriate diet recommendations would be presented.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PAEP
  - 3-phosphoglycerate
  - 2-phosphoglycerate
  - oxaloacetate
genes:
- word: PEP
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
chemicals:
- word: 3-phosphoglycerate
  source: MESH
  identifier: C005156
- word: 2-phosphoglycerate
  source: MESH
  identifier: C008885
- word: oxaloacetate
  source: MESH
  identifier: D010071
diseases: []
figid_alias: PMC2430757__F2
redirect_from: /figures/PMC2430757__F2
figtype: Figure
---
