---
figid: PMC2693436__1471-2148-9-118-1
figlink: /pmc/articles/PMC2693436/figure/F1/
number: F1
caption: Schematic representation of the metabolism of sialic acid among Bacteria.
  Summary of varied pathways of sialic acid utilization in Bacteria. The catabolic
  pathway of sialic acid involves several steps beginning with NanA. Highlighted in
  orange is the donor-scavenging synthesis of sialic acid. Highlighted in pink is
  the de novo pathway for the synthesis of sialic acid. For a more comprehensive review
  of sialic acid utilization see refs. 1 and 29. NanH, Neuraminidase; Neu5Ac, N-acetylneuraminic
  acid, sialic acid; T, sialic acid transporter; NanA, N-acetylneuraminic acid lyase;
  ManNAc, N-acetylmannosamine; NanK, N-acetylmannosamine kinase; ManNAc-6-P, N-acetylmannosamine-6-phosphate;
  NanE, N-acetylmannosamine-6-P epimerase; GlcNAc-6-P, N-acetylglucosamine-6-phosphate;
  NagA, N-acetylglucosamine-6-phosphate deacteylase; GlcN-6-P, Glucosamine-6-phosphate;
  NagB, Glucosamine-6-phosphate deaminase; Fru-6-P, Fructose-6-phosphate; NeuA, CMP-N-acetylneuraminic
  acid synthetase; CMP-NeuAc, CMP-N-acetylneuraminic acid; NeuB, N-acetylneuraminic
  acid synthase; LPS, lipopolysaccharide.
pmcid: PMC2693436
papertitle: Insights into the evolution of sialic acid catabolism among bacteria.
reftext: Salvador Almagro-Moreno, et al. BMC Evol Biol. 2009;9:118-118.
pmc_ranked_result_index: '73680'
pathway_score: 0.927474
filename: 1471-2148-9-118-1.jpg
figtitle: Schematic representation of the metabolism of sialic acid among Bacteria
year: '2009'
organisms:
- Mus musculus
- Rattus norvegicus
- Gemmata obscuriglobus
- Staphylococcus aureus
- Streptococcus gordonii
- Streptococcus sanguinis
- Streptococcus agalactiae
- Streptococcus pneumoniae
- Streptococcus pyogenes
- Lactococcus plantarum
- Escherichia coli K1
- Clostridium botulinum
- Clostridium butyricum
- Clostridioides difficile
- Clostridium perfringens
- Clostridium leptum
- Clostridium tertium
- Staphylococcus aureus subsp. aureus N315
- Latilactobacillus sakei
- Streptococcus pyogenes M1 GAS
- Ligilactobacillus salivarius
- Anaerotruncus colihominis
- Streptococcus pneumoniae TIGR4
- Yersinia pestis KIM10
- Vibrio vulnificus YJ016
- Campylobacter jejuni
- Shigella flexneri 2a str. 301
- Flavobacteriales bacterium
- Histophilus somni 129PT
- Streptococcus agalactiae 2603VR
- Enterocloster bolteae
- Mycoplasma capricolum
- Mycoplasma mycoides
- Lactiplantibacillus plantarum WCFS1
- Vibrio parahaemolyticus RIMD 2210633
- Pseudoalteromonas haloplanktis
- Mycoplasmopsis synoviae 53
- Pasteurella multocida subsp. multocida str. Pm70
- Yersinia pseudotuberculosis IP 32953
- Staphylococcus haemolyticus JCSC1435
- Bacteroides ovatus
- Salmonella enterica
- Clostridium perfringens SM101
- Clostridium scindens
- Photobacterium profundum SS9
- Shigella dysenteriae Sd197
- Shigella boydii Sb227
- Shigella sonnei Ss046
- Aliivibrio fischeri ES114
- Latilactobacillus sakei subsp. sakei 23K
- Pseudoalteromonas translucida TAC125
- Ruminococcus gnavus
- Deuterostomia
- Yersinia frederiksenii ATCC 33641
- Yersinia mollaretii ATCC 43969
- Yersinia bercovieri ATCC 43970
- Ligilactobacillus salivarius UCC118
- Clostridium botulinum C
- Escherichia coli O157:H7 str. Sakai
- Streptococcus sanguinis SK36
- Yersinia enterocolitica subsp. enterocolitica 8081
- Dorea formicigenerans
- Shewanella pealeana ATCC 700345
- Mycoplasma mycoides subsp. capri
- human gut metagenome
- Dorea formicigenerans ATCC 27755
- Dorea longicatena DSM 13814
- Vibrio sp. RS1
- Clostridium botulinum C str. Eklund
- Clostridium perfringens B
- Clostridium perfringens E
- Parabacteroides merdae
- Bacteroides stercoris
- Streptococcus gordonii str. Challis substr. CH1
- Bacteroides caccae
- Neisseria gonorrhoeae
- Neisseria meningitidis
- Streptococcus pneumoniae Hungary19A-6
- Streptococcus pneumoniae CGSP14
- Escherichia coli
- Trichomonas vaginalis
- Proteus mirabilis
- Shigella boydii
- Shigella dysenteriae
- Shigella flexneri
- Shigella sonnei
- Yersinia enterocolitica
- Yersinia pestis
- Vibrio cholerae
- Aliivibrio fischeri
- Vibrio parahaemolyticus
- Vibrio vulnificus
- Yersinia pestis KIM D27
- Shewanella pealeana
- Haemophilus influenzae Rd KW20
- Actinobacillus pleuropneumoniae
- Haemophilus influenzae
- Haemophilus ducreyi
- Histophilus somni
- Photobacterium profundum
- Pasteurella multocida
- Bacteroides fragilis
- Bacteroides uniformis
- Phocaeicola vulgatus
- Parabacteroides distasonis
- Escherichia coli K-12
- Fusobacterium nucleatum
- Faecalibacterium prausnitzii
- Dorea longicatena
- Gallus gallus
- Salmonella enterica subsp. enterica serovar Typhi
- Salmonella enterica subsp. enterica serovar Typhimurium
- Ornithorhynchus anatinus
- Homo sapiens
- Sus scrofa
- Salmonella enterica subsp. enterica serovar Typhimurium str. LT2
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2693436__1471-2148-9-118-1.html
  '@type': Dataset
  description: Schematic representation of the metabolism of sialic acid among Bacteria.
    Summary of varied pathways of sialic acid utilization in Bacteria. The catabolic
    pathway of sialic acid involves several steps beginning with NanA. Highlighted
    in orange is the donor-scavenging synthesis of sialic acid. Highlighted in pink
    is the de novo pathway for the synthesis of sialic acid. For a more comprehensive
    review of sialic acid utilization see refs. 1 and 29. NanH, Neuraminidase; Neu5Ac,
    N-acetylneuraminic acid, sialic acid; T, sialic acid transporter; NanA, N-acetylneuraminic
    acid lyase; ManNAc, N-acetylmannosamine; NanK, N-acetylmannosamine kinase; ManNAc-6-P,
    N-acetylmannosamine-6-phosphate; NanE, N-acetylmannosamine-6-P epimerase; GlcNAc-6-P,
    N-acetylglucosamine-6-phosphate; NagA, N-acetylglucosamine-6-phosphate deacteylase;
    GlcN-6-P, Glucosamine-6-phosphate; NagB, Glucosamine-6-phosphate deaminase; Fru-6-P,
    Fructose-6-phosphate; NeuA, CMP-N-acetylneuraminic acid synthetase; CMP-NeuAc,
    CMP-N-acetylneuraminic acid; NeuB, N-acetylneuraminic acid synthase; LPS, lipopolysaccharide.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - IRF6
  - NAGA
  - ManNAc-6-P
  - CMP-Neu5Ac
  - UDP
genes:
- word: CPS/LPS
  symbol: LPS
  source: hgnc_prev_symbol
  hgnc_symbol: IRF6
  entrez: '3664'
- word: NagA
  symbol: NAGA
  source: hgnc_symbol
  hgnc_symbol: NAGA
  entrez: '4668'
chemicals:
- word: ManNAc-6-P
  source: MESH
  identifier: C430285
- word: CMP-Neu5Ac
  source: MESH
  identifier: C561601
- word: UDP
  source: MESH
  identifier: D014530
diseases: []
figid_alias: PMC2693436__F1
redirect_from: /figures/PMC2693436__F1
figtype: Figure
---
