---
figid: PMC2996932__1471-2164-11-404-4
figlink: /pmc/articles/PMC2996932/figure/F4/
number: F4
caption: 'Transcriptional evidence for metabolic differences between sessile (biofilm)
  and swimming (planktonic) modes of life. Genes encoding mixed acid fermentation
  enzymes are up-regulated in the biofilm (bottom part). As a result, two metabolites
  can be produced: acetyl-phosphate (acP) and acetate. The former can be used as a
  phospho-donor intermediate, while the later can be transformed to acetyl-CoA or
  released to the medium. They could be produced by fermentation of sugars via the
  Embden-Meyerhof-Parnas pathway to pyruvate, then to acetyl phosphate (acP) by pyruvate
  oxidase (PoxB), and finally to acetate by AckA. In contrast, genes encoding key
  TCA cycle enzymes are up-regulated in the swimming (planktonic) cells (top part;
  see also text for explanations). Circles indicate the name of the enzymes whose
  genes are being expressed in the Río Tinto samples (red, preferentially induced
  in biofilm; green, in planktonic cells; and yellow transcribed in both places).
  All of them have been identified in the L. ferrooxidans genome sequence draft (to
  be published elsewhere; see Additional file  Table S2, and Additional file ). POR,
  pyruvate oxidoreductase system; Pta, phosphotransacetylase; Idh, NAD+ dependent
  isocitrate dehydrogenase; Acn, Aconitase; Mdh, Malate dehydrogenase; FumC, fumarate
  hydratase component C; SdhA, succinate dehydrogenase component A; SucCD, Succinate
  dehydrogenase; PpsA, phosphoenolpyruvate synthase; AccACD, acetyl-CoA carboxylase.'
pmcid: PMC2996932
papertitle: Environmental transcriptome analysis reveals physiological differences
  between biofilm and planktonic modes of life of the iron oxidizing bacteria Leptospirillum
  spp. in their natural microbial community.
reftext: Mercedes Moreno-Paz, et al. BMC Genomics. 2010;11:404-404.
pmc_ranked_result_index: '98932'
pathway_score: 0.945357
filename: 1471-2164-11-404-4.jpg
figtitle: Transcriptional evidence for metabolic differences between sessile (biofilm)
  and swimming (planktonic) modes of life
year: '2010'
organisms:
- Octodon degus
- environmental samples
- Bacillus subtilis
- Lactiplantibacillus plantarum
- Listeria monocytogenes
- Leptospirillum ferriphilum
- Leptospirillum ferrooxidans
- Bermanella marisrubri
- Pseudomonas aeruginosa
- Acidithiobacillus caldus
- Xanthomonas campestris
- Leptospirillum ferrodiazotrophum
- Leptospirillum rubarum
- Hordeum vulgare
- Acidimicrobium ferrooxidans
- Escherichia coli
- Salmonella sp.
- Escherichia coli K-12
- Acidithiobacillus ferrooxidans
- Acidithiobacillus thiooxidans
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC2996932__1471-2164-11-404-4.html
  '@type': Dataset
  description: 'Transcriptional evidence for metabolic differences between sessile
    (biofilm) and swimming (planktonic) modes of life. Genes encoding mixed acid fermentation
    enzymes are up-regulated in the biofilm (bottom part). As a result, two metabolites
    can be produced: acetyl-phosphate (acP) and acetate. The former can be used as
    a phospho-donor intermediate, while the later can be transformed to acetyl-CoA
    or released to the medium. They could be produced by fermentation of sugars via
    the Embden-Meyerhof-Parnas pathway to pyruvate, then to acetyl phosphate (acP)
    by pyruvate oxidase (PoxB), and finally to acetate by AckA. In contrast, genes
    encoding key TCA cycle enzymes are up-regulated in the swimming (planktonic) cells
    (top part; see also text for explanations). Circles indicate the name of the enzymes
    whose genes are being expressed in the Río Tinto samples (red, preferentially
    induced in biofilm; green, in planktonic cells; and yellow transcribed in both
    places). All of them have been identified in the L. ferrooxidans genome sequence
    draft (to be published elsewhere; see Additional file  Table S2, and Additional
    file ). POR, pyruvate oxidoreductase system; Pta, phosphotransacetylase; Idh,
    NAD+ dependent isocitrate dehydrogenase; Acn, Aconitase; Mdh, Malate dehydrogenase;
    FumC, fumarate hydratase component C; SdhA, succinate dehydrogenase component
    A; SucCD, Succinate dehydrogenase; PpsA, phosphoenolpyruvate synthase; AccACD,
    acetyl-CoA carboxylase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SDHA
  - ACSS2
  - PTCRA
  - PAEP
  - POR
  - NDUFAB1
  - NF2
  - WDTC1
  - Succinyl-CoA
  - Acetate
  - Fatty acids
  - Glucose
  - Isocitrate
  - Malonyl-CoA
  - Pyruvate
  - Succinate
genes:
- word: SdhA)
  symbol: SDHA
  source: hgnc_symbol
  hgnc_symbol: SDHA
  entrez: '6389'
- word: ACSA
  symbol: ACSA
  source: hgnc_alias_symbol
  hgnc_symbol: ACSS2
  entrez: '55902'
- word: Pta
  symbol: PT-ALPHA
  source: hgnc_alias_symbol
  hgnc_symbol: PTCRA
  entrez: '171558'
- word: PEP
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
- word: POR
  symbol: POR
  source: hgnc_symbol
  hgnc_symbol: POR
  entrez: '5447'
- word: acP
  symbol: ACP
  source: hgnc_alias_symbol
  hgnc_symbol: NDUFAB1
  entrez: '4706'
- word: Acn
  symbol: ACN
  source: hgnc_alias_symbol
  hgnc_symbol: NF2
  entrez: '4771'
- word: ADP
  symbol: ADP
  source: hgnc_alias_symbol
  hgnc_symbol: WDTC1
  entrez: '23038'
chemicals:
- word: Succinyl-CoA
  source: MESH
  identifier: C012046
- word: Acetate
  source: MESH
  identifier: D000085
- word: Fatty acids
  source: MESH
  identifier: D005227
- word: Glucose
  source: MESH
  identifier: D005947
- word: Isocitrate
  source: MESH
  identifier: D007523
- word: Malonyl-CoA
  source: MESH
  identifier: D008316
- word: Pyruvate
  source: MESH
  identifier: D011773
- word: Succinate
  source: MESH
  identifier: D013386
diseases: []
figid_alias: PMC2996932__F4
redirect_from: /figures/PMC2996932__F4
figtype: Figure
---
