---
figid: PMC3840736__1471-2164-14-781-2
figlink: /pmc/articles/PMC3840736/figure/F2/
number: F2
caption: Summary diagram of pathways related to carotenoid metabolism (MVA, MEP, carotenoid
  biosynthetic and catabolism pathways). See below for legend. 1. ACAT, acetyl-CoA
  acetyltransferase; 2. HMGS, 3-hydroxy-3-methylglutaryl-CoA synthase; 3. HMGR, 3-hydroxy-3-methylglutaryl-CoA
  reductase; 4. MVK, mevalonate kinase; 5. PMK, phosphomevalonate kinase; 6. PMD,
  diphosphomevalonate decarboxylase; 7. IDI, isopentenyl-diphosphate δ-isomerase;
  8. GPS, geranyl-diphosphate synthase; 9. FPS, farnesyl diphosphate synthase; 10.
  DXS, 1-deoxy-d-xylulose-5-phosphate synthase; 11. DXR, 1-deoxy-d-xylulose-5-phosphate
  reductoisomerase; 12. MCT, 2-C-methyl-d-erythritol-4-phosphate cytidyltransferase;
  13. CMK, 4-diphosphocytidyl-2-C-methyl-d-erythritol kinase; 14. MDS, 2-C-methyl-d-erythritol
  2,4-cyclodiphosphate synthase; 15. HDS, 4-hydroxy-3-methylbut-2-en-1-yl diphosphate
  synthase; 16. HDR, 4-hydroxy-3-methylbut-2-enyl diphosphate reductase; 17. GGPS,
  geranylgeranyl diphosphate synthase; 18. KSB, ent-kaur-16-ene synthase; 19. GGPR,
  geranylgeranyl diphosphate reductase; 20. PREA, prenyl transferase; 21. PSY, phytoene
  synthase; 22. PDS, phytoene desaturase; 23. Z-ISO, ζ-carotene isomerase; 24. ZDS,
  ζ-carotene desaturase; 25. CRTISO, carotene isomerase; 26. LCYE, lycopene ϵ-cyclase;
  27. LCYB, lycopene β-cyclase; 28. CHYB, β-carotene hydroxylase; 29. CHYE, ϵ-carotene
  hydroxylase; 30. ZEP, zeaxanthin epoxidase; 31. VDE, violaxanthin de-epoxidase;
  32. NXS, neoxanthin synthase; 33. XISO, xanthophyll isomerase; 34. NCED, 9-cis-epoxycarotenoid
  dioxygenase; 35. SDR, short chain dehydrogenase; 36. AAO-MoCo, abscisic aldehyde
  oxidase-molybdenum co-factor. In the diagram, the enzymes with down-regulated gene
  expression level are indicated in red, the enzymes with up-regulated gene expression
  level in blue, while the enzymes with gene expression level that increases in the
  intermediate stages and decreases in the last stage in green. The asterisks identify
  enzymes encoded by several gene isoforms; the gene expression level of the predominant
  isoform is shown in the diagram.
pmcid: PMC3840736
papertitle: Comparative genomics reveals candidate carotenoid pathway regulators of
  ripening watermelon fruit.
reftext: Stefania Grassi, et al. BMC Genomics. 2013;14:781-781.
pmc_ranked_result_index: '201966'
pathway_score: 0.9274831
filename: 1471-2164-14-781-2.jpg
figtitle: Summary diagram of pathways related to carotenoid metabolism (MVA, MEP,
  carotenoid biosynthetic and catabolism pathways)
year: '2013'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3840736__1471-2164-14-781-2.html
  '@type': Dataset
  description: Summary diagram of pathways related to carotenoid metabolism (MVA,
    MEP, carotenoid biosynthetic and catabolism pathways). See below for legend. 1.
    ACAT, acetyl-CoA acetyltransferase; 2. HMGS, 3-hydroxy-3-methylglutaryl-CoA synthase;
    3. HMGR, 3-hydroxy-3-methylglutaryl-CoA reductase; 4. MVK, mevalonate kinase;
    5. PMK, phosphomevalonate kinase; 6. PMD, diphosphomevalonate decarboxylase; 7.
    IDI, isopentenyl-diphosphate δ-isomerase; 8. GPS, geranyl-diphosphate synthase;
    9. FPS, farnesyl diphosphate synthase; 10. DXS, 1-deoxy-d-xylulose-5-phosphate
    synthase; 11. DXR, 1-deoxy-d-xylulose-5-phosphate reductoisomerase; 12. MCT, 2-C-methyl-d-erythritol-4-phosphate
    cytidyltransferase; 13. CMK, 4-diphosphocytidyl-2-C-methyl-d-erythritol kinase;
    14. MDS, 2-C-methyl-d-erythritol 2,4-cyclodiphosphate synthase; 15. HDS, 4-hydroxy-3-methylbut-2-en-1-yl
    diphosphate synthase; 16. HDR, 4-hydroxy-3-methylbut-2-enyl diphosphate reductase;
    17. GGPS, geranylgeranyl diphosphate synthase; 18. KSB, ent-kaur-16-ene synthase;
    19. GGPR, geranylgeranyl diphosphate reductase; 20. PREA, prenyl transferase;
    21. PSY, phytoene synthase; 22. PDS, phytoene desaturase; 23. Z-ISO, ζ-carotene
    isomerase; 24. ZDS, ζ-carotene desaturase; 25. CRTISO, carotene isomerase; 26.
    LCYE, lycopene ϵ-cyclase; 27. LCYB, lycopene β-cyclase; 28. CHYB, β-carotene hydroxylase;
    29. CHYE, ϵ-carotene hydroxylase; 30. ZEP, zeaxanthin epoxidase; 31. VDE, violaxanthin
    de-epoxidase; 32. NXS, neoxanthin synthase; 33. XISO, xanthophyll isomerase; 34.
    NCED, 9-cis-epoxycarotenoid dioxygenase; 35. SDR, short chain dehydrogenase; 36.
    AAO-MoCo, abscisic aldehyde oxidase-molybdenum co-factor. In the diagram, the
    enzymes with down-regulated gene expression level are indicated in red, the enzymes
    with up-regulated gene expression level in blue, while the enzymes with gene expression
    level that increases in the intermediate stages and decreases in the last stage
    in green. The asterisks identify enzymes encoded by several gene isoforms; the
    gene expression level of the predominant isoform is shown in the diagram.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SLC26A4
  - GRHL1
  - IPP
  - FES
  - PMVK
  - PAFAH1B1
  - GATA3
  - geranylgeranyl diphosphate
  - 9,15-cis-phytofluene
  - farnesyl diphosphate
  - isopentenyl-diphosphate
  - 9-cis-violaxanthin
  - violaxanthin
  - 9'-tri-cis-neurosporene
  - 9-cis-neoxanthin
  - neoxanthin
  - xanthoxin
  - 3-hydroxy-3-methyl-glutaryl-CoA
  - antheraxanthin
  - dimethylallyl diphosphate
  - mevalonate 5-phosphate
  - mevalonate 5-diphosphate
  - 15-cis-phytoene
  - 2-C-methyl-D-erythritol 4-phosphate
  - 4-diphosphocytidyl-2-C-methyl-D-erythritol
  - 2-C-methyl-D-erythritol 2,4-cyclodiphosphate
  - zeinoxanthin
  - geranyl diphosphate
  - phytyl diphosphate
  - abscissic acid
  - aldehyde
  - 15,9'-tri-cis-C-carotene
  - 9'-di-cis--carotene
  - carotene
  - o-carotene
  - chlorophylls
  - gibberellins
  - D-glyceraldehyde-3-phosphate pyruvate
  - mevalonate
  - plastoquinones
  - sesquiterpenes
  - sterols
  - triterpenes
  - lutein
  - tocopherols
  - ent-kaurene
  - zeaxanthin
genes:
- word: 22|PDS
  symbol: PDS
  source: hgnc_alias_symbol
  hgnc_symbol: SLC26A4
  entrez: '5172'
- word: НMGR
  symbol: MGR
  source: hgnc_alias_symbol
  hgnc_symbol: GRHL1
  entrez: '29841'
- word: IPP
  symbol: IPP
  source: hgnc_symbol
  hgnc_symbol: IPP
  entrez: '3652'
- word: FPS*
  symbol: FPS
  source: hgnc_alias_symbol
  hgnc_symbol: FES
  entrez: '2242'
- word: PMK
  symbol: PMK
  source: hgnc_alias_symbol
  hgnc_symbol: PMVK
  entrez: '10654'
- word: MDS
  symbol: MDS
  source: hgnc_prev_symbol
  hgnc_symbol: PAFAH1B1
  entrez: '5048'
- word: 16|HDR*
  symbol: HDR
  source: hgnc_alias_symbol
  hgnc_symbol: GATA3
  entrez: '2625'
chemicals:
- word: geranylgeranyl diphosphate
  source: MESH
  identifier: C002963
- word: 9,15-cis-phytofluene
  source: MESH
  identifier: C003140
- word: farnesyl diphosphate
  source: MESH
  identifier: C004808
- word: isopentenyl-diphosphate
  source: MESH
  identifier: C004809
- word: 9-cis-violaxanthin
  source: MESH
  identifier: C005613
- word: violaxanthin
  source: MESH
  identifier: C005613
- word: 9'-tri-cis-neurosporene
  source: MESH
  identifier: C008604
- word: 9-cis-neoxanthin
  source: MESH
  identifier: C011947
- word: neoxanthin
  source: MESH
  identifier: C011947
- word: xanthoxin
  source: MESH
  identifier: C012475
- word: 3-hydroxy-3-methyl-glutaryl-CoA
  source: MESH
  identifier: C015901
- word: antheraxanthin
  source: MESH
  identifier: C031140
- word: dimethylallyl diphosphate
  source: MESH
  identifier: C043060
- word: mevalonate 5-phosphate
  source: MESH
  identifier: C045038
- word: mevalonate 5-diphosphate
  source: MESH
  identifier: C046285
- word: 15-cis-phytoene
  source: MESH
  identifier: C100185
- word: 2-C-methyl-D-erythritol 4-phosphate
  source: MESH
  identifier: C114232
- word: 4-diphosphocytidyl-2-C-methyl-D-erythritol
  source: MESH
  identifier: C400182
- word: 2-C-methyl-D-erythritol 2,4-cyclodiphosphate
  source: MESH
  identifier: C443879
- word: zeinoxanthin
  source: MESH
  identifier: C503770
- word: geranyl diphosphate
  source: MESH
  identifier: C511282
- word: phytyl diphosphate
  source: MESH
  identifier: C529546
- word: abscissic acid
  source: MESH
  identifier: D000040
- word: aldehyde
  source: MESH
  identifier: D000447
- word: 15,9'-tri-cis-C-carotene
  source: MESH
  identifier: D002338
- word: 9'-di-cis--carotene
  source: MESH
  identifier: D002338
- word: carotene
  source: MESH
  identifier: D002338
- word: o-carotene
  source: MESH
  identifier: D002338
- word: chlorophylls
  source: MESH
  identifier: D002734
- word: gibberellins
  source: MESH
  identifier: D005875
- word: D-glyceraldehyde-3-phosphate pyruvate
  source: MESH
  identifier: D005986
- word: mevalonate
  source: MESH
  identifier: D008798
- word: plastoquinones
  source: MESH
  identifier: D010971
- word: sesquiterpenes
  source: MESH
  identifier: D012717
- word: sterols
  source: MESH
  identifier: D013261
- word: triterpenes
  source: MESH
  identifier: D014315
- word: lutein
  source: MESH
  identifier: D014975
- word: tocopherols
  source: MESH
  identifier: D024505
- word: ent-kaurene
  source: MESH
  identifier: D045786
- word: zeaxanthin
  source: MESH
  identifier: D065146
diseases: []
figid_alias: PMC3840736__F2
redirect_from: /figures/PMC3840736__F2
figtype: Figure
---
