---
figid: PMC3916724__fpls-05-00028-g001
figlink: /pmc/articles/PMC3916724/figure/F1/
number: FIGURE 1
caption: Molybdenum metabolism in higher plant cells. The main components of molybdenum
  metabolism in plants are shown including the Moco biosynthetic pathway (CNX proteins)
  in mitochondria and cytosol, the Moco user enzymes and their respective main functions
  in nitrogen assimilation (NR), ABA synthesis (AAO3), purine catabolism (XDH1), and
  sulfite detoxification (SO). mARC enzymes are proposed to function in reduction
  of certain N-hydroxylated substrates, which have not yet been identified. While
  one of the two mARC isoforms (mARC2) contains an NH2-terminal mitochondrial targeting
  sequence, such targeting sequence is absent at the second isoform, which therefore
  is assumed to act in the cytosol. In contrast to the molybdate transporter MOT2,
  which functions at the vacuolar membrane as a molybdate exporter, MOT1 might localize
  to the endomembrane system, possibly to the endoplasmic reticulum. A mitochondrial
  localization of MOT1 has also been reported but appears less likely as no obvious
  reason exists for import or export of molybdate into or out of the mitochondria,
  respectively. The plant homolog (MFS-MOT) of the major facilitator superfamily molybdate
  transporter MOT2 from Chlamydomonas might be required for molybdate import across
  the plasma membrane. The function of the Moco sulfurase ABA3 in activation of AO
  and XDH is indicated, just as the functions of the mitochondrial ABC transporter
  ATM3 in export of cPMP from mitochondria and in cytosolic iron-sulfur cluster ([Fe-S])
  assembly for AO and XDH (and other extra-mitochondrial proteins). Further details
  are given in the main text and are reviewed by  and . Molybdo-enzymes are indicated
  by blue letters, other components of molybdenum metabolism by orange letters; dotted
  arrows indicate requirement for Moco by molybdo-enzymes.
pmcid: PMC3916724
papertitle: Molybdenum metabolism in plants and crosstalk to iron.
reftext: Florian Bittner. Front Plant Sci. 2014;5:28.
pmc_ranked_result_index: '92130'
pathway_score: 0.9410256
filename: fpls-05-00028-g001.jpg
figtitle: Molybdenum metabolism in higher plant cells
year: '2014'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3916724__fpls-05-00028-g001.html
  '@type': Dataset
  description: Molybdenum metabolism in higher plant cells. The main components of
    molybdenum metabolism in plants are shown including the Moco biosynthetic pathway
    (CNX proteins) in mitochondria and cytosol, the Moco user enzymes and their respective
    main functions in nitrogen assimilation (NR), ABA synthesis (AAO3), purine catabolism
    (XDH1), and sulfite detoxification (SO). mARC enzymes are proposed to function
    in reduction of certain N-hydroxylated substrates, which have not yet been identified.
    While one of the two mARC isoforms (mARC2) contains an NH2-terminal mitochondrial
    targeting sequence, such targeting sequence is absent at the second isoform, which
    therefore is assumed to act in the cytosol. In contrast to the molybdate transporter
    MOT2, which functions at the vacuolar membrane as a molybdate exporter, MOT1 might
    localize to the endomembrane system, possibly to the endoplasmic reticulum. A
    mitochondrial localization of MOT1 has also been reported but appears less likely
    as no obvious reason exists for import or export of molybdate into or out of the
    mitochondria, respectively. The plant homolog (MFS-MOT) of the major facilitator
    superfamily molybdate transporter MOT2 from Chlamydomonas might be required for
    molybdate import across the plasma membrane. The function of the Moco sulfurase
    ABA3 in activation of AO and XDH is indicated, just as the functions of the mitochondrial
    ABC transporter ATM3 in export of cPMP from mitochondria and in cytosolic iron-sulfur
    cluster ([Fe-S]) assembly for AO and XDH (and other extra-mitochondrial proteins).
    Further details are given in the main text and are reviewed by  and . Molybdo-enzymes
    are indicated by blue letters, other components of molybdenum metabolism by orange
    letters; dotted arrows indicate requirement for Moco by molybdo-enzymes.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - FES
  - XDH
  - BTAF1
  - CCL7
  - HSPA9
  - 9-cis-Violaxanthin
  - Violaxanthin
  - 9-cis-Neoxanthin
  - Neoxanthin
  - Xanthoxin
  - Antheraxanthin
  - Abscisic acid
  - aldehyde
  - Purine nucleotide
  - Hypoxanthine
  - Xanthine
  - Zeaxanthin
genes:
- word: (Fe-S)
  symbol: FES
  source: hgnc_symbol
  hgnc_symbol: FES
  entrez: '2242'
- word: XDH
  symbol: XDH
  source: hgnc_symbol
  hgnc_symbol: XDH
  entrez: '7498'
- word: MOT1
  symbol: MOT1
  source: hgnc_alias_symbol
  hgnc_symbol: BTAF1
  entrez: '9044'
- word: (m)ARC
  symbol: MARC
  source: hgnc_alias_symbol
  hgnc_symbol: CCL7
  entrez: '6354'
- word: MOT2
  symbol: mot-2
  source: hgnc_alias_symbol
  hgnc_symbol: HSPA9
  entrez: '3313'
chemicals:
- word: 9-cis-Violaxanthin
  source: MESH
  identifier: C005613
- word: Violaxanthin
  source: MESH
  identifier: C005613
- word: 9-cis-Neoxanthin
  source: MESH
  identifier: C011947
- word: Neoxanthin
  source: MESH
  identifier: C011947
- word: Xanthoxin
  source: MESH
  identifier: C012475
- word: Antheraxanthin
  source: MESH
  identifier: C031140
- word: Abscisic acid
  source: MESH
  identifier: D000040
- word: aldehyde
  source: MESH
  identifier: D000447
- word: Purine nucleotide
  source: MESH
  identifier: D011685
- word: Hypoxanthine
  source: MESH
  identifier: D019271
- word: Xanthine
  source: MESH
  identifier: D019820
- word: Zeaxanthin
  source: MESH
  identifier: D065146
diseases: []
---
