---
figid: PMC3013041__zbc0051145520005
figlink: /pmc/articles/PMC3013041/figure/F5/
number: F5
caption: 'Relationship between transcription factors and their regulatory factors.
  Transcription factors (TF, rectangles) show different motif activity scores between
  WT and TamR cells and their regulatory genes (triangles), and signal mediators (circles)
  are visualized as a network (A, E2; B, HRG). Node color represents phosphorylation
  and TF activation differences between the two cell lines: red, higher activity in
  TamR than in WT; blue, higher activity in WT than in TamR; green, phosphorylation
  change was observed in only one cell or down-regulation only was observed; gray,
  no change. The edges were color-coded according to TF and regulator activation level
  in WT and TamR: red, higher activation of TFs and their regulators in TamR than
  in WT; blue, higher activation of TFs and their regulators in WT than in TamR; black,
  others. TFs with differential motif activity were inferred from gene expression
  data (see the supplemental materials for detail), and then factors regulating the
  predicted TFs were investigated by the KEGG pathway, PhosphoSitePlus, and NetworKIN
  databases. Relationships extracted from KEGG and PhosphoSitePlus are shown.'
pmcid: PMC3013041
papertitle: Integrated Quantitative Analysis of the Phosphoproteome and Transcriptome
  in Tamoxifen-resistant Breast Cancer.
reftext: Masaaki Oyama, et al. J Biol Chem. 2011 Jan 7;286(1):818-829.
pmc_ranked_result_index: '42511'
pathway_score: 0.9197988
filename: zbc0051145520005.jpg
figtitle: Relationship between transcription factors and their regulatory factors
year: '2011'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3013041__zbc0051145520005.html
  '@type': Dataset
  description: 'Relationship between transcription factors and their regulatory factors.
    Transcription factors (TF, rectangles) show different motif activity scores between
    WT and TamR cells and their regulatory genes (triangles), and signal mediators
    (circles) are visualized as a network (A, E2; B, HRG). Node color represents phosphorylation
    and TF activation differences between the two cell lines: red, higher activity
    in TamR than in WT; blue, higher activity in WT than in TamR; green, phosphorylation
    change was observed in only one cell or down-regulation only was observed; gray,
    no change. The edges were color-coded according to TF and regulator activation
    level in WT and TamR: red, higher activation of TFs and their regulators in TamR
    than in WT; blue, higher activation of TFs and their regulators in WT than in
    TamR; black, others. TFs with differential motif activity were inferred from gene
    expression data (see the supplemental materials for detail), and then factors
    regulating the predicted TFs were investigated by the KEGG pathway, PhosphoSitePlus,
    and NetworKIN databases. Relationships extracted from KEGG and PhosphoSitePlus
    are shown.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TF
  - ELF4
  - GSK3B
  - JUNB
  - JUND
  - ELK4
genes:
- word: TF
  symbol: TF
  source: hgnc_symbol
  hgnc_symbol: TF
  entrez: '7018'
- word: ELF4
  symbol: ELF4
  source: hgnc_symbol
  hgnc_symbol: ELF4
  entrez: '2000'
- word: GSK3B
  symbol: GSK3B
  source: hgnc_symbol
  hgnc_symbol: GSK3B
  entrez: '2932'
- word: JUNB
  symbol: JUNB
  source: hgnc_symbol
  hgnc_symbol: JUNB
  entrez: '3726'
- word: JUND
  symbol: JUND
  source: hgnc_symbol
  hgnc_symbol: JUND
  entrez: '3727'
- word: ELK4
  symbol: ELK4
  source: hgnc_symbol
  hgnc_symbol: ELK4
  entrez: '2005'
chemicals: []
diseases: []
figid_alias: PMC3013041__F5
redirect_from: /figures/PMC3013041__F5
figtype: Figure
---
