---
figid: PMC3002239__molbiolevolmsq204f01_3c
figlink: /pmc/articles/PMC3002239/figure/fig1/
number: 'F <span style="font-variant: small-caps" class="small-caps">IG</span> . 1'
caption: Metabolic pathways in 35 metabolites in Arabidopsis. (A) Open circles and
  dots denote chemical compounds and enzyme reactions, respectively. The chemical
  compounds of 17 amino acids (red circles) were quantified in this study. All metabolic
  pathways synthesized by less than five enzyme reactions from the 17 amino acids
  are shown in this figure. All the pathways were collected from the TAIR Aracyc database
  (version 5.0). As small molecules, including water, O2, ATP, NADP, NADPH, polyphosphates,
  CO2, H+, phosphates, ADP, UDP, Co-A, NAD, NADH, AMP, and Acetyl-CoA, are involved
  in many reactions or are used as carriers for transferring electrons, they were
  excluded from this map. (B) As an example of the structure of a metabolic pathway,
  two reactions as a minimum metabolic pathway from Gly to Trp are shown. (C) Metabolic
  pathway of 18 secondary metabolites in Arabidopsis. The chemical compounds shown
  in red were quantified in this study. Abbreviation of each metabolite is shown in
  supplementary table S1, Supplementary Material online. The metabolic pathways are
  modified from .
pmcid: PMC3002239
papertitle: Functional Compensation of Primary and Secondary Metabolites by Duplicate
  Genes in Arabidopsis thaliana.
reftext: Kousuke Hanada, et al. Mol Biol Evol. 2011 Jan;28(1):377-382.
pmc_ranked_result_index: '28773'
pathway_score: 0.8732076
filename: molbiolevolmsq204f01_3c.jpg
figtitle: Functional Compensation of Primary and Secondary Metabolites by Duplicate
  Genes in Arabidopsis thaliana
year: '2011'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3002239__molbiolevolmsq204f01_3c.html
  '@type': Dataset
  description: Metabolic pathways in 35 metabolites in Arabidopsis. (A) Open circles
    and dots denote chemical compounds and enzyme reactions, respectively. The chemical
    compounds of 17 amino acids (red circles) were quantified in this study. All metabolic
    pathways synthesized by less than five enzyme reactions from the 17 amino acids
    are shown in this figure. All the pathways were collected from the TAIR Aracyc
    database (version 5.0). As small molecules, including water, O2, ATP, NADP, NADPH,
    polyphosphates, CO2, H+, phosphates, ADP, UDP, Co-A, NAD, NADH, AMP, and Acetyl-CoA,
    are involved in many reactions or are used as carriers for transferring electrons,
    they were excluded from this map. (B) As an example of the structure of a metabolic
    pathway, two reactions as a minimum metabolic pathway from Gly to Trp are shown.
    (C) Metabolic pathway of 18 secondary metabolites in Arabidopsis. The chemical
    compounds shown in red were quantified in this study. Abbreviation of each metabolite
    is shown in supplementary table S1, Supplementary Material online. The metabolic
    pathways are modified from .
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TYRP1
  - PTPN5
  - indole-3-glycerol-phosphate
  - 4MTB
  - D-glyceraldehyde-3-phosphate
  - glyoxylate
  - Ser
  - Trp
genes:
- word: Trp
  symbol: TRP
  source: hgnc_alias_symbol
  hgnc_symbol: TYRP1
  entrez: '7306'
- word: steps
  symbol: STEP
  source: hgnc_alias_symbol
  hgnc_symbol: PTPN5
  entrez: '84867'
chemicals:
- word: indole-3-glycerol-phosphate
  source: MESH
  identifier: C008117
- word: 4MTB
  source: MESH
  identifier: C066074
- word: D-glyceraldehyde-3-phosphate
  source: MESH
  identifier: D005986
- word: glyoxylate
  source: MESH
  identifier: D006038
- word: Ser
  source: MESH
  identifier: D012694
- word: Trp
  source: MESH
  identifier: D014364
diseases: []
---
