---
figid: PMC3787301__elife01102f004
figlink: /pmc/articles/PMC3787301/figure/fig4/
number: Figure 4
caption: 'Metabolic predictions for Melainabacteria based on genes identified in .
  Genes in pathways detected in the genomes of the subsurface and at least one gut
  genome (white box), only in the subsurface genome (grey box), only in at least one
  gut genome (orange box), and genes missing from pathways in all genomes (red box).
  Glycolysis proceeds via the canonical Embden-Meyerhof-Parnas (EMP) pathway with
  the exception of fructose-6-phosphate 1-phosphotransferase (EC:2.7.1.90, gene 3).
  Names of pathways and fermentation end-products are bolded and ATP generated by
  substrate-level phosphorylation are noted. All Melainabacteria genomes sampled lack
  electron transport chain components (including cytochromes (Cyto), succinate dehydrogenase
  (sdh), flavins, quinones), terminal respiratory oxidases or reductases, and photosystem
  I or II (PS1, PS2). The genomes also lack a complete TCA cycle (absent enzymes noted
  by red boxes), with the TCA enzymes instead linked to the fermentation of amino
  acids and organic acids denoted (pathways, blue arrows). Ferredoxin (Fd, green text)
  is important for hydrogen (H2) production via hydrogenases (yellow background box).
  Proton translocation mechanisms (green background box) may be achieved by the activity
  of trimeric oxaloacetate (OAA) decarboxylase and sodium-hydrogen antiporter, pyrophosphate
  (PPi) hydrolysis with pyrophosphatases, 11 subunit NADH dehydrogenase, and an annotated
  NiFe hydrogenase (green enzyme). Annotations for the gene numbers are in . The complete
  metabolic comparison of the Melainabacteria can be accessed at http://ggkbase.berkeley.edu/genome_summaries/81-MEL-Metabolic-Overview-June2013.DOI:
  http://dx.doi.org/10.7554/eLife.01102.01010.7554/eLife.01102.011Figure 4—source
  data 1.Examination of enzymes (steps) in near-complete KEGG based modules shared
  among or unique to subsurface ACD20 and gut Melainabacteria genomes MEL.A1, MEL.B1,
  and MEL.B2.Analysis is based on the KEGG Module database ().DOI: http://dx.doi.org/10.7554/eLife.01102.01110.7554/eLife.01102.012Figure
  4—source data 2.Gene annotations corresponding to the numbers in .If the gene occurs
  in both the ACD20 and gut genomes, the reported annotation is based on ACD20.DOI:
  http://dx.doi.org/10.7554/eLife.01102.012'
pmcid: PMC3787301
papertitle: The human gut and groundwater harbor non-photosynthetic bacteria belonging
  to a new candidate phylum sibling to Cyanobacteria.
reftext: Sara C Di Rienzi, et al. eLife. 2013;2:e01102.
pmc_ranked_result_index: '108381'
pathway_score: 0.901181
filename: elife01102f004.jpg
figtitle: The physiological and metabolic landscape of Melainabacteria
year: '2013'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3787301__elife01102f004.html
  '@type': Dataset
  description: 'Metabolic predictions for Melainabacteria based on genes identified
    in . Genes in pathways detected in the genomes of the subsurface and at least
    one gut genome (white box), only in the subsurface genome (grey box), only in
    at least one gut genome (orange box), and genes missing from pathways in all genomes
    (red box). Glycolysis proceeds via the canonical Embden-Meyerhof-Parnas (EMP)
    pathway with the exception of fructose-6-phosphate 1-phosphotransferase (EC:2.7.1.90,
    gene 3). Names of pathways and fermentation end-products are bolded and ATP generated
    by substrate-level phosphorylation are noted. All Melainabacteria genomes sampled
    lack electron transport chain components (including cytochromes (Cyto), succinate
    dehydrogenase (sdh), flavins, quinones), terminal respiratory oxidases or reductases,
    and photosystem I or II (PS1, PS2). The genomes also lack a complete TCA cycle
    (absent enzymes noted by red boxes), with the TCA enzymes instead linked to the
    fermentation of amino acids and organic acids denoted (pathways, blue arrows).
    Ferredoxin (Fd, green text) is important for hydrogen (H2) production via hydrogenases
    (yellow background box). Proton translocation mechanisms (green background box)
    may be achieved by the activity of trimeric oxaloacetate (OAA) decarboxylase and
    sodium-hydrogen antiporter, pyrophosphate (PPi) hydrolysis with pyrophosphatases,
    11 subunit NADH dehydrogenase, and an annotated NiFe hydrogenase (green enzyme).
    Annotations for the gene numbers are in . The complete metabolic comparison of
    the Melainabacteria can be accessed at http://ggkbase.berkeley.edu/genome_summaries/81-MEL-Metabolic-Overview-June2013.DOI:
    http://dx.doi.org/10.7554/eLife.01102.01010.7554/eLife.01102.011Figure 4—source
    data 1.Examination of enzymes (steps) in near-complete KEGG based modules shared
    among or unique to subsurface ACD20 and gut Melainabacteria genomes MEL.A1, MEL.B1,
    and MEL.B2.Analysis is based on the KEGG Module database ().DOI: http://dx.doi.org/10.7554/eLife.01102.01110.7554/eLife.01102.012Figure
    4—source data 2.Gene annotations corresponding to the numbers in .If the gene
    occurs in both the ACD20 and gut genomes, the reported annotation is based on
    ACD20.DOI: http://dx.doi.org/10.7554/eLife.01102.012'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SRC
  - MALL
  - CIT
  - PDS5B
  - ADH5
  - SDHB
  - AIP
  - MAEA
  - WDTC1
  - Acetate
  - Ethanol
  - Butyrate
  - dicarboxylic acids
  - flavin
  - Formate
  - Fructose
  - Glucose
  - Hexose
  - Trehalose87 Maltose
  - Pentose phosphate
  - Pyruvate
  - Ribose
  - Sorbitol
  - Glu
  - Lactate
genes:
- word: As(V)
  symbol: ASV
  source: hgnc_alias_symbol
  hgnc_symbol: SRC
  entrez: '6714'
- word: Mall
  symbol: MALL
  source: hgnc_symbol
  hgnc_symbol: MALL
  entrez: '7851'
- word: Cit
  symbol: CIT
  source: hgnc_symbol
  hgnc_symbol: CIT
  entrez: '11113'
- word: as3
  symbol: AS3
  source: hgnc_alias_symbol
  hgnc_symbol: PDS5B
  entrez: '23047'
- word: fdh
  symbol: FDH
  source: hgnc_prev_symbol
  hgnc_symbol: ADH5
  entrez: '128'
- word: sdh
  symbol: SDH
  source: hgnc_prev_symbol
  hgnc_symbol: SDHB
  entrez: '6390'
- word: '"""AIP"'
  symbol: AIP
  source: hgnc_symbol
  hgnc_symbol: AIP
  entrez: '9049'
- word: EMP
  symbol: EMP
  source: hgnc_alias_symbol
  hgnc_symbol: MAEA
  entrez: '10296'
- word: ADP
  symbol: ADP
  source: hgnc_alias_symbol
  hgnc_symbol: WDTC1
  entrez: '23038'
chemicals:
- word: Acetate
  source: MESH
  identifier: D000085
- word: Ethanol
  source: MESH
  identifier: D000431
- word: Butyrate
  source: MESH
  identifier: D002087
- word: dicarboxylic acids
  source: MESH
  identifier: D003998
- word: flavin
  source: MESH
  identifier: D005415
- word: Formate
  source: MESH
  identifier: D005561
- word: Fructose
  source: MESH
  identifier: D005632
- word: Glucose
  source: MESH
  identifier: D005947
- word: Hexose
  source: MESH
  identifier: D006601
- word: Trehalose87 Maltose
  source: MESH
  identifier: D008320
- word: Pentose phosphate
  source: MESH
  identifier: D010429
- word: Pyruvate
  source: MESH
  identifier: D011773
- word: Ribose
  source: MESH
  identifier: D012266
- word: Sorbitol
  source: MESH
  identifier: D013012
- word: Glu
  source: MESH
  identifier: D018698
- word: Lactate
  source: MESH
  identifier: D019344
diseases: []
---
