---
figid: PMC3339861__zh70081237210003
figlink: /pmc/articles/PMC3339861/figure/F3/
number: F3
caption: 'EGF decreases sphingolipid biosynthesis transcripts. A: sphingolipid metabolism.
  This pathway was constructed based on the sphingolipid metabolism pathway in the
  KEGG database and figures in Refs.  and  and was drawn using ChemBioDraw Ultra 12.0
  (CambridgeSoft, Cambridge, MA). The initial condensation, catalyzed by SPTLC, is
  also the rate-limiting step for de novo synthesis, forming 3-ketodihydro-sphingosine
  (). This product is then rapidly reduced to sphinganine, which is then acylated
  by a member of the LASS family of enzymes to form dihydroceramide. LASS3 is the
  most relevant isozyme for ceramides of the epidermal barrier due to its fatty acyl-CoA
  chain length specificity (). Dihydroceramide is then desaturated by DEGS1/2 or hydroxylated
  by DEGS2 (exclusively), to form ceramides and phytoceramides, respectively. Ceramides
  Cer5(AS) and Cer2(NS) may also be formed via the Salvage Pathway by LASS acylation
  of sphingosine already present in the cell. These reactions occur in the endoplasmic
  reticulum. At this point, the pool of newly formed ceramides and phytoceramides
  are trafficked to the Golgi apparatus. UGCG glycosylates all ceramides and phytoceramides
  to form glucosylceramides. SGMS catalyzes the addition of choline phosphate (from
  phosphatidylcholine) to Cer5 and Cer2 to form sphingomyelin. These 2 products (glucosylceramides
  and sphingomyelin) are packaged into the lamellar bodies, which are extruded at
  the intersection of the stratum granulosum and stratum corneum. EGF relative fold
  changes determined by qRT-PCR are shown. *Effects of EGF are significant by Student''s
  t-test (P < 0.05). B: qRT-PCR analysis of transcripts encoding enzymes involved
  in sphingolipid biochemical pathways (n = 3–4). Results are expressed relative to
  values obtained in samples grown to 50% confluent cell density without EGF following
  normalization to values of TUBA1C. †Comparison between the untreated samples (confluent
  cell density effect). *Comparison between control and EGF at 100% confluent cell
  density. All bars indicate means ± SD. Student''s t-test with Ŝidák-Bonferroni correction
  was used to evaluate statistical significance. ††P < 0.01, †††P < 0.001, *P < 0.0253
  (Ŝidák-Bonferroni correction P value cut-off), **P < 0.01, ***P < 0.001.'
pmcid: PMC3339861
papertitle: EGFR regulation of epidermal barrier function.
reftext: Quynh T. Tran, et al. Physiol Genomics. 2012 Apr 15;44(8):455-469.
pmc_ranked_result_index: '192732'
pathway_score: 0.8397976
filename: zh70081237210003.jpg
figtitle: EGF decreases sphingolipid biosynthesis transcripts
year: '2012'
organisms: Homo sapiens
ndex: e74fcb6e-de8e-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3339861__zh70081237210003.html
  '@type': Dataset
  description: 'EGF decreases sphingolipid biosynthesis transcripts. A: sphingolipid
    metabolism. This pathway was constructed based on the sphingolipid metabolism
    pathway in the KEGG database and figures in Refs.  and  and was drawn using ChemBioDraw
    Ultra 12.0 (CambridgeSoft, Cambridge, MA). The initial condensation, catalyzed
    by SPTLC, is also the rate-limiting step for de novo synthesis, forming 3-ketodihydro-sphingosine
    (). This product is then rapidly reduced to sphinganine, which is then acylated
    by a member of the LASS family of enzymes to form dihydroceramide. LASS3 is the
    most relevant isozyme for ceramides of the epidermal barrier due to its fatty
    acyl-CoA chain length specificity (). Dihydroceramide is then desaturated by DEGS1/2
    or hydroxylated by DEGS2 (exclusively), to form ceramides and phytoceramides,
    respectively. Ceramides Cer5(AS) and Cer2(NS) may also be formed via the Salvage
    Pathway by LASS acylation of sphingosine already present in the cell. These reactions
    occur in the endoplasmic reticulum. At this point, the pool of newly formed ceramides
    and phytoceramides are trafficked to the Golgi apparatus. UGCG glycosylates all
    ceramides and phytoceramides to form glucosylceramides. SGMS catalyzes the addition
    of choline phosphate (from phosphatidylcholine) to Cer5 and Cer2 to form sphingomyelin.
    These 2 products (glucosylceramides and sphingomyelin) are packaged into the lamellar
    bodies, which are extruded at the intersection of the stratum granulosum and stratum
    corneum. EGF relative fold changes determined by qRT-PCR are shown. *Effects of
    EGF are significant by Student''s t-test (P < 0.05). B: qRT-PCR analysis of transcripts
    encoding enzymes involved in sphingolipid biochemical pathways (n = 3–4). Results
    are expressed relative to values obtained in samples grown to 50% confluent cell
    density without EGF following normalization to values of TUBA1C. †Comparison between
    the untreated samples (confluent cell density effect). *Comparison between control
    and EGF at 100% confluent cell density. All bars indicate means ± SD. Student''s
    t-test with Ŝidák-Bonferroni correction was used to evaluate statistical significance.
    ††P < 0.01, †††P < 0.001, *P < 0.0253 (Ŝidák-Bonferroni correction P value cut-off),
    **P < 0.01, ***P < 0.001.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MBTPS1
  - CERS3
  - KDSR
  - SGMS2
  - CERS4
  - IGLL1
  - ACER1
  - GBA
  - PDGFD
  - EGF
  - FA2H
  - SPTLC3
  - WDTC1
  - DEGS2
  - DEGS1
  - UGCG
  - SPHK1
  - SGPL1
  - DAND5
  - SMPD3
  - ASAH1
  - SGPP2
  - NEUROG1
  - Palmitaldehyde
  - Phosphoethanolamine
  - Dihydrosphingosine
  - Sphingosine 1-Phosphate
  - Dihydroceramide
  - Acyl-CoA
  - Ceramide
  - Fatty Acid
  - Glucosylceramide
  - Palmitoyl-CoA
  - Phosphatidylcholine
  - Choline Phosphate
  - Serine
  - Sphingomyelin
  - 3-ketodihydro-sphingosine
  - Sphingosine
genes:
- word: S1P
  symbol: S1P
  source: hgnc_alias_symbol
  hgnc_symbol: MBTPS1
  entrez: '8720'
- word: LASS3,4
  symbol: LASS3
  source: hgnc_prev_symbol
  hgnc_symbol: CERS3
  entrez: '204219'
- word: KDSR
  symbol: KDSR
  source: hgnc_symbol
  hgnc_symbol: KDSR
  entrez: '2531'
- word: SGMS2
  symbol: SGMS2
  source: hgnc_symbol
  hgnc_symbol: SGMS2
  entrez: '166929'
- word: LASS3,4
  symbol: LASS4
  source: hgnc_prev_symbol
  hgnc_symbol: CERS4
  entrez: '79603'
- word: '141'
  symbol: '14.1'
  source: hgnc_alias_symbol
  hgnc_symbol: IGLL1
  entrez: '3543'
- word: ACER1
  symbol: ACER1
  source: hgnc_symbol
  hgnc_symbol: ACER1
  entrez: '125981'
- word: GBA
  symbol: GBA
  source: hgnc_symbol
  hgnc_symbol: GBA
  entrez: '2629'
- word: IEGF
  symbol: IEGF
  source: hgnc_alias_symbol
  hgnc_symbol: PDGFD
  entrez: '80310'
- word: EGF
  symbol: EGF
  source: hgnc_symbol
  hgnc_symbol: EGF
  entrez: '1950'
- word: FA2H
  symbol: FA2H
  source: hgnc_symbol
  hgnc_symbol: FA2H
  entrez: '79152'
- word: SPTLC3
  symbol: SPTLC3
  source: hgnc_symbol
  hgnc_symbol: SPTLC3
  entrez: '55304'
- word: ADP
  symbol: ADP
  source: hgnc_alias_symbol
  hgnc_symbol: WDTC1
  entrez: '23038'
- word: DEGS2
  symbol: DEGS2
  source: hgnc_symbol
  hgnc_symbol: DEGS2
  entrez: '123099'
- word: DEGS1,2
  symbol: DEGS1
  source: hgnc_symbol
  hgnc_symbol: DEGS1
  entrez: '8560'
- word: UGCG
  symbol: UGCG
  source: hgnc_symbol
  hgnc_symbol: UGCG
  entrez: '7357'
- word: SPHK1
  symbol: SPHK1
  source: hgnc_symbol
  hgnc_symbol: SPHK1
  entrez: '8877'
- word: SGPL1
  symbol: SGPL1
  source: hgnc_symbol
  hgnc_symbol: SGPL1
  entrez: '8879'
- word: Cer2
  symbol: CER2
  source: hgnc_alias_symbol
  hgnc_symbol: DAND5
  entrez: '199699'
- word: SMPD3
  symbol: SMPD3
  source: hgnc_symbol
  hgnc_symbol: SMPD3
  entrez: '55512'
- word: ASAH1
  symbol: ASAH1
  source: hgnc_symbol
  hgnc_symbol: ASAH1
  entrez: '427'
- word: SGPP2
  symbol: SGPP2
  source: hgnc_symbol
  hgnc_symbol: SGPP2
  entrez: '130367'
- word: (aka
  symbol: AKA
  source: hgnc_alias_symbol
  hgnc_symbol: NEUROG1
  entrez: '4762'
chemicals:
- word: Palmitaldehyde
  source: MESH
  identifier: C001647
- word: Phosphoethanolamine
  source: MESH
  identifier: C005448
- word: Dihydrosphingosine
  source: MESH
  identifier: C005682
- word: Sphingosine 1-Phosphate
  source: MESH
  identifier: C060506
- word: Dihydroceramide
  source: MESH
  identifier: C109343
- word: Acyl-CoA
  source: MESH
  identifier: D000214
- word: Ceramide
  source: MESH
  identifier: D002518
- word: Fatty Acid
  source: MESH
  identifier: D005227
- word: Glucosylceramide
  source: MESH
  identifier: D005963
- word: Palmitoyl-CoA
  source: MESH
  identifier: D010171
- word: Phosphatidylcholine
  source: MESH
  identifier: D010713
- word: Choline Phosphate
  source: MESH
  identifier: D010767
- word: Serine
  source: MESH
  identifier: D012694
- word: Sphingomyelin
  source: MESH
  identifier: D013109
- word: 3-ketodihydro-sphingosine
  source: MESH
  identifier: D013110
- word: Sphingosine
  source: MESH
  identifier: D013110
diseases: []
figid_alias: PMC3339861__F3
redirect_from: /figures/PMC3339861__F3
figtype: Figure
---
