---
figid: PMC3428307__pone.0043595.g001
figlink: /pmc/articles/PMC3428307/figure/pone-0043595-g001/
number: F1
caption: The numbers inside the boxes correspond to the number (N) of samples. Samples
  from PD patients were randomly chosen to be part of the training or test sets of
  samples. (A) Identifying the PD risk markers using a training set. In order to identify
  putative biomarkers, the training set (training set 1) was used in the microarray
  screen. The data from the microarray analysis was analyzed by three methods in order
  to identify markers with good specificity, sensitivity and predictive accuracy.
  PD patients were compared to each control group separately, compared to pooled controls
  as a single group and using LOOCV. Splice variants that were up- or down-regulated
  by 2-fold (P<0.05) in PD patients were considered candidate risk markers. The candidate
  risk markers were manually curated to include those that may play a role in PD based
  on pathway and disease analysis (Ingenuity Systems software) and to exclude those
  for which primers could not be designed or could not be detected by qPCR. Thirteen
  of the risk markers were replicated by qPCR (Training set 1). (B) Experimentally
  testing the PD risk markers with training and test sets. The 13 risk markers were
  validated in two independent test sets (training set 2 and test set) using qPCR.
  Clinical diagnosis of the participants was based on neurological exam. (C) Statistical
  analysis. LOOCV KNN was used to determine the predictive accuracy of the samples
  from the training set (training set 1). In addition, linear discriminant analysis
  was used to test the predictive accuracy on the training and test sets. PD is Parkinson’s
  disease patients represented in green, HC is healthy controls represented in red,
  MSA is multiple system atrophy controls represented in blue, PSP is progressive
  supranuclear palsy controls represented in purple and Control is HC+MSA+PSP controls
  represented in yellow. TP = true positive; TN = true negative; FP = false positive;
  FN = false negative; PPV = positive predictive value; NPV = negative predictive
  value.
pmcid: PMC3428307
papertitle: Biosignatures for Parkinson’s Disease and Atypical Parkinsonian Disorders
  Patients.
reftext: Judith A. Potashkin, et al. PLoS One. 2012;7(8):e43595.
pmc_ranked_result_index: '216758'
pathway_score: 0.5728821
filename: pone.0043595.g001.jpg
figtitle: Biosignatures for Parkinson’s Disease and Atypical Parkinsonian Disorders
  Patients
year: '2012'
organisms:
- Mus musculus
- Sequoia sempervirens
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3428307__pone.0043595.g001.html
  '@type': Dataset
  description: The numbers inside the boxes correspond to the number (N) of samples.
    Samples from PD patients were randomly chosen to be part of the training or test
    sets of samples. (A) Identifying the PD risk markers using a training set. In
    order to identify putative biomarkers, the training set (training set 1) was used
    in the microarray screen. The data from the microarray analysis was analyzed by
    three methods in order to identify markers with good specificity, sensitivity
    and predictive accuracy. PD patients were compared to each control group separately,
    compared to pooled controls as a single group and using LOOCV. Splice variants
    that were up- or down-regulated by 2-fold (P<0.05) in PD patients were considered
    candidate risk markers. The candidate risk markers were manually curated to include
    those that may play a role in PD based on pathway and disease analysis (Ingenuity
    Systems software) and to exclude those for which primers could not be designed
    or could not be detected by qPCR. Thirteen of the risk markers were replicated
    by qPCR (Training set 1). (B) Experimentally testing the PD risk markers with
    training and test sets. The 13 risk markers were validated in two independent
    test sets (training set 2 and test set) using qPCR. Clinical diagnosis of the
    participants was based on neurological exam. (C) Statistical analysis. LOOCV KNN
    was used to determine the predictive accuracy of the samples from the training
    set (training set 1). In addition, linear discriminant analysis was used to test
    the predictive accuracy on the training and test sets. PD is Parkinson’s disease
    patients represented in green, HC is healthy controls represented in red, MSA
    is multiple system atrophy controls represented in blue, PSP is progressive supranuclear
    palsy controls represented in purple and Control is HC+MSA+PSP controls represented
    in yellow. TP = true positive; TN = true negative; FP = false positive; FN = false
    negative; PPV = positive predictive value; NPV = negative predictive value.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TNR
  - TNXB
  - TUSC3
  - SETD1A
  - BCR
  - TNN
  - TNC
  - PSPH
  - SET
genes:
- word: TN
  symbol: TN
  source: bioentities_symbol
  hgnc_symbol: TNR
  entrez: '7143'
- word: TN
  symbol: TN
  source: bioentities_symbol
  hgnc_symbol: TNXB
  entrez: '7148'
- word: (N=33)
  symbol: N33
  source: hgnc_alias_symbol
  hgnc_symbol: TUSC3
  entrez: '7991'
- word: set1
  symbol: Set1
  source: hgnc_alias_symbol
  hgnc_symbol: SETD1A
  entrez: '9739'
- word: all
  symbol: ALL
  source: hgnc_alias_symbol
  hgnc_symbol: BCR
  entrez: '613'
- word: TN
  symbol: TN
  source: bioentities_symbol
  hgnc_symbol: TNN
  entrez: '63923'
- word: TN
  symbol: TN
  source: bioentities_symbol
  hgnc_symbol: TNC
  entrez: '3371'
- word: PSP
  symbol: PSP
  source: hgnc_prev_symbol
  hgnc_symbol: PSPH
  entrez: '5723'
- word: set.
  symbol: SET
  source: hgnc_symbol
  hgnc_symbol: SET
  entrez: '6418'
chemicals: []
diseases: []
figid_alias: PMC3428307__F1
redirect_from: /figures/PMC3428307__F1
figtype: Figure
---
