---
figid: PMC3667846__pone.0065622.g006
figlink: /pmc/articles/PMC3667846/figure/pone-0065622-g006/
number: F6
caption: 'Enzymes for which information about cDNAs is not known are shown by broken
  arrows. Enzymes in red show up-regulation in pap1 in comparison to BR086, whereas
  in green (7OMT) shows down-regulation. Transcripts with no change in expression
  are shown in black. Transcripts identified through comparative transcriptome analysis
  with putative involvement in papaverine are underlined. Abbreviations: TYDC, tyrosine
  decarboxylase; NCS, norcoclaurine synthase; 6OMT, (S)-norcoclaurine 6-O-methyltransferase;
  CNMT, (S)-coclaurine N-methyltransferase; NMCH, (S)-N-methylcoclaurine 3′-hydroxylase;
  3′OHase, 3′-hydroxylase; 4′OMT, (S)-3′-hydroxyN-methylcoclaurine 4′-O-methyltransferase;
  N7OMT, norreticuline 7-O-methyltransferase; 7OMT, reticuline 7-O-methyltransferase;
  3′OMT, 3′-O-methyltransferase; deHase, dehydrogenase; LNdeMT, laudanosine N-demethylase.'
pmcid: PMC3667846
papertitle: Comparative Transcriptome Analysis Using High Papaverine Mutant of Papaver
  somniferum Reveals Pathway and Uncharacterized Steps of Papaverine Biosynthesis.
reftext: Sumya Pathak, et al. PLoS One. 2013;8(5):e65622.
pmc_ranked_result_index: '96377'
pathway_score: 0.9553669
filename: pone.0065622.g006.jpg
figtitle: Comparative Transcriptome Analysis Using High Papaverine Mutant of Papaver
  somniferum Reveals Pathway and Uncharacterized Steps of Papaverine Biosynthesis
year: '2013'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3667846__pone.0065622.g006.html
  '@type': Dataset
  description: 'Enzymes for which information about cDNAs is not known are shown by
    broken arrows. Enzymes in red show up-regulation in pap1 in comparison to BR086,
    whereas in green (7OMT) shows down-regulation. Transcripts with no change in expression
    are shown in black. Transcripts identified through comparative transcriptome analysis
    with putative involvement in papaverine are underlined. Abbreviations: TYDC, tyrosine
    decarboxylase; NCS, norcoclaurine synthase; 6OMT, (S)-norcoclaurine 6-O-methyltransferase;
    CNMT, (S)-coclaurine N-methyltransferase; NMCH, (S)-N-methylcoclaurine 3′-hydroxylase;
    3′OHase, 3′-hydroxylase; 4′OMT, (S)-3′-hydroxyN-methylcoclaurine 4′-O-methyltransferase;
    N7OMT, norreticuline 7-O-methyltransferase; 7OMT, reticuline 7-O-methyltransferase;
    3′OMT, 3′-O-methyltransferase; deHase, dehydrogenase; LNdeMT, laudanosine N-demethylase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - LYST
  - (S)--Laudanosine
  - (S)--6-0
  - Papaverine
  - L-Tyrosine
genes:
- word: CHS
  symbol: CHS
  source: hgnc_alias_symbol
  hgnc_symbol: LYST
  entrez: '1130'
chemicals:
- word: (S)--Laudanosine
  source: MESH
  identifier: C001522
- word: (S)--6-0
  source: MESH
  identifier: C012008
- word: Papaverine
  source: MESH
  identifier: D010208
- word: L-Tyrosine
  source: MESH
  identifier: D014443
diseases: []
figid_alias: PMC3667846__F6
redirect_from: /figures/PMC3667846__F6
figtype: Figure
---
