---
figid: PMC3746496__JC-12-14-g002
figlink: /pmc/articles/PMC3746496/figure/F2/
number: F2
caption: PKM2 activity is controlled by a tight regulatory network at the transcriptional,
  proteomic and metabolic level. (a) Upon EGFR activation parallel signaling pathways
  initiate transcription of c-MYC and HIF1a. Since PKM2 is both, transcriptional target
  and transcriptional coactivator, parallel in combination with forward control initiates
  and amplifies the signal. (b) Post - translational modifications control the equilibrium
  between metabolically active tetramer (PKM2)4 and transcriptional active dimer (PKM2)2
  with protein kinase activity. In the nucleus PKM2 serves as coactivator of glycolytic
  genes including itself. This further amplifies the signal. (c) A network of feedback
  mechanisms provides a metabolic phenotype that is robust against perturbations.
  The combination of positive and negative feedback control guarantees stable flux
  through the glycolytic pathway with PKM2 as terminal master regulator. PKM2 transcript,
  protein tetramer, and metabolic activity are highlighted in blue. The gate keeping
  pyruvate dehydrogenase (PDH) reaction between cytosol and mitochondria is negatively
  regulated by pyruvate dehydrogenase kinase (PDK) and labeled in grey. Genes are
  displayed in Italic. Enzymes are abbreviated in uppercase as enolase (ENO), glucose
  transporter (GLUT), hexokinase (HK), lactate dehydrogenase (LDH), glutaminase (GLS),
  PDH, PDK, PKM2. Metabolites are abbreviated in lowercase as acetyl - CoA (aco),
  alanine (ala), adenosine triphosphate (atp), fructose - 1,6 - bisphosphatase (fbp),
  fructose - 6 - phosphatase (f6p), glucose - 6 - phosphate (g6p), lactate (lac),
  phosphoenol pyruvate (pep), 3 - phosphoglycerate (3pg), pyruvate (pyr), reactive
  oxygen species (ros), ribose - 5 - phosphate (r5p), succinylaminoimidazolecarboxamide
  ribose - 5 - phosphate (saicar), serine (ser).
pmcid: PMC3746496
papertitle: 'Cancer metabolism meets systems biology: Pyruvate kinase isoform PKM2
  is a metabolic master regulator.'
reftext: Fabian V. Filipp. J Carcinog. 2013;12:14.
pmc_ranked_result_index: '49416'
pathway_score: 0.9015083
filename: JC-12-14-g002.jpg
figtitle: PKM2 activity is controlled by a tight regulatory network at the transcriptional,
  proteomic and metabolic level
year: '2013'
organisms:
- Homo sapiens
ndex: f2bfa6dc-de8e-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3746496__JC-12-14-g002.html
  '@type': Dataset
  description: PKM2 activity is controlled by a tight regulatory network at the transcriptional,
    proteomic and metabolic level. (a) Upon EGFR activation parallel signaling pathways
    initiate transcription of c-MYC and HIF1a. Since PKM2 is both, transcriptional
    target and transcriptional coactivator, parallel in combination with forward control
    initiates and amplifies the signal. (b) Post - translational modifications control
    the equilibrium between metabolically active tetramer (PKM2)4 and transcriptional
    active dimer (PKM2)2 with protein kinase activity. In the nucleus PKM2 serves
    as coactivator of glycolytic genes including itself. This further amplifies the
    signal. (c) A network of feedback mechanisms provides a metabolic phenotype that
    is robust against perturbations. The combination of positive and negative feedback
    control guarantees stable flux through the glycolytic pathway with PKM2 as terminal
    master regulator. PKM2 transcript, protein tetramer, and metabolic activity are
    highlighted in blue. The gate keeping pyruvate dehydrogenase (PDH) reaction between
    cytosol and mitochondria is negatively regulated by pyruvate dehydrogenase kinase
    (PDK) and labeled in grey. Genes are displayed in Italic. Enzymes are abbreviated
    in uppercase as enolase (ENO), glucose transporter (GLUT), hexokinase (HK), lactate
    dehydrogenase (LDH), glutaminase (GLS), PDH, PDK, PKM2. Metabolites are abbreviated
    in lowercase as acetyl - CoA (aco), alanine (ala), adenosine triphosphate (atp),
    fructose - 1,6 - bisphosphatase (fbp), fructose - 6 - phosphatase (f6p), glucose
    - 6 - phosphate (g6p), lactate (lac), phosphoenol pyruvate (pep), 3 - phosphoglycerate
    (3pg), pyruvate (pyr), reactive oxygen species (ros), ribose - 5 - phosphate (r5p),
    succinylaminoimidazolecarboxamide ribose - 5 - phosphate (saicar), serine (ser).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PDK4
  - PDK2
  - PDK3
  - PKM
  - STAT3
  - LDHC
  - CTNNB1
  - MYC
  - SLC2A1
  - PDK1
  - LDHA
  - FBP1
  - GLS
  - PAEP
  - KLK15
  - H3F3C
  - HIST3H3
  - POU5F1
  - LDHB
  - PDHX
  - ENO1
  - ENO2
  - PDHA2
  - HIF1A
  - ENO4
  - PDHA1
  - ENO3
  - DLAT
  - PDHB
  - CCND1
  - DLD
  - EGFR
  - ser
genes:
- word: PDK-
  symbol: PDK
  source: bioentities_symbol
  hgnc_symbol: PDK4
  entrez: '5166'
- word: PDK-
  symbol: PDK
  source: bioentities_symbol
  hgnc_symbol: PDK2
  entrez: '5164'
- word: PDK-
  symbol: PDK
  source: bioentities_symbol
  hgnc_symbol: PDK3
  entrez: '5165'
- word: PKM2
  symbol: PKM2
  source: hgnc_prev_symbol
  hgnc_symbol: PKM
  entrez: '5315'
- word: STAT3
  symbol: STAT3
  source: hgnc_symbol
  hgnc_symbol: STAT3
  entrez: '6774'
- word: LDH.
  symbol: LDH
  source: bioentities_symbol
  hgnc_symbol: LDHC
  entrez: '3948'
- word: B-catenin
  symbol: beta-catenin
  source: hgnc_alias_symbol
  hgnc_symbol: CTNNB1
  entrez: '1499'
- word: c-MYC
  symbol: c-Myc
  source: hgnc_alias_symbol
  hgnc_symbol: MYC
  entrez: '4609'
- word: GLUT
  symbol: GLUT
  source: hgnc_prev_symbol
  hgnc_symbol: SLC2A1
  entrez: '6513'
- word: PDK-
  symbol: PDK
  source: bioentities_symbol
  hgnc_symbol: PDK1
  entrez: '5163'
- word: LDH.
  symbol: LDH
  source: bioentities_symbol
  hgnc_symbol: LDHA
  entrez: '3939'
- word: fbp
  symbol: FBP
  source: hgnc_prev_symbol
  hgnc_symbol: FBP1
  entrez: '2203'
- word: GLS
  symbol: GLS
  source: hgnc_symbol
  hgnc_symbol: GLS
  entrez: '2744'
- word: pep
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
- word: aco
  symbol: ACO
  source: hgnc_alias_symbol
  hgnc_symbol: KLK15
  entrez: '55554'
- word: histoneH3
  symbol: Histone_H3
  source: bioentities_symbol
  hgnc_symbol: H3F3C
  entrez: '440093'
- word: histoneH3
  symbol: Histone_H3
  source: bioentities_symbol
  hgnc_symbol: HIST3H3
  entrez: '8290'
- word: OCT4
  symbol: Oct4
  source: hgnc_alias_symbol
  hgnc_symbol: POU5F1
  entrez: '5460'
- word: LDH.
  symbol: LDH
  source: bioentities_symbol
  hgnc_symbol: LDHB
  entrez: '3945'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHX
  entrez: '8050'
- word: ENO
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO1
  entrez: '2023'
- word: ENO
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO2
  entrez: '2026'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHA2
  entrez: '5161'
- word: HIF1A
  symbol: HIF1A
  source: hgnc_symbol
  hgnc_symbol: HIF1A
  entrez: '3091'
- word: ENO
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO4
  entrez: '387712'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHA1
  entrez: '5160'
- word: ENO
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO3
  entrez: '2027'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: DLAT
  entrez: '1737'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHB
  entrez: '5162'
- word: CyclinD1
  symbol: Cyclin-D1
  source: bioentities_symbol
  hgnc_symbol: CCND1
  entrez: '595'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: DLD
  entrez: '1738'
- word: EGFR
  symbol: EGFR
  source: hgnc_symbol
  hgnc_symbol: EGFR
  entrez: '1956'
chemicals:
- word: ser
  source: MESH
  identifier: D012694
diseases: []
figid_alias: PMC3746496__F2
redirect_from: /figures/PMC3746496__F2
figtype: Figure
---
