---
figid: PMC3596326__pone.0059300.g004
figlink: /pmc/articles/PMC3596326/figure/pone-0059300-g004/
number: Figure 4
caption: A. Origin of nucleotide metabolism ∼3.8 Gy ago; ndFF  = 0). B. Emergence
  of the nucleotide interconversion (INT), catabolism and salvage (CAT) and biosynthetic
  (BIO) pathways ∼3.5 Gy ago (ndFF  = 0.061–0.073). C. Fully connected INT, BIO and
  CAT pathways ∼3 Gy ago (ndFF  = 0.187). Pathways mediated by prebiotic chemistries
  that are plausible and most parsimonious are depicted in red and enable the growth
  of the emergent protein enzyme-mediated pathways of purine metabolism by structural
  and functional innovation and piecemeal recruitment (recruited FFs are indicated
  with numbers). Unknown candidate or withering prebiotic pathways are indicated with
  dashed lines. We note that primordial reactions of the BIO pathway (top of metabolic
  diagram) in B could have been non-operational in the absence of suitable prebiotic
  chemistries until later in evolution. FF structures associated with individual enzymatic
  activities (described in EC nomenclature) are painted according to their age, in
  a scale of node distance (ndFF) that ranges from 0 (the oldest enzymes; ∼3.8 Gy
  ago) to 0.2 (∼3.0 Gy ago).
pmcid: PMC3596326
papertitle: Structural Phylogenomics Reveals Gradual Evolutionary Replacement of Abiotic
  Chemistries by Protein Enzymes in Purine Metabolism.
reftext: Kelsey Caetano-Anollés, et al. PLoS One. 2013;8(3):e59300.
pmc_ranked_result_index: '22533'
pathway_score: 0.8411128
filename: pone.0059300.g004.jpg
figtitle: Structural Phylogenomics Reveals Gradual Evolutionary Replacement of Abiotic
  Chemistries by Protein Enzymes in Purine Metabolism
year: '2013'
organisms: Homo sapiens
ndex: 627986dd-de9b-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3596326__pone.0059300.g004.html
  '@type': Dataset
  description: A. Origin of nucleotide metabolism ∼3.8 Gy ago; ndFF  = 0). B. Emergence
    of the nucleotide interconversion (INT), catabolism and salvage (CAT) and biosynthetic
    (BIO) pathways ∼3.5 Gy ago (ndFF  = 0.061–0.073). C. Fully connected INT, BIO
    and CAT pathways ∼3 Gy ago (ndFF  = 0.187). Pathways mediated by prebiotic chemistries
    that are plausible and most parsimonious are depicted in red and enable the growth
    of the emergent protein enzyme-mediated pathways of purine metabolism by structural
    and functional innovation and piecemeal recruitment (recruited FFs are indicated
    with numbers). Unknown candidate or withering prebiotic pathways are indicated
    with dashed lines. We note that primordial reactions of the BIO pathway (top of
    metabolic diagram) in B could have been non-operational in the absence of suitable
    prebiotic chemistries until later in evolution. FF structures associated with
    individual enzymatic activities (described in EC nomenclature) are painted according
    to their age, in a scale of node distance (ndFF) that ranges from 0 (the oldest
    enzymes; ∼3.8 Gy ago) to 0.2 (∼3.0 Gy ago).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - WDTC1
  - NF2
  - MFAP1
  - NEUROG1
  - BRAP
  - S100A4
  - AMN
  - EMP2
  - NT5C2
  - CAT
  - Xanthosine O
  - Glycineamideribotide
  - AICAR 5-aminoimidazole-4-carbaramideribotide
  - 5-formamidoimidazole-4 carboxamideribotide
  - O FAICAR
  - 5-aminoimidazole-4N-succinylocarboxamideribotide
  - Ribosylamine
  - Diaminofumaronitrile
  - Adenine
  - Adenine O
  - O Adenine
  - Adenosine
  - O Formaldehyde
  - L-Glutamine
  - Glyceraldehyde
  - Glycine
  - Guanine
  - O Guanosine
  - ITP
  - Succinate
  - Hypoxanthine
  - Xanthine
genes:
- word: ADP
  symbol: ADP
  source: hgnc_alias_symbol
  hgnc_symbol: WDTC1
  entrez: '23038'
- word: ACN
  symbol: ACN
  source: hgnc_alias_symbol
  hgnc_symbol: NF2
  entrez: '4771'
- word: AMP
  symbol: AMP
  source: hgnc_alias_symbol
  hgnc_symbol: MFAP1
  entrez: '4236'
- word: AKA
  symbol: AKA
  source: hgnc_alias_symbol
  hgnc_symbol: NEUROG1
  entrez: '4762'
- word: IMP
  symbol: IMP
  source: hgnc_alias_symbol
  hgnc_symbol: BRAP
  entrez: '8315'
- word: 42.a
  symbol: 42A
  source: hgnc_alias_symbol
  hgnc_symbol: S100A4
  entrez: '6275'
- word: AMN
  symbol: AMN
  source: hgnc_symbol
  hgnc_symbol: AMN
  entrez: '81693'
- word: XMP
  symbol: XMP
  source: hgnc_alias_symbol
  hgnc_symbol: EMP2
  entrez: '2013'
- word: GMP
  symbol: GMP
  source: hgnc_alias_symbol
  hgnc_symbol: NT5C2
  entrez: '22978'
- word: CAT
  symbol: CAT
  source: hgnc_symbol
  hgnc_symbol: CAT
  entrez: '847'
chemicals:
- word: Xanthosine O
  source: MESH
  identifier: C005893
- word: Glycineamideribotide
  source: MESH
  identifier: C014978
- word: AICAR 5-aminoimidazole-4-carbaramideribotide
  source: MESH
  identifier: C031143
- word: 5-formamidoimidazole-4 carboxamideribotide
  source: MESH
  identifier: C062030
- word: O FAICAR
  source: MESH
  identifier: C062030
- word: 5-aminoimidazole-4N-succinylocarboxamideribotide
  source: MESH
  identifier: C098330
- word: Ribosylamine
  source: MESH
  identifier: C481638
- word: Diaminofumaronitrile
  source: MESH
  identifier: C572732
- word: Adenine
  source: MESH
  identifier: D000225
- word: Adenine O
  source: MESH
  identifier: D000225
- word: O Adenine
  source: MESH
  identifier: D000225
- word: Adenosine
  source: MESH
  identifier: D000241
- word: O Formaldehyde
  source: MESH
  identifier: D005557
- word: L-Glutamine
  source: MESH
  identifier: D005973
- word: Glyceraldehyde
  source: MESH
  identifier: D005985
- word: Glycine
  source: MESH
  identifier: D005998
- word: Guanine
  source: MESH
  identifier: D006147
- word: O Guanosine
  source: MESH
  identifier: D006151
- word: ITP
  source: MESH
  identifier: D007293
- word: Succinate
  source: MESH
  identifier: D013386
- word: Hypoxanthine
  source: MESH
  identifier: D019271
- word: Xanthine
  source: MESH
  identifier: D019820
diseases: []
---
