---
figid: PMC3936743__gkt1150f7p
figlink: /pmc/articles/PMC3936743/figure/gkt1150-F7/
number: Figure 7
caption: 'Control of the concentration of transcripts for enzymes involved in central
  carbon metabolism. Ratios of mRNA concentrations (white bar) and mRNA half-lives
  (grey bar) are represented for the three growth rate comparisons (left to right:
  0.20 versus 0.10 h−1, 0.40 versus 0.20 h−1 and 0.63 versus 0.40 h−1) for 34 genes
  encoding enzymes of the central carbon metabolism. Growth rate comparisons for which
  the mRNA concentration is controlled mainly by degradation (ρD classes IV and V)
  are colored in red. The pathway names are given in boxes (PP: pentose phosphate
  pathway, ED: Entner–Doudoroff pathway, TCA: tricarboxylic acid cycle) and the direction
  of flow are indicated by arrows. Metabolites, GLC: glucose, PEP: phosphoenolpyruvate,
  6PG: 6-phosphogluconate, PYR: pyruvate, OAA: oxaloacetate, AcCoA: acetyl-CoA and
  ACE: acetate. Enzymes, galM: galactose-1-epimerase, ptsH/ptsI/crr: PTS system, pgm:
  phosphoglucomutase, zwf: glucose 6-phosphate-1-dehydrogenase, fbaA: fructose bisphosphate
  aldolase, gapA: glyceraldehyde 3-phosphate dehydrogenase, pgk: phosphoglycerate
  kinase, gpmA: 2,3–bisphosphoglycerate-dependent phosphoglycerate mutase, gpmM: 2,3–bisphosphoglycerate-independent
  phosphoglycerate mutase, edd: phosphogluconate dehydratase, eda: Entner-Doudoroff
  aldolase, ppc: PEP carboxylase, pck: PEP carboxykinase, ppsA: PEP synthetase, lpd/aceF:
  pyruvate dehydrogenase, ackA: acetate kinase, acs: acetyl-CoA synthetase, gltA:
  citrate synthase, acnB: aconitase B, icd: isocitrate dehydrogenase, aceK: isocitrate
  dehydrogenase kinase/phosphatase, sdhA/B/C/D: succinate dehydrogenase, frdA/B: fumarate
  reductase, fumC: fumarase C, mqo: malate:quinone oxidoreductase and maeA/maeB: malic
  enzymes.'
pmcid: PMC3936743
papertitle: Dual role of transcription and transcript stability in the regulation
  of gene expression in Escherichia coli cells cultured on glucose at different growth
  rates.
reftext: Thomas Esquerré, et al. Nucleic Acids Res. 2014 Feb;42(4):2460-2472.
pmc_ranked_result_index: '164216'
pathway_score: 0.8778202
filename: gkt1150f7p.jpg
figtitle: Control of the concentration of transcripts for enzymes involved in central
  carbon metabolism
year: '2014'
organisms: Homo sapiens
ndex: edae6a64-df11-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3936743__gkt1150f7p.html
  '@type': Dataset
  description: 'Control of the concentration of transcripts for enzymes involved in
    central carbon metabolism. Ratios of mRNA concentrations (white bar) and mRNA
    half-lives (grey bar) are represented for the three growth rate comparisons (left
    to right: 0.20 versus 0.10 h−1, 0.40 versus 0.20 h−1 and 0.63 versus 0.40 h−1)
    for 34 genes encoding enzymes of the central carbon metabolism. Growth rate comparisons
    for which the mRNA concentration is controlled mainly by degradation (ρD classes
    IV and V) are colored in red. The pathway names are given in boxes (PP: pentose
    phosphate pathway, ED: Entner–Doudoroff pathway, TCA: tricarboxylic acid cycle)
    and the direction of flow are indicated by arrows. Metabolites, GLC: glucose,
    PEP: phosphoenolpyruvate, 6PG: 6-phosphogluconate, PYR: pyruvate, OAA: oxaloacetate,
    AcCoA: acetyl-CoA and ACE: acetate. Enzymes, galM: galactose-1-epimerase, ptsH/ptsI/crr:
    PTS system, pgm: phosphoglucomutase, zwf: glucose 6-phosphate-1-dehydrogenase,
    fbaA: fructose bisphosphate aldolase, gapA: glyceraldehyde 3-phosphate dehydrogenase,
    pgk: phosphoglycerate kinase, gpmA: 2,3–bisphosphoglycerate-dependent phosphoglycerate
    mutase, gpmM: 2,3–bisphosphoglycerate-independent phosphoglycerate mutase, edd:
    phosphogluconate dehydratase, eda: Entner-Doudoroff aldolase, ppc: PEP carboxylase,
    pck: PEP carboxykinase, ppsA: PEP synthetase, lpd/aceF: pyruvate dehydrogenase,
    ackA: acetate kinase, acs: acetyl-CoA synthetase, gltA: citrate synthase, acnB:
    aconitase B, icd: isocitrate dehydrogenase, aceK: isocitrate dehydrogenase kinase/phosphatase,
    sdhA/B/C/D: succinate dehydrogenase, frdA/B: fumarate reductase, fumC: fumarase
    C, mqo: malate:quinone oxidoreductase and maeA/maeB: malic enzymes.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MAEA
  - PAEP
  - GALM
  - SDHD
  - EDA
  - ACE
  - FXN
  - PRKG1
  - VCAN
  - UBR5
  - SDHA
  - SDHB
genes:
- word: maeA
  symbol: MAEA
  source: hgnc_symbol
  hgnc_symbol: MAEA
  entrez: '10296'
- word: PEP
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
- word: galM
  symbol: GALM
  source: hgnc_symbol
  hgnc_symbol: GALM
  entrez: '130589'
- word: sdhD
  symbol: SDHD
  source: hgnc_symbol
  hgnc_symbol: SDHD
  entrez: '6392'
- word: eda
  symbol: EDA
  source: hgnc_symbol
  hgnc_symbol: EDA
  entrez: '1896'
- word: ACE
  symbol: ACE
  source: hgnc_symbol
  hgnc_symbol: ACE
  entrez: '1636'
- word: frdA
  symbol: FRDA
  source: hgnc_prev_symbol
  hgnc_symbol: FXN
  entrez: '2395'
- word: pgk
  symbol: PGK
  source: hgnc_alias_symbol
  hgnc_symbol: PRKG1
  entrez: '5592'
- word: pgm
  symbol: PG-M
  source: hgnc_alias_symbol
  hgnc_symbol: VCAN
  entrez: '1462'
- word: edd
  symbol: EDD
  source: hgnc_alias_symbol
  hgnc_symbol: UBR5
  entrez: '51366'
- word: sdhA/sdhB/sdhCIsdhD
  symbol: SDHA
  source: hgnc_symbol
  hgnc_symbol: SDHA
  entrez: '6389'
- word: sdhA/sdhB/sdhCIsdhD
  symbol: SDHB
  source: hgnc_symbol
  hgnc_symbol: SDHB
  entrez: '6390'
chemicals: []
diseases: []
---
