---
figid: PMC3338748__pone.0035003.g002
figlink: /pmc/articles/PMC3338748/figure/pone-0035003-g002/
number: F2
caption: (A) The flow diagram depicts the experimental strategy for SNV discovery
  and confirmation. For the AGRE and control cohorts, orthogonal multiplexing was
  performed to prepare two distinct sets of sample pools (15 pools of 20 samples each,
  or 20 pools of 15 samples each). Following enrichment of exonic target regions for
  all 18 mGluR pathway genes, SNVs were identified and confirmed by deep resequencing
  of orthogonal pools on two independent NGS platforms (Illumina GAII and the Helicos
  HeliScope). SNVs concordantly detected on both platforms were then analyzed as shown
  in panel B. (B) The flow diagram depicts the procedure used to classify the presumptive
  functional effects of identified variants. SNVs concordantly detected on both NGS
  platforms were classified as common or rare using a minor allele frequency (maf)
  threshold of 1%. Common SNVs, rare SNVs occurring in both autism and control populations,
  and rare synonymous (silent) SNVs were considered likely to be benign. Rare SNVs
  in intronic sequences flanking exons that did not affect conserved splice donor
  or acceptor sites or in 5′ untranslated regions were classified as not benign but
  of unknown significance. Rare SNVs causing missense substitutions or occurring in
  mRNA 3′ untranslated regions (and therefore possibly affecting mRNA stability or
  translation , ) were considered possibly deleterious, and rare SNVs causing nonsense
  mutations or affecting conserved splice donor or acceptor sequences were considered
  probably deleterious. These latter two categories of SNVs were together considered
  potentially deleterious.
pmcid: PMC3338748
papertitle: High-Throughput Sequencing of mGluR Signaling Pathway Genes Reveals Enrichment
  of Rare Variants in Autism.
reftext: Raymond J. Kelleher III, et al. PLoS One. 2012;7(4):e35003.
pmc_ranked_result_index: '214432'
pathway_score: 0.8925843
filename: pone.0035003.g002.jpg
figtitle: High-Throughput Sequencing of mGluR Signaling Pathway Genes Reveals Enrichment
  of Rare Variants in Autism
year: '2012'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3338748__pone.0035003.g002.html
  '@type': Dataset
  description: (A) The flow diagram depicts the experimental strategy for SNV discovery
    and confirmation. For the AGRE and control cohorts, orthogonal multiplexing was
    performed to prepare two distinct sets of sample pools (15 pools of 20 samples
    each, or 20 pools of 15 samples each). Following enrichment of exonic target regions
    for all 18 mGluR pathway genes, SNVs were identified and confirmed by deep resequencing
    of orthogonal pools on two independent NGS platforms (Illumina GAII and the Helicos
    HeliScope). SNVs concordantly detected on both platforms were then analyzed as
    shown in panel B. (B) The flow diagram depicts the procedure used to classify
    the presumptive functional effects of identified variants. SNVs concordantly detected
    on both NGS platforms were classified as common or rare using a minor allele frequency
    (maf) threshold of 1%. Common SNVs, rare SNVs occurring in both autism and control
    populations, and rare synonymous (silent) SNVs were considered likely to be benign.
    Rare SNVs in intronic sequences flanking exons that did not affect conserved splice
    donor or acceptor sites or in 5′ untranslated regions were classified as not benign
    but of unknown significance. Rare SNVs causing missense substitutions or occurring
    in mRNA 3′ untranslated regions (and therefore possibly affecting mRNA stability
    or translation , ) were considered possibly deleterious, and rare SNVs causing
    nonsense mutations or affecting conserved splice donor or acceptor sequences were
    considered probably deleterious. These latter two categories of SNVs were together
    considered potentially deleterious.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - YES1
  - ETFA
  - MAF
  - BRD2
genes:
- word: 'yes'
  symbol: 'Yes'
  source: hgnc_alias_symbol
  hgnc_symbol: YES1
  entrez: '7525'
- word: GAII
  symbol: GA2
  source: hgnc_alias_symbol
  hgnc_symbol: ETFA
  entrez: '2108'
- word: maf
  symbol: MAF
  source: hgnc_symbol
  hgnc_symbol: MAF
  entrez: '4094'
- word: nat
  symbol: NAT
  source: hgnc_alias_symbol
  hgnc_symbol: BRD2
  entrez: '6046'
chemicals: []
diseases: []
figid_alias: PMC3338748__F2
redirect_from: /figures/PMC3338748__F2
figtype: Figure
---
