---
figid: PMC3585121__pgen.1003323.g006
figlink: /pmc/articles/PMC3585121/figure/pgen-1003323-g006/
number: Figure 6
caption: 'Tracks from top to bottom of each map represent the following: genes, repeats,
  MITEs, and graphs of AT (red) and GC (blue) contents. Each gene is represented by
  one or more boxes representing the coding sequences in exons, and an arrow indicating
  the direction of transcription. Double-slash marks (//) indicate sequence gaps within
  scaffolds of the assembled E. festucae genome sequences. Closed circles indicate
  telomeres, and distances from the telomere on the E. festucae map are indicated
  in kilobasepairs (kb). Cyan bars beneath each map represent repeat sequences, and
  are labeled with names or numbers to indicate relationships between repeats in the
  different species. Vertical bars beneath the repeat maps indicate MITEs. Gene names
  are abbreviated A through P for easA through easP, W for dmaW, and clo for cloA.
  Genes for synthesis of the ergoline ring system (skeleton) are shown in dark blue
  for the steps to chanoclavine-I (W, F, E, and C), and in light blue (D, A, and G)
  for steps to agroclavine. Genes for subsequent chemical decorations are shown in
  red (clo, H, O, P, lpsA, lpsB, and lpsC). Identifiable genes flanking the clusters
  are indicated in gray, and unfilled arrows indicate pseudogenes. The major pathway
  end-products for each strain are listed below each species name, abbreviated as
  indicated in , and in bold for those confirmed in this study. Note that LAH is a
  reported product of C. paspali, but the sequenced strain is predicted not to synthesize
  it due to a defective easE gene.'
pmcid: PMC3585121
papertitle: 'Plant-Symbiotic Fungi as Chemical Engineers: Multi-Genome Analysis of
  the Clavicipitaceae Reveals Dynamics of Alkaloid Loci.'
reftext: Christopher L. Schardl, et al. PLoS Genet. 2013 Feb;9(2):e1003323.
pmc_ranked_result_index: '191862'
pathway_score: 0.5570285
filename: pgen.1003323.g006.jpg
figtitle: 'Plant-Symbiotic Fungi as Chemical Engineers: Multi-Genome Analysis of the
  Clavicipitaceae Reveals Dynamics of Alkaloid Loci'
year: '2013'
organisms: Homo sapiens
ndex: da58205e-df1f-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3585121__pgen.1003323.g006.html
  '@type': Dataset
  description: 'Tracks from top to bottom of each map represent the following: genes,
    repeats, MITEs, and graphs of AT (red) and GC (blue) contents. Each gene is represented
    by one or more boxes representing the coding sequences in exons, and an arrow
    indicating the direction of transcription. Double-slash marks (//) indicate sequence
    gaps within scaffolds of the assembled E. festucae genome sequences. Closed circles
    indicate telomeres, and distances from the telomere on the E. festucae map are
    indicated in kilobasepairs (kb). Cyan bars beneath each map represent repeat sequences,
    and are labeled with names or numbers to indicate relationships between repeats
    in the different species. Vertical bars beneath the repeat maps indicate MITEs.
    Gene names are abbreviated A through P for easA through easP, W for dmaW, and
    clo for cloA. Genes for synthesis of the ergoline ring system (skeleton) are shown
    in dark blue for the steps to chanoclavine-I (W, F, E, and C), and in light blue
    (D, A, and G) for steps to agroclavine. Genes for subsequent chemical decorations
    are shown in red (clo, H, O, P, lpsA, lpsB, and lpsC). Identifiable genes flanking
    the clusters are indicated in gray, and unfilled arrows indicate pseudogenes.
    The major pathway end-products for each strain are listed below each species name,
    abbreviated as indicated in , and in bold for those confirmed in this study. Note
    that LAH is a reported product of C. paspali, but the sequenced strain is predicted
    not to synthesize it due to a defective easE gene.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - GNB5
  - TSTD1
  - UBXN11
  - DSG4
  - ESR1
  - SGCB
  - GNB1
  - GC
  - GNB2
  - GNB4
  - SAT1
  - CSPG5
  - LAT
  - GNB3
  - PRL
  - SELENBP1
  - ESR2
genes:
- word: GB
  symbol: G_beta
  source: bioentities_symbol
  hgnc_symbol: GNB5
  entrez: '10681'
- word: KAT
  symbol: KAT
  source: hgnc_alias_symbol
  hgnc_symbol: TSTD1
  entrez: '100131187'
- word: SOC
  symbol: SOC
  source: hgnc_alias_symbol
  hgnc_symbol: UBXN11
  entrez: '91544'
- word: LAH
  symbol: LAH
  source: hgnc_alias_symbol
  hgnc_symbol: DSG4
  entrez: '147409'
- word: ERA
  symbol: ER-alpha
  source: hgnc_alias_symbol
  hgnc_symbol: ESR1
  entrez: '2099'
- word: SGC
  symbol: SGC
  source: hgnc_alias_symbol
  hgnc_symbol: SGCB
  entrez: '6443'
- word: GB
  symbol: G_beta
  source: bioentities_symbol
  hgnc_symbol: GNB1
  entrez: '2782'
- word: '%GC'
  symbol: GC
  source: hgnc_symbol
  hgnc_symbol: GC
  entrez: '2638'
- word: GB
  symbol: G_beta
  source: bioentities_symbol
  hgnc_symbol: GNB2
  entrez: '2783'
- word: GB
  symbol: G_beta
  source: bioentities_symbol
  hgnc_symbol: GNB4
  entrez: '59345'
- word: SAT
  symbol: SAT
  source: hgnc_prev_symbol
  hgnc_symbol: SAT1
  entrez: '6303'
- word: NGC
  symbol: NGC
  source: hgnc_alias_symbol
  hgnc_symbol: CSPG5
  entrez: '10675'
- word: LAT
  symbol: LAT
  source: hgnc_symbol
  hgnc_symbol: LAT
  entrez: '27040'
- word: GB
  symbol: G_beta
  source: bioentities_symbol
  hgnc_symbol: GNB3
  entrez: '2784'
- word: PRL
  symbol: PRL
  source: hgnc_symbol
  hgnc_symbol: PRL
  entrez: '5617'
- word: lpsB
  symbol: LPSB
  source: hgnc_alias_symbol
  hgnc_symbol: SELENBP1
  entrez: '8991'
- word: ERB
  symbol: ER-beta
  source: hgnc_alias_symbol
  hgnc_symbol: ESR2
  entrez: '2100'
chemicals: []
diseases: []
---
