---
figid: PMC3134078__1345fig4
figlink: /pmc/articles/PMC3134078/figure/F4/
number: Figure 4
caption: Role of Dot1 in DNA damage checkpoint response and repair. The yeast proteins
  are shown, and the steps in which Dot1 has been implicated are marked by red stars.
  (A) In yeast, the PI-3 kinases Mec1/Ddc2 (human ATR/ATRIP) sense exposed ssDNA formed
  during repair of UV-, oxidative-, and alkylating-damaged DNA as well as resected
  DSBs. The sensor PI-3 kinase Tel (human ATM) recognizes blunt-end DSBs. After recognition
  and binding of sensor proteins to the site of DNA damage, mediator proteins such
  as Rad9, Dpb11, and Mrc1 are recruited and activated by phosphorylation. Dot1-mediated
  H3K79 methylation is required for the recruitment and activation of Rad9 and phosphorylation
  of the downstream transducer protein Rad53. Dot1 is also required for proper checkpoint
  function at both G1 and G2. (B) Several different DNA repair mechanisms can be used
  by the cell in response to different types of DNA damage. Damage by UV radiation
  can be repaired by NER, RR, and PRR. Loss of H3K79 methylation impairs all three
  pathways, resulting in hypersensitivity to UV damage. Damage by alkylating agents
  can be repaired by the BER pathway. In rare instances where the damage cannot be
  repaired, the error-prone TLS pathway can be activated to bypass the DNA lesion.
  It was demonstrated that Dot1 maintains DNA integrity by inhibiting TLS. Finally,
  IR, which generates DSBs, can be repaired by NHEJ. Alternatively, the DSBs can undergo
  5′-to-3′ resection and be repaired by HR. Methylation of H3K79 and the subsequent
  recruitment of Rad9 are required for regulating resection. Additionally, H3K79 methylation
  is required for the recruitment of cohesin, which is important for maintaining chromosome
  structure and for efficient SCR to repair DSBs.
pmcid: PMC3134078
papertitle: The diverse functions of Dot1 and H3K79 methylation.
reftext: Anh Tram Nguyen, et al. Genes Dev. 2011 Jul 1;25(13):1345-1358.
pmc_ranked_result_index: '171792'
pathway_score: 0.952072
filename: 1345fig4.jpg
figtitle: Role of Dot1 in DNA damage checkpoint response and repair
year: '2011'
organisms: Homo sapiens
ndex: a3a7046b-deb5-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3134078__1345fig4.html
  '@type': Dataset
  description: Role of Dot1 in DNA damage checkpoint response and repair. The yeast
    proteins are shown, and the steps in which Dot1 has been implicated are marked
    by red stars. (A) In yeast, the PI-3 kinases Mec1/Ddc2 (human ATR/ATRIP) sense
    exposed ssDNA formed during repair of UV-, oxidative-, and alkylating-damaged
    DNA as well as resected DSBs. The sensor PI-3 kinase Tel (human ATM) recognizes
    blunt-end DSBs. After recognition and binding of sensor proteins to the site of
    DNA damage, mediator proteins such as Rad9, Dpb11, and Mrc1 are recruited and
    activated by phosphorylation. Dot1-mediated H3K79 methylation is required for
    the recruitment and activation of Rad9 and phosphorylation of the downstream transducer
    protein Rad53. Dot1 is also required for proper checkpoint function at both G1
    and G2. (B) Several different DNA repair mechanisms can be used by the cell in
    response to different types of DNA damage. Damage by UV radiation can be repaired
    by NER, RR, and PRR. Loss of H3K79 methylation impairs all three pathways, resulting
    in hypersensitivity to UV damage. Damage by alkylating agents can be repaired
    by the BER pathway. In rare instances where the damage cannot be repaired, the
    error-prone TLS pathway can be activated to bypass the DNA lesion. It was demonstrated
    that Dot1 maintains DNA integrity by inhibiting TLS. Finally, IR, which generates
    DSBs, can be repaired by NHEJ. Alternatively, the DSBs can undergo 5′-to-3′ resection
    and be repaired by HR. Methylation of H3K79 and the subsequent recruitment of
    Rad9 are required for regulating resection. Additionally, H3K79 methylation is
    required for the recruitment of cohesin, which is important for maintaining chromosome
    structure and for efficient SCR to repair DSBs.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MRC1
  - CDC25B
  - ATM
  - NR1H2
  - CHEK2
  - FUS
  - RAD9A
  - ATR
  - CDC25A
  - TP53
  - CHEK1
  - CDC25C
genes:
- word: Mrc1
  symbol: MRC1
  source: hgnc_symbol
  hgnc_symbol: MRC1
  entrez: '4360'
- word: (Cdc25
  symbol: CDC25
  source: bioentities_symbol
  hgnc_symbol: CDC25B
  entrez: '994'
- word: Tel1
  symbol: TEL1
  source: hgnc_alias_symbol
  hgnc_symbol: ATM
  entrez: '472'
- word: '*NER'
  symbol: NER
  source: hgnc_alias_symbol
  hgnc_symbol: NR1H2
  entrez: '7376'
- word: Rad53
  symbol: RAD53
  source: hgnc_prev_symbol
  hgnc_symbol: CHEK2
  entrez: '11200'
- word: '*TLS'
  symbol: TLS
  source: hgnc_alias_symbol
  hgnc_symbol: FUS
  entrez: '2521'
- word: Rad9
  symbol: RAD9
  source: hgnc_prev_symbol
  hgnc_symbol: RAD9A
  entrez: '5883'
- word: Mec1
  symbol: MEC1
  source: hgnc_alias_symbol
  hgnc_symbol: ATR
  entrez: '545'
- word: (Cdc25
  symbol: CDC25
  source: bioentities_symbol
  hgnc_symbol: CDC25A
  entrez: '993'
- word: p53
  symbol: p53
  source: hgnc_alias_symbol
  hgnc_symbol: TP53
  entrez: '7157'
- word: Chk1
  symbol: CHK1
  source: hgnc_alias_symbol
  hgnc_symbol: CHEK1
  entrez: '1111'
- word: (Cdc25
  symbol: CDC25
  source: bioentities_symbol
  hgnc_symbol: CDC25C
  entrez: '995'
chemicals: []
diseases: []
---
