---
figid: PMC3905242__nihms-548067-f0002
figlink: /pmc/articles/PMC3905242/figure/F1/
number: Figure 1
caption: 'Double-stranded DNA (dsDNA) breaks are recognized by the MRE11 complex,
  which catalyses the activation of ataxia-telangiectasia mutated (ATM) in conjunction
  with other proteins such as the tat-interactive protein 60 kDa (TIP60; also known
  as KAT5) acetyltransferase and p53-binding protein 1 (53BP1),. ATM activation promotes
  cell-cycle checkpoint induction, influences DNA repair, and can activate apoptosis
  and senescence in certain cellular contexts. Depending on the cell-cycle phase and
  end-binding complexes or end modifications, breaks can be directed into two major
  repair pathways: homology-directed repair (HDR) or non-homologous end-joining (NHEJ;
  also known as classical (C)-NHEJ). a | HDR requires the 5′–3′ resection of dsDNA
  to generate single-stranded DNA (ssDNA)–dsDNA junctions. This is initiated by the
  MRE11 complex and CtBP-interacting protein (CtIP) and further bulk resection is
  carried out by exonuclease 1 (EXO1), BLM and DNA2 (,,–). 3′ ssDNA tails generated
  by resection are bound by replication protein A (RPA), which activates ATR via ATR-interacting
  protein (ATRIP) binding to influence the checkpoint response. RPA on these 3′ tails
  is exchanged for RAD51 to promote strand invasion, HDR repair and resolution of
  repair intermediates. b | Ends bound by the Ku70–Ku80 heterodimer can be repaired
  by NHEJ in conjunction with the DNA-dependent protein kinase catalytic subunit (DNA-PKcs),
  Artemis nuclease and DNA ligase 4, with the help of additional factors involved
  in end-modifications, gap filling and ligation. This NHEJ pathway is independent
  of the MRE11 complex. c | The MRE11 complex, in conjunction with CtIP, also regulates
  the poorly defined alternative NHEJ (A-NHEJ) pathway, which is characterized by
  large deletions and the frequent use of short microhomologies–,–. This pathway is
  resection-dependent and requires several enzymatic activities for resection, flap
  trimming, synthesis and ligation. CDK, cell division protein kinase; DSB, double-strand
  break.'
pmcid: PMC3905242
papertitle: 'The MRE11 complex: starting from the ends.'
reftext: Travis H. Stracker, et al. Nat Rev Mol Cell Biol. ;12(2):90-103.
pmc_ranked_result_index: '128419'
pathway_score: 0.9458759
filename: nihms-548067-f0002.jpg
figtitle: 'MRE11 complex: starting from the ends'
year: ''
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3905242__nihms-548067-f0002.html
  '@type': Dataset
  description: 'Double-stranded DNA (dsDNA) breaks are recognized by the MRE11 complex,
    which catalyses the activation of ataxia-telangiectasia mutated (ATM) in conjunction
    with other proteins such as the tat-interactive protein 60 kDa (TIP60; also known
    as KAT5) acetyltransferase and p53-binding protein 1 (53BP1),. ATM activation
    promotes cell-cycle checkpoint induction, influences DNA repair, and can activate
    apoptosis and senescence in certain cellular contexts. Depending on the cell-cycle
    phase and end-binding complexes or end modifications, breaks can be directed into
    two major repair pathways: homology-directed repair (HDR) or non-homologous end-joining
    (NHEJ; also known as classical (C)-NHEJ). a | HDR requires the 5′–3′ resection
    of dsDNA to generate single-stranded DNA (ssDNA)–dsDNA junctions. This is initiated
    by the MRE11 complex and CtBP-interacting protein (CtIP) and further bulk resection
    is carried out by exonuclease 1 (EXO1), BLM and DNA2 (,,–). 3′ ssDNA tails generated
    by resection are bound by replication protein A (RPA), which activates ATR via
    ATR-interacting protein (ATRIP) binding to influence the checkpoint response.
    RPA on these 3′ tails is exchanged for RAD51 to promote strand invasion, HDR repair
    and resolution of repair intermediates. b | Ends bound by the Ku70–Ku80 heterodimer
    can be repaired by NHEJ in conjunction with the DNA-dependent protein kinase catalytic
    subunit (DNA-PKcs), Artemis nuclease and DNA ligase 4, with the help of additional
    factors involved in end-modifications, gap filling and ligation. This NHEJ pathway
    is independent of the MRE11 complex. c | The MRE11 complex, in conjunction with
    CtIP, also regulates the poorly defined alternative NHEJ (A-NHEJ) pathway, which
    is characterized by large deletions and the frequent use of short microhomologies–,–.
    This pathway is resection-dependent and requires several enzymatic activities
    for resection, flap trimming, synthesis and ligation. CDK, cell division protein
    kinase; DSB, double-strand break.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PRKCQ
  - RBBP8
  - CHEK2
  - PRKCE
  - RASA1
  - CHEK1
  - PRKCZ
  - GATA3
  - PRKCG
  - PRKCI
  - CDK2
  - ALOX5AP
  - BLM
  - ENG
  - CDK1
  - PRKCD
  - ATR
  - CDC25A
  - PRKCH
  - CDC25C
  - PRKCA
  - PRKCB
  - PRKD3
  - DCLRE1C
  - ATM
  - MRE11
  - 5-25 nucleotides
genes:
- word: PKcs
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCQ
  entrez: '5588'
- word: CtIP
  symbol: CtIP
  source: hgnc_alias_symbol
  hgnc_symbol: RBBP8
  entrez: '5932'
- word: CHK2
  symbol: CHK2
  source: hgnc_alias_symbol
  hgnc_symbol: CHEK2
  entrez: '11200'
- word: PKcs
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCE
  entrez: '5581'
- word: Gap
  symbol: GAP
  source: hgnc_alias_symbol
  hgnc_symbol: RASA1
  entrez: '5921'
- word: CHK1
  symbol: CHK1
  source: hgnc_alias_symbol
  hgnc_symbol: CHEK1
  entrez: '1111'
- word: PKcs
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCZ
  entrez: '5590'
- word: HDR,
  symbol: HDR
  source: hgnc_alias_symbol
  hgnc_symbol: GATA3
  entrez: '2625'
- word: PKcs
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCG
  entrez: '5582'
- word: PKcs
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCI
  entrez: '5584'
- word: CDK1,CDK2
  symbol: CDK2
  source: hgnc_symbol
  hgnc_symbol: CDK2
  entrez: '1017'
- word: Flap
  symbol: FLAP
  source: hgnc_alias_symbol
  hgnc_symbol: ALOX5AP
  entrez: '241'
- word: BLM
  symbol: BLM
  source: hgnc_symbol
  hgnc_symbol: BLM
  entrez: '641'
- word: End
  symbol: END
  source: hgnc_alias_symbol
  hgnc_symbol: ENG
  entrez: '2022'
- word: CDK1,
  symbol: CDK1
  source: hgnc_symbol
  hgnc_symbol: CDK1
  entrez: '983'
- word: PKcs
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCD
  entrez: '5580'
- word: ATR
  symbol: ATR
  source: hgnc_symbol
  hgnc_symbol: ATR
  entrez: '545'
- word: CDC25A,
  symbol: CDC25A
  source: hgnc_symbol
  hgnc_symbol: CDC25A
  entrez: '993'
- word: PKcs
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCH
  entrez: '5583'
- word: CDC25C
  symbol: CDC25C
  source: hgnc_symbol
  hgnc_symbol: CDC25C
  entrez: '995'
- word: PKcs
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCA
  entrez: '5578'
- word: PKcs
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCB
  entrez: '5579'
- word: PKcs
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKD3
  entrez: '23683'
- word: Artemis
  symbol: ARTEMIS
  source: hgnc_alias_symbol
  hgnc_symbol: DCLRE1C
  entrez: '64421'
- word: ATM
  symbol: ATM
  source: hgnc_symbol
  hgnc_symbol: ATM
  entrez: '472'
- word: MRE11
  symbol: MRE11
  source: hgnc_symbol
  hgnc_symbol: MRE11
  entrez: '4361'
chemicals:
- word: 5-25 nucleotides
  source: MESH
  identifier: D009711
diseases: []
---
