---
figid: PMC3514620__fonc-02-00186-g001
figlink: /pmc/articles/PMC3514620/figure/F1/
number: F1
caption: Temozolomide function and activated DNA repair pathway. Temozolomide (TMZ)
  is chemically converted to MTIC (5-3-(methyltriazen-1-yl) imidazole-4-carboximide)
  at physiologic pH and degrades to a methyldiazonium cation, which transfers a methyl
  group to DNA. The most common site of methylation is N7-MeG (60–80%) followed by
  N3-MeA (10–20%) and O6-MeG (5–10%). When active MGMT is present, O6-MeG is repaired
  without cytotoxicity. When MGMT is inactivated or does not have the potential to
  completely repair O6-MeG, unrepaired O6-MeG is continuously repaired by the futile
  cycle of MMR, which ultimately induces cell death by provoking double-strand breaks
  (DSB). When MMR does not function properly, genomic instability is amplified. N7-MeG
  and N3-MeA is repaired by BER. If not repaired, alkylated bases cause a replication
  stall and collapse of the replication fork, generating single-strand breaks (SSBs),
  which ultimately induce DSB. It is possible that SSB and DSB repair pathways are
  activated and diminish the cytotoxic effects of TMZ.
pmcid: PMC3514620
papertitle: Complex DNA repair pathways as possible therapeutic targets to overcome
  temozolomide resistance in glioblastoma.
reftext: Koji Yoshimoto, et al. Front Oncol. 2012;2:186.
pmc_ranked_result_index: '130187'
pathway_score: 0.8533549
filename: fonc-02-00186-g001.jpg
figtitle: Temozolomide function and activated DNA repair pathway
year: '2012'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3514620__fonc-02-00186-g001.html
  '@type': Dataset
  description: Temozolomide function and activated DNA repair pathway. Temozolomide
    (TMZ) is chemically converted to MTIC (5-3-(methyltriazen-1-yl) imidazole-4-carboximide)
    at physiologic pH and degrades to a methyldiazonium cation, which transfers a
    methyl group to DNA. The most common site of methylation is N7-MeG (60–80%) followed
    by N3-MeA (10–20%) and O6-MeG (5–10%). When active MGMT is present, O6-MeG is
    repaired without cytotoxicity. When MGMT is inactivated or does not have the potential
    to completely repair O6-MeG, unrepaired O6-MeG is continuously repaired by the
    futile cycle of MMR, which ultimately induces cell death by provoking double-strand
    breaks (DSB). When MMR does not function properly, genomic instability is amplified.
    N7-MeG and N3-MeA is repaired by BER. If not repaired, alkylated bases cause a
    replication stall and collapse of the replication fork, generating single-strand
    breaks (SSBs), which ultimately induce DSB. It is possible that SSB and DSB repair
    pathways are activated and diminish the cytotoxic effects of TMZ.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MGMT
  - SSB
  - N7-meG
  - adenine
  - guanine
genes:
- word: MGMT
  symbol: MGMT
  source: hgnc_symbol
  hgnc_symbol: MGMT
  entrez: '4255'
- word: SSB
  symbol: SSB
  source: hgnc_symbol
  hgnc_symbol: SSB
  entrez: '6741'
chemicals:
- word: N7-meG
  source: MESH
  identifier: C008450
- word: adenine
  source: MESH
  identifier: D000225
- word: guanine
  source: MESH
  identifier: D006147
diseases: []
figid_alias: PMC3514620__F1
redirect_from: /figures/PMC3514620__F1
figtype: Figure
---
