---
figid: PMC3616974__pgen.1003373.g006
figlink: /pmc/articles/PMC3616974/figure/pgen-1003373-g006/
number: F6
caption: (A) Regions in CpG islands D and E that are DNA-methylated in non-malignant
  B-cells (left) become demethylated in the vast majority of CLL patients (right).
  This coincides with relaxed chromatin characterized by absence of macroH2A and enrichment
  of H3K4me2 at the promoters of the lncRNA genes DLEU1 and DLEU2/Alt1. (B) DNA-hypomethylation
  is correlated with transcriptional upregulation of splicing variants of the two
  lncRNA genes DLEU1 and DLEU2 and inversely correlated with the protein-coding genes
  in 13q14.3. No correlation could be found with levels of mature miR-15a and miR-16,
  probably because these transcripts are also deregulated by a posttranscriptional
  processing defect in CLL cells (Allegra, manuscript submitted). (C) Candidate genes
  localized in the critical region in 13q14.3 are functionally related and all modulate
  NF-kB signalling, albeit with different impact. Nuclear transporter KPNA3 is binding
  NF-kB components p65 and p50 and is therefore likely a positive regulator like e.g.
  RFP2. RFP2 binds to C13ORF1 and induces NF-kB activity via the canonical pathway
  components p50 and p65, for which its ubiquitin ligase actvity is required. The
  miRNA genes miR-15a and miR-16-1 were identified together with other members of
  this miR family to be among the strongest activators of NF-kB activity. Previously
  they have been shown to both modulate cell cycle regulators and inhibit NF-kB via
  TAB3/TAK1. DLEU7 has recently also been reported to inhibit NF-kB by binding and
  inactivating the TACI/BCMA receptors.
pmcid: PMC3616974
papertitle: Epigenetic Upregulation of lncRNAs at 13q14.3 in Leukemia Is Linked to
  the In Cis Downregulation of a Gene Cluster That Targets NF-kB.
reftext: Angela Garding, et al. PLoS Genet. 2013 Apr;9(4):e1003373.
pmc_ranked_result_index: '86872'
pathway_score: 0.373841
filename: pgen.1003373.g006.jpg
figtitle: Epigenetic Upregulation of lncRNAs at 13q14.3 in Leukemia Is Linked to the
  In Cis Downregulation of a Gene Cluster That Targets NF-kB
year: '2013'
organisms: Homo sapiens
ndex: 40f21b3e-df1f-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3616974__pgen.1003373.g006.html
  '@type': Dataset
  description: (A) Regions in CpG islands D and E that are DNA-methylated in non-malignant
    B-cells (left) become demethylated in the vast majority of CLL patients (right).
    This coincides with relaxed chromatin characterized by absence of macroH2A and
    enrichment of H3K4me2 at the promoters of the lncRNA genes DLEU1 and DLEU2/Alt1.
    (B) DNA-hypomethylation is correlated with transcriptional upregulation of splicing
    variants of the two lncRNA genes DLEU1 and DLEU2 and inversely correlated with
    the protein-coding genes in 13q14.3. No correlation could be found with levels
    of mature miR-15a and miR-16, probably because these transcripts are also deregulated
    by a posttranscriptional processing defect in CLL cells (Allegra, manuscript submitted).
    (C) Candidate genes localized in the critical region in 13q14.3 are functionally
    related and all modulate NF-kB signalling, albeit with different impact. Nuclear
    transporter KPNA3 is binding NF-kB components p65 and p50 and is therefore likely
    a positive regulator like e.g. RFP2. RFP2 binds to C13ORF1 and induces NF-kB activity
    via the canonical pathway components p50 and p65, for which its ubiquitin ligase
    actvity is required. The miRNA genes miR-15a and miR-16-1 were identified together
    with other members of this miR family to be among the strongest activators of
    NF-kB activity. Previously they have been shown to both modulate cell cycle regulators
    and inhibit NF-kB via TAB3/TAK1. DLEU7 has recently also been reported to inhibit
    NF-kB by binding and inactivating the TACI/BCMA receptors.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CHML
  - HIST2H2AC
  - NFKB1
  - IKBKG
  - TRIM13
  - TNFRSF17
  - DLEU7
  - KCNRG
  - IKBKB
  - GPT
  - KPNA3
  - CHUK
  - SPRYD7
  - TAB3
  - NR2C2
  - TNFRSF13B
genes:
- word: REP2
  symbol: REP-2
  source: hgnc_alias_symbol
  hgnc_symbol: CHML
  entrez: '1122'
- word: H2A
  symbol: H2A
  source: hgnc_prev_symbol
  hgnc_symbol: HIST2H2AC
  entrez: '8338'
- word: NF-kB
  symbol: NF-kappaB
  source: hgnc_alias_symbol
  hgnc_symbol: NFKB1
  entrez: '4790'
- word: IKK
  symbol: IKK
  source: bioentities_symbol
  hgnc_symbol: IKBKG
  entrez: '8517'
- word: RFP2
  symbol: RFP2
  source: hgnc_prev_symbol
  hgnc_symbol: TRIM13
  entrez: '10206'
- word: BCMA
  symbol: BCMA
  source: hgnc_prev_symbol
  hgnc_symbol: TNFRSF17
  entrez: '608'
- word: DLEU7
  symbol: DLEU7
  source: hgnc_symbol
  hgnc_symbol: DLEU7
  entrez: '220107'
- word: KCNRG|
  symbol: KCNRG
  source: hgnc_symbol
  hgnc_symbol: KCNRG
  entrez: '283518'
- word: IKK
  symbol: IKK
  source: bioentities_symbol
  hgnc_symbol: IKBKB
  entrez: '3551'
- word: Alt1
  symbol: ALT1
  source: hgnc_alias_symbol
  hgnc_symbol: GPT
  entrez: '2875'
- word: KPNA3
  symbol: KPNA3
  source: hgnc_symbol
  hgnc_symbol: KPNA3
  entrez: '3839'
- word: IKK
  symbol: IKK
  source: bioentities_symbol
  hgnc_symbol: CHUK
  entrez: '1147'
- word: C13ORF1
  symbol: C13orf1
  source: hgnc_prev_symbol
  hgnc_symbol: SPRYD7
  entrez: '57213'
- word: TAB3/
  symbol: TAB3
  source: hgnc_symbol
  hgnc_symbol: TAB3
  entrez: '257397'
- word: TAK1
  symbol: TAK1
  source: hgnc_alias_symbol
  hgnc_symbol: NR2C2
  entrez: '7182'
- word: TACI
  symbol: TACI
  source: hgnc_alias_symbol
  hgnc_symbol: TNFRSF13B
  entrez: '23495'
chemicals: []
diseases: []
figid_alias: PMC3616974__F6
redirect_from: /figures/PMC3616974__F6
figtype: Figure
---
