---
figid: PMC3731064__nihms492698f2
figlink: /pmc/articles/PMC3731064/figure/F2/
number: F2
caption: 'Following its formation, a double-strand break (DSB) is initially bound
  by the yeast Mre11–Rad50–Xrs2 (MRX) or human MRE11–RAD50–NBS1 (MRN) complex. If
  a DSB is formed in the G1 phase of the cell cycle, the DSB is preferentially repaired
  by the non-homologous end-joining (NHEJ) pathway (left panel). In this case, binding
  of the Ku heterodimer (Ku70–Ku80) prevents extensive nucleolytic processing of the
  DSB and promotes subsequent ligation of the DSB by the Dnl4–Lif1–Nej1 complex in
  budding yeast and the LIG4–XRCC4 factors in mammals. If a DSB is formed in the S
  or G2 cell cycle phase, the homologous recombination repair pathway predominates
  (right panel). Key steps for successful repair by homologous recombination include:
  nucleolytic processing of the 5′ ends of the dsDNA ends into 3′ single-stranded
  tails by the initial action of the yeast MRX complex and Sae2 enzyme (known as RBBP8
  or CTIP in mammals); extensive processing by either the exodeoxyribonuclease 1 (Exo1)
  or the Dna2–Sgs1 DNA-end-processing enzymes (the human Bloom’s syndrome protein
  (BLM) helicase is the homologue of yeast Sgs1); assembly of a recombination protein
  A (RPA)–ssDNA filament that is subsequently converted into a Rad51–ssDNA filament;
  completion of a successful homology search and formation of heteroduplex DNA; DNA
  synthesis that uses the 3′ end of the broken DNA, resolution of the heteroduplex
  or a double-sided Holliday junction; and, finally, ligation of the ssDNA nicks and
  termination of the process. Note that formation of the RPA–ssDNA filament is also
  essential for activation of the DNA damage cell cycle checkpoint. For a recent Review,
  see REF. . Protein or complex names given in brackets in the figure are the human
  homologues.'
pmcid: PMC3731064
papertitle: Chromatin and the genome integrity network.
reftext: Manolis Papamichos-Chronakis, et al. Nat Rev Genet. ;14(1):62-75.
pmc_ranked_result_index: '99431'
pathway_score: 0.9649352
filename: nihms492698f2.jpg
figtitle: Chromatin and the genome integrity network
year: ''
organisms:
- Mus musculus
- Arabidopsis thaliana
- Schizosaccharomyces pombe
- Saccharomyces cerevisiae
- Drosophila melanogaster
- Xenopus laevis
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3731064__nihms492698f2.html
  '@type': Dataset
  description: 'Following its formation, a double-strand break (DSB) is initially
    bound by the yeast Mre11–Rad50–Xrs2 (MRX) or human MRE11–RAD50–NBS1 (MRN) complex.
    If a DSB is formed in the G1 phase of the cell cycle, the DSB is preferentially
    repaired by the non-homologous end-joining (NHEJ) pathway (left panel). In this
    case, binding of the Ku heterodimer (Ku70–Ku80) prevents extensive nucleolytic
    processing of the DSB and promotes subsequent ligation of the DSB by the Dnl4–Lif1–Nej1
    complex in budding yeast and the LIG4–XRCC4 factors in mammals. If a DSB is formed
    in the S or G2 cell cycle phase, the homologous recombination repair pathway predominates
    (right panel). Key steps for successful repair by homologous recombination include:
    nucleolytic processing of the 5′ ends of the dsDNA ends into 3′ single-stranded
    tails by the initial action of the yeast MRX complex and Sae2 enzyme (known as
    RBBP8 or CTIP in mammals); extensive processing by either the exodeoxyribonuclease
    1 (Exo1) or the Dna2–Sgs1 DNA-end-processing enzymes (the human Bloom’s syndrome
    protein (BLM) helicase is the homologue of yeast Sgs1); assembly of a recombination
    protein A (RPA)–ssDNA filament that is subsequently converted into a Rad51–ssDNA
    filament; completion of a successful homology search and formation of heteroduplex
    DNA; DNA synthesis that uses the 3′ end of the broken DNA, resolution of the heteroduplex
    or a double-sided Holliday junction; and, finally, ligation of the ssDNA nicks
    and termination of the process. Note that formation of the RPA–ssDNA filament
    is also essential for activation of the DNA damage cell cycle checkpoint. For
    a recent Review, see REF. . Protein or complex names given in brackets in the
    figure are the human homologues.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - DNA2
  - BLM
  - UBA2
  - RBBP8
  - EXO1
genes:
- word: Dna2
  symbol: DNA2
  source: hgnc_symbol
  hgnc_symbol: DNA2
  entrez: '1763'
- word: (BLM)
  symbol: BLM
  source: hgnc_symbol
  hgnc_symbol: BLM
  entrez: '641'
- word: Sae2
  symbol: SAE2
  source: hgnc_prev_symbol
  hgnc_symbol: UBA2
  entrez: '10054'
- word: (RBBP8)
  symbol: RBBP8
  source: hgnc_symbol
  hgnc_symbol: RBBP8
  entrez: '5932'
- word: Exo1
  symbol: EXO1
  source: hgnc_symbol
  hgnc_symbol: EXO1
  entrez: '9156'
chemicals: []
diseases: []
figid_alias: PMC3731064__F2
redirect_from: /figures/PMC3731064__F2
figtype: Figure
---
