---
figid: PMC3561014__PP_209254_f2
figlink: /pmc/articles/PMC3561014/figure/fig2/
number: F2
caption: 'Adjustments in root metabolism promote acclimation to  deficiency. Shown
  are modifications in white lupin cluster root metabolism that facilitate acclimation
  to  deficiency as evidenced by transcript expression. Increased expression in  was
  confirmed by  as indicated by fold change in parentheses. Red arrows represent genes
  known to be up-regulated due to  deficiency. Increased Suc metabolism via glycolysis
  and organic acid production provide carbon for malate and citrate exudation into
  the rhizosphere. Organic acids lost through exudation are replenished through anaplerotic
  pathways involving phosphoenolpyruvate carboxylase (PEPC) and a glyoxylate-like
  cycle malate synthase (MS). One-carbon metabolism is enhanced through a formate
  and  pathway. The  pathway contributes to Met and ethylene production. Increased
  expression of transcripts involved in phospholipid degradation releases  for recycling
  and carbon for acetyl-CoA synthesis. Acetyl-CoA and glyoxylate provide carbon for
  malate synthesis through malate synthase. Formate may be carboxylated to glyoxylate
  by a putative glyoxylate synthase (Glox-S). Additional abbreviations are as follows:
  Suc synthase (SS); phosphoenolpyruvate carboxylase kinase (PEPCK); phosphoenolpyruvate
  carboxylase carboxykinase (PEPCRK); malate dehydrogenase (MDH); citrate synthase
  (CS); formamidase (FORM); formate dehydrogenase (FDH); Ala glyoxylate transaminase
  (AGTA); THF deformylase (THFD); methylene- reductase (MTHFR); S-adenosyl-Met synthase
  (SAM Synth); aminocyclopropane synthase (ACC Synth); aminocyclopropane oxidase (ACC
  Oxidase); phospholipase A (PLA1); glycerophosphodiester-phosphodiesterase (GPX-PDE);
  acid phosphatase (AP); monogalactosyldiacylglycerol synthase (MGDG synthase); sulfoquinovosyltransferase
  (SQD).'
pmcid: PMC3561014
papertitle: An RNA-Seq Transcriptome Analysis of Orthophosphate-Deficient White Lupin
  Reveals Novel Insights into Phosphorus Acclimation in Plants.
reftext: Jamie A. Oâ€™Rourke, et al. Plant Physiol. 2013 Feb;161(2):705-724.
pmc_ranked_result_index: '56738'
pathway_score: 0.9487874
filename: PP_209254_f2.jpg
figtitle: Adjustments in root metabolism promote acclimation to  deficiency
year: '2013'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3561014__PP_209254_f2.html
  '@type': Dataset
  description: 'Adjustments in root metabolism promote acclimation to  deficiency.
    Shown are modifications in white lupin cluster root metabolism that facilitate
    acclimation to  deficiency as evidenced by transcript expression. Increased expression
    in  was confirmed by  as indicated by fold change in parentheses. Red arrows represent
    genes known to be up-regulated due to  deficiency. Increased Suc metabolism via
    glycolysis and organic acid production provide carbon for malate and citrate exudation
    into the rhizosphere. Organic acids lost through exudation are replenished through
    anaplerotic pathways involving phosphoenolpyruvate carboxylase (PEPC) and a glyoxylate-like
    cycle malate synthase (MS). One-carbon metabolism is enhanced through a formate
    and  pathway. The  pathway contributes to Met and ethylene production. Increased
    expression of transcripts involved in phospholipid degradation releases  for recycling
    and carbon for acetyl-CoA synthesis. Acetyl-CoA and glyoxylate provide carbon
    for malate synthesis through malate synthase. Formate may be carboxylated to glyoxylate
    by a putative glyoxylate synthase (Glox-S). Additional abbreviations are as follows:
    Suc synthase (SS); phosphoenolpyruvate carboxylase kinase (PEPCK); phosphoenolpyruvate
    carboxylase carboxykinase (PEPCRK); malate dehydrogenase (MDH); citrate synthase
    (CS); formamidase (FORM); formate dehydrogenase (FDH); Ala glyoxylate transaminase
    (AGTA); THF deformylase (THFD); methylene- reductase (MTHFR); S-adenosyl-Met synthase
    (SAM Synth); aminocyclopropane synthase (ACC Synth); aminocyclopropane oxidase
    (ACC Oxidase); phospholipase A (PLA1); glycerophosphodiester-phosphodiesterase
    (GPX-PDE); acid phosphatase (AP); monogalactosyldiacylglycerol synthase (MGDG
    synthase); sulfoquinovosyltransferase (SQD).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SLTM
  - PAEP
  - ADH5
  - PCK2
  - ACACA
  - MTHFR
  - CS
  - C2
  - ACACB
genes:
- word: Met
  symbol: Met
  source: hgnc_alias_symbol
  hgnc_symbol: SLTM
  entrez: '79811'
- word: PEP
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
- word: FDH
  symbol: FDH
  source: hgnc_prev_symbol
  hgnc_symbol: ADH5
  entrez: '128'
- word: PEPCK
  symbol: PEPCK
  source: hgnc_alias_symbol
  hgnc_symbol: PCK2
  entrez: '5106'
- word: ACC
  symbol: ACC
  source: bioentities_symbol
  hgnc_symbol: ACACA
  entrez: '31'
- word: MTHFR
  symbol: MTHFR
  source: hgnc_symbol
  hgnc_symbol: MTHFR
  entrez: '4524'
- word: cs
  symbol: CS
  source: hgnc_symbol
  hgnc_symbol: CS
  entrez: '1431'
- word: (C2
  symbol: C2
  source: hgnc_symbol
  hgnc_symbol: C2
  entrez: '717'
- word: ACC
  symbol: ACC
  source: bioentities_symbol
  hgnc_symbol: ACACB
  entrez: '32'
chemicals: []
diseases: []
figid_alias: PMC3561014__F2
redirect_from: /figures/PMC3561014__F2
figtype: Figure
---
