---
figid: PMC3952039__nihms523529f1
figlink: /pmc/articles/PMC3952039/figure/F1/
number: Figure 1
caption: '(A) Illustration shows RET gene structure with 20 exons (solid boxes) and
  the putative two major RET isoforms: short RET 9 (1072 aa) or long RET 51 (1114
  aa). The different isoforms are generated by alternative splicing and differ only
  in the number of aa residues in the cytosolic tail. RET 9 has nine amino acids at
  the carboxyl terminus that are different than the 51 terminal residues in RET 51.
  The extracellular domain consists of 4 cadherin motifs. The key tyrosine (Y) residues
  in the cytoplasmic domain are labeled. Note that the RET 51 isoform has an additional
  tyrosine residue Y1096. (B) Schematic shows RET signaling components using RET 51
  as an example. GFRα1, the essential co-receptor of RET, is a glycophosphatidylinositol
  linked protein present on cell surface in the epithelia or in the mesenchyme. Two
  molecules of GDNF, ligand for RET signaling, bind to homodimers of Gfrα1 which then
  recruit two RET molecules in the cell membrane. Formation of this hexameric complex
  induces autophosphorylation of specific tyrosine residues in the intracellular kinase
  domain that serve as docking sites for the intracellular adaptors and activates
  the indicated downstream signaling cascades. Asterisks on SHC denote that other
  adaptors including SHC, IRS1–2, FRS2, DOK 1–6, and ENIGMA can also bind to the multi-docking
  Y1062 site. Activated SHC can either trigger the PI3K-AKT pathway or the ERK-MAPK
  pathway depending on the adaptor complex recruited. RET 51 has the additional residue
  Y1096 which can also activate the PI3K-AKT pathway. (aa = amino acids; UB = ureteric
  bud; PLCγ = phospholipase Cγ; PKC = protein kinase C)'
pmcid: PMC3952039
papertitle: 'To Bud or not to Bud: The RET perspective in CAKUT.'
reftext: T. Keefe Davis, et al. Pediatr Nephrol. ;29(4):597-608.
pmc_ranked_result_index: '125530'
pathway_score: 0.5857289
filename: nihms523529f1.jpg
figtitle: 'To Bud or not to Bud: The RET perspective in CAKUT'
year: ''
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC3952039__nihms523529f1.html
  '@type': Dataset
  description: '(A) Illustration shows RET gene structure with 20 exons (solid boxes)
    and the putative two major RET isoforms: short RET 9 (1072 aa) or long RET 51
    (1114 aa). The different isoforms are generated by alternative splicing and differ
    only in the number of aa residues in the cytosolic tail. RET 9 has nine amino
    acids at the carboxyl terminus that are different than the 51 terminal residues
    in RET 51. The extracellular domain consists of 4 cadherin motifs. The key tyrosine
    (Y) residues in the cytoplasmic domain are labeled. Note that the RET 51 isoform
    has an additional tyrosine residue Y1096. (B) Schematic shows RET signaling components
    using RET 51 as an example. GFRα1, the essential co-receptor of RET, is a glycophosphatidylinositol
    linked protein present on cell surface in the epithelia or in the mesenchyme.
    Two molecules of GDNF, ligand for RET signaling, bind to homodimers of Gfrα1 which
    then recruit two RET molecules in the cell membrane. Formation of this hexameric
    complex induces autophosphorylation of specific tyrosine residues in the intracellular
    kinase domain that serve as docking sites for the intracellular adaptors and activates
    the indicated downstream signaling cascades. Asterisks on SHC denote that other
    adaptors including SHC, IRS1–2, FRS2, DOK 1–6, and ENIGMA can also bind to the
    multi-docking Y1062 site. Activated SHC can either trigger the PI3K-AKT pathway
    or the ERK-MAPK pathway depending on the adaptor complex recruited. RET 51 has
    the additional residue Y1096 which can also activate the PI3K-AKT pathway. (aa
    = amino acids; UB = ureteric bud; PLCγ = phospholipase Cγ; PKC = protein kinase
    C)'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MAPK8
  - GDNF
  - MAPK3
  - SHC2
  - MAPK11
  - MAPK1
  - MAPK13
  - HRAS
  - MAPK9
  - PLCG2
  - PRKD3
  - NRAS
  - SHC1
  - PRKCI
  - RET
  - KRAS
  - MAPK12
  - PRKCD
  - SHC3
  - PRKCB
  - PLCG1
  - MAPK10
  - MAPK14
  - PRKCA
  - SRC
  - PRKCH
  - PRKCQ
  - PRKCZ
  - PRKCE
  - SHC4
  - PRKCG
  - Cancer
genes:
- word: '>RAS>ERK/MAPK'
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK8
  entrez: '5599'
- word: GDNF
  symbol: GDNF
  source: hgnc_symbol
  hgnc_symbol: GDNF
  entrez: '2668'
- word: ERK/MAPK
  symbol: ERK
  source: bioentities_symbol
  hgnc_symbol: MAPK3
  entrez: '5595'
- word: SHC*
  symbol: SHC
  source: bioentities_symbol
  hgnc_symbol: SHC2
  entrez: '25759'
- word: '>RAS>ERK/MAPK'
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK11
  entrez: '5600'
- word: ERK/MAPK
  symbol: ERK
  source: bioentities_symbol
  hgnc_symbol: MAPK1
  entrez: '5594'
- word: '>RAS>ERK/MAPK'
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK13
  entrez: '5603'
- word: RAS>
  symbol: RAS
  source: bioentities_symbol
  hgnc_symbol: HRAS
  entrez: '3265'
- word: '>RAS>ERK/MAPK'
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK9
  entrez: '5601'
- word: PLCY→
  symbol: PLCG
  source: bioentities_symbol
  hgnc_symbol: PLCG2
  entrez: '5336'
- word: PKC
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKD3
  entrez: '23683'
- word: RAS>
  symbol: RAS
  source: bioentities_symbol
  hgnc_symbol: NRAS
  entrez: '4893'
- word: SHC*
  symbol: SHC
  source: bioentities_symbol
  hgnc_symbol: SHC1
  entrez: '6464'
- word: PKC
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCI
  entrez: '5584'
- word: RET51
  symbol: RET51
  source: hgnc_alias_symbol
  hgnc_symbol: RET
  entrez: '5979'
- word: RAS>
  symbol: RAS
  source: bioentities_symbol
  hgnc_symbol: KRAS
  entrez: '3845'
- word: '>RAS>ERK/MAPK'
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK3
  entrez: '5595'
- word: '>RAS>ERK/MAPK'
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK12
  entrez: '6300'
- word: PKC
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCD
  entrez: '5580'
- word: SHC*
  symbol: SHC
  source: bioentities_symbol
  hgnc_symbol: SHC3
  entrez: '53358'
- word: '>RAS>ERK/MAPK'
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK1
  entrez: '5594'
- word: PKC
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCB
  entrez: '5579'
- word: PLCY→
  symbol: PLCG
  source: bioentities_symbol
  hgnc_symbol: PLCG1
  entrez: '5335'
- word: RET
  symbol: RET
  source: hgnc_symbol
  hgnc_symbol: RET
  entrez: '5979'
- word: '>RAS>ERK/MAPK'
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK10
  entrez: '5602'
- word: '>RAS>ERK/MAPK'
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK14
  entrez: '1432'
- word: PKC
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCA
  entrez: '5578'
- word: SRC
  symbol: SRC
  source: hgnc_symbol
  hgnc_symbol: SRC
  entrez: '6714'
- word: PKC
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCH
  entrez: '5583'
- word: PKC
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCQ
  entrez: '5588'
- word: PKC
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCZ
  entrez: '5590'
- word: PKC
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCE
  entrez: '5581'
- word: SHC*
  symbol: SHC
  source: bioentities_symbol
  hgnc_symbol: SHC4
  entrez: '399694'
- word: PKC
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCG
  entrez: '5582'
chemicals: []
diseases:
- word: Cancer
  source: ''
  identifier: ''
---
