---
figid: PMC4981676__zmr0031624340002
figlink: /pmc/articles/PMC4981676/figure/F2/
number: FIG 2
caption: Structure and function of JIP1 acting within the JNK pathway. (A) The domain
  architecture of the JIP1 protein. The N-terminal regulatory “tail” is largely disordered,
  while the C-terminal half of JIP1 contains three folded domains as well as a kinesin
  light chain (KLC)-binding linear motif. The precise function of the intrinsically
  disordered N terminus (with its conserved acidic motifs) is unknown, yet it is highly
  phosphorylated by JNK in a D-motif-dependent manner. Currently, only two target
  sites (T103 and S421) are known to have a role in JNK-dependent physiological regulation
  of JIP1. This model was built by combining domain signature searches (PFAM), folding
  tendency predictors (IUPRED), and conservation analyses (multiple alignments among
  vertebrate proteins) as well as curated data from the literature. The lower line
  shows the results of conservation analyses (red, highly conserved sequence; blue,
  nonconserved sequence), when sequences of vertebrate JIP1 and the closely related
  JIP2 proteins are aligned with each other. Structural domains and key motifs are
  preserved in both proteins (including the JNK-binding D-motif), while most regulatory
  phosphorylation sites differ between JIP1 and JIP2. (B) A model of JIP1 actions
  on the microtubule-dependent transport processes in neurons. The JIP1/2 dimers (turquoise)
  are capable of transporting a diverse set of membrane-associated proteins (e.g.,
  β-APP, APoE2-R) as well as certain MAP2Ks (MKK7 [red]) and inactive MAP3Ks (MLK3,
  DLK [magenta]). These complexes are moved along the microtubule filaments with the
  help of kinesin 1-kinesin light chain 1 motors (blue). At the end of their journey,
  the transport complexes are uncoupled by a JNK-dependent phosphorylation of JIP1.
  Since this step also results in the release of upstream components and activators
  belonging to the JNK pathway, it leads to a positive feedback loop and helps to
  maintain subcellular compartments with high local JNK activity. The JIP1/2 proteins
  uncoupled from their cargo are also transported in a reverse direction (likely through
  a dynein-driven process), although the structural details of the latter complex
  are poorly known.
pmcid: PMC4981676
papertitle: 'JNK Signaling: Regulation and Functions Based on Complex Protein-Protein
  Partnerships.'
reftext: András Zeke, et al. Microbiol Mol Biol Rev. 2016 Sep;80(3):793-835.
pmc_ranked_result_index: '163353'
pathway_score: 0.7173681
filename: zmr0031624340002.jpg
figtitle: Structure and function of JIP1 acting within the JNK pathway
year: '2016'
organisms: Homo sapiens
ndex: 669b3dd2-df32-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4981676__zmr0031624340002.html
  '@type': Dataset
  description: Structure and function of JIP1 acting within the JNK pathway. (A) The
    domain architecture of the JIP1 protein. The N-terminal regulatory “tail” is largely
    disordered, while the C-terminal half of JIP1 contains three folded domains as
    well as a kinesin light chain (KLC)-binding linear motif. The precise function
    of the intrinsically disordered N terminus (with its conserved acidic motifs)
    is unknown, yet it is highly phosphorylated by JNK in a D-motif-dependent manner.
    Currently, only two target sites (T103 and S421) are known to have a role in JNK-dependent
    physiological regulation of JIP1. This model was built by combining domain signature
    searches (PFAM), folding tendency predictors (IUPRED), and conservation analyses
    (multiple alignments among vertebrate proteins) as well as curated data from the
    literature. The lower line shows the results of conservation analyses (red, highly
    conserved sequence; blue, nonconserved sequence), when sequences of vertebrate
    JIP1 and the closely related JIP2 proteins are aligned with each other. Structural
    domains and key motifs are preserved in both proteins (including the JNK-binding
    D-motif), while most regulatory phosphorylation sites differ between JIP1 and
    JIP2. (B) A model of JIP1 actions on the microtubule-dependent transport processes
    in neurons. The JIP1/2 dimers (turquoise) are capable of transporting a diverse
    set of membrane-associated proteins (e.g., β-APP, APoE2-R) as well as certain
    MAP2Ks (MKK7 [red]) and inactive MAP3Ks (MLK3, DLK [magenta]). These complexes
    are moved along the microtubule filaments with the help of kinesin 1-kinesin light
    chain 1 motors (blue). At the end of their journey, the transport complexes are
    uncoupled by a JNK-dependent phosphorylation of JIP1. Since this step also results
    in the release of upstream components and activators belonging to the JNK pathway,
    it leads to a positive feedback loop and helps to maintain subcellular compartments
    with high local JNK activity. The JIP1/2 proteins uncoupled from their cargo are
    also transported in a reverse direction (likely through a dynein-driven process),
    although the structural details of the latter complex are poorly known.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MAPK8IP1
  - MAPK9
  - PTBP2
  - KLC1
  - MAPK8
  - MAP2K7
  - MAPK10
  - LRP8
  - MAPK8IP2
genes:
- word: JIP1
  symbol: JIP-1
  source: hgnc_alias_symbol
  hgnc_symbol: MAPK8IP1
  entrez: '9479'
- word: JNK
  symbol: JNK
  source: bioentities_symbol
  hgnc_symbol: MAPK9
  entrez: '5601'
- word: PTB
  symbol: PTB
  source: hgnc_alias_symbol
  hgnc_symbol: PTBP2
  entrez: '58155'
- word: KLC
  symbol: KLC
  source: hgnc_alias_symbol
  hgnc_symbol: KLC1
  entrez: '3831'
- word: JNK
  symbol: JNK
  source: bioentities_symbol
  hgnc_symbol: MAPK8
  entrez: '5599'
- word: MKK7
  symbol: MKK7
  source: hgnc_alias_symbol
  hgnc_symbol: MAP2K7
  entrez: '5609'
- word: JNK
  symbol: JNK
  source: bioentities_symbol
  hgnc_symbol: MAPK10
  entrez: '5602'
- word: ApoER2
  symbol: APOER2
  source: hgnc_alias_symbol
  hgnc_symbol: LRP8
  entrez: '7804'
- word: KLC1
  symbol: KLC1
  source: hgnc_symbol
  hgnc_symbol: KLC1
  entrez: '3831'
- word: (JIP1/JIP2)
  symbol: JIP2
  source: hgnc_alias_symbol
  hgnc_symbol: MAPK8IP2
  entrez: '23542'
chemicals: []
diseases: []
---
