---
figid: PMC4696776__nihms379045f2a
figlink: /pmc/articles/PMC149053/figure/fig7/
number: F2a
caption: 'Figure 2A. Predicting response to induction chemotherapy in AMLA. Modulators
  and networks assessed. Schematic of the cell signalling pathways probed using SCNP.
  Signalling node is defined as a combination of a specific proteomic read out in
  the presence or absence of a specific modulator. Text in blue indicates assay readouts.
  Biological categories of signalling pathways and nodes are indicated by colour:
  1) Chemokine, Cytokine, Growth factor Signal Induction (green), 2) Functional Apoptosis
  (pink), 3) Phosphatase Regulation/Reactive Oxygen Species (ROS) Stress(purple) and
  4) Surface Markers/Drug Transporters (orange).Figure 2B. Apoptosis pathway distinguishes
  CR in younger patients with AML. Functional apoptosis assays can stratify NR and
  CR responses in diagnostic AML leukemia samples. Shown in the side panel is a schema
  for functional apoptosis after DNA damage. In this example the modulator is etoposide
  and the readouts are cleaved PARP and p-Chk2. The upper left panel shows the AUC
  of the ROC (0.81) for the signalling node etoposide → p-Chk2, cleaved PARP+. The
  upper right panel shows the box and whisker plots with standard deviation shown.
  Circles represent individual patient samples percentage of PARP positive and p–Chk2
  negative cells. The frequency of p-CHK2-, Cleaved PARP+ (c-PARP+) apoptotic cells
  (upper left quadrant of the flow cytometry plots) after overnight exposure to Etoposide
  was used to quantify apoptosis.Figure 2C. Signalling nodes add value to age alone
  for induction response stratification. PMA induced p-CREB readout improves upon
  age as a classifier for response to induction therapy. Significant p-values (slope
  = 0) for the signalling node were found after accounting for age in the logistic
  regression model. The ROC plot illustrates CR and NR stratification using the model
  incorporating the signalling node (PMA → p-CREB) with age.'
pmcid: PMC4696776
papertitle: Insights into acute myeloid leukemia via single cell network profiling.
reftext: Steven M. Kornblau, et al. Clin Lab Int. 2010 Feb-Mar;34(1):12-15.
pmc_ranked_result_index: '31391'
pathway_score: 0.9054432
filename: nihms379045f2a.jpg
figtitle: Insights into acute myeloid leukemia via single cell network profiling
year: '2010'
organisms:
- clinical samples
- Homo sapiens
ndex: 074d5e89-df06-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4696776__nihms379045f2a.html
  '@type': Dataset
  description: 'Figure 2A. Predicting response to induction chemotherapy in AMLA.
    Modulators and networks assessed. Schematic of the cell signalling pathways probed
    using SCNP. Signalling node is defined as a combination of a specific proteomic
    read out in the presence or absence of a specific modulator. Text in blue indicates
    assay readouts. Biological categories of signalling pathways and nodes are indicated
    by colour: 1) Chemokine, Cytokine, Growth factor Signal Induction (green), 2)
    Functional Apoptosis (pink), 3) Phosphatase Regulation/Reactive Oxygen Species
    (ROS) Stress(purple) and 4) Surface Markers/Drug Transporters (orange).Figure
    2B. Apoptosis pathway distinguishes CR in younger patients with AML. Functional
    apoptosis assays can stratify NR and CR responses in diagnostic AML leukemia samples.
    Shown in the side panel is a schema for functional apoptosis after DNA damage.
    In this example the modulator is etoposide and the readouts are cleaved PARP and
    p-Chk2. The upper left panel shows the AUC of the ROC (0.81) for the signalling
    node etoposide → p-Chk2, cleaved PARP+. The upper right panel shows the box and
    whisker plots with standard deviation shown. Circles represent individual patient
    samples percentage of PARP positive and p–Chk2 negative cells. The frequency of
    p-CHK2-, Cleaved PARP+ (c-PARP+) apoptotic cells (upper left quadrant of the flow
    cytometry plots) after overnight exposure to Etoposide was used to quantify apoptosis.Figure
    2C. Signalling nodes add value to age alone for induction response stratification.
    PMA induced p-CREB readout improves upon age as a classifier for response to induction
    therapy. Significant p-values (slope = 0) for the signalling node were found after
    accounting for age in the logistic regression model. The ROC plot illustrates
    CR and NR stratification using the model incorporating the signalling node (PMA
    → p-CREB) with age.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PRKD3
  - PRKCA
  - NRAS
  - STAT5B
  - CREB3L3
  - AKT3
  - CREB3L2
  - PLCG2
  - PRKCD
  - PRKCZ
  - PRKCQ
  - AKT1
  - STAT6
  - CREM
  - EIF2AK3
  - PRKCE
  - PRKCG
  - PRKCH
  - PRKCI
  - CXCL2
  - CXCL3
  - STAT5A
  - LCP2
  - STAT1
  - STAT3
  - AKT2
  - CHEK2
  - ATF1
  - HRAS
  - CXCL11
  - CXCL5
  - CXCL14
  - STAT4
  - CREB3L4
  - STAT2
  - CD9
  - KRAS
  - PRKCB
  - ABCG2
  - CXCL12
  - CREB3L1
  - CREB5
  - CREB1
  - CXCR4
  - CREB3
  - ATF2
  - IL27
  - CXCL8
  - CXCL10
  - CXCL9
  - PF4
  - PPBP
  - CXCL6
  - CXCL13
  - CXCL1
  - KIT
  - KITLG
  - CSF3
  - PI3
  - MAPK1
  - MAPK3
  - Etoposide
  - Cancer
genes:
- word: PKC,
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKD3
  entrez: '23683'
- word: PKC,
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCA
  entrez: '5578'
- word: Ras
  symbol: RAS
  source: bioentities_symbol
  hgnc_symbol: NRAS
  entrez: '4893'
- word: Stats
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT5B
  entrez: '6777'
- word: p-Creb
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L3
  entrez: '84699'
- word: p-Akt
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT3
  entrez: '10000'
- word: p-Creb
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L2
  entrez: '64764'
- word: p-Plcy2
  symbol: PLCG2
  source: hgnc_symbol
  hgnc_symbol: PLCG2
  entrez: '5336'
- word: PKC,
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCD
  entrez: '5580'
- word: PKC,
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCZ
  entrez: '5590'
- word: PKC,
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCQ
  entrez: '5588'
- word: p-Akt
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT1
  entrez: '207'
- word: Stats
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT6
  entrez: '6778'
- word: p-Creb
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREM
  entrez: '1390'
- word: p-Erk
  symbol: PERK
  source: hgnc_alias_symbol
  hgnc_symbol: EIF2AK3
  entrez: '9451'
- word: PKC,
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCE
  entrez: '5581'
- word: PKC,
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCG
  entrez: '5582'
- word: PKC,
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCH
  entrez: '5583'
- word: PKC,
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCI
  entrez: '5584'
- word: Chemokines,
  symbol: Chemokine
  source: bioentities_symbol
  hgnc_symbol: CXCL2
  entrez: '2920'
- word: Chemokines,
  symbol: Chemokine
  source: bioentities_symbol
  hgnc_symbol: CXCL3
  entrez: '2921'
- word: Stats
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT5A
  entrez: '6776'
- word: p-SLP76
  symbol: SLP-76
  source: hgnc_alias_symbol
  hgnc_symbol: LCP2
  entrez: '3937'
- word: p-Stat1
  symbol: STAT1
  source: hgnc_symbol
  hgnc_symbol: STAT1
  entrez: '6772'
- word: p-Stat3
  symbol: STAT3
  source: hgnc_symbol
  hgnc_symbol: STAT3
  entrez: '6774'
- word: p-Akt
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT2
  entrez: '208'
- word: p-Chk2
  symbol: CHK2
  source: hgnc_alias_symbol
  hgnc_symbol: CHEK2
  entrez: '11200'
- word: p-Creb
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: ATF1
  entrez: '466'
- word: Ras
  symbol: RAS
  source: bioentities_symbol
  hgnc_symbol: HRAS
  entrez: '3265'
- word: Chemokines,
  symbol: Chemokine
  source: bioentities_symbol
  hgnc_symbol: CXCL11
  entrez: '6373'
- word: Chemokines,
  symbol: Chemokine
  source: bioentities_symbol
  hgnc_symbol: CXCL5
  entrez: '6374'
- word: Chemokines,
  symbol: Chemokine
  source: bioentities_symbol
  hgnc_symbol: CXCL14
  entrez: '9547'
- word: Stats
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT1
  entrez: '6772'
- word: Stats
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT4
  entrez: '6775'
- word: p-Creb
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L4
  entrez: '148327'
- word: Stats
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT3
  entrez: '6774'
- word: Stats
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT2
  entrez: '6773'
- word: MRP-1
  symbol: MRP-1
  source: hgnc_alias_symbol
  hgnc_symbol: CD9
  entrez: '928'
- word: p-Stat5
  symbol: STAT5
  source: bioentities_symbol
  hgnc_symbol: STAT5B
  entrez: '6777'
- word: p-Stat5
  symbol: STAT5
  source: bioentities_symbol
  hgnc_symbol: STAT5A
  entrez: '6776'
- word: Ras
  symbol: RAS
  source: bioentities_symbol
  hgnc_symbol: KRAS
  entrez: '3845'
- word: PKC,
  symbol: PKC
  source: bioentities_symbol
  hgnc_symbol: PRKCB
  entrez: '5579'
- word: ABCG2
  symbol: ABCG2
  source: hgnc_symbol
  hgnc_symbol: ABCG2
  entrez: '9429'
- word: Chemokines,
  symbol: Chemokine
  source: bioentities_symbol
  hgnc_symbol: CXCL12
  entrez: '6387'
- word: p-Creb
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L1
  entrez: '90993'
- word: p-Creb
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB5
  entrez: '9586'
- word: p-Creb
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB1
  entrez: '1385'
- word: CXCR4
  symbol: CXCR4
  source: hgnc_symbol
  hgnc_symbol: CXCR4
  entrez: '7852'
- word: p-Creb
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3
  entrez: '10488'
- word: p-Creb
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: ATF2
  entrez: '1386'
- word: IL-27
  symbol: IL27
  source: hgnc_symbol
  hgnc_symbol: IL27
  entrez: '246778'
- word: Chemokines,
  symbol: Chemokine
  source: bioentities_symbol
  hgnc_symbol: CXCL8
  entrez: '3576'
- word: Chemokines,
  symbol: Chemokine
  source: bioentities_symbol
  hgnc_symbol: CXCL10
  entrez: '3627'
- word: Chemokines,
  symbol: Chemokine
  source: bioentities_symbol
  hgnc_symbol: CXCL9
  entrez: '4283'
- word: Chemokines,
  symbol: Chemokine
  source: bioentities_symbol
  hgnc_symbol: PF4
  entrez: '5196'
- word: Chemokines,
  symbol: Chemokine
  source: bioentities_symbol
  hgnc_symbol: PPBP
  entrez: '5473'
- word: Chemokines,
  symbol: Chemokine
  source: bioentities_symbol
  hgnc_symbol: CXCL6
  entrez: '6372'
- word: Chemokines,
  symbol: Chemokine
  source: bioentities_symbol
  hgnc_symbol: CXCL13
  entrez: '10563'
- word: Chemokines,
  symbol: Chemokine
  source: bioentities_symbol
  hgnc_symbol: CXCL1
  entrez: '2919'
- word: c-Kit
  symbol: C-Kit
  source: hgnc_alias_symbol
  hgnc_symbol: KIT
  entrez: '3815'
- word: FLT3L,SCF
  symbol: SCF
  source: hgnc_alias_symbol
  hgnc_symbol: KITLG
  entrez: '4254'
- word: G-CSF
  symbol: G-CSF
  source: hgnc_prev_symbol
  hgnc_symbol: CSF3
  entrez: '1440'
- word: PI3
  symbol: PI3
  source: hgnc_symbol
  hgnc_symbol: PI3
  entrez: '5266'
- word: p-Erk
  symbol: ERK
  source: bioentities_symbol
  hgnc_symbol: MAPK1
  entrez: '5594'
- word: p-Erk
  symbol: ERK
  source: bioentities_symbol
  hgnc_symbol: MAPK3
  entrez: '5595'
chemicals:
- word: Etoposide
  source: MESH
  identifier: D005047
diseases:
- word: Cancer
  source: ''
  identifier: ''
figid_alias: PMC4696776__F2a
redirect_from: /figures/PMC4696776__F2a
figtype: Figure
---
