---
figid: PMC4396512__elife05604f001
figlink: /pmc/articles/PMC4396512/figure/fig1/
number: Figure 1
caption: 'Arrows represent the steps involved in aliphatic glucosinolate (GSL) biosynthesis
  that have been validated through laboratory experiments and are naturally variable
  within A. thaliana. Gene names are listed next to or above the arrows. (A) Regulation
  of aliphatic GSL biosynthesis. The transcription factors MYB28 and MYB29 control
  accumulation of aliphatic GLS. A double knockout in these genes results in no aliphatic
  GLS accumulation, while a single knockout in these genes leads to a 50% reduction
  in aliphatic GSL (myb28) or a 25% reduction in aliphatic GSL (myb29), compared to
  WT Col-0. The biosynthetic enzymes MAM1 and AOP2 also influence aliphatic GLS accumulation
  and a non-functional allele at either locus leads to decreased GSL accumulation.
  (B) Amino acid chain elongation. During chain elongation, carbons are added to a
  methionine precursor through a series of reactions producing an elongated amino
  acid. Variation at the Elong locus controls the number of carbons added to the amino
  acid precursor and therefore, the length of the GSL side chain, R. A functional
  allele at this locus, MAM1, leads primarily to accumulation of GSL with four carbon
  (4C) length side chains, whereas a non-functional allele, gsm1 leads to accumulation
  of GSL with three carbon (3C) length side chains. The elongated amino acid is subsequently
  converted into a GLS via the core pathway (not shown). (C) Side chain modification.
  The GSL compounds produced can then undergo a series of side chain modifications
  that lead to the suite of diverse GSL compounds found in Arabidopsis. Side chain
  modification is controlled by variation of GSOX1, GSOX3, AOP2, AOP3 and GSOH. GSOX1
  & GSOX3 oxygenate a methylthio (MT) to methylsulfinyl (MSO) GSL. AOP2 converts MSO
  to alkenyl, such as allyl and but-3-enyl. AOP3, on the other hand converts only
  3C length MSO to OH-propyl GSL and cannot act on the 4MSO GSL. GSOH oxygenates the
  4C but-3-enyl to the OH-alkenyl GSL, OH-but-3-enyl. Since GSOH acts on but-3-enyl
  GSL, which is a product of AOP2, a functional AOP2 is necessary for GSOH to function
  and AOP2 is said to be epistatic to GSOH. Col-0 is functional for MAM1 and the GSOX''s,
  null for both AOP2 and AOP3, and functional for GSOH, resulting in accumulation
  of primarily 4MSO GLS. See  for images of GSL traces for each GSL genotypes in our
  mutant laboratory population. (D) GSL Activation. Once produced, GLS are stored
  in the vacuole in their stable, unreactive form until activation occurs. Upon cellular
  disruption, such as occurs during pathogen attack, insect herbivory or even wind
  damage, GLS come into contact with their own plant-made activating enzyme, myrosinase.
  After production, myrosinase is stored in vacuoles of idioblastic cells called myrosin
  bodies. Myrosinase activates the GSL compound by cleaving the glucose moiety, yielding
  an unstable aglycone structure that non-enzymatically rearranges to either nitriles
  or isothiocyanates, depending on the presence of the co-activators ESM1 and ESP.DOI:
  http://dx.doi.org/10.7554/eLife.05604.003'
pmcid: PMC4396512
papertitle: Natural genetic variation in Arabidopsis thaliana defense metabolism genes
  modulates field fitness.
reftext: Rachel Kerwin, et al. eLife. 2015;4:e05604.
pmc_ranked_result_index: '230508'
pathway_score: 0.9000879
filename: elife05604f001.jpg
figtitle: Overview of aliphatic GSL biosynthesis and activation in Arabidopsis thaliana
year: '2015'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4396512__elife05604f001.html
  '@type': Dataset
  description: 'Arrows represent the steps involved in aliphatic glucosinolate (GSL)
    biosynthesis that have been validated through laboratory experiments and are naturally
    variable within A. thaliana. Gene names are listed next to or above the arrows.
    (A) Regulation of aliphatic GSL biosynthesis. The transcription factors MYB28
    and MYB29 control accumulation of aliphatic GLS. A double knockout in these genes
    results in no aliphatic GLS accumulation, while a single knockout in these genes
    leads to a 50% reduction in aliphatic GSL (myb28) or a 25% reduction in aliphatic
    GSL (myb29), compared to WT Col-0. The biosynthetic enzymes MAM1 and AOP2 also
    influence aliphatic GLS accumulation and a non-functional allele at either locus
    leads to decreased GSL accumulation. (B) Amino acid chain elongation. During chain
    elongation, carbons are added to a methionine precursor through a series of reactions
    producing an elongated amino acid. Variation at the Elong locus controls the number
    of carbons added to the amino acid precursor and therefore, the length of the
    GSL side chain, R. A functional allele at this locus, MAM1, leads primarily to
    accumulation of GSL with four carbon (4C) length side chains, whereas a non-functional
    allele, gsm1 leads to accumulation of GSL with three carbon (3C) length side chains.
    The elongated amino acid is subsequently converted into a GLS via the core pathway
    (not shown). (C) Side chain modification. The GSL compounds produced can then
    undergo a series of side chain modifications that lead to the suite of diverse
    GSL compounds found in Arabidopsis. Side chain modification is controlled by variation
    of GSOX1, GSOX3, AOP2, AOP3 and GSOH. GSOX1 & GSOX3 oxygenate a methylthio (MT)
    to methylsulfinyl (MSO) GSL. AOP2 converts MSO to alkenyl, such as allyl and but-3-enyl.
    AOP3, on the other hand converts only 3C length MSO to OH-propyl GSL and cannot
    act on the 4MSO GSL. GSOH oxygenates the 4C but-3-enyl to the OH-alkenyl GSL,
    OH-but-3-enyl. Since GSOH acts on but-3-enyl GSL, which is a product of AOP2,
    a functional AOP2 is necessary for GSOH to function and AOP2 is said to be epistatic
    to GSOH. Col-0 is functional for MAM1 and the GSOX''s, null for both AOP2 and
    AOP3, and functional for GSOH, resulting in accumulation of primarily 4MSO GLS.
    See  for images of GSL traces for each GSL genotypes in our mutant laboratory
    population. (D) GSL Activation. Once produced, GLS are stored in the vacuole in
    their stable, unreactive form until activation occurs. Upon cellular disruption,
    such as occurs during pathogen attack, insect herbivory or even wind damage, GLS
    come into contact with their own plant-made activating enzyme, myrosinase. After
    production, myrosinase is stored in vacuoles of idioblastic cells called myrosin
    bodies. Myrosinase activates the GSL compound by cleaving the glucose moiety,
    yielding an unstable aglycone structure that non-enzymatically rearranges to either
    nitriles or isothiocyanates, depending on the presence of the co-activators ESM1
    and ESP.DOI: http://dx.doi.org/10.7554/eLife.05604.003'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CTSA
  - ESM1
  - MAML1
  - PRDX6
  - Methionine
  - Nitrile
  - Isothiocyanate
genes:
- word: GSL
  symbol: GSL
  source: hgnc_prev_symbol
  hgnc_symbol: CTSA
  entrez: '5476'
- word: ESM1
  symbol: ESM1
  source: hgnc_symbol
  hgnc_symbol: ESM1
  entrez: '11082'
- word: mam1
  symbol: Mam-1
  source: hgnc_alias_symbol
  hgnc_symbol: MAML1
  entrez: '9794'
- word: AOP2
  symbol: AOP2
  source: hgnc_alias_symbol
  hgnc_symbol: PRDX6
  entrez: '9588'
chemicals:
- word: Methionine
  source: MESH
  identifier: D008715
- word: Nitrile
  source: MESH
  identifier: D009570
- word: Isothiocyanate
  source: MESH
  identifier: D017879
diseases: []
---
