---
figid: PMC4001401__pc_118919_f4
figlink: /pmc/articles/PMC4001401/figure/fig4/
number: F4
caption: 'Correlation Loading Plots of the Lipids and Transcripts in the O2PLS-DA
  Model.Lipids are denoted as triangles and transcripts as squares. The location of
  each variable on the plot is described by the correlation of its joint variance
  projection with light-specific (x axis) and temperature-specific (y axis) latent
  variables of the O2PLS-DA model. The gradients are from darkness- to high light–specific
  responses (left to right) on the x axis and from cold- to heat-specific responses
  (bottom to top) on the y axis. Each plot shows the distribution of all analyzed
  variables (small gray symbols), but on each of them a different group is highlighted
  by magnified and color-coded symbols.(A) Glycerolipids. The total number of double
  bonds in glycerolipid acyl chains is represented by the color scale from red (six
  double bonds) to blue (no double bonds). Sum changes of the lipid classes are represented
  by green arrows. From all plotted lipids, only those with significant correlation
  (P < 0.01) with one of the latent variables have been plotted.(B) Transcripts of
  the  biosynthesis pathway. Genes belonging to the  biosynthesis pathway are highlighted
  in green. Sum changes of all lipid biochemical pathways are plotted as gray arrows.
  The sum changes of the  biosynthesis pathway are highlighted by a green arrow. The
  abbreviations for the pathways are as follows: Cutin, cutin biosynthesis; EuP, eukaryotic
  phospholipid biosynthesis; FA,  biosynthesis; FA&TAG deg,  and  degradation; FAE&Wax,  elongation
  and wax biosynthesis; LT, lipid trafficking; Oxylipin, oxylipin metabolism; P sig,
  phospholipid signaling; Pro GSP, prokaryotic galactolipid, sulfolipid, and phospholipid
  metabolism; Sph, sphingolipid biosynthesis; Suberin, suberin biosynthesis; TAG synth,  biosynthesis.(C)
  Genes of prokaryotic and eukaryotic galactolipid, sulfolipid, and phospholipid metabolism
  pathways.(D) Genes involved in storage oil metabolism.'
pmcid: PMC4001401
papertitle: Linking Gene Expression and Membrane Lipid Composition of Arabidopsis.
reftext: Jedrzej Szymanski, et al. Plant Cell. 2014 Mar;26(3):915-928.
pmc_ranked_result_index: '158749'
pathway_score: 0.8371981
filename: pc_118919_f4.jpg
figtitle: Linking Gene Expression and Membrane Lipid Composition
year: '2014'
organisms: Arabidopsis thaliana
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4001401__pc_118919_f4.html
  '@type': Dataset
  description: 'Correlation Loading Plots of the Lipids and Transcripts in the O2PLS-DA
    Model.Lipids are denoted as triangles and transcripts as squares. The location
    of each variable on the plot is described by the correlation of its joint variance
    projection with light-specific (x axis) and temperature-specific (y axis) latent
    variables of the O2PLS-DA model. The gradients are from darkness- to high light–specific
    responses (left to right) on the x axis and from cold- to heat-specific responses
    (bottom to top) on the y axis. Each plot shows the distribution of all analyzed
    variables (small gray symbols), but on each of them a different group is highlighted
    by magnified and color-coded symbols.(A) Glycerolipids. The total number of double
    bonds in glycerolipid acyl chains is represented by the color scale from red (six
    double bonds) to blue (no double bonds). Sum changes of the lipid classes are
    represented by green arrows. From all plotted lipids, only those with significant
    correlation (P < 0.01) with one of the latent variables have been plotted.(B)
    Transcripts of the  biosynthesis pathway. Genes belonging to the  biosynthesis
    pathway are highlighted in green. Sum changes of all lipid biochemical pathways
    are plotted as gray arrows. The sum changes of the  biosynthesis pathway are highlighted
    by a green arrow. The abbreviations for the pathways are as follows: Cutin, cutin
    biosynthesis; EuP, eukaryotic phospholipid biosynthesis; FA,  biosynthesis; FA&TAG
    deg,  and  degradation; FAE&Wax,  elongation and wax biosynthesis; LT, lipid trafficking;
    Oxylipin, oxylipin metabolism; P sig, phospholipid signaling; Pro GSP, prokaryotic
    galactolipid, sulfolipid, and phospholipid metabolism; Sph, sphingolipid biosynthesis;
    Suberin, suberin biosynthesis; TAG synth,  biosynthesis.(C) Genes of prokaryotic
    and eukaryotic galactolipid, sulfolipid, and phospholipid metabolism pathways.(D)
    Genes involved in storage oil metabolism.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PC
  - FANCE
  - FANCA
  - GNAS
  - ACP1
  - ACSL1
  - CHKA
  - CDIPT
  - GNAL
  - COL4A5
  - PES1
genes:
- word: PC
  symbol: PC
  source: hgnc_symbol
  hgnc_symbol: PC
  entrez: '5091'
- word: FAE&Wax
  symbol: FAE
  source: hgnc_alias_symbol
  hgnc_symbol: FANCE
  entrez: '2178'
- word: FaA
  symbol: FAA
  source: hgnc_alias_symbol
  hgnc_symbol: FANCA
  entrez: '2175'
- word: GS
  symbol: Gs
  source: bioentities_symbol
  hgnc_symbol: GNAS
  entrez: '2778'
- word: ACP1
  symbol: ACP1
  source: hgnc_symbol
  hgnc_symbol: ACP1
  entrez: '52'
- word: LACS
  symbol: LACS
  source: hgnc_alias_symbol
  hgnc_symbol: ACSL1
  entrez: '2180'
- word: CKI
  symbol: CKI
  source: hgnc_alias_symbol
  hgnc_symbol: CHKA
  entrez: '1119'
- word: PIS
  symbol: PIS
  source: hgnc_alias_symbol
  hgnc_symbol: CDIPT
  entrez: '10423'
- word: GS
  symbol: Gs
  source: bioentities_symbol
  hgnc_symbol: GNAL
  entrez: '2774'
- word: ATS
  symbol: ATS
  source: hgnc_prev_symbol
  hgnc_symbol: COL4A5
  entrez: '1287'
- word: PES
  symbol: PES
  source: hgnc_alias_symbol
  hgnc_symbol: PES1
  entrez: '23481'
chemicals: []
diseases: []
figid_alias: PMC4001401__F4
redirect_from: /figures/PMC4001401__F4
figtype: Figure
---
