---
figid: PMC4905226__kmco-02-01-968020-g002
figlink: /pmc/articles/PMC4905226/figure/f0002/
number: F2
caption: (See previous page). Mechanisms of non-homologous end joining. (A). Canonical
  NHEJ (C-NHEJ). C-NHEJ is initiated by binding of the Ku80-Ku70 heterodimer to the
  DSB, which recruits the DNA-PK catalytic subunit, DNA-PKcs. Several proteins, including
  Artemis, the polynucleotide kinase (PNK) and members of the polymerase X family,
  process the DNA ends to make them competent for the subsequent ligation steps. Finally,
  ligase IV, in association with Xrcc4 and Cernunos/Xlf, seals the double-strand ends.
  (B) Alternative end-joining (A-EJ). In the absence of Ku70/Ku80, the DNA ends are
  resected in a reaction favored by the nuclease activity of Mre11 and CtIP. Note
  that Parp1 is involved in A-EJ initiation. The resulting ssDNA reveals complementary
  microhomologies (2–4 nt or more) that can be annealed; gap filling completes the
  end joining. Finally, Xrcc1 and ligase III complete the A-EJ process. Notably, A-EJ
  is always associated with deletions at the junctions and frequently, but not systematically,
  involves microhomologies that are distant from the DSB. (C) The 2-step DSB repair
  pathway choice model. After signaling of the DSB by ATM and MRN, (I) binding of
  Ku80/Ku70 protects the DSB ends from resection, routing DSB repair toward the conservative
  C-NHEJ pathway. (II) The nuclease activity of Mre11 and CtIP favor ssDNA resection,
  which can then initiate the HR or A-EJ pathway. A short ssDNA resection is sufficient
  for A-EJ but not for HR. A-EJ is an exclusively non-conservative mutagenic process.
pmcid: PMC4905226
papertitle: Role of the double-strand break repair pathway in the maintenance of genomic
  stability.
reftext: Tangui Le Guen, et al. Mol Cell Oncol. 2015 Jan-Mar;2(1):e968020.
pmc_ranked_result_index: '120564'
pathway_score: 0.8598784
filename: kmco-02-01-968020-g002.jpg
figtitle: (See previous page)
year: '2015'
organisms:
- Eucalyptus intertexta
- Schizosaccharomyces pombe
- Saccharomyces cerevisiae
- Escherichia coli
- Gallus gallus
- Homo sapiens
ndex: 53a9f51d-df01-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4905226__kmco-02-01-968020-g002.html
  '@type': Dataset
  description: (See previous page). Mechanisms of non-homologous end joining. (A).
    Canonical NHEJ (C-NHEJ). C-NHEJ is initiated by binding of the Ku80-Ku70 heterodimer
    to the DSB, which recruits the DNA-PK catalytic subunit, DNA-PKcs. Several proteins,
    including Artemis, the polynucleotide kinase (PNK) and members of the polymerase
    X family, process the DNA ends to make them competent for the subsequent ligation
    steps. Finally, ligase IV, in association with Xrcc4 and Cernunos/Xlf, seals the
    double-strand ends. (B) Alternative end-joining (A-EJ). In the absence of Ku70/Ku80,
    the DNA ends are resected in a reaction favored by the nuclease activity of Mre11
    and CtIP. Note that Parp1 is involved in A-EJ initiation. The resulting ssDNA
    reveals complementary microhomologies (2–4 nt or more) that can be annealed; gap
    filling completes the end joining. Finally, Xrcc1 and ligase III complete the
    A-EJ process. Notably, A-EJ is always associated with deletions at the junctions
    and frequently, but not systematically, involves microhomologies that are distant
    from the DSB. (C) The 2-step DSB repair pathway choice model. After signaling
    of the DSB by ATM and MRN, (I) binding of Ku80/Ku70 protects the DSB ends from
    resection, routing DSB repair toward the conservative C-NHEJ pathway. (II) The
    nuclease activity of Mre11 and CtIP favor ssDNA resection, which can then initiate
    the HR or A-EJ pathway. A short ssDNA resection is sufficient for A-EJ but not
    for HR. A-EJ is an exclusively non-conservative mutagenic process.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MRE11
  - ATM
  - PRKDC
  - XRCC5
  - NHEJ1
  - RBBP8
  - DCLRE1C
  - XRCC6
  - XRCC4
  - PARP1
genes:
- word: Mre11
  symbol: MRE11
  source: hgnc_symbol
  hgnc_symbol: MRE11
  entrez: '4361'
- word: ATM
  symbol: ATM
  source: hgnc_symbol
  hgnc_symbol: ATM
  entrez: '472'
- word: DNA-PKcs
  symbol: DNA-PKcs
  source: hgnc_alias_symbol
  hgnc_symbol: PRKDC
  entrez: '5591'
- word: Ku80
  symbol: KU80
  source: hgnc_alias_symbol
  hgnc_symbol: XRCC5
  entrez: '7520'
- word: Cernunnos
  symbol: Cernunnos
  source: hgnc_alias_symbol
  hgnc_symbol: NHEJ1
  entrez: '79840'
- word: CtIP
  symbol: CtIP
  source: hgnc_alias_symbol
  hgnc_symbol: RBBP8
  entrez: '5932'
- word: Artemis
  symbol: ARTEMIS
  source: hgnc_alias_symbol
  hgnc_symbol: DCLRE1C
  entrez: '64421'
- word: Ku70
  symbol: KU70
  source: hgnc_alias_symbol
  hgnc_symbol: XRCC6
  entrez: '2547'
- word: Xrcc4
  symbol: XRCC4
  source: hgnc_symbol
  hgnc_symbol: XRCC4
  entrez: '7518'
- word: Parp1
  symbol: PARP1
  source: hgnc_symbol
  hgnc_symbol: PARP1
  entrez: '142'
chemicals: []
diseases: []
figid_alias: PMC4905226__F2
redirect_from: /figures/PMC4905226__F2
figtype: Figure
---
