---
figid: PMC4082330__nihms583789f2
figlink: /pmc/articles/PMC4082330/figure/F2/
number: F2
caption: Activated signalling cascades can alter the epigenetic landscape directly
  or indirectly by communicating with chromatin regulators. a, Metabolites and nutrient
  availability influence the activity of many enzymes regulating the epigenome, as
  these small molecules are used as cofactors. Forexample, oxygen is required for
  the activity of the JumonjiC (JMJC) family of Lys demethylases. The extent to which
  a hypoxic environment inhibits the various JMJC demethylases is not known. Hypoxia
  can lead to histone acetylation around hypoxia-inducible factor 1α (HIF1α) and HIF1β
  target genes via the recruitment of p300. Dietary Met is a precursor of the methyl
  donor S-adenosylmethionine (SAM), which is required for DNA, Lys or Arg methylation
  (Me) events, and glucose metabolism through the tricarboxylic acid (TCA) cycle produces
  α-ketoglutarate, which is also required for JMJC demethylase activity. Glucose metabolism
  through thehexosamine biosynthetic pathway produces uridine diphosphate N-acetylglucosamine
  (UDP-GlcNAc), which is used by O-linked β-d-N-acetylglucosamine transferase (OGT)
  to trigger O-GlcNAcylation of enzymes that regulate chromatin modification, histones
  and transcription factors (TFs). Acetyl-CoA that is produced from citrate exiting
  the TCA cycle also provides the donor group for protein acetylation (Ac). b, A simplified
  and generic kinase cascade that collides with chromatin modifying factors in the
  nucleus. Signal-induced post-translational modifications (PTMs) of enzymes (such
  as phosphorylation) may alter enzyme activity in modifying histone or DNA residues.
  Enzyme modification of ATP-dependent chromatin remodellers can also alter their
  effects on nucleosome repositioning, changing the modification landscape or DNA
  accessibility around particular gene promoters or enhancers. Controlling enhancer
  activity or responsiveness to signals through nearby histone tail modifications
  and transcription factor binding site accessibility may be a crucial process in
  cell fate decisions, as enhancer utilization drives tissue-specific gene expression.
  Phosphorylation–dephosphorylation events on histone chaperones can alter their ability
  to deposit histone variants into functionally important genomic areas, such as around
  the transcriptional start sites of genes. Reader proteins may also be direct targets
  of signalling cascades; changing the ability of these proteins to recognize the
  PTMs on chromatin and therefore the recruitment of additional regulators can completely
  switch the biological consequence of a chromatin modification. Another mechanism
  at work is complex nucleation initiated by a signalling event. In this case, one
  or a few proteins are maintained on or nearby their target genes, poised but not
  active, and acting as a ‘placeholder’ for rapid induction. Upon a signalling event,
  the placeholders nucleate the remaining protein complex and a function is executed.
  Due to the rapid nature of this response, complex nucleation may be most prevalent
  during primary response gene (PRG) induction.
pmcid: PMC4082330
papertitle: Emerging roles for chromatin as a signal integration and storage platform.
reftext: Aimee I. Badeaux, et al. Nat Rev Mol Cell Biol. ;14(4):211-224.
pmc_ranked_result_index: '99483'
pathway_score: 0.942869
filename: nihms583789f2.jpg
figtitle: Emerging roles for chromatin as a signal integration and storage platform
year: ''
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4082330__nihms583789f2.html
  '@type': Dataset
  description: Activated signalling cascades can alter the epigenetic landscape directly
    or indirectly by communicating with chromatin regulators. a, Metabolites and nutrient
    availability influence the activity of many enzymes regulating the epigenome,
    as these small molecules are used as cofactors. Forexample, oxygen is required
    for the activity of the JumonjiC (JMJC) family of Lys demethylases. The extent
    to which a hypoxic environment inhibits the various JMJC demethylases is not known.
    Hypoxia can lead to histone acetylation around hypoxia-inducible factor 1α (HIF1α)
    and HIF1β target genes via the recruitment of p300. Dietary Met is a precursor
    of the methyl donor S-adenosylmethionine (SAM), which is required for DNA, Lys
    or Arg methylation (Me) events, and glucose metabolism through the tricarboxylic
    acid (TCA) cycle produces α-ketoglutarate, which is also required for JMJC demethylase
    activity. Glucose metabolism through thehexosamine biosynthetic pathway produces
    uridine diphosphate N-acetylglucosamine (UDP-GlcNAc), which is used by O-linked
    β-d-N-acetylglucosamine transferase (OGT) to trigger O-GlcNAcylation of enzymes
    that regulate chromatin modification, histones and transcription factors (TFs).
    Acetyl-CoA that is produced from citrate exiting the TCA cycle also provides the
    donor group for protein acetylation (Ac). b, A simplified and generic kinase cascade
    that collides with chromatin modifying factors in the nucleus. Signal-induced
    post-translational modifications (PTMs) of enzymes (such as phosphorylation) may
    alter enzyme activity in modifying histone or DNA residues. Enzyme modification
    of ATP-dependent chromatin remodellers can also alter their effects on nucleosome
    repositioning, changing the modification landscape or DNA accessibility around
    particular gene promoters or enhancers. Controlling enhancer activity or responsiveness
    to signals through nearby histone tail modifications and transcription factor
    binding site accessibility may be a crucial process in cell fate decisions, as
    enhancer utilization drives tissue-specific gene expression. Phosphorylation–dephosphorylation
    events on histone chaperones can alter their ability to deposit histone variants
    into functionally important genomic areas, such as around the transcriptional
    start sites of genes. Reader proteins may also be direct targets of signalling
    cascades; changing the ability of these proteins to recognize the PTMs on chromatin
    and therefore the recruitment of additional regulators can completely switch the
    biological consequence of a chromatin modification. Another mechanism at work
    is complex nucleation initiated by a signalling event. In this case, one or a
    few proteins are maintained on or nearby their target genes, poised but not active,
    and acting as a ‘placeholder’ for rapid induction. Upon a signalling event, the
    placeholders nucleate the remaining protein complex and a function is executed.
    Due to the rapid nature of this response, complex nucleation may be most prevalent
    during primary response gene (PRG) induction.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - HIF1A
  - EPAS1
  - ARNTL
  - ARNT2
  - ARNT
  - HIF3A
  - CCL1
  - SLTM
  - Arg
  - Glucose
  - Met
  - UDP
genes:
- word: HIF
  symbol: HIF
  source: bioentities_symbol
  hgnc_symbol: HIF1A
  entrez: '3091'
- word: HIF
  symbol: HIF
  source: bioentities_symbol
  hgnc_symbol: EPAS1
  entrez: '2034'
- word: HIF
  symbol: HIF
  source: bioentities_symbol
  hgnc_symbol: ARNTL
  entrez: '406'
- word: HIF
  symbol: HIF
  source: bioentities_symbol
  hgnc_symbol: ARNT2
  entrez: '9915'
- word: HIF
  symbol: HIF
  source: bioentities_symbol
  hgnc_symbol: ARNT
  entrez: '405'
- word: HIF
  symbol: HIF
  source: bioentities_symbol
  hgnc_symbol: HIF3A
  entrez: '64344'
- word: p500
  symbol: P500
  source: hgnc_alias_symbol
  hgnc_symbol: CCL1
  entrez: '6346'
- word: Met
  symbol: Met
  source: hgnc_alias_symbol
  hgnc_symbol: SLTM
  entrez: '79811'
chemicals:
- word: Arg
  source: MESH
  identifier: D001120
- word: Glucose
  source: MESH
  identifier: D005947
- word: Met
  source: MESH
  identifier: D008715
- word: UDP
  source: MESH
  identifier: D014530
diseases: []
figid_alias: PMC4082330__F2
redirect_from: /figures/PMC4082330__F2
figtype: Figure
---
