---
figid: PMC4515750__nihms706610f1
figlink: /pmc/articles/PMC4515750/figure/F1/
number: Figure 1
caption: 'Drosophila gene names are used.a) miRNA pathway: Canonical miRNAs are processed
  from long primary transcripts by Drosha/Pasha complex in nucleus to produce pre-miRNAs.
  Processed pre-miRNAs are then processed by Dcr-1/loqs complex after exported to
  cytoplasm. One strand of a small RNA duplex of miRNA is preferentially incorporated
  into AGO1 complex to regulate target expression. Mirtron pathway is an alternative
  pathway to produce pre-miRNAs. Mirtrons are directly produced by splicing reactions
  followed by de-branching of the intron lariats. Thus mirtrons are produced by a
  Drosha/Pasha independent manner. Several mirtron genes are predicted to have single
  stranded extensions (tailed mirtrons). Although the first tailed mirtron found in
  Drosophila has a 3′ tail, the tails appeared to be located on 5′ sides of hairpins
  in vertebrates. Endo-shRNAs (only found in mouse so far) are proposed to directly
  produce pre-miRNAs by RNA polymerase III transcription. Small regulatory RNAs from
  tRNAs and snoRNAs are also found in mammalian large scale libraries.b) Exogenous
  siRNA pathway. The exogenous siRNA pathway was originally found during studies on
  molecular mechanisms of artificial RNA interference using synthetic dsRNAs. Later
  this pathway was found to play roles in anti-viral response. siRNAs are processed
  from exogenous dsRNAs (such as those derived from viral genomes or their replication
  intermediates, or artificial dsRNAs) by the complex containing Dcr-2 and its cofactor,
  R2D2. Resulting siRNAs are loaded to the effector AGO2 by the Dcr-2/R2D2 loading
  complex.c) Endogenous siRNA pathway. siRNAs are endogenously generated from dsRNAs
  produced from various sources including transcripts containing inverted-repeat (hairpin-RNAs),
  sense-antisense transcript pairs (cis- or trans-Natural Antisense Transcripts) and
  dsRNAs produced from transposons. Endogenously produced dsRNAs are cleaved by the
  Dcr-2/Loqs-PD complex to produce endogenous siRNAs, and siRNAs are loaded into AGO2.
  Endogenous siRNAs play roles in transposon repression and gene regulation.'
pmcid: PMC4515750
papertitle: Diversity and complexity of dicer dependent small RNA networks in animals.
reftext: Katsutomo Okamura, et al. Seikagaku. ;81(10):904-909.
pmc_ranked_result_index: '21413'
pathway_score: 0.9546363
filename: nihms706610f1.jpg
figtitle: Diversity and complexity of dicer dependent small RNA networks in animals
year: ''
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4515750__nihms706610f1.html
  '@type': Dataset
  description: 'Drosophila gene names are used.a) miRNA pathway: Canonical miRNAs
    are processed from long primary transcripts by Drosha/Pasha complex in nucleus
    to produce pre-miRNAs. Processed pre-miRNAs are then processed by Dcr-1/loqs complex
    after exported to cytoplasm. One strand of a small RNA duplex of miRNA is preferentially
    incorporated into AGO1 complex to regulate target expression. Mirtron pathway
    is an alternative pathway to produce pre-miRNAs. Mirtrons are directly produced
    by splicing reactions followed by de-branching of the intron lariats. Thus mirtrons
    are produced by a Drosha/Pasha independent manner. Several mirtron genes are predicted
    to have single stranded extensions (tailed mirtrons). Although the first tailed
    mirtron found in Drosophila has a 3′ tail, the tails appeared to be located on
    5′ sides of hairpins in vertebrates. Endo-shRNAs (only found in mouse so far)
    are proposed to directly produce pre-miRNAs by RNA polymerase III transcription.
    Small regulatory RNAs from tRNAs and snoRNAs are also found in mammalian large
    scale libraries.b) Exogenous siRNA pathway. The exogenous siRNA pathway was originally
    found during studies on molecular mechanisms of artificial RNA interference using
    synthetic dsRNAs. Later this pathway was found to play roles in anti-viral response.
    siRNAs are processed from exogenous dsRNAs (such as those derived from viral genomes
    or their replication intermediates, or artificial dsRNAs) by the complex containing
    Dcr-2 and its cofactor, R2D2. Resulting siRNAs are loaded to the effector AGO2
    by the Dcr-2/R2D2 loading complex.c) Endogenous siRNA pathway. siRNAs are endogenously
    generated from dsRNAs produced from various sources including transcripts containing
    inverted-repeat (hairpin-RNAs), sense-antisense transcript pairs (cis- or trans-Natural
    Antisense Transcripts) and dsRNAs produced from transposons. Endogenously produced
    dsRNAs are cleaved by the Dcr-2/Loqs-PD complex to produce endogenous siRNAs,
    and siRNAs are loaded into AGO2. Endogenous siRNAs play roles in transposon repression
    and gene regulation.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - AGO2
  - DGCR8
  - FBXW7
  - AGO3
  - DROSHA
  - TES
  - DICER1
  - PNMT
genes:
- word: AGO2
  symbol: AGO2
  source: hgnc_symbol
  hgnc_symbol: AGO2
  entrez: '27161'
- word: Pasha
  symbol: pasha
  source: hgnc_alias_symbol
  hgnc_symbol: DGCR8
  entrez: '54487'
- word: AGO
  symbol: AGO
  source: hgnc_alias_symbol
  hgnc_symbol: FBXW7
  entrez: '55294'
- word: AGO3
  symbol: AGO3
  source: hgnc_symbol
  hgnc_symbol: AGO3
  entrez: '192669'
- word: Drosha
  symbol: DROSHA
  source: hgnc_symbol
  hgnc_symbol: DROSHA
  entrez: '29102'
- word: TES
  symbol: TES
  source: hgnc_symbol
  hgnc_symbol: TES
  entrez: '26136'
- word: Dicer-1
  symbol: DICER1
  source: hgnc_symbol
  hgnc_symbol: DICER1
  entrez: '23405'
- word: Pent
  symbol: PENT
  source: hgnc_prev_symbol
  hgnc_symbol: PNMT
  entrez: '5409'
chemicals: []
diseases: []
---
