---
figid: PMC4065383__nihms583137f2
figlink: /pmc/articles/PMC4065383/figure/F2/
number: F2
caption: Origins and removal of ribonucleotides in DNA. (A) Incorporation and removal
  of ribonucleotides incorporated by RNA primase during OF synthesis and maturation.
  Two of the pathways by which the RNA primer synthesized by Pol α-primase is removed
  are displayed. RNase H2 is able to digest and remove the RNA primer up until the
  terminal 3′ ribonucleotide that is removed by FEN1. Alternatively, Pol δ can perform
  strand displacement synthesis to generate a flap that is degraded by FEN1. (B) Proofreading
  efficiency is determined by the balance between excision and extension. The polymerase
  exonuclease activity can excise ribonucleotides from the 3′ terminus in primer-template
  DNA, an event that is more efficient on a mismatched 3′ terminal dNMP primer than
  it is on an rNMP. The kinetics of proofreading (both excision and extension) by
  a replicative DNA polymerase in the presence of a terminal rNMP remain to be biochemically
  determined. (C) The ribonucleotide excision repair (RER) pathway is initiated by
  incision 5′ to the ribonucleotide by RNase H2. Strand displacement synthesis by
  the polymerase is followed by nucleolytic flap processing and ligation. There are
  redundant functions of the polymerases (Pol δ and Polε) as well as the nucleases
  (FEN1 and Exo1; Adapted from Fig. 4 in []). (D) Loss of RNase H2 activity due to
  deletion of RNH201 or RNase H2-deficiency in AGS leaves unrepaired genomic ribonucleotides
  that may become targets for Top1-cleavage and removal. The resulting unligatable
  DNA ends possess a 2′–3′-cyclic phosphate and a 5′-hydroxyl that may promote the
  formation of DNA breaks and/or recombination and cause genome instability. (Adapted
  from Fig. 4 in []). (E) The pathway by which RNase H2 cleavage at ribonucleotides
  can generate abortive ligation intermediates (adenylated 5′-RNA–DNA junctions) that
  require APTXHNT3 for resolution. Failure of this pathway due to deletion of HNT3
  or APTX deficiency in AOA1 causes phenotypes that may reflect the persistence of
  adenylated RNA–DNA junctions (Adapted from Fig. 2E in [].
pmcid: PMC4065383
papertitle: 'Ribonucleotides in DNA: Origins, repair and consequences.'
reftext: Jessica S. Williams, et al. DNA Repair (Amst). ;19:27-37.
pmc_ranked_result_index: '149889'
pathway_score: 0.8059683
filename: nihms583137f2.jpg
figtitle: Origins and removal of ribonucleotides in DNA
year: ''
organisms:
- Homo sapiens
ndex: 059bf853-df1b-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4065383__nihms583137f2.html
  '@type': Dataset
  description: Origins and removal of ribonucleotides in DNA. (A) Incorporation and
    removal of ribonucleotides incorporated by RNA primase during OF synthesis and
    maturation. Two of the pathways by which the RNA primer synthesized by Pol α-primase
    is removed are displayed. RNase H2 is able to digest and remove the RNA primer
    up until the terminal 3′ ribonucleotide that is removed by FEN1. Alternatively,
    Pol δ can perform strand displacement synthesis to generate a flap that is degraded
    by FEN1. (B) Proofreading efficiency is determined by the balance between excision
    and extension. The polymerase exonuclease activity can excise ribonucleotides
    from the 3′ terminus in primer-template DNA, an event that is more efficient on
    a mismatched 3′ terminal dNMP primer than it is on an rNMP. The kinetics of proofreading
    (both excision and extension) by a replicative DNA polymerase in the presence
    of a terminal rNMP remain to be biochemically determined. (C) The ribonucleotide
    excision repair (RER) pathway is initiated by incision 5′ to the ribonucleotide
    by RNase H2. Strand displacement synthesis by the polymerase is followed by nucleolytic
    flap processing and ligation. There are redundant functions of the polymerases
    (Pol δ and Polε) as well as the nucleases (FEN1 and Exo1; Adapted from Fig. 4
    in []). (D) Loss of RNase H2 activity due to deletion of RNH201 or RNase H2-deficiency
    in AGS leaves unrepaired genomic ribonucleotides that may become targets for Top1-cleavage
    and removal. The resulting unligatable DNA ends possess a 2′–3′-cyclic phosphate
    and a 5′-hydroxyl that may promote the formation of DNA breaks and/or recombination
    and cause genome instability. (Adapted from Fig. 4 in []). (E) The pathway by
    which RNase H2 cleavage at ribonucleotides can generate abortive ligation intermediates
    (adenylated 5′-RNA–DNA junctions) that require APTXHNT3 for resolution. Failure
    of this pathway due to deletion of HNT3 or APTX deficiency in AOA1 causes phenotypes
    that may reflect the persistence of adenylated RNA–DNA junctions (Adapted from
    Fig. 2E in [].
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - INS
  - FEN1
  - RFC2
  - EXO1
  - RPA1
  - RNASEH1
  - FASTK
  - POLA1
  - RFC1
  - TOP1
  - PCNA
  - DNA2
  - C5
  - RPA3
  - ALOX5AP
  - APTX
  - RFC4
  - RFC3
  - RPA2
  - JAG1
  - RFC5
genes:
- word: ins
  symbol: INS
  source: hgnc_symbol
  hgnc_symbol: INS
  entrez: '3630'
- word: FEN1
  symbol: FEN1
  source: hgnc_symbol
  hgnc_symbol: FEN1
  entrez: '2237'
- word: RFC,
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC2
  entrez: '5982'
- word: FEN1/Exo1
  symbol: EXO1
  source: hgnc_symbol
  hgnc_symbol: EXO1
  entrez: '9156'
- word: RFC,RPA
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA1
  entrez: '6117'
- word: (#RNaseH1)
  symbol: RNASEH1
  source: hgnc_symbol
  hgnc_symbol: RNASEH1
  entrez: '246243'
- word: fast
  symbol: FAST
  source: hgnc_alias_symbol
  hgnc_symbol: FASTK
  entrez: '10922'
- word: Pola-
  symbol: POLA
  source: hgnc_prev_symbol
  hgnc_symbol: POLA1
  entrez: '5422'
- word: RFC,
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC1
  entrez: '5981'
- word: Top1
  symbol: TOP1
  source: hgnc_symbol
  hgnc_symbol: TOP1
  entrez: '7150'
- word: +PCNA,
  symbol: PCNA
  source: hgnc_symbol
  hgnc_symbol: PCNA
  entrez: '5111'
- word: DNA2)
  symbol: DNA2
  source: hgnc_symbol
  hgnc_symbol: DNA2
  entrez: '1763'
- word: C5.
  symbol: C5
  source: hgnc_symbol
  hgnc_symbol: C5
  entrez: '727'
- word: RFC,RPA
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA3
  entrez: '6119'
- word: FLAP
  symbol: FLAP
  source: hgnc_alias_symbol
  hgnc_symbol: ALOX5AP
  entrez: '241'
- word: AOA1
  symbol: AOA1
  source: hgnc_alias_symbol
  hgnc_symbol: APTX
  entrez: '54840'
- word: RFC,
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC4
  entrez: '5984'
- word: RFC,
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC3
  entrez: '5983'
- word: RFC,RPA
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA2
  entrez: '6118'
- word: AGS
  symbol: AGS
  source: hgnc_prev_symbol
  hgnc_symbol: JAG1
  entrez: '182'
- word: RFC,
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC5
  entrez: '5985'
chemicals: []
diseases: []
figid_alias: PMC4065383__F2
redirect_from: /figures/PMC4065383__F2
figtype: Figure
---
