---
figid: PMC4287430__12864_2014_6788_Fig1_HTML
figlink: /pmc/articles/PMC4287430/figure/Fig1/
number: F1
caption: 'Predicted model of the S. velum symbiont cell. The diagram, based on the
  gene annotation of the symbiont genome, depicts key functional systems and metabolic
  pathways: sulfur oxidation, electron transport, ATP synthases, CO2-fixation via
  the Calvin Cycle, gluconeogenesis, polyglucose synthesis, glycolysis, TCA and glyoxylate
  cycles, synthesis of amino acids, fatty acids, lipids, isoprenoids via non-mevalonate
  pathway, and the cell wall, solute transporters, protein secretion systems, and
  the type IV pilus. Different protein categories are color-coded and the individual
  subunits indicated by shape symbols. The direction of substrate transport across
  the membrane is shown with arrows. Components of the electron transport chain are
  arranged from the lowest to the highest electronegativity of the electron donors
  (blue) and acceptors (red). The corresponding electronegativity values are listed
  next to the respective enzymes. Enzymes shared between glycolysis, gluconeogenesis,
  and the Calvin cycle are designated in green. Enzymes unique to these pathways are
  designated in red. Enzymes shared between the Calvin cycle and the pentose phosphate
  cycle are designated in blue. Amino acids which may be essential for the host are
  designated in red. Speculated pathways are designated with a question mark. The
  abbreviations used, the respective full gene product names, and the corresponding
  NCBI protein ID references are listed in Additional file : Table S3.'
pmcid: PMC4287430
papertitle: 'The genome of the intracellular bacterium of the coastal bivalve, Solemya
  velum: a blueprint for thriving in and out of symbiosis.'
reftext: Oleg Dmytrenko, et al. BMC Genomics. 2014;15:924.
pmc_ranked_result_index: '19726'
pathway_score: 0.9386604
filename: 12864_2014_6788_Fig1_HTML.jpg
figtitle: Predicted model of the Solemya velum symbiont cell
year: '2014'
organisms:
- XXX
- Thioalkalivibrio sulfidiphilus
- Allochromatium vinosum
- Thiocapsa roseopersicina
- Chlorobaculum tepidum
- Pinna nobilis
- Candidatus Carsonella ruddii
- Solemya reidi
- Diaporthe sclerotioides
- Staphylococcus aureus
- Phreagena okutanii
- Bradyrhizobium japonicum is5
- Solemya velum
- Escherichia coli IS1
- uncultured organism
- Wolbachia endosymbiont of Drosophila melanogaster
- Corynebacterium glutamicum
- Olavius algarvensis
- Campylobacter jejuni
- Crossidium squamiferum
- Helicobacter pylori
- Desulforapulum autotrophicum
- Solemya velum gill symbiont
- Methylococcus capsulatus str. Bath
- Thiocapsa marina
- metagenome
- Ridgeia piscesae
- Thermodesulfovibrio yellowstonii DSM 11347
- Hydrogenovibrio crunogenus XCL-2
- sulfur-oxidizing symbionts
- Sulfurimonas denitrificans DSM 1251
- Bradyrhizobium japonicum
- Cupriavidus necator H16
- Sinorhizobium meliloti
- Rhizobium leguminosarum
- Rhizobium leguminosarum bv. viciae
- Aliivibrio fischeri MJ11
- Hydrogenovibrio crunogenus
- metagenomes
- Methylococcus capsulatus
- Neisseria meningitidis
- Escherichia coli str. K-12 substr. MG1655
- Wigglesworthia glossinidia
- Escherichia coli DH1
- Tevnia jerichonana
- endosymbiont of Riftia pachyptila
- Escherichia coli
- PX clade
- Allochromatium vinosum DSM 180
- Klebsiella pneumoniae
- Sideroxydans lithotrophicus
- Riftia pachyptila
- Sipunculus nudus
- Calyptogena magnifica
- Rickettsia prowazekii
- Escherichia coli K-12
- uncultured Gammaproteobacteria bacterium
- Buchnera aphidicola
- Wolbachia pipientis
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4287430__12864_2014_6788_Fig1_HTML.html
  '@type': Dataset
  description: 'Predicted model of the S. velum symbiont cell. The diagram, based
    on the gene annotation of the symbiont genome, depicts key functional systems
    and metabolic pathways: sulfur oxidation, electron transport, ATP synthases, CO2-fixation
    via the Calvin Cycle, gluconeogenesis, polyglucose synthesis, glycolysis, TCA
    and glyoxylate cycles, synthesis of amino acids, fatty acids, lipids, isoprenoids
    via non-mevalonate pathway, and the cell wall, solute transporters, protein secretion
    systems, and the type IV pilus. Different protein categories are color-coded and
    the individual subunits indicated by shape symbols. The direction of substrate
    transport across the membrane is shown with arrows. Components of the electron
    transport chain are arranged from the lowest to the highest electronegativity
    of the electron donors (blue) and acceptors (red). The corresponding electronegativity
    values are listed next to the respective enzymes. Enzymes shared between glycolysis,
    gluconeogenesis, and the Calvin cycle are designated in green. Enzymes unique
    to these pathways are designated in red. Enzymes shared between the Calvin cycle
    and the pentose phosphate cycle are designated in blue. Amino acids which may
    be essential for the host are designated in red. Speculated pathways are designated
    with a question mark. The abbreviations used, the respective full gene product
    names, and the corresponding NCBI protein ID references are listed in Additional
    file : Table S3.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ADCY5
  - ADCY6
  - ADCY8
  - ADCY9
  - ADCY10
  - ACACA
  - ADCY2
  - ADCY3
  - CS
  - DCT
  - ADCY7
  - ENO4
  - FH
  - ADCY4
  - DGCR2
  - ACACB
  - GPR3
  - KLK15
  - ADCY1
  - HCCS
  - NDUFAB1
  - OMP
  - PRKG1
  - VCAN
  - CDIPT
  - A4GALT
  - GPI
  - PSD
  - PTDSS1
  - LAT
  - LPA
  - SLTM
  - TRRAP
  - TRIM54
  - PNPLA6
  - OGDH
  - TWIST1
  - ENO1
  - ENO2
  - ENO3
  - PYGL
  - RPE
  - phosphane
  - amine
  - Cys
  - FATTY ACIDS
  - Leu
  - Met
  - UDP
genes:
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY5
  entrez: '111'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY6
  entrez: '112'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY8
  entrez: '114'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY9
  entrez: '115'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY10
  entrez: '55811'
- word: Acc
  symbol: ACC
  source: bioentities_symbol
  hgnc_symbol: ACACA
  entrez: '31'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY2
  entrez: '108'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY3
  entrez: '109'
- word: CS
  symbol: CS
  source: hgnc_symbol
  hgnc_symbol: CS
  entrez: '1431'
- word: dct
  symbol: DCT
  source: hgnc_symbol
  hgnc_symbol: DCT
  entrez: '1638'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY7
  entrez: '113'
- word: Eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO4
  entrez: '387712'
- word: FH
  symbol: FH
  source: hgnc_symbol
  hgnc_symbol: FH
  entrez: '2271'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY4
  entrez: '196883'
- word: Lan
  symbol: LAN
  source: hgnc_alias_symbol
  hgnc_symbol: DGCR2
  entrez: '9993'
- word: Acc
  symbol: ACC
  source: bioentities_symbol
  hgnc_symbol: ACACB
  entrez: '32'
- word: AcCA
  symbol: ACCA
  source: hgnc_alias_symbol
  hgnc_symbol: GPR3
  entrez: '2827'
- word: ACO
  symbol: ACO
  source: hgnc_alias_symbol
  hgnc_symbol: KLK15
  entrez: '55554'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY1
  entrez: '107'
- word: MLS
  symbol: MLS
  source: hgnc_prev_symbol
  hgnc_symbol: HCCS
  entrez: '3052'
- word: ACP.
  symbol: ACP
  source: hgnc_alias_symbol
  hgnc_symbol: NDUFAB1
  entrez: '4706'
- word: Omp
  symbol: OMP
  source: hgnc_symbol
  hgnc_symbol: OMP
  entrez: '4975'
- word: PGK
  symbol: PGK
  source: hgnc_alias_symbol
  hgnc_symbol: PRKG1
  entrez: '5592'
- word: PGM
  symbol: PG-M
  source: hgnc_alias_symbol
  hgnc_symbol: VCAN
  entrez: '1462'
- word: Pis
  symbol: PIS
  source: hgnc_alias_symbol
  hgnc_symbol: CDIPT
  entrez: '10423'
- word: PK
  symbol: P(k)
  source: hgnc_alias_symbol
  hgnc_symbol: A4GALT
  entrez: '53947'
- word: GPI
  symbol: GPI
  source: hgnc_symbol
  hgnc_symbol: GPI
  entrez: '2821'
- word: psd
  symbol: PSD
  source: hgnc_symbol
  hgnc_symbol: PSD
  entrez: '5662'
- word: Pssa,
  symbol: PSSA
  source: hgnc_alias_symbol
  hgnc_symbol: PTDSS1
  entrez: '9791'
- word: Lat
  symbol: LAT
  source: hgnc_symbol
  hgnc_symbol: LAT
  entrez: '27040'
- word: Lpa
  symbol: LPA
  source: hgnc_symbol
  hgnc_symbol: LPA
  entrez: '4018'
- word: Met
  symbol: Met
  source: hgnc_alias_symbol
  hgnc_symbol: SLTM
  entrez: '79811'
- word: TRAP
  symbol: TR-AP
  source: hgnc_alias_symbol
  hgnc_symbol: TRRAP
  entrez: '8295'
- word: Murf
  symbol: MURF
  source: hgnc_alias_symbol
  hgnc_symbol: TRIM54
  entrez: '57159'
- word: nte
  symbol: NTE
  source: hgnc_alias_symbol
  hgnc_symbol: PNPLA6
  entrez: '10908'
- word: OGDH
  symbol: OGDH
  source: hgnc_symbol
  hgnc_symbol: OGDH
  entrez: '4967'
- word: SCS
  symbol: SCS
  source: hgnc_alias_symbol
  hgnc_symbol: TWIST1
  entrez: '7291'
- word: Eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO1
  entrez: '2023'
- word: Eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO2
  entrez: '2026'
- word: Eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO3
  entrez: '2027'
- word: PYGL
  symbol: PYGL
  source: hgnc_symbol
  hgnc_symbol: PYGL
  entrez: '5836'
- word: RPE
  symbol: RPE
  source: hgnc_symbol
  hgnc_symbol: RPE
  entrez: '6120'
chemicals:
- word: phosphane
  source: MESH
  identifier: C044646
- word: amine
  source: MESH
  identifier: D000588
- word: Cys
  source: MESH
  identifier: D003545
- word: FATTY ACIDS
  source: MESH
  identifier: D005227
- word: Leu
  source: MESH
  identifier: D007930
- word: Met
  source: MESH
  identifier: D008715
- word: UDP
  source: MESH
  identifier: D014530
diseases: []
figid_alias: PMC4287430__F1
redirect_from: /figures/PMC4287430__F1
figtype: Figure
---
