---
figid: PMC4149734__pp_243014_f5
figlink: /pmc/articles/PMC4149734/figure/fig5/
number: F5
caption: Turnover of proteins in metabolic pathways in barley leaves. A, Biosynthetic
  pathway for the cofactor TPP. NAD (NAD+), Cys, and Gly are converted to 4-methyl-5-(β-hydroxyethyl)
  thiazole phosphate (HET-P) via the rapidly degraded THI1. In the other branch of
  the pathway, another fast turnover enzyme, THIC, catalyzes the conversion of S-adenosyl
  Met (SAM), 5-aminoimidazole ribonucleotide (AIR), and hydroxymethylpyrimidine (HMP)
  to HMP phosphate (HMP-P), which is then converted to HMP pyrophosphate (HMP-PP).
  HET-P and HMP-PP are then converted to thiamine phosphate (Thiamine) and subsequently
  to TPP for use as an enzymatic cofactor. B, The tetrapyrrole biosynthetic pathway.
  In the top portion of the pathway, five proteins (glutamyl tRNA synthetase [tRNAGlu
  syn], Glu-1-semialdehyde aminomutase [GSA], porphobilinogen [PBG] deaminase, uroporyphrinogen
  [Uro] decarboxylase, and coprogen oxidase) all had fairly typical degradation rates.
  In the Mg2+ branch of the pathway, leading to chlorophyll biosynthesis, two of the
  most rapidly degraded proteins in the data set were observed, an -chelatase subunit
  (GUN5) and  cyclase (CRD1), as well as . Also observed in this branch were a CHLM
  protein from the -protoporphyrin IX methyltransferase complex and another subunit
  of the magnesium chelatase complex (CHLI). One protein was classified as a heme-binding
  protein (SOUL) and involved in tetrapyrrole metabolism but has no clearly assigned
  function. C, Photorespiratory pathway. The oxygenic reaction of Rubisco leads to
  the formation of 2-phosphogycolate, which is converted to glycolate that is exported
  to the cytosol and enters peroxisomes, where it is oxidized to glyoxylate and aminated
  to Gly by the fast turnover of glycolate oxidase and Glu:glyoxylate aminotransferase
  (AT). After decarboxylation of Gly and the synthesis of Ser in mitochondria by the
  combined action of Gly decarboxylase (GDC) and Ser hydroxymethyltransferase (SHMT),
  the Ser reenters peroxisomes, where it is deaminated to hydroxypyruvate and reduced
  to glycerate by two rapidly degrading enzymes, Ser glyoxylate aminotransferase (SGAT)
  and hydroxypyruvate reductase (HPR). Glycerate is then returned to the chloroplast,
  where it is phosphorylated by glycerate kinase (GK) to reenter the Calvin cycle
  as 3-P-glycolate and then converted to glycolate by phosphoglycolate phosphatase
  (PGP).
pmcid: PMC4149734
papertitle: Proteins with High Turnover Rate in Barley Leaves Estimated by Proteome
  Analysis Combined with in Planta Isotope Labeling.
reftext: Clark J. Nelson, et al. Plant Physiol. 2014 Sep;166(1):91-108.
pmc_ranked_result_index: '2600'
pathway_score: 0.8919101
filename: pp_243014_f5.jpg
figtitle: Turnover of proteins in metabolic pathways in barley leaves
year: '2014'
organisms:
- Hordeum vulgare
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4149734__pp_243014_f5.html
  '@type': Dataset
  description: Turnover of proteins in metabolic pathways in barley leaves. A, Biosynthetic
    pathway for the cofactor TPP. NAD (NAD+), Cys, and Gly are converted to 4-methyl-5-(β-hydroxyethyl)
    thiazole phosphate (HET-P) via the rapidly degraded THI1. In the other branch
    of the pathway, another fast turnover enzyme, THIC, catalyzes the conversion of
    S-adenosyl Met (SAM), 5-aminoimidazole ribonucleotide (AIR), and hydroxymethylpyrimidine
    (HMP) to HMP phosphate (HMP-P), which is then converted to HMP pyrophosphate (HMP-PP).
    HET-P and HMP-PP are then converted to thiamine phosphate (Thiamine) and subsequently
    to TPP for use as an enzymatic cofactor. B, The tetrapyrrole biosynthetic pathway.
    In the top portion of the pathway, five proteins (glutamyl tRNA synthetase [tRNAGlu
    syn], Glu-1-semialdehyde aminomutase [GSA], porphobilinogen [PBG] deaminase, uroporyphrinogen
    [Uro] decarboxylase, and coprogen oxidase) all had fairly typical degradation
    rates. In the Mg2+ branch of the pathway, leading to chlorophyll biosynthesis,
    two of the most rapidly degraded proteins in the data set were observed, an -chelatase
    subunit (GUN5) and  cyclase (CRD1), as well as . Also observed in this branch
    were a CHLM protein from the -protoporphyrin IX methyltransferase complex and
    another subunit of the magnesium chelatase complex (CHLI). One protein was classified
    as a heme-binding protein (SOUL) and involved in tetrapyrrole metabolism but has
    no clearly assigned function. C, Photorespiratory pathway. The oxygenic reaction
    of Rubisco leads to the formation of 2-phosphogycolate, which is converted to
    glycolate that is exported to the cytosol and enters peroxisomes, where it is
    oxidized to glyoxylate and aminated to Gly by the fast turnover of glycolate oxidase
    and Glu:glyoxylate aminotransferase (AT). After decarboxylation of Gly and the
    synthesis of Ser in mitochondria by the combined action of Gly decarboxylase (GDC)
    and Ser hydroxymethyltransferase (SHMT), the Ser reenters peroxisomes, where it
    is deaminated to hydroxypyruvate and reduced to glycerate by two rapidly degrading
    enzymes, Ser glyoxylate aminotransferase (SGAT) and hydroxypyruvate reductase
    (HPR). Glycerate is then returned to the chloroplast, where it is phosphorylated
    by glycerate kinase (GK) to reenter the Calvin cycle as 3-P-glycolate and then
    converted to glycolate by phosphoglycolate phosphatase (PGP).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - HEBP2
  - PGP
  - IMMT
  - HPR
  - POR
  - SHMT1
  - SAFB
  - Divinyl protochlorophyllide
  - Hydroxymethylbilane
  - Mg-protoporphyrin
  - Biliverdin
  - Coproporphyrinogen
  - Cys
  - glycine
  - Gly
  - 3-P-glycolate
  - glyoxylate
  - malate
  - Monovinyl protochlorophyllide
  - Fe-protoporphyrin
  - Protoporphyrin
  - serine
  - Uroporphyrinogen
  - Glutamate
  - glutamate
genes:
- word: 'SOUL:'
  symbol: SOUL
  source: hgnc_alias_symbol
  hgnc_symbol: HEBP2
  entrez: '23593'
- word: PGP
  symbol: PGP
  source: hgnc_symbol
  hgnc_symbol: PGP
  entrez: '283871'
- word: HMP
  symbol: HMP
  source: hgnc_alias_symbol
  hgnc_symbol: IMMT
  entrez: '10989'
- word: 'HPR:'
  symbol: HPR
  source: hgnc_symbol
  hgnc_symbol: HPR
  entrez: '3250'
- word: POR
  symbol: POR
  source: hgnc_symbol
  hgnc_symbol: POR
  entrez: '5447'
- word: SHMT
  symbol: SHMT
  source: hgnc_alias_symbol
  hgnc_symbol: SHMT1
  entrez: '6470'
- word: HET-P
  symbol: HET
  source: hgnc_alias_symbol
  hgnc_symbol: SAFB
  entrez: '6294'
chemicals:
- word: Divinyl protochlorophyllide
  source: MESH
  identifier: C020301
- word: Hydroxymethylbilane
  source: MESH
  identifier: C024393
- word: Mg-protoporphyrin
  source: MESH
  identifier: C039629
- word: Biliverdin
  source: MESH
  identifier: D001664
- word: Coproporphyrinogen
  source: MESH
  identifier: D003305
- word: Cys
  source: MESH
  identifier: D003545
- word: glycine
  source: MESH
  identifier: D005998
- word: Gly
  source: MESH
  identifier: D005998
- word: 3-P-glycolate
  source: MESH
  identifier: D006016
- word: glyoxylate
  source: MESH
  identifier: D006038
- word: malate
  source: MESH
  identifier: D008293
- word: Monovinyl protochlorophyllide
  source: MESH
  identifier: D011521
- word: Fe-protoporphyrin
  source: MESH
  identifier: D011524
- word: Protoporphyrin
  source: MESH
  identifier: D011524
- word: serine
  source: MESH
  identifier: D012694
- word: Uroporphyrinogen
  source: MESH
  identifier: D014577
- word: Glutamate
  source: MESH
  identifier: D018698
- word: glutamate
  source: MESH
  identifier: D018698
diseases: []
figid_alias: PMC4149734__F5
redirect_from: /figures/PMC4149734__F5
figtype: Figure
---
