---
figid: PMC4908344__btw260f4p
figlink: /pmc/articles/PMC4908344/figure/btw260-F4/
number: F4
caption: Examples of extracted features as enzyme-specific chemical transformation
  patterns. (a) The left panel shows two substrate–product pairs (RP01073 and RP01958)
  associated with enzyme ortholog K01592, tyrosine decarboxylase. (b) The left panel
  shows three substrate–product pairs (RP01224, RP04067 and RP01667) associated with
  enzyme ortholog K00052, 3-isopropylmalate dehydrogenase. In (a) and (b), the chemical
  graph alignments of the compounds are shown in the middle. Red dashed lines indicate
  the elimination of chemical bonds, red dotted lines indicate the atoms that change
  their labels (functional groups), and blue dotted lines indicate the atoms that
  are preserved during the reaction. The corresponding PACHA feature vectors are shown
  at the right. Features representing conserved chemical substructures are colored
  black and the features representing chemical changes are colored red
pmcid: PMC4908344
papertitle: Simultaneous prediction of enzyme orthologs from chemical transformation
  patterns for de novo metabolic pathway reconstruction.
reftext: Yasuo Tabei, et al. Bioinformatics. 2016 Jun 15;32(12):i278-i287.
pmc_ranked_result_index: '187466'
pathway_score: 0.8008669
filename: btw260f4p.jpg
figtitle: Examples of extracted features as enzyme-specific chemical transformation
  patterns
year: '2016'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4908344__btw260f4p.html
  '@type': Dataset
  description: Examples of extracted features as enzyme-specific chemical transformation
    patterns. (a) The left panel shows two substrate–product pairs (RP01073 and RP01958)
    associated with enzyme ortholog K01592, tyrosine decarboxylase. (b) The left panel
    shows three substrate–product pairs (RP01224, RP04067 and RP01667) associated
    with enzyme ortholog K00052, 3-isopropylmalate dehydrogenase. In (a) and (b),
    the chemical graph alignments of the compounds are shown in the middle. Red dashed
    lines indicate the elimination of chemical bonds, red dotted lines indicate the
    atoms that change their labels (functional groups), and blue dotted lines indicate
    the atoms that are preserved during the reaction. The corresponding PACHA feature
    vectors are shown at the right. Features representing conserved chemical substructures
    are colored black and the features representing chemical changes are colored red
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CBX1
  - HIST1H3C
  - CBY1
  - 3-isopropylmalate
  - tyrosine
genes:
- word: (CBx)
  symbol: CBX
  source: hgnc_alias_symbol
  hgnc_symbol: CBX1
  entrez: '10951'
- word: H3C
  symbol: H3/c
  source: hgnc_alias_symbol
  hgnc_symbol: HIST1H3C
  entrez: '8352'
- word: (CBy
  symbol: Cby
  source: hgnc_alias_symbol
  hgnc_symbol: CBY1
  entrez: '25776'
chemicals:
- word: 3-isopropylmalate
  source: MESH
  identifier: C025313
- word: tyrosine
  source: MESH
  identifier: D014443
diseases: []
figid_alias: PMC4908344__F4
redirect_from: /figures/PMC4908344__F4
figtype: Figure
---
