---
figid: PMC4320441__13104_2014_3461_Fig3_HTML
figlink: /pmc/articles/PMC4320441/figure/Fig3/
number: F3
caption: Pathway analysis of 49 genes whose functional domains were changed by DAS
  exons. The EGFR/mTOR signaling pathway was analyzed using 49 genes, where the functionalities
  of the changed domains were related to gene annotations. Thus, representative functions
  of these genes may be regulated by alternative splicing events. Seven of them were
  shown with larger fonts with an inner color. The lines indicate the intermolecular
  relationships. Medium fonts indicate differentially expressed genes suggested by
  a previous analysis []. Outer colors indicate the elevation or reduction of their
  expressions. Smaller fonts indicate closely related upstream or downstream proteins.
  Blue or red indicate successfully validated gain of function and elevation of expression
  or loss of function and reduction of expression, respectively. Success of the validation
  by semi-quantitative RT-PCR was determined by an Fc value > 1.5 between undifferentiated
  P19 cells (day 0) and the cells at the neuronal stage (day 7) (see also Figure ).
  Light blue or orange colors indicate unsuccessful validation, even though these
  genes were suggested by a previous microarray experiment. The violet inner color
  of Kitlg indicates that its domains were suggested to change but the gain or loss
  of function was difficult to determine. Tsc2 was not one of the extracted 49 genes,
  but was one of 262 DAS exon genes. Indeed, its alternative isoforms have no difference
  in their functional protein domains. Additionally, the transcript of p53 (TP53)
  was not listed as a differentially expressed gene; however, its protein level was
  likely to be upregulated, as described in a previous report [].
pmcid: PMC4320441
papertitle: Computational extraction of a neural molecular network through alternative
  splicing.
reftext: Shafiul Alam, et al. BMC Res Notes. 2014;7:934.
pmc_ranked_result_index: '181641'
pathway_score: 0.937713
filename: 13104_2014_3461_Fig3_HTML.jpg
figtitle: 49 genes whose functional domains were changed by DAS exons
year: '2014'
organisms:
- Homo sapiens
ndex: 0b4e6333-defd-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4320441__13104_2014_3461_Fig3_HTML.html
  '@type': Dataset
  description: Pathway analysis of 49 genes whose functional domains were changed
    by DAS exons. The EGFR/mTOR signaling pathway was analyzed using 49 genes, where
    the functionalities of the changed domains were related to gene annotations. Thus,
    representative functions of these genes may be regulated by alternative splicing
    events. Seven of them were shown with larger fonts with an inner color. The lines
    indicate the intermolecular relationships. Medium fonts indicate differentially
    expressed genes suggested by a previous analysis []. Outer colors indicate the
    elevation or reduction of their expressions. Smaller fonts indicate closely related
    upstream or downstream proteins. Blue or red indicate successfully validated gain
    of function and elevation of expression or loss of function and reduction of expression,
    respectively. Success of the validation by semi-quantitative RT-PCR was determined
    by an Fc value > 1.5 between undifferentiated P19 cells (day 0) and the cells
    at the neuronal stage (day 7) (see also Figure ). Light blue or orange colors
    indicate unsuccessful validation, even though these genes were suggested by a
    previous microarray experiment. The violet inner color of Kitlg indicates that
    its domains were suggested to change but the gain or loss of function was difficult
    to determine. Tsc2 was not one of the extracted 49 genes, but was one of 262 DAS
    exon genes. Indeed, its alternative isoforms have no difference in their functional
    protein domains. Additionally, the transcript of p53 (TP53) was not listed as
    a differentially expressed gene; however, its protein level was likely to be upregulated,
    as described in a previous report [].
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - EGFR
  - RHEB
  - MAPK3
  - STAT3
  - AKT1
  - ARAP1
  - MAPK1
  - EGF
  - TGFA
  - EP400
  - CDKN2A
  - TSC2
  - MDM2
  - SHC1
  - GRB2
  - STAT2
  - KITLG
  - MAPK8
  - MAPK11
  - ARHGEF12
  - MAPK9
  - MAPK10
  - MAPK13
  - MAPK12
  - STAT5B
  - MTOR
  - STAT1
  - KIT
  - TSC1
  - MAPK14
  - STAT4
  - STAT5A
  - STAT6
  - TP53
  - HBEGF
  - Cancer
genes:
- word: EGFR
  symbol: EGFR
  source: hgnc_symbol
  hgnc_symbol: EGFR
  entrez: '1956'
- word: RHEB
  symbol: RHEB
  source: hgnc_symbol
  hgnc_symbol: RHEB
  entrez: '6009'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK3
  entrez: '5595'
- word: STAT
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT3
  entrez: '6774'
- word: AKT1
  symbol: AKT1
  source: hgnc_symbol
  hgnc_symbol: AKT1
  entrez: '207'
- word: ARAP1
  symbol: ARAP1
  source: hgnc_symbol
  hgnc_symbol: ARAP1
  entrez: '116985'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK1
  entrez: '5594'
- word: EGF
  symbol: EGF
  source: hgnc_symbol
  hgnc_symbol: EGF
  entrez: '1950'
- word: TGFA
  symbol: TGFA
  source: hgnc_symbol
  hgnc_symbol: TGFA
  entrez: '7039'
- word: EP400
  symbol: EP400
  source: hgnc_symbol
  hgnc_symbol: EP400
  entrez: '57634'
- word: CDKN2A
  symbol: CDKN2A
  source: hgnc_symbol
  hgnc_symbol: CDKN2A
  entrez: '1029'
- word: TSC2
  symbol: TSC2
  source: hgnc_symbol
  hgnc_symbol: TSC2
  entrez: '7249'
- word: MDM2
  symbol: MDM2
  source: hgnc_symbol
  hgnc_symbol: MDM2
  entrez: '4193'
- word: SHC1
  symbol: SHC1
  source: hgnc_symbol
  hgnc_symbol: SHC1
  entrez: '6464'
- word: GRB2
  symbol: GRB2
  source: hgnc_symbol
  hgnc_symbol: GRB2
  entrez: '2885'
- word: STAT
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT2
  entrez: '6773'
- word: KITLG
  symbol: KITLG
  source: hgnc_symbol
  hgnc_symbol: KITLG
  entrez: '4254'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK8
  entrez: '5599'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK11
  entrez: '5600'
- word: ARHGEF12
  symbol: ARHGEF12
  source: hgnc_symbol
  hgnc_symbol: ARHGEF12
  entrez: '23365'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK9
  entrez: '5601'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK10
  entrez: '5602'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK13
  entrez: '5603'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK12
  entrez: '6300'
- word: MAPK1
  symbol: MAPK1
  source: hgnc_symbol
  hgnc_symbol: MAPK1
  entrez: '5594'
- word: STAT
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT5B
  entrez: '6777'
- word: MTOR
  symbol: MTOR
  source: hgnc_symbol
  hgnc_symbol: MTOR
  entrez: '2475'
- word: STAT
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT1
  entrez: '6772'
- word: KIT
  symbol: KIT
  source: hgnc_symbol
  hgnc_symbol: KIT
  entrez: '3815'
- word: TSC1
  symbol: TSC1
  source: hgnc_symbol
  hgnc_symbol: TSC1
  entrez: '7248'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK14
  entrez: '1432'
- word: STAT
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT4
  entrez: '6775'
- word: STAT
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT5A
  entrez: '6776'
- word: STAT
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT6
  entrez: '6778'
- word: TP53*
  symbol: TP53
  source: hgnc_symbol
  hgnc_symbol: TP53
  entrez: '7157'
- word: HBEGF
  symbol: HBEGF
  source: hgnc_symbol
  hgnc_symbol: HBEGF
  entrez: '1839'
chemicals: []
diseases:
- word: Cancer
  source: ''
  identifier: ''
figid_alias: PMC4320441__F3
redirect_from: /figures/PMC4320441__F3
figtype: Figure
---
