---
figid: PMC4137073__12864_2014_6353_Fig3_HTML
figlink: /pmc/articles/PMC4137073/figure/Fig3/
number: F3
caption: 'Examples of emergent amino acid metabolism shared between the Moranella
  endobia and Tremblaya princeps genomes. This figure illustrates examples of emergent
  metabolic pathways predicted between symbiotic prokaryotes Candidatus Moranella
  endobia and Candidatus Tremblaya princeps. Enyzmes found in Moranella (red), Tremblaya
  (blue), or both taxa (purple) are highlighted in the pathway glyph diagrams, showing
  patterns of potentially emergent metabolism. A complete description of all amino
  acid pathways can be found in Additional file : Figure S3.'
pmcid: PMC4137073
papertitle: Metabolic pathways for the whole community.
reftext: Niels W Hanson, et al. BMC Genomics. 2014;15(1):619.
pmc_ranked_result_index: '41653'
pathway_score: 0.7745508
filename: 12864_2014_6353_Fig3_HTML.jpg
figtitle: Examples of emergent amino acid metabolism shared between the Moranella
  endobia and Tremblaya princeps genomes
year: '2014'
organisms:
- Candidatus Moranella endobia
- environmental samples
- unclassified sequences
- Planococcus citri
- Candidatus Tremblaya princeps
- Bacillus subtilis subsp. subtilis str. 168
- metagenome
- metagenomes
- Nitrospira moscoviensis
- Escherichia coli str. K-12 substr. MG1655
- Escherichia coli
- Caulobacter vibrioides NA1000
- human metagenome
- Aurantimonas manganoxydans
- Escherichia coli K-12
- Helicobacter pylori 26695
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4137073__12864_2014_6353_Fig3_HTML.html
  '@type': Dataset
  description: 'Examples of emergent amino acid metabolism shared between the Moranella
    endobia and Tremblaya princeps genomes. This figure illustrates examples of emergent
    metabolic pathways predicted between symbiotic prokaryotes Candidatus Moranella
    endobia and Candidatus Tremblaya princeps. Enyzmes found in Moranella (red), Tremblaya
    (blue), or both taxa (purple) are highlighted in the pathway glyph diagrams, showing
    patterns of potentially emergent metabolism. A complete description of all amino
    acid pathways can be found in Additional file : Figure S3.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - WDTC1
  - DAND5
  - indole
  - indole-Sgycerol
  - Arginine
  - aspartate
  - bicarbonate
  - Lysine
  - 7-phosphate
  - L-asporyl--phosphate
  - cartumoyl phosphate
  - phosphate
  - L-serine
  - L-tryptophan
  - Tryptophan
  - L-glutamate
genes:
- word: ADP
  symbol: ADP
  source: hgnc_alias_symbol
  hgnc_symbol: WDTC1
  entrez: '23038'
- word: dte
  symbol: DTE
  source: hgnc_alias_symbol
  hgnc_symbol: DAND5
  entrez: '199699'
chemicals:
- word: indole
  source: MESH
  identifier: C030374
- word: indole-Sgycerol
  source: MESH
  identifier: C030374
- word: Arginine
  source: MESH
  identifier: D001120
- word: aspartate
  source: MESH
  identifier: D001224
- word: bicarbonate
  source: MESH
  identifier: D001639
- word: Lysine
  source: MESH
  identifier: D008239
- word: 7-phosphate
  source: MESH
  identifier: D010710
- word: L-asporyl--phosphate
  source: MESH
  identifier: D010710
- word: cartumoyl phosphate
  source: MESH
  identifier: D010710
- word: phosphate
  source: MESH
  identifier: D010710
- word: L-serine
  source: MESH
  identifier: D012694
- word: L-tryptophan
  source: MESH
  identifier: D014364
- word: Tryptophan
  source: MESH
  identifier: D014364
- word: L-glutamate
  source: MESH
  identifier: D018698
diseases: []
figid_alias: PMC4137073__F3
redirect_from: /figures/PMC4137073__F3
figtype: Figure
---
