---
figid: PMC4816099__cdd2015122f2
figlink: /pmc/articles/PMC4816099/figure/fig2/
number: F2
caption: Controlling a variety of extracellular and intracellular processes via the
  phospholipid code. (a) During apoptosis, loss of phospholipid asymmetry due to inactivation
  of ATP11C flippase induces PS exposure. PS acts as an ‘eat me' signal and can be
  detected by receptors on neighboring phagocytes (directly) and serum opsonins (indirectly)
  to trigger engulfment. (b) Similarly, cells undergoing necrosis exposes additional
  phospholipids such as PI(4)P and PA, which can be recognized by serum opsonins such
  as histidine-rich glycoprotein and mannose-binding lectin to trigger phagocytic
  engulfment. (c) The caspase-independent programmed cell death pathway of necroptosis
  involves formation of the ‘necrosome' that activates MLKL, leading to its translocation
  to the plasma membrane. MLKL oligomerization upon contact with PIP2 (PI(4,5)P2)
  and PI(5)P then mediates membrane permeabilization. (d) In the tumor microenvironment,
  elevated cytosolic Ca2+ and ROS production can trigger a loss in phospholipid asymmetry
  and expose anionic phospholipids on tumor vascular endothelium. PS exposure and
  TF overexpression can then promote assembly of circulating coagulation factors such
  as FIXa and FVIIIa on the tumor cell membrane to initiate blood clotting, which
  can drive tumor progression through elevated growth factor signaling (left). Abberant
  regulation of PI3K can lead to elevated levels of the plasma membrane second messenger
  PIP3 (PI(3,4,5)P3), a process that could mediate recruitment and activation of Akt
  and subsequently mTOR to promote tumor proliferation and survival (right). (e) The
  Gram-negative bacteria, V. parahaemolyticus uses MAM7, an adhesion molecule that
  interacts with both PA and fibronectin on intestinal epithelial cells, to mediate
  host cell entry (left). The oomycetes P. sojae translocates an effector molecule,
  Avr1b into the host cell membrane to repress host defenses, via binding of its RXLR
  motif to outer membrane PI(3)P (right). (f) Viruses such as the ebola virus use
  ‘apoptotic mimicry' to enter target cells. Ebola virus exposes PS on its viral envelope
  that binds to the TIM-1 receptor on host cells to facilitate viral uptake. (g) In
  HIV-1 infected cells, the HIV-1 Gag protein is trafficked to the plasma membrane
  via its matrix (MA) domain where it interacts with PIP2 to initiate viral budding.
  (h) Enzymatic modification of host cell phospholipids by microbial virulence factors
  can aid infection. S. flexneri secretes IpgD into intestinal host cells that acts
  as phosphoinositol 4-phosphatase to deplete plasma membrane PIP2, leading to actin
  remodeling and membrane ruffling (left). SigD from S. typhimurium displays similar
  phosphatase activity to induce membrane ruffling and facilitate formation of Salmonella-containing
  vacuoles, thereby enabling bacterial replication (right). (i) The plant defensin
  NaD1 induces cytolysis of both fungal and mammalian tumor cells through binding
  to plasma membrane PIP2, with which it forms an oligomeric complex, leading to membrane
  permeabilization. TAM, Tyro-3, Axl, Mer; Gas6, growth arrest-specific 6; MGF-E8,
  milk fat globule-EGF factor 8; TIM-4, T cell/transmembrane, Ig, mucin-4; BAI-1,
  brain-specific angiogenesis inhibitor 1; HRG, histidine-rich glycoprotein; CRT,
  calreticulin; MBL, mannose-binding lectin; RhoA, ras homolog gene family, member
  A; ROCK, rho-associated protein kinase; LIMK, LIM motif-containing protein kinase;
  Co, cofilin; EBOV, Ebola virus; TIM-1, T cell immunoglobulin and mucin domain 1;
  Env, envelope glycoprotein
pmcid: PMC4816099
papertitle: 'The phospholipid code: a key component of dying cell recognition, tumor
  progression and host–microbe interactions.'
reftext: A A Baxter, et al. Cell Death Differ. 2015 Dec;22(12):1893-1905.
pmc_ranked_result_index: '235006'
pathway_score: 0.8415906
filename: cdd2015122f2.jpg
figtitle: Controlling a variety of extracellular and intracellular processes via the
  phospholipid code
year: '2015'
organisms:
- Homo sapiens
ndex: 0a811f48-de9c-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4816099__cdd2015122f2.html
  '@type': Dataset
  description: Controlling a variety of extracellular and intracellular processes
    via the phospholipid code. (a) During apoptosis, loss of phospholipid asymmetry
    due to inactivation of ATP11C flippase induces PS exposure. PS acts as an ‘eat
    me' signal and can be detected by receptors on neighboring phagocytes (directly)
    and serum opsonins (indirectly) to trigger engulfment. (b) Similarly, cells undergoing
    necrosis exposes additional phospholipids such as PI(4)P and PA, which can be
    recognized by serum opsonins such as histidine-rich glycoprotein and mannose-binding
    lectin to trigger phagocytic engulfment. (c) The caspase-independent programmed
    cell death pathway of necroptosis involves formation of the ‘necrosome' that activates
    MLKL, leading to its translocation to the plasma membrane. MLKL oligomerization
    upon contact with PIP2 (PI(4,5)P2) and PI(5)P then mediates membrane permeabilization.
    (d) In the tumor microenvironment, elevated cytosolic Ca2+ and ROS production
    can trigger a loss in phospholipid asymmetry and expose anionic phospholipids
    on tumor vascular endothelium. PS exposure and TF overexpression can then promote
    assembly of circulating coagulation factors such as FIXa and FVIIIa on the tumor
    cell membrane to initiate blood clotting, which can drive tumor progression through
    elevated growth factor signaling (left). Abberant regulation of PI3K can lead
    to elevated levels of the plasma membrane second messenger PIP3 (PI(3,4,5)P3),
    a process that could mediate recruitment and activation of Akt and subsequently
    mTOR to promote tumor proliferation and survival (right). (e) The Gram-negative
    bacteria, V. parahaemolyticus uses MAM7, an adhesion molecule that interacts with
    both PA and fibronectin on intestinal epithelial cells, to mediate host cell entry
    (left). The oomycetes P. sojae translocates an effector molecule, Avr1b into the
    host cell membrane to repress host defenses, via binding of its RXLR motif to
    outer membrane PI(3)P (right). (f) Viruses such as the ebola virus use ‘apoptotic
    mimicry' to enter target cells. Ebola virus exposes PS on its viral envelope that
    binds to the TIM-1 receptor on host cells to facilitate viral uptake. (g) In HIV-1
    infected cells, the HIV-1 Gag protein is trafficked to the plasma membrane via
    its matrix (MA) domain where it interacts with PIP2 to initiate viral budding.
    (h) Enzymatic modification of host cell phospholipids by microbial virulence factors
    can aid infection. S. flexneri secretes IpgD into intestinal host cells that acts
    as phosphoinositol 4-phosphatase to deplete plasma membrane PIP2, leading to actin
    remodeling and membrane ruffling (left). SigD from S. typhimurium displays similar
    phosphatase activity to induce membrane ruffling and facilitate formation of Salmonella-containing
    vacuoles, thereby enabling bacterial replication (right). (i) The plant defensin
    NaD1 induces cytolysis of both fungal and mammalian tumor cells through binding
    to plasma membrane PIP2, with which it forms an oligomeric complex, leading to
    membrane permeabilization. TAM, Tyro-3, Axl, Mer; Gas6, growth arrest-specific
    6; MGF-E8, milk fat globule-EGF factor 8; TIM-4, T cell/transmembrane, Ig, mucin-4;
    BAI-1, brain-specific angiogenesis inhibitor 1; HRG, histidine-rich glycoprotein;
    CRT, calreticulin; MBL, mannose-binding lectin; RhoA, ras homolog gene family,
    member A; ROCK, rho-associated protein kinase; LIMK, LIM motif-containing protein
    kinase; Co, cofilin; EBOV, Ebola virus; TIM-1, T cell immunoglobulin and mucin
    domain 1; Env, envelope glycoprotein
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ACTC1
  - ACTG2
  - ACTA1
  - ACTG1
  - AKT2
  - ACTA2
  - ACTB
  - CALR
  - GAST
  - MT-ND1
  - RHOA
  - ROCK1
  - AKT1
  - BAAT
  - AKT3
  - TNFRSF1A
  - TRADD
  - ROCK2
  - TNF
genes:
- word: actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTC1
  entrez: '70'
- word: actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTG2
  entrez: '72'
- word: Actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTA1
  entrez: '58'
- word: actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTG1
  entrez: '71'
- word: Akt
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT2
  entrez: '208'
- word: actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTA2
  entrez: '59'
- word: actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTB
  entrez: '60'
- word: CRT
  symbol: CRT
  source: hgnc_alias_symbol
  hgnc_symbol: CALR
  entrez: '811'
- word: Gass
  symbol: GAS
  source: hgnc_prev_symbol
  hgnc_symbol: GAST
  entrez: '2520'
- word: NaD1
  symbol: NAD1
  source: hgnc_alias_symbol
  hgnc_symbol: MT-ND1
  entrez: '4535'
- word: RhoA
  symbol: RhoA
  source: hgnc_alias_symbol
  hgnc_symbol: RHOA
  entrez: '387'
- word: ROCK
  symbol: ROCK
  source: bioentities_symbol
  hgnc_symbol: ROCK1
  entrez: '6093'
- word: Akt
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT1
  entrez: '207'
- word: Actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTA2
  entrez: '59'
- word: actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTA1
  entrez: '58'
- word: BAT
  symbol: BAT
  source: hgnc_alias_symbol
  hgnc_symbol: BAAT
  entrez: '570'
- word: Akt
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT3
  entrez: '10000'
- word: TNFR
  symbol: TNF-R
  source: hgnc_alias_symbol
  hgnc_symbol: TNFRSF1A
  entrez: '7132'
- word: Actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTB
  entrez: '60'
- word: Actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTG2
  entrez: '72'
- word: Actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTC1
  entrez: '70'
- word: Actin
  symbol: Actin
  source: bioentities_symbol
  hgnc_symbol: ACTG1
  entrez: '71'
- word: TRADD
  symbol: TRADD
  source: hgnc_symbol
  hgnc_symbol: TRADD
  entrez: '8717'
- word: ROCK
  symbol: ROCK
  source: bioentities_symbol
  hgnc_symbol: ROCK2
  entrez: '9475'
- word: TNF-a
  symbol: TNF-alpha
  source: hgnc_alias_symbol
  hgnc_symbol: TNF
  entrez: '7124'
chemicals: []
diseases: []
figid_alias: PMC4816099__F2
redirect_from: /figures/PMC4816099__F2
figtype: Figure
---
