---
figid: PMC4978736__fpls-07-01138-g0002
figlink: /pmc/articles/PMC4978736/figure/F2/
number: Figure 2
caption: 'Gene expression, proteome, and metabolome contrasts in young and immature
  tissues, assigned to the central carbon metabolism. Gene expression and proteomics
  data are provided as log2FC (fold change) but for clarity of the figure only provided
  if either proteome or transcriptome changed by more than logFC of 0.5. Metabolite
  data is provided as concentration fold change. Omics data are highlighted in green,
  red, and yellow. Green: Increase in gene expression, protein accumulation, or in
  metabolite concentration in mature tissues compared to immature tissues. Red: Decrease
  in gene expression, in protein accumulation, or in metabolite concentration in mature
  tissues compared to immature tissues. Yellow: Up and down regulation of genes/enzymes
  that characterize isoforms in different tissues and organelles. Gray: No statistically
  significant difference between tissues. Numbers refer to enzymatic step reactions.
  Numbers (1–34) refers to enzymatic step reactions. 1, hexokinase; 2, glucose-6-phosphate
  isomerase; 3, glyceraldehyde 3-phosphate dehydrogenase (cytosolic and plastidial
  isoforms); 4, enolase (cytosolic and plastidial isoforms); 5, pyruvate kinase (cytosolic
  and plastidial isoforms); 6, pyruvate dehydrogenase complex; 7, citrate synthase
  (microchondrial and glyoxomal isoforms); 8, isocitrate dehydrogenase (mitochondrial);
  9, malate dehydrohenase (mitochondrial, plastidic, glyoxysomal and cytosolic isoforms);
  10, nitrate reductase; 11, glutamine synthetase; 12, glutamate synthase; 13, ribose
  5-phosphate isomerase; 14, histidyl-tRNA synthetase; 15, 2-oxoglutarate aminotransferase;
  16, glycine hydroxymethyltransferase; 17, tryptophan synthase; 18,transaminase;
  19, aromatic amino acid aminotransferases; 20, peroxidase; 21, branched-chain amino
  acid aminotransferase; 22, valyl-tRNA synthetase; 23, aminoacyl-tRNA synthetase
  family protein; 23, alanine aminotransferase; 24, aldehyde dehydrogenase; 25, acetyl-CoA
  synthetase (acetate-CoA ligase); 26, fatty acid synthase (acyl-ACP synthases); 27,
  aspartate aminotransferase; 28, asparagine synthetase; 29, threonine synthase, threonyl-tRNA
  synthetase; 30, branched-chain amino acid aminotransferase; 31, isoleucyl-tRNA synthetase;
  32, diaminopimelate decarboxylase; 33, glycosyl hydrolase family; 34, cellulose
  synthase; 35, Rubisco (carboxylation). Some steps were omitted for the sake of simplicity.'
pmcid: PMC4978736
papertitle: 'Metabolic Reconstruction of Setaria italica: A Systems Biology Approach
  for Integrating Tissue-Specific Omics and Pathway Analysis of Bioenergy Grasses.'
reftext: Cristiana G. de Oliveira Dal'Molin, et al. Front Plant Sci. 2016;7:1138.
pmc_ranked_result_index: '234258'
pathway_score: 0.897742
filename: fpls-07-01138-g0002.jpg
figtitle: Gene expression, proteome, and metabolome contrasts in young and immature
  tissues, assigned to the central carbon metabolism
year: '2016'
organisms: Homo sapiens
ndex: 41202604-df13-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC4978736__fpls-07-01138-g0002.html
  '@type': Dataset
  description: 'Gene expression, proteome, and metabolome contrasts in young and immature
    tissues, assigned to the central carbon metabolism. Gene expression and proteomics
    data are provided as log2FC (fold change) but for clarity of the figure only provided
    if either proteome or transcriptome changed by more than logFC of 0.5. Metabolite
    data is provided as concentration fold change. Omics data are highlighted in green,
    red, and yellow. Green: Increase in gene expression, protein accumulation, or
    in metabolite concentration in mature tissues compared to immature tissues. Red:
    Decrease in gene expression, in protein accumulation, or in metabolite concentration
    in mature tissues compared to immature tissues. Yellow: Up and down regulation
    of genes/enzymes that characterize isoforms in different tissues and organelles.
    Gray: No statistically significant difference between tissues. Numbers refer to
    enzymatic step reactions. Numbers (1–34) refers to enzymatic step reactions. 1,
    hexokinase; 2, glucose-6-phosphate isomerase; 3, glyceraldehyde 3-phosphate dehydrogenase
    (cytosolic and plastidial isoforms); 4, enolase (cytosolic and plastidial isoforms);
    5, pyruvate kinase (cytosolic and plastidial isoforms); 6, pyruvate dehydrogenase
    complex; 7, citrate synthase (microchondrial and glyoxomal isoforms); 8, isocitrate
    dehydrogenase (mitochondrial); 9, malate dehydrohenase (mitochondrial, plastidic,
    glyoxysomal and cytosolic isoforms); 10, nitrate reductase; 11, glutamine synthetase;
    12, glutamate synthase; 13, ribose 5-phosphate isomerase; 14, histidyl-tRNA synthetase;
    15, 2-oxoglutarate aminotransferase; 16, glycine hydroxymethyltransferase; 17,
    tryptophan synthase; 18,transaminase; 19, aromatic amino acid aminotransferases;
    20, peroxidase; 21, branched-chain amino acid aminotransferase; 22, valyl-tRNA
    synthetase; 23, aminoacyl-tRNA synthetase family protein; 23, alanine aminotransferase;
    24, aldehyde dehydrogenase; 25, acetyl-CoA synthetase (acetate-CoA ligase); 26,
    fatty acid synthase (acyl-ACP synthases); 27, aspartate aminotransferase; 28,
    asparagine synthetase; 29, threonine synthase, threonyl-tRNA synthetase; 30, branched-chain
    amino acid aminotransferase; 31, isoleucyl-tRNA synthetase; 32, diaminopimelate
    decarboxylase; 33, glycosyl hydrolase family; 34, cellulose synthase; 35, Rubisco
    (carboxylation). Some steps were omitted for the sake of simplicity.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - AKR1B10
  - TYR
  - KLK15
  - MET
  - NOS1
  - MAL
  - THRB
  - TYRP1
  - RASA1
  - NOS2
  - THRA
  - PAEP
  - NOS3
  - Acetate
  - ALA
  - O Ethanol
  - ARG
  - ASN
  - CYS
  - Fatty acids
  - Glycerol
  - GLY
  - LEU
  - PRO
  - Sucrose
  - TYR
  - UDP
  - VAL
genes:
- word: HIS
  symbol: HIS
  source: hgnc_alias_symbol
  hgnc_symbol: AKR1B10
  entrez: '57016'
- word: TYR
  symbol: TYR
  source: hgnc_symbol
  hgnc_symbol: TYR
  entrez: '7299'
- word: ACO
  symbol: ACO
  source: hgnc_alias_symbol
  hgnc_symbol: KLK15
  entrez: '55554'
- word: MET
  symbol: MET
  source: hgnc_symbol
  hgnc_symbol: MET
  entrez: '4233'
- word: NOs
  symbol: NOS
  source: bioentities_symbol
  hgnc_symbol: NOS1
  entrez: '4842'
- word: MAL
  symbol: MAL
  source: hgnc_symbol
  hgnc_symbol: MAL
  entrez: '4118'
- word: THR
  symbol: THR
  source: bioentities_symbol
  hgnc_symbol: THRB
  entrez: '7068'
- word: TRP
  symbol: TRP
  source: hgnc_alias_symbol
  hgnc_symbol: TYRP1
  entrez: '7306'
- word: GAЗP
  symbol: GAP
  source: hgnc_alias_symbol
  hgnc_symbol: RASA1
  entrez: '5921'
- word: NOs
  symbol: NOS
  source: bioentities_symbol
  hgnc_symbol: NOS2
  entrez: '4843'
- word: THR
  symbol: THR
  source: bioentities_symbol
  hgnc_symbol: THRA
  entrez: '7067'
- word: PEP
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
- word: NOs
  symbol: NOS
  source: bioentities_symbol
  hgnc_symbol: NOS3
  entrez: '4846'
chemicals:
- word: Acetate
  source: MESH
  identifier: D000085
- word: ALA
  source: MESH
  identifier: D000409
- word: O Ethanol
  source: MESH
  identifier: D000431
- word: ARG
  source: MESH
  identifier: D001120
- word: ASN
  source: MESH
  identifier: D001216
- word: CYS
  source: MESH
  identifier: D003545
- word: Fatty acids
  source: MESH
  identifier: D005227
- word: Glycerol
  source: MESH
  identifier: D005990
- word: GLY
  source: MESH
  identifier: D005998
- word: LEU
  source: MESH
  identifier: D007930
- word: PRO
  source: MESH
  identifier: D011392
- word: Sucrose
  source: MESH
  identifier: D013395
- word: TYR
  source: MESH
  identifier: D014443
- word: UDP
  source: MESH
  identifier: D014530
- word: VAL
  source: MESH
  identifier: D014633
diseases: []
---
