---
figid: PMC5145879__fphys-07-00606-g0001
figlink: /pmc/articles/PMC5145879/figure/F1/
number: F1
caption: Bioinformatics workflow of microbiome profiling. The first step is the data
  acquisition that can be derived from any NGS technology (Illumina, IonProton, PacBio)
  and generating of the FASTQ file to proceed with the analysis. In the quality control
  step, the aim is to clean and eliminate possible errors in data, for example, to
  discard low quality score and very short reads, quimeric and adapter sequences.
  In addition, it is important to evaluate the presence of some contaminants from
  other organisms, specific GC content bias or repeated sequences that may interfere
  with the assembly step. The following steps depend on the nature of data, whether
  the aim is to sequence a marker gene, such as the 16S rRNA gene or ITS, or to perform
  shotgun metagenomic sequencing. OTU clustering is a critical step and many algorithms
  and strategies have emerged to accomplish a proper classification of sequences for
  a more accurate determination of taxa proportions and diversity indexes (diversity
  assessment). Good assemblies and alignments are an important aspect to reach correct
  gene predictions in the whole genome pipeline. In the functional assignment step,
  we gather a biological understanding for regulation and gene pathway reconstruction,
  obtaining finally the microbiome profiling.
pmcid: PMC5145879
papertitle: 'Host-Microbiome Interaction and Cancer: Potential Application in Precision
  Medicine.'
reftext: Alejandra V. Contreras, et al. Front Physiol. 2016;7:606.
pmc_ranked_result_index: '223009'
pathway_score: 0.6575608
filename: fphys-07-00606-g0001.jpg
figtitle: Bioinformatics workflow of microbiome profiling
year: '2016'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5145879__fphys-07-00606-g0001.html
  '@type': Dataset
  description: Bioinformatics workflow of microbiome profiling. The first step is
    the data acquisition that can be derived from any NGS technology (Illumina, IonProton,
    PacBio) and generating of the FASTQ file to proceed with the analysis. In the
    quality control step, the aim is to clean and eliminate possible errors in data,
    for example, to discard low quality score and very short reads, quimeric and adapter
    sequences. In addition, it is important to evaluate the presence of some contaminants
    from other organisms, specific GC content bias or repeated sequences that may
    interfere with the assembly step. The following steps depend on the nature of
    data, whether the aim is to sequence a marker gene, such as the 16S rRNA gene
    or ITS, or to perform shotgun metagenomic sequencing. OTU clustering is a critical
    step and many algorithms and strategies have emerged to accomplish a proper classification
    of sequences for a more accurate determination of taxa proportions and diversity
    indexes (diversity assessment). Good assemblies and alignments are an important
    aspect to reach correct gene predictions in the whole genome pipeline. In the
    functional assignment step, we gather a biological understanding for regulation
    and gene pathway reconstruction, obtaining finally the microbiome profiling.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - LUC7L3
  - NELFE
  - GC
genes:
- word: CROP
  symbol: CROP
  source: hgnc_alias_symbol
  hgnc_symbol: LUC7L3
  entrez: '51747'
- word: RDP
  symbol: RDP
  source: hgnc_alias_symbol
  hgnc_symbol: NELFE
  entrez: '7936'
- word: GC
  symbol: GC
  source: hgnc_symbol
  hgnc_symbol: GC
  entrez: '2638'
chemicals: []
diseases: []
figid_alias: PMC5145879__F1
redirect_from: /figures/PMC5145879__F1
figtype: Figure
---
