---
figid: PMC5813551__PP_PP2017UP01225R2_f1
figlink: /pmc/articles/PMC5813551/figure/fig1/
number: Figure 1
caption: 'Overview of the regulation of group-VII ERF factor stability in Arabidopsis.
  The stability of plant ERF-VII proteins is controlled by intracellular O2 and NO
  levels by means of the Arg-Cys/N-end rule pathway (NERP). The N-terminal Cys, exposed
  upon Met cleavage by MAP (Met aminopeptidase) enzymes, is susceptible to oxidation.
  Arginyl transferase (ATE) enzymes conjugate oxidized Cys (*Cys) to Arg, which in
  turn recruits the Arg-specific N-recognin PRT6 (), an N-end rule pathway-specialized
  E3 ubiquitin ligase that labels the substrate for degradation through the 26S proteasome
  (see Box 1 for additional details of the pathway). Cys oxidation can be promoted
  by specific thiol oxygenases called plant Cys oxidases (PCOs): In the presence of
  oxygen, PCOs convert Cys into Cys-sulfinic acid, which acts as an ATE substrate
  (). Therefore, PCOs target ERF-VII proteins to the proteasome in an oxygen-dependent
  fashion. Besides oxygen, nitric oxide (NO) also promotes ERF-VII turnover via the
  Arg-Cys/NERP, through a still-undetermined Cys-dependent mechanism (; ). Finally,
  although N-terminal Cys reactivity to hydrogen peroxide (H2O2) has not been assessed
  in ERF-VII proteins, *Cys forms generated in relation to H2O2 concentration () could
  in principle play a role in the pathway, by either working as an alternative ATE
  substrate or interfering with Cys-sulfinic acid catalysis. Plasma membrane localization
  has been observed for RAP2.3 () and RAP2.12 (; ). As the latter has been found to
  be associated with peripheral membrane proteins belonging to the ACBP (Acyl-CoA
  binding protein) family, it has been proposed that this interaction is useful to
  maintain an inactive pool of RAP2.12 factor at the plasma membrane (). ERF-VIIs
  have a primary role as master activators of the hypoxic metabolism. ERF-VII transcription
  factors (represented in this figure by the five subfamily members from Arabidopsis,
  AtRAP2.2/2.3/2.12 and AtHRE1/2) exert direct control on the hypoxia-inducible expression
  of plant anaerobic genes by binding an HRPE (hypoxia response promoter element)
  motif present in their promoters (e.g. ADH, PDC, LBD41, HRE1 and HRE2, and HRA1;
  ). The hypoxia-inducible factors HRE1 and HRE2 are further controlled at the posttranscriptional
  level through the Arg-Cys/NERP (). During hypoxic regulation, HRA1 acts as a feedback
  repressor of anaerobic gene expression, by interaction with RAP2.12 (). The SINAT
  pathway, which is an N-end rule pathway-independent proteolysis, is also shown.
  RAP2.12 can be ubiquitinated by the E3 ligases SINAT1/2 (). These proteins modulate
  the autophagy pathway and thereby enhance Arabidopsis tolerance to nutrient starvation
  (). In fact, autophagy responses are also activated during hypoxia and contribute
  to plant submergence tolerance (,  and ). This evidence suggests that an additional
  tier of regulation might connect the ERF-VIIs to submergence responses, through
  the SINAT factors. Solid lines refer to experimentally established reactions or
  relationships, dashed lines to hypothetical relationships drawn from observed regulation,
  and dotted lines depict hypothetical reactions.'
pmcid: PMC5813551
papertitle: 'Group VII Ethylene Response Factors in Arabidopsis: Regulation and Physiological
  Roles.'
reftext: Beatrice Giuntoli, et al. Plant Physiol. 2018 Feb;176(2):1143-1155.
pmc_ranked_result_index: '228093'
pathway_score: 0.948871
filename: PP_PP2017UP01225R2_f1.jpg
figtitle: Regulation of group-VII ERF factor stability in Arabidopsis
year: '2018'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5813551__PP_PP2017UP01225R2_f1.html
  '@type': Dataset
  description: 'Overview of the regulation of group-VII ERF factor stability in Arabidopsis.
    The stability of plant ERF-VII proteins is controlled by intracellular O2 and
    NO levels by means of the Arg-Cys/N-end rule pathway (NERP). The N-terminal Cys,
    exposed upon Met cleavage by MAP (Met aminopeptidase) enzymes, is susceptible
    to oxidation. Arginyl transferase (ATE) enzymes conjugate oxidized Cys (*Cys)
    to Arg, which in turn recruits the Arg-specific N-recognin PRT6 (), an N-end rule
    pathway-specialized E3 ubiquitin ligase that labels the substrate for degradation
    through the 26S proteasome (see Box 1 for additional details of the pathway).
    Cys oxidation can be promoted by specific thiol oxygenases called plant Cys oxidases
    (PCOs): In the presence of oxygen, PCOs convert Cys into Cys-sulfinic acid, which
    acts as an ATE substrate (). Therefore, PCOs target ERF-VII proteins to the proteasome
    in an oxygen-dependent fashion. Besides oxygen, nitric oxide (NO) also promotes
    ERF-VII turnover via the Arg-Cys/NERP, through a still-undetermined Cys-dependent
    mechanism (; ). Finally, although N-terminal Cys reactivity to hydrogen peroxide
    (H2O2) has not been assessed in ERF-VII proteins, *Cys forms generated in relation
    to H2O2 concentration () could in principle play a role in the pathway, by either
    working as an alternative ATE substrate or interfering with Cys-sulfinic acid
    catalysis. Plasma membrane localization has been observed for RAP2.3 () and RAP2.12
    (; ). As the latter has been found to be associated with peripheral membrane proteins
    belonging to the ACBP (Acyl-CoA binding protein) family, it has been proposed
    that this interaction is useful to maintain an inactive pool of RAP2.12 factor
    at the plasma membrane (). ERF-VIIs have a primary role as master activators of
    the hypoxic metabolism. ERF-VII transcription factors (represented in this figure
    by the five subfamily members from Arabidopsis, AtRAP2.2/2.3/2.12 and AtHRE1/2)
    exert direct control on the hypoxia-inducible expression of plant anaerobic genes
    by binding an HRPE (hypoxia response promoter element) motif present in their
    promoters (e.g. ADH, PDC, LBD41, HRE1 and HRE2, and HRA1; ). The hypoxia-inducible
    factors HRE1 and HRE2 are further controlled at the posttranscriptional level
    through the Arg-Cys/NERP (). During hypoxic regulation, HRA1 acts as a feedback
    repressor of anaerobic gene expression, by interaction with RAP2.12 (). The SINAT
    pathway, which is an N-end rule pathway-independent proteolysis, is also shown.
    RAP2.12 can be ubiquitinated by the E3 ligases SINAT1/2 (). These proteins modulate
    the autophagy pathway and thereby enhance Arabidopsis tolerance to nutrient starvation
    (). In fact, autophagy responses are also activated during hypoxia and contribute
    to plant submergence tolerance (,  and ). This evidence suggests that an additional
    tier of regulation might connect the ERF-VIIs to submergence responses, through
    the SINAT factors. Solid lines refer to experimentally established reactions or
    relationships, dashed lines to hypothetical relationships drawn from observed
    regulation, and dotted lines depict hypothetical reactions.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - LRPAP1
  - ADH1A
  - ADH1B
  - ADH4
  - ADH5
  - ADHFE1
  - TRPM6
  - PDC
  - DBI
  - ADH1C
  - ADH7
  - ADH6
  - Arg
  - Cys
  - Met
  - Cys-sulfinic acid
genes:
- word: RAP
  symbol: RAP
  source: hgnc_prev_symbol
  hgnc_symbol: LRPAP1
  entrez: '4043'
- word: ADH
  symbol: ADH
  source: bioentities_symbol
  hgnc_symbol: ADH1A
  entrez: '124'
- word: ADH
  symbol: ADH
  source: bioentities_symbol
  hgnc_symbol: ADH1B
  entrez: '125'
- word: ADH
  symbol: ADH
  source: bioentities_symbol
  hgnc_symbol: ADH4
  entrez: '127'
- word: ADH
  symbol: ADH
  source: bioentities_symbol
  hgnc_symbol: ADH5
  entrez: '128'
- word: ADH
  symbol: ADH
  source: bioentities_symbol
  hgnc_symbol: ADHFE1
  entrez: '137872'
- word: HSH
  symbol: HSH
  source: hgnc_prev_symbol
  hgnc_symbol: TRPM6
  entrez: '140803'
- word: PDC
  symbol: PDC
  source: hgnc_symbol
  hgnc_symbol: PDC
  entrez: '5132'
- word: ACBP
  symbol: ACBP
  source: hgnc_alias_symbol
  hgnc_symbol: DBI
  entrez: '1622'
- word: ADH
  symbol: ADH
  source: bioentities_symbol
  hgnc_symbol: ADH1C
  entrez: '126'
- word: ADH
  symbol: ADH
  source: bioentities_symbol
  hgnc_symbol: ADH7
  entrez: '131'
- word: ADH
  symbol: ADH
  source: bioentities_symbol
  hgnc_symbol: ADH6
  entrez: '130'
chemicals:
- word: Arg
  source: MESH
  identifier: D001120
- word: Cys
  source: MESH
  identifier: D003545
- word: Met
  source: MESH
  identifier: D008715
- word: Cys-sulfinic acid
  source: MESH
  identifier: D013441
diseases: []
---
