---
figid: PMC5299434__srep42333-f8
figlink: /pmc/articles/PMC5299434/figure/f8/
number: Figure 8
caption: 'Pathways are shown by grey circles and the component enzymes are shown as:
  1, ribulose-1,5-bisphosphate carboxylase-oxygenase; 2, phosphoglycerate kinase;
  3, fructose-bisphosphate aldolase; 4, fructose-1,6-bisphosphatase; 5, phosphoribulokinase;
  6, triose-phosphate isomerase. The abbreviations used are: ADP, Adenosine diphosphate;
  APX ascorbate peroxidase; ATP, adenosine triphosphate; ATPSase, ATP synthase; BPGA,
  1,3-bisphosphoglyceric acid; CA, carbonic anhydrase; Chl, chlorophyll; DHAP, dihydroxyacetone
  phosphate; F6P, fructose-6-phosphate; FBP, fructose-1,6-bisphosphate; Fd, ferredoxin;
  FNR, ferredoxin-NADP reductase; GAPDH, glyceraldehyde-3-phosphate dehydrogenase;
  G3P, glyceraldehyde-3-phosphate; hv, light; LCIP63, low CO2 inducible protein 63 kDa;
  LRPa, light-repressed protein a; NADPH, nicotinamide adenine dinucleotide phosphate
  (reduced); OPP, oxidative pentose phosphate pathway; PEP, phosphoenol pyruvate;
  PFK, phosphofructokinase; PGA, 3-phosphoglyceric acid; Pi, inorganic phosphate;
  PK, pyruvate kinase; PSI, photosystem I; PSII, photosystem II; Pyr, pyruvate; Ru5P,
  ribulose-5-phosphate; RuBP, ribulose-1,5-bisphosphate; Sti1, stress-inducible protein
  sti1; THP, transmembrane hypothetical protein. Changes in protein level in response
  to CO2-limitation are shown as increases (green arrows) or decreases (red arrows).
  Major interactions between the pathways and organelles are shown by grey lines.
  Also shown, as coloured boxes adjacent to proteins or pathways, are changes reported
  in the literature using proteomic or transcriptomic approaches in response to starvation
  by inorganic carbon (lilac) for T. pseudonana; nitrogen (green) for T. pseudonana,
  for P. tricornutum; phosphorus (blue) for T. pseudonana, for P. tricornutum; silica
  for T. pseudonana (grey); iron (russet) for T. pseudonana29, for P. tricornutum;
  fluctuating light (yellow-brown) for T. pseudonana; temperature (turquoise) for
  T. pseudonana. Responses for T. pseudonana (above the line) and for P. tricornutum
  (below the line) are denoted as an increase (+), decrease (−), both decrease and
  increase (±), no change (o). If the response is not commented on in the paper or
  if it is not obvious from , no information is presented.'
pmcid: PMC5299434
papertitle: 'Responses of the marine diatom Thalassiosira pseudonana to changes in
  CO2 concentration: a proteomic approach.'
reftext: Romain Clement, et al. Sci Rep. 2017;7:42333.
pmc_ranked_result_index: '210188'
pathway_score: 0.9662563
filename: srep42333-f8.jpg
figtitle: 'Responses of the marine diatom Thalassiosira pseudonana to changes in CO2
  concentration: a proteomic approach'
year: '2017'
organisms: Homo sapiens
ndex: c8d8f6ed-decf-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5299434__srep42333-f8.html
  '@type': Dataset
  description: 'Pathways are shown by grey circles and the component enzymes are shown
    as: 1, ribulose-1,5-bisphosphate carboxylase-oxygenase; 2, phosphoglycerate kinase;
    3, fructose-bisphosphate aldolase; 4, fructose-1,6-bisphosphatase; 5, phosphoribulokinase;
    6, triose-phosphate isomerase. The abbreviations used are: ADP, Adenosine diphosphate;
    APX ascorbate peroxidase; ATP, adenosine triphosphate; ATPSase, ATP synthase;
    BPGA, 1,3-bisphosphoglyceric acid; CA, carbonic anhydrase; Chl, chlorophyll; DHAP,
    dihydroxyacetone phosphate; F6P, fructose-6-phosphate; FBP, fructose-1,6-bisphosphate;
    Fd, ferredoxin; FNR, ferredoxin-NADP reductase; GAPDH, glyceraldehyde-3-phosphate
    dehydrogenase; G3P, glyceraldehyde-3-phosphate; hv, light; LCIP63, low CO2 inducible
    protein 63 kDa; LRPa, light-repressed protein a; NADPH, nicotinamide adenine dinucleotide
    phosphate (reduced); OPP, oxidative pentose phosphate pathway; PEP, phosphoenol
    pyruvate; PFK, phosphofructokinase; PGA, 3-phosphoglyceric acid; Pi, inorganic
    phosphate; PK, pyruvate kinase; PSI, photosystem I; PSII, photosystem II; Pyr,
    pyruvate; Ru5P, ribulose-5-phosphate; RuBP, ribulose-1,5-bisphosphate; Sti1, stress-inducible
    protein sti1; THP, transmembrane hypothetical protein. Changes in protein level
    in response to CO2-limitation are shown as increases (green arrows) or decreases
    (red arrows). Major interactions between the pathways and organelles are shown
    by grey lines. Also shown, as coloured boxes adjacent to proteins or pathways,
    are changes reported in the literature using proteomic or transcriptomic approaches
    in response to starvation by inorganic carbon (lilac) for T. pseudonana; nitrogen
    (green) for T. pseudonana, for P. tricornutum; phosphorus (blue) for T. pseudonana,
    for P. tricornutum; silica for T. pseudonana (grey); iron (russet) for T. pseudonana29,
    for P. tricornutum; fluctuating light (yellow-brown) for T. pseudonana; temperature
    (turquoise) for T. pseudonana. Responses for T. pseudonana (above the line) and
    for P. tricornutum (below the line) are denoted as an increase (+), decrease (−),
    both decrease and increase (±), no change (o). If the response is not commented
    on in the paper or if it is not obvious from , no information is presented.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PAK1
  - GAPDH
  - PAK5
  - PAK3
  - APEX1
  - PAEP
  - PAK4
  - FBP1
  - PAK6
  - CYP1A2
  - SI
  - PAK2
  - STIP1
  - CHRDL1
  - A4GALT
  - Glucose
genes:
- word: PAK
  symbol: PAK
  source: bioentities_symbol
  hgnc_symbol: PAK1
  entrez: '5058'
- word: GAPDH
  symbol: GAPDH
  source: hgnc_symbol
  hgnc_symbol: GAPDH
  entrez: '2597'
- word: PAK
  symbol: PAK
  source: bioentities_symbol
  hgnc_symbol: PAK5
  entrez: '57144'
- word: PAK
  symbol: PAK
  source: bioentities_symbol
  hgnc_symbol: PAK3
  entrez: '5063'
- word: APX
  symbol: APX
  source: hgnc_alias_symbol
  hgnc_symbol: APEX1
  entrez: '328'
- word: PEP
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
- word: PAK
  symbol: PAK
  source: bioentities_symbol
  hgnc_symbol: PAK4
  entrez: '10298'
- word: FBP
  symbol: FBP
  source: hgnc_prev_symbol
  hgnc_symbol: FBP1
  entrez: '2203'
- word: PAK
  symbol: PAK
  source: bioentities_symbol
  hgnc_symbol: PAK6
  entrez: '56924'
- word: '|CP12'
  symbol: CP12
  source: hgnc_alias_symbol
  hgnc_symbol: CYP1A2
  entrez: '1544'
- word: Si
  symbol: SI
  source: hgnc_symbol
  hgnc_symbol: SI
  entrez: '6476'
- word: PAK
  symbol: PAK
  source: bioentities_symbol
  hgnc_symbol: PAK2
  entrez: '5062'
- word: sti1
  symbol: STI1
  source: hgnc_alias_symbol
  hgnc_symbol: STIP1
  entrez: '10963'
- word: Chl
  symbol: CHL
  source: hgnc_alias_symbol
  hgnc_symbol: CHRDL1
  entrez: '91851'
- word: PK
  symbol: P(k)
  source: hgnc_alias_symbol
  hgnc_symbol: A4GALT
  entrez: '53947'
chemicals:
- word: Glucose
  source: MESH
  identifier: D005947
diseases: []
---
