---
figid: PMC5437397__12864_2017_3769_Fig4_HTML
figlink: /pmc/articles/PMC5437397/figure/Fig4/
number: Fig. 4
caption: 'Immune deficiency (IMD) pathway members in malacostracans. a PGRPs recognises
  Gram-negative bacteria and activate the IMD pathway through the RHIM motifs. Although
  the IMD pathway is typically activated by PGRPs in Drosophila melanogaster, PGRPs
  are not necessary for IMD signalling and it was posited that an unknown protein
  is present upstream of the IMD signalling cascade. Like DIF from the Toll pathway,
  in the IMD pathway, differential activation of another NF-κB transcription factor,
  Relish, occurs. Relish is phosphorylated through the activation of IκB kinase (IKK)
  complexes and transforming growth factor-β-activated kinase 1 (TAK1). The caspase-8
  homolog death-related ced-3/Nedd2-like protein (DREDD) and FAS-associated death
  domain (FADD) proteins are required for IKK and TAK1 activation and Relish is cleaved
  through DREDD. Caspar, a homologue of mammalian Fas-associating factor 1 that is
  essential for antifungal immunity, negatively regulates the IMD-mediated immune
  response by preventing nuclear translocation of Relish. Caspar also suppresses the
  IMD pathway through targeting Dredd-dependent cleavage of Relish. Phylogenetic trees
  of b IMD, c Caspar, d Relish and e DREDD are constructed using the maximum-likelihood
  method from an amino acid multiple sequence alignment. Taxa labels are depicted
  as their respective colour codes. Bootstrap support values (n = 1000) for all trees
  can be found in Additional file : Figure S14. Scale bar represents substitution
  per site'
pmcid: PMC5437397
papertitle: Comparative genomic analysis of innate immunity reveals novel and conserved
  components in crustacean food crop species.
reftext: Alvina G. Lai, et al. BMC Genomics. 2017;18:389.
pmc_ranked_result_index: '12791'
pathway_score: 0.7548196
filename: 12864_2017_3769_Fig4_HTML.jpg
figtitle: Immune deficiency (IMD) pathway members in malacostracans
year: '2017'
organisms:
- XXX
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5437397__12864_2017_3769_Fig4_HTML.html
  '@type': Dataset
  description: 'Immune deficiency (IMD) pathway members in malacostracans. a PGRPs
    recognises Gram-negative bacteria and activate the IMD pathway through the RHIM
    motifs. Although the IMD pathway is typically activated by PGRPs in Drosophila
    melanogaster, PGRPs are not necessary for IMD signalling and it was posited that
    an unknown protein is present upstream of the IMD signalling cascade. Like DIF
    from the Toll pathway, in the IMD pathway, differential activation of another
    NF-κB transcription factor, Relish, occurs. Relish is phosphorylated through the
    activation of IκB kinase (IKK) complexes and transforming growth factor-β-activated
    kinase 1 (TAK1). The caspase-8 homolog death-related ced-3/Nedd2-like protein
    (DREDD) and FAS-associated death domain (FADD) proteins are required for IKK and
    TAK1 activation and Relish is cleaved through DREDD. Caspar, a homologue of mammalian
    Fas-associating factor 1 that is essential for antifungal immunity, negatively
    regulates the IMD-mediated immune response by preventing nuclear translocation
    of Relish. Caspar also suppresses the IMD pathway through targeting Dredd-dependent
    cleavage of Relish. Phylogenetic trees of b IMD, c Caspar, d Relish and e DREDD
    are constructed using the maximum-likelihood method from an amino acid multiple
    sequence alignment. Taxa labels are depicted as their respective colour codes.
    Bootstrap support values (n = 1000) for all trees can be found in Additional file
    : Figure S14. Scale bar represents substitution per site'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PDE4D
  - PDE4C
  - PDE3A
  - PDE6A
  - PDE4B
  - PDE7A
  - PDE1A
  - PDE1C
  - PDE6C
  - PDE2A
  - PDE3B
  - PDE4A
  - PGLYRP1
  - CHUK
  - TECR
  - PDE8B
  - NR2C2
  - PDE9A
  - IKBKB
  - ATG9B
  - PDE7B
  - HHEX
  - ZBTB32
  - IKBKG
  - ARHGAP26
  - PDE8A
  - MAPK10
  - AATF
  - PDE1B
  - PDE6B
  - PDE10A
  - MAPK8
  - MAPK9
  - PDE11A
genes:
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE4D
  entrez: '5144'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE4C
  entrez: '5143'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE3A
  entrez: '5139'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE6A
  entrez: '5145'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE4B
  entrez: '5142'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE7A
  entrez: '5150'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE1A
  entrez: '5136'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE1C
  entrez: '5137'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE6C
  entrez: '5146'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE2A
  entrez: '5138'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE3B
  entrez: '5140'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE4A
  entrez: '5141'
- word: PGRP
  symbol: PGRP
  source: hgnc_alias_symbol
  hgnc_symbol: PGLYRP1
  entrez: '8993'
- word: IKK
  symbol: IKK
  source: bioentities_symbol
  hgnc_symbol: CHUK
  entrez: '1147'
- word: SC2
  symbol: SC2
  source: hgnc_prev_symbol
  hgnc_symbol: TECR
  entrez: '9524'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE8B
  entrez: '8622'
- word: TAK1
  symbol: TAK1
  source: hgnc_alias_symbol
  hgnc_symbol: NR2C2
  entrez: '7182'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE9A
  entrez: '5152'
- word: IKK
  symbol: IKK
  source: bioentities_symbol
  hgnc_symbol: IKBKB
  entrez: '3551'
- word: sone
  symbol: SONE
  source: hgnc_alias_symbol
  hgnc_symbol: ATG9B
  entrez: '285973'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE7B
  entrez: '27115'
- word: hex
  symbol: HEX
  source: hgnc_alias_symbol
  hgnc_symbol: HHEX
  entrez: '3087'
- word: rog
  symbol: Rog
  source: hgnc_alias_symbol
  hgnc_symbol: ZBTB32
  entrez: '27033'
- word: IKK
  symbol: IKK
  source: bioentities_symbol
  hgnc_symbol: IKBKG
  entrez: '8517'
- word: graf
  symbol: GRAF
  source: hgnc_alias_symbol
  hgnc_symbol: ARHGAP26
  entrez: '23092'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE8A
  entrez: '5151'
- word: JNK
  symbol: JNK
  source: bioentities_symbol
  hgnc_symbol: MAPK10
  entrez: '5602'
- word: DED
  symbol: DED
  source: hgnc_alias_symbol
  hgnc_symbol: AATF
  entrez: '26574'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE1B
  entrez: '5153'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE6B
  entrez: '5158'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE10A
  entrez: '10846'
- word: JNK
  symbol: JNK
  source: bioentities_symbol
  hgnc_symbol: MAPK8
  entrez: '5599'
- word: JNK
  symbol: JNK
  source: bioentities_symbol
  hgnc_symbol: MAPK9
  entrez: '5601'
- word: pdes
  symbol: PDE
  source: bioentities_symbol
  hgnc_symbol: PDE11A
  entrez: '50940'
chemicals: []
diseases: []
---
