---
figid: PMC5178258__fmicb-07-02082-g0003
figlink: /pmc/articles/PMC5178258/figure/F3/
number: Figure 3
caption: Graphical representation of potential metabolic activities of acidophilic
  Sideroxydans strains. The figure represents the pool of metabolic pathways that
  were reconstructed from the metagenomic sequence data of contigs assigned to strains
  of the genus Sideroxydans. It therefore does not express the co-occurrence of various
  pathways within the same strain. Biochemical reactions do not reflect real stoichiometry
  with the exception of cases were it was judged to be of relevance. Question marks
  indicate that it is unknown which sulfur compound is oxidized via the Sox enzyme
  system. Based on its pKa (3.86) lactate is depicted in its acid form outside the
  cell, though the narrow difference to the external pH (3.5) means that only a small
  fraction of molecules will be protonated. Please also note in this context that,
  no lactate was added to the culture medium. Dotted arrows indicate that no details
  are provided on the multiple enzymes involved in the pathway while dashed lines
  indicate the potential path of a volatile compound (i.e., H2). For reasons of clarity
  no details are provided on the tricarboxylic acid (TCA) cycle. Arrows deviating
  from the the TCA cycle indicate its relevance as a central carbon metabolic pathway,
  though this line of metabolic function was not further investigated in this study.
  Carbon dioxide appears to be reduced to 3-phosphoglycerate via the Calvin-Benson-Bassham
  cycle indicated by its key enzyme ribulose-1,5-bisphosphate carboxylase/oxygenase
  (RuBisCO). Names for the nitrate transporter and nitrate and nitrite reductases
  are not indicated due to contradicting nomenclature in the databases. FHL, formate
  hydrogenlyase; AC III, alternative complex III; ETF, electron transfer flavoprotein
  oxidoreductase; SDH, succinate dehydrogenase; SQR, sulfide quinone oxidoreductase;
  CA, carbonic anhydrase; NADH dh, NADH dehydrogenase; Phn, phosphonate uptake system.
  Enzymes involved in metabolic pathways are abbreviated by their acronyms. For details
  see Supplementary Table .
pmcid: PMC5178258
papertitle: Reconstruction of the Metabolic Potential of Acidophilic Sideroxydans
  Strains from the Metagenome of an Microaerophilic Enrichment Culture of Acidophilic
  Iron-Oxidizing Bacteria from a Pilot Plant for the Treatment of Acid Mine Drainage
  Reveals Metabolic Versatility and Adaptation to Life at Low pH.
reftext: Martin MÃ¼hling, et al. Front Microbiol. 2016;7:2082.
pmc_ranked_result_index: '63680'
pathway_score: 0.9414911
filename: fmicb-07-02082-g0003.jpg
figtitle: Graphical representation of potential metabolic activities of acidophilic
  Sideroxydans strains
year: '2016'
organisms: Homo sapiens
ndex: 159280fd-df03-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5178258__fmicb-07-02082-g0003.html
  '@type': Dataset
  description: Graphical representation of potential metabolic activities of acidophilic
    Sideroxydans strains. The figure represents the pool of metabolic pathways that
    were reconstructed from the metagenomic sequence data of contigs assigned to strains
    of the genus Sideroxydans. It therefore does not express the co-occurrence of
    various pathways within the same strain. Biochemical reactions do not reflect
    real stoichiometry with the exception of cases were it was judged to be of relevance.
    Question marks indicate that it is unknown which sulfur compound is oxidized via
    the Sox enzyme system. Based on its pKa (3.86) lactate is depicted in its acid
    form outside the cell, though the narrow difference to the external pH (3.5) means
    that only a small fraction of molecules will be protonated. Please also note in
    this context that, no lactate was added to the culture medium. Dotted arrows indicate
    that no details are provided on the multiple enzymes involved in the pathway while
    dashed lines indicate the potential path of a volatile compound (i.e., H2). For
    reasons of clarity no details are provided on the tricarboxylic acid (TCA) cycle.
    Arrows deviating from the the TCA cycle indicate its relevance as a central carbon
    metabolic pathway, though this line of metabolic function was not further investigated
    in this study. Carbon dioxide appears to be reduced to 3-phosphoglycerate via
    the Calvin-Benson-Bassham cycle indicated by its key enzyme ribulose-1,5-bisphosphate
    carboxylase/oxygenase (RuBisCO). Names for the nitrate transporter and nitrate
    and nitrite reductases are not indicated due to contradicting nomenclature in
    the databases. FHL, formate hydrogenlyase; AC III, alternative complex III; ETF,
    electron transfer flavoprotein oxidoreductase; SDH, succinate dehydrogenase; SQR,
    sulfide quinone oxidoreductase; CA, carbonic anhydrase; NADH dh, NADH dehydrogenase;
    Phn, phosphonate uptake system. Enzymes involved in metabolic pathways are abbreviated
    by their acronyms. For details see Supplementary Table .
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ADCY2
  - ZBTB25
  - PPP4C
  - SQOR
  - TEAD2
  - ADCY1
  - ADCY9
  - ADCY3
  - WDTC1
  - SH2B2
  - ADCY10
  - TRIM31
  - ADCY8
  - ADCY6
  - ADCY4
  - KLK4
  - ADCY7
  - ADCY5
  - cysteine
  - fumarate
  - succinate
genes:
- word: '[AC'
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY2
  entrez: '108'
- word: '[ACII'
  symbol: AC2
  source: hgnc_alias_symbol
  hgnc_symbol: ADCY2
  entrez: '108'
- word: Kup
  symbol: KUP
  source: hgnc_alias_symbol
  hgnc_symbol: ZBTB25
  entrez: '7597'
- word: Ppx
  symbol: PPX
  source: hgnc_alias_symbol
  hgnc_symbol: PPP4C
  entrez: '5531'
- word: SQR
  symbol: SQR
  source: hgnc_alias_symbol
  hgnc_symbol: SQOR
  entrez: '58472'
- word: ETF
  symbol: ETF
  source: hgnc_alias_symbol
  hgnc_symbol: TEAD2
  entrez: '8463'
- word: '[AC'
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY1
  entrez: '107'
- word: '[AC'
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY9
  entrez: '115'
- word: '[AC'
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY3
  entrez: '109'
- word: ADP
  symbol: ADP
  source: hgnc_alias_symbol
  hgnc_symbol: WDTC1
  entrez: '23038'
- word: APS
  symbol: APS
  source: hgnc_alias_symbol
  hgnc_symbol: SH2B2
  entrez: '10603'
- word: '[AC'
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY10
  entrez: '55811'
- word: Rnf
  symbol: RNF
  source: hgnc_alias_symbol
  hgnc_symbol: TRIM31
  entrez: '11074'
- word: '[AC'
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY8
  entrez: '114'
- word: '[AC'
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY6
  entrez: '112'
- word: '[AC'
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY4
  entrez: '196883'
- word: PstS
  symbol: PSTS
  source: hgnc_alias_symbol
  hgnc_symbol: KLK4
  entrez: '9622'
- word: '[AC'
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY7
  entrez: '113'
- word: '[AC'
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY5
  entrez: '111'
chemicals:
- word: cysteine
  source: MESH
  identifier: D003545
- word: fumarate
  source: MESH
  identifier: D005650
- word: succinate
  source: MESH
  identifier: D013386
diseases: []
---
