---
figid: PMC5395668__mbo0021732700004
figlink: /pmc/articles/PMC5395668/figure/fig4/
number: F4
caption: A proposed metabolic schema for cells of the SAR202 clade. Oxidative enzymes
  encoded in the SAR202 genomes provide semispecific initial oxidation of heterogenous
  recalcitrant DOM on the cell surface or upon initial transport of these compounds
  into the cell. Carboxylate degradation products are ligated into a pool of CoA-linked
  substrates via acyl-CoA transferases. CoA-linked substrates move to the beta- and
  possibly alpha-oxidation pathways. Partial degradation products of fatty acids and
  substituted alkanes released from the partial oxidation of recalcitrant DOM may
  provide intermediates for absorption via the 3-hydroxypropionate cycle (3HPC). Incompletely
  degraded substrates may be substituted for more active substrates using acyl-CoA
  transferase activity, with released products being exported from the cell via a
  number of MFS1 transport proteins with documented promiscuous transport activities,
  thus contributing to a more refractory environmental pool of marine dissolved organic
  matter. 3HPC pathway gaps for which no suitable gene candidates were found in the
  draft genomes are marked in red. While it is uncertain as to whether the enzymes
  of the 3HPC shown here participate in carbon fixation, even an incomplete 3HPC could
  provide a means of assimilating a number of small carbon intermediates into central
  metabolism. Several mechanisms for cleavage of sulfite from sulfur and organosulfur
  compounds are encoded by the genome and may provide a source of sulfur for growth
  or a substrate for sulfite oxidase.
pmcid: PMC5395668
papertitle: SAR202 Genomes from the Dark Ocean Predict Pathways for the Oxidation
  of Recalcitrant Dissolved Organic Matter.
reftext: Zachary Landry, et al. mBio. 2017 Mar-Apr;8(2):e00413-17.
pmc_ranked_result_index: '76673'
pathway_score: 0.9497317
filename: mbo0021732700004.jpg
figtitle: A proposed metabolic schema for cells of the SAR202 clade
year: '2017'
organisms:
- uncultured actinomycete
- Chloroflexus aurantiacus
- Chloroflexi bacterium SCGC AB-629-P13
- Streptococcus agalactiae
- Thermogemmatispora carboxidivorans
- Rhodococcus erythropolis
- Dehalococcoides mccartyi CBDB1
- Clostridium scindens
- Catenulispora acidiphila
- Ktedonobacter racemifer
- metagenomes
- marine metagenome
- Pirellula staleyi DSM 6068
- Dehalococcoides mccartyi
- SAR202 cluster
- uncultured bacterium EC5
- uncultured bacterium
- Chloroflexi bacterium SCGC AAA001-F05
- Chloroflexi bacterium SCGC AAA007-M09
- Chloroflexi bacterium SCGC AAA240-N13
- Chloroflexi bacterium SCGC AAA240-O15
- Homo sapiens
- Cerdocyon thous
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5395668__mbo0021732700004.html
  '@type': Dataset
  description: A proposed metabolic schema for cells of the SAR202 clade. Oxidative
    enzymes encoded in the SAR202 genomes provide semispecific initial oxidation of
    heterogenous recalcitrant DOM on the cell surface or upon initial transport of
    these compounds into the cell. Carboxylate degradation products are ligated into
    a pool of CoA-linked substrates via acyl-CoA transferases. CoA-linked substrates
    move to the beta- and possibly alpha-oxidation pathways. Partial degradation products
    of fatty acids and substituted alkanes released from the partial oxidation of
    recalcitrant DOM may provide intermediates for absorption via the 3-hydroxypropionate
    cycle (3HPC). Incompletely degraded substrates may be substituted for more active
    substrates using acyl-CoA transferase activity, with released products being exported
    from the cell via a number of MFS1 transport proteins with documented promiscuous
    transport activities, thus contributing to a more refractory environmental pool
    of marine dissolved organic matter. 3HPC pathway gaps for which no suitable gene
    candidates were found in the draft genomes are marked in red. While it is uncertain
    as to whether the enzymes of the 3HPC shown here participate in carbon fixation,
    even an incomplete 3HPC could provide a means of assimilating a number of small
    carbon intermediates into central metabolism. Several mechanisms for cleavage
    of sulfite from sulfur and organosulfur compounds are encoded by the genome and
    may provide a source of sulfur for growth or a substrate for sulfite oxidase.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SOX10
  - TRRAP
  - FBN1
  - Propionyl-CoA
  - Succinyl-CoA
  - Dimethylsulfoniopropionate
  - Fatty Acids
  - Formate
  - Isocitrate
  - Malonyl-CoA
  - Oxaloacetate
  - Pyruvate
  - Succinate
  - Thiosulfate
  - Pyruvate Lactate
genes:
- word: DOM
  symbol: DOM
  source: hgnc_alias_symbol
  hgnc_symbol: SOX10
  entrez: '6663'
- word: TRAP
  symbol: TR-AP
  source: hgnc_alias_symbol
  hgnc_symbol: TRRAP
  entrez: '8295'
- word: MFS1
  symbol: MFS1
  source: hgnc_prev_symbol
  hgnc_symbol: FBN1
  entrez: '2200'
chemicals:
- word: Propionyl-CoA
  source: MESH
  identifier: C009061
- word: Succinyl-CoA
  source: MESH
  identifier: C012046
- word: Dimethylsulfoniopropionate
  source: MESH
  identifier: C068078
- word: Fatty Acids
  source: MESH
  identifier: D005227
- word: Formate
  source: MESH
  identifier: D005561
- word: Isocitrate
  source: MESH
  identifier: D007523
- word: Malonyl-CoA
  source: MESH
  identifier: D008316
- word: Oxaloacetate
  source: MESH
  identifier: D010071
- word: Pyruvate
  source: MESH
  identifier: D011773
- word: Succinate
  source: MESH
  identifier: D013386
- word: Thiosulfate
  source: MESH
  identifier: D013885
- word: Pyruvate Lactate
  source: MESH
  identifier: D019344
diseases: []
figid_alias: PMC5395668__F4
redirect_from: /figures/PMC5395668__F4
figtype: Figure
---
