---
figid: PMC5222806__fmicb-07-02167-g0001
figlink: /pmc/articles/PMC5222806/figure/F1/
number: Figure 1
caption: 'Metabolic pathways for methane oxidation (A) and FA synthesis (B) in M.
  buryatense 5G(B1) based on a gene inventory study, transcriptomic data analysis,
  and genome-scale modeling (De la Torre et al., ). Numbers in blue represent that
  particular reaction''s fluxes according to the developed genome-scale model (De
  la Torre et al., ); numbers in black after gene names denote abundances of respective
  transcript normalized to an abundance of methanol dehydrogenase. Gene names coding
  enzymes of the methane assimilation pathways are extracted from BioCyc Database
  Collection (http://biocyc.org/). Enzyme EC numbers and corresponding gene IDs are
  listed in Table . Methane utilization starts from oxidation to methanol by methane
  monooxygenase enzyme in periplasm; methanol is then metabolized to formaldehyde
  by a periplasmic pyrroloquinoline quinone-linked methanol dehydrogenase. Formaldehyde
  is either oxidized to CO2 or assimilated in cytoplasm through ribulose monophosphate
  (RuMP) and EMP/EDD (Embden-Meyerhof-Parnas/Entner–Doudoroff) pathways (A), fraction
  of which are then converted through pyruvate to acetyl-CoA, a precursor for FA synthesis
  (B). Designations: CH4, methane; CH20, formaldehyde; CHO2, formate; H4MTP, tetrahydromethanopterin
  pathway; H4F, methylene tetrahydrofolate pathway; CO2, carbon dioxide; f6p, fructose
  6-phosphate; acCoA, acetyl-CoA; NADH, nicotinamide adenine dinucleotide reduced;
  NADPH, dihydronicotinamide adenine dinucleotide phosphate reduced; ATP, adenosine-triphosphate.'
pmcid: PMC5222806
papertitle: Fatty Acid Biosynthesis Pathways in Methylomicrobium buryatense 5G(B1).
reftext: Aleksandr Demidenko, et al. Front Microbiol. 2016;7:2167.
pmc_ranked_result_index: '53580'
pathway_score: 0.6230619
filename: fmicb-07-02167-g0001.jpg
figtitle: Metabolic pathways for methane oxidation (A) and FA synthesis (B) in M
year: '2016'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5222806__fmicb-07-02167-g0001.html
  '@type': Dataset
  description: 'Metabolic pathways for methane oxidation (A) and FA synthesis (B)
    in M. buryatense 5G(B1) based on a gene inventory study, transcriptomic data analysis,
    and genome-scale modeling (De la Torre et al., ). Numbers in blue represent that
    particular reaction''s fluxes according to the developed genome-scale model (De
    la Torre et al., ); numbers in black after gene names denote abundances of respective
    transcript normalized to an abundance of methanol dehydrogenase. Gene names coding
    enzymes of the methane assimilation pathways are extracted from BioCyc Database
    Collection (http://biocyc.org/). Enzyme EC numbers and corresponding gene IDs
    are listed in Table . Methane utilization starts from oxidation to methanol by
    methane monooxygenase enzyme in periplasm; methanol is then metabolized to formaldehyde
    by a periplasmic pyrroloquinoline quinone-linked methanol dehydrogenase. Formaldehyde
    is either oxidized to CO2 or assimilated in cytoplasm through ribulose monophosphate
    (RuMP) and EMP/EDD (Embden-Meyerhof-Parnas/Entner–Doudoroff) pathways (A), fraction
    of which are then converted through pyruvate to acetyl-CoA, a precursor for FA
    synthesis (B). Designations: CH4, methane; CH20, formaldehyde; CHO2, formate;
    H4MTP, tetrahydromethanopterin pathway; H4F, methylene tetrahydrofolate pathway;
    CO2, carbon dioxide; f6p, fructose 6-phosphate; acCoA, acetyl-CoA; NADH, nicotinamide
    adenine dinucleotide reduced; NADPH, dihydronicotinamide adenine dinucleotide
    phosphate reduced; ATP, adenosine-triphosphate.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PDHA2
  - TNK2
  - ACACB
  - DLAT
  - EBP
  - ACACA
  - PRKG1
  - EDA
  - DLD
  - PDHX
  - PDHB
  - PHKA2
  - PMCH
  - PDHA1
  - FANCE
  - GAPDH
  - HPS1
  - FBXO3
  - ENO2
  - UBR5
  - MAEA
  - ENO3
  - ENO1
  - SDHD
  - VCAN
  - MCAT
  - ENO4
  - PTCRA
  - acetate
  - malonyl-CoA
  - palmitate
  - 3-phosphate
  - pyruvate
  - serine
genes:
- word: 'pdh:'
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHA2
  entrez: '5161'
- word: 'ack:'
  symbol: ACK
  source: hgnc_alias_symbol
  hgnc_symbol: TNK2
  entrez: '10188'
- word: 'acc:'
  symbol: ACC
  source: bioentities_symbol
  hgnc_symbol: ACACB
  entrez: '32'
- word: 'pdh:'
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: DLAT
  entrez: '1737'
- word: CHO2
  symbol: CHO2
  source: hgnc_alias_symbol
  hgnc_symbol: EBP
  entrez: '10682'
- word: 'acc:'
  symbol: ACC
  source: bioentities_symbol
  hgnc_symbol: ACACA
  entrez: '31'
- word: 'pgk:'
  symbol: PGK
  source: hgnc_alias_symbol
  hgnc_symbol: PRKG1
  entrez: '5592'
- word: 'eda:'
  symbol: EDA
  source: hgnc_symbol
  hgnc_symbol: EDA
  entrez: '1896'
- word: 'pdh:'
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: DLD
  entrez: '1738'
- word: 'pdh:'
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHX
  entrez: '8050'
- word: 'pdh:'
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHB
  entrez: '5162'
- word: 'pyk:'
  symbol: PYK
  source: hgnc_prev_symbol
  hgnc_symbol: PHKA2
  entrez: '5256'
- word: 'mch:'
  symbol: MCH
  source: hgnc_alias_symbol
  hgnc_symbol: PMCH
  entrez: '5367'
- word: 'pdh:'
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHA1
  entrez: '5160'
- word: 'fae:'
  symbol: FAE
  source: hgnc_alias_symbol
  hgnc_symbol: FANCE
  entrez: '2178'
- word: 'gapdh:'
  symbol: GAPDH
  source: hgnc_symbol
  hgnc_symbol: GAPDH
  entrez: '2597'
- word: 'hps:'
  symbol: HPS
  source: hgnc_prev_symbol
  hgnc_symbol: HPS1
  entrez: '3257'
- word: 'fba:'
  symbol: FBA
  source: hgnc_alias_symbol
  hgnc_symbol: FBXO3
  entrez: '26273'
- word: 'eno:'
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO2
  entrez: '2026'
- word: 'edd:'
  symbol: EDD
  source: hgnc_alias_symbol
  hgnc_symbol: UBR5
  entrez: '51366'
- word: 'EMP:'
  symbol: EMP
  source: hgnc_alias_symbol
  hgnc_symbol: MAEA
  entrez: '10296'
- word: 'eno:'
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO3
  entrez: '2027'
- word: 'eno:'
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO1
  entrez: '2023'
- word: 'pgl:'
  symbol: PGL
  source: hgnc_prev_symbol
  hgnc_symbol: SDHD
  entrez: '6392'
- word: 'pgm:'
  symbol: PG-M
  source: hgnc_alias_symbol
  hgnc_symbol: VCAN
  entrez: '1462'
- word: 'fabD:'
  symbol: fabD
  source: hgnc_alias_symbol
  hgnc_symbol: MCAT
  entrez: '27349'
- word: 'eno:'
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO4
  entrez: '387712'
- word: 'pta:'
  symbol: PT-ALPHA
  source: hgnc_alias_symbol
  hgnc_symbol: PTCRA
  entrez: '171558'
chemicals:
- word: acetate
  source: MESH
  identifier: D000085
- word: malonyl-CoA
  source: MESH
  identifier: D008316
- word: palmitate
  source: MESH
  identifier: D010168
- word: 3-phosphate
  source: MESH
  identifier: D010710
- word: pyruvate
  source: MESH
  identifier: D011773
- word: serine
  source: MESH
  identifier: D012694
diseases: []
---
