---
figid: PMC5552966__MBO3-6-na-g009
figlink: /pmc/articles/PMC5552966/figure/mbo3485-fig-0009/
number: F9
caption: A general diagram of Cu‐responsive paths in the cells of filamentous fungi
  at the proteome level. This diagram depicts the paths and components that respond
  to Cu stress and indicates how the fungi change their proteomes to address changes
  in Cu concentration. The details have been described in the Discussion. The broken
  lines with arrows suggest the Cu stress signaling paths and involved components.
  ADDH, ATP‐dependent DNA helicase II; ADRH, ATP‐dependent RNA helicase; CP, catalase‐peroxidase;
  CSP, calcium signaling pathway; DPs, differentially expressed proteins; ECS, epithelial
  cell signaling in H. pylori infection; GnRH, GnRH signaling pathway; HD, Hedgehog
  signaling pathway; Hsp, heat shock protein; 14‐3‐3, 14‐3‐3 proteins; ISP, insulin
  signaling pathway; MAPK, MAPK signaling pathway; NOD, NOD‐like receptor signaling
  pathway; NSP, neurotrophin signaling pathway; OP, oxidative phosphorylation; RIG,
  RIG‐I‐like receptor signaling pathway; ROS, reactive oxygen species; RuvBH, ATP‐dependent
  RuvB helicase; SOD, superoxide dismutase; SS, spermidine synthase; STPP, serine/threonine‐protein
  phosphatase; TCA, citrate cycle
pmcid: PMC5552966
papertitle: 'A proteomic‐based investigation of potential copper‐responsive biomarkers:
  Proteins, conceptual networks, and metabolic pathways featuring Penicillium janthinellum
  from a heavy metal‐polluted ecological niche.'
reftext: Xin Feng, et al. Microbiologyopen. 2017 Aug;6(4):e00485.
pmc_ranked_result_index: '132'
pathway_score: 0.8855227
filename: MBO3-6-na-g009.jpg
figtitle: General diagram of Cu‐responsive paths in the cells of filamentous fungi
  at the proteome level
year: '2017'
organisms:
- Mus musculus
- Mus spretus
- Cupriavidus metallidurans
- Elasmopus rapax
- Tigriopus japonicus
- Aspergillus nidulans
- Helicobacter pylori
- Syringa vulgaris
- Thalassiosira pseudonana
- Arabidopsis thaliana
- Meyerozyma carpophila
- Botrytis cinerea
- Solanum tuberosum
- Schizosaccharomyces pombe
- Saccharomyces cerevisiae
- Yarrowia lipolytica
- Penicillium janthinellum
- Rhodotorula mucilaginosa
- Homo sapiens
ndex: 811bdc82-df17-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5552966__MBO3-6-na-g009.html
  '@type': Dataset
  description: A general diagram of Cu‐responsive paths in the cells of filamentous
    fungi at the proteome level. This diagram depicts the paths and components that
    respond to Cu stress and indicates how the fungi change their proteomes to address
    changes in Cu concentration. The details have been described in the Discussion.
    The broken lines with arrows suggest the Cu stress signaling paths and involved
    components. ADDH, ATP‐dependent DNA helicase II; ADRH, ATP‐dependent RNA helicase;
    CP, catalase‐peroxidase; CSP, calcium signaling pathway; DPs, differentially expressed
    proteins; ECS, epithelial cell signaling in H. pylori infection; GnRH, GnRH signaling
    pathway; HD, Hedgehog signaling pathway; Hsp, heat shock protein; 14‐3‐3, 14‐3‐3
    proteins; ISP, insulin signaling pathway; MAPK, MAPK signaling pathway; NOD, NOD‐like
    receptor signaling pathway; NSP, neurotrophin signaling pathway; OP, oxidative
    phosphorylation; RIG, RIG‐I‐like receptor signaling pathway; ROS, reactive oxygen
    species; RuvBH, ATP‐dependent RuvB helicase; SOD, superoxide dismutase; SS, spermidine
    synthase; STPP, serine/threonine‐protein phosphatase; TCA, citrate cycle
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MAPK9
  - MAPK8
  - MAPK11
  - MAPK12
  - SPECC1
  - SOD1
  - MAPK10
  - MAPK14
  - MAPK3
  - YWHAQ
  - MAPK13
  - SOD3
  - SOD2
  - MAPK1
  - Aldehyde
  - Fatty acid
  - Glucose
  - (hydroxymethyl)glutathione
  - Trehalose
  - Citric acid
genes:
- word: -MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK9
  entrez: '5601'
- word: -MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK8
  entrez: '5599'
- word: -MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK11
  entrez: '5600'
- word: -MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK12
  entrez: '6300'
- word: •NSP
  symbol: NSP
  source: hgnc_alias_symbol
  hgnc_symbol: SPECC1
  entrez: '92521'
- word: SOD,
  symbol: SOD
  source: bioentities_symbol
  hgnc_symbol: SOD1
  entrez: '6647'
- word: -MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK10
  entrez: '5602'
- word: -MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK14
  entrez: '1432'
- word: -MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK3
  entrez: '5595'
- word: 14-3-3
  symbol: 14-3-3
  source: hgnc_alias_symbol
  hgnc_symbol: YWHAQ
  entrez: '10971'
- word: -MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK13
  entrez: '5603'
- word: SOD,
  symbol: SOD
  source: bioentities_symbol
  hgnc_symbol: SOD3
  entrez: '6649'
- word: SOD,
  symbol: SOD
  source: bioentities_symbol
  hgnc_symbol: SOD2
  entrez: '6648'
- word: -MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK1
  entrez: '5594'
chemicals:
- word: Aldehyde
  source: MESH
  identifier: D000447
- word: Fatty acid
  source: MESH
  identifier: D005227
- word: Glucose
  source: MESH
  identifier: D005947
- word: (hydroxymethyl)glutathione
  source: MESH
  identifier: D005978
- word: Trehalose
  source: MESH
  identifier: D014199
- word: Citric acid
  source: MESH
  identifier: D019343
diseases: []
figid_alias: PMC5552966__F9
redirect_from: /figures/PMC5552966__F9
figtype: Figure
---
