---
figid: PMC5898751__pone.0195908.g006
figlink: /pmc/articles/PMC5898751/figure/pone.0195908.g006/
number: F6
caption: 'The proposed pathways show phytohormone biosynthesis (light orange) and
  MAPK signaling pathways (light blue). This figure shows only the genes that have
  been associated with the pathway from transcriptome analysis. Red boxes show plant
  hormones, blue boxes show the genes up-regulated stress, green arrow shows genes
  involved in drought tolerance through ubiquitin mediated proteolysis and red arrows
  shows the direct involvement in the drought tolerance. Light blue lines with a bar
  denotes inhibition and simple arrows denotes activation. -p indicates de phosphorylation
  process. Aux/IAA: auxin-responsive protein IAA; GH3: Auxin-responsive GH3 family
  protein; SAUR: SAUR-like auxin-responsive protein family; CRE: Histidine kinase
  2/3/4 (cytokinin receptor); B-ARR: Two-component response regulator ARR-B family;
  GID1: Alpha/beta-Hydrolases superfamily protein; TF: Phytochrome interacting factor
  3 (PIF3); PP2C: Protein phosphatase 2C; SnRK2: Serine/threonine-protein kinase SRK2;
  ABF: ABA responsive element binding factor; ETR: Ethylene receptor; EIN3: ETHYLENE-INSENSITIVE3-like
  3; JAZ: Jasmonate ZIM domain-containing protein; RTE1: Transmembrane protein 222;
  MKK9: Mitogen-activated protein kinase kinase 9; ChiB: Basic endochitinase B; MAP3K17/18:
  Mitogen-activated protein kinase kinase kinase 17/18; MAPK1: Mitogen-activated protein
  kinase 1; MAPK6: Mitogen-activated protein kinase 6; CAT7: Catalase.'
pmcid: PMC5898751
papertitle: Transcriptomic analysis reveals the differentially expressed genes and
  pathways involved in drought tolerance in pearl millet [Pennisetum glaucum (L.)
  R. Br].
reftext: Ambika Dudhate, et al. PLoS One. 2018;13(4):e0195908.
pmc_ranked_result_index: '73168'
pathway_score: 0.9435495
filename: pone.0195908.g006.jpg
figtitle: Phytohormone and MAPK signaling pathway
year: '2018'
organisms:
- Pennisetum glaucum
- Synechocystis sp.
- Gossypium hirsutum
- Arabidopsis thaliana
- Cicer arietinum
- Pisum sativum
- Oryza sativa
- Cenchrus americanus
- Setaria italica
- Sorghum bicolor
- Triticum aestivum
- Zea mays
- Saccharomyces cerevisiae
- Triticum monococcum subsp. aegilopoides
- Saccharomyces cerevisiae CAT-1
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5898751__pone.0195908.g006.html
  '@type': Dataset
  description: 'The proposed pathways show phytohormone biosynthesis (light orange)
    and MAPK signaling pathways (light blue). This figure shows only the genes that
    have been associated with the pathway from transcriptome analysis. Red boxes show
    plant hormones, blue boxes show the genes up-regulated stress, green arrow shows
    genes involved in drought tolerance through ubiquitin mediated proteolysis and
    red arrows shows the direct involvement in the drought tolerance. Light blue lines
    with a bar denotes inhibition and simple arrows denotes activation. -p indicates
    de phosphorylation process. Aux/IAA: auxin-responsive protein IAA; GH3: Auxin-responsive
    GH3 family protein; SAUR: SAUR-like auxin-responsive protein family; CRE: Histidine
    kinase 2/3/4 (cytokinin receptor); B-ARR: Two-component response regulator ARR-B
    family; GID1: Alpha/beta-Hydrolases superfamily protein; TF: Phytochrome interacting
    factor 3 (PIF3); PP2C: Protein phosphatase 2C; SnRK2: Serine/threonine-protein
    kinase SRK2; ABF: ABA responsive element binding factor; ETR: Ethylene receptor;
    EIN3: ETHYLENE-INSENSITIVE3-like 3; JAZ: Jasmonate ZIM domain-containing protein;
    RTE1: Transmembrane protein 222; MKK9: Mitogen-activated protein kinase kinase
    9; ChiB: Basic endochitinase B; MAP3K17/18: Mitogen-activated protein kinase kinase
    kinase 17/18; MAPK1: Mitogen-activated protein kinase 1; MAPK6: Mitogen-activated
    protein kinase 6; CAT7: Catalase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MAPK14
  - MAPK13
  - MAPK9
  - MAPK8
  - MAPK12
  - MAPK10
  - ZNF346
  - TF
  - MAPK11
  - MAPK1
  - MAPK3
  - MAPK6
  - MAP2K6
  - Tryptophan
  - Diterpenoid
  - Cysteine
  - Methionine
  - Etlylene
  - Abscisic acid
  - Linolenic acid
genes:
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK14
  entrez: '1432'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK13
  entrez: '5603'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK9
  entrez: '5601'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK8
  entrez: '5599'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK12
  entrez: '6300'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK10
  entrez: '5602'
- word: JAZ
  symbol: JAZ
  source: hgnc_alias_symbol
  hgnc_symbol: ZNF346
  entrez: '23567'
- word: TF
  symbol: TF
  source: hgnc_symbol
  hgnc_symbol: TF
  entrez: '7018'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK11
  entrez: '5600'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK1
  entrez: '5594'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK3
  entrez: '5595'
- word: MAPK6
  symbol: MAPK6
  source: hgnc_symbol
  hgnc_symbol: MAPK6
  entrez: '5597'
- word: MKK6
  symbol: MKK6
  source: hgnc_alias_symbol
  hgnc_symbol: MAP2K6
  entrez: '5608'
chemicals:
- word: Tryptophan
  source: ''
  identifier: ''
- word: Diterpenoid
  source: MESH
  identifier: C503616
- word: Cysteine
  source: MESH
  identifier: D003545
- word: Methionine
  source: MESH
  identifier: D008715
- word: Etlylene
  source: ''
  identifier: ''
- word: Abscisic acid
  source: MESH
  identifier: D000040
- word: Linolenic acid
  source: MESH
  identifier: D008042
diseases: []
figid_alias: PMC5898751__F6
redirect_from: /figures/PMC5898751__F6
figtype: Figure
---
