---
figid: PMC5272919__nihms-845162-f0003
figlink: /pmc/articles/PMC5272919/figure/F2/
number: Figure 2
caption: 'a | Entry-associated disassembly. Viruses must disassemble their capsid
  to release their genome. Members of the Polyomaviridae disassemble their capsid
  by co-opting components of the endoplasmic reticulum (ER)-associated degradation
  (ERAD) pathway. To cause infection, polyomaviruses undergo receptor-mediated endocytosis
  and traffic to the ER (step 1). Once at the ER, they use protein disulfide isomerase
  (PDI)-family members to isomerize and reduce viral disulfide bonds (step 2). These
  events partially disassemble the virus particles to form hydrophobic viruses that
  engage binding immunoglobulin protein (BiP) through the activity of ER DNAJ domain-containing
  protein 3 (ERDJ3; step 3). 170 kDa glucose-regulated protein (GRP170) then releases
  polyomavirus from BiP, enabling the hydrophobic virus to insert into the ER membrane
  (step 4). The membrane-inserted virus reorganizes selective ER membrane proteins
  (B cell receptor-associated protein 31 (BAP31) and the J proteins DNAJ homologue
  subfamily B member 12 (DNAJB12), DNAJB14 and DNAJ C18) in the lipid bilayer to form
  foci (step 5). The membrane-attached cytosolic extraction machinery (heat shock
  cognate protein 70 (HSC70)–heat shock protein 105 (HSP105)–small glutamine-rich
  tetratricopeptide repeat-containing protein-α (SGTA)) then ejects the virus from
  the foci into the cytosol in a reaction that simultaneously disassembles the virus
  (step 6). Cytosolic disassembly enables the resulting core virus particle to move
  into the nucleus to cause infection. b | Genome translation. Some viruses exploit
  the ER-associated biosynthetic machinery to translate their genetic code in two
  ways: the translation of viral structural proteins that are incorporated into virions
  (for example, the envelope glycoprotein of HIV and haemagglutinin of influenza A
  virus (IAV)) and the translation of viral non-structural proteins that promote the
  subsequent viral replication step. This is evident during translation of the positive-sense
  RNA ((+)RNA) genome of viruses in the Flaviviridae (hepatitis C virus (HCV), dengue
  virus (DENV) and West Nile virus WNV) and Coronaviridae (severe acute respiratory
  syndrome coronavirus (SARS-CoV)) families, in which the newly synthesized replication
  proteins target to sites of virus replication on the ER (or ER-derived) membrane
  compartments in preparation for replication. At these sites, the physical architecture
  of the ER membrane effectively acts as a scaffold to recruit the viral replication
  proteins.'
pmcid: PMC5272919
papertitle: 'Opportunistic intruders: how viruses orchestrate ER functions to infect
  cells.'
reftext: Madhu Sudhan Ravindran, et al. Nat Rev Microbiol. ;14(7):407-420.
pmc_ranked_result_index: '229676'
pathway_score: 0.9480831
filename: nihms-845162-f0003.jpg
figtitle: 'Opportunistic intruders: how viruses orchestrate ER functions to infect
  cells'
year: ''
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5272919__nihms-845162-f0003.html
  '@type': Dataset
  description: 'a | Entry-associated disassembly. Viruses must disassemble their capsid
    to release their genome. Members of the Polyomaviridae disassemble their capsid
    by co-opting components of the endoplasmic reticulum (ER)-associated degradation
    (ERAD) pathway. To cause infection, polyomaviruses undergo receptor-mediated endocytosis
    and traffic to the ER (step 1). Once at the ER, they use protein disulfide isomerase
    (PDI)-family members to isomerize and reduce viral disulfide bonds (step 2). These
    events partially disassemble the virus particles to form hydrophobic viruses that
    engage binding immunoglobulin protein (BiP) through the activity of ER DNAJ domain-containing
    protein 3 (ERDJ3; step 3). 170 kDa glucose-regulated protein (GRP170) then releases
    polyomavirus from BiP, enabling the hydrophobic virus to insert into the ER membrane
    (step 4). The membrane-inserted virus reorganizes selective ER membrane proteins
    (B cell receptor-associated protein 31 (BAP31) and the J proteins DNAJ homologue
    subfamily B member 12 (DNAJB12), DNAJB14 and DNAJ C18) in the lipid bilayer to
    form foci (step 5). The membrane-attached cytosolic extraction machinery (heat
    shock cognate protein 70 (HSC70)–heat shock protein 105 (HSP105)–small glutamine-rich
    tetratricopeptide repeat-containing protein-α (SGTA)) then ejects the virus from
    the foci into the cytosol in a reaction that simultaneously disassembles the virus
    (step 6). Cytosolic disassembly enables the resulting core virus particle to move
    into the nucleus to cause infection. b | Genome translation. Some viruses exploit
    the ER-associated biosynthetic machinery to translate their genetic code in two
    ways: the translation of viral structural proteins that are incorporated into
    virions (for example, the envelope glycoprotein of HIV and haemagglutinin of influenza
    A virus (IAV)) and the translation of viral non-structural proteins that promote
    the subsequent viral replication step. This is evident during translation of the
    positive-sense RNA ((+)RNA) genome of viruses in the Flaviviridae (hepatitis C
    virus (HCV), dengue virus (DENV) and West Nile virus WNV) and Coronaviridae (severe
    acute respiratory syndrome coronavirus (SARS-CoV)) families, in which the newly
    synthesized replication proteins target to sites of virus replication on the ER
    (or ER-derived) membrane compartments in preparation for replication. At these
    sites, the physical architecture of the ER membrane effectively acts as a scaffold
    to recruit the viral replication proteins.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - DNAJC18
  - DNAJB14
  - HSPA8
  - DNAJB11
  - HYOU1
  - SGTA
  - HSPA5
  - DNAJB12
  - BCAP31
genes:
- word: DNAJC18,
  symbol: DNAJC18
  source: hgnc_symbol
  hgnc_symbol: DNAJC18
  entrez: '202052'
- word: DNAJB14,
  symbol: DNAJB14
  source: hgnc_symbol
  hgnc_symbol: DNAJB14
  entrez: '79982'
- word: HSC70
  symbol: HSC70
  source: hgnc_alias_symbol
  hgnc_symbol: HSPA8
  entrez: '3312'
- word: ERDJ3
  symbol: ERdj3
  source: hgnc_alias_symbol
  hgnc_symbol: DNAJB11
  entrez: '51726'
- word: GRP170
  symbol: Grp170
  source: hgnc_alias_symbol
  hgnc_symbol: HYOU1
  entrez: '10525'
- word: SGTA
  symbol: SGTA
  source: hgnc_symbol
  hgnc_symbol: SGTA
  entrez: '6449'
- word: BiP
  symbol: BiP
  source: hgnc_alias_symbol
  hgnc_symbol: HSPA5
  entrez: '3309'
- word: DNAJB12,
  symbol: DNAJB12
  source: hgnc_symbol
  hgnc_symbol: DNAJB12
  entrez: '54788'
- word: BAP31
  symbol: BAP31
  source: hgnc_alias_symbol
  hgnc_symbol: BCAP31
  entrez: '10134'
chemicals: []
diseases: []
---
