---
figid: PMC5047111__PP_PP201601097DR1_f8
figlink: /pmc/articles/PMC5047111/figure/fig8/
number: Figure 8
caption: Summary diagram showing the core pathway of Pro metabolism in relation to
  lipid, cuticle, and redox-related genes found to affect Pro accumulation. Multiple
  genes and pharmacological treatments disrupting lipid metabolism or NADPH oxidase
  activity were found to affect Pro accumulation and PDH1pro:LUC2 expression. This
  included CYP86A2 and LACS2 identified in the PDH1pro:LUC2 forward genetic screen;
  CYP86A4, CED1, MYB16, and MYB106 identified by reverse genetics; CAC3 and KCS genes
  identified by p5cs1-4 RNA sequencing; as well as strong effects of the NADPH oxidase
  inhibitor . The reactions catalyzed by these proteins are spread across several
  compartments, including endoplasmic reticulum (ER), chloroplast, and plasma membrane
  (PM). The commonality among these pathways and Pro metabolism itself is an effect
  on redox status by the consumption of NADPH and the regeneration of NADP+, either
  directly, as for P5CS1, P5CR, LACS2, CYP86A2, and NADPH oxidases, or indirectly
  by modifying the flux through lipid synthesis, as for CAC3, KCS, CED1, MYB16, and
  MYB106. This commonality, along with data obtained with  scavengers and  treatment,
  indicates that redox status is a key factor linking these pathways together. Consistent
  with this, RNA sequencing data showed that p5cs1-4 had substantial alterations in
  gene expression-related chloroplast and mitochondrial redox metabolism, further
  indicating the relationship of Pro to redox status. The similar effects of p5cs1-4
  on gene expression at both high and low ψw indicated the importance of Pro metabolism,
  and likely a high rate of flux through the Pro cycle of synthesis and catabolism,
  even in unstressed plants where Pro level is low. P5CS1 is drawn in its own box
  to indicate that its subcellular localization is unclear. How changes in redox state
  are communicated between these subcellular compartments is unknown.
pmcid: PMC5047111
papertitle: Proline Coordination with Fatty Acid Synthesis and Redox Metabolism of
  Chloroplast and Mitochondria.
reftext: Suhas Shinde, et al. Plant Physiol. 2016 Oct;172(2):1074-1088.
pmc_ranked_result_index: '110229'
pathway_score: 0.9752289
filename: PP_PP201601097DR1_f8.jpg
figtitle: Summary diagram showing the core pathway of Pro metabolism in relation to
  lipid, cuticle, and redox-related genes found to affect Pro accumulation
year: '2016'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5047111__PP_PP201601097DR1_f8.html
  '@type': Dataset
  description: Summary diagram showing the core pathway of Pro metabolism in relation
    to lipid, cuticle, and redox-related genes found to affect Pro accumulation. Multiple
    genes and pharmacological treatments disrupting lipid metabolism or NADPH oxidase
    activity were found to affect Pro accumulation and PDH1pro:LUC2 expression. This
    included CYP86A2 and LACS2 identified in the PDH1pro:LUC2 forward genetic screen;
    CYP86A4, CED1, MYB16, and MYB106 identified by reverse genetics; CAC3 and KCS
    genes identified by p5cs1-4 RNA sequencing; as well as strong effects of the NADPH
    oxidase inhibitor . The reactions catalyzed by these proteins are spread across
    several compartments, including endoplasmic reticulum (ER), chloroplast, and plasma
    membrane (PM). The commonality among these pathways and Pro metabolism itself
    is an effect on redox status by the consumption of NADPH and the regeneration
    of NADP+, either directly, as for P5CS1, P5CR, LACS2, CYP86A2, and NADPH oxidases,
    or indirectly by modifying the flux through lipid synthesis, as for CAC3, KCS,
    CED1, MYB16, and MYB106. This commonality, along with data obtained with  scavengers
    and  treatment, indicates that redox status is a key factor linking these pathways
    together. Consistent with this, RNA sequencing data showed that p5cs1-4 had substantial
    alterations in gene expression-related chloroplast and mitochondrial redox metabolism,
    further indicating the relationship of Pro to redox status. The similar effects
    of p5cs1-4 on gene expression at both high and low ψw indicated the importance
    of Pro metabolism, and likely a high rate of flux through the Pro cycle of synthesis
    and catabolism, even in unstressed plants where Pro level is low. P5CS1 is drawn
    in its own box to indicate that its subcellular localization is unclear. How changes
    in redox state are communicated between these subcellular compartments is unknown.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TBCE
  - DSP
  - PYCR1
  - ALDH4A1
  - Cafenstrole
  - Fatty Acid Malonyl-CoA
  - Malonyl-CoA
  - pyruvate
genes:
- word: Cafenstrole-|KCS
  symbol: KCS
  source: hgnc_prev_symbol
  hgnc_symbol: TBCE
  entrez: '6905'
- word: DPI
  symbol: DPI
  source: hgnc_alias_symbol
  hgnc_symbol: DSP
  entrez: '1832'
- word: P5C
  symbol: P5C
  source: hgnc_alias_symbol
  hgnc_symbol: PYCR1
  entrez: '5831'
- word: P5CDH
  symbol: P5CDh
  source: hgnc_alias_symbol
  hgnc_symbol: ALDH4A1
  entrez: '8659'
chemicals:
- word: Cafenstrole
  source: MESH
  identifier: C444649
- word: Fatty Acid Malonyl-CoA
  source: MESH
  identifier: D008316
- word: Malonyl-CoA
  source: MESH
  identifier: D008316
- word: pyruvate
  source: MESH
  identifier: D011773
diseases: []
---
