---
figid: PMC5919953__fmicb-09-00746-g0004
figlink: /pmc/articles/PMC5919953/figure/F4/
number: F4
caption: 'Nitrogen and sulfur metabolic pathways identified in the six Microcystis-epibiont
  communities. (A) Nitrogen metabolism; (B) Sulfur metabolism. Diagrams are based
  on KEGG pathway maps. Available enzyme classification numbers for each step are
  included in boxes. Each column represents a different community. DA, TA, DF, TF,
  DW, and TW are the abbreviations for the six Microcystis-epibiont communities. Box
  colors indicate bacteria from different classes: dark green, Cyanobacteria (Microcystis);
  pink, Alphaproteobacteria; gold, Betaproteobacteria; light green, Gammaproteobacteria;
  and blue, other phyla such as Bacteroidetes or Gemmatimonadetes. Gray indicates
  absence of the corresponding enzyme from the community.'
pmcid: PMC5919953
papertitle: A Large-Scale Comparative Metagenomic Study Reveals the Functional Interactions
  in Six Bloom-Forming Microcystis-Epibiont Communities.
reftext: Qi Li, et al. Front Microbiol. 2018;9:746.
pmc_ranked_result_index: '93577'
pathway_score: 0.8712137
filename: fmicb-09-00746-g0004.jpg
figtitle: Nitrogen and sulfur metabolic pathways identified in the six Microcystis-epibiont
  communities
year: '2018'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5919953__fmicb-09-00746-g0004.html
  '@type': Dataset
  description: 'Nitrogen and sulfur metabolic pathways identified in the six Microcystis-epibiont
    communities. (A) Nitrogen metabolism; (B) Sulfur metabolism. Diagrams are based
    on KEGG pathway maps. Available enzyme classification numbers for each step are
    included in boxes. Each column represents a different community. DA, TA, DF, TF,
    DW, and TW are the abbreviations for the six Microcystis-epibiont communities.
    Box colors indicate bacteria from different classes: dark green, Cyanobacteria
    (Microcystis); pink, Alphaproteobacteria; gold, Betaproteobacteria; light green,
    Gammaproteobacteria; and blue, other phyla such as Bacteroidetes or Gemmatimonadetes.
    Gray indicates absence of the corresponding enzyme from the community.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - Sulfur
  - Nitrogen
  - Nitrate
  - ONitroalkane
  - Sulfite
  - Sulfide
  - Nitrite
  - Nitric oxideo
  - Sulfate
  - O Sulfate
  - Thiosulfate
  - oxideC
  - SO
  - OThiosulfate
  - HydroxylamineO
  - SOXYZ-S-SH
  - SOXYZ-S-S-SO3
  - Nitrileo
genes: []
chemicals:
- word: Sulfur
  source: MESH
  identifier: D013455
- word: Nitrogen
  source: MESH
  identifier: D009584
- word: Nitrate
  source: MESH
  identifier: D009566
- word: ONitroalkane
  source: ''
  identifier: ''
- word: Sulfite
  source: MESH
  identifier: D013447
- word: Sulfide
  source: MESH
  identifier: D013440
- word: Nitrite
  source: MESH
  identifier: D009573
- word: Nitric oxideo
  source: MESH
  identifier: D017942
- word: Sulfate
  source: MESH
  identifier: D013431
- word: O Sulfate
  source: MESH
  identifier: D013431
- word: Thiosulfate
  source: MESH
  identifier: D013885
- word: oxideC
  source: MESH
  identifier: D010087
- word: SO
  source: MESH
  identifier: C443440
- word: OThiosulfate
  source: ''
  identifier: ''
- word: HydroxylamineO
  source: MESH
  identifier: D019811
- word: SOXYZ-S-SH
  source: MESH
  identifier: C014393
- word: SOXYZ-S-S-SO3
  source: MESH
  identifier: C443440
- word: Nitrileo
  source: ''
  identifier: ''
diseases: []
figid_alias: PMC5919953__F4
redirect_from: /figures/PMC5919953__F4
figtype: Figure
---
