---
figid: PMC55327__gb-2001-2-8-research0030-1
figlink: /pmc/articles/PMC55327/figure/F1/
number: Figure 1
caption: 'Biochemical pathway for phenylalanine and tyrosine biosynthesis. Small-molecule
  abbreviations: AGN, L-arogenate; CHA, chorismate; DAHP, 2-keto-3-deoxy-D-arabino-heptulosonate
  7-P; DHQ, dehydroquinate; DHS, dehydroshikimate; E4P, erythrose-4-P; EPSP, enolpyruvylshikimate
  3-P; HPP, 4-hydroxyphenylpyruvate; PEP, phosphoenolpyruvate; PHE, phenylalanine;
  PPA, prephenate; PPY, phenylpyruvate; SHK, shikimate; S3P, shikimate-3-P; TYR, tyrosine.
  Enzyme acronyms: [Aat], aromatic aminotransferase; [AroA], DAHP synthase; [AroB]
  DHQ synthase; [AroC], dehydroquinase; [AroD], SHK dehydrogenase; [AroE], SHK kinase;
  [AroF], EPSP synthase; [AroG], CHA synthase; [AroQ/AroR/AroH], CHA mutase; [PheA],
  prephenate dehydratase; [TyrAc], prephenate (cyclohexadienyl) dehydrogenase; [*Aat],
  periplasmic aromatic aminotransferase; [*AroQ], periplasmic chorismate mutase; [*PheC],
  periplasmic cyclohexadienyl dehydratase. (In P. aeruginosa, the only organism so
  far shown to have *Aat, it belongs to the *AatIβ grouping.) The flow route that
  progresses from PPA → AGN → TYR is shown in grey because this is a minor route in
  both P. aeruginosa and S. typhimurium. Different homology groups known so far that
  correspond with a given catalytic step are distinguished by Roman numeral subscripts,
  for example AroAI or AroAII. Subgroups within a homology group carry a further identifier,
  for example AroAIα or AroAIβ . As far as possible, these identifiers have been chosen
  to accommodate (within the constraints of logical naming) established precedents
  of naming. For example, the two homology groups of AroA were originally distinguished
  as class I or class II [50], and α and β subgroups for family-I DAHP synthases were
  defined recently [24]. AroCI and AroCII have been distinguished as biosynthetic
  or catabolic dehydroquinase, respectively, but this is a misnomer as many genomes
  use AroCII for aromatic amino acid biosynthesis (G.X. and R.A.J., unpublished data).
  Thus far, AroEI is used exclusively in Bacteria, and AroEII exclusively in Archaea
  [14]. The AroQIα group of chorismate mutases has been so named to reflect previous
  reference to them as AroQ [22, 46]. Likewise, AroRIβreflects previous reference
  to it as AroR [32]. The α, β and γ subgrouping identifiers for Aat reflect the subgroupings
  within family-I aminotransferases defined by Jensen and Gu [3]. Color-coding is
  used to illustrate the substantial difference between P.aeruginosa and S. typhimurium
  in deployment of different homolog or analog proteins (numbers within boxes indicate
  the number of paralogs), chorismate mutase fusions (peg-and-hole matches) and periplasm-localized
  enzymes.'
pmcid: PMC55327
papertitle: The emerging periplasm-localized subclass of AroQ chorismate mutases,
  exemplified by those from Salmonella typhimurium and Pseudomonas aeruginosa.
reftext: David H Calhoun, et al. Genome Biol. 2001;2(8):research0030.1-research0030.16.
pmc_ranked_result_index: '211183'
pathway_score: 0.9039492
filename: gb-2001-2-8-research0030-1.jpg
figtitle: Biochemical pathway for phenylalanine and tyrosine biosynthesis
year: '2001'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC55327__gb-2001-2-8-research0030-1.html
  '@type': Dataset
  description: 'Biochemical pathway for phenylalanine and tyrosine biosynthesis. Small-molecule
    abbreviations: AGN, L-arogenate; CHA, chorismate; DAHP, 2-keto-3-deoxy-D-arabino-heptulosonate
    7-P; DHQ, dehydroquinate; DHS, dehydroshikimate; E4P, erythrose-4-P; EPSP, enolpyruvylshikimate
    3-P; HPP, 4-hydroxyphenylpyruvate; PEP, phosphoenolpyruvate; PHE, phenylalanine;
    PPA, prephenate; PPY, phenylpyruvate; SHK, shikimate; S3P, shikimate-3-P; TYR,
    tyrosine. Enzyme acronyms: [Aat], aromatic aminotransferase; [AroA], DAHP synthase;
    [AroB] DHQ synthase; [AroC], dehydroquinase; [AroD], SHK dehydrogenase; [AroE],
    SHK kinase; [AroF], EPSP synthase; [AroG], CHA synthase; [AroQ/AroR/AroH], CHA
    mutase; [PheA], prephenate dehydratase; [TyrAc], prephenate (cyclohexadienyl)
    dehydrogenase; [*Aat], periplasmic aromatic aminotransferase; [*AroQ], periplasmic
    chorismate mutase; [*PheC], periplasmic cyclohexadienyl dehydratase. (In P. aeruginosa,
    the only organism so far shown to have *Aat, it belongs to the *AatIβ grouping.)
    The flow route that progresses from PPA → AGN → TYR is shown in grey because this
    is a minor route in both P. aeruginosa and S. typhimurium. Different homology
    groups known so far that correspond with a given catalytic step are distinguished
    by Roman numeral subscripts, for example AroAI or AroAII. Subgroups within a homology
    group carry a further identifier, for example AroAIα or AroAIβ . As far as possible,
    these identifiers have been chosen to accommodate (within the constraints of logical
    naming) established precedents of naming. For example, the two homology groups
    of AroA were originally distinguished as class I or class II [50], and α and β
    subgroups for family-I DAHP synthases were defined recently [24]. AroCI and AroCII
    have been distinguished as biosynthetic or catabolic dehydroquinase, respectively,
    but this is a misnomer as many genomes use AroCII for aromatic amino acid biosynthesis
    (G.X. and R.A.J., unpublished data). Thus far, AroEI is used exclusively in Bacteria,
    and AroEII exclusively in Archaea [14]. The AroQIα group of chorismate mutases
    has been so named to reflect previous reference to them as AroQ [22, 46]. Likewise,
    AroRIβreflects previous reference to it as AroR [32]. The α, β and γ subgrouping
    identifiers for Aat reflect the subgroupings within family-I aminotransferases
    defined by Jensen and Gu [3]. Color-coding is used to illustrate the substantial
    difference between P.aeruginosa and S. typhimurium in deployment of different
    homolog or analog proteins (numbers within boxes indicate the number of paralogs),
    chorismate mutase fusions (peg-and-hole matches) and periplasm-localized enzymes.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PAEP
  - TCFL5
  - PPY
  - SERPINA1
  - SHPK
  - TYR
  - PSPH
  - PHE
  - TYR
genes:
- word: PEP
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
- word: CHA
  symbol: CHA
  source: hgnc_alias_symbol
  hgnc_symbol: TCFL5
  entrez: '10732'
- word: PPY
  symbol: PPY
  source: hgnc_symbol
  hgnc_symbol: PPY
  entrez: '5539'
- word: '[Aat)'
  symbol: AAT
  source: hgnc_alias_symbol
  hgnc_symbol: SERPINA1
  entrez: '5265'
- word: SHK-
  symbol: SHK
  source: hgnc_alias_symbol
  hgnc_symbol: SHPK
  entrez: '23729'
- word: TYR
  symbol: TYR
  source: hgnc_symbol
  hgnc_symbol: TYR
  entrez: '7299'
- word: ЕPSP
  symbol: PSP
  source: hgnc_prev_symbol
  hgnc_symbol: PSPH
  entrez: '5723'
chemicals:
- word: PHE
  source: MESH
  identifier: D010649
- word: TYR
  source: MESH
  identifier: D014443
diseases: []
---
