---
figid: PMC5080403__sys0051620540002
figlink: /pmc/articles/PMC5080403/figure/fig2/
number: F2
caption: Selected metabolic features of the unFirm_1 phylotype as inferred from genome
  and proteome comparisons. The unFirm_1 phylotype encoded limited sugar utilization
  capabilities; however, a complete WL pathway and enzymes inferred to function in
  fatty acid degradation were observed. Metaproteomic analysis (genes, enzyme complexes,
  and transport systems are highlighted to indicate quantitative MaxQuant LFQ values)
  indicated that unFirm_1 was much more detectable in the Link_ADIb sample, with all
  WL pathway enzymes being detected, as well as genes associated with fatty acid degradation.
  In particular, the high relative abundance of a unidirectional fructose 1,6-bisphosphate
  aldolase/phosphatase (ALDOA) supported the hypotheses that unFirm_1 performs gluconeogenesis
  (anabolic glucose metabolism). Broken lines indicate annotations for which representative
  genes were not identified in the respective reconstructed genomes. Gene names and
  identification (IMG gene ID) numbers can be found in  in the supplemental material.
  THF, tetrahydrofolate; FTHFS, formyl-tetrahydrofolate synthase; PK, pyruvate kinase;
  PEP, phosphoenolpyruvate; G6P, glucose-6-phosphate; F6P, fructose-6-phosphate; M6P,
  mannose-6-phosphate; PMM, phosphomannomutase; ACSL, long-chain-fatty-acid–CoA ligase;
  HADH, 3-hydroxyacyl-CoA dehydrogenase; GMPPB, mannose-1-phosphate guanylyltransferase;
  ACAT, acetyl-CoA acetyltransferase; XLF, d-xylulose; G3H, d-glyceraldehyde 3-phosphate;
  E4P, d-erythrose 4-phosphate; 5SP, d-xylulose 5-phosphate; S7P, sedoheptulose 7-phosphate;
  5RP, d-ribulose 5-phosphate; RP5, d-ribose 5-phosphate.
pmcid: PMC5080403
papertitle: Novel Syntrophic Populations Dominate an Ammonia-Tolerant Methanogenic
  Microbiome.
reftext: J. A. Frank, et al. mSystems. 2016 Sep-Oct;1(5):e00092-16.
pmc_ranked_result_index: '150010'
pathway_score: 0.743546
filename: sys0051620540002.jpg
figtitle: Selected metabolic features of the unFirm_1 phylotype as inferred from genome
  and proteome comparisons
year: '2016'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5080403__sys0051620540002.html
  '@type': Dataset
  description: Selected metabolic features of the unFirm_1 phylotype as inferred from
    genome and proteome comparisons. The unFirm_1 phylotype encoded limited sugar
    utilization capabilities; however, a complete WL pathway and enzymes inferred
    to function in fatty acid degradation were observed. Metaproteomic analysis (genes,
    enzyme complexes, and transport systems are highlighted to indicate quantitative
    MaxQuant LFQ values) indicated that unFirm_1 was much more detectable in the Link_ADIb
    sample, with all WL pathway enzymes being detected, as well as genes associated
    with fatty acid degradation. In particular, the high relative abundance of a unidirectional
    fructose 1,6-bisphosphate aldolase/phosphatase (ALDOA) supported the hypotheses
    that unFirm_1 performs gluconeogenesis (anabolic glucose metabolism). Broken lines
    indicate annotations for which representative genes were not identified in the
    respective reconstructed genomes. Gene names and identification (IMG gene ID)
    numbers can be found in  in the supplemental material. THF, tetrahydrofolate;
    FTHFS, formyl-tetrahydrofolate synthase; PK, pyruvate kinase; PEP, phosphoenolpyruvate;
    G6P, glucose-6-phosphate; F6P, fructose-6-phosphate; M6P, mannose-6-phosphate;
    PMM, phosphomannomutase; ACSL, long-chain-fatty-acid–CoA ligase; HADH, 3-hydroxyacyl-CoA
    dehydrogenase; GMPPB, mannose-1-phosphate guanylyltransferase; ACAT, acetyl-CoA
    acetyltransferase; XLF, d-xylulose; G3H, d-glyceraldehyde 3-phosphate; E4P, d-erythrose
    4-phosphate; 5SP, d-xylulose 5-phosphate; S7P, sedoheptulose 7-phosphate; 5RP,
    d-ribulose 5-phosphate; RP5, d-ribose 5-phosphate.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PAEP
  - Acetyl phosphate
  - Acetate
  - D-alanine
  - alanine
  - L-arginine
  - Fatty acids
  - formate
  - glucose
  - mannose
  - Pyruvate
genes:
- word: PÉP
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
chemicals:
- word: Acetyl phosphate
  source: MESH
  identifier: C011632
- word: Acetate
  source: MESH
  identifier: D000085
- word: D-alanine
  source: MESH
  identifier: D000409
- word: alanine
  source: MESH
  identifier: D000409
- word: L-arginine
  source: MESH
  identifier: D001120
- word: Fatty acids
  source: MESH
  identifier: D005227
- word: formate
  source: MESH
  identifier: D005561
- word: glucose
  source: MESH
  identifier: D005947
- word: mannose
  source: MESH
  identifier: D008358
- word: Pyruvate
  source: MESH
  identifier: D011773
diseases: []
figid_alias: PMC5080403__F2
redirect_from: /figures/PMC5080403__F2
figtype: Figure
---
