---
figid: PMC5441870__elife-18990-fig3
figlink: /pmc/articles/PMC5441870/figure/fig3/
number: Figure 3
caption: 'Enzyme names are reported together with the gene locus number (Theam_number).
  Primary compounds involved in reactions were also reported, however visualized reactions
  are not complete. Pathways were arbitrarily color coded according to their reconstructed
  function and are consistent throughout the paper. Solid shapes represent genes for
  which the enzyme was found in the proteome, while outlined shapes were only identified
  in the genome. Circled numbers 1 to 6 represent reaction numbers of the putative
  reductive acetyl-CoA pathway as described in the text. Abbreviations: NITRATE AMMONIFICATION:
  NapCMADGH – Periplasmic nitrate reductase complex; NirA – Putative nitrite reductase;
  AmtB – ammonia transporter; GlnA – L-Glutamine synthetase; GltA – Glutamate synthetase.
  HYDROGEN OXIDATION: HynABC – Ni-Fe Membrane bound hydrogenase; FrhACB – Cytosolic
  Ni-Fe hydrogenase/putative coenzyme F420 hydrogenase; HupD – Cyrosolic Ni-Fe hydrogenase
  maturation protease; HypA-F – Hydrogenases expression/synthesis accessory proteins;
  EchABCEF – Ech membrane bound hydrogenase complex; HydAB – Cytosolic Ni-Fe hydrogenases
  potentially involved in ferredoxin reduction; Hdr? – Missing heterodisulfide reductase
  CoB-CoM; FdhABC – Formate dehydrogenase. ENERGY PRODUCTION: NADH dehydrogenase and
  ATP synthetase are reported without the names of the single units; Nhe – Sodium/hydrogen
  symporter. SULFUR REDUCTION: Sqr – Putative sulfate quinone reductase involved in
  sulfur respiration; Nsr? – FAD/NAD nucleotide-disulphide oxidoreductase; Aps – Sulfate
  adenylyl transferase and kinase involved in assimilation of sulfur. FLAGELLAR COMPLEX:
  for simplicity single unit names are not reported. REDUCTIVE ACETYL-CoA: Fhs – Proposed
  reverse formyl-THF deformylase; PurHT – Formyltransferases (phosphoribosylaminoimidazolecarboxamide
  and phosphoribosylglycinamide) FolD – Methenyl-THF cyclohydrolase and dehydrogenase;
  MetVF – Methylene-THF reductase; MetR – Putative methyl-transferase; CodH – Carbon
  monoxyde dehydrogenase; AcsA – Acetyl-CoA ligase/synthase; AccABCD – Acetyl-CoA
  carboxylase. REDUCTIVE CITRIC ACID CYCLE: AclAB – ATP-citrate lyase; Mdh – Malate
  dehydrogenase; FumAB – Fumarate hydratase; FrdAB – Fumarate reductase; SucCD – Succinyl-CoA
  synthetase; OorABCD – 2-Oxoglutarate synthase; Idh2 – Isocitrate dehydrogenase/2-oxoglutarate
  carboxylase; AcnB – Aconitate hydratase; PorABDG – Pyruvate synthase; PycAB - Pyruvate
  carboxylase; PpsA – Phosphoenolpyruvate synthase water dikinase; PyK – Pyruvate:water
  dikinase. GLUCONEOGENESIS: Eno – Enolase; Pgm – Phosphoglycerate mutase; Pgk – Phosphoglycerate
  kinase; Gapor – Glyceraldehyde-3-phosphate dehydrogenase; Gapdh – Glyceraldehyde
  3-phosphate dehydrogenase; Fba – Predicted fructose-bisphosphate aldolase; Tim –
  Triosephosphate isomerase; Fbp – Fructose-1,6-bisphosphatase I; Pgi – Phosphoglucose
  isomerase; Pgm – Phosphoglucomutase.DOI: http://dx.doi.org/10.7554/eLife.18990.00710.7554/eLife.18990.008Figure
  3—source data 1.List of the proteins identified in the proteome of T. ammonificans
  grown under nitrate reducing conditions.DOI: http://dx.doi.org/10.7554/eLife.18990.008'
pmcid: PMC5441870
papertitle: Insight into the evolution of microbial metabolism from the deep-branching
  bacterium, Thermovibrio ammonificans.
reftext: Donato Giovannelli, et al. eLife. 2017;6:e18990.
pmc_ranked_result_index: '233222'
pathway_score: 0.8993864
filename: elife-18990-fig3.jpg
figtitle: Central metabolism of T
year: '2017'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5441870__elife-18990-fig3.html
  '@type': Dataset
  description: 'Enzyme names are reported together with the gene locus number (Theam_number).
    Primary compounds involved in reactions were also reported, however visualized
    reactions are not complete. Pathways were arbitrarily color coded according to
    their reconstructed function and are consistent throughout the paper. Solid shapes
    represent genes for which the enzyme was found in the proteome, while outlined
    shapes were only identified in the genome. Circled numbers 1 to 6 represent reaction
    numbers of the putative reductive acetyl-CoA pathway as described in the text.
    Abbreviations: NITRATE AMMONIFICATION: NapCMADGH – Periplasmic nitrate reductase
    complex; NirA – Putative nitrite reductase; AmtB – ammonia transporter; GlnA –
    L-Glutamine synthetase; GltA – Glutamate synthetase. HYDROGEN OXIDATION: HynABC
    – Ni-Fe Membrane bound hydrogenase; FrhACB – Cytosolic Ni-Fe hydrogenase/putative
    coenzyme F420 hydrogenase; HupD – Cyrosolic Ni-Fe hydrogenase maturation protease;
    HypA-F – Hydrogenases expression/synthesis accessory proteins; EchABCEF – Ech
    membrane bound hydrogenase complex; HydAB – Cytosolic Ni-Fe hydrogenases potentially
    involved in ferredoxin reduction; Hdr? – Missing heterodisulfide reductase CoB-CoM;
    FdhABC – Formate dehydrogenase. ENERGY PRODUCTION: NADH dehydrogenase and ATP
    synthetase are reported without the names of the single units; Nhe – Sodium/hydrogen
    symporter. SULFUR REDUCTION: Sqr – Putative sulfate quinone reductase involved
    in sulfur respiration; Nsr? – FAD/NAD nucleotide-disulphide oxidoreductase; Aps
    – Sulfate adenylyl transferase and kinase involved in assimilation of sulfur.
    FLAGELLAR COMPLEX: for simplicity single unit names are not reported. REDUCTIVE
    ACETYL-CoA: Fhs – Proposed reverse formyl-THF deformylase; PurHT – Formyltransferases
    (phosphoribosylaminoimidazolecarboxamide and phosphoribosylglycinamide) FolD –
    Methenyl-THF cyclohydrolase and dehydrogenase; MetVF – Methylene-THF reductase;
    MetR – Putative methyl-transferase; CodH – Carbon monoxyde dehydrogenase; AcsA
    – Acetyl-CoA ligase/synthase; AccABCD – Acetyl-CoA carboxylase. REDUCTIVE CITRIC
    ACID CYCLE: AclAB – ATP-citrate lyase; Mdh – Malate dehydrogenase; FumAB – Fumarate
    hydratase; FrdAB – Fumarate reductase; SucCD – Succinyl-CoA synthetase; OorABCD
    – 2-Oxoglutarate synthase; Idh2 – Isocitrate dehydrogenase/2-oxoglutarate carboxylase;
    AcnB – Aconitate hydratase; PorABDG – Pyruvate synthase; PycAB - Pyruvate carboxylase;
    PpsA – Phosphoenolpyruvate synthase water dikinase; PyK – Pyruvate:water dikinase.
    GLUCONEOGENESIS: Eno – Enolase; Pgm – Phosphoglycerate mutase; Pgk – Phosphoglycerate
    kinase; Gapor – Glyceraldehyde-3-phosphate dehydrogenase; Gapdh – Glyceraldehyde
    3-phosphate dehydrogenase; Fba – Predicted fructose-bisphosphate aldolase; Tim
    – Triosephosphate isomerase; Fbp – Fructose-1,6-bisphosphatase I; Pgi – Phosphoglucose
    isomerase; Pgm – Phosphoglucomutase.DOI: http://dx.doi.org/10.7554/eLife.18990.00710.7554/eLife.18990.008Figure
    3—source data 1.List of the proteins identified in the proteome of T. ammonificans
    grown under nitrate reducing conditions.DOI: http://dx.doi.org/10.7554/eLife.18990.008'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ENO2
  - ADCY4
  - ADCY2
  - ENO1
  - ADCY3
  - ADCY1
  - VCAN
  - ADCY9
  - CPSF4
  - NDP
  - ENO4
  - RTCA
  - ENO3
  - HADH
  - JTB
  - FANCA
  - NAPA
  - ADCY8
  - ADCY5
  - ADCY10
  - CTNNBL1
  - ADCY7
  - IRF1
  - PMCH
  - ADCY6
  - ETHE1
  - Nitrate
genes:
- word: Eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO2
  entrez: '2026'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY4
  entrez: '196883'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY2
  entrez: '108'
- word: Eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO1
  entrez: '2023'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY3
  entrez: '109'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY1
  entrez: '107'
- word: Pgm
  symbol: PG-M
  source: hgnc_alias_symbol
  hgnc_symbol: VCAN
  entrez: '1462'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY9
  entrez: '115'
- word: Nar
  symbol: NAR
  source: hgnc_alias_symbol
  hgnc_symbol: CPSF4
  entrez: '10898'
- word: Ndp
  symbol: NDP
  source: hgnc_symbol
  hgnc_symbol: NDP
  entrez: '4693'
- word: Eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO4
  entrez: '387712'
- word: rTCA
  symbol: RTCA
  source: hgnc_symbol
  hgnc_symbol: RTCA
  entrez: '8634'
- word: Eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO3
  entrez: '2027'
- word: HADH
  symbol: HADH
  source: hgnc_symbol
  hgnc_symbol: HADH
  entrez: '3033'
- word: par
  symbol: PAR
  source: hgnc_alias_symbol
  hgnc_symbol: JTB
  entrez: '10899'
- word: Faa
  symbol: FAA
  source: hgnc_alias_symbol
  hgnc_symbol: FANCA
  entrez: '2175'
- word: NapA
  symbol: NAPA
  source: hgnc_symbol
  hgnc_symbol: NAPA
  entrez: '8775'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY8
  entrez: '114'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY5
  entrez: '111'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY10
  entrez: '55811'
- word: Nap
  symbol: NAP
  source: hgnc_alias_symbol
  hgnc_symbol: CTNNBL1
  entrez: '56259'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY7
  entrez: '113'
- word: Mar
  symbol: MAR
  source: hgnc_alias_symbol
  hgnc_symbol: IRF1
  entrez: '3659'
- word: Mch
  symbol: MCH
  source: hgnc_alias_symbol
  hgnc_symbol: PMCH
  entrez: '5367'
- word: Ac
  symbol: AC
  source: bioentities_symbol
  hgnc_symbol: ADCY6
  entrez: '112'
- word: HS-Co
  symbol: HSCO
  source: hgnc_alias_symbol
  hgnc_symbol: ETHE1
  entrez: '23474'
chemicals:
- word: Nitrate
  source: MESH
  identifier: D009566
diseases: []
---
