---
figid: PMC5688106__41598_2017_15923_Fig3_HTML
figlink: /pmc/articles/PMC5688106/figure/Fig3/
number: Figure 3
caption: 'Effect of SAT1/2, SAT3 and CS3 gene silencing and L-cysteine depletion on
  sulfur-containing amino acid metabolism. Levels of metabolites extracted from SAT1/2,
  SAT3 and CS gene silenced (SAT1/2gs, SAT3gs, and CSgs) and control (harboring plasmid
  psAP2G) strains cultured under normal and L-cysteine lacking (CD) BI-S-33 medium
  conditions is shown. Data shown are the means ± standard deviations of three biological
  replicates. In the metabolic pathway schemes, solid arrows represent the steps catalyzed
  by the enzymes whose encoding genes are present in the E. histolytica genome, whereas
  an arrow with a dashed line indicates those likely absent in the genome or not yet
  identified so far. Abbreviations: RPA, relative peak area; 3PGA, 3-phosphoglycerate;
  SAH, S-adenosylhomocysteine; PGDH, phosphoglycerate dehydrogenase (EC 1.1.1.95);
  PSAT, phosphoserine aminotransferase (EC 2.6.1.52); PSP, phosphoserine phosphatase
  (EC 3.1.3.3); MGL, methionine γ-lyase (L-methioninase, EC 4.4.1.11); SAT, serine
  O-acetyltransferase (EC 2.3.1.30); CS, cysteine synthase (O-acetyl-L-serine sulfhydrylase,
  EC 2.5.1.47); MAT, methionine adenosyltransferase (S-adenosyl-L-methionine synthetase,
  EC 2.5.1.6); MT, various methyltransferases (EC 2.1.1.X); and SAHH, adenosylhomocysteinase
  (S-adenosyl-L-homocysteine hydrolase, EC 3.3.1.1).'
pmcid: PMC5688106
papertitle: Genetic, metabolomic and transcriptomic analyses of the de novo L-cysteine
  biosynthetic pathway in the enteric protozoan parasite Entamoeba histolytica.
reftext: Ghulam Jeelani, et al. Sci Rep. 2017;7:15649.
pmc_ranked_result_index: '66908'
pathway_score: 0.4386863
filename: 41598_2017_15923_Fig3_HTML.jpg
figtitle: Effect of SAT1/2, SAT3 and CS3 gene silencing and L-cysteine depletion on
  sulfur-containing amino acid metabolism
year: '2017'
organisms: Homo sapiens
ndex: 2aa7f974-de9b-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5688106__41598_2017_15923_Fig3_HTML.html
  '@type': Dataset
  description: 'Effect of SAT1/2, SAT3 and CS3 gene silencing and L-cysteine depletion
    on sulfur-containing amino acid metabolism. Levels of metabolites extracted from
    SAT1/2, SAT3 and CS gene silenced (SAT1/2gs, SAT3gs, and CSgs) and control (harboring
    plasmid psAP2G) strains cultured under normal and L-cysteine lacking (CD) BI-S-33
    medium conditions is shown. Data shown are the means ± standard deviations of
    three biological replicates. In the metabolic pathway schemes, solid arrows represent
    the steps catalyzed by the enzymes whose encoding genes are present in the E.
    histolytica genome, whereas an arrow with a dashed line indicates those likely
    absent in the genome or not yet identified so far. Abbreviations: RPA, relative
    peak area; 3PGA, 3-phosphoglycerate; SAH, S-adenosylhomocysteine; PGDH, phosphoglycerate
    dehydrogenase (EC 1.1.1.95); PSAT, phosphoserine aminotransferase (EC 2.6.1.52);
    PSP, phosphoserine phosphatase (EC 3.1.3.3); MGL, methionine γ-lyase (L-methioninase,
    EC 4.4.1.11); SAT, serine O-acetyltransferase (EC 2.3.1.30); CS, cysteine synthase
    (O-acetyl-L-serine sulfhydrylase, EC 2.5.1.47); MAT, methionine adenosyltransferase
    (S-adenosyl-L-methionine synthetase, EC 2.5.1.6); MT, various methyltransferases
    (EC 2.1.1.X); and SAHH, adenosylhomocysteinase (S-adenosyl-L-homocysteine hydrolase,
    EC 3.3.1.1).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - RPA2
  - PSPH
  - MGLL
  - RPA3
  - PHGDH
  - MAT1A
  - RPA1
  - SAT1
  - AHCY
  - CS
  - ACSM3
  - Methanethiol
  - L-Cystathionine
  - L-Cysteine
  - S-Methyl cysteine
  - L-Cystine
  - Glucose
  - Homocysteine
  - L-Methionine
  - S-Adenosylmethionine
  - Serine
  - Sulfide
genes:
- word: RPA/
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA2
  entrez: '6118'
- word: PSP
  symbol: PSP
  source: hgnc_prev_symbol
  hgnc_symbol: PSPH
  entrez: '5723'
- word: MGL
  symbol: MGL
  source: hgnc_alias_symbol
  hgnc_symbol: MGLL
  entrez: '11343'
- word: RPA/
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA3
  entrez: '6119'
- word: PGDH
  symbol: PGDH
  source: hgnc_alias_symbol
  hgnc_symbol: PHGDH
  entrez: '26227'
- word: MAT
  symbol: MAT
  source: hgnc_alias_symbol
  hgnc_symbol: MAT1A
  entrez: '4143'
- word: RPA/
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA1
  entrez: '6117'
- word: SAT
  symbol: SAT
  source: hgnc_prev_symbol
  hgnc_symbol: SAT1
  entrez: '6303'
- word: SAHH
  symbol: SAHH
  source: hgnc_alias_symbol
  hgnc_symbol: AHCY
  entrez: '191'
- word: SAT1/2gs
  symbol: SAT1
  source: hgnc_symbol
  hgnc_symbol: SAT1
  entrez: '6303'
- word: CS
  symbol: CS
  source: hgnc_symbol
  hgnc_symbol: CS
  entrez: '1431'
- word: SAH
  symbol: SAH
  source: hgnc_prev_symbol
  hgnc_symbol: ACSM3
  entrez: '6296'
chemicals:
- word: Methanethiol
  source: MESH
  identifier: C005231
- word: L-Cystathionine
  source: MESH
  identifier: D003540
- word: L-Cysteine
  source: MESH
  identifier: D003545
- word: S-Methyl cysteine
  source: MESH
  identifier: D003545
- word: L-Cystine
  source: MESH
  identifier: D003553
- word: Glucose
  source: MESH
  identifier: D005947
- word: Homocysteine
  source: MESH
  identifier: D006710
- word: L-Methionine
  source: MESH
  identifier: D008715
- word: S-Adenosylmethionine
  source: MESH
  identifier: D012436
- word: Serine
  source: MESH
  identifier: D012694
- word: Sulfide
  source: MESH
  identifier: D013440
diseases: []
---
