---
figid: PMC523366__pp1363043f05
figlink: /pmc/articles/PMC523366/figure/fig5/
number: Figure 5
caption: 'Metabolic flux map of primary and intermediate metabolic pathways in developing
  soybean cotyledons cultured on Suc (10% [w/w] U-13C) and Gln. Fluxes are proportional
  to arrow widths. (Fluxes less than 0.4 μmol d−1 cotyledon−1 are shown with lines
  of lowest visible thickness.) Arrows indicate the direction of net flux. A complete
  numerical listing of estimated flux values and reaction reversibilities is provided
  in . Intracellular metabolites are shown in white ovals, and gray ovals show sink
  metabolites (proteinogenic amino acids, polysaccharides, etc.). Metabolites taken
  up from/secreted into the medium are shown in blue ovals. Metabolic pathways are
  color coded as follows: dark red, glycolysis and Suc metabolism; pale blue, pentose
  phosphate pathway; orange, TCA cycle; blue-gray, pyruvate dehydrogenase link; mauve,
  anaplerotic fluxes; dark yellow, glyoxylate shunt; green, Glu metabolism, GABA shunt,
  and associated intercompartmental transport fluxes; gray, fluxes toward biomass
  synthesis; and black, all intercompartmental transport fluxes except those involved
  in Gln metabolism and GABA shunt. Abbreviations for intracellular metabolites not
  defined in the text are as follows: P5P, pentose-5-P; S7P, sedoheptulose-7-P; E4P,
  erythrose-4-P; acetCoA, acetyl CoA; iCit, isocitrate; aKG, α-ketoglutarate; Scn,
  succinate; SSA, succinic semi-aldehyde; and GOx, glyoxylate. Sink metabolite abbreviations
  are as follows: Psac, polysaccharides; Nuc, carbon skeleton of nucleotides; and
  Sta, starch. Asp and Asn are denoted together as Asx. Glu and Gln are denoted together
  as Glx. F6P and T3P appear at two different locations each in the cytosol and plastid,
  to avoid confusing intersections of lines. Each flux is assigned a short name based
  on the name of the gene encoding one of the metabolic reactions represented by it.
  Intracellular metabolites and fluxes with a superscript are located in specific
  subcellular compartments: c, cytosol; p, plastid; and m, mitochondrion. If a flux
  has no superscript, its compartmentation could not be unambiguously determined (such
  as gap, eno, and pyk, and some fluxes toward biosynthesis).'
pmcid: PMC523366
papertitle: Quantification of Compartmented Metabolic Fluxes in Developing Soybean
  Embryos by Employing Biosynthetically Directed Fractional 13C Labeling, Two-Dimensional
  [13C, 1H] Nuclear Magnetic Resonance, and Comprehensive Isotopomer Balancing.
reftext: Ganesh Sriram, et al. Plant Physiol. 2004 Oct;136(2):3043-3057.
pmc_ranked_result_index: '72546'
pathway_score: 0.9391505
filename: pp1363043f05.jpg
figtitle: Metabolic flux map of primary and intermediate metabolic pathways in developing
  soybean cotyledons cultured on Suc (10% [w/w] U-13C) and Gln
year: '2004'
organisms: Homo sapiens
ndex: e5e2bbc5-dee5-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC523366__pp1363043f05.html
  '@type': Dataset
  description: 'Metabolic flux map of primary and intermediate metabolic pathways
    in developing soybean cotyledons cultured on Suc (10% [w/w] U-13C) and Gln. Fluxes
    are proportional to arrow widths. (Fluxes less than 0.4 μmol d−1 cotyledon−1 are
    shown with lines of lowest visible thickness.) Arrows indicate the direction of
    net flux. A complete numerical listing of estimated flux values and reaction reversibilities
    is provided in . Intracellular metabolites are shown in white ovals, and gray
    ovals show sink metabolites (proteinogenic amino acids, polysaccharides, etc.).
    Metabolites taken up from/secreted into the medium are shown in blue ovals. Metabolic
    pathways are color coded as follows: dark red, glycolysis and Suc metabolism;
    pale blue, pentose phosphate pathway; orange, TCA cycle; blue-gray, pyruvate dehydrogenase
    link; mauve, anaplerotic fluxes; dark yellow, glyoxylate shunt; green, Glu metabolism,
    GABA shunt, and associated intercompartmental transport fluxes; gray, fluxes toward
    biomass synthesis; and black, all intercompartmental transport fluxes except those
    involved in Gln metabolism and GABA shunt. Abbreviations for intracellular metabolites
    not defined in the text are as follows: P5P, pentose-5-P; S7P, sedoheptulose-7-P;
    E4P, erythrose-4-P; acetCoA, acetyl CoA; iCit, isocitrate; aKG, α-ketoglutarate;
    Scn, succinate; SSA, succinic semi-aldehyde; and GOx, glyoxylate. Sink metabolite
    abbreviations are as follows: Psac, polysaccharides; Nuc, carbon skeleton of nucleotides;
    and Sta, starch. Asp and Asn are denoted together as Asx. Glu and Gln are denoted
    together as Glx. F6P and T3P appear at two different locations each in the cytosol
    and plastid, to avoid confusing intersections of lines. Each flux is assigned
    a short name based on the name of the gene encoding one of the metabolic reactions
    represented by it. Intracellular metabolites and fluxes with a superscript are
    located in specific subcellular compartments: c, cytosol; p, plastid; and m, mitochondrion.
    If a flux has no superscript, its compartmentation could not be unambiguously
    determined (such as gap, eno, and pyk, and some fluxes toward biosynthesis).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - DLD
  - THRB
  - DLAT
  - PDHA1
  - MAL
  - MET
  - THRA
  - BLOC1S2
  - BIRC3
  - ENO2
  - NUCB1
  - PDHA2
  - ENO1
  - GLYAT
  - ENO3
  - GBA3
  - PACRG
  - AKR1B10
  - EMD
  - PRG4
  - SDHB
  - RASA1
  - A4GALT
  - TYRP1
  - ENO4
  - PAEP
  - H6PD
  - PDHB
  - PDHX
  - TYR
  - PHKA2
  - ACSM3
  - Ala
  - Arg
  - Cys
  - fructose F6PEso
  - GABA
  - Leu
  - Lys
  - Met
  - Phe
  - Ser
  - Thr
  - Trp
  - Tyr
  - Val
genes:
- word: pdh
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: DLD
  entrez: '1738'
- word: Thr
  symbol: THR
  source: bioentities_symbol
  hgnc_symbol: THRB
  entrez: '7068'
- word: pdh
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: DLAT
  entrez: '1737'
- word: pdh
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHA1
  entrez: '5160'
- word: Mal
  symbol: MAL
  source: hgnc_symbol
  hgnc_symbol: MAL
  entrez: '4118'
- word: (Met
  symbol: MET
  source: hgnc_symbol
  hgnc_symbol: MET
  entrez: '4233'
- word: Thr
  symbol: THR
  source: bioentities_symbol
  hgnc_symbol: THRA
  entrez: '7067'
- word: blos2,Cys
  symbol: BLOS2
  source: hgnc_alias_symbol
  hgnc_symbol: BLOC1S2
  entrez: '282991'
- word: malT2
  symbol: MALT2
  source: hgnc_alias_symbol
  hgnc_symbol: BIRC3
  entrez: '330'
- word: eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO2
  entrez: '2026'
- word: (Nuc
  symbol: NUC
  source: hgnc_alias_symbol
  hgnc_symbol: NUCB1
  entrez: '4924'
- word: pdh
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHA2
  entrez: '5161'
- word: eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO1
  entrez: '2023'
- word: gat
  symbol: GAT
  source: hgnc_alias_symbol
  hgnc_symbol: GLYAT
  entrez: '10249'
- word: eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO3
  entrez: '2027'
- word: Gluc
  symbol: GLUC
  source: hgnc_alias_symbol
  hgnc_symbol: GBA3
  entrez: '57733'
- word: Glup
  symbol: Glup
  source: hgnc_alias_symbol
  hgnc_symbol: PACRG
  entrez: '135138'
- word: (His.
  symbol: HIS
  source: hgnc_alias_symbol
  hgnc_symbol: AKR1B10
  entrez: '57016'
- word: Sta
  symbol: STA
  source: hgnc_alias_symbol
  hgnc_symbol: EMD
  entrez: '2010'
- word: SZP
  symbol: SZP
  source: hgnc_alias_symbol
  hgnc_symbol: PRG4
  entrez: '10216'
- word: sdh
  symbol: SDH
  source: hgnc_prev_symbol
  hgnc_symbol: SDHB
  entrez: '6390'
- word: gap
  symbol: GAP
  source: hgnc_alias_symbol
  hgnc_symbol: RASA1
  entrez: '5921'
- word: pk
  symbol: P(k)
  source: hgnc_alias_symbol
  hgnc_symbol: A4GALT
  entrez: '53947'
- word: Trp
  symbol: TRP
  source: hgnc_alias_symbol
  hgnc_symbol: TYRP1
  entrez: '7306'
- word: eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO4
  entrez: '387712'
- word: (PEP
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
- word: gdh
  symbol: GDH
  source: hgnc_prev_symbol
  hgnc_symbol: H6PD
  entrez: '9563'
- word: pdh
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHB
  entrez: '5162'
- word: pdh
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHX
  entrez: '8050'
- word: Tyr
  symbol: TYR
  source: hgnc_symbol
  hgnc_symbol: TYR
  entrez: '7299'
- word: pyk
  symbol: PYK
  source: hgnc_prev_symbol
  hgnc_symbol: PHKA2
  entrez: '5256'
- word: sah
  symbol: SAH
  source: hgnc_prev_symbol
  hgnc_symbol: ACSM3
  entrez: '6296'
chemicals:
- word: Ala
  source: MESH
  identifier: D000409
- word: Arg
  source: MESH
  identifier: D001120
- word: Cys
  source: MESH
  identifier: D003545
- word: fructose F6PEso
  source: MESH
  identifier: D005632
- word: GABA
  source: MESH
  identifier: D005680
- word: Leu
  source: MESH
  identifier: D007930
- word: Lys
  source: MESH
  identifier: D008239
- word: Met
  source: MESH
  identifier: D008715
- word: Phe
  source: MESH
  identifier: D010649
- word: Ser
  source: MESH
  identifier: D012694
- word: Thr
  source: MESH
  identifier: D013912
- word: Trp
  source: MESH
  identifier: D014364
- word: Tyr
  source: MESH
  identifier: D014443
- word: Val
  source: MESH
  identifier: D014633
diseases: []
---
