---
figid: PMC5743771__nihms894598f1
figlink: /pmc/articles/PMC5743771/figure/F1/
number: Figure 1
caption: Genomic mechanisms, including gene transactivation (A) and transrepression
  (B), are the best characterized means of LXR activity in cells, whereas recently
  described non-genomic mechanisms (C) remain somewhat less well understood. In transactivation,
  two general models, represented by induction of the target genes ATP Binding Cassette
  (ABC)A1 and ABCG1, have been identified. For ABCA1, it is thought that LXR bound
  to promoter LXR response elements (LXREs) in the steady state tonically represses
  gene expression by recruiting co-repressors such as nuclear receptor co-repressor
  1 (NCoR). Upon LXR ligand binding, co-repressors are shed in exchange for co-activators,
  driving gene expression. More recent studies suggest that for a majority of LXR
  targets including ABCG1, LXRs bind to LXREs only after ligand-induced activation
  and histone demethylation. In both of these models, LXR binds LXREs directly in
  the form of a heterodimer with retinoid X receptor (RXR). In the case of transrepression
  of pro-inflammatory genes (B), two general mechanisms of LXR action have been identified
  in macrophages/hepatocytes and astrocytes. In both cases, lysines in the ligand-binding
  domain of LXR are SUMOylated after LXR ligation. In the case of LPS-stimulated macrophages
  or cytokine-stimulated hepatocytes, SUMOylated LXR binds in monomeric form to a
  multimolecular co-repressor complex, inhibiting release of co-repressors from gene
  promoters, thereby blocking gene expression. In astrocytes, SUMOylated LXRs inhibit
  transcription by blocking the binding of signal transducer and activator of transcription
  1 (STAT1) to promoters. Recently, examples of non-genomic (extranuclear) LXRβ action
  have been identified, as shown in panel C. In colon cancer cells, cytoplasmic LXRβ
  drives NLRP3- and caspase-1-dependent pyroptotic cell death by inducing pannexin-1-mediated
  ATP release. In platelets, LXRβ inhibits kinase signaling to degranulation and aggregation
  downstream of the GPVI receptor. In endothelial cells, lipid raft-localized LXRβ
  mediates a signaling pathway to cell migration involving estrogen receptor (ER)-α
  and the kinase AKT. The relative importance of genomic and non-genomic mechanisms
  remains poorly understood. The distinct binding partners of LXRs in these various
  contexts, however, creates the exciting potential for targeted, potentially cell
  type- and gene-specific, pharmacologic interventions. eNOS, endothelial nitric oxide
  synthase; GPVI, glycoprotein VI; GPS2, G-protein pathway suppressor 2; HDAC, histone
  deacetylase; iNOS, inducible nitric oxide synthase; P2X7, P2X purinoceptor 7; PIAS1,
  protein inhibitor of STAT1; SAA, serum amyloid A; SUMO, small ubiquitin-like modifier;
  TBLR, transducin beta-like 1X-related protein 1; UBC9, SUMO-conjugating enzyme UBC9.
pmcid: PMC5743771
papertitle: The Challenges and Promise of Targeting the Liver X Receptors for Treatment
  of Inflammatory Disease.
reftext: Michael B. Fessler. Pharmacol Ther. ;181:1-12.
pmc_ranked_result_index: '153886'
pathway_score: 0.8775026
filename: nihms894598f1.jpg
figtitle: Challenges and Promise of Targeting the Liver X Receptors for Treatment
  of Inflammatory Disease
year: ''
organisms: Homo sapiens
ndex: 9284c65c-df1a-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5743771__nihms894598f1.html
  '@type': Dataset
  description: Genomic mechanisms, including gene transactivation (A) and transrepression
    (B), are the best characterized means of LXR activity in cells, whereas recently
    described non-genomic mechanisms (C) remain somewhat less well understood. In
    transactivation, two general models, represented by induction of the target genes
    ATP Binding Cassette (ABC)A1 and ABCG1, have been identified. For ABCA1, it is
    thought that LXR bound to promoter LXR response elements (LXREs) in the steady
    state tonically represses gene expression by recruiting co-repressors such as
    nuclear receptor co-repressor 1 (NCoR). Upon LXR ligand binding, co-repressors
    are shed in exchange for co-activators, driving gene expression. More recent studies
    suggest that for a majority of LXR targets including ABCG1, LXRs bind to LXREs
    only after ligand-induced activation and histone demethylation. In both of these
    models, LXR binds LXREs directly in the form of a heterodimer with retinoid X
    receptor (RXR). In the case of transrepression of pro-inflammatory genes (B),
    two general mechanisms of LXR action have been identified in macrophages/hepatocytes
    and astrocytes. In both cases, lysines in the ligand-binding domain of LXR are
    SUMOylated after LXR ligation. In the case of LPS-stimulated macrophages or cytokine-stimulated
    hepatocytes, SUMOylated LXR binds in monomeric form to a multimolecular co-repressor
    complex, inhibiting release of co-repressors from gene promoters, thereby blocking
    gene expression. In astrocytes, SUMOylated LXRs inhibit transcription by blocking
    the binding of signal transducer and activator of transcription 1 (STAT1) to promoters.
    Recently, examples of non-genomic (extranuclear) LXRβ action have been identified,
    as shown in panel C. In colon cancer cells, cytoplasmic LXRβ drives NLRP3- and
    caspase-1-dependent pyroptotic cell death by inducing pannexin-1-mediated ATP
    release. In platelets, LXRβ inhibits kinase signaling to degranulation and aggregation
    downstream of the GPVI receptor. In endothelial cells, lipid raft-localized LXRβ
    mediates a signaling pathway to cell migration involving estrogen receptor (ER)-α
    and the kinase AKT. The relative importance of genomic and non-genomic mechanisms
    remains poorly understood. The distinct binding partners of LXRs in these various
    contexts, however, creates the exciting potential for targeted, potentially cell
    type- and gene-specific, pharmacologic interventions. eNOS, endothelial nitric
    oxide synthase; GPVI, glycoprotein VI; GPS2, G-protein pathway suppressor 2; HDAC,
    histone deacetylase; iNOS, inducible nitric oxide synthase; P2X7, P2X purinoceptor
    7; PIAS1, protein inhibitor of STAT1; SAA, serum amyloid A; SUMO, small ubiquitin-like
    modifier; TBLR, transducin beta-like 1X-related protein 1; UBC9, SUMO-conjugating
    enzyme UBC9.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - IL18
  - IRF1
  - NR1H3
  - UBE2I
  - AKT1
  - SAA1
  - HDAC2
  - HDAC8
  - HDAC3
  - HDAC4
  - SAA2
  - NR1H2
  - RXRB
  - HDAC1
  - RXRG
  - RXRA
  - JUND
  - FOSL2
  - JUN
  - JUNB
  - ABCG1
  - AKT3
  - NFKB1
  - TBL1XR1
  - FOS
  - FOSB
  - STAT1
  - AKT2
  - ABCA1
  - SAA4
  - GP6
  - NCOR2
  - NLRP3
  - GPS2
  - PIAS1
  - SYK
  - SUMO1
  - P2RX7
  - NCOR1
  - TBL1Y
  - SUMO2
  - NOS3
  - FOSL1
  - ESR1
  - ATP
  - Pyroptosin
  - Calcium
  - Colon Cancer
genes:
- word: IL-18
  symbol: IL-18
  source: hgnc_alias_symbol
  hgnc_symbol: IL18
  entrez: '3606'
- word: IRF1
  symbol: IRF1
  source: hgnc_symbol
  hgnc_symbol: IRF1
  entrez: '3659'
- word: LXRA
  symbol: LXR-a
  source: hgnc_alias_symbol
  hgnc_symbol: NR1H3
  entrez: '10062'
- word: UBC9
  symbol: UBC9
  source: hgnc_alias_symbol
  hgnc_symbol: UBE2I
  entrez: '7329'
- word: AKT
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT1
  entrez: '207'
- word: SAA
  symbol: SAA
  source: bioentities_symbol
  hgnc_symbol: SAA1
  entrez: '6288'
- word: HDACI
  symbol: HDAC_I
  source: bioentities_symbol
  hgnc_symbol: HDAC2
  entrez: '3066'
- word: HDACI
  symbol: HDAC_I
  source: bioentities_symbol
  hgnc_symbol: HDAC8
  entrez: '55869'
- word: HDACI
  symbol: HDAC_I
  source: bioentities_symbol
  hgnc_symbol: HDAC3
  entrez: '8841'
- word: HDACA
  symbol: HDAC-A
  source: hgnc_alias_symbol
  hgnc_symbol: HDAC4
  entrez: '9759'
- word: SAA
  symbol: SAA
  source: bioentities_symbol
  hgnc_symbol: SAA2
  entrez: '6289'
- word: LXRB
  symbol: LXR-b
  source: hgnc_alias_symbol
  hgnc_symbol: NR1H2
  entrez: '7376'
- word: HDAC4
  symbol: HDAC4
  source: hgnc_symbol
  hgnc_symbol: HDAC4
  entrez: '9759'
- word: RXR
  symbol: RXR
  source: bioentities_symbol
  hgnc_symbol: RXRB
  entrez: '6257'
- word: HDACI
  symbol: HDAC_I
  source: bioentities_symbol
  hgnc_symbol: HDAC1
  entrez: '3065'
- word: RXR
  symbol: RXR
  source: bioentities_symbol
  hgnc_symbol: RXRG
  entrez: '6258'
- word: RXR
  symbol: RXR
  source: bioentities_symbol
  hgnc_symbol: RXRA
  entrez: '6256'
- word: AP-1
  symbol: AP1
  source: bioentities_symbol
  hgnc_symbol: JUND
  entrez: '3727'
- word: AP-1
  symbol: AP1
  source: bioentities_symbol
  hgnc_symbol: FOSL2
  entrez: '2355'
- word: AP-1
  symbol: AP1
  source: bioentities_symbol
  hgnc_symbol: JUN
  entrez: '3725'
- word: AP-1
  symbol: AP1
  source: bioentities_symbol
  hgnc_symbol: JUNB
  entrez: '3726'
- word: ABCG1
  symbol: ABCG1
  source: hgnc_symbol
  hgnc_symbol: ABCG1
  entrez: '9619'
- word: AKT
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT3
  entrez: '10000'
- word: NF-KB
  symbol: NF-kappaB
  source: hgnc_alias_symbol
  hgnc_symbol: NFKB1
  entrez: '4790'
- word: TBLR1
  symbol: TBLR1
  source: hgnc_alias_symbol
  hgnc_symbol: TBL1XR1
  entrez: '79718'
- word: AP-1
  symbol: AP1
  source: bioentities_symbol
  hgnc_symbol: FOS
  entrez: '2353'
- word: AP-1
  symbol: AP1
  source: bioentities_symbol
  hgnc_symbol: FOSB
  entrez: '2354'
- word: STAT1
  symbol: STAT1
  source: hgnc_symbol
  hgnc_symbol: STAT1
  entrez: '6772'
- word: AKT
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT2
  entrez: '208'
- word: ABCA1
  symbol: ABCA1
  source: hgnc_symbol
  hgnc_symbol: ABCA1
  entrez: '19'
- word: SAA
  symbol: SAA
  source: bioentities_symbol
  hgnc_symbol: SAA4
  entrez: '6291'
- word: GPVI
  symbol: GPVI
  source: hgnc_alias_symbol
  hgnc_symbol: GP6
  entrez: '51206'
- word: NCOR
  symbol: NCOR
  source: bioentities_symbol
  hgnc_symbol: NCOR2
  entrez: '9612'
- word: NLRP3
  symbol: NLRP3
  source: hgnc_symbol
  hgnc_symbol: NLRP3
  entrez: '114548'
- word: GPS2
  symbol: GPS2
  source: hgnc_symbol
  hgnc_symbol: GPS2
  entrez: '2874'
- word: PIAS1
  symbol: PIAS1
  source: hgnc_symbol
  hgnc_symbol: PIAS1
  entrez: '8554'
- word: Syk
  symbol: SYK
  source: hgnc_symbol
  hgnc_symbol: SYK
  entrez: '6850'
- word: SUMO1
  symbol: SUMO1
  source: hgnc_symbol
  hgnc_symbol: SUMO1
  entrez: '7341'
- word: P2X7
  symbol: P2X7
  source: hgnc_alias_symbol
  hgnc_symbol: P2RX7
  entrez: '5027'
- word: NCOR
  symbol: NCOR
  source: bioentities_symbol
  hgnc_symbol: NCOR1
  entrez: '9611'
- word: TBL1
  symbol: TBL1
  source: hgnc_alias_symbol
  hgnc_symbol: TBL1Y
  entrez: '90665'
- word: SUMO2
  symbol: SUMO2
  source: hgnc_symbol
  hgnc_symbol: SUMO2
  entrez: '6613'
- word: ENOS
  symbol: eNOS
  source: hgnc_alias_symbol
  hgnc_symbol: NOS3
  entrez: '4846'
- word: AP-1
  symbol: AP1
  source: bioentities_symbol
  hgnc_symbol: FOSL1
  entrez: '8061'
- word: ERa
  symbol: ER-alpha
  source: hgnc_alias_symbol
  hgnc_symbol: ESR1
  entrez: '2099'
chemicals:
- word: ATP
  source: MESH
  identifier: D000255
- word: Pyroptosin
  source: ''
  identifier: ''
- word: Calcium
  source: MESH
  identifier: D002118
diseases:
- word: Colon Cancer
  source: MESH
  identifier: D015179
---
