---
figid: PMC5704294__oc-2017-00224x_0001
figlink: /pmc/articles/PMC5704294/figure/fig1/
number: F1
caption: Proposed activation pathway and domain structure of Nrf1. (A) (1) Full length
  Nrf1 is glycosylated in the ER lumen (Pro-Nrf1) and subsequently retrotranslocated
  to the cytosol by VCP/p97. (2) ER membrane-bound Pro-Nrf1 is de-N-glycosylated by
  NGLY1. (3) The protease DDI2 cleaves Nrf1 between W103 and L104 and releases the
  active p95 form into the cytosol. (4) Nrf1 is immediately degraded by the proteasome
  and thus maintained at low levels in the cell. (5) In cells with insufficient proteasome
  capacity due to chemical inhibition or an overload of misfolded proteins, active
  Nrf1 accumulates and migrates to the nucleus, where it heterodimerizes with cofactors
  (small Maf proteins), binds to chromosomal targets, and activates the synthesis
  of PSMs. (B) Domain structure of Nrf1 with ER transmembrane domain, site of DDI2
  proteolysis, and possible N-glycosylation sites labeled in red. The N-terminal domain
  (NTD) contains the transmembrane sequence that anchors Nrf1 within the ER membrane
  and the proteolytic cleavage site for DDI2 between W103 and L104. The transactivation
  domain (TAD) comprises the two acidic domains (AD1, AD2) and the Asn/Ser/Thr-rich
  region (NST) with eight predicted N-glycosylation sites (orange). The serine-rich
  region (SR) was found to be multiply O-GlcNAcylated, and its glycosylation status
  dictates the ubiquitination of the transcription factor. The Nrf2-ECH homology 6-like
  domain (Neh6L) is conserved in two relatives of Nrf1, Nrf2 and Nrf3. The DNA binding
  domain comprises the cap ’n’ collar (CNC) and the basic leucine zipper domain (bZIP),
  which enable heterodimerization with Maf proteins before binding to the DNA. The
  C-terminal domain (CTD) also contributes to transcription factor activity.
pmcid: PMC5704294
papertitle: Inhibition of NGLY1 Inactivates the Transcription Factor Nrf1 and Potentiates
  Proteasome Inhibitor Cytotoxicity.
reftext: Frederick M. Tomlin, et al. ACS Cent Sci. 2017 Nov 22;3(11):1143-1155.
pmc_ranked_result_index: '209771'
pathway_score: 0.9063521
filename: oc-2017-00224x_0001.jpg
figtitle: Proposed activation pathway and domain structure of Nrf1
year: '2017'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5704294__oc-2017-00224x_0001.html
  '@type': Dataset
  description: Proposed activation pathway and domain structure of Nrf1. (A) (1) Full
    length Nrf1 is glycosylated in the ER lumen (Pro-Nrf1) and subsequently retrotranslocated
    to the cytosol by VCP/p97. (2) ER membrane-bound Pro-Nrf1 is de-N-glycosylated
    by NGLY1. (3) The protease DDI2 cleaves Nrf1 between W103 and L104 and releases
    the active p95 form into the cytosol. (4) Nrf1 is immediately degraded by the
    proteasome and thus maintained at low levels in the cell. (5) In cells with insufficient
    proteasome capacity due to chemical inhibition or an overload of misfolded proteins,
    active Nrf1 accumulates and migrates to the nucleus, where it heterodimerizes
    with cofactors (small Maf proteins), binds to chromosomal targets, and activates
    the synthesis of PSMs. (B) Domain structure of Nrf1 with ER transmembrane domain,
    site of DDI2 proteolysis, and possible N-glycosylation sites labeled in red. The
    N-terminal domain (NTD) contains the transmembrane sequence that anchors Nrf1
    within the ER membrane and the proteolytic cleavage site for DDI2 between W103
    and L104. The transactivation domain (TAD) comprises the two acidic domains (AD1,
    AD2) and the Asn/Ser/Thr-rich region (NST) with eight predicted N-glycosylation
    sites (orange). The serine-rich region (SR) was found to be multiply O-GlcNAcylated,
    and its glycosylation status dictates the ubiquitination of the transcription
    factor. The Nrf2-ECH homology 6-like domain (Neh6L) is conserved in two relatives
    of Nrf1, Nrf2 and Nrf3. The DNA binding domain comprises the cap ’n’ collar (CNC)
    and the basic leucine zipper domain (bZIP), which enable heterodimerization with
    Maf proteins before binding to the DNA. The C-terminal domain (CTD) also contributes
    to transcription factor activity.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - NGLY1
  - MAF
  - APOE
  - NRF1
  - DZIP1
  - APP
  - DDI2
genes:
- word: NGLY1
  symbol: NGLY1
  source: hgnc_symbol
  hgnc_symbol: NGLY1
  entrez: '55768'
- word: Maf
  symbol: MAF
  source: hgnc_symbol
  hgnc_symbol: MAF
  entrez: '4094'
- word: AD2
  symbol: AD2
  source: hgnc_prev_symbol
  hgnc_symbol: APOE
  entrez: '348'
- word: Nrf1
  symbol: NRF1
  source: hgnc_symbol
  hgnc_symbol: NRF1
  entrez: '4899'
- word: DZIP
  symbol: DZIP
  source: hgnc_alias_symbol
  hgnc_symbol: DZIP1
  entrez: '22873'
- word: AD1
  symbol: AD1
  source: hgnc_prev_symbol
  hgnc_symbol: APP
  entrez: '351'
- word: DDI2
  symbol: DDI2
  source: hgnc_symbol
  hgnc_symbol: DDI2
  entrez: '84301'
chemicals: []
diseases: []
figid_alias: PMC5704294__F1
redirect_from: /figures/PMC5704294__F1
figtype: Figure
---
