---
figid: PMC5428490__41598_2017_438_Fig2_HTML
figlink: /pmc/articles/PMC5428490/figure/Fig2/
number: Figure 2
caption: Transcriptome pattern differs between two oil biosynthesis tissues in oil
  palm. (A) The ratio of up- to down-regulated genes when compared the transcription
  level of Tenera against Dura in mesocarp and endosperm. The gene expression level
  of their respective key stage (2.5 MAF for endosperm, 3.5 MAF for mesocarp) were
  used to compare. Assumed at least 2 folds change as up- or down-regulated genes.
  Whole transcriptome and some specific categorized pathways are listed out. Plastid
  FAS refers to the pathway of fatty acid synthesis in plastid. Noticed that genes
  of lipid related or plastid FAS are highly upregulated in mesocarp and endosperm
  of Tenera respectively. (B) Transcripts patterns for enzymes involved in glysolysis
  reactions for Dura and Tenera. Values in red indicate the fold increase in reads
  of mesocarp of Tenera to that of Dura at 3.5 MAF. Values in blue indicate the fold
  increase in reads of endosperm of Tenera to that of Dura at 2.5 MAF. Samples collected
  from different tissues for Dura, Pisifera and Tenera (#53) were used for transcriptome
  analysis. Orange arrow indicates the cytosolic glycolytic process is the major carbon
  flow increased in Tenera compared with Dura both in mesocarp and endosperm, as deduced
  by the fold increase in reads. Light yellow arrows indicate the major increased
  cytosol-to-plastid carbon flow for mesocarp. Light blue arrow indicates the major
  increased cytosol-to-plastid carbon flow for endosperm. (C) Transcripts patterns
  for enzymes involved in plastidial and extraplasidial reactions for Dura and Tenera.
  Values in red indicate the fold increase in reads of mesocarp of Tenera to that
  of Dura at 3.5 MAF. Values in blue indicate the fold increase in reads of endosperm
  of Tenera to that of Dura at 2.5 MAF. Samples collected from different tissues for
  Dura, Pisifera and Tenera (#53) were used for transcriptome analysis. Reads for
  enzymes with multiple isoforms were summed. For details on abbreviations, annotations,
  and transcriptome levels at each samples, see Dataset S1. 16:0, palmitic acid; 18:0,
  stearic acid; 18:1, oleic acid, 18:2, Linoleic acid; 18:3, linolenic acid. For details
  on abbreviations, annotations, and transcription levels at each stage, see Supplemental
  Dataset .
pmcid: PMC5428490
papertitle: Transcriptome and functional analysis reveals hybrid vigor for oil biosynthesis
  in oil palm.
reftext: Jingjing Jin, et al. Sci Rep. 2017;7:439.
pmc_ranked_result_index: '184244'
pathway_score: 0.9093914
filename: 41598_2017_438_Fig2_HTML.jpg
figtitle: Transcriptome pattern differs between two oil biosynthesis tissues in oil
  palm
year: '2017'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5428490__41598_2017_438_Fig2_HTML.html
  '@type': Dataset
  description: Transcriptome pattern differs between two oil biosynthesis tissues
    in oil palm. (A) The ratio of up- to down-regulated genes when compared the transcription
    level of Tenera against Dura in mesocarp and endosperm. The gene expression level
    of their respective key stage (2.5 MAF for endosperm, 3.5 MAF for mesocarp) were
    used to compare. Assumed at least 2 folds change as up- or down-regulated genes.
    Whole transcriptome and some specific categorized pathways are listed out. Plastid
    FAS refers to the pathway of fatty acid synthesis in plastid. Noticed that genes
    of lipid related or plastid FAS are highly upregulated in mesocarp and endosperm
    of Tenera respectively. (B) Transcripts patterns for enzymes involved in glysolysis
    reactions for Dura and Tenera. Values in red indicate the fold increase in reads
    of mesocarp of Tenera to that of Dura at 3.5 MAF. Values in blue indicate the
    fold increase in reads of endosperm of Tenera to that of Dura at 2.5 MAF. Samples
    collected from different tissues for Dura, Pisifera and Tenera (#53) were used
    for transcriptome analysis. Orange arrow indicates the cytosolic glycolytic process
    is the major carbon flow increased in Tenera compared with Dura both in mesocarp
    and endosperm, as deduced by the fold increase in reads. Light yellow arrows indicate
    the major increased cytosol-to-plastid carbon flow for mesocarp. Light blue arrow
    indicates the major increased cytosol-to-plastid carbon flow for endosperm. (C)
    Transcripts patterns for enzymes involved in plastidial and extraplasidial reactions
    for Dura and Tenera. Values in red indicate the fold increase in reads of mesocarp
    of Tenera to that of Dura at 3.5 MAF. Values in blue indicate the fold increase
    in reads of endosperm of Tenera to that of Dura at 2.5 MAF. Samples collected
    from different tissues for Dura, Pisifera and Tenera (#53) were used for transcriptome
    analysis. Reads for enzymes with multiple isoforms were summed. For details on
    abbreviations, annotations, and transcriptome levels at each samples, see Dataset
    S1. 16:0, palmitic acid; 18:0, stearic acid; 18:1, oleic acid, 18:2, Linoleic
    acid; 18:3, linolenic acid. For details on abbreviations, annotations, and transcription
    levels at each stage, see Supplemental Dataset .
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ACACB
  - ACACA
  - ENO1
  - ENO3
  - FH
  - PCYT1B
  - PDHX
  - GPI
  - GAPDH
  - NDUFAB1
  - ENO4
  - GPT2
  - DGAT1
  - PC
  - PPT1
  - ZBTB22
  - KLK15
  - HSD17B12
  - PPAT
  - DLD
  - PDHA1
  - REG3A
  - ENO2
  - AGPAT1
  - PDHB
  - PCSK7
  - A4GALT
  - FBXO3
  - VCAN
  - PDHA2
  - FANCD2
  - DLAT
  - PRKG1
  - LPA
  - PRF1
  - FAS
  - MAT1A
  - Acyl-CoA
  - Glycerol
  - Malonyl-CoA
  - Pyruvate
  - Succinate
genes:
- word: ACC
  symbol: ACC
  source: bioentities_symbol
  hgnc_symbol: ACACB
  entrez: '32'
- word: ACC
  symbol: ACC
  source: bioentities_symbol
  hgnc_symbol: ACACA
  entrez: '31'
- word: ENO
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO1
  entrez: '2023'
- word: ENO
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO3
  entrez: '2027'
- word: Fumarases
  symbol: fumarase
  source: hgnc_alias_symbol
  hgnc_symbol: FH
  entrez: '2271'
- word: (CTb)
  symbol: CTB
  source: hgnc_alias_symbol
  hgnc_symbol: PCYT1B
  entrez: '9468'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHX
  entrez: '8050'
- word: GPI
  symbol: GPI
  source: hgnc_symbol
  hgnc_symbol: GPI
  entrez: '2821'
- word: GAPDH
  symbol: GAPDH
  source: hgnc_symbol
  hgnc_symbol: GAPDH
  entrez: '2597'
- word: ACP
  symbol: ACP
  source: hgnc_alias_symbol
  hgnc_symbol: NDUFAB1
  entrez: '4706'
- word: ENO
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO4
  entrez: '387712'
- word: GPT2
  symbol: GPT2
  source: hgnc_symbol
  hgnc_symbol: GPT2
  entrez: '84706'
- word: DGAT1
  symbol: DGAT1
  source: hgnc_symbol
  hgnc_symbol: DGAT1
  entrez: '8694'
- word: PC
  symbol: PC
  source: hgnc_symbol
  hgnc_symbol: PC
  entrez: '5091'
- word: PPT1
  symbol: PPT1
  source: hgnc_symbol
  hgnc_symbol: PPT1
  entrez: '5538'
- word: Fru
  symbol: fru
  source: hgnc_alias_symbol
  hgnc_symbol: ZBTB22
  entrez: '9278'
- word: ACO
  symbol: ACO
  source: hgnc_alias_symbol
  hgnc_symbol: KLK15
  entrez: '55554'
- word: KAR
  symbol: KAR
  source: hgnc_alias_symbol
  hgnc_symbol: HSD17B12
  entrez: '51144'
- word: GPAT
  symbol: GPAT
  source: hgnc_alias_symbol
  hgnc_symbol: PPAT
  entrez: '5471'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: DLD
  entrez: '1738'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHA1
  entrez: '5160'
- word: PAP
  symbol: PAP
  source: hgnc_prev_symbol
  hgnc_symbol: REG3A
  entrez: '5068'
- word: ENO
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO2
  entrez: '2026'
- word: LPAAT-A
  symbol: LPAAT-alpha
  source: hgnc_alias_symbol
  hgnc_symbol: AGPAT1
  entrez: '10554'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHB
  entrez: '5162'
- word: LPC-
  symbol: LPC
  source: hgnc_alias_symbol
  hgnc_symbol: PCSK7
  entrez: '9159'
- word: PK
  symbol: P(k)
  source: hgnc_alias_symbol
  hgnc_symbol: A4GALT
  entrez: '53947'
- word: FBA
  symbol: FBA
  source: hgnc_alias_symbol
  hgnc_symbol: FBXO3
  entrez: '26273'
- word: PGM
  symbol: PG-M
  source: hgnc_alias_symbol
  hgnc_symbol: VCAN
  entrez: '1462'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHA2
  entrez: '5161'
- word: FAD2
  symbol: FA-D2
  source: hgnc_alias_symbol
  hgnc_symbol: FANCD2
  entrez: '2177'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: DLAT
  entrez: '1737'
- word: PGK
  symbol: PGK
  source: hgnc_alias_symbol
  hgnc_symbol: PRKG1
  entrez: '5592'
- word: (LPA)
  symbol: LPA
  source: hgnc_symbol
  hgnc_symbol: LPA
  entrez: '4018'
- word: PFP
  symbol: PFP
  source: hgnc_alias_symbol
  hgnc_symbol: PRF1
  entrez: '5551'
- word: FAS
  symbol: FAS
  source: hgnc_symbol
  hgnc_symbol: FAS
  entrez: '355'
- word: MAT
  symbol: MAT
  source: hgnc_alias_symbol
  hgnc_symbol: MAT1A
  entrez: '4143'
chemicals:
- word: Acyl-CoA
  source: MESH
  identifier: D000214
- word: Glycerol
  source: MESH
  identifier: D005990
- word: Malonyl-CoA
  source: MESH
  identifier: D008316
- word: Pyruvate
  source: MESH
  identifier: D011773
- word: Succinate
  source: MESH
  identifier: D013386
diseases: []
---
