---
figid: PMC5811630__sys0011821800007
figlink: /pmc/articles/PMC5811630/figure/fig7/
number: FIG 7
caption: Pathway analysis of iron starvation responses. (A and B) Association networks
  for differentially expressed genes, with a P value of ≤0.05 and a minimum of 10
  mapped reads, were generated for the early-cycle response (6 + 3 h BPDL) (A) and
  midcycle response (12 + 3 h BPDL) (B) using STRING-db v.10.5. The thickness of the
  lines connecting the nodes (genes) correlates with confidence of gene association,
  with a minimum confidence cutoff value of 0.7. (C and D) Midcycle (12 + 3 h BPDL
  or 12 + 6 h BPDL) clustered genes were mapped to nucleotide metabolism (C) and acetyl-CoA
  synthesis (D) pathways using KEGGMapper v.2.8. Data corresponding to upregulated
  and downregulated genes in panels C and D are shown with green and red backgrounds,
  respectively. Data corresponding to unchanged genes have a white background.
pmcid: PMC5811630
papertitle: Genomewide Transcriptional Responses of Iron-Starved Chlamydia trachomatis
  Reveal Prioritization of Metabolic Precursor Synthesis over Protein Translation.
reftext: Amanda J. Brinkworth, et al. mSystems. 2018 Jan-Feb;3(1):e00184-17.
pmc_ranked_result_index: '11525'
pathway_score: 0.8878221
filename: sys0011821800007.jpg
figtitle: Iron starvation responses
year: '2018'
organisms: Homo sapiens
ndex: 4154a282-def9-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5811630__sys0011821800007.html
  '@type': Dataset
  description: Pathway analysis of iron starvation responses. (A and B) Association
    networks for differentially expressed genes, with a P value of ≤0.05 and a minimum
    of 10 mapped reads, were generated for the early-cycle response (6 + 3 h BPDL)
    (A) and midcycle response (12 + 3 h BPDL) (B) using STRING-db v.10.5. The thickness
    of the lines connecting the nodes (genes) correlates with confidence of gene association,
    with a minimum confidence cutoff value of 0.7. (C and D) Midcycle (12 + 3 h BPDL
    or 12 + 6 h BPDL) clustered genes were mapped to nucleotide metabolism (C) and
    acetyl-CoA synthesis (D) pathways using KEGGMapper v.2.8. Data corresponding to
    upregulated and downregulated genes in panels C and D are shown with green and
    red backgrounds, respectively. Data corresponding to unchanged genes have a white
    background.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PDHB
  - PDHA1
  - OFCC1
  - LPA
  - CMPK2
  - ENO3
  - MATN1
  - PAEP
  - SPAG9
  - CUX1
  - ENO4
  - HMGA2
  - UEVLD
  - DCD
  - CXCL9
  - SLC25A1
  - ENO1
  - DUT
  - PNP
  - ENO2
  - AATF
  - UTP
  - UDP
  - CDP
  - CMP
  - dCDP
  - DUDP
  - Acetyl-CoA
  - S-Acotyldihidro
  - ethyl-THPP
  - Pyruvate
  - amide
  - Oxaloacetato
  - lysine
  - LPA
  - octanoyl
  - TCA CYCLE
  - 2-Phosphate
  - AMINO-ACYL
genes:
- word: pdhB
  symbol: PDHB
  source: hgnc_symbol
  hgnc_symbol: PDHB
  entrez: '5162'
- word: pdhA
  symbol: PDHA
  source: hgnc_prev_symbol
  hgnc_symbol: PDHA1
  entrez: '5160'
- word: opo]
  symbol: Opo
  source: hgnc_alias_symbol
  hgnc_symbol: OFCC1
  entrez: '266553'
- word: LpA
  symbol: LPA
  source: hgnc_symbol
  hgnc_symbol: LPA
  entrez: '4018'
- word: ndk
  symbol: NDK
  source: hgnc_alias_symbol
  hgnc_symbol: CMPK2
  entrez: '129607'
- word: eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO3
  entrez: '2027'
- word: CMP
  symbol: CMP
  source: hgnc_prev_symbol
  hgnc_symbol: MATN1
  entrez: '4146'
- word: PEP
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
- word: phet
  symbol: PHET
  source: hgnc_alias_symbol
  hgnc_symbol: SPAG9
  entrez: '9043'
- word: CDP
  symbol: CDP
  source: hgnc_alias_symbol
  hgnc_symbol: CUX1
  entrez: '1523'
- word: eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO4
  entrez: '387712'
- word: Lipo-
  symbol: LIPO
  source: hgnc_alias_symbol
  hgnc_symbol: HMGA2
  entrez: '8091'
- word: ATTP
  symbol: Attp
  source: hgnc_alias_symbol
  hgnc_symbol: UEVLD
  entrez: '55293'
- word: dcd
  symbol: DCD
  source: hgnc_symbol
  hgnc_symbol: DCD
  entrez: '117159'
- word: cmk
  symbol: CMK
  source: hgnc_prev_symbol
  hgnc_symbol: CXCL9
  entrez: '4283'
- word: CTP
  symbol: CTP
  source: hgnc_alias_symbol
  hgnc_symbol: SLC25A1
  entrez: '6576'
- word: eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO1
  entrez: '2023'
- word: dut
  symbol: DUT
  source: hgnc_symbol
  hgnc_symbol: DUT
  entrez: '1854'
- word: pnp
  symbol: PNP
  source: hgnc_symbol
  hgnc_symbol: PNP
  entrez: '4860'
- word: eno
  symbol: ENO
  source: bioentities_symbol
  hgnc_symbol: ENO2
  entrez: '2026'
- word: ded
  symbol: DED
  source: hgnc_alias_symbol
  hgnc_symbol: AATF
  entrez: '26574'
chemicals:
- word: UTP
  source: MESH
  identifier: D014544
- word: UDP
  source: MESH
  identifier: D014530
- word: CDP
  source: MESH
  identifier: D003565
- word: CMP
  source: MESH
  identifier: D003568
- word: dCDP
  source: MESH
  identifier: C045813
- word: DUDP
  source: ''
  identifier: ''
- word: Acetyl-CoA
  source: MESH
  identifier: D000105
- word: S-Acotyldihidro
  source: MESH
  identifier: D013455
- word: ethyl-THPP
  source: MESH
  identifier: C014870
- word: Pyruvate
  source: MESH
  identifier: D011773
- word: amide
  source: MESH
  identifier: D000577
- word: Oxaloacetato
  source: ''
  identifier: ''
- word: lysine
  source: MESH
  identifier: C114808
- word: LPA
  source: MESH
  identifier: C522988
- word: octanoyl
  source: MESH
  identifier: C034681
- word: TCA CYCLE
  source: MESH
  identifier: C000589078
- word: 2-Phosphate
  source: MESH
  identifier: D010710
- word: AMINO-ACYL
  source: MESH
  identifier: D000596
diseases: []
---
