---
figid: PMC5431987__41598_2017_1314_Fig8_HTML
figlink: /pmc/articles/PMC5431987/figure/Fig8/
number: F8
caption: 'Material and methods workflow diagram. The workflow starts with the data
  acquisition block which gathers level three breast invasive cancer gene expression
  raw counts for tumour and normal datasets from The Cancer Genome Atlas (TCGA). Complementary
  annotation data are obtained from BioMart (Chromosome, %GC content, Entrez Gene
  IDs, and HUGO Gene Nomenclature Committee - HGNC - symbols). Then, the pre-processing
  block integrates both expression and annotation data necessary for quality control,
  such as bias detection (e.g. %GC content, gene length, etc.), within/between normalization
  and multidimensional principal component (PC) analysis noise reduction. After that,
  the processing block diverges into two complementary analyses: i) Differentially
  expressed genes (DEGs) discovery is carried out to find potential candidate genes
  between cases and controls; ii) Two whole genome-genome mutual information density
  distributions are built for each condition in order to construct similar networks
  and visualized with Cytoscape. Both DEGs and networks results are then submitted
  to functional analysis with Ingenuity Pathway Analysis (IPA). Finally, result exploration
  integrates PubMed literature together with functional results to obtain biological
  insight of the problem at hand.'
pmcid: PMC5431987
papertitle: RNA-Seq based genome-wide analysis reveals loss of inter-chromosomal regulation
  in breast cancer.
reftext: Jesús Espinal-Enríquez, et al. Sci Rep. 2017;7:1760.
pmc_ranked_result_index: '173126'
pathway_score: 0.8795233
filename: 41598_2017_1314_Fig8_HTML.jpg
figtitle: Material and methods workflow diagram
year: '2017'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5431987__41598_2017_1314_Fig8_HTML.html
  '@type': Dataset
  description: 'Material and methods workflow diagram. The workflow starts with the
    data acquisition block which gathers level three breast invasive cancer gene expression
    raw counts for tumour and normal datasets from The Cancer Genome Atlas (TCGA).
    Complementary annotation data are obtained from BioMart (Chromosome, %GC content,
    Entrez Gene IDs, and HUGO Gene Nomenclature Committee - HGNC - symbols). Then,
    the pre-processing block integrates both expression and annotation data necessary
    for quality control, such as bias detection (e.g. %GC content, gene length, etc.),
    within/between normalization and multidimensional principal component (PC) analysis
    noise reduction. After that, the processing block diverges into two complementary
    analyses: i) Differentially expressed genes (DEGs) discovery is carried out to
    find potential candidate genes between cases and controls; ii) Two whole genome-genome
    mutual information density distributions are built for each condition in order
    to construct similar networks and visualized with Cytoscape. Both DEGs and networks
    results are then submitted to functional analysis with Ingenuity Pathway Analysis
    (IPA). Finally, result exploration integrates PubMed literature together with
    functional results to obtain biological insight of the problem at hand.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PCSK1
  - GC
  - MSMP
  - PKD2
genes:
- word: PC-1
  symbol: PC1
  source: hgnc_alias_symbol
  hgnc_symbol: PCSK1
  entrez: '5122'
- word: '%GC,'
  symbol: GC
  source: hgnc_symbol
  hgnc_symbol: GC
  entrez: '2638'
- word: PC-3
  symbol: PC-3
  source: hgnc_alias_symbol
  hgnc_symbol: MSMP
  entrez: '692094'
- word: PC-2
  symbol: Pc-2
  source: hgnc_alias_symbol
  hgnc_symbol: PKD2
  entrez: '5311'
chemicals: []
diseases: []
figid_alias: PMC5431987__F8
redirect_from: /figures/PMC5431987__F8
figtype: Figure
---
