---
figid: PMC5006394__nihms812310f2
figlink: /pmc/articles/PMC5006394/figure/F2/
number: Figure 2
caption: Effects of metabolism on Foxp3 expression and the generation of Treg cells.
  There are various scenarios in which Treg cell signaling and metabolic pathways
  might integrate and potentially affect Foxp3 expression. One of the downstream effects
  of an enhancement in glycolytic metabolism is the production of metabolic intermediates
  that can also function as signaling molecules (1). For example, NAD+ and NADH might
  control Foxp3 stability via the activation of histone deacetylases such as SIRT
  proteins, which directly deacetylate Foxp3 in the nucleus and lead to its proteosomal
  degradation in the cytoplasm. Furthermore, the glycolytic enzyme enolase-1 can repress
  the FOXP3 splice variant containing exon 2 (E2) in human Treg cells, and its engagement
  in glycolysis serves as a mechanism by which glycolysis can control Foxp3 expression
  (2). Signaling molecules from glycolysis and mitochondrial metabolism (PEP and ROS)
  activate NFAT via Ca2+ mobilization during T cell activation, a process that could
  potentially affect Foxp3 expression in Treg cells as well (3). Activation of Foxo
  transcription factors and HIF-1α downstream of the PI(3)K-Akt-mTOR signaling pathway
  can reciprocally affect Foxp3 expression (4). Finally, the chromatin-modifying enzyme
  EZH2 that is crucial for the establishment of a repressive Treg cell gene program
  can be inhibited by specific microRNAs (such as miR-101 and miR-26a) under circumstances
  of glucose deprivation, which leads to Treg cell instability (5). ETC, electron-transport
  chain; PRC, polycomb repressive complex; IP3 and IP3R, inositol-1,3,4-trisphosphate
  and its receptor.
pmcid: PMC5006394
papertitle: Immunometabolism of regulatory T cells.
reftext: Ryan Newton, et al. Nat Immunol. 2016 May 19;17(6):618-625.
pmc_ranked_result_index: '29530'
pathway_score: 0.9593951
filename: nihms812310f2.jpg
figtitle: Metabolism on Foxp3 expression and the generation of Treg cells
year: '2016'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5006394__nihms812310f2.html
  '@type': Dataset
  description: Effects of metabolism on Foxp3 expression and the generation of Treg
    cells. There are various scenarios in which Treg cell signaling and metabolic
    pathways might integrate and potentially affect Foxp3 expression. One of the downstream
    effects of an enhancement in glycolytic metabolism is the production of metabolic
    intermediates that can also function as signaling molecules (1). For example,
    NAD+ and NADH might control Foxp3 stability via the activation of histone deacetylases
    such as SIRT proteins, which directly deacetylate Foxp3 in the nucleus and lead
    to its proteosomal degradation in the cytoplasm. Furthermore, the glycolytic enzyme
    enolase-1 can repress the FOXP3 splice variant containing exon 2 (E2) in human
    Treg cells, and its engagement in glycolysis serves as a mechanism by which glycolysis
    can control Foxp3 expression (2). Signaling molecules from glycolysis and mitochondrial
    metabolism (PEP and ROS) activate NFAT via Ca2+ mobilization during T cell activation,
    a process that could potentially affect Foxp3 expression in Treg cells as well
    (3). Activation of Foxo transcription factors and HIF-1α downstream of the PI(3)K-Akt-mTOR
    signaling pathway can reciprocally affect Foxp3 expression (4). Finally, the chromatin-modifying
    enzyme EZH2 that is crucial for the establishment of a repressive Treg cell gene
    program can be inhibited by specific microRNAs (such as miR-101 and miR-26a) under
    circumstances of glucose deprivation, which leads to Treg cell instability (5).
    ETC, electron-transport chain; PRC, polycomb repressive complex; IP3 and IP3R,
    inositol-1,3,4-trisphosphate and its receptor.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - FOXP3
  - HIF1A
  - FOXO3
  - PTEN
  - EZH2
  - FOXO1
  - MTOR
  - NFAT5
  - NFATC1
  - PRKAB1
  - PRKAB2
  - PRKAG1
  - CD28
  - ATP2A1
  - ATP2A2
  - NFATC2
  - ATP2A3
  - SIRT1
  - TRA
  - PDCD1
  - PRKAA1
  - PRKAA2
  - NFATC4
  - TRG
  - PAEP
  - PPRC1
  - CTLA4
  - PRKAG3
  - TRD
  - NFATC3
  - TRB
  - PRKAG2
  - Glucose
  - Pyruvate
  - Lactate
genes:
- word: Foxp3
  symbol: FOXP3
  source: hgnc_symbol
  hgnc_symbol: FOXP3
  entrez: '50943'
- word: HIF1-a
  symbol: HIF1A
  source: hgnc_symbol
  hgnc_symbol: HIF1A
  entrez: '3091'
- word: Foxo3
  symbol: FOXO3
  source: hgnc_symbol
  hgnc_symbol: FOXO3
  entrez: '2309'
- word: PTEN
  symbol: PTEN
  source: hgnc_symbol
  hgnc_symbol: PTEN
  entrez: '5728'
- word: EZH2
  symbol: EZH2
  source: hgnc_symbol
  hgnc_symbol: EZH2
  entrez: '2146'
- word: Foxo1
  symbol: FOXO1
  source: hgnc_symbol
  hgnc_symbol: FOXO1
  entrez: '2308'
- word: MTOR
  symbol: MTOR
  source: hgnc_symbol
  hgnc_symbol: MTOR
  entrez: '2475'
- word: NFAT
  symbol: NFAT
  source: bioentities_symbol
  hgnc_symbol: NFAT5
  entrez: '10725'
- word: NFAT
  symbol: NFAT
  source: bioentities_symbol
  hgnc_symbol: NFATC1
  entrez: '4772'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAB1
  entrez: '5564'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAB2
  entrez: '5565'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAG1
  entrez: '5571'
- word: CD28
  symbol: CD28
  source: hgnc_symbol
  hgnc_symbol: CD28
  entrez: '940'
- word: SERCA
  symbol: SERCA
  source: bioentities_symbol
  hgnc_symbol: ATP2A1
  entrez: '487'
- word: SERCA
  symbol: SERCA
  source: bioentities_symbol
  hgnc_symbol: ATP2A2
  entrez: '488'
- word: NFAT
  symbol: NFAT
  source: bioentities_symbol
  hgnc_symbol: NFATC2
  entrez: '4773'
- word: SERCA
  symbol: SERCA
  source: bioentities_symbol
  hgnc_symbol: ATP2A3
  entrez: '489'
- word: SIRT1
  symbol: SIRT1
  source: hgnc_symbol
  hgnc_symbol: SIRT1
  entrez: '23411'
- word: TCR
  symbol: TCR
  source: bioentities_symbol
  hgnc_symbol: TRA
  entrez: '6955'
- word: PD-1
  symbol: PD-1
  source: hgnc_alias_symbol
  hgnc_symbol: PDCD1
  entrez: '5133'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAA1
  entrez: '5562'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAA2
  entrez: '5563'
- word: NFAT
  symbol: NFAT
  source: bioentities_symbol
  hgnc_symbol: NFATC4
  entrez: '4776'
- word: TCR
  symbol: TCR
  source: bioentities_symbol
  hgnc_symbol: TRG
  entrez: '6965'
- word: PEP
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
- word: PRC
  symbol: PRC
  source: hgnc_alias_symbol
  hgnc_symbol: PPRC1
  entrez: '23082'
- word: CTLA-4
  symbol: CTLA4
  source: hgnc_symbol
  hgnc_symbol: CTLA4
  entrez: '1493'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAG3
  entrez: '53632'
- word: TCR
  symbol: TCR
  source: bioentities_symbol
  hgnc_symbol: TRD
  entrez: '6964'
- word: NFAT
  symbol: NFAT
  source: bioentities_symbol
  hgnc_symbol: NFATC3
  entrez: '4775'
- word: TCR
  symbol: TCR
  source: bioentities_symbol
  hgnc_symbol: TRB
  entrez: '6957'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAG2
  entrez: '51422'
chemicals:
- word: Glucose
  source: MESH
  identifier: D005947
- word: Pyruvate
  source: MESH
  identifier: D011773
- word: Lactate
  source: MESH
  identifier: D019344
diseases: []
---
