---
figid: PMC5615194__mbo0051734950007
figlink: /pmc/articles/PMC5615194/figure/fig7/
number: F7
caption: Rickettsia metabolic network reconstruction highlights reductive genome evolution
  and addiction to host cell metabolites. The network focuses on the biosynthesis
  pathways discussed in the text. For brevity, pathways for most amino acids are not
  shown. Red stars indicate six pathway holes (see ). GLY, glycolysis; MEP/DOXP, nonmevalonate
  terpenoid biosynthesis; THF, 5,6,7,8-THF; GSH, glutathione. (A) Theoretical Rickettsia
  metabolic network in the absence of imported metabolites. Rickettsia metabolic pathways
  are supplemented with typical Gram-negative biosynthetic pathways to create a complete
  metabolic network. (B) Reconstructed Rickettsia metabolic network, including imported
  metabolites. Pathways removed from panel A have been purged from Rickettsia genomes
  throughout evolution, a consequence of pilfering of metabolites from the eukaryotic
  host. Red ellipses, metabolites known to be imported by Rickettsia species; yellow
  ellipses, metabolites predicted to be imported on the basis of metabolic network
  reconstruction. The import of S-adenosyl-l-methionine, phosphorylcholine, and the
  majority of amino acids is not included in the reconstruction. Pathway lines are
  highlighted in red to indicate cofactors that are synthesized directly from imported
  metabolites. Additionally, if the cofactor is directly imported from the host, the
  pathway line is yellow. The network is based on a phylogenomics analyses of 84 Rickettsia
  genomes (see ).
pmcid: PMC5615194
papertitle: Wholly Rickettsia! Reconstructed Metabolic Profile of the Quintessential
  Bacterial Parasite of Eukaryotic Cells.
reftext: Timothy P. Driscoll, et al. mBio. 2017 Sep-Oct;8(5):e00859-17.
pmc_ranked_result_index: '4100'
pathway_score: 0.8147641
filename: mbo0051734950007.jpg
figtitle: Rickettsia metabolic network reconstruction highlights reductive genome
  evolution and addiction to host cell metabolites
year: '2017'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5615194__mbo0051734950007.html
  '@type': Dataset
  description: Rickettsia metabolic network reconstruction highlights reductive genome
    evolution and addiction to host cell metabolites. The network focuses on the biosynthesis
    pathways discussed in the text. For brevity, pathways for most amino acids are
    not shown. Red stars indicate six pathway holes (see ). GLY, glycolysis; MEP/DOXP,
    nonmevalonate terpenoid biosynthesis; THF, 5,6,7,8-THF; GSH, glutathione. (A)
    Theoretical Rickettsia metabolic network in the absence of imported metabolites.
    Rickettsia metabolic pathways are supplemented with typical Gram-negative biosynthetic
    pathways to create a complete metabolic network. (B) Reconstructed Rickettsia
    metabolic network, including imported metabolites. Pathways removed from panel
    A have been purged from Rickettsia genomes throughout evolution, a consequence
    of pilfering of metabolites from the eukaryotic host. Red ellipses, metabolites
    known to be imported by Rickettsia species; yellow ellipses, metabolites predicted
    to be imported on the basis of metabolic network reconstruction. The import of
    S-adenosyl-l-methionine, phosphorylcholine, and the majority of amino acids is
    not included in the reconstruction. Pathway lines are highlighted in red to indicate
    cofactors that are synthesized directly from imported metabolites. Additionally,
    if the cofactor is directly imported from the host, the pathway line is yellow.
    The network is based on a phylogenomics analyses of 84 Rickettsia genomes (see
    ).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PLP1
  - PHB
  - PDC
  - MFAP1
  - IPP
  - FMN1
  - SLC25A1
  - NT5C2
  - Pyrimidines
  - DMAPP
  - Ala
  - Asp
  - Biotin
  - Coenzyme A
  - FAD
  - Fatty Acid
  - glucose
  - Malate
  - Queuosine
  - Porphyrins
  - Pyruvate
  - D-ribose 5-P
  - UDP
  - Glu
genes:
- word: PLP
  symbol: PLP
  source: hgnc_prev_symbol
  hgnc_symbol: PLP1
  entrez: '5354'
- word: PHB
  symbol: PHB
  source: hgnc_symbol
  hgnc_symbol: PHB
  entrez: '5245'
- word: PDC
  symbol: PDC
  source: hgnc_symbol
  hgnc_symbol: PDC
  entrez: '5132'
- word: AMP
  symbol: AMP
  source: hgnc_alias_symbol
  hgnc_symbol: MFAP1
  entrez: '4236'
- word: IPP)
  symbol: IPP
  source: hgnc_symbol
  hgnc_symbol: IPP
  entrez: '3652'
- word: FMN
  symbol: FMN
  source: hgnc_prev_symbol
  hgnc_symbol: FMN1
  entrez: '342184'
- word: CTP
  symbol: CTP
  source: hgnc_alias_symbol
  hgnc_symbol: SLC25A1
  entrez: '6576'
- word: GMP
  symbol: GMP
  source: hgnc_alias_symbol
  hgnc_symbol: NT5C2
  entrez: '22978'
chemicals:
- word: Pyrimidines
  source: MESH
  identifier: C030986
- word: DMAPP
  source: MESH
  identifier: C043060
- word: Ala
  source: MESH
  identifier: D000409
- word: Asp
  source: MESH
  identifier: D001224
- word: Biotin
  source: MESH
  identifier: D001710
- word: Coenzyme A
  source: MESH
  identifier: D003067
- word: FAD
  source: MESH
  identifier: D005182
- word: Fatty Acid
  source: MESH
  identifier: D005227
- word: glucose
  source: MESH
  identifier: D005947
- word: Malate
  source: MESH
  identifier: D008293
- word: Queuosine
  source: MESH
  identifier: D009704
- word: Porphyrins
  source: MESH
  identifier: D011166
- word: Pyruvate
  source: MESH
  identifier: D011773
- word: D-ribose 5-P
  source: MESH
  identifier: D012266
- word: UDP
  source: MESH
  identifier: D014530
- word: Glu
  source: MESH
  identifier: D018698
diseases: []
figid_alias: PMC5615194__F7
redirect_from: /figures/PMC5615194__F7
figtype: Figure
---
