---
figid: PMC5618938__epi-08-1637-g2
figlink: /pmc/articles/PMC5618938/figure/F0002/
number: F2
caption: '(A) A model of interaction of environmental factors with epigenetic modulators
  during aging. Environmental factors affect cell function through direct or indirect
  mechanisms by influencing many epigenetic modulators including chromatin writers
  (e.g., DNMTs, HMTs and HATs), chromatin eraser enzymes (e.g., TET, HDMTs and HDACs)
  and various transcriptional factors resulting in dynamic chromatin modifications,
  which in turn influence gene expression. Epigenetic-mediated phenotypic plasticity
  attenuation during aging is characterized by a loss of the normal balance between
  the regulatory epigenetic landmarks and plasticity of response to environmental
  signals. For example, in young cells, the chromatin (hypothetically) with increased
  active histone modifications (orange circle, methylated histone tail; green circle,
  acetylated histone tail) and hypomethylated DNA pattern (red diamond, methylated
  CpG island, left panel) leads to gene activation in response to balanced environmental
  stimuli. However, the gene is relatively refractory to environmentally induced activation
  due to age-dependent epigenetic changes in the gene regulatory region including
  altered epigenetic landmarks and malfunction of epigenetic modulators such as changes
  in gene expression, binding ability or enzymatic activity of chromatin writers,
  erasers and relevant transcriptional factors (represented as distorted shapes, right
  panel). This results in an altered chromatin structure (compacted chromatin) and
  regulatory machinery (repressor complex recruitment) in the loci leading to gene
  silencing (right panel). (B) Schematic representation of mechanistic pathway indicated
  in (A).DNMT: DNA methyltransferase; HAT: Histone acetylase; HDAC: Histone deacetylase;
  HDMT: Histone demethylase; HMT: Histone methyltransferase; TET: Ten-eleven translocation.'
pmcid: PMC5618938
papertitle: 'Age-related epigenetic drift and phenotypic plasticity loss: implications
  in prevention of age-related human diseases.'
reftext: Yuanyuan Li, et al. Epigenomics. 2016 Dec;8(12):1637-1651.
pmc_ranked_result_index: '99146'
pathway_score: 0.852232
filename: epi-08-1637-g2.jpg
figtitle: 'Age-related epigenetic drift and phenotypic plasticity loss: implications
  in prevention of age-related human diseases'
year: '2016'
organisms:
- Homo sapiens
ndex: 998dfd6c-df18-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5618938__epi-08-1637-g2.html
  '@type': Dataset
  description: '(A) A model of interaction of environmental factors with epigenetic
    modulators during aging. Environmental factors affect cell function through direct
    or indirect mechanisms by influencing many epigenetic modulators including chromatin
    writers (e.g., DNMTs, HMTs and HATs), chromatin eraser enzymes (e.g., TET, HDMTs
    and HDACs) and various transcriptional factors resulting in dynamic chromatin
    modifications, which in turn influence gene expression. Epigenetic-mediated phenotypic
    plasticity attenuation during aging is characterized by a loss of the normal balance
    between the regulatory epigenetic landmarks and plasticity of response to environmental
    signals. For example, in young cells, the chromatin (hypothetically) with increased
    active histone modifications (orange circle, methylated histone tail; green circle,
    acetylated histone tail) and hypomethylated DNA pattern (red diamond, methylated
    CpG island, left panel) leads to gene activation in response to balanced environmental
    stimuli. However, the gene is relatively refractory to environmentally induced
    activation due to age-dependent epigenetic changes in the gene regulatory region
    including altered epigenetic landmarks and malfunction of epigenetic modulators
    such as changes in gene expression, binding ability or enzymatic activity of chromatin
    writers, erasers and relevant transcriptional factors (represented as distorted
    shapes, right panel). This results in an altered chromatin structure (compacted
    chromatin) and regulatory machinery (repressor complex recruitment) in the loci
    leading to gene silencing (right panel). (B) Schematic representation of mechanistic
    pathway indicated in (A).DNMT: DNA methyltransferase; HAT: Histone acetylase;
    HDAC: Histone deacetylase; HDMT: Histone demethylase; HMT: Histone methyltransferase;
    TET: Ten-eleven translocation.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SIRT5
  - SIRT2
  - SIRT1
  - SIRT6
  - HDAC1
  - HDAC5
  - HDAC4
  - HDAC8
  - HDAC7
  - HDAC2
  - SIRT7
  - HDAC9
  - SIRT4
  - HDAC3
  - HDAC10
  - SIRT3
  - HDAC11
  - HDAC6
  - vitamin C
  - isoflavones
  - polyphenols
genes:
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: SIRT5
  entrez: '23408'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: SIRT2
  entrez: '22933'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: SIRT1
  entrez: '23411'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: SIRT6
  entrez: '51548'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: HDAC1
  entrez: '3065'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: HDAC5
  entrez: '10014'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: HDAC4
  entrez: '9759'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: HDAC8
  entrez: '55869'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: HDAC7
  entrez: '51564'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: HDAC2
  entrez: '3066'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: SIRT7
  entrez: '51547'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: HDAC9
  entrez: '9734'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: SIRT4
  entrez: '23409'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: HDAC3
  entrez: '8841'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: HDAC10
  entrez: '83933'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: SIRT3
  entrez: '23410'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: HDAC11
  entrez: '79885'
- word: HDAC
  symbol: HDAC
  source: bioentities_symbol
  hgnc_symbol: HDAC6
  entrez: '10013'
chemicals:
- word: vitamin C
  source: MESH
  identifier: D001205
- word: isoflavones
  source: MESH
  identifier: D007529
- word: polyphenols
  source: MESH
  identifier: D059808
diseases: []
figid_alias: PMC5618938__F2
redirect_from: /figures/PMC5618938__F2
figtype: Figure
---
