---
figid: PMC5423858__nihms829667f1
figlink: /pmc/articles/PMC5423858/figure/F1/
number: F1
caption: MiR gene expression is regulated in ways similar to protein coding genes.
  Neuronal activity (depolarization, neurotrophins, sensory stimuli, etc.) can induce
  miR expression [,,]. MiR-132, for example, possesses a CRE sequence in its promoter
  allowing CREB to regulate its activity. The NMDA receptor, CaMKII and the MAPK-ERK
  pathway are known to control miR-132 expression although different upstream signaling
  pathways that may regulate other miR genes []. There are two miR biogenesis pathways
  in the nucleus, the canonical and the ‘miRtrons’ pathway. In the canonical pathway,
  RNA polymerase II transcribes miR genes into pri-miRs that are further processed
  into pre-miRs by the Pasha/Drosha microprocessor complex. In the miRtron pathway,
  miRs are spliced from introns by the spliceosome to pre-miRs. Exportin5 along with
  Ran-GTP actively transports pre-miRs to the cytoplasm. Dicer/Loquacious complex
  further processes the pre-miRs in the cytoplasm to mature duplexes - with a guide
  strand (miR) and its passenger (miR*). A member of the Argonaute family of proteins
  (Ago1 or Ago2 depending on the pathway) loads one of the strands into the miRISC
  complex (lower right corner). The complex includes the Gw182 protein and other auxiliary
  proteins such as dFmr1. This outcome leads to mRNA degradation or translational
  inhibition depending on the match quality of miR/mRNA hybrid. In rare cases (upper
  right corner), duplexes with a very high level of complementarity between strands
  are sorted to an alternative pathway using Dicer2, leading to mRNA degradation by
  Ago2 and inducing competition between the processing pathways []. Neural activity,
  genetic manipulations or disorders have been shown to positively or negatively influence
  the processing enzymes involved in miRNA biogenesis (grey squares with red and green
  arrows) [,,].
pmcid: PMC5423858
papertitle: MicroRNA function in Drosophila memory formation.
reftext: Germain U. Busto, et al. Curr Opin Neurobiol. ;43:15-24.
pmc_ranked_result_index: '120053'
pathway_score: 0.9574236
filename: nihms829667f1.jpg
figtitle: MicroRNA function in Drosophila memory formation
year: ''
organisms: Homo sapiens
ndex: 69af6cfa-dee6-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5423858__nihms829667f1.html
  '@type': Dataset
  description: MiR gene expression is regulated in ways similar to protein coding
    genes. Neuronal activity (depolarization, neurotrophins, sensory stimuli, etc.)
    can induce miR expression [,,]. MiR-132, for example, possesses a CRE sequence
    in its promoter allowing CREB to regulate its activity. The NMDA receptor, CaMKII
    and the MAPK-ERK pathway are known to control miR-132 expression although different
    upstream signaling pathways that may regulate other miR genes []. There are two
    miR biogenesis pathways in the nucleus, the canonical and the ‘miRtrons’ pathway.
    In the canonical pathway, RNA polymerase II transcribes miR genes into pri-miRs
    that are further processed into pre-miRs by the Pasha/Drosha microprocessor complex.
    In the miRtron pathway, miRs are spliced from introns by the spliceosome to pre-miRs.
    Exportin5 along with Ran-GTP actively transports pre-miRs to the cytoplasm. Dicer/Loquacious
    complex further processes the pre-miRs in the cytoplasm to mature duplexes - with
    a guide strand (miR) and its passenger (miR*). A member of the Argonaute family
    of proteins (Ago1 or Ago2 depending on the pathway) loads one of the strands into
    the miRISC complex (lower right corner). The complex includes the Gw182 protein
    and other auxiliary proteins such as dFmr1. This outcome leads to mRNA degradation
    or translational inhibition depending on the match quality of miR/mRNA hybrid.
    In rare cases (upper right corner), duplexes with a very high level of complementarity
    between strands are sorted to an alternative pathway using Dicer2, leading to
    mRNA degradation by Ago2 and inducing competition between the processing pathways
    []. Neural activity, genetic manipulations or disorders have been shown to positively
    or negatively influence the processing enzymes involved in miRNA biogenesis (grey
    squares with red and green arrows) [,,].
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MAPK1
  - MAPK3
  - MAPK8
  - MAPK11
  - MAPK14
  - TNKS
  - PARP2
  - TIPARP
  - MAPK9
  - MAPK10
  - DGCR8
  - TNRC6A
  - CREB5
  - DICER1
  - PARP4
  - PARP15
  - PARP10
  - FBXW7
  - PARP14
  - PARP16
  - CREB3L4
  - ATF1
  - MAPK13
  - CREB3L3
  - CREB3L1
  - MECP2
  - PARP3
  - PARP1
  - CREB3
  - PARP6
  - PARP11
  - CAMK2B
  - PARP8
  - MAPK12
  - PARP9
  - PARP12
  - CREM
  - CREB3L2
  - CREB1
  - ATF2
  - POLL
  - TNKS2
  - RAN
  - AMAA
  - Glutamate
genes:
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK1
  entrez: '5594'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK3
  entrez: '5595'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK8
  entrez: '5599'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK11
  entrez: '5600'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK14
  entrez: '1432'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: TNKS
  entrez: '8658'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP2
  entrez: '10038'
- word: ERK
  symbol: ERK
  source: bioentities_symbol
  hgnc_symbol: MAPK3
  entrez: '5595'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: TIPARP
  entrez: '25976'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK9
  entrez: '5601'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK10
  entrez: '5602'
- word: Pasha,
  symbol: pasha
  source: hgnc_alias_symbol
  hgnc_symbol: DGCR8
  entrez: '54487'
- word: Gw182
  symbol: GW182
  source: hgnc_alias_symbol
  hgnc_symbol: TNRC6A
  entrez: '27327'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB5
  entrez: '9586'
- word: Dicer
  symbol: Dicer
  source: hgnc_alias_symbol
  hgnc_symbol: DICER1
  entrez: '23405'
- word: ERK
  symbol: ERK
  source: bioentities_symbol
  hgnc_symbol: MAPK1
  entrez: '5594'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP4
  entrez: '143'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP15
  entrez: '165631'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP10
  entrez: '84875'
- word: Ago
  symbol: AGO
  source: hgnc_alias_symbol
  hgnc_symbol: FBXW7
  entrez: '55294'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP14
  entrez: '54625'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP16
  entrez: '54956'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L4
  entrez: '148327'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: ATF1
  entrez: '466'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK13
  entrez: '5603'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L3
  entrez: '84699'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L1
  entrez: '90993'
- word: MECP2
  symbol: MECP2
  source: hgnc_symbol
  hgnc_symbol: MECP2
  entrez: '4204'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP3
  entrez: '10039'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP1
  entrez: '142'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3
  entrez: '10488'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP6
  entrez: '56965'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP11
  entrez: '57097'
- word: CAMKII
  symbol: CAMK2
  source: hgnc_alias_symbol
  hgnc_symbol: CAMK2B
  entrez: '816'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP8
  entrez: '79668'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK12
  entrez: '6300'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP9
  entrez: '83666'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP12
  entrez: '64761'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREM
  entrez: '1390'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L2
  entrez: '64764'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB1
  entrez: '1385'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: ATF2
  entrez: '1386'
- word: Poll
  symbol: POLL
  source: hgnc_symbol
  hgnc_symbol: POLL
  entrez: '27343'
- word: Parp
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: TNKS2
  entrez: '80351'
- word: Ran-GTP
  symbol: RAN
  source: hgnc_symbol
  hgnc_symbol: RAN
  entrez: '5901'
chemicals:
- word: AMAA
  source: MESH
  identifier: C024314
- word: Glutamate
  source: MESH
  identifier: D018698
diseases: []
figid_alias: PMC5423858__F1
redirect_from: /figures/PMC5423858__F1
figtype: Figure
---
