---
figid: PMC5377334__srep45557-f3
figlink: /pmc/articles/PMC5377334/figure/f3/
number: Figure 3
caption: (a) Reconstruction process of LCSC and non-LCSC GEMs. Transcriptome and metabolome
  data were integrated with a generic human GEM, HMR 2.0. During the GEM reconstruction
  process, draft GEMs were examined and validated by simulating the predefined 56
  metabolic tasks under the cultivation condition of RPMI-1640 medium (). The GrowMatch
  gap-filling algorithm was performed to enable the cell type-specific GEMs to demonstrate
  growth. Reactions involved in lipid and amino acid metabolisms were more included
  in LCSC GEMs, whereas keratan sulfate biosynthetic pathway (largely mediated by
  MAN1A1) was more represented in non-LCSC GEMs. (b) Scheme of metabolic simulation
  using flux response analysis of the LCSC and non-LCSC GEMs to predict transcription
  factors regulating biomass generation (i.e., cell growth rate). The mean flux value
  (purple line) of each reaction was used when calculating the correlation between
  each reaction’s flux value and cell growth rate. In the right panel, upon prediction
  of reactions with flux values positively correlated with cell growth rate (red lines).
  Transcription factors (TF) that are known to bind to and regulate genes of the predicted
  reactions were next searched from RegNetwork. (c) Transcription factors that potentially
  regulate genes responsible for the metabolic reactions with flux values positively
  correlated with cell growth rate. Names of the top 10 transcription factors (out
  of 184 transcription factors) are presented.
pmcid: PMC5377334
papertitle: Systems approach to characterize the metabolism of liver cancer stem cells
  expressing CD133.
reftext: Wonhee Hur, et al. Sci Rep. 2017;7:45557.
pmc_ranked_result_index: '18477'
pathway_score: 0.564702
filename: srep45557-f3.jpg
figtitle: Systems approach to characterize the metabolism of liver cancer stem cells
  expressing CD133
year: '2017'
organisms: Homo sapiens
ndex: bfef1a3f-df35-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5377334__srep45557-f3.html
  '@type': Dataset
  description: (a) Reconstruction process of LCSC and non-LCSC GEMs. Transcriptome
    and metabolome data were integrated with a generic human GEM, HMR 2.0. During
    the GEM reconstruction process, draft GEMs were examined and validated by simulating
    the predefined 56 metabolic tasks under the cultivation condition of RPMI-1640
    medium (). The GrowMatch gap-filling algorithm was performed to enable the cell
    type-specific GEMs to demonstrate growth. Reactions involved in lipid and amino
    acid metabolisms were more included in LCSC GEMs, whereas keratan sulfate biosynthetic
    pathway (largely mediated by MAN1A1) was more represented in non-LCSC GEMs. (b)
    Scheme of metabolic simulation using flux response analysis of the LCSC and non-LCSC
    GEMs to predict transcription factors regulating biomass generation (i.e., cell
    growth rate). The mean flux value (purple line) of each reaction was used when
    calculating the correlation between each reaction’s flux value and cell growth
    rate. In the right panel, upon prediction of reactions with flux values positively
    correlated with cell growth rate (red lines). Transcription factors (TF) that
    are known to bind to and regulate genes of the predicted reactions were next searched
    from RegNetwork. (c) Transcription factors that potentially regulate genes responsible
    for the metabolic reactions with flux values positively correlated with cell growth
    rate. Names of the top 10 transcription factors (out of 184 transcription factors)
    are presented.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - AR
  - NR4A1
  - RXRA
  - PPARD
  - PPARG
  - GEM
  - MYC
  - PPARA
  - SP1
  - TFAP2A
  - TF
  - KCNK3
  - E2F1
  - sulfate
genes:
- word: ÁR
  symbol: AR
  source: hgnc_symbol
  hgnc_symbol: AR
  entrez: '367'
- word: HMR
  symbol: HMR
  source: hgnc_prev_symbol
  hgnc_symbol: NR4A1
  entrez: '3164'
- word: RXRa,
  symbol: RXRA
  source: hgnc_symbol
  hgnc_symbol: RXRA
  entrez: '6256'
- word: PPAR
  symbol: PPAR
  source: bioentities_symbol
  hgnc_symbol: PPARD
  entrez: '5467'
- word: PPAR
  symbol: PPAR
  source: bioentities_symbol
  hgnc_symbol: PPARG
  entrez: '5468'
- word: GEM
  symbol: GEM
  source: hgnc_symbol
  hgnc_symbol: GEM
  entrez: '2669'
- word: MYC
  symbol: MYC
  source: hgnc_symbol
  hgnc_symbol: MYC
  entrez: '4609'
- word: PPAR
  symbol: PPAR
  source: bioentities_symbol
  hgnc_symbol: PPARA
  entrez: '5465'
- word: SP1
  symbol: SP1
  source: hgnc_symbol
  hgnc_symbol: SP1
  entrez: '6667'
- word: TFAP2A
  symbol: TFAP2A
  source: hgnc_symbol
  hgnc_symbol: TFAP2A
  entrez: '7020'
- word: TF
  symbol: TF
  source: hgnc_symbol
  hgnc_symbol: TF
  entrez: '7018'
- word: tasks
  symbol: TASK
  source: hgnc_alias_symbol
  hgnc_symbol: KCNK3
  entrez: '3777'
- word: E2F1
  symbol: E2F1
  source: hgnc_symbol
  hgnc_symbol: E2F1
  entrez: '1869'
chemicals:
- word: sulfate
  source: MESH
  identifier: D013431
diseases: []
---
