---
figid: PMC5025700__zbc0411652280001
figlink: /pmc/articles/PMC5025700/figure/F1/
number: F1
caption: 'Schematic showing the sources of NADPH in the mitochondrial matrix. This
  schematic depicts the simplified intermediary oxidative catabolism of energy substrates
  via the mitochondrial Krebs cycle. NADP+ is reduced into NADPH during the oxidation
  of the substrates isocitrate (Isoc), malate (Mal), and glutamate (Glut) via IDH2,
  MEs, and GDH, respectively. In addition, NADPH is also a product of the forward
  reaction of NNT, an enzyme located in the IMM, whose activity is dependent on the
  NAD redox state and the electrochemical potential across the IMM. Enzymes that regenerate
  NADPH are shown in red, and their oxidative reactions are indicated by blue arrows.
  Two isoforms of ME occur in mitochondria and can reduce NADP+; one is specific for
  the NADP coenzyme only. The GDH reaction is coupled to either NAD+ or NADP+ reduction.
  Pyruvate (Pyr), malate, and glutamate, shown in bold, are exogenous substrates used
  to support mitochondrial energy metabolism during most of the experiments in this
  study. Dashed lines represent the aspartate transamination reaction that consumes
  glutamate to form and feed α-ketoglutarate (α-Kg) into the Krebs cycle, which is
  canonically a principal entry pathway for glutamate carbon skeleton. Other abbreviations
  used are as follows: Ac-CoA, acetyl-coenzyme A; Citr, citrate; Succ-CoA, succinyl-coenzyme
  A; Succ, succinate; OAA, oxaloacetate; Asp, aspartate. For simplicity, fluxes through
  other Krebs cycle enzymes, such as citrate synthase and aconitase, can be equally
  important to NADPH supply via a putative pathway but are not depicted.'
pmcid: PMC5025700
papertitle: The Contribution of Nicotinamide Nucleotide Transhydrogenase to Peroxide
  Detoxification Is Dependent on the Respiratory State and Counterbalanced by Other
  Sources of NADPH in Liver Mitochondria.
reftext: Juliana Aparecida Ronchi, et al. J Biol Chem. 2016 Sep 16;291(38):20173-20187.
pmc_ranked_result_index: '131096'
pathway_score: 0.9629918
filename: zbc0411652280001.jpg
figtitle: Schematic showing the sources of NADPH in the mitochondrial matrix
year: '2016'
organisms:
- Homo sapiens
ndex: 958075c2-decc-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5025700__zbc0411652280001.html
  '@type': Dataset
  description: 'Schematic showing the sources of NADPH in the mitochondrial matrix.
    This schematic depicts the simplified intermediary oxidative catabolism of energy
    substrates via the mitochondrial Krebs cycle. NADP+ is reduced into NADPH during
    the oxidation of the substrates isocitrate (Isoc), malate (Mal), and glutamate
    (Glut) via IDH2, MEs, and GDH, respectively. In addition, NADPH is also a product
    of the forward reaction of NNT, an enzyme located in the IMM, whose activity is
    dependent on the NAD redox state and the electrochemical potential across the
    IMM. Enzymes that regenerate NADPH are shown in red, and their oxidative reactions
    are indicated by blue arrows. Two isoforms of ME occur in mitochondria and can
    reduce NADP+; one is specific for the NADP coenzyme only. The GDH reaction is
    coupled to either NAD+ or NADP+ reduction. Pyruvate (Pyr), malate, and glutamate,
    shown in bold, are exogenous substrates used to support mitochondrial energy metabolism
    during most of the experiments in this study. Dashed lines represent the aspartate
    transamination reaction that consumes glutamate to form and feed α-ketoglutarate
    (α-Kg) into the Krebs cycle, which is canonically a principal entry pathway for
    glutamate carbon skeleton. Other abbreviations used are as follows: Ac-CoA, acetyl-coenzyme
    A; Citr, citrate; Succ-CoA, succinyl-coenzyme A; Succ, succinate; OAA, oxaloacetate;
    Asp, aspartate. For simplicity, fluxes through other Krebs cycle enzymes, such
    as citrate synthase and aconitase, can be equally important to NADPH supply via
    a putative pathway but are not depicted.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PDHA2
  - SLC2A1
  - PDHX
  - PDHA1
  - DLAT
  - H6PD
  - PDHB
  - IDH2
  - DLD
  - MAL
  - Asp
genes:
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHA2
  entrez: '5161'
- word: Glut
  symbol: GLUT
  source: hgnc_prev_symbol
  hgnc_symbol: SLC2A1
  entrez: '6513'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHX
  entrez: '8050'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHA1
  entrez: '5160'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: DLAT
  entrez: '1737'
- word: GDH
  symbol: GDH
  source: hgnc_prev_symbol
  hgnc_symbol: H6PD
  entrez: '9563'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: PDHB
  entrez: '5162'
- word: IDH2
  symbol: IDH2
  source: hgnc_symbol
  hgnc_symbol: IDH2
  entrez: '3418'
- word: PDH
  symbol: PDH
  source: bioentities_symbol
  hgnc_symbol: DLD
  entrez: '1738'
- word: -Mal
  symbol: MAL
  source: hgnc_symbol
  hgnc_symbol: MAL
  entrez: '4118'
chemicals:
- word: Asp
  source: MESH
  identifier: D001224
diseases: []
figid_alias: PMC5025700__F1
redirect_from: /figures/PMC5025700__F1
figtype: Figure
---
