---
figid: PMC5462018__PP_PP201601848R1_f5
figlink: /pmc/articles/PMC5462018/figure/fig5/
number: Figure 5
caption: 'Conserved ABA signaling pathway in epidermis of H. vulgare and P. proliferum.
  The signaling network is formed by four main functional categories: ABA biosynthesis
  (pink), ABA transportation (green), signal transduction (purple), and the downstream
  ion channels and transporters (light blue). Heatmap of transcriptome of H. vulgare
  epidermis in drought stress (left and middle) and qPCR of P. proliferum epidermis
  in ABA treatment (right) are shown. ABCC, ATP-binding cassette C transporter; ABCG,
  ATP-binding cassette G; ABF, ABA responsive elements-binding factor; ABI4, ABA insensitive4;
  AHA, Arabidopsis plasma membrane H+-ATPase; AKT1, Ser/Thr kinase1; ALMT, aluminum-activated
  malate transporter; AREB, AREB-like protein; AVP, Arabidopsis vacuolar H+-pyrophosphatase;
  CAS, calcium sensing receptor; CHLH, protoporphyrin IX magnesium chelatase, subunit
  H; CLC-C, chloride channel C; CNGC, cyclic nucleotide gated channel; CYP707A, cytochromeP450
  family 707 superfamily A; GORK, gated outwardly rectifying K+ channel; GTG, GPCR-type
  g protein1; KAT, guard cell inwardly rectifying K+ channel; MAPK, mitogen activated
  kinase-like protein; MYB, MYB domain protein; NADPBRF, NAD(P)-binding Rossmann-fold
  superfamily protein; NCED3, 9-cis-epoxycarotenoid dioxygenase6; OST1, open stomata1;
  PLDa1, phospholipase Dα1; PP2C, Protein Phosphatase 2C; RBOH, respiratory burst
  oxidase homolog protein; RCAR, regulatory component of ABA receptor; SLAC, slow
  anion channel; STP1, sugar transporter1; VHA, vacuolar H+-ATPase; ZEP, zeaxanthin
  epoxidase.'
pmcid: PMC5462018
papertitle: Evolutionary Conservation of ABA Signaling for Stomatal Closure.
reftext: Shengguan Cai, et al. Plant Physiol. 2017 Jun;174(2):732-747.
pmc_ranked_result_index: '107971'
pathway_score: 0.868024
filename: PP_PP201601848R1_f5.jpg
figtitle: Conserved ABA signaling pathway in epidermis of H
year: '2017'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5462018__PP_PP201601848R1_f5.html
  '@type': Dataset
  description: 'Conserved ABA signaling pathway in epidermis of H. vulgare and P.
    proliferum. The signaling network is formed by four main functional categories:
    ABA biosynthesis (pink), ABA transportation (green), signal transduction (purple),
    and the downstream ion channels and transporters (light blue). Heatmap of transcriptome
    of H. vulgare epidermis in drought stress (left and middle) and qPCR of P. proliferum
    epidermis in ABA treatment (right) are shown. ABCC, ATP-binding cassette C transporter;
    ABCG, ATP-binding cassette G; ABF, ABA responsive elements-binding factor; ABI4,
    ABA insensitive4; AHA, Arabidopsis plasma membrane H+-ATPase; AKT1, Ser/Thr kinase1;
    ALMT, aluminum-activated malate transporter; AREB, AREB-like protein; AVP, Arabidopsis
    vacuolar H+-pyrophosphatase; CAS, calcium sensing receptor; CHLH, protoporphyrin
    IX magnesium chelatase, subunit H; CLC-C, chloride channel C; CNGC, cyclic nucleotide
    gated channel; CYP707A, cytochromeP450 family 707 superfamily A; GORK, gated outwardly
    rectifying K+ channel; GTG, GPCR-type g protein1; KAT, guard cell inwardly rectifying
    K+ channel; MAPK, mitogen activated kinase-like protein; MYB, MYB domain protein;
    NADPBRF, NAD(P)-binding Rossmann-fold superfamily protein; NCED3, 9-cis-epoxycarotenoid
    dioxygenase6; OST1, open stomata1; PLDa1, phospholipase Dα1; PP2C, Protein Phosphatase
    2C; RBOH, respiratory burst oxidase homolog protein; RCAR, regulatory component
    of ABA receptor; SLAC, slow anion channel; STP1, sugar transporter1; VHA, vacuolar
    H+-ATPase; ZEP, zeaxanthin epoxidase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - AKT2
  - AVP
  - GYPA
  - AKT1
  - MAPK9
  - MAPK13
  - MAPK12
  - MAPK11
  - KRIT1
  - TSTD1
  - MAPK1
  - AKT3
  - MAPK8
  - MYB
  - MAPK3
  - MAPK14
  - MAPK10
  - TRIP10
genes:
- word: AKT
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT2
  entrez: '208'
- word: AVP
  symbol: AVP
  source: hgnc_symbol
  hgnc_symbol: AVP
  entrez: '551'
- word: GPA
  symbol: GPA
  source: hgnc_alias_symbol
  hgnc_symbol: GYPA
  entrez: '2993'
- word: AKT
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT1
  entrez: '207'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK9
  entrez: '5601'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK13
  entrez: '5603'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK12
  entrez: '6300'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK11
  entrez: '5600'
- word: CAM
  symbol: CAM
  source: hgnc_alias_symbol
  hgnc_symbol: KRIT1
  entrez: '889'
- word: KAT
  symbol: KAT
  source: hgnc_alias_symbol
  hgnc_symbol: TSTD1
  entrez: '100131187'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK1
  entrez: '5594'
- word: AKT
  symbol: AKT
  source: bioentities_symbol
  hgnc_symbol: AKT3
  entrez: '10000'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK8
  entrez: '5599'
- word: MYB
  symbol: MYB
  source: hgnc_symbol
  hgnc_symbol: MYB
  entrez: '4602'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK3
  entrez: '5595'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK14
  entrez: '1432'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK10
  entrez: '5602'
- word: STP
  symbol: STP
  source: hgnc_alias_symbol
  hgnc_symbol: TRIP10
  entrez: '9322'
chemicals: []
diseases: []
---
