---
figid: PMC5937716__nihms961952f3
figlink: /pmc/articles/PMC5937716/figure/F3/
number: F3
caption: (A) The list of regeneration genes from  and  with Z scores > 2.0 was assessed
  for Gene Ontology (GO) pathway enrichment using BINGO software (; ). Those pathways
  significantly enriched (Bonferroni-corrected family-wise error rate [FWER] p < 0.01)
  are shown with colored circles of greater intensity for greater significance. The
  size of each circle reflects the number of genes in that category.(B) The list of
  regeneration genes from  and  with Z scores > 2.0 that are in the “transport” GO
  function group was analyzed using GeneMania software () for interactions. Each red
  circle is regeneration gene, each gray diamond is a protein domain, and each gray
  circle is predicted regeneration gene on the basis of sequence homology. Protein-protein
  interactions, co-localization, and co-expression are shown by connecting lines.
  The “transport” pathways includes multiple Rab proteins. Statistical significance
  for enrichment of this process by genome-wide false discovery rate (FDR) was calculated
  using the STRING database ().(C) Similar analysis as in (B) but for the “receptor
  binding” GO process.(D) Similar analysis as in (B) but for the “cytokine” plus “Jak-Stat
  signaling” KEGG pathways.
pmcid: PMC5937716
papertitle: Functional Genome-wide Screen Identifies Pathways Restricting Central
  Nervous System Axonal Regeneration.
reftext: Yuichi Sekine, et al. Cell Rep. ;23(2):415-428.
pmc_ranked_result_index: '171863'
pathway_score: 0.783758
filename: nihms961952f3.jpg
figtitle: Functional Genome-wide Screen Identifies Pathways Restricting Central Nervous
  System Axonal Regeneration
year: ''
organisms:
- Homo sapiens
ndex: c578bef8-df25-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC5937716__nihms961952f3.html
  '@type': Dataset
  description: (A) The list of regeneration genes from  and  with Z scores > 2.0 was
    assessed for Gene Ontology (GO) pathway enrichment using BINGO software (; ).
    Those pathways significantly enriched (Bonferroni-corrected family-wise error
    rate [FWER] p < 0.01) are shown with colored circles of greater intensity for
    greater significance. The size of each circle reflects the number of genes in
    that category.(B) The list of regeneration genes from  and  with Z scores > 2.0
    that are in the “transport” GO function group was analyzed using GeneMania software
    () for interactions. Each red circle is regeneration gene, each gray diamond is
    a protein domain, and each gray circle is predicted regeneration gene on the basis
    of sequence homology. Protein-protein interactions, co-localization, and co-expression
    are shown by connecting lines. The “transport” pathways includes multiple Rab
    proteins. Statistical significance for enrichment of this process by genome-wide
    false discovery rate (FDR) was calculated using the STRING database ().(C) Similar
    analysis as in (B) but for the “receptor binding” GO process.(D) Similar analysis
    as in (B) but for the “cytokine” plus “Jak-Stat signaling” KEGG pathways.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - FASLG
  - IFNA1
  - JAK2
  - TSPAN4
  - CALY
  - JAK1
  - CS
  - RARA
  - VEGFD
  - CAT
  - POMC
  - STAT3
  - JAK3
  - GABARAP
  - STAT2
  - STAT5B
  - STAT4
  - STAT1
  - KIRREL3
  - RARG
  - TCOF1
  - NAT2
  - RARB
  - TYK2
  - LEP
  - ARSB
  - STAT5A
  - VEGFA
  - MECR
  - SLTM
  - PGF
  - STAT6
  - NAA50
  - ABRA
  - VEGFC
  - SOST
  - VEGFB
  - Al
  - Ca
  - Ran
genes:
- word: Fasl
  symbol: FasL
  source: hgnc_alias_symbol
  hgnc_symbol: FASLG
  entrez: '356'
- word: ifl
  symbol: IFL
  source: hgnc_alias_symbol
  hgnc_symbol: IFNA1
  entrez: '3439'
- word: Jak
  symbol: JAK
  source: bioentities_symbol
  hgnc_symbol: JAK2
  entrez: '3717'
- word: Nag2
  symbol: NAG-2
  source: hgnc_alias_symbol
  hgnc_symbol: TSPAN4
  entrez: '7106'
- word: Caly
  symbol: CALY
  source: hgnc_symbol
  hgnc_symbol: CALY
  entrez: '50632'
- word: Jak
  symbol: JAK
  source: bioentities_symbol
  hgnc_symbol: JAK1
  entrez: '3716'
- word: CS
  symbol: CS
  source: hgnc_symbol
  hgnc_symbol: CS
  entrez: '1431'
- word: RAR
  symbol: RAR
  source: bioentities_symbol
  hgnc_symbol: RARA
  entrez: '5914'
- word: Figf
  symbol: FIGF
  source: hgnc_prev_symbol
  hgnc_symbol: VEGFD
  entrez: '2277'
- word: Cat
  symbol: CAT
  source: hgnc_symbol
  hgnc_symbol: CAT
  entrez: '847'
- word: Poc
  symbol: POC
  source: hgnc_alias_symbol
  hgnc_symbol: POMC
  entrez: '5443'
- word: STAT
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT3
  entrez: '6774'
- word: Jak
  symbol: JAK
  source: bioentities_symbol
  hgnc_symbol: JAK3
  entrez: '3718'
- word: Gabarap
  symbol: GABARAP
  source: hgnc_symbol
  hgnc_symbol: GABARAP
  entrez: '11337'
- word: STAT
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT2
  entrez: '6773'
- word: STAT
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT5B
  entrez: '6777'
- word: STAT
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT4
  entrez: '6775'
- word: STAT
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT1
  entrez: '6772'
- word: Neph2
  symbol: NEPH2
  source: hgnc_alias_symbol
  hgnc_symbol: KIRREL3
  entrez: '84623'
- word: RAR
  symbol: RAR
  source: bioentities_symbol
  hgnc_symbol: RARG
  entrez: '5916'
- word: TCS
  symbol: TCS
  source: hgnc_alias_symbol
  hgnc_symbol: TCOF1
  entrez: '6949'
- word: Nat2
  symbol: NAT2
  source: hgnc_symbol
  hgnc_symbol: NAT2
  entrez: '10'
- word: RAR
  symbol: RAR
  source: bioentities_symbol
  hgnc_symbol: RARB
  entrez: '5915'
- word: Jak
  symbol: JAK
  source: bioentities_symbol
  hgnc_symbol: TYK2
  entrez: '7297'
- word: Lep
  symbol: LEP
  source: hgnc_symbol
  hgnc_symbol: LEP
  entrez: '3952'
- word: Vegf
  symbol: VEGF
  source: bioentities_symbol
  hgnc_symbol: VEGFD
  entrez: '2277'
- word: Arsb
  symbol: ARSB
  source: hgnc_symbol
  hgnc_symbol: ARSB
  entrez: '411'
- word: STAT
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT5A
  entrez: '6776'
- word: Vegf
  symbol: VEGF
  source: bioentities_symbol
  hgnc_symbol: VEGFA
  entrez: '7422'
- word: Mecr
  symbol: MECR
  source: hgnc_symbol
  hgnc_symbol: MECR
  entrez: '51102'
- word: Met
  symbol: Met
  source: hgnc_alias_symbol
  hgnc_symbol: SLTM
  entrez: '79811'
- word: Vegf
  symbol: VEGF
  source: bioentities_symbol
  hgnc_symbol: PGF
  entrez: '5228'
- word: STAT
  symbol: STAT
  source: bioentities_symbol
  hgnc_symbol: STAT6
  entrez: '6778'
- word: San
  symbol: San
  source: hgnc_alias_symbol
  hgnc_symbol: NAA50
  entrez: '80218'
- word: Stars
  symbol: STARS
  source: hgnc_alias_symbol
  hgnc_symbol: ABRA
  entrez: '137735'
- word: Vegf
  symbol: VEGF
  source: bioentities_symbol
  hgnc_symbol: VEGFC
  entrez: '7424'
- word: Sost
  symbol: SOST
  source: hgnc_symbol
  hgnc_symbol: SOST
  entrez: '50964'
- word: Vegf
  symbol: VEGF
  source: bioentities_symbol
  hgnc_symbol: VEGFB
  entrez: '7423'
chemicals:
- word: Al
  source: MESH
  identifier: D000535
- word: Ca
  source: MESH
  identifier: D002118
- word: Ran
  source: MESH
  identifier: C046045
diseases: []
figid_alias: PMC5937716__F3
redirect_from: /figures/PMC5937716__F3
figtype: Figure
---
