---
figid: PMC6528166__fpls-10-00560-g0001
figlink: /pmc/articles/PMC6528166/figure/F1/
number: F1
caption: Phylogenetic relationships and gene clustering of JA-responsive ERF (A) and
  bHLH (B) transcription factors of clade II, subgroup IXa and IVa, respectively.
  Full-length amino acid sequences were aligned using ClustalW (Thompson et al., ),
  and the trees were generated with the neighbor-joining algorithm using MEGA-X (Kumar
  et al., ). The Arabidopsis genes are included in the trees as an outgroup. The scale
  bars indicate the number of amino acid substitutions per residue. For isoprenoid-derived
  metabolites, the contribution of the mevalonate (MVA) or methylerythritol phosphate
  (MEP) pathway to their production is indicated on the right. Apparent orthologs
  from Solanaceae plants are connected by dotted lines. Gene clusters are depicted
  schematically. The region deleted in a low-nicotine tobacco cultivar (Kajikawa et
  al., ) is highlighted in light green.
pmcid: PMC6528166
papertitle: 'The Recruitment Model of Metabolic Evolution: Jasmonate-Responsive Transcription
  Factors and a Conceptual Model for the Evolution of Metabolic Pathways.'
reftext: Tsubasa Shoji. Front Plant Sci. 2019;10:560.
pmc_ranked_result_index: '151951'
pathway_score: 0.8649476
filename: fpls-10-00560-g0001.jpg
figtitle: Phylogenetic relationships and gene clustering of JA-responsive ERF (A)
  and bHLH (B) transcription factors of clade II, subgroup IXa and IVa, respectively
year: '2019'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6528166__fpls-10-00560-g0001.html
  '@type': Dataset
  description: Phylogenetic relationships and gene clustering of JA-responsive ERF
    (A) and bHLH (B) transcription factors of clade II, subgroup IXa and IVa, respectively.
    Full-length amino acid sequences were aligned using ClustalW (Thompson et al.,
    ), and the trees were generated with the neighbor-joining algorithm using MEGA-X
    (Kumar et al., ). The Arabidopsis genes are included in the trees as an outgroup.
    The scale bars indicate the number of amino acid substitutions per residue. For
    isoprenoid-derived metabolites, the contribution of the mevalonate (MVA) or methylerythritol
    phosphate (MEP) pathway to their production is indicated on the right. Apparent
    orthologs from Solanaceae plants are connected by dotted lines. Gene clusters
    are depicted schematically. The region deleted in a low-nicotine tobacco cultivar
    (Kajikawa et al., ) is highlighted in light green.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ORC1
  - ESS2
  - nicotine
  - MEP
  - artenimisinin
  - triperpenoid saponins
genes:
- word: ORC1
  symbol: ORC1
  source: hgnc_symbol
  hgnc_symbol: ORC1
  entrez: '4998'
- word: BIS1
  symbol: bis1
  source: hgnc_alias_symbol
  hgnc_symbol: ESS2
  entrez: '8220'
chemicals:
- word: nicotine
  source: MESH
  identifier: D009538
- word: MEP
  source: MESH
  identifier: C071507
- word: artenimisinin
  source: ''
  identifier: ''
- word: triperpenoid saponins
  source: MESH
  identifier: D012503
diseases: []
figid_alias: PMC6528166__F1
redirect_from: /figures/PMC6528166__F1
figtype: Figure
---
