---
figid: PMC6355812__41598_2018_37768_Fig1_HTML
figlink: /pmc/articles/PMC6355812/figure/Fig1/
number: F1
caption: Illustration of polyP metabolism and pathway distributions in archaea. (a)
  Red arrows represent synthesis pathways (PPK1 and Arp_C) that convert (phosphate)n
  to (phosphate)n+1 while deep blue arrows represent independent degradation pathways
  (PPGK, SurE, PPX, PAP, PPK2 and NadK) that break down (phosphate)n into (phosphate)n-1.
  Arp_C is an abbreviation of actin protein complex of ArpABCEFGH that shares the
  same Pfam domain actin (PF00022). (b) Eight polyP metabolism pathways and nine archaeal
  physiology types are presented. Full_Path (7 archaea) encompasses polyP-related
  six enzymes while No_Path (64 archaea) have lost all six enzymes from archaeal proteomes.
  Functional_Path (99 archaea) means archaea having PPK1 and at least one polyP-degrading
  enzyme. PPK1_PPK2 (18 archaea), PPK1_PAP (30 archaea), PPK1_SurE (88 archaea), PPK1_PPX
  (39 archaea), and PPK2_NADK (78 archaea) means archaea having PPK1 and the designated
  polyP-degrading enzyme. For each pathway, average proteome size (APS) and GC content
  were given at the bottom of each boxplot.
pmcid: PMC6355812
papertitle: Bioinformatics Analysis of Metabolism Pathways of Archaeal Energy Reserves.
reftext: Liang Wang, et al. Sci Rep. 2019;9:1034.
pmc_ranked_result_index: '112967'
pathway_score: 0.8533368
filename: 41598_2018_37768_Fig1_HTML.jpg
figtitle: Illustration of polyP metabolism and pathway distributions in archaea
year: '2019'
organisms:
- Thiomicrospira cyclica
- Haloferax massiliensis
- Mycobacterium tuberculosis
- Thermococcus gammatolerans
- Methanosarcina acetivorans C2A
- Methanosarcina mazei Go1
- Acinetobacter baylyi
- Candidatus Lokiarchaeota archaeon
- candidate division MSBL1
- Methanohalophilus mahii
- Methanosarcina thermophila
- Methanosarcina acetivorans
- Archaeoglobus fulgidus
- Haloarcula marismortui
- Haloferax mediterranei
- Saccharolobus solfataricus
- Sulfolobus sp.
- Burkholderia pseudomallei
- Pseudomonas aeruginosa
- Pyrococcus abyssi
- Methanothrix harundinacea
- Thermococcus kodakarensis
- Acidiplasma cupricumulans
- Euryarchaeota incertae sedis
- Haladaptatus paucihalophilus
- Escherichia coli B
- Sinorhizobium fredii
- Metallosphaera sedula DSM 5348
- Metallosphaera sedula
- Dictyostelium discoideum
- Sulfuracidifex metallicus
- Saccharomyces cerevisiae
- Haloarcula hispanica
- Pyrococcus horikoshii
- Escherichia coli
- Acinetobacter baylyi ADP1
- Aeromonas hydrophila
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6355812__41598_2018_37768_Fig1_HTML.html
  '@type': Dataset
  description: Illustration of polyP metabolism and pathway distributions in archaea.
    (a) Red arrows represent synthesis pathways (PPK1 and Arp_C) that convert (phosphate)n
    to (phosphate)n+1 while deep blue arrows represent independent degradation pathways
    (PPGK, SurE, PPX, PAP, PPK2 and NadK) that break down (phosphate)n into (phosphate)n-1.
    Arp_C is an abbreviation of actin protein complex of ArpABCEFGH that shares the
    same Pfam domain actin (PF00022). (b) Eight polyP metabolism pathways and nine
    archaeal physiology types are presented. Full_Path (7 archaea) encompasses polyP-related
    six enzymes while No_Path (64 archaea) have lost all six enzymes from archaeal
    proteomes. Functional_Path (99 archaea) means archaea having PPK1 and at least
    one polyP-degrading enzyme. PPK1_PPK2 (18 archaea), PPK1_PAP (30 archaea), PPK1_SurE
    (88 archaea), PPK1_PPX (39 archaea), and PPK2_NADK (78 archaea) means archaea
    having PPK1 and the designated polyP-degrading enzyme. For each pathway, average
    proteome size (APS) and GC content were given at the bottom of each boxplot.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - REG3A
  - NADK
  - MFAP1
  - PPP4C
  - WDTC1
  - NT5C2
  - Phosphate
  - ADP
  - ATP
  - GDP
  - GMP
  - NAD
  - Glucose
  - AMP
  - Glucose-6-Phosphate
  - GTP
  - NADP
genes:
- word: PAP
  symbol: PAP
  source: hgnc_prev_symbol
  hgnc_symbol: REG3A
  entrez: '5068'
- word: Nadk
  symbol: NADK
  source: hgnc_symbol
  hgnc_symbol: NADK
  entrez: '65220'
- word: AMP
  symbol: AMP
  source: hgnc_alias_symbol
  hgnc_symbol: MFAP1
  entrez: '4236'
- word: PPX
  symbol: PPX
  source: hgnc_alias_symbol
  hgnc_symbol: PPP4C
  entrez: '5531'
- word: ADP+
  symbol: ADP
  source: hgnc_alias_symbol
  hgnc_symbol: WDTC1
  entrez: '23038'
- word: GMP
  symbol: GMP
  source: hgnc_alias_symbol
  hgnc_symbol: NT5C2
  entrez: '22978'
chemicals:
- word: Phosphate
  source: MESH
  identifier: D010710
- word: ADP
  source: MESH
  identifier: D000244
- word: ATP
  source: MESH
  identifier: D000255
- word: GDP
  source: MESH
  identifier: D006153
- word: GMP
  source: MESH
  identifier: C068152
- word: NAD
  source: MESH
  identifier: D009243
- word: Glucose
  source: MESH
  identifier: D005947
- word: AMP
  source: MESH
  identifier: D000249
- word: Glucose-6-Phosphate
  source: MESH
  identifier: D019298
- word: GTP
  source: MESH
  identifier: D006160
- word: NADP
  source: MESH
  identifier: D009249
diseases: []
figid_alias: PMC6355812__F1
redirect_from: /figures/PMC6355812__F1
figtype: Figure
---
