---
figid: PMC6391848__fgene-10-00091-g004
figlink: /pmc/articles/PMC6391848/figure/F4/
number: F4
caption: Enrichment analysis of differentially expressed mRNAs. (A) Advanced bubble
  chart of GO-BP. Y-axis label represents terms name, and X-axis label typifies rich
  factor which is defined as the percentage of target genes per term. Size and color
  of the bubble are measured as count and –log10qvalue to represent the amount of
  differentially expressed mRNAs enriched in these terms and the enrichment significance,
  respectively, as well as three shapes of SampleGroup represent three functional
  clusters. (B) KEGG interaction network. Terms are represented as nodes based on
  the kappa score level (= 0.4), and node size indicates the term enrichment significant.
  Edges represent shared genes, the wider the edge, the larger the overlap is. The
  colors indicating different functional groups are shown at the right top. (C) Bos
  taurus Wnt signaling pathway map (). Upregulated genes in our network are marked
  in red.
pmcid: PMC6391848
papertitle: Characterization of lncRNA–miRNA–mRNA Network to Reveal Potential Functional
  ceRNAs in Bovine Skeletal Muscle.
reftext: Binglin Yue, et al. Front Genet. 2019;10:91.
pmc_ranked_result_index: '150001'
pathway_score: 0.6352031
filename: fgene-10-00091-g004.jpg
figtitle: Enrichment analysis of differentially expressed mRNAs
year: '2019'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6391848__fgene-10-00091-g004.html
  '@type': Dataset
  description: Enrichment analysis of differentially expressed mRNAs. (A) Advanced
    bubble chart of GO-BP. Y-axis label represents terms name, and X-axis label typifies
    rich factor which is defined as the percentage of target genes per term. Size
    and color of the bubble are measured as count and –log10qvalue to represent the
    amount of differentially expressed mRNAs enriched in these terms and the enrichment
    significance, respectively, as well as three shapes of SampleGroup represent three
    functional clusters. (B) KEGG interaction network. Terms are represented as nodes
    based on the kappa score level (= 0.4), and node size indicates the term enrichment
    significant. Edges represent shared genes, the wider the edge, the larger the
    overlap is. The colors indicating different functional groups are shown at the
    right top. (C) Bos taurus Wnt signaling pathway map (). Upregulated genes in our
    network are marked in red.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SFRP2
  - RB1
  - GPR162
  - AR
  - HNRNPM
  - ACSM3
  - GO-BP
  - chramcsone
  - thymocyne
  - proteomal ubijulirdependefi
  - Colorectal cancer
  - SAH
  - mitoea rucieer
genes:
- word: SFRP2
  symbol: SFRP2
  source: hgnc_symbol
  hgnc_symbol: SFRP2
  entrez: '6423'
- word: RB1
  symbol: RB1
  source: hgnc_symbol
  hgnc_symbol: RB1
  entrez: '5925'
- word: a2
  symbol: A-2
  source: hgnc_alias_symbol
  hgnc_symbol: GPR162
  entrez: '27239'
- word: ais
  symbol: AIS
  source: hgnc_alias_symbol
  hgnc_symbol: AR
  entrez: '367'
- word: cear
  symbol: CEAR
  source: hgnc_alias_symbol
  hgnc_symbol: HNRNPM
  entrez: '4670'
- word: SAH
  symbol: SAH
  source: hgnc_prev_symbol
  hgnc_symbol: ACSM3
  entrez: '6296'
chemicals:
- word: GO-BP
  source: MESH
  identifier: C110781
- word: chramcsone
  source: ''
  identifier: ''
- word: thymocyne
  source: ''
  identifier: ''
diseases:
- word: proteomal ubijulirdependefi
  source: ''
  identifier: ''
- word: Colorectal cancer
  source: MESH
  identifier: D015179
- word: SAH
  source: MESH
  identifier: D013345
- word: mitoea rucieer
  source: ''
  identifier: ''
figid_alias: PMC6391848__F4
redirect_from: /figures/PMC6391848__F4
figtype: Figure
---
