---
figid: PMC6410037__genes-10-00118-g002
figlink: /pmc/articles/PMC6410037/figure/genes-10-00118-f002/
number: F2
caption: Telomere length maintenance mechanisms. (A) A simplified view of telomere
  length control by telomerase and the CST-complex. Telomerase adds telomeric repeats
  to the 3’-end of the G-rich overhang. Its activity is controlled by the CST-complex,
  which displaces telomerase, removes secondary structures and recruits the DNA polymerase
  α/primase complex to synthesize the C-rich strand. Note that regulation of telomerase
  by the CST-complex is more complex in yeast Saccharomyces cerevisiae (see text for
  details). (B) A general representation of the Alternative Lengthening of Telomeres
  (ALT) mechanism. ALT is currently viewed as the BIR-dependent elongation of telomeres
  after the replication fork stalling, ultimately leading to the formation of a one-ended
  double-strand break. This fork stalling can be due to the formation of unusual secondary
  structures in telomeric DNA (such as G-quadruplexes), the presence of stable DNA-RNA
  structures (R-loops generated during transcription of telomere-specific RNA called
  TERRA) or potent protein barriers (a circle with question mark). The replicative
  helicase is pictured as a red ring, RNA polymerase is a yellow oval and DNA polymerases
  are green and blue torpedoes with rings on their rear ends representing PCNA. One-ended
  DSBs are then channeled into the RAD52-dependent DNA damage repair pathway. This
  pathway is either RAD51-dependent, which involves invasion into a homologous or
  homeologous duplex, or RAD51-independent, which involves single-strand annealing.
  (These two pathways lead to the formation of Type I and Type II survivors in telomerase-deficient
  S. cerevisiae, respectively.) Upon template switching, the replisome is assembled
  and BIR-dependent synthesis of the telomere proceeds. BIR at telomeres requires
  DNA polymerase δ, its accessory subunit POLD3 (Pol32 in S. cerevisiae) as well as
  Replication Factor C (RFC) and PCNA (see text for details). The source of templates
  for RAD51-independent annealing is not limited to single-stranded regions released
  after G-quadruplex formation in the complimentary strand. For a comprehensive review
  of all eligible substrates see []. Importantly, such templates may be represented
  by extrachromosomal telomeric circular DNAs believed to be generated from the resolution
  of recombination intermediates at telomeres. Other sources for template DNA during
  BIR-dependent telomere-lengthening could be T-loops or interstitial telomeric sequences
  (ITSs) (for further discussion of these issues see ).
pmcid: PMC6410037
papertitle: 'At the Beginning of the End and in the Middle of the Beginning: Structure
  and Maintenance of Telomeric DNA Repeats and Interstitial Telomeric Sequences.'
reftext: Anna Y. Aksenova, et al. Genes (Basel). 2019 Feb;10(2):118.
pmc_ranked_result_index: '230741'
pathway_score: 0.6826566
filename: genes-10-00118-g002.jpg
figtitle: Telomere length maintenance mechanisms
year: '2019'
organisms:
- Homo sapiens
ndex: 504bb93a-df32-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6410037__genes-10-00118-g002.html
  '@type': Dataset
  description: Telomere length maintenance mechanisms. (A) A simplified view of telomere
    length control by telomerase and the CST-complex. Telomerase adds telomeric repeats
    to the 3’-end of the G-rich overhang. Its activity is controlled by the CST-complex,
    which displaces telomerase, removes secondary structures and recruits the DNA
    polymerase α/primase complex to synthesize the C-rich strand. Note that regulation
    of telomerase by the CST-complex is more complex in yeast Saccharomyces cerevisiae
    (see text for details). (B) A general representation of the Alternative Lengthening
    of Telomeres (ALT) mechanism. ALT is currently viewed as the BIR-dependent elongation
    of telomeres after the replication fork stalling, ultimately leading to the formation
    of a one-ended double-strand break. This fork stalling can be due to the formation
    of unusual secondary structures in telomeric DNA (such as G-quadruplexes), the
    presence of stable DNA-RNA structures (R-loops generated during transcription
    of telomere-specific RNA called TERRA) or potent protein barriers (a circle with
    question mark). The replicative helicase is pictured as a red ring, RNA polymerase
    is a yellow oval and DNA polymerases are green and blue torpedoes with rings on
    their rear ends representing PCNA. One-ended DSBs are then channeled into the
    RAD52-dependent DNA damage repair pathway. This pathway is either RAD51-dependent,
    which involves invasion into a homologous or homeologous duplex, or RAD51-independent,
    which involves single-strand annealing. (These two pathways lead to the formation
    of Type I and Type II survivors in telomerase-deficient S. cerevisiae, respectively.)
    Upon template switching, the replisome is assembled and BIR-dependent synthesis
    of the telomere proceeds. BIR at telomeres requires DNA polymerase δ, its accessory
    subunit POLD3 (Pol32 in S. cerevisiae) as well as Replication Factor C (RFC) and
    PCNA (see text for details). The source of templates for RAD51-independent annealing
    is not limited to single-stranded regions released after G-quadruplex formation
    in the complimentary strand. For a comprehensive review of all eligible substrates
    see []. Importantly, such templates may be represented by extrachromosomal telomeric
    circular DNAs believed to be generated from the resolution of recombination intermediates
    at telomeres. Other sources for template DNA during BIR-dependent telomere-lengthening
    could be T-loops or interstitial telomeric sequences (ITSs) (for further discussion
    of these issues see ).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - POLD1
  - RAD51
  - RFC1
  - RFC3
  - RFC4
  - RFC2
  - KCNJ11
  - POLD3
  - TERT
  - RAD52
  - ACOT1
  - DKC1
  - PCNA
  - POLA1
  - CTC1
  - RFC5
  - STN1
  - TEN1
genes:
- word: Poldelta
  symbol: POLD
  source: hgnc_prev_symbol
  hgnc_symbol: POLD1
  entrez: '5424'
- word: RAD51-dependent
  symbol: RAD51
  source: hgnc_symbol
  hgnc_symbol: RAD51
  entrez: '5888'
- word: RFC
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC1
  entrez: '5981'
- word: RFC
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC3
  entrez: '5983'
- word: RFC
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC4
  entrez: '5984'
- word: RFC
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC2
  entrez: '5982'
- word: BIR-dependent
  symbol: BIR
  source: hgnc_alias_symbol
  hgnc_symbol: KCNJ11
  entrez: '3767'
- word: POLD3/Pol32
  symbol: POLD3
  source: hgnc_symbol
  hgnc_symbol: POLD3
  entrez: '10714'
- word: TERT
  symbol: TERT
  source: hgnc_symbol
  hgnc_symbol: TERT
  entrez: '7015'
- word: RAD52-dependent
  symbol: RAD52
  source: hgnc_symbol
  hgnc_symbol: RAD52
  entrez: '5893'
- word: Cte1
  symbol: CTE-1
  source: hgnc_alias_symbol
  hgnc_symbol: ACOT1
  entrez: '641371'
- word: dyskerin
  symbol: dyskerin
  source: hgnc_alias_symbol
  hgnc_symbol: DKC1
  entrez: '1736'
- word: PCNA
  symbol: PCNA
  source: hgnc_symbol
  hgnc_symbol: PCNA
  entrez: '5111'
- word: Pola/Primase
  symbol: POLA
  source: hgnc_prev_symbol
  hgnc_symbol: POLA1
  entrez: '5422'
- word: Ctc1
  symbol: CTC1
  source: hgnc_symbol
  hgnc_symbol: CTC1
  entrez: '80169'
- word: RFC
  symbol: RFC
  source: bioentities_symbol
  hgnc_symbol: RFC5
  entrez: '5985'
- word: Stn1
  symbol: STN1
  source: hgnc_symbol
  hgnc_symbol: STN1
  entrez: '79991'
- word: Ten1
  symbol: TEN1
  source: hgnc_symbol
  hgnc_symbol: TEN1
  entrez: '100134934'
chemicals: []
diseases: []
figid_alias: PMC6410037__F2
redirect_from: /figures/PMC6410037__F2
figtype: Figure
---
