---
figid: PMC6330604__atm-06-S2-S122-f1
figlink: /pmc/articles/PMC6330604/figure/f1/
number: Figure 1
caption: 'DSBR pathways and the effect of RAD51 paralog loss on pathway choice. (A)
  Schematic of homologous recombination. Double strand breaks are detected by the
  sensor protein MDC1, which recruits the ATM dimer. ATM autophosphorylates, becomes
  a monomer and then phosphorylates H2A.X, which serves as a recruitment signal for
  additional complexes including exonucleases. Exonucleases resect the broken ends
  in a 5'' to 3'' direction to generate 3'' overhanging ssDNA fibres, which are then
  coated with RPA. BRCA2 loads RAD51 onto the fibres by exchanging it with RPA. The
  RAD51-ssDNA fibres are remodelled and stabilized by RAD51 paralogs, and filaments
  are engaged in homology searching the genome. RAD51 catalyses the annealing of the
  ssDNA fibres to their homologous template for resynthesis. The junction structure
  is then resolved and ligated to generate intact duplex. (B) Schematic of alternative
  DSBR mechanisms: (i) alt-NHEJ. Binding of the KU70/80 heterodimer to the break recruits
  DNA-PK or PARP-1 and their associated end processing factors. Once 3'' hydroxyl
  and 5'' phosphate termini are restored ends are blunt-ligated; (ii) SSA. Regions
  of microhomology between ssDNA fibres are annealed, generating 3'' overhanging flaps.
  These are cleaved by structure-specific endonucleases and the remaining gaps are
  filled and ligated; (iii) SDSA. Single strand invasion occurs as in HR, but only
  one end of a break aligns with a homologous template for polymerase extension. The
  newly synthesized DNA is then melted from its template and annealed to the existing
  3'' ssDNA overhang on the other break end; (iv) effect of RAD51 loading defects
  on pathway choice. RAD51 paralog or BRCA2 deficiency lead to instability or loss
  of RAD51 ssDNA fibres. Without RAD51 fibres HR cannot be completed and alternative
  error prone repair mechanisms are engaged. In these circumstances PARP-dependent
  alt-NHEJ is a critical repair mechanism. Chemical inhibition of PARP causes an overuse
  of classical DNA-PK driven NHEJ with subsequent cytotoxic deletions and chromosome
  structural aberrations. NHEJ, non-homologous end joining; ssDNA, single-stranded
  DNA; SSBR, single strand break repair; DSBR, double strand break repair; RPA, replication
  protein A; SSA, single strand annealing; SDSA, synthesis dependent strand annealing;
  HR, homologous recombination; DDR, DNA damage repair.'
pmcid: PMC6330604
papertitle: RAD51 paralogs promote genomic integrity and chemoresistance in cancer
  by facilitating homologous recombination.
reftext: Janelle Louise Harris, et al. Ann Transl Med. 2018 Dec;6(Suppl 2):S122.
pmc_ranked_result_index: '188253'
pathway_score: 0.9319291
filename: atm-06-S2-S122-f1.jpg
figtitle: DSBR pathways and the effect of RAD51 paralog loss on pathway choice
year: '2018'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6330604__atm-06-S2-S122-f1.html
  '@type': Dataset
  description: 'DSBR pathways and the effect of RAD51 paralog loss on pathway choice.
    (A) Schematic of homologous recombination. Double strand breaks are detected by
    the sensor protein MDC1, which recruits the ATM dimer. ATM autophosphorylates,
    becomes a monomer and then phosphorylates H2A.X, which serves as a recruitment
    signal for additional complexes including exonucleases. Exonucleases resect the
    broken ends in a 5'' to 3'' direction to generate 3'' overhanging ssDNA fibres,
    which are then coated with RPA. BRCA2 loads RAD51 onto the fibres by exchanging
    it with RPA. The RAD51-ssDNA fibres are remodelled and stabilized by RAD51 paralogs,
    and filaments are engaged in homology searching the genome. RAD51 catalyses the
    annealing of the ssDNA fibres to their homologous template for resynthesis. The
    junction structure is then resolved and ligated to generate intact duplex. (B)
    Schematic of alternative DSBR mechanisms: (i) alt-NHEJ. Binding of the KU70/80
    heterodimer to the break recruits DNA-PK or PARP-1 and their associated end processing
    factors. Once 3'' hydroxyl and 5'' phosphate termini are restored ends are blunt-ligated;
    (ii) SSA. Regions of microhomology between ssDNA fibres are annealed, generating
    3'' overhanging flaps. These are cleaved by structure-specific endonucleases and
    the remaining gaps are filled and ligated; (iii) SDSA. Single strand invasion
    occurs as in HR, but only one end of a break aligns with a homologous template
    for polymerase extension. The newly synthesized DNA is then melted from its template
    and annealed to the existing 3'' ssDNA overhang on the other break end; (iv) effect
    of RAD51 loading defects on pathway choice. RAD51 paralog or BRCA2 deficiency
    lead to instability or loss of RAD51 ssDNA fibres. Without RAD51 fibres HR cannot
    be completed and alternative error prone repair mechanisms are engaged. In these
    circumstances PARP-dependent alt-NHEJ is a critical repair mechanism. Chemical
    inhibition of PARP causes an overuse of classical DNA-PK driven NHEJ with subsequent
    cytotoxic deletions and chromosome structural aberrations. NHEJ, non-homologous
    end joining; ssDNA, single-stranded DNA; SSBR, single strand break repair; DSBR,
    double strand break repair; RPA, replication protein A; SSA, single strand annealing;
    SDSA, synthesis dependent strand annealing; HR, homologous recombination; DDR,
    DNA damage repair.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TIPARP
  - TNKS2
  - BRCA2
  - PARP15
  - DDR2
  - ATM
  - PARP14
  - RBBP8
  - ENG
  - DDR1
  - NSD1
  - MDC1
  - MRE11
  - PARP9
  - PARP2
  - PARP3
  - PARP1
  - PARP4
  - RRAD
  - RAD51
  - RPA1
  - PARP16
  - RUNX1T1
  - RASA1
  - PARP11
  - PARP12
  - PARP8
  - PARP10
  - TNKS
  - LNPEP
  - RPA3
  - CALR
  - PARP6
  - RPA2
  - PRKDC
  - RAD
genes:
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: TIPARP
  entrez: '25976'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: TNKS2
  entrez: '80351'
- word: BRCA2
  symbol: BRCA2
  source: hgnc_symbol
  hgnc_symbol: BRCA2
  entrez: '675'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP15
  entrez: '165631'
- word: DDR
  symbol: DDR
  source: bioentities_symbol
  hgnc_symbol: DDR2
  entrez: '4921'
- word: ATM
  symbol: ATM
  source: hgnc_symbol
  hgnc_symbol: ATM
  entrez: '472'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP14
  entrez: '54625'
- word: CTIP)
  symbol: CtIP
  source: hgnc_alias_symbol
  hgnc_symbol: RBBP8
  entrez: '5932'
- word: ends
  symbol: END
  source: hgnc_alias_symbol
  hgnc_symbol: ENG
  entrez: '2022'
- word: DDR
  symbol: DDR
  source: bioentities_symbol
  hgnc_symbol: DDR1
  entrez: '780'
- word: S'to
  symbol: STO
  source: hgnc_prev_symbol
  hgnc_symbol: NSD1
  entrez: '64324'
- word: MDC1
  symbol: MDC1
  source: hgnc_symbol
  hgnc_symbol: MDC1
  entrez: '9656'
- word: (MRE11,CTIP)
  symbol: MRE11
  source: hgnc_symbol
  hgnc_symbol: MRE11
  entrez: '4361'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP9
  entrez: '83666'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP2
  entrez: '10038'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP3
  entrez: '10039'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP1
  entrez: '142'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP4
  entrez: '143'
- word: RAD
  symbol: RAD
  source: hgnc_alias_symbol
  hgnc_symbol: RRAD
  entrez: '6236'
- word: RAD51
  symbol: RAD51
  source: hgnc_symbol
  hgnc_symbol: RAD51
  entrez: '5888'
- word: RPA
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA1
  entrez: '6117'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP16
  entrez: '54956'
- word: eto.
  symbol: ETO
  source: hgnc_alias_symbol
  hgnc_symbol: RUNX1T1
  entrez: '862'
- word: Gap
  symbol: GAP
  source: hgnc_alias_symbol
  hgnc_symbol: RASA1
  entrez: '5921'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP11
  entrez: '57097'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP12
  entrez: '64761'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP8
  entrez: '79668'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP10
  entrez: '84875'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: TNKS
  entrez: '8658'
- word: PARP-1
  symbol: PARP-1
  source: hgnc_alias_symbol
  hgnc_symbol: TIPARP
  entrez: '25976'
- word: PLAP
  symbol: P-LAP
  source: hgnc_alias_symbol
  hgnc_symbol: LNPEP
  entrez: '4012'
- word: RPA
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA3
  entrez: '6119'
- word: SSA
  symbol: SSA
  source: hgnc_alias_symbol
  hgnc_symbol: CALR
  entrez: '811'
- word: PARP
  symbol: PARP
  source: bioentities_symbol
  hgnc_symbol: PARP6
  entrez: '56965'
- word: RPA
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA2
  entrez: '6118'
- word: DNA-PK
  symbol: DNAPK
  source: hgnc_alias_symbol
  hgnc_symbol: PRKDC
  entrez: '5591'
chemicals: []
diseases:
- word: RAD
  source: MESH
  identifier: C535729
---
