---
figid: PMC6295671__nihms-999703-f0001
figlink: /pmc/articles/PMC6295671/figure/F1/
number: Figure 1
caption: Identification of neoantigen-reactive T cells from patients with cancer.
  Next-generation sequencing (whole exome and whole transcriptome) is performed on
  tumor and matched normal cells to identify non-synonymous somatic mutations expressed
  by the cancer (left). Next, two approaches that do not rely on predictions of HLA–peptide
  binding can be used to investigate the reactivity of T cells to neoantigens encoded
  by the identified mutations. In the first approach (middle), minigenes encoding
  the mutation flanked by nucleotides encoding 12 amino acids from the wild-type gene
  can be synthesized in tandem to create TMG constructs, which are then cloned into
  an appropriate expression vector. Linking multiple minigenes in tandem allows a
  relatively large number of mutations to be evaluated at once. Plasmids encoding
  TMGs or TMG RNAs transcribed in vitro are then introduced into the appropriate antigen-presenting
  cells (APCs), such as autologous dendritic cells or B cells, through techniques
  such as electroporation or lipid-based transfection, to allow processing and presentation
  of the neoantigens in the context of the patient’s own HLA class I and II molecules.
  T cells derived from tumor (TILs) or from the blood (right) are then co-cultured
  with the antigen-presenting cells expressing the TMGs, and T cell reactivity is
  evaluated by immunological methods such as cytokine ELISPOT or ELISA or the analysis
  of T cell–activation molecules such as CD137 (4–1BB) or CD134 (OX40) by flow cytometry
  (far right). The second approach (bottom) is identical to the first approach, except
  that instead of genetic constructs encoding the mutations, long peptides containing
  the mutant amino acid flanked by 12 amino acids from the wild-type protein are synthesized
  and then pulsed onto antigen-presenting cells, which process and present the mutant
  peptides to T cells. Similar to the minigenes and TMG concept, in this approach,
  a variable number of individual long peptides can be combined to generate peptide
  pools, which increases the throughput of neoantigen screening.
pmcid: PMC6295671
papertitle: '‘Final common pathway’ of human cancer immunotherapy: targeting random
  somatic mutations.'
reftext: Eric Tran, et al. Nat Immunol. 2017 Feb 15;18(3):255-262.
pmc_ranked_result_index: '227621'
pathway_score: 0.8639321
filename: nihms-999703-f0001.jpg
figtitle: Identification of neoantigen-reactive T cells from patients with cancer
year: '2017'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6295671__nihms-999703-f0001.html
  '@type': Dataset
  description: Identification of neoantigen-reactive T cells from patients with cancer.
    Next-generation sequencing (whole exome and whole transcriptome) is performed
    on tumor and matched normal cells to identify non-synonymous somatic mutations
    expressed by the cancer (left). Next, two approaches that do not rely on predictions
    of HLA–peptide binding can be used to investigate the reactivity of T cells to
    neoantigens encoded by the identified mutations. In the first approach (middle),
    minigenes encoding the mutation flanked by nucleotides encoding 12 amino acids
    from the wild-type gene can be synthesized in tandem to create TMG constructs,
    which are then cloned into an appropriate expression vector. Linking multiple
    minigenes in tandem allows a relatively large number of mutations to be evaluated
    at once. Plasmids encoding TMGs or TMG RNAs transcribed in vitro are then introduced
    into the appropriate antigen-presenting cells (APCs), such as autologous dendritic
    cells or B cells, through techniques such as electroporation or lipid-based transfection,
    to allow processing and presentation of the neoantigens in the context of the
    patient’s own HLA class I and II molecules. T cells derived from tumor (TILs)
    or from the blood (right) are then co-cultured with the antigen-presenting cells
    expressing the TMGs, and T cell reactivity is evaluated by immunological methods
    such as cytokine ELISPOT or ELISA or the analysis of T cell–activation molecules
    such as CD137 (4–1BB) or CD134 (OX40) by flow cytometry (far right). The second
    approach (bottom) is identical to the first approach, except that instead of genetic
    constructs encoding the mutations, long peptides containing the mutant amino acid
    flanked by 12 amino acids from the wild-type protein are synthesized and then
    pulsed onto antigen-presenting cells, which process and present the mutant peptides
    to T cells. Similar to the minigenes and TMG concept, in this approach, a variable
    number of individual long peptides can be combined to generate peptide pools,
    which increases the throughput of neoantigen screening.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CD8A
  - CD4
  - APCS
  - BCR
  - TNFRSF4
  - CD8B
  - TNFRSF9
genes:
- word: CD8
  symbol: CD8
  source: bioentities_symbol
  hgnc_symbol: CD8A
  entrez: '925'
- word: CD4
  symbol: CD4
  source: hgnc_symbol
  hgnc_symbol: CD4
  entrez: '920'
- word: APCS
  symbol: APCS
  source: hgnc_symbol
  hgnc_symbol: APCS
  entrez: '325'
- word: all
  symbol: ALL
  source: hgnc_alias_symbol
  hgnc_symbol: BCR
  entrez: '613'
- word: OX40
  symbol: OX40
  source: hgnc_alias_symbol
  hgnc_symbol: TNFRSF4
  entrez: '7293'
- word: CD8
  symbol: CD8
  source: bioentities_symbol
  hgnc_symbol: CD8B
  entrez: '926'
- word: 4-1BB
  symbol: 4-1BB
  source: hgnc_alias_symbol
  hgnc_symbol: TNFRSF9
  entrez: '3604'
chemicals: []
diseases: []
---
