---
figid: PMC6255977__fpls-09-01687-g001
figlink: /pmc/articles/PMC6255977/figure/F1/
number: F1
caption: 'Transcriptome response of Salvia miltiorrhiza to overexpression of AtPAP1.
  (A) Comparison of gene expression between two libraries, with each normalized to
  one million tags. Red dots represent transcripts more prevalent in AtPAP1-14 library;
  green dots, those present at lower frequency in transgenic plants; and blue dots,
  transcript levels with no significant change. Parameters “FDR < 0.001” and “log2Ratio
  ≥ 1” were used as threshold to judge significance of differences in expression.
  (B) Proposed biosynthetic pathway for phenolics, beginning with core phenylpropanoid
  and tyrosine pathways, and leading to three major branch pathways: phenolic acid,
  flavonoid, and lignin (red, upregulated genes; green, downregulated). TAT, tyrosine
  aminotransferase; HPPR, hydroxyl phenylpyruvate reductase; PAL, phenylalanine ammonia
  lyase; C4H, cinnamate 4-hydroxylase; 4CL, hydroxycinnamate-CoA ligase; RAS, rosmarinic
  acid synthase; CYP, cytochrome P450 enzymes; CHS, chalcone synthase; CHI, chalcone
  isomerase; FLS, flavonol synthase; F3H, flavanone 3-hydroxylase; F3’H, flavonoid
  3’-hydroxylase; F3′5′H, flavonoid 3′,5′-hydroxylase; DFR, dihydroflavonol 4-reductase;
  ANS, anthocyanin synthase; HCT, hydroxyl cinnamoyl transferase; CCR, cinnamoyl-CoA
  reductase; C3’ H, coumarate 3’-hydroxylase; COMT, caffeic acid O-methyltransferase;
  CCoAMT, caffeoyl-CoA O-methyltransferase; F5H, ferulate 5-hydroxylase; GT, glycosyltransferase;
  CAD, cinnamyl alcohol dehydrogenase.'
pmcid: PMC6255977
papertitle: AtPAP1 Interacts With and Activates SmbHLH51, a Positive Regulator to
  Phenolic Acids Biosynthesis in Salvia miltiorrhiza.
reftext: Yucui Wu, et al. Front Plant Sci. 2018;9:1687.
pmc_ranked_result_index: '86572'
pathway_score: 0.510411
filename: fpls-09-01687-g001.jpg
figtitle: Transcriptome response of Salvia miltiorrhiza to overexpression of AtPAP1
year: '2018'
organisms: Homo sapiens
ndex: 9ab4ebee-de96-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6255977__fpls-09-01687-g001.html
  '@type': Dataset
  description: 'Transcriptome response of Salvia miltiorrhiza to overexpression of
    AtPAP1. (A) Comparison of gene expression between two libraries, with each normalized
    to one million tags. Red dots represent transcripts more prevalent in AtPAP1-14
    library; green dots, those present at lower frequency in transgenic plants; and
    blue dots, transcript levels with no significant change. Parameters “FDR < 0.001”
    and “log2Ratio ≥ 1” were used as threshold to judge significance of differences
    in expression. (B) Proposed biosynthetic pathway for phenolics, beginning with
    core phenylpropanoid and tyrosine pathways, and leading to three major branch
    pathways: phenolic acid, flavonoid, and lignin (red, upregulated genes; green,
    downregulated). TAT, tyrosine aminotransferase; HPPR, hydroxyl phenylpyruvate
    reductase; PAL, phenylalanine ammonia lyase; C4H, cinnamate 4-hydroxylase; 4CL,
    hydroxycinnamate-CoA ligase; RAS, rosmarinic acid synthase; CYP, cytochrome P450
    enzymes; CHS, chalcone synthase; CHI, chalcone isomerase; FLS, flavonol synthase;
    F3H, flavanone 3-hydroxylase; F3’H, flavonoid 3’-hydroxylase; F3′5′H, flavonoid
    3′,5′-hydroxylase; DFR, dihydroflavonol 4-reductase; ANS, anthocyanin synthase;
    HCT, hydroxyl cinnamoyl transferase; CCR, cinnamoyl-CoA reductase; C3’ H, coumarate
    3’-hydroxylase; COMT, caffeic acid O-methyltransferase; CCoAMT, caffeoyl-CoA O-methyltransferase;
    F5H, ferulate 5-hydroxylase; GT, glycosyltransferase; CAD, cinnamyl alcohol dehydrogenase.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CS
  - PDXP
  - COMT
  - CAD
  - CGA
  - NRAS
  - FSHB
  - HRAS
  - CCR2
  - KRAS
  - phosphate
  - Phenylpropanold
  - Phenolic acid
  - Tyrosine
  - Lphenylalanine
  - Cinnamate
  - 4Hydranyphanylpyreie acid CAN
  - Cu
  - 4Hydrenyphenylare acid
  - FAN
  - CHI
  - ethoeyanina
  - PAPI-14<TPM
  - CIN
genes:
- word: CS
  symbol: CS
  source: hgnc_symbol
  hgnc_symbol: CS
  entrez: '1431'
- word: CIN
  symbol: CIN
  source: hgnc_alias_symbol
  hgnc_symbol: PDXP
  entrez: '57026'
- word: COMT
  symbol: COMT
  source: hgnc_symbol
  hgnc_symbol: COMT
  entrez: '1312'
- word: CAD
  symbol: CAD
  source: hgnc_symbol
  hgnc_symbol: CAD
  entrez: '790'
- word: FSH
  symbol: FSH
  source: bioentities_symbol
  hgnc_symbol: CGA
  entrez: '1081'
- word: RAS
  symbol: RAS
  source: bioentities_symbol
  hgnc_symbol: NRAS
  entrez: '4893'
- word: FSH
  symbol: FSH
  source: bioentities_symbol
  hgnc_symbol: FSHB
  entrez: '2488'
- word: RAS
  symbol: RAS
  source: bioentities_symbol
  hgnc_symbol: HRAS
  entrez: '3265'
- word: CCR2
  symbol: CCR2
  source: hgnc_symbol
  hgnc_symbol: CCR2
  entrez: '729230'
- word: RAS
  symbol: RAS
  source: bioentities_symbol
  hgnc_symbol: KRAS
  entrez: '3845'
chemicals:
- word: phosphate
  source: MESH
  identifier: D010710
- word: Phenylpropanold
  source: ''
  identifier: ''
- word: Phenolic acid
  source: MESH
  identifier: C017616
- word: Tyrosine
  source: ''
  identifier: ''
- word: Lphenylalanine
  source: ''
  identifier: ''
- word: Cinnamate
  source: MESH
  identifier: D002934
- word: 4Hydranyphanylpyreie acid CAN
  source: MESH
  identifier: C000596933
- word: Cu
  source: MESH
  identifier: D003300
- word: 4Hydrenyphenylare acid
  source: MESH
  identifier: C082768
- word: FAN
  source: MESH
  identifier: C110072
- word: CHI
  source: MESH
  identifier: C084533
- word: ethoeyanina
  source: ''
  identifier: ''
- word: PAPI-14<TPM
  source: MESH
  identifier: C112586
diseases:
- word: CIN
  source: MESH
  identifier: D007674
figid_alias: PMC6255977__F1
redirect_from: /figures/PMC6255977__F1
figtype: Figure
---
