---
figid: PMC6307301__40168_2018_613_Fig6_HTML
figlink: /pmc/articles/PMC6307301/figure/Fig6/
number: F6
caption: 'Comparative genomics analysis of correlations between Marinilabiliales microbes
  and the microorganisms enriched during enrichment culturing. Overview of biotin
  biosynthesis and precursor salvage. Biosynthetic enzymes: BioH, pimeloyl-[acyl-carrier
  protein] methyl ester esterase; BioC, malonyl-CoA O-methyltransferase; BioF, 8-amino-7-oxononanoate
  synthase; BioA, adenosylmethionine-8-amino-7-oxononanoate aminotransferase; BioD,
  dethiobiotin synthetase; BioB, biotin synthase; and BirA, biotin-[acetyl-CoA-carboxylase]
  ligase. Biotin translocases include the BioYM ECF-type transporter. The auxotrophic
  groups, which mainly contain Marinilabiliales bacteria, are unable to synthesize
  biotin de novo and are missing most of the related pathway genes. Phototrophic groups,
  which mainly contain Vibrionaceae, Clostridiaceae, Desulfobulbaceae, Desulfovibrionaceae,
  and Desulfobacteraceae bacteria, can synthesize biotin de novo. The Desulfobacteraceae
  node has no correlations with the Prolixibacteraceae node and the Marinilabiliaceae
  node; however, the bacteria within this family were highly enriched and became the
  dominant family during enrichment culturing. We also analyzed the genomes of this
  family’s members. Pie slices indicate the total fragments per kilobase of transcript
  per million fragments mapped (FPKM) of the corresponding gene in each sediment sample'
pmcid: PMC6307301
papertitle: Metatranscriptomic and comparative genomic insights into resuscitation
  mechanisms during enrichment culturing.
reftext: Da-Shuai Mu, et al. Microbiome. 2018;6:230.
pmc_ranked_result_index: '96858'
pathway_score: 0.9122996
filename: 40168_2018_613_Fig6_HTML.jpg
figtitle: Comparative genomics analysis of correlations between Marinilabiliales microbes
  and the microorganisms enriched during enrichment culturing
year: '2018'
organisms:
- Draconibacterium orientale
- Diaporthe sclerotioides
- Bacillus subtilis At2
- mixed culture
- Salinivirga cyanobacteriivorans
- unidentified marine bacterioplankton
- Bacillus subtilis
- Clostridium pasteurianum
- Bradymonas sediminis
- enrichment culture
- enrichment cultures
- Ancylomarina subtilis
- Mycobacterium tuberculosis
- metagenome
- Pseudomonas aeruginosa
- Bacteroides sp.
- Populus deltoides
- Halomonas denitrificans
- Saccharomyces cerevisiae
- Escherichia coli
- human metagenome
- Vibrio cholerae
- gut metagenome
- uncultured bacterium
- Escherichia coli O157:H7
- Nitrosomonas europaea
- Homo sapiens
- Marinobacter xestospongiae
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6307301__40168_2018_613_Fig6_HTML.html
  '@type': Dataset
  description: 'Comparative genomics analysis of correlations between Marinilabiliales
    microbes and the microorganisms enriched during enrichment culturing. Overview
    of biotin biosynthesis and precursor salvage. Biosynthetic enzymes: BioH, pimeloyl-[acyl-carrier
    protein] methyl ester esterase; BioC, malonyl-CoA O-methyltransferase; BioF, 8-amino-7-oxononanoate
    synthase; BioA, adenosylmethionine-8-amino-7-oxononanoate aminotransferase; BioD,
    dethiobiotin synthetase; BioB, biotin synthase; and BirA, biotin-[acetyl-CoA-carboxylase]
    ligase. Biotin translocases include the BioYM ECF-type transporter. The auxotrophic
    groups, which mainly contain Marinilabiliales bacteria, are unable to synthesize
    biotin de novo and are missing most of the related pathway genes. Phototrophic
    groups, which mainly contain Vibrionaceae, Clostridiaceae, Desulfobulbaceae, Desulfovibrionaceae,
    and Desulfobacteraceae bacteria, can synthesize biotin de novo. The Desulfobacteraceae
    node has no correlations with the Prolixibacteraceae node and the Marinilabiliaceae
    node; however, the bacteria within this family were highly enriched and became
    the dominant family during enrichment culturing. We also analyzed the genomes
    of this family’s members. Pie slices indicate the total fragments per kilobase
    of transcript per million fragments mapped (FPKM) of the corresponding gene in
    each sediment sample'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CYTH1
  - Biotin
  - pimeloyl
  - methyl ester
  - pimyeloyl-lacp
  - fatty acids
  - pimyeloyl
  - 8-amino-7-oxononanoate
  - 8-dlaminononanoate
  - 8-diaminononanoate
  - dethiobiotin
  - biotin
  - biotinyl
  - Coenzyme A
  - JCM 21150
  - O DOO
genes:
- word: B2-1
  symbol: B2-1
  source: hgnc_alias_symbol
  hgnc_symbol: CYTH1
  entrez: '9267'
chemicals:
- word: Biotin
  source: MESH
  identifier: D001710
- word: pimeloyl
  source: MESH
  identifier: C019527
- word: methyl ester
  source: MESH
  identifier: D004952
- word: pimyeloyl-lacp
  source: ''
  identifier: ''
- word: fatty acids
  source: MESH
  identifier: D005227
- word: pimyeloyl
  source: ''
  identifier: ''
- word: 8-amino-7-oxononanoate
  source: MESH
  identifier: D000596
- word: 8-dlaminononanoate
  source: MESH
  identifier: C090700
- word: 8-diaminononanoate
  source: MESH
  identifier: C090700
- word: dethiobiotin
  source: MESH
  identifier: C004749
- word: biotin
  source: MESH
  identifier: D001710
- word: biotinyl
  source: MESH
  identifier: C025360
- word: Coenzyme A
  source: MESH
  identifier: D003067
- word: JCM 21150
  source: MESH
  identifier: C549084
- word: O DOO
  source: MESH
  identifier: D013481
diseases: []
figid_alias: PMC6307301__F6
redirect_from: /figures/PMC6307301__F6
figtype: Figure
---
