---
figid: PMC6166676__zjw0101857910001
figlink: /pmc/articles/PMC6166676/figure/F1/
number: F1
caption: Study design. A, Biological model. Schematic representation of the MAPK signaling
  pathway whereby BRAF(V600E), the drugs and KO target sites are highlighted. B, Experimental
  design. WiDr and WiDr PTPN11 KO cells were cultured for transcriptome, proteome
  and phosphoproteome analysis. For each of the six treatments the time-course of
  events is indicated (Experimental Procedures). Color codes are utilized to differentiate
  the effect due to drug inhibition in PTPN11 WT with respect to the knock-out of
  PTPN11 KO by distinguishing BRAFi-treated samples (red and shades of purple) from
  not BRAFi-treated samples (gray and shades of blue). C, Workflow employed in this
  study. For the transcriptomics analysis, mRNA libraries were prepared and sequenced
  using Illumina NextSeq. Quantitation in proteomics and phosphoproteomics analysis
  was done by label-free quantitation. Phosphoproteomics was performed after Ti4+-IMAC
  phosphopeptide enrichment. For the integration of transcriptome, proteome and phosphoproteome
  approach, a customized protein database was developed to include identified missense
  mutations for peptide identification and quantification (Experimental Procedures).
pmcid: PMC6166676
papertitle: A System-wide Approach to Monitor Responses to Synergistic BRAF and EGFR
  Inhibition in Colorectal Cancer Cells.
reftext: Anna Ressa, et al. Mol Cell Proteomics. 2018 Oct;17(10):1892-1908.
pmc_ranked_result_index: '24555'
pathway_score: 0.6443132
filename: zjw0101857910001.jpg
figtitle: Study design
year: '2018'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6166676__zjw0101857910001.html
  '@type': Dataset
  description: Study design. A, Biological model. Schematic representation of the
    MAPK signaling pathway whereby BRAF(V600E), the drugs and KO target sites are
    highlighted. B, Experimental design. WiDr and WiDr PTPN11 KO cells were cultured
    for transcriptome, proteome and phosphoproteome analysis. For each of the six
    treatments the time-course of events is indicated (Experimental Procedures). Color
    codes are utilized to differentiate the effect due to drug inhibition in PTPN11
    WT with respect to the knock-out of PTPN11 KO by distinguishing BRAFi-treated
    samples (red and shades of purple) from not BRAFi-treated samples (gray and shades
    of blue). C, Workflow employed in this study. For the transcriptomics analysis,
    mRNA libraries were prepared and sequenced using Illumina NextSeq. Quantitation
    in proteomics and phosphoproteomics analysis was done by label-free quantitation.
    Phosphoproteomics was performed after Ti4+-IMAC phosphopeptide enrichment. For
    the integration of transcriptome, proteome and phosphoproteome approach, a customized
    protein database was developed to include identified missense mutations for peptide
    identification and quantification (Experimental Procedures).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - L1CAM
  - PTPN11
  - CXXC5
  - ETNK1
  - Lys
  - Urea
  - phospho
genes:
- word: s10
  symbol: S10
  source: hgnc_prev_symbol
  hgnc_symbol: L1CAM
  entrez: '3897'
- word: PTPN11
  symbol: PTPN11
  source: hgnc_symbol
  hgnc_symbol: PTPN11
  entrez: '5781'
- word: WID
  symbol: WID
  source: hgnc_alias_symbol
  hgnc_symbol: CXXC5
  entrez: '51523'
- word: EKI
  symbol: EKI
  source: hgnc_alias_symbol
  hgnc_symbol: ETNK1
  entrez: '55500'
chemicals:
- word: Lys
  source: MESH
  identifier: C026591
- word: Urea
  source: MESH
  identifier: D014508
- word: phospho
  source: MESH
  identifier: C033601
diseases: []
figid_alias: PMC6166676__F1
redirect_from: /figures/PMC6166676__F1
figtype: Figure
---
