---
figid: PMC6391339__fgene-10-00125-g001
figlink: /pmc/articles/PMC6391339/figure/F1/
number: F1
caption: Biogenesis of miRNAs. In the canonical pathway, miRNAs are transcribed from
  their loci by RNA polymerase II into a long primary transcript of about 80 nucleotides
  called the pri-miRNA. Cleavage follows and is done by Drosha, a type III RNase along
  with the DGCR8 protein to produce pre-miRNA. In the non-canonical pathway, Mirtrons
  are spliced by the spliceosome to form looped intermediates referred to as lariat
  which then refold into pre-miRNAs. Next, the exportin 5, a RAN-GTP dependent transporter,
  mediates the movement of pre-miRNAs from the nucleus into the cytoplasm. Further
  processing by Dicer and TARBP2 protein generates mature miRNAs, producing double-stranded
  structure of miRNA of about 21-22 nt in length. The duplex is loaded into an AGO
  protein. The passenger strand (miRNA∗) is degraded, whereas the guide strand is
  incorporated by the Ago into the miRNA-induced silencing complex (miRISC). In animals,
  imperfect complementarity occurs when the miRNA seed region, nucleotides 2-8, BPs
  perfectly with the complementary seed match site in the 3′ UTR of the target mRNA
  resulting in translational repression or degradation. GW182 a core component of
  miRISC, mediates deadenylation of mRNAs by interacting with AGO and PABP consequently
  leading to recruitment of deadenylases like CCR4 and CAF1. Translation repression
  can result from inhibited binding of PABP to the poly (A) tail of the mRNA, responsible
  for attracting the elFs to mRNA to initiate translation. The formation of the CCR4-CAF1-NOT
  complex, a poly A tail-truncating enzyme, mediated by binding of miRISC to mRNA
  results in truncation of the downstream poly A tail, reduced binding of translation
  initiation factors and translation repression. The shortening or complete removal
  of the poly (A) tail induces the removal of the 5′ cap of the mRNA. Decapping is
  also mediated by DCP1 and DCP2. Consequently, the uncapped mRNA is rapidly degraded
  by 5′-3′ exoribonucleases.
pmcid: PMC6391339
papertitle: MicroRNAs, Hypoxia and the Stem-Like State as Contributors to Cancer Aggressiveness.
reftext: Lucy Wanjiku Macharia, et al. Front Genet. 2019;10:125.
pmc_ranked_result_index: '221818'
pathway_score: 0.849714
filename: fgene-10-00125-g001.jpg
figtitle: Biogenesis of miRNAs
year: '2019'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6391339__fgene-10-00125-g001.html
  '@type': Dataset
  description: Biogenesis of miRNAs. In the canonical pathway, miRNAs are transcribed
    from their loci by RNA polymerase II into a long primary transcript of about 80
    nucleotides called the pri-miRNA. Cleavage follows and is done by Drosha, a type
    III RNase along with the DGCR8 protein to produce pre-miRNA. In the non-canonical
    pathway, Mirtrons are spliced by the spliceosome to form looped intermediates
    referred to as lariat which then refold into pre-miRNAs. Next, the exportin 5,
    a RAN-GTP dependent transporter, mediates the movement of pre-miRNAs from the
    nucleus into the cytoplasm. Further processing by Dicer and TARBP2 protein generates
    mature miRNAs, producing double-stranded structure of miRNA of about 21-22 nt
    in length. The duplex is loaded into an AGO protein. The passenger strand (miRNA∗)
    is degraded, whereas the guide strand is incorporated by the Ago into the miRNA-induced
    silencing complex (miRISC). In animals, imperfect complementarity occurs when
    the miRNA seed region, nucleotides 2-8, BPs perfectly with the complementary seed
    match site in the 3′ UTR of the target mRNA resulting in translational repression
    or degradation. GW182 a core component of miRISC, mediates deadenylation of mRNAs
    by interacting with AGO and PABP consequently leading to recruitment of deadenylases
    like CCR4 and CAF1. Translation repression can result from inhibited binding of
    PABP to the poly (A) tail of the mRNA, responsible for attracting the elFs to
    mRNA to initiate translation. The formation of the CCR4-CAF1-NOT complex, a poly
    A tail-truncating enzyme, mediated by binding of miRISC to mRNA results in truncation
    of the downstream poly A tail, reduced binding of translation initiation factors
    and translation repression. The shortening or complete removal of the poly (A)
    tail induces the removal of the 5′ cap of the mRNA. Decapping is also mediated
    by DCP1 and DCP2. Consequently, the uncapped mRNA is rapidly degraded by 5′-3′
    exoribonucleases.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ACE
  - AGO1
  - DGCR8
  - TNRC6A
  - DCP2
  - DROSHA
  - CCR4
  - RAN
  - DICER1
  - O Ran
  - GTP
genes:
- word: DCP1
  symbol: DCP1
  source: hgnc_prev_symbol
  hgnc_symbol: ACE
  entrez: '1636'
- word: AGO1
  symbol: AGO1
  source: hgnc_symbol
  hgnc_symbol: AGO1
  entrez: '26523'
- word: DGCR8
  symbol: DGCR8
  source: hgnc_symbol
  hgnc_symbol: DGCR8
  entrez: '54487'
- word: GW182
  symbol: GW182
  source: hgnc_alias_symbol
  hgnc_symbol: TNRC6A
  entrez: '27327'
- word: DCP2
  symbol: DCP2
  source: hgnc_symbol
  hgnc_symbol: DCP2
  entrez: '167227'
- word: DROSHA
  symbol: DROSHA
  source: hgnc_symbol
  hgnc_symbol: DROSHA
  entrez: '29102'
- word: CCR4,
  symbol: CCR4
  source: hgnc_symbol
  hgnc_symbol: CCR4
  entrez: '1233'
- word: Ran
  symbol: RAN
  source: hgnc_symbol
  hgnc_symbol: RAN
  entrez: '5901'
- word: '|DICER'
  symbol: Dicer
  source: hgnc_alias_symbol
  hgnc_symbol: DICER1
  entrez: '23405'
chemicals:
- word: O Ran
  source: MESH
  identifier: C046045
- word: GTP
  source: MESH
  identifier: D006160
diseases: []
figid_alias: PMC6391339__F1
redirect_from: /figures/PMC6391339__F1
figtype: Figure
---
