---
figid: PMC6423516__nihms-1006895-f0003
figlink: /pmc/articles/PMC6423516/figure/F3/
number: F3
caption: Various cell-intrinsic and extrinsic inputs regulate the autophagy-lysosome
  machinery. Organelle damage and impaired protein quality control with dysfunction
  of the ubiquitin-proteasome system activates selective autophagy to remove the damaged
  organelles (mitochondria, ER, golgi and lysososomes) and protein aggregates (reviewed
  in ()). Aging induces ROS upregulation and accumulation of DNA damage, which induce
  senescence in cardiac progenitors through telomere attrition and DNA damage (reviewed
  in (). While SIRT1 expression is increased, experimental evidence indicates that
  further stimulation of SIRT1 function enhances autophagy via activation of FOXO1.
  Serum levels of GDF11, a TGFβ family member, decreases with age in the serum and
  increasing GDF11 levels beneficially affects age-related cardiac phenotypes. While
  GDF11 stimulates autophagy in skeletal muscle; its effects on the heart have not
  been studied. miR-216a increases in the aging heart and is known to downregulate
  Beclin-1, a protein essential for autophagy induction. Nicotinamide phosphoribosyl
  transferase (Nampt), a key enzyme in the salvage pathway of NAD+ synthesis in cardiomyocytes
  is down regulated with aging, and restoring Nampt and NAD+ levels restores autophagic
  flux in the heart. Multiple transcriptional pathways intricately regulate the autophagy-lysosome
  machinery (reviewed in ()). The FOXO family of transcription factors are autophagy
  activators and are activated by phosphorylation effected by AMPK, JNK and MST1 kinases.
  However, Akt-mediated phosphorylation of FOXO1 at a different site inactivates FOXO1
  and holds it in the cytosol. Activation of the stress sensor ERN1/IRE1α signaling
  also holds FOXO activity in check. TP53 (tumor protein p53), a transcription factor
  that regulates cell cycle activates autophagy when localized to the nucleus by directly
  activating transcriprion of various ATG genes, but inhibits autophagy when in the
  cytosol. Lysosomal lipases are held in check by mxl3 in C. elegans and by its orthologue
  MAX in mice. Both are transcriptional repressors that bind to the CLEAR response
  element where TFEB binds to inhibit lipophagy and lipolysis for generation of energy
  (reviewed in (). CREB, or cAMP response element-binding protein is held in check
  by its interaction with FXR, farnesoid X receptor in the fed state. Fasting induced
  FXR degradation and binding of CREB to CRTC2, its co-activator to induce TFEB transcription.
  Fasting also causes LYNUS machinery to deactivate mTOR which results in dephosphorylation
  and activation of TFEB. TFEB stimulates transcription of autophagy and lysosome
  pathway genes. Circadian rhythm plays a central role in regulation of the fasting
  feeding responses and controls autophagy-lysosome machinery. Emerging evidence points
  to counter-regulation of TFEB family of transcription factors by circadian rhythm
  regulators either indirectly via mTOR activity or directly via TFEB binding to Per
  genes (reviewed in (). Numerous metabolic influences also regulate autophagy in
  the heart, as shown (and reviewed in ()). Please see section titled ‘Cardiac energetics,
  substrate metabolism and pathophysiology under fasting states’ for a detailed discussion
  of cardiac metabolism under stress. ‘’ indicates , whereas ‘’ indicates inhibition.
pmcid: PMC6423516
papertitle: 'Lysosomes Mediate Benefits of Intermittent Fasting in Cardiometabolic
  Disease: The Janitor is the Undercover Boss..'
reftext: Kartik Mani, et al. Compr Physiol. 2018 Sep 14;8(4):1639-1667.
pmc_ranked_result_index: '120966'
pathway_score: 0.5349956
filename: nihms-1006895-f0003.jpg
figtitle: 'Lysosomes Mediate Benefits of Intermittent Fasting in Cardiometabolic Disease:
  The Janitor is the Undercover Boss.'
year: '2018'
organisms: Homo sapiens
ndex: 7d346db4-df35-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6423516__nihms-1006895-f0003.html
  '@type': Dataset
  description: Various cell-intrinsic and extrinsic inputs regulate the autophagy-lysosome
    machinery. Organelle damage and impaired protein quality control with dysfunction
    of the ubiquitin-proteasome system activates selective autophagy to remove the
    damaged organelles (mitochondria, ER, golgi and lysososomes) and protein aggregates
    (reviewed in ()). Aging induces ROS upregulation and accumulation of DNA damage,
    which induce senescence in cardiac progenitors through telomere attrition and
    DNA damage (reviewed in (). While SIRT1 expression is increased, experimental
    evidence indicates that further stimulation of SIRT1 function enhances autophagy
    via activation of FOXO1. Serum levels of GDF11, a TGFβ family member, decreases
    with age in the serum and increasing GDF11 levels beneficially affects age-related
    cardiac phenotypes. While GDF11 stimulates autophagy in skeletal muscle; its effects
    on the heart have not been studied. miR-216a increases in the aging heart and
    is known to downregulate Beclin-1, a protein essential for autophagy induction.
    Nicotinamide phosphoribosyl transferase (Nampt), a key enzyme in the salvage pathway
    of NAD+ synthesis in cardiomyocytes is down regulated with aging, and restoring
    Nampt and NAD+ levels restores autophagic flux in the heart. Multiple transcriptional
    pathways intricately regulate the autophagy-lysosome machinery (reviewed in ()).
    The FOXO family of transcription factors are autophagy activators and are activated
    by phosphorylation effected by AMPK, JNK and MST1 kinases. However, Akt-mediated
    phosphorylation of FOXO1 at a different site inactivates FOXO1 and holds it in
    the cytosol. Activation of the stress sensor ERN1/IRE1α signaling also holds FOXO
    activity in check. TP53 (tumor protein p53), a transcription factor that regulates
    cell cycle activates autophagy when localized to the nucleus by directly activating
    transcriprion of various ATG genes, but inhibits autophagy when in the cytosol.
    Lysosomal lipases are held in check by mxl3 in C. elegans and by its orthologue
    MAX in mice. Both are transcriptional repressors that bind to the CLEAR response
    element where TFEB binds to inhibit lipophagy and lipolysis for generation of
    energy (reviewed in (). CREB, or cAMP response element-binding protein is held
    in check by its interaction with FXR, farnesoid X receptor in the fed state. Fasting
    induced FXR degradation and binding of CREB to CRTC2, its co-activator to induce
    TFEB transcription. Fasting also causes LYNUS machinery to deactivate mTOR which
    results in dephosphorylation and activation of TFEB. TFEB stimulates transcription
    of autophagy and lysosome pathway genes. Circadian rhythm plays a central role
    in regulation of the fasting feeding responses and controls autophagy-lysosome
    machinery. Emerging evidence points to counter-regulation of TFEB family of transcription
    factors by circadian rhythm regulators either indirectly via mTOR activity or
    directly via TFEB binding to Per genes (reviewed in (). Numerous metabolic influences
    also regulate autophagy in the heart, as shown (and reviewed in ()). Please see
    section titled ‘Cardiac energetics, substrate metabolism and pathophysiology under
    fasting states’ for a detailed discussion of cardiac metabolism under stress.
    ‘’ indicates , whereas ‘’ indicates inhibition.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CLOCK
  - PRKAA1
  - CREB3L4
  - PRKN
  - BCL2
  - RICTOR
  - MAPKAP1
  - PINK1
  - PRKAG3
  - PGC
  - CRY1
  - FOXO1
  - NR1H4
  - CREB3
  - CREB1
  - ATF2
  - CREM
  - TFEB
  - TP53
  - ULK1
  - PRKAG2
  - CREB5
  - PER2
  - PPARGC1A
  - FXR1
  - MST1
  - MTOR
  - RPTOR
  - MLST8
  - PRKAG1
  - PRKAA2
  - CREB3L1
  - PPARA
  - SIRT1
  - SIRT4
  - SIRT3
  - PRKAB1
  - PRKAB2
  - CREB3L2
  - CREB3L3
  - ATF1
  - STK11
  - SIRT5
  - NADH
  - NAD
  - Calcium
  - Cholesterol
  - Saturated Fatty acids
  - Unsaturated Fatty acids
  - Ketone
  - Carbohydrates
  - Glucose
  - Hexosamine
  - N-AC-Glc
  - Trehalose
  - Amino Acids
  - Mitochondrial Stress
  - Lysosomal Stress
  - Glucose -6-P
  - Mitochondrial dysfunction
genes:
- word: CLOCK/BMAL
  symbol: CLOCK
  source: hgnc_symbol
  hgnc_symbol: CLOCK
  entrez: '9575'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAA1
  entrez: '5562'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L4
  entrez: '148327'
- word: PARKIN/PINK1
  symbol: parkin
  source: hgnc_alias_symbol
  hgnc_symbol: PRKN
  entrez: '5071'
- word: Bcl-2
  symbol: Bcl-2
  source: hgnc_alias_symbol
  hgnc_symbol: BCL2
  entrez: '596'
- word: •MTORC2!
  symbol: mTORC2
  source: bioentities_symbol
  hgnc_symbol: RICTOR
  entrez: '253260'
- word: •MTORC2!
  symbol: mTORC2
  source: bioentities_symbol
  hgnc_symbol: MAPKAP1
  entrez: '79109'
- word: PARKIN/PINK1
  symbol: PINK1
  source: hgnc_symbol
  hgnc_symbol: PINK1
  entrez: '65018'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAG3
  entrez: '53632'
- word: AMPK/PGC-
  symbol: PGC
  source: hgnc_symbol
  hgnc_symbol: PGC
  entrez: '5225'
- word: CRY1
  symbol: CRY1
  source: hgnc_symbol
  hgnc_symbol: CRY1
  entrez: '1407'
- word: FOXO-1
  symbol: FOXO1
  source: hgnc_symbol
  hgnc_symbol: FOXO1
  entrez: '2308'
- word: FXR
  symbol: FXR
  source: hgnc_alias_symbol
  hgnc_symbol: NR1H4
  entrez: '9971'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3
  entrez: '10488'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB1
  entrez: '1385'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: ATF2
  entrez: '1386'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREM
  entrez: '1390'
- word: TFEB
  symbol: TFEB
  source: hgnc_symbol
  hgnc_symbol: TFEB
  entrez: '7942'
- word: TP53
  symbol: TP53
  source: hgnc_symbol
  hgnc_symbol: TP53
  entrez: '7157'
- word: ULK1
  symbol: ULK1
  source: hgnc_symbol
  hgnc_symbol: ULK1
  entrez: '8408'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAG2
  entrez: '51422'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB5
  entrez: '9586'
- word: •CREB1
  symbol: CREB1
  source: hgnc_symbol
  hgnc_symbol: CREB1
  entrez: '1385'
- word: PER2
  symbol: PER2
  source: hgnc_symbol
  hgnc_symbol: PER2
  entrez: '8864'
- word: PGC1
  symbol: PGC1
  source: hgnc_alias_symbol
  hgnc_symbol: PPARGC1A
  entrez: '10891'
- word: PGC1-a
  symbol: PGC-1alpha
  source: hgnc_alias_symbol
  hgnc_symbol: PPARGC1A
  entrez: '10891'
- word: FXR1
  symbol: FXR1
  source: hgnc_symbol
  hgnc_symbol: FXR1
  entrez: '8087'
- word: Mst1
  symbol: MST1
  source: hgnc_symbol
  hgnc_symbol: MST1
  entrez: '4485'
- word: MTORC1
  symbol: mTORC1
  source: bioentities_symbol
  hgnc_symbol: MTOR
  entrez: '2475'
- word: MTORC1
  symbol: mTORC1
  source: bioentities_symbol
  hgnc_symbol: RPTOR
  entrez: '57521'
- word: •MTORC2!
  symbol: mTORC2
  source: bioentities_symbol
  hgnc_symbol: MLST8
  entrez: '64223'
- word: •MTORC2!
  symbol: mTORC2
  source: bioentities_symbol
  hgnc_symbol: MTOR
  entrez: '2475'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAG1
  entrez: '5571'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAA2
  entrez: '5563'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L1
  entrez: '90993'
- word: PPAR-a
  symbol: PPARA
  source: hgnc_symbol
  hgnc_symbol: PPARA
  entrez: '5465'
- word: SIRT1
  symbol: SIRT1
  source: hgnc_symbol
  hgnc_symbol: SIRT1
  entrez: '23411'
- word: SIRT3,4,5
  symbol: SIRT4
  source: hgnc_symbol
  hgnc_symbol: SIRT4
  entrez: '23409'
- word: SIRT3,4,5
  symbol: SIRT3
  source: hgnc_symbol
  hgnc_symbol: SIRT3
  entrez: '23410'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAB1
  entrez: '5564'
- word: AMPK
  symbol: AMPK
  source: bioentities_symbol
  hgnc_symbol: PRKAB2
  entrez: '5565'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L2
  entrez: '64764'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L3
  entrez: '84699'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: ATF1
  entrez: '466'
- word: STK11
  symbol: STK11
  source: hgnc_symbol
  hgnc_symbol: STK11
  entrez: '6794'
- word: SIRT3,4,5
  symbol: SIRT5
  source: hgnc_symbol
  hgnc_symbol: SIRT5
  entrez: '23408'
chemicals:
- word: NADH
  source: MESH
  identifier: D009243
- word: NAD
  source: MESH
  identifier: D009243
- word: Calcium
  source: MESH
  identifier: D002118
- word: Cholesterol
  source: MESH
  identifier: D002784
- word: Saturated Fatty acids
  source: MESH
  identifier: D005227
- word: Unsaturated Fatty acids
  source: MESH
  identifier: D005231
- word: Ketone
  source: MESH
  identifier: D007659
- word: Carbohydrates
  source: MESH
  identifier: D002241
- word: Glucose
  source: MESH
  identifier: D005947
- word: Hexosamine
  source: MESH
  identifier: D006595
- word: N-AC-Glc
  source: MESH
  identifier: C576651
- word: Trehalose
  source: MESH
  identifier: D014199
- word: Amino Acids
  source: MESH
  identifier: D000596
diseases:
- word: Mitochondrial Stress
  source: MESH
  identifier: D004194
- word: Lysosomal Stress
  source: MESH
  identifier: D004194
- word: Glucose -6-P
  source: MESH
  identifier: C537241
- word: Mitochondrial dysfunction
  source: MESH
  identifier: D028361
figid_alias: PMC6423516__F3
redirect_from: /figures/PMC6423516__F3
figtype: Figure
---
