---
figid: PMC6038301__13568_2018_635_Fig4_HTML
figlink: /pmc/articles/PMC6038301/figure/Fig4/
number: F4
caption: Three proposed ammonia oxidation pathways in AOA. Pathway 1 was proposed
  by Walker et al. In this pathway, ammonia is oxidized to hydroxylamine by the membrane
  enzyme complex AMO/CuMMO (Walker et al. ). Subsequently, hydroxylamine is oxidized
  to nitrite in the periplasm by a heme-rich hydroxylamine oxidoreductase (CuHAO)
  complex. Four electrons from this oxidation are transferred to the quinone pool.
  Two electrons from the reduced quinone pool return to AMO (marked by red) and are
  required to initiate ammonia oxidation. The remaining two electrons enter the electron
  transport chain composed of pcy protein to generate the proton motive force (PMF)
  necessary for ATP synthesis and NADH synthesis. In Pathway 2 (Schleper and Nicol
  ; Stahl and de la Torre ), NO is speculated functioning as a redox shuttle to deliver
  electrons to the AMO (marked by green) since measurable amounts of NO are produced
  during ammonia oxidation. In pathway 3, iterative production and consumption of
  NO is involved in conversion of hydroxylamine to nitrite facilitated by a proposed
  novel copper enzyme capable of performing known P460 activity (CuP460) (Kozlowski
  et al. ). N2O was formed abiotically from NO by interaction with media components
  or with debris in killed cell. AMO/CuMMO, ammonia monooxygenase; CuHAO, hydroxylamine
  dehydrogenase; NIR, Cu-containing NO-forming nitrite reductase; pcy, plastocyanin;
  Q/QH2, quinone/quinol pool
pmcid: PMC6038301
papertitle: Genome-scale metabolic model analysis indicates low energy production
  efficiency in marine ammonia-oxidizing archaea.
reftext: Feiran Li, et al. AMB Express. 2018;8:106.
pmc_ranked_result_index: '34849'
pathway_score: 0.8905784
filename: 13568_2018_635_Fig4_HTML.jpg
figtitle: Three proposed ammonia oxidation pathways in AOA
year: '2018'
organisms:
- Nitrosopumilus adriaticus
- Methanosarcina acetivorans
- Nitrosomonas europaea ATCC 19718
- Sulfolobus acidocaldarius
- metagenomes
- Nitrosopumilus maritimus SCM1
- Escherichia coli
- Nitrobacter winogradskyi
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6038301__13568_2018_635_Fig4_HTML.html
  '@type': Dataset
  description: Three proposed ammonia oxidation pathways in AOA. Pathway 1 was proposed
    by Walker et al. In this pathway, ammonia is oxidized to hydroxylamine by the
    membrane enzyme complex AMO/CuMMO (Walker et al. ). Subsequently, hydroxylamine
    is oxidized to nitrite in the periplasm by a heme-rich hydroxylamine oxidoreductase
    (CuHAO) complex. Four electrons from this oxidation are transferred to the quinone
    pool. Two electrons from the reduced quinone pool return to AMO (marked by red)
    and are required to initiate ammonia oxidation. The remaining two electrons enter
    the electron transport chain composed of pcy protein to generate the proton motive
    force (PMF) necessary for ATP synthesis and NADH synthesis. In Pathway 2 (Schleper
    and Nicol ; Stahl and de la Torre ), NO is speculated functioning as a redox shuttle
    to deliver electrons to the AMO (marked by green) since measurable amounts of
    NO are produced during ammonia oxidation. In pathway 3, iterative production and
    consumption of NO is involved in conversion of hydroxylamine to nitrite facilitated
    by a proposed novel copper enzyme capable of performing known P460 activity (CuP460)
    (Kozlowski et al. ). N2O was formed abiotically from NO by interaction with media
    components or with debris in killed cell. AMO/CuMMO, ammonia monooxygenase; CuHAO,
    hydroxylamine dehydrogenase; NIR, Cu-containing NO-forming nitrite reductase;
    pcy, plastocyanin; Q/QH2, quinone/quinol pool
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ASPSCR1
  - METTL2B
  - GOLGA6A
  - PRNP
  - MAL
  - PAEP
  - GLUL
  - Purine
  - tyr
  - ser
  - citramalatel
  - citraconate
  - Amino Acids
  - HCO3
genes:
- word: asp-L
  symbol: ASPL
  source: hgnc_alias_symbol
  hgnc_symbol: ASPSCR1
  entrez: '79058'
- word: met-L
  symbol: METL
  source: hgnc_alias_symbol
  hgnc_symbol: METTL2B
  entrez: '55798'
- word: glp
  symbol: GLP
  source: hgnc_alias_symbol
  hgnc_symbol: GOLGA6A
  entrez: '342096'
- word: prip
  symbol: PRIP
  source: hgnc_prev_symbol
  hgnc_symbol: PRNP
  entrez: '5621'
- word: mal
  symbol: MAL
  source: hgnc_symbol
  hgnc_symbol: MAL
  entrez: '4118'
- word: pep
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
- word: glu-L
  symbol: GLUL
  source: hgnc_symbol
  hgnc_symbol: GLUL
  entrez: '2752'
chemicals:
- word: Purine
  source: MESH
  identifier: D011687
- word: tyr
  source: MESH
  identifier: C042696
- word: ser
  source: MESH
  identifier: C530429
- word: citramalatel
  source: ''
  identifier: ''
- word: citraconate
  source: ''
  identifier: ''
- word: Amino Acids
  source: MESH
  identifier: D000596
- word: HCO3
  source: MESH
  identifier: C504136
diseases: []
figid_alias: PMC6038301__F4
redirect_from: /figures/PMC6038301__F4
figtype: Figure
---
