---
figid: PMC6523720__cells-08-00300-g001
figlink: /pmc/articles/PMC6523720/figure/cells-08-00300-f001/
number: F1
caption: 'Epigenetic mediators involved in the establishment and erasure of histone
  post-translational modifications and DNA methylation, as described in the text.
  Histone acetyltransferases (HATs) are responsible for the transfer of an acetyl
  group to different histone amino acid residues, including lysines 4 and 9 (K4 and
  K9). Histone deacetylases (HDACs) catalyze the removal of this group. Methyl groups
  can also be added to different amino acids in the tails of the histones by histone
  methyltransferases (HMTs) and removed by histone demethylases (HDMs). Histone acetylation
  always results in an active chromatin status, while the effects of histone methylation
  depends on the number of groups added and the amino acid residue involved. As exemplified
  in the figure, H3K27me3 and H3K9me3 are repressive marks, while H3K4me3 is an active
  mark. DNA methylation takes place more frequently in cytosines followed by guanines
  in the so-called CpG sites, reaction catalyzed by DNA-methyltransferases (DNMTs)
  using S-adenosylmethionine (SAM) as methyl donor. The erasure of DNA methylation
  can be driven by an active pathway, either by sequential oxidations or deamination.
  In the oxidation pathway (depicted in red), Ten-eleven translocation enzymes (TETs)
  catalyze the conversion of 5-methylcytosine (5mC) in 5-hydroxymethylcytosine (5hmC),
  which is further oxidized to form 5-formylcytosine (5fC) and 5-carboxylcytosine
  (5caC). Both 5fC and 5caC are recognized by the base excision repair (BER) machinery
  and removed from the DNA strand, resulting in the reincorporation of a non-methylated
  cytosine. In the deamination pathway (depicted in purple), 5mC and 5hmC can be deaminated
  by AID/APOBEC (Activity Induced Deaminase/Apolipoprotein B mRNA Editing Catalytic
  Polypeptide-like) family of enzymes, generating thymine and 5-hydroxymethyluracil
  (5hmU), respectively. The mispairing of these bases with guanine in the opposite
  strand activates the BER machinery and results in the reincorporation of a non-methylated
  cytosine. G9a: histone methyltransferase G9a; SET7/9: histone-lysine N-methyltransferase
  SETD7; JARID: Jumonji/ARID Domain-Containing Protein; JMJD: Jumonji Domain-Containing
  Protein; LSD1: Lysine-specific demethylase 1; SAH: S -adenosylhomocysteine.'
pmcid: PMC6523720
papertitle: 'Regulation Is in the Air: The Relationship between Hypoxia and Epigenetics
  in Cancer.'
reftext: Diego Camuzi, et al. Cells. 2019 Apr;8(4):300.
pmc_ranked_result_index: '218623'
pathway_score: 0.9290723
filename: cells-08-00300-g001.jpg
figtitle: Epigenetic mediators involved in the establishment and erasure of histone
  post-translational modifications and DNA methylation, as described in the text
year: '2019'
organisms:
- Mus musculus
- Rattus norvegicus
- Citrus macrophylla
- Human papillomavirus type 16
- Homo sapiens
ndex: 65bd725b-df0b-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6523720__cells-08-00300-g001.html
  '@type': Dataset
  description: 'Epigenetic mediators involved in the establishment and erasure of
    histone post-translational modifications and DNA methylation, as described in
    the text. Histone acetyltransferases (HATs) are responsible for the transfer of
    an acetyl group to different histone amino acid residues, including lysines 4
    and 9 (K4 and K9). Histone deacetylases (HDACs) catalyze the removal of this group.
    Methyl groups can also be added to different amino acids in the tails of the histones
    by histone methyltransferases (HMTs) and removed by histone demethylases (HDMs).
    Histone acetylation always results in an active chromatin status, while the effects
    of histone methylation depends on the number of groups added and the amino acid
    residue involved. As exemplified in the figure, H3K27me3 and H3K9me3 are repressive
    marks, while H3K4me3 is an active mark. DNA methylation takes place more frequently
    in cytosines followed by guanines in the so-called CpG sites, reaction catalyzed
    by DNA-methyltransferases (DNMTs) using S-adenosylmethionine (SAM) as methyl donor.
    The erasure of DNA methylation can be driven by an active pathway, either by sequential
    oxidations or deamination. In the oxidation pathway (depicted in red), Ten-eleven
    translocation enzymes (TETs) catalyze the conversion of 5-methylcytosine (5mC)
    in 5-hydroxymethylcytosine (5hmC), which is further oxidized to form 5-formylcytosine
    (5fC) and 5-carboxylcytosine (5caC). Both 5fC and 5caC are recognized by the base
    excision repair (BER) machinery and removed from the DNA strand, resulting in
    the reincorporation of a non-methylated cytosine. In the deamination pathway (depicted
    in purple), 5mC and 5hmC can be deaminated by AID/APOBEC (Activity Induced Deaminase/Apolipoprotein
    B mRNA Editing Catalytic Polypeptide-like) family of enzymes, generating thymine
    and 5-hydroxymethyluracil (5hmU), respectively. The mispairing of these bases
    with guanine in the opposite strand activates the BER machinery and results in
    the reincorporation of a non-methylated cytosine. G9a: histone methyltransferase
    G9a; SET7/9: histone-lysine N-methyltransferase SETD7; JARID: Jumonji/ARID Domain-Containing
    Protein; JMJD: Jumonji Domain-Containing Protein; LSD1: Lysine-specific demethylase
    1; SAH: S -adenosylhomocysteine.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - KDM5A
  - KDM5B
  - HDAC2
  - HDAC3
  - EHMT2
  - HDAC10
  - HDAC8
  - HDAC9
  - HDAC1
  - KDM4D
  - KDM6B
  - KDM3A
  - KDM4B
  - SETD7
  - HDAC7
  - KDM1A
  - ACSM3
  - COL4A5
  - HDAC5
  - HDAC6
  - HDAC4
  - CH3-SAM
  - SAH
  - AID
genes:
- word: JARID1A
  symbol: JARID1A
  source: hgnc_prev_symbol
  hgnc_symbol: KDM5A
  entrez: '5927'
- word: JARID1B
  symbol: JARID1B
  source: hgnc_prev_symbol
  hgnc_symbol: KDM5B
  entrez: '10765'
- word: HDAC1-10
  symbol: HDAC2
  source: hgnc_symbol
  hgnc_symbol: HDAC2
  entrez: '3066'
- word: HDAC1-10
  symbol: HDAC3
  source: hgnc_symbol
  hgnc_symbol: HDAC3
  entrez: '8841'
- word: G9a
  symbol: G9A
  source: hgnc_alias_symbol
  hgnc_symbol: EHMT2
  entrez: '10919'
- word: HDAC1-10
  symbol: HDAC10
  source: hgnc_symbol
  hgnc_symbol: HDAC10
  entrez: '83933'
- word: HDAC1-10
  symbol: HDAC8
  source: hgnc_symbol
  hgnc_symbol: HDAC8
  entrez: '55869'
- word: HDAC1-10
  symbol: HDAC9
  source: hgnc_symbol
  hgnc_symbol: HDAC9
  entrez: '9734'
- word: HDAC1-10
  symbol: HDAC1
  source: hgnc_symbol
  hgnc_symbol: HDAC1
  entrez: '3065'
- word: JMJD2D
  symbol: JMJD2D
  source: hgnc_prev_symbol
  hgnc_symbol: KDM4D
  entrez: '55693'
- word: JMJD3
  symbol: JMJD3
  source: hgnc_prev_symbol
  hgnc_symbol: KDM6B
  entrez: '23135'
- word: JMJD1A
  symbol: JMJD1A
  source: hgnc_prev_symbol
  hgnc_symbol: KDM3A
  entrez: '55818'
- word: JMJD2B
  symbol: JMJD2B
  source: hgnc_prev_symbol
  hgnc_symbol: KDM4B
  entrez: '23030'
- word: SET7/9
  symbol: SET7/9
  source: hgnc_alias_symbol
  hgnc_symbol: SETD7
  entrez: '80854'
- word: HDAC1-10
  symbol: HDAC7
  source: hgnc_symbol
  hgnc_symbol: HDAC7
  entrez: '51564'
- word: LSD1
  symbol: LSD1
  source: hgnc_alias_symbol
  hgnc_symbol: KDM1A
  entrez: '23028'
- word: SAH
  symbol: SAH
  source: hgnc_prev_symbol
  hgnc_symbol: ACSM3
  entrez: '6296'
- word: –ùATS
  symbol: ATS
  source: hgnc_prev_symbol
  hgnc_symbol: COL4A5
  entrez: '1287'
- word: HDAC1-10
  symbol: HDAC5
  source: hgnc_symbol
  hgnc_symbol: HDAC5
  entrez: '10014'
- word: HDAC1-10
  symbol: HDAC6
  source: hgnc_symbol
  hgnc_symbol: HDAC6
  entrez: '10013'
- word: HDAC1-10
  symbol: HDAC4
  source: hgnc_symbol
  hgnc_symbol: HDAC4
  entrez: '9759'
chemicals:
- word: CH3-SAM
  source: MESH
  identifier: C455845
diseases:
- word: SAH
  source: MESH
  identifier: D013345
- word: AID
  source: ''
  identifier: ''
figid_alias: PMC6523720__F1
redirect_from: /figures/PMC6523720__F1
figtype: Figure
---
