---
figid: PMC6021929__pgs-19-413-g1
figlink: /pmc/articles/PMC6021929/figure/F0001/
number: F1
caption: '(A) The permissive candidacy portion of the PIP. GWAS lead SNPs and known
  significant genes for the phenotype of interest are encoded by the user in an MVF
  and expanded by population-specific linkage and body SNP mapping, respectively,
  to generate a list of permissive candidate variants (PCVs). (B) The ERV workflow.
  All of the PCVs are evaluated as putative regulatory SNPs using tissue-specific
  omics data on phenotype-relevant tissues encoded by the user in an MTF. They are
  evaluated according to the regulatory function of the chromatin state segment in
  which they reside, the dependence of that regulatory function on the variant allele
  and the presence of identifiable target gene relationships. (C) The coding variant
  (CV) workflow. In the CV workflow, all the PCVs are analyzed to investigate whether
  they are nonsynonymous CVs with a meaningful minor allele frequency. (D) The hierarchical
  mapping portion of the PIP. Variants passing either workflow (intermediate candidate
  variants) are associated with target genes, and those expressed in relevant tissues
  (per the MTF) are subjected to pathway mapping and gene-set enrichment in IPA®.
  Gene sets associated with significant and related pathways are regarded as substantive,
  and the variants influencing them (regulatory and coding) are considered candidate
  variants.ERV: Expression regulatory variant; GWAS: Genome-wide association study;
  IPA®: Ingenuity®Pathway Analysis; MTF: Master tissue file; MVF: Master variant file;
  PIP: Pharmacoepigenomics informatics pipeline.'
pmcid: PMC6021929
papertitle: The pharmacoepigenomics informatics pipeline defines a pathway of novel
  and known warfarin pharmacogenomics variants.
reftext: Ari Allyn-Feuer, et al. Pharmacogenomics. 2018 Apr;19(5):413-434.
pmc_ranked_result_index: '49715'
pathway_score: 0.9020135
filename: pgs-19-413-g1.jpg
figtitle: The pharmacoepigenomics informatics pipeline defines a pathway of novel
  and known warfarin pharmacogenomics variants
year: '2018'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6021929__pgs-19-413-g1.html
  '@type': Dataset
  description: '(A) The permissive candidacy portion of the PIP. GWAS lead SNPs and
    known significant genes for the phenotype of interest are encoded by the user
    in an MVF and expanded by population-specific linkage and body SNP mapping, respectively,
    to generate a list of permissive candidate variants (PCVs). (B) The ERV workflow.
    All of the PCVs are evaluated as putative regulatory SNPs using tissue-specific
    omics data on phenotype-relevant tissues encoded by the user in an MTF. They are
    evaluated according to the regulatory function of the chromatin state segment
    in which they reside, the dependence of that regulatory function on the variant
    allele and the presence of identifiable target gene relationships. (C) The coding
    variant (CV) workflow. In the CV workflow, all the PCVs are analyzed to investigate
    whether they are nonsynonymous CVs with a meaningful minor allele frequency. (D)
    The hierarchical mapping portion of the PIP. Variants passing either workflow
    (intermediate candidate variants) are associated with target genes, and those
    expressed in relevant tissues (per the MTF) are subjected to pathway mapping and
    gene-set enrichment in IPA®. Gene sets associated with significant and related
    pathways are regarded as substantive, and the variants influencing them (regulatory
    and coding) are considered candidate variants.ERV: Expression regulatory variant;
    GWAS: Genome-wide association study; IPA®: Ingenuity®Pathway Analysis; MTF: Master
    tissue file; MVF: Master variant file; PIP: Pharmacoepigenomics informatics pipeline.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MARK1
  - SET
  - NR4A3
  - STUB1
genes:
- word: marks
  symbol: MARK
  source: hgnc_alias_symbol
  hgnc_symbol: MARK1
  entrez: '4139'
- word: set
  symbol: SET
  source: hgnc_symbol
  hgnc_symbol: SET
  entrez: '6418'
- word: Minor
  symbol: MINOR
  source: hgnc_alias_symbol
  hgnc_symbol: NR4A3
  entrez: '8013'
- word: -CHIP
  symbol: CHIP
  source: hgnc_alias_symbol
  hgnc_symbol: STUB1
  entrez: '10273'
chemicals: []
diseases: []
figid_alias: PMC6021929__F1
redirect_from: /figures/PMC6021929__F1
figtype: Figure
---
