---
figid: PMC6591400__41467_2019_10706_Fig6_HTML
figlink: /pmc/articles/PMC6591400/figure/Fig6/
number: F6
caption: pAB5 modulates chromosomally-encoded proteins. Quantitative proteome analysis
  of the effect of pAB5 on the secretome (a), the whole proteome with shaking (b),
  and whole proteome under static conditions (c). Manhattan plots demonstrating the
  significance of protein alteration, –log10(p-value), vs position in the genome are
  shown. The direction of the protein alteration is colored coded according the provided
  heat map. Selected proteins and accession numbers are shown. Full data sets are
  shown in Supplementary Fig. . Pink shadow highlights the genes encoding for CUP1
  pili, yellow shadow highlights the genes encoding for CUP2 pili, orange shadow highlights
  the genes of PNAG biosynthetic pathway and green shadow highlights the genes encoding
  for the T6SS. Four biological replicates of each condition were analyzed. The default
  Maxquant FDR setting of 1% FDR at the protein and peptide levels were used
pmcid: PMC6591400
papertitle: Urinary tract colonization is enhanced by a plasmid that regulates uropathogenic
  Acinetobacter baumannii chromosomal genes.
reftext: Gisela Di Venanzio, et al. Nat Commun. 2019;10:2763.
pmc_ranked_result_index: '86531'
pathway_score: 0.5072047
filename: 41467_2019_10706_Fig6_HTML.jpg
figtitle: pAB5 modulates chromosomally-encoded proteins
year: '2019'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6591400__41467_2019_10706_Fig6_HTML.html
  '@type': Dataset
  description: pAB5 modulates chromosomally-encoded proteins. Quantitative proteome
    analysis of the effect of pAB5 on the secretome (a), the whole proteome with shaking
    (b), and whole proteome under static conditions (c). Manhattan plots demonstrating
    the significance of protein alteration, –log10(p-value), vs position in the genome
    are shown. The direction of the protein alteration is colored coded according
    the provided heat map. Selected proteins and accession numbers are shown. Full
    data sets are shown in Supplementary Fig. . Pink shadow highlights the genes encoding
    for CUP1 pili, yellow shadow highlights the genes encoding for CUP2 pili, orange
    shadow highlights the genes of PNAG biosynthetic pathway and green shadow highlights
    the genes encoding for the T6SS. Four biological replicates of each condition
    were analyzed. The default Maxquant FDR setting of 1% FDR at the protein and peptide
    levels were used
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SHC3
  - SHC1
  - SHC2
  - SHC4
  - STC1
  - d0335 02155
genes:
- word: SHC
  symbol: SHC
  source: bioentities_symbol
  hgnc_symbol: SHC3
  entrez: '53358'
- word: SHC
  symbol: SHC
  source: bioentities_symbol
  hgnc_symbol: SHC1
  entrez: '6464'
- word: SHC
  symbol: SHC
  source: bioentities_symbol
  hgnc_symbol: SHC2
  entrez: '25759'
- word: SHC
  symbol: SHC
  source: bioentities_symbol
  hgnc_symbol: SHC4
  entrez: '399694'
- word: STC
  symbol: STC
  source: hgnc_prev_symbol
  hgnc_symbol: STC1
  entrez: '6781'
chemicals:
- word: d0335 02155
  source: MESH
  identifier: C116248
diseases: []
figid_alias: PMC6591400__F6
redirect_from: /figures/PMC6591400__F6
figtype: Figure
---
