---
figid: PMC6291580__atm-06-S1-S14-f1
figlink: /pmc/articles/PMC6291580/figure/f1/
number: F1
caption: Potential genetic determinants of cardiovascular risk in SLE. The picture
  summarises some of the main potential pathways under genetic control and potentially
  involved in cardiovascular risk in patients with SLE. Genes marked with (^) have
  been showed to be involved in general SLE susceptibility. For genes marked with
  (*), evidence has been provided for a potential specific involvement in cardiovascular
  risk in patients with SLE. Polymorphisms in genes involved in antigen recognition
  (such as mannose-binding lectin, MBL, or C-reactive protein, CRP) and presentation
  (human leukocyte antigen, HLA) constitute examples of this latter category and might
  be pathogenic by affecting macrophage (Mϕ) and dendritic cell (DC) function. Several
  mechanisms of potential vascular pathogenic significance have been linked to IFNα
  pathways. IFNα has previously been linked to endothelial dysfunction and enhanced
  release of pro-inflammatory microparticles (EMP). The STAT4 pathway downstream IFNα-receptor
  plays a crucial role in T-helper 1 cell function and promotion of Mϕ activation
  through IFNγ. Recent evidence suggests an association between polymorphisms in the
  SRP54-AS1 gene and ischaemic events in patients with SLE. Impaired regulatory response
  to IFNα due to defective binding of STAT1 to SRP54-AS1 sequence has been proposed
  as a potential mechanism accounting for the observed phenotype-genotype association.
  SRP54-AS1 might have a regulatory role in multiple cell lines including B and T
  cells and Mϕs. In addition, its defective activation in vessels affects the expression
  of the neighbour gene FAM177A1, which could contribute to vascular inflammation.
  Impaired binding of transcription factors might also be the key mechanism for the
  pathogenicity of some novel SNPs in the IL19 gene. IL19 sequence regulation is thought
  to affect IL10 expression and ultimately B cell activation, proliferation and production
  of autoantibodies including anti-phospholipid antibodies (aPL). Among potential
  transcription factors binding to IL19 sequence, IKZF3 is of particular interest
  due to its known association with SLE and its functional place downstream IFNα and
  IRF8, the latter being another recently described locus for potential cardiovascular
  susceptibility in patients with SLE. aPL, anti-phospholipid antibodies; B, B-lymphocyte;
  CRP, C-reactive protein; DC, dendritic cell; EMP, endothelium-derived microparticles;
  HLA, human leukocyte antigen; IFNα, interferon α; IFNαR, interferon α receptor;
  IFNγ, interferon γ; IFNγR, interferon γ receptor; IKZF3, Ikaros family zinc finger
  3; IL, interleukin; IRF8, interferon-regulatory factor 8; MBL, mannose-binding lectin;
  Mϕ, macrophage; SRP54-AS1, signal recognition particle 54 - antisense 1; STAT 1,4,
  signal transducer and activator of transcription 1,4; T, T-lymphocyte.
pmcid: PMC6291580
papertitle: 'Genetics in systemic lupus erythematosus: entering the borough of cardiovascular
  risk.'
reftext: Giuseppe A. Ramirez. Ann Transl Med. 2018 Nov;6(Suppl 1):S14.
pmc_ranked_result_index: '197472'
pathway_score: 0.95752
filename: atm-06-S1-S14-f1.jpg
figtitle: Genetic determinants of cardiovascular risk in SLE
year: '2018'
organisms:
- Homo sapiens
ndex: 719aa7b4-dee1-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6291580__atm-06-S1-S14-f1.html
  '@type': Dataset
  description: Potential genetic determinants of cardiovascular risk in SLE. The picture
    summarises some of the main potential pathways under genetic control and potentially
    involved in cardiovascular risk in patients with SLE. Genes marked with (^) have
    been showed to be involved in general SLE susceptibility. For genes marked with
    (*), evidence has been provided for a potential specific involvement in cardiovascular
    risk in patients with SLE. Polymorphisms in genes involved in antigen recognition
    (such as mannose-binding lectin, MBL, or C-reactive protein, CRP) and presentation
    (human leukocyte antigen, HLA) constitute examples of this latter category and
    might be pathogenic by affecting macrophage (Mϕ) and dendritic cell (DC) function.
    Several mechanisms of potential vascular pathogenic significance have been linked
    to IFNα pathways. IFNα has previously been linked to endothelial dysfunction and
    enhanced release of pro-inflammatory microparticles (EMP). The STAT4 pathway downstream
    IFNα-receptor plays a crucial role in T-helper 1 cell function and promotion of
    Mϕ activation through IFNγ. Recent evidence suggests an association between polymorphisms
    in the SRP54-AS1 gene and ischaemic events in patients with SLE. Impaired regulatory
    response to IFNα due to defective binding of STAT1 to SRP54-AS1 sequence has been
    proposed as a potential mechanism accounting for the observed phenotype-genotype
    association. SRP54-AS1 might have a regulatory role in multiple cell lines including
    B and T cells and Mϕs. In addition, its defective activation in vessels affects
    the expression of the neighbour gene FAM177A1, which could contribute to vascular
    inflammation. Impaired binding of transcription factors might also be the key
    mechanism for the pathogenicity of some novel SNPs in the IL19 gene. IL19 sequence
    regulation is thought to affect IL10 expression and ultimately B cell activation,
    proliferation and production of autoantibodies including anti-phospholipid antibodies
    (aPL). Among potential transcription factors binding to IL19 sequence, IKZF3 is
    of particular interest due to its known association with SLE and its functional
    place downstream IFNα and IRF8, the latter being another recently described locus
    for potential cardiovascular susceptibility in patients with SLE. aPL, anti-phospholipid
    antibodies; B, B-lymphocyte; CRP, C-reactive protein; DC, dendritic cell; EMP,
    endothelium-derived microparticles; HLA, human leukocyte antigen; IFNα, interferon
    α; IFNαR, interferon α receptor; IFNγ, interferon γ; IFNγR, interferon γ receptor;
    IKZF3, Ikaros family zinc finger 3; IL, interleukin; IRF8, interferon-regulatory
    factor 8; MBL, mannose-binding lectin; Mϕ, macrophage; SRP54-AS1, signal recognition
    particle 54 - antisense 1; STAT 1,4, signal transducer and activator of transcription
    1,4; T, T-lymphocyte.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MAEA
  - IFNGR1
  - IFNG
  - IL19
  - FAM177A1
  - IFNAR1
  - HLA-A
  - STAT4
  - IL10
  - IRF8
  - Mo
  - Atherosclerosis
  - Vascular inflammation
genes:
- word: EMP
  symbol: EMP
  source: hgnc_alias_symbol
  hgnc_symbol: MAEA
  entrez: '10296'
- word: IFNYR
  symbol: IFNGR
  source: hgnc_prev_symbol
  hgnc_symbol: IFNGR1
  entrez: '3459'
- word: IFNY
  symbol: IFNG
  source: hgnc_symbol
  hgnc_symbol: IFNG
  entrez: '3458'
- word: IL19--
  symbol: IL19
  source: hgnc_symbol
  hgnc_symbol: IL19
  entrez: '29949'
- word: FAM177A1
  symbol: FAM177A1
  source: hgnc_symbol
  hgnc_symbol: FAM177A1
  entrez: '283635'
- word: IFNAR
  symbol: IFNAR
  source: hgnc_prev_symbol
  hgnc_symbol: IFNAR1
  entrez: '3454'
- word: HLAA
  symbol: HLA-A
  source: hgnc_symbol
  hgnc_symbol: HLA-A
  entrez: '3105'
- word: STAT4^
  symbol: STAT4
  source: hgnc_symbol
  hgnc_symbol: STAT4
  entrez: '6775'
- word: IL10A
  symbol: IL10A
  source: hgnc_alias_symbol
  hgnc_symbol: IL10
  entrez: '3586'
- word: IRF8*
  symbol: IRF8
  source: hgnc_symbol
  hgnc_symbol: IRF8
  entrez: '3394'
chemicals:
- word: Mo
  source: MESH
  identifier: D008982
diseases:
- word: Atherosclerosis
  source: MESH
  identifier: D050197
- word: Vascular inflammation
  source: MESH
  identifier: D007249
figid_alias: PMC6291580__F1
redirect_from: /figures/PMC6291580__F1
figtype: Figure
---
