---
figid: PMC6314074__cshperspectmed-HAE-031724_F3
figlink: pmc/articles/PMC6314074/figure/A031724F3/
number: F3
caption: Life cycle of hepatitis E virus (HEV). (1) Nonenveloped HEV virions bind
  to cellular receptors. Heparin sulfate proteoglycans (HSPGs) and heat shock cognate
  protein 70 (HSC70) are thought to be the attachment receptors for HEV but more research
  is required to determine the entry receptors for HEV. Once bound to the cell, virions
  are taken into the cell through a dynamin-2, clathrin, and cholesterol-dependent
  process. (2) Quasi-enveloped HEV virions are capable of entering the cell via a
  clathrin-mediated pathway. This entry pathway appears to be dependent on Rab 5 and
  Rab7 GTPases and, unlike the unenveloped HEV virions, also requires lysosome function
  before uncoating within the host cell. (3) Heat shock proteins including HSP90 and
  Grp78 have been implicated in transporting virions to sites of genome release. (4)
  Once released into the cytoplasm, the positive-strand genomic RNA serves as messenger
  RNA (mRNA) for the translation of the open reading frame (ORF)1 nonstructural polyprotein.
  (5) Whether or not processing occurs for the ORF1 polyprotein remains debatable,
  but the viral nonstructural proteins within ORF1 transcribe the positive-sense RNA
  genome into a negative-strand intermediate. This step occurs at either the rough
  endoplasmic reticulum (RER) or RER-derived membrane vesicles and involves host proteins,
  including elongation initiation factors, and viral proteins, including ORF4 for
  genotype (gt)1 HEV replication. The subgenomic bicistronic mRNA is generated from
  the negative-stranded RNA and serves as messenger for translation of ORF2 and ORF3
  proteins. (6) Host ribosomes translate the newly synthesized RNAs into more nonstructural
  proteins and the ORF2 and ORF3 proteins. (7) There are a number of unknown steps
  to virus assembly. ORF2 binds to viral genomic RNA and begins to multimerize. ORF3
  also binds to ORF2 during transition to the plasma membrane. (8) Particles become
  membrane-associated likely by budding into intracellular vesicles via interaction
  of the ORF3 protein with host vacuolar sorting proteins such as tumor suppressor
  gene 101. These enveloped particles retain the trans-Golgi network protein 2 protein
  and CD63 on their lipid surface until particle release. (9) The plasma membrane
  serves as the final site for release of enveloped virions via the exosomal release
  pathway and is sensitive to disruption of Rab27a and Hrs. (10) At some point during
  transit through the host digestive system, the viral envelope is lost. HEV released
  into the bloodstream appears to retain its quasi-envelope, which is thought to mask
  the capsid shell from the host immune system.
pmcid: PMC6314074
papertitle: Hepatitis E Virus Genome Structure and Replication Strategy.
reftext: Scott P. Kenney, et al. Cold Spring Harb Perspect Med. 2019 Jan;9(1):a031724.
pmc_ranked_result_index: '123011'
pathway_score: 0.7580741
filename: cshperspectmed-HAE-031724_F3.jpg
figtitle: Life cycle of hepatitis E virus (HEV)
year: '2019'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6314074__cshperspectmed-HAE-031724_F3.html
  '@type': Dataset
  description: Life cycle of hepatitis E virus (HEV). (1) Nonenveloped HEV virions
    bind to cellular receptors. Heparin sulfate proteoglycans (HSPGs) and heat shock
    cognate protein 70 (HSC70) are thought to be the attachment receptors for HEV
    but more research is required to determine the entry receptors for HEV. Once bound
    to the cell, virions are taken into the cell through a dynamin-2, clathrin, and
    cholesterol-dependent process. (2) Quasi-enveloped HEV virions are capable of
    entering the cell via a clathrin-mediated pathway. This entry pathway appears
    to be dependent on Rab 5 and Rab7 GTPases and, unlike the unenveloped HEV virions,
    also requires lysosome function before uncoating within the host cell. (3) Heat
    shock proteins including HSP90 and Grp78 have been implicated in transporting
    virions to sites of genome release. (4) Once released into the cytoplasm, the
    positive-strand genomic RNA serves as messenger RNA (mRNA) for the translation
    of the open reading frame (ORF)1 nonstructural polyprotein. (5) Whether or not
    processing occurs for the ORF1 polyprotein remains debatable, but the viral nonstructural
    proteins within ORF1 transcribe the positive-sense RNA genome into a negative-strand
    intermediate. This step occurs at either the rough endoplasmic reticulum (RER)
    or RER-derived membrane vesicles and involves host proteins, including elongation
    initiation factors, and viral proteins, including ORF4 for genotype (gt)1 HEV
    replication. The subgenomic bicistronic mRNA is generated from the negative-stranded
    RNA and serves as messenger for translation of ORF2 and ORF3 proteins. (6) Host
    ribosomes translate the newly synthesized RNAs into more nonstructural proteins
    and the ORF2 and ORF3 proteins. (7) There are a number of unknown steps to virus
    assembly. ORF2 binds to viral genomic RNA and begins to multimerize. ORF3 also
    binds to ORF2 during transition to the plasma membrane. (8) Particles become membrane-associated
    likely by budding into intracellular vesicles via interaction of the ORF3 protein
    with host vacuolar sorting proteins such as tumor suppressor gene 101. These enveloped
    particles retain the trans-Golgi network protein 2 protein and CD63 on their lipid
    surface until particle release. (9) The plasma membrane serves as the final site
    for release of enveloped virions via the exosomal release pathway and is sensitive
    to disruption of Rab27a and Hrs. (10) At some point during transit through the
    host digestive system, the viral envelope is lost. HEV released into the bloodstream
    appears to retain its quasi-envelope, which is thought to mask the capsid shell
    from the host immune system.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - RAB27A
  - CLTB
  - CLTA
  - CLTC
  - RAB5A
  - RAB7A
  - SDC2
  - HSPA8
  - HSP90B1
  - TRAP1
  - HSP90AB1
  - HSPA5
genes:
- word: Rab27a,
  symbol: RAB27A
  source: hgnc_symbol
  hgnc_symbol: RAB27A
  entrez: '5873'
- word: Rab27a,Hrs
  symbol: RAB27A
  source: hgnc_symbol
  hgnc_symbol: RAB27A
  entrez: '5873'
- word: Clathrin
  symbol: Clathrin
  source: bioentities_symbol
  hgnc_symbol: CLTB
  entrez: '1212'
- word: Clathrin
  symbol: Clathrin
  source: bioentities_symbol
  hgnc_symbol: CLTA
  entrez: '1211'
- word: Clathrin
  symbol: Clathrin
  source: bioentities_symbol
  hgnc_symbol: CLTC
  entrez: '1213'
- word: Rab5,
  symbol: RAB5
  source: hgnc_prev_symbol
  hgnc_symbol: RAB5A
  entrez: '5868'
- word: Rab5,Rab7
  symbol: RAB7
  source: hgnc_prev_symbol
  hgnc_symbol: RAB7A
  entrez: '7879'
- word: Rab5,Rab7
  symbol: RAB5
  source: hgnc_prev_symbol
  hgnc_symbol: RAB5A
  entrez: '5868'
- word: Rab7
  symbol: RAB7
  source: hgnc_prev_symbol
  hgnc_symbol: RAB7A
  entrez: '7879'
- word: HSPGS,HSC70,?
  symbol: HSPG
  source: hgnc_prev_symbol
  hgnc_symbol: SDC2
  entrez: '6383'
- word: HSC70,
  symbol: HSC70
  source: hgnc_alias_symbol
  hgnc_symbol: HSPA8
  entrez: '3312'
- word: HSP90/Grp78
  symbol: HSP90
  source: bioentities_symbol
  hgnc_symbol: HSP90B1
  entrez: '7184'
- word: HSP90/Grp78
  symbol: HSP90
  source: bioentities_symbol
  hgnc_symbol: TRAP1
  entrez: '10131'
- word: HSP90/Grp78
  symbol: HSP90
  source: bioentities_symbol
  hgnc_symbol: HSP90AB1
  entrez: '3326'
- word: HSP90/Grp78
  symbol: GRP78
  source: hgnc_prev_symbol
  hgnc_symbol: HSPA5
  entrez: '3309'
chemicals: []
diseases: []
figid_alias: PMC6314074__F3
redirect_from: /figures/PMC6314074__F3
figtype: Figure
---
