---
figid: PMC6785598__18_2019_3199_Fig4_HTML
figlink: /pmc/articles/PMC6785598/figure/Fig4/
number: Fig. 4
caption: Epigenetic regulation of stem cells. a Active regulatory elements are typically
  enriched for 5hmC, H3K27ac, H3K4me, and bound Mediator complex. During gene repression,
  the activating histone modifications are eliminated, and repressive marks, such
  as H3K27me3 and nucleosomal compaction, are established. In pluripotent stem cells,
  multiple enhancers combined with master pluripotency transcription factors, such
  as OCT4, SOX2 and KLF4, establish a super-enhancer, which supports the activation
  of pluripotency genes. The absence of master pluripotency factors could induce the
  disassembly of the enhancer–promoter complex in the assembly of repressive inputs.
  The nucleosome remodeling and deacetylase (NuRD) complex induces nucleosome formation
  at the binding region of pluripotency factors and alters the histone modifications
  that correspond to transcriptional activity. Similar disassembly of activating modifications
  occurs in the promoter by PRC2-associated histone demethylases KDM2B and KDM5A.
  Furthermore, H3K9me2 or H3K9me3 is deposited in the enhancer by G9A in complex with
  GLP and SETDB1, and DNA methylation by DNMT3A. Equally, PRC2 deposits H3K27me3 in
  promoters, which initiate chromatin compaction by recruiting the canonical PRC1
  complex and monoubiquitylate H2AK119 (H2AK119ub). K27ac, Lys27 acetylation; Pol
  II, RNA polymerase II. PRC2, Polycomb repressive complex 2. GLP, G9A-like protein.
  DNMT3A, DNA methyltransferase 3A. CCTC-binding factor (CTCF). DNMT, DNA methyltransferase;
  HDAC, histone deacetylase; HAT, histone acetyltransferase; MED, mediator complex;
  MBD, methyl-DNA-binding domain protein; POL II, RNA polymerase II; TET, ten eleven
  translocation dioxygenase; TrxG, trithorax group complex. TF, transcription factor.
  b In differentiated cells, H3K27me3 and H2AK119ub are enriched in CpG island-containing
  promoters of some pluripotency genes. At the initiation of reprogramming, OCT4,
  SOX2 and KLF4 (known as OSK) bind to partial motif sequences of some select enhancers
  to engage pioneer factor-like activity. These OSK binding sites are modified with
  H3K4me1 and H3K4me2. OSK binding also promotes H3K4 methylation at the promoter
  via the MLL component WDR5 and is associated with local erasure of H3K27me3 by the
  histone demethylase UTX. Subsequently, OSK cooperates with unknown factors to establish
  a canonical enhancer architecture, including H3K4me2 and H3K27ac, and stable topological
  cognation to the promoter via the cohesin and Mediator complexes. Embryonic stem
  cell-specific BAF (esBAF) further stabilizes the enhancer-promoter complex, and
  the activating inputs predominate at this stage and direct the expression of the
  target genes. LSD1, Lys-specific demethylase 1; MBD3, methyl CpG-binding domain
  protein 3; MLL, mixed lineage leukemia. Pol II, RNA polymerase II, WDR5, WD repeat-containing
  protein 5. c The mechanisms of DNA methylation and demethylation. Cytosine is converted
  to 5mC by DNMTs. TET1 is responsible for the conversion of 5mC to 5hmC. Three TET
  family proteins could subsequently oxidize 5hmC to 5fC (5-formylcytosine) and then
  to 5caC (5-carboxylcytosine). Moreover, 5hmC can be converted to 5hmU (5-hydroxymethyluracil)
  by deaminase activation-induced cytidine deaminase (AID) and apolipoprotein B mRNA-editing
  enzyme catalytic polypeptides (APOBECs). 5fC, 5caC and 5hmU can be excised by thymine
  DNA glycosylase and replaced by an unmodified cytosine via the base-excision repair
  (BER) pathway. d Key epigenetic regulators that are involved in HSC self-renewal
  and lineage commitment during differentiation, and DNA methylation levels changes
  during hematopoietic lineage commitment. CLP, common lymphoid progenitor; CMP, common
  myeloid progenitor; DNMT, DNA methyltransferase; DNA me, DNA methylation; GLP, G9A-like
  protein. GMP, granulocyte–monocyte progenitor; MPP, multipotent progenitor; PRC,
  polycomb repressive complex; TET, Ten-eleven translocation; TrxG, Trithorax group.
  e In the embryonic neurogenesis of mice, neuroepithelial cells develop into radial
  glial cells (RGCs) around embryonic day 14. RGCs can either produce neurons directly
  or give rise to intermediate progenitor cells (IPCs), which in turn generate neurons.
  During later embryonic development, RGCs also give rise to oligodendrocytes (OligoD)
  and astrocytes
pmcid: PMC6785598
papertitle: Stem cells in tissues, organoids, and cancers.
reftext: Xusheng Wang. Cell Mol Life Sci. 2019;76(20):4043-4070.
pmc_ranked_result_index: '234988'
pathway_score: 0.7775275
filename: 18_2019_3199_Fig4_HTML.jpg
figtitle: Epigenetic regulation of stem cells
year: '2019'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6785598__18_2019_3199_Fig4_HTML.html
  '@type': Dataset
  description: Epigenetic regulation of stem cells. a Active regulatory elements are
    typically enriched for 5hmC, H3K27ac, H3K4me, and bound Mediator complex. During
    gene repression, the activating histone modifications are eliminated, and repressive
    marks, such as H3K27me3 and nucleosomal compaction, are established. In pluripotent
    stem cells, multiple enhancers combined with master pluripotency transcription
    factors, such as OCT4, SOX2 and KLF4, establish a super-enhancer, which supports
    the activation of pluripotency genes. The absence of master pluripotency factors
    could induce the disassembly of the enhancer–promoter complex in the assembly
    of repressive inputs. The nucleosome remodeling and deacetylase (NuRD) complex
    induces nucleosome formation at the binding region of pluripotency factors and
    alters the histone modifications that correspond to transcriptional activity.
    Similar disassembly of activating modifications occurs in the promoter by PRC2-associated
    histone demethylases KDM2B and KDM5A. Furthermore, H3K9me2 or H3K9me3 is deposited
    in the enhancer by G9A in complex with GLP and SETDB1, and DNA methylation by
    DNMT3A. Equally, PRC2 deposits H3K27me3 in promoters, which initiate chromatin
    compaction by recruiting the canonical PRC1 complex and monoubiquitylate H2AK119
    (H2AK119ub). K27ac, Lys27 acetylation; Pol II, RNA polymerase II. PRC2, Polycomb
    repressive complex 2. GLP, G9A-like protein. DNMT3A, DNA methyltransferase 3A.
    CCTC-binding factor (CTCF). DNMT, DNA methyltransferase; HDAC, histone deacetylase;
    HAT, histone acetyltransferase; MED, mediator complex; MBD, methyl-DNA-binding
    domain protein; POL II, RNA polymerase II; TET, ten eleven translocation dioxygenase;
    TrxG, trithorax group complex. TF, transcription factor. b In differentiated cells,
    H3K27me3 and H2AK119ub are enriched in CpG island-containing promoters of some
    pluripotency genes. At the initiation of reprogramming, OCT4, SOX2 and KLF4 (known
    as OSK) bind to partial motif sequences of some select enhancers to engage pioneer
    factor-like activity. These OSK binding sites are modified with H3K4me1 and H3K4me2.
    OSK binding also promotes H3K4 methylation at the promoter via the MLL component
    WDR5 and is associated with local erasure of H3K27me3 by the histone demethylase
    UTX. Subsequently, OSK cooperates with unknown factors to establish a canonical
    enhancer architecture, including H3K4me2 and H3K27ac, and stable topological cognation
    to the promoter via the cohesin and Mediator complexes. Embryonic stem cell-specific
    BAF (esBAF) further stabilizes the enhancer-promoter complex, and the activating
    inputs predominate at this stage and direct the expression of the target genes.
    LSD1, Lys-specific demethylase 1; MBD3, methyl CpG-binding domain protein 3; MLL,
    mixed lineage leukemia. Pol II, RNA polymerase II, WDR5, WD repeat-containing
    protein 5. c The mechanisms of DNA methylation and demethylation. Cytosine is
    converted to 5mC by DNMTs. TET1 is responsible for the conversion of 5mC to 5hmC.
    Three TET family proteins could subsequently oxidize 5hmC to 5fC (5-formylcytosine)
    and then to 5caC (5-carboxylcytosine). Moreover, 5hmC can be converted to 5hmU
    (5-hydroxymethyluracil) by deaminase activation-induced cytidine deaminase (AID)
    and apolipoprotein B mRNA-editing enzyme catalytic polypeptides (APOBECs). 5fC,
    5caC and 5hmU can be excised by thymine DNA glycosylase and replaced by an unmodified
    cytosine via the base-excision repair (BER) pathway. d Key epigenetic regulators
    that are involved in HSC self-renewal and lineage commitment during differentiation,
    and DNA methylation levels changes during hematopoietic lineage commitment. CLP,
    common lymphoid progenitor; CMP, common myeloid progenitor; DNMT, DNA methyltransferase;
    DNA me, DNA methylation; GLP, G9A-like protein. GMP, granulocyte–monocyte progenitor;
    MPP, multipotent progenitor; PRC, polycomb repressive complex; TET, Ten-eleven
    translocation; TrxG, Trithorax group. e In the embryonic neurogenesis of mice,
    neuroepithelial cells develop into radial glial cells (RGCs) around embryonic
    day 14. RGCs can either produce neurons directly or give rise to intermediate
    progenitor cells (IPCs), which in turn generate neurons. During later embryonic
    development, RGCs also give rise to oligodendrocytes (OligoD) and astrocytes
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MMP16
  - MMP24
  - MMP17
  - MMP19
  - MMP3
  - MMP7
  - MMP8
  - MMP9
  - MMP27
  - MMP25
  - MMP28
  - MMP23B
  - MMP20
  - BMI1
  - MMP21
  - MMP1
  - MMP2
  - RAD21
  - SMC3
  - DNMT1
  - DNMT3A
  - EZH1
  - EZH2
  - GOLGA6A
  - KLF4
  - MED27
  - KMT2A
  - MMP11
  - MMP12
  - MMP13
  - MMP14
  - MMP15
  - KDM6A
  - ME2
  - EP300
  - POLI
  - PRC1
  - MMP10
  - TRG
  - MATN1
  - TDG
  - TET2
genes:
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP16
  entrez: '4325'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP24
  entrez: '10893'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP17
  entrez: '4326'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP19
  entrez: '4327'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP3
  entrez: '4314'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP7
  entrez: '4316'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP8
  entrez: '4317'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP9
  entrez: '4318'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP27
  entrez: '64066'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP25
  entrez: '64386'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP28
  entrez: '79148'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP23B
  entrez: '8510'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP20
  entrez: '9313'
- word: BMI1
  symbol: BMI1
  source: hgnc_symbol
  hgnc_symbol: BMI1
  entrez: '648'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP21
  entrez: '118856'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP1
  entrez: '4312'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP2
  entrez: '4313'
- word: Cohesin-
  symbol: Cohesin
  source: bioentities_symbol
  hgnc_symbol: RAD21
  entrez: '5885'
- word: Cohesin-
  symbol: Cohesin
  source: bioentities_symbol
  hgnc_symbol: SMC3
  entrez: '9126'
- word: DNMT1
  symbol: DNMT1
  source: hgnc_symbol
  hgnc_symbol: DNMT1
  entrez: '1786'
- word: DNMT3A
  symbol: DNMT3A
  source: hgnc_symbol
  hgnc_symbol: DNMT3A
  entrez: '1788'
- word: EZH1/2
  symbol: EZH1
  source: hgnc_symbol
  hgnc_symbol: EZH1
  entrez: '2145'
- word: EZH1/2
  symbol: EZH2
  source: hgnc_symbol
  hgnc_symbol: EZH2
  entrez: '2146'
- word: GLP
  symbol: GLP
  source: hgnc_alias_symbol
  hgnc_symbol: GOLGA6A
  entrez: '342096'
- word: KLF4
  symbol: KLF4
  source: hgnc_symbol
  hgnc_symbol: KLF4
  entrez: '9314'
- word: MED3
  symbol: MED3
  source: hgnc_alias_symbol
  hgnc_symbol: MED27
  entrez: '9442'
- word: MLL
  symbol: MLL
  source: hgnc_prev_symbol
  hgnc_symbol: KMT2A
  entrez: '4297'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP11
  entrez: '4320'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP12
  entrez: '4321'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP13
  entrez: '4322'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP14
  entrez: '4323'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP15
  entrez: '4324'
- word: UTX
  symbol: UTX
  source: hgnc_prev_symbol
  hgnc_symbol: KDM6A
  entrez: '7403'
- word: нзКАme2
  symbol: ME2
  source: hgnc_symbol
  hgnc_symbol: ME2
  entrez: '4200'
- word: P300
  symbol: p300
  source: hgnc_alias_symbol
  hgnc_symbol: EP300
  entrez: '2033'
- word: POLI
  symbol: POLI
  source: hgnc_symbol
  hgnc_symbol: POLI
  entrez: '11201'
- word: PRC1
  symbol: PRC1
  source: hgnc_symbol
  hgnc_symbol: PRC1
  entrez: '9055'
- word: MMP
  symbol: MMP
  source: bioentities_symbol
  hgnc_symbol: MMP10
  entrez: '4319'
- word: Cohesin
  symbol: Cohesin
  source: bioentities_symbol
  hgnc_symbol: SMC3
  entrez: '9126'
- word: Cohesin
  symbol: Cohesin
  source: bioentities_symbol
  hgnc_symbol: RAD21
  entrez: '5885'
- word: TrG
  symbol: TRG
  source: hgnc_symbol
  hgnc_symbol: TRG
  entrez: '6965'
- word: CMP
  symbol: CMP
  source: hgnc_prev_symbol
  hgnc_symbol: MATN1
  entrez: '4146'
- word: TDG
  symbol: TDG
  source: hgnc_symbol
  hgnc_symbol: TDG
  entrez: '6996'
- word: TET2
  symbol: TET2
  source: hgnc_symbol
  hgnc_symbol: TET2
  entrez: '54790'
chemicals: []
diseases: []
---
