---
figid: PMC6784888__41564_2018_261_Fig4_HTML
figlink: /pmc/articles/PMC6784888/figure/Fig4/
number: Fig. 4
caption: 'a, Metabolic pathway map with enzymes found in ~70% of ‘free-living’ fungi
  and absent in at least one of the single-cell fungi shown as a gradient: darker
  red shades indicate more single-cell species missing the same enzyme in a given
  pathway. b–f, Certain pathways with a high degree of such common losses: assimilatory
  sulfate reduction (sulfate to sulfide) (b), thiamine biosynthesis (HMP and TZE to
  thiamine phosphate) (c), spermidine synthesis (S-adenosylmethionine to spermidine)
  (d), biotin metabolism (dethiobiotin to biotin (and biotinyl-5′-AMP)) (e), and citrate
  synthesis (citrate to/from oxaloacetate and acetyl-CoA) (f). APS, adenylyl sulfate;
  HMP, 4-amino-5-hydroxymethyl-2-methylpyrimidine; HMP-P, HMP phosphate; HMP-PP, HMP
  diphosphate; PAPS, phosphoadenylyl sulfate; TZE, 5-(2-hydroxyethyl)-4-methylthiazole;
  TZP, TZE phosphate. The coloured boxes for b–f indicate the presence (solid) or
  the absence (shaded) of a given enzyme in a genome: Ra, R. allomycis (red); Bh,
  B. helicus (blue); Cp, C. protostelioides (green); Dc, D. cristalligena (purple);
  Pc, P. cylindrospora (orange); Ts, T. sphaerospora (yellow); Sp, S. pseudoplumigaleata
  (brown); Mb, M. bicuspidata (pink).'
pmcid: PMC6784888
papertitle: Leveraging single-cell genomics to expand the fungal tree of life.
reftext: Steven R. Ahrendt, et al. Nat Microbiol. 2018;3(12):1417-1428.
pmc_ranked_result_index: '38824'
pathway_score: 0.5408732
filename: 41564_2018_261_Fig4_HTML.jpg
figtitle: Leveraging single-cell genomics to expand the fungal tree of life
year: '2018'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6784888__41564_2018_261_Fig4_HTML.html
  '@type': Dataset
  description: 'a, Metabolic pathway map with enzymes found in ~70% of ‘free-living’
    fungi and absent in at least one of the single-cell fungi shown as a gradient:
    darker red shades indicate more single-cell species missing the same enzyme in
    a given pathway. b–f, Certain pathways with a high degree of such common losses:
    assimilatory sulfate reduction (sulfate to sulfide) (b), thiamine biosynthesis
    (HMP and TZE to thiamine phosphate) (c), spermidine synthesis (S-adenosylmethionine
    to spermidine) (d), biotin metabolism (dethiobiotin to biotin (and biotinyl-5′-AMP))
    (e), and citrate synthesis (citrate to/from oxaloacetate and acetyl-CoA) (f).
    APS, adenylyl sulfate; HMP, 4-amino-5-hydroxymethyl-2-methylpyrimidine; HMP-P,
    HMP phosphate; HMP-PP, HMP diphosphate; PAPS, phosphoadenylyl sulfate; TZE, 5-(2-hydroxyethyl)-4-methylthiazole;
    TZP, TZE phosphate. The coloured boxes for b–f indicate the presence (solid) or
    the absence (shaded) of a given enzyme in a genome: Ra, R. allomycis (red); Bh,
    B. helicus (blue); Cp, C. protostelioides (green); Dc, D. cristalligena (purple);
    Pc, P. cylindrospora (orange); Ts, T. sphaerospora (yellow); Sp, S. pseudoplumigaleata
    (brown); Mb, M. bicuspidata (pink).'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SH2B2
  - CP
  - IMMT
  - PC
  - Carbohydate
  - terpenoids
  - amino acid
  - Hydroxymethyl- Phosphooxymethyl
  - pyrimidine
  - ATP
  - Sulfite
  - Cp Dc Pc
  - Sulfate
  - Sulfide
  - HMP
  - Thiamine phosphate
  - Adenosylmethionine
  - Spermidine
  - Hydroxyethylthiazole
  - Thiamine
  - phosphate
  - S-Adenosylmethionine
  - S-Adenosylmethioninamine
  - Biotin
  - ATP citrate
  - acetyl-CoA
  - Oxaloacetate Acetyl-CoA Citrate
  - Dethiobiotin
  - Citrate
  - Biotinyl-5'-AMP
  - APS
genes:
- word: APS
  symbol: APS
  source: hgnc_alias_symbol
  hgnc_symbol: SH2B2
  entrez: '10603'
- word: Cp
  symbol: CP
  source: hgnc_symbol
  hgnc_symbol: CP
  entrez: '1356'
- word: HMP
  symbol: HMP
  source: hgnc_alias_symbol
  hgnc_symbol: IMMT
  entrez: '10989'
- word: Pc
  symbol: PC
  source: hgnc_symbol
  hgnc_symbol: PC
  entrez: '5091'
chemicals:
- word: Carbohydate
  source: ''
  identifier: ''
- word: terpenoids
  source: MESH
  identifier: D013729
- word: amino acid
  source: MESH
  identifier: D000596
- word: Hydroxymethyl- Phosphooxymethyl
  source: MESH
  identifier: C020420
- word: pyrimidine
  source: MESH
  identifier: C030986
- word: ATP
  source: MESH
  identifier: D000255
- word: Sulfite
  source: MESH
  identifier: D013447
- word: Cp Dc Pc
  source: MESH
  identifier: C024660
- word: Sulfate
  source: MESH
  identifier: D013431
- word: Sulfide
  source: MESH
  identifier: D013440
- word: HMP
  source: MESH
  identifier: C040698
- word: Thiamine phosphate
  source: MESH
  identifier: D013831
- word: Adenosylmethionine
  source: MESH
  identifier: D012436
- word: Spermidine
  source: MESH
  identifier: D013095
- word: Hydroxyethylthiazole
  source: MESH
  identifier: C012572
- word: Thiamine
  source: MESH
  identifier: D013831
- word: phosphate
  source: MESH
  identifier: D010710
- word: S-Adenosylmethionine
  source: MESH
  identifier: D012436
- word: S-Adenosylmethioninamine
  source: MESH
  identifier: D013455
- word: Biotin
  source: MESH
  identifier: D001710
- word: ATP citrate
  source: MESH
  identifier: C102006
- word: acetyl-CoA
  source: MESH
  identifier: D000105
- word: Oxaloacetate Acetyl-CoA Citrate
  source: MESH
  identifier: C102006
- word: Dethiobiotin
  source: MESH
  identifier: C004749
- word: Citrate
  source: MESH
  identifier: C102006
- word: Biotinyl-5'-AMP
  source: MESH
  identifier: D000249
diseases:
- word: APS
  source: MESH
  identifier: D016884
---
