---
figid: PMC6659691__41598_2019_47381_Fig1_HTML
figlink: /pmc/articles/PMC6659691/figure/Fig1/
number: F1
caption: An illustration shows the general process for creating single cell and multi-cell
  computational models that were validated directly by comparing the predicted results
  against the chemokine and cytokine profiles produced from the same cells grown in
  single cell and multi-cell cultures. (A) Cell-specific computational models were
  created in a series of successive steps. Information from high quality genomic,
  transcriptomic, proteomic, and metabolomic datasets were downloaded into the computational
  network library and used to create computational models for single GE KER, DC, and
  HTL. We used these individual models to create GE KER + DC + HTL multi-cell computational
  models to predict the effects of inflammatory events. The predicted responses were
  validated against the observed responses of the same cells grown in multi-cell cultures.
  (B) In multi-cell cultures, each cell type reacts to both the presence of an added
  LPS agonist as well as the presence of chemokines and cytokines produced from adjacent
  cells, and vice versa. (C) The autocrine and paracrine responses of the multi-cell
  interaction were modeled and the IFNG pathway is included as an example. The predictive
  responses were calculated using Michaelis Menton kinetics; mass action kinetics;
  as ordinary differential equations (ODEs). ODEs contained the reaction; enzyme;
  initial concentrations of protein intermediate reactants; and the Ka, Km, kcat,
  and Vmax parameters of the reactants at each step of the signaling process.
pmcid: PMC6659691
papertitle: Computational Models Accurately Predict Multi-Cell Biomarker Profiles
  in Inflammation and Cancer.
reftext: Carol L. Fischer, et al. Sci Rep. 2019;9:10877.
pmc_ranked_result_index: '116677'
pathway_score: 0.8700723
filename: 41598_2019_47381_Fig1_HTML.jpg
figtitle: Computational Models Accurately Predict Multi-Cell Biomarker Profiles in
  Inflammation and Cancer.
year: '2019'
organisms: Homo sapiens
ndex: 1ea2060e-df1a-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6659691__41598_2019_47381_Fig1_HTML.html
  '@type': Dataset
  description: An illustration shows the general process for creating single cell
    and multi-cell computational models that were validated directly by comparing
    the predicted results against the chemokine and cytokine profiles produced from
    the same cells grown in single cell and multi-cell cultures. (A) Cell-specific
    computational models were created in a series of successive steps. Information
    from high quality genomic, transcriptomic, proteomic, and metabolomic datasets
    were downloaded into the computational network library and used to create computational
    models for single GE KER, DC, and HTL. We used these individual models to create
    GE KER + DC + HTL multi-cell computational models to predict the effects of inflammatory
    events. The predicted responses were validated against the observed responses
    of the same cells grown in multi-cell cultures. (B) In multi-cell cultures, each
    cell type reacts to both the presence of an added LPS agonist as well as the presence
    of chemokines and cytokines produced from adjacent cells, and vice versa. (C)
    The autocrine and paracrine responses of the multi-cell interaction were modeled
    and the IFNG pathway is included as an example. The predictive responses were
    calculated using Michaelis Menton kinetics; mass action kinetics; as ordinary
    differential equations (ODEs). ODEs contained the reaction; enzyme; initial concentrations
    of protein intermediate reactants; and the Ka, Km, kcat, and Vmax parameters of
    the reactants at each step of the signaling process.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CD28
  - CD40
  - CSF2
  - IFNG
  - IFNGR1
  - IFNGR2
  - CCL20
  - IL10
  - IL12A
  - IL12B
  - IL13
  - IL17A
  - IL17RA
  - IL18
  - CD80
  - CXCL10
  - IL22RA1
  - IL2RA
  - IL4
  - JAK2
  - IRF6
  - TLR4
  - TNF
  - TNFRSF1A
  - RORC
  - VEGFA
  - CD86
  - IL22
  - IL2
  - Co80
  - LPS
genes:
- word: CD28
  symbol: CD28
  source: hgnc_symbol
  hgnc_symbol: CD28
  entrez: '940'
- word: CD40
  symbol: CD40
  source: hgnc_symbol
  hgnc_symbol: CD40
  entrez: '958'
- word: CSF2
  symbol: CSF2
  source: hgnc_symbol
  hgnc_symbol: CSF2
  entrez: '1437'
- word: IFNG
  symbol: IFNG
  source: hgnc_symbol
  hgnc_symbol: IFNG
  entrez: '3458'
- word: IFNGR
  symbol: IFNGR
  source: hgnc_prev_symbol
  hgnc_symbol: IFNGR1
  entrez: '3459'
- word: IFNGR2
  symbol: IFNGR2
  source: hgnc_symbol
  hgnc_symbol: IFNGR2
  entrez: '3460'
- word: CCL20
  symbol: CCL20
  source: hgnc_symbol
  hgnc_symbol: CCL20
  entrez: '6364'
- word: IL10
  symbol: IL10
  source: hgnc_symbol
  hgnc_symbol: IL10
  entrez: '3586'
- word: IL12
  symbol: IL12
  source: bioentities_symbol
  hgnc_symbol: IL12A
  entrez: '3592'
- word: IL12
  symbol: IL12
  source: bioentities_symbol
  hgnc_symbol: IL12B
  entrez: '3593'
- word: IL13
  symbol: IL13
  source: hgnc_symbol
  hgnc_symbol: IL13
  entrez: '3596'
- word: IL17
  symbol: IL-17
  source: hgnc_alias_symbol
  hgnc_symbol: IL17A
  entrez: '3605'
- word: IL17R
  symbol: IL17R
  source: hgnc_prev_symbol
  hgnc_symbol: IL17RA
  entrez: '23765'
- word: IL18
  symbol: IL18
  source: hgnc_symbol
  hgnc_symbol: IL18
  entrez: '3606'
- word: CD80/
  symbol: CD80
  source: hgnc_symbol
  hgnc_symbol: CD80
  entrez: '941'
- word: CXCL10
  symbol: CXCL10
  source: hgnc_symbol
  hgnc_symbol: CXCL10
  entrez: '3627'
- word: IL22R
  symbol: IL22R
  source: hgnc_prev_symbol
  hgnc_symbol: IL22RA1
  entrez: '58985'
- word: IL2R
  symbol: IL2R
  source: hgnc_prev_symbol
  hgnc_symbol: IL2RA
  entrez: '3559'
- word: IL4
  symbol: IL4
  source: hgnc_symbol
  hgnc_symbol: IL4
  entrez: '3565'
- word: JAK2
  symbol: JAK2
  source: hgnc_symbol
  hgnc_symbol: JAK2
  entrez: '3717'
- word: LPS
  symbol: LPS
  source: hgnc_prev_symbol
  hgnc_symbol: IRF6
  entrez: '3664'
- word: TLR4
  symbol: TLR4
  source: hgnc_symbol
  hgnc_symbol: TLR4
  entrez: '7099'
- word: TNF
  symbol: TNF
  source: hgnc_symbol
  hgnc_symbol: TNF
  entrez: '7124'
- word: TNFR
  symbol: TNF-R
  source: hgnc_alias_symbol
  hgnc_symbol: TNFRSF1A
  entrez: '7132'
- word: TOR
  symbol: TOR
  source: hgnc_alias_symbol
  hgnc_symbol: RORC
  entrez: '6097'
- word: VEGFA
  symbol: VEGFA
  source: hgnc_symbol
  hgnc_symbol: VEGFA
  entrez: '7422'
- word: CD86
  symbol: CD86
  source: hgnc_symbol
  hgnc_symbol: CD86
  entrez: '942'
- word: IL22
  symbol: IL22
  source: hgnc_symbol
  hgnc_symbol: IL22
  entrez: '50616'
- word: IL2
  symbol: IL2
  source: hgnc_symbol
  hgnc_symbol: IL2
  entrez: '3558'
chemicals:
- word: Co80
  source: MESH
  identifier: C094863
diseases:
- word: LPS
  source: MESH
  identifier: C536528
figid_alias: PMC6659691__F1
redirect_from: /figures/PMC6659691__F1
figtype: Figure
---
