---
figid: PMC6854030__fneur-10-01152-g0002
figlink: /pmc/articles/PMC6854030/figure/F2/
number: F2
caption: Amyloidogenesis and the influence of lipid managing proteins. (A) Alternative
  proteolytic processing of amyloid precursor protein (APP). Plasma membrane α-secretase
  initiates the non-amyloidogenic pathway, releasing a soluble ectodomain (sAPPα)
  and generating a membrane-anchored C-terminal fragment (CTF) of 83 amino acids.
  In contrast, the amyloidogenic pathway is initiated by β-secretase in intracellular
  acidic organelles. Its processing results in a 99-amino-acid CTF and a soluble extracellular
  sAPPβ. CTFs are subsequently processed by γ-secretases generating p3 or Aβ peptides
  and releasing an APP intracellular domain (AICD). The AICD resulting from the amyloidogenic
  pathway translocates to the nucleus and regulates amyloidogenic and lipid metabolism
  gene transcription. The AICD generated by the non-amyloidogenic pathway is rapidly
  degraded in the cytosol. (B) Effects of ApoE on the production and dynamics of Aβ
  peptides. Direct interaction of ApoE with APP modulates its internalization. Also,
  LRP1 receptor promotes APP internalization. Because ApoE2/ApoE3 isoforms have more
  affinity for this receptor than ApoE4, ApoE2, and ApoE3 avoid APP internalization,
  preventing the amyloidogenic pathway (disfavored interactions depicted as red crosses).
  Moreover, ApoE triggers a signaling pathway that stimulates APP transcription. ApoE4
  is the most potent isoform in triggering this pathway and thus Aβ production. Finally,
  ApoE also binds hydrophobic Aβ peptides through its lipid-binding domain. The lower
  lipid load of ApoE4 isoform results in a lower efficiency in Aβ clearance.
pmcid: PMC6854030
papertitle: Lipid-Binding Proteins in Brain Health and Disease.
reftext: Miriam Corraliza-Gomez, et al. Front Neurol. 2019;10:1152.
pmc_ranked_result_index: '14831'
pathway_score: 0.9546714
filename: fneur-10-01152-g0002.jpg
figtitle: Amyloidogenesis and the influence of lipid managing proteins
year: '2019'
organisms:
- Homo sapiens
ndex: f5617bb6-ded8-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6854030__fneur-10-01152-g0002.html
  '@type': Dataset
  description: Amyloidogenesis and the influence of lipid managing proteins. (A) Alternative
    proteolytic processing of amyloid precursor protein (APP). Plasma membrane α-secretase
    initiates the non-amyloidogenic pathway, releasing a soluble ectodomain (sAPPα)
    and generating a membrane-anchored C-terminal fragment (CTF) of 83 amino acids.
    In contrast, the amyloidogenic pathway is initiated by β-secretase in intracellular
    acidic organelles. Its processing results in a 99-amino-acid CTF and a soluble
    extracellular sAPPβ. CTFs are subsequently processed by γ-secretases generating
    p3 or Aβ peptides and releasing an APP intracellular domain (AICD). The AICD resulting
    from the amyloidogenic pathway translocates to the nucleus and regulates amyloidogenic
    and lipid metabolism gene transcription. The AICD generated by the non-amyloidogenic
    pathway is rapidly degraded in the cytosol. (B) Effects of ApoE on the production
    and dynamics of Aβ peptides. Direct interaction of ApoE with APP modulates its
    internalization. Also, LRP1 receptor promotes APP internalization. Because ApoE2/ApoE3
    isoforms have more affinity for this receptor than ApoE4, ApoE2, and ApoE3 avoid
    APP internalization, preventing the amyloidogenic pathway (disfavored interactions
    depicted as red crosses). Moreover, ApoE triggers a signaling pathway that stimulates
    APP transcription. ApoE4 is the most potent isoform in triggering this pathway
    and thus Aβ production. Finally, ApoE also binds hydrophobic Aβ peptides through
    its lipid-binding domain. The lower lipid load of ApoE4 isoform results in a lower
    efficiency in Aβ clearance.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - APP
  - ASS1
  - LRP1
  - MAPK9
  - MAPK14
  - MAPK1
  - MAPK3
  - MAPK8
  - MAPK11
  - MAPK10
  - MAPK13
  - MAPK12
  - 'N'
  - O SAPPB
genes:
- word: APP
  symbol: APP
  source: hgnc_symbol
  hgnc_symbol: APP
  entrez: '351'
- word: Aß
  symbol: ASS
  source: hgnc_prev_symbol
  hgnc_symbol: ASS1
  entrez: '445'
- word: LRP1
  symbol: LRP1
  source: hgnc_symbol
  hgnc_symbol: LRP1
  entrez: '4035'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK9
  entrez: '5601'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK14
  entrez: '1432'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK1
  entrez: '5594'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK3
  entrez: '5595'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK8
  entrez: '5599'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK11
  entrez: '5600'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK10
  entrez: '5602'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK13
  entrez: '5603'
- word: MAPK
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK12
  entrez: '6300'
chemicals:
- word: 'N'
  source: MESH
  identifier: D009584
- word: O SAPPB
  source: MESH
  identifier: D013481
diseases: []
figid_alias: PMC6854030__F2
redirect_from: /figures/PMC6854030__F2
figtype: Figure
---
