---
figid: PMC6039108__nihms975461f3
figlink: /pmc/articles/PMC6039108/figure/F3/
number: F3
caption: 'Double stranded break (DSB) is induced by a targeted nuclease (represented
  by scissors). DSB ends may or may not be resected (dashed or solid line, respectively),.
  The ultimate gene editing outcome (light blue boxes on the bottom) depends on several
  factors: the type of donor template provided (yellow box), the phase of cell cycle
  (light green box) and the presumed DNA repair proteins available (pink box). Gray
  boxes indicate the names of the repair mechanisms. It should be noted that the figure
  illustrates the common pathways described to date, however modification of DNA repair
  pathways and their utilization for gene editing purposes is an area of active research.
  (From left to right). A DSB with no end resection and no donor available is likely
  to result in insertions and deletions (indels) and lead to gene disruption via the
  non-homologous end joining (NHEJ) pathway. NHEJ may occur in any phase of cell cycle.
  Exogenously providing a double-stranded donor (dsDonor), which contains nuclease
  cut sites (scissors) around the gene of interest (green rectangle), may result in
  homology-independent targeted integration (HITI). The presence of microhomology
  on opposite strands of DNA around the cut site may result in gene disruption via
  the microhomology-mediated end joining (MMEJ) pathway. A recently reported method
  of gene integration, termed precise integration into target chromosome (PITCh),
  utilizes MMEJ machinery to integrate a gene of interest, which is provided by dsDonor
  with short homology arms (HA) to the DNA (HA are highlighted in orange and blue).
  The three pathways on the right are generally only active in S/G2 phases of cell
  cycle and may be used to correct a single nucleotide mutation in the DNA (represented
  by a red line). Exogenously providing a dsDonor with long homology arms may lead
  to either gene integration or gene correction via homologous recombination (HR)
  mechanism, depending on the length of the donor template. A new type of repair mechanism
  for gene correction was recently described, termed single stranded template repair
  (SSTR). Although resulting in the same outcome as HR-mediated gene correction, SSTR
  is presumed to utilize the Fanconi Anemia (FA) pathway and be RAD-51 independent.'
pmcid: PMC6039108
papertitle: Hematopoietic Stem Cell Gene Therapy –Progress and Lessons Learned.
reftext: Richard A. Morgan, et al. Cell Stem Cell. 2017 Nov 2;21(5):574-590.
pmc_ranked_result_index: '191814'
pathway_score: 0.7227168
filename: nihms975461f3.jpg
figtitle: Hematopoietic Stem Cell Gene Therapy –Progress and Lessons Learned
year: '2017'
organisms:
- Homo sapiens
ndex: db015f7e-df1c-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6039108__nihms975461f3.html
  '@type': Dataset
  description: 'Double stranded break (DSB) is induced by a targeted nuclease (represented
    by scissors). DSB ends may or may not be resected (dashed or solid line, respectively),.
    The ultimate gene editing outcome (light blue boxes on the bottom) depends on
    several factors: the type of donor template provided (yellow box), the phase of
    cell cycle (light green box) and the presumed DNA repair proteins available (pink
    box). Gray boxes indicate the names of the repair mechanisms. It should be noted
    that the figure illustrates the common pathways described to date, however modification
    of DNA repair pathways and their utilization for gene editing purposes is an area
    of active research. (From left to right). A DSB with no end resection and no donor
    available is likely to result in insertions and deletions (indels) and lead to
    gene disruption via the non-homologous end joining (NHEJ) pathway. NHEJ may occur
    in any phase of cell cycle. Exogenously providing a double-stranded donor (dsDonor),
    which contains nuclease cut sites (scissors) around the gene of interest (green
    rectangle), may result in homology-independent targeted integration (HITI). The
    presence of microhomology on opposite strands of DNA around the cut site may result
    in gene disruption via the microhomology-mediated end joining (MMEJ) pathway.
    A recently reported method of gene integration, termed precise integration into
    target chromosome (PITCh), utilizes MMEJ machinery to integrate a gene of interest,
    which is provided by dsDonor with short homology arms (HA) to the DNA (HA are
    highlighted in orange and blue). The three pathways on the right are generally
    only active in S/G2 phases of cell cycle and may be used to correct a single nucleotide
    mutation in the DNA (represented by a red line). Exogenously providing a dsDonor
    with long homology arms may lead to either gene integration or gene correction
    via homologous recombination (HR) mechanism, depending on the length of the donor
    template. A new type of repair mechanism for gene correction was recently described,
    termed single stranded template repair (SSTR). Although resulting in the same
    outcome as HR-mediated gene correction, SSTR is presumed to utilize the Fanconi
    Anemia (FA) pathway and be RAD-51 independent.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - EXO1
  - LIG3
  - XRCC6
  - ENG
  - RAD51
  - GNAL
  - CUX1
  - XRCC5
  - RPA3
  - ATRX
  - RPA1
  - PRKDC
  - BRCA2
  - NHEJ1
  - XRCC1
  - GNAS
  - RPA2
  - PARP1
  - POLE
  - BRCA1
  - DCLRE1C
  - PALB2
  - RBBP8
genes:
- word: EXO1
  symbol: EXO1
  source: hgnc_symbol
  hgnc_symbol: EXO1
  entrez: '9156'
- word: Lig3
  symbol: LIG3
  source: hgnc_symbol
  hgnc_symbol: LIG3
  entrez: '3980'
- word: Ku70/80
  symbol: KU70
  source: hgnc_alias_symbol
  hgnc_symbol: XRCC6
  entrez: '2547'
- word: end
  symbol: END
  source: hgnc_alias_symbol
  hgnc_symbol: ENG
  entrez: '2022'
- word: RAD51-
  symbol: RAD51
  source: hgnc_symbol
  hgnc_symbol: RAD51
  entrez: '5888'
- word: G1,S/G2,M
  symbol: Gs
  source: bioentities_symbol
  hgnc_symbol: GNAL
  entrez: '2774'
- word: cut
  symbol: CUT
  source: hgnc_alias_symbol
  hgnc_symbol: CUX1
  entrez: '1523'
- word: Ku70/80
  symbol: KU80
  source: hgnc_alias_symbol
  hgnc_symbol: XRCC5
  entrez: '7520'
- word: RPA
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA3
  entrez: '6119'
- word: RAD51,54
  symbol: RAD54
  source: hgnc_prev_symbol
  hgnc_symbol: ATRX
  entrez: '546'
- word: RPA
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA1
  entrez: '6117'
- word: DNA-PKcs
  symbol: DNA-PKcs
  source: hgnc_alias_symbol
  hgnc_symbol: PRKDC
  entrez: '5591'
- word: BRCA2
  symbol: BRCA2
  source: hgnc_symbol
  hgnc_symbol: BRCA2
  entrez: '675'
- word: XRCC1/XLF
  symbol: XLF
  source: hgnc_alias_symbol
  hgnc_symbol: NHEJ1
  entrez: '79840'
- word: XRCC1/XLF
  symbol: XRCC1
  source: hgnc_symbol
  hgnc_symbol: XRCC1
  entrez: '7515'
- word: G1,S/G2,M
  symbol: Gs
  source: bioentities_symbol
  hgnc_symbol: GNAS
  entrez: '2778'
- word: RPA
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA2
  entrez: '6118'
- word: PARP1
  symbol: PARP1
  source: hgnc_symbol
  hgnc_symbol: PARP1
  entrez: '142'
- word: Pole
  symbol: POLE
  source: hgnc_symbol
  hgnc_symbol: POLE
  entrez: '5426'
- word: BRCA1
  symbol: BRCA1
  source: hgnc_symbol
  hgnc_symbol: BRCA1
  entrez: '672'
- word: Artemis
  symbol: ARTEMIS
  source: hgnc_alias_symbol
  hgnc_symbol: DCLRE1C
  entrez: '64421'
- word: PALB2
  symbol: PALB2
  source: hgnc_symbol
  hgnc_symbol: PALB2
  entrez: '79728'
- word: MRN/CtIP/
  symbol: CtIP
  source: hgnc_alias_symbol
  hgnc_symbol: RBBP8
  entrez: '5932'
chemicals: []
diseases: []
figid_alias: PMC6039108__F3
redirect_from: /figures/PMC6039108__F3
figtype: Figure
---
