---
figid: PMC6668592__kobej-65-e19f2a
figlink: /pmc/articles/PMC149053/figure/fig7/
number: ''
caption: 'Comparison between CRC and normal tissue samples.(A) Differences in the
  levels of metabolites related to glycolysis, the TCA cycle, or the surrounding pathways
  between CRC and normal tissue samples. (B) Differences in the levels of metabolites
  related to nucleotide de novo synthesis or salvage pathways between CRC and normal
  tissue samples. The values indicate the mean relative levels of metabolites in the
  CRC or normal tissue samples divided by tissue weight and adjusted for the level
  of the relevant internal standard. In this figure, the differences in the levels
  of some metabolites related to glycolysis, the pentose phosphate pathway, the TCA
  cycle, glutathione biosynthesis, or amino acid pathways between the CRC and normal
  tissue samples are shown. Adenine, Uracil, Thymine, Hypoxanthine, Hypotaurine, β-Alanine,
  Taurine, Tryptophan, Glutamine, Glutamic acid, and Putrescine are the results of
  GC/MS analysis. L-Cystathionine, L-Cysteine, Glycine, L-Ornithine, L-Kynurenine,
  L-Asparate, L-Asparagine, L-Phenylalanine, L-Serine, L-Threonine, L-Tyrosine, L-Proline,
  L-Arginine, L-Valine, L-Lysine, L-Leucine, L-Isoleucine, Guanine, Guanosine, Uric
  acid, Uridine, Adenosine, and Xanthine are the results of LC/MS-based cation analysis.
  UMP, AMP, Orotic acid, S7P, 6-Phosphogluconic acid, R5P, Ru5P/Xu5P, G1P, G6P, F6P,
  2-Phosohogleceric acid, PEP, Pyruvic acid, DHAP, Quinolinic acid, NAD, Malic acid,
  Oxalic acid, Fumaric acid, and Succinic acid are the results of LC/MS-based anion
  analysis. The p-values were calculated using the Mann-Whitney U test (*p <0.05;
  **p <0.01; ***p <0.005). The white and black bars show the results for the normal
  and CRC tissue samples, respectively. Ketogenic amino acids, glucogenic amino acids,
  and ketoglucogenic amino acids are indicated in blue, red, and green, respectively.
  PEP: phosphoenolpyruvic acid, TKT: transketoaldolase, TDLO: transaldolase 1, CDO:
  cysteine dioxygenase, IDO: indolamine-2,3-dioxygenase, QPRT: quinolinate phosphoribosyltransferase,
  NAD: nicotinamide adenine dinucleotide, DHAP: dihydroxyacetone phosphate, G1P: glucose
  1-phosphate, G3P: glyceraldehyde 3-phosphate, G6P: glucose 6-phosphate, S7P: sedoheptulose
  7-phosphate, R5P: ribulose 5-phosphate, Ru5P/Xu5P: xylulose 5-phosphate, PRPP: phosphoribosyl
  pyrophosphate, F6P: fructose 6-phosphate, HPRT: hypoxanthine-guanine phosphoribosyltransferase,
  APRT: adenine phosphoribosyltransferase, UMP: uridine monophosphate, TMP: thymine
  monophosphate, AMP: adenosine monophosphate, ATP: adenosine triphosphate, IMP: inosine
  monophosphate, XMP: xanthosine monophosphate, R5P: ribulose 5-phosphate.'
pmcid: PMC6668592
papertitle: GC/MS and LC/MS-based Tissue Metabolomic Analysis Detected Increased Levels
  of Antioxidant Metabolites in Colorectal Cancer.
reftext: MEGUMI KIBI, et al. Kobe J Med Sci. 2019;65(1):E19-E27.
pmc_ranked_result_index: '134072'
pathway_score: 0.8996218
filename: kobej-65-e19f2a.jpg
figtitle: GC/MS and LC/MS-based Tissue Metabolomic Analysis Detected Increased Levels
  of Antioxidant Metabolites in Colorectal Cancer
year: '2019'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6668592__kobej-65-e19f2a.html
  '@type': Dataset
  description: 'Comparison between CRC and normal tissue samples.(A) Differences in
    the levels of metabolites related to glycolysis, the TCA cycle, or the surrounding
    pathways between CRC and normal tissue samples. (B) Differences in the levels
    of metabolites related to nucleotide de novo synthesis or salvage pathways between
    CRC and normal tissue samples. The values indicate the mean relative levels of
    metabolites in the CRC or normal tissue samples divided by tissue weight and adjusted
    for the level of the relevant internal standard. In this figure, the differences
    in the levels of some metabolites related to glycolysis, the pentose phosphate
    pathway, the TCA cycle, glutathione biosynthesis, or amino acid pathways between
    the CRC and normal tissue samples are shown. Adenine, Uracil, Thymine, Hypoxanthine,
    Hypotaurine, β-Alanine, Taurine, Tryptophan, Glutamine, Glutamic acid, and Putrescine
    are the results of GC/MS analysis. L-Cystathionine, L-Cysteine, Glycine, L-Ornithine,
    L-Kynurenine, L-Asparate, L-Asparagine, L-Phenylalanine, L-Serine, L-Threonine,
    L-Tyrosine, L-Proline, L-Arginine, L-Valine, L-Lysine, L-Leucine, L-Isoleucine,
    Guanine, Guanosine, Uric acid, Uridine, Adenosine, and Xanthine are the results
    of LC/MS-based cation analysis. UMP, AMP, Orotic acid, S7P, 6-Phosphogluconic
    acid, R5P, Ru5P/Xu5P, G1P, G6P, F6P, 2-Phosohogleceric acid, PEP, Pyruvic acid,
    DHAP, Quinolinic acid, NAD, Malic acid, Oxalic acid, Fumaric acid, and Succinic
    acid are the results of LC/MS-based anion analysis. The p-values were calculated
    using the Mann-Whitney U test (*p <0.05; **p <0.01; ***p <0.005). The white and
    black bars show the results for the normal and CRC tissue samples, respectively.
    Ketogenic amino acids, glucogenic amino acids, and ketoglucogenic amino acids
    are indicated in blue, red, and green, respectively. PEP: phosphoenolpyruvic acid,
    TKT: transketoaldolase, TDLO: transaldolase 1, CDO: cysteine dioxygenase, IDO:
    indolamine-2,3-dioxygenase, QPRT: quinolinate phosphoribosyltransferase, NAD:
    nicotinamide adenine dinucleotide, DHAP: dihydroxyacetone phosphate, G1P: glucose
    1-phosphate, G3P: glyceraldehyde 3-phosphate, G6P: glucose 6-phosphate, S7P: sedoheptulose
    7-phosphate, R5P: ribulose 5-phosphate, Ru5P/Xu5P: xylulose 5-phosphate, PRPP:
    phosphoribosyl pyrophosphate, F6P: fructose 6-phosphate, HPRT: hypoxanthine-guanine
    phosphoribosyltransferase, APRT: adenine phosphoribosyltransferase, UMP: uridine
    monophosphate, TMP: thymine monophosphate, AMP: adenosine monophosphate, ATP:
    adenosine triphosphate, IMP: inosine monophosphate, XMP: xanthosine monophosphate,
    R5P: ribulose 5-phosphate.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CBS
  - QPRT
  - CDON
  - PAEP
  - IDO1
  - TALDO1
  - TKT
  - gluconic acid Ru5P
  - amino acid
  - Nucleotide
  - L-Serine
  - Glycine
  - DHAP
  - Cysteine
  - Cystathionine
  - Tryptophan
  - L-Leucine
  - Hypotaurine
  - Glutamine
  - L-Isoleucine
  - L-Kynurenine
  - Pyruvic acid
  - NAD
  - Taurine
  - Glutamic acid
  - L-Lysine
  - Quinolinic acid
  - L-Proline
  - L-Ornithine
  - Acetyl CoA
  - Oxalic acid
  - ketoglutarate
  - L-Arginine
  - Asparagine
  - L-Asparate
  - succinyl-CoA
  - Putrescine
  - Malic acid
  - TSA
  - L-Threonine
  - Fumaric acid
  - L-Phenylalanine
  - L-Tyrosine
  - L-Valine
genes:
- word: CBS
  symbol: CBS
  source: hgnc_symbol
  hgnc_symbol: CBS
  entrez: '875'
- word: QPRT
  symbol: QPRT
  source: hgnc_symbol
  hgnc_symbol: QPRT
  entrez: '23475'
- word: CDO
  symbol: CDO
  source: hgnc_alias_symbol
  hgnc_symbol: CDON
  entrez: '50937'
- word: PEP
  symbol: PEP
  source: hgnc_alias_symbol
  hgnc_symbol: PAEP
  entrez: '5047'
- word: IDO
  symbol: IDO
  source: hgnc_prev_symbol
  hgnc_symbol: IDO1
  entrez: '3620'
- word: TALDO1
  symbol: TALDO1
  source: hgnc_symbol
  hgnc_symbol: TALDO1
  entrez: '6888'
- word: TKT
  symbol: TKT
  source: hgnc_symbol
  hgnc_symbol: TKT
  entrez: '7086'
chemicals:
- word: gluconic acid Ru5P
  source: MESH
  identifier: C030691
- word: amino acid
  source: MESH
  identifier: D000596
- word: Nucleotide
  source: MESH
  identifier: D009711
- word: L-Serine
  source: MESH
  identifier: C047902
- word: Glycine
  source: MESH
  identifier: D005998
- word: DHAP
  source: MESH
  identifier: C017202
- word: Cysteine
  source: MESH
  identifier: D003545
- word: Cystathionine
  source: MESH
  identifier: D003540
- word: Tryptophan
  source: ''
  identifier: ''
- word: L-Leucine
  source: MESH
  identifier: C027819
- word: Hypotaurine
  source: MESH
  identifier: C003949
- word: Glutamine
  source: MESH
  identifier: C578860
- word: L-Isoleucine
  source: MESH
  identifier: C043801
- word: L-Kynurenine
  source: MESH
  identifier: C029366
- word: Pyruvic acid
  source: MESH
  identifier: D019289
- word: NAD
  source: MESH
  identifier: D009243
- word: Taurine
  source: MESH
  identifier: D013654
- word: Glutamic acid
  source: MESH
  identifier: C030030
- word: L-Lysine
  source: MESH
  identifier: C114808
- word: Quinolinic acid
  source: MESH
  identifier: D017378
- word: L-Proline
  source: MESH
  identifier: C489032
- word: L-Ornithine
  source: MESH
  identifier: C008973
- word: Acetyl CoA
  source: MESH
  identifier: D000105
- word: Oxalic acid
  source: MESH
  identifier: D019815
- word: ketoglutarate
  source: MESH
  identifier: C029743
- word: L-Arginine
  source: ''
  identifier: ''
- word: Asparagine
  source: MESH
  identifier: C050801
- word: L-Asparate
  source: MESH
  identifier: C025076
- word: succinyl-CoA
  source: MESH
  identifier: C012046
- word: Putrescine
  source: MESH
  identifier: D011700
- word: Malic acid
  source: MESH
  identifier: C030298
- word: TSA
  source: MESH
  identifier: C070842
- word: L-Threonine
  source: MESH
  identifier: C033059
- word: Fumaric acid
  source: MESH
  identifier: C032005
- word: L-Phenylalanine
  source: MESH
  identifier: C119108
- word: L-Tyrosine
  source: ''
  identifier: ''
- word: L-Valine
  source: MESH
  identifier: C035810
diseases: []
---
