---
figid: PMC6226483__41598_2018_34730_Fig4_HTML
figlink: /pmc/articles/PMC6226483/figure/Fig4/
number: Figure 4
caption: The complexity of the carbon assimilation metabolic pathway of cassava based
  on reaction-enzyme association. (a) The schematic of relationship among annotated
  enzymatic proteins (P), enzymes (EC), enzymatic reactions (RXN), and metabolites
  (MET); (b) Examples of the reaction-enzyme association in MeRecon; (c) The relationship
  between enzymes (EC numbers) and reactions in MeRecon, represented as nodes and
  edges. Nodes consist of enzymes (circles) and enzymatic reactions (rectangles),
  and the edges between the nodes represent reaction-enzyme association of each metabolic
  reaction. Size of enzyme nodes represents node degree, and the number of enzymatic
  reactions catalyzed by that enzyme. AMI - amino acid biosynthesis, CAL - Calvin
  cycle, CEL - cell wall biosynthesis, FAT - fatty acid biosynthesis, NUC - nucleotide
  biosynthesis, RES – respiration, STA - starch biosynthesis, and SUC - sucrose biosynthesis.
pmcid: PMC6226483
papertitle: Unlocking conserved and diverged metabolic characteristics in cassava
  carbon assimilation via comparative genomics approach.
reftext: Wanatsanan Siriwat, et al. Sci Rep. 2018;8:16593.
pmc_ranked_result_index: '15843'
pathway_score: 0.7659873
filename: 41598_2018_34730_Fig4_HTML.jpg
figtitle: The complexity of the carbon assimilation metabolic pathway of cassava based
  on reaction-enzyme association
year: '2018'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6226483__41598_2018_34730_Fig4_HTML.html
  '@type': Dataset
  description: The complexity of the carbon assimilation metabolic pathway of cassava
    based on reaction-enzyme association. (a) The schematic of relationship among
    annotated enzymatic proteins (P), enzymes (EC), enzymatic reactions (RXN), and
    metabolites (MET); (b) Examples of the reaction-enzyme association in MeRecon;
    (c) The relationship between enzymes (EC numbers) and reactions in MeRecon, represented
    as nodes and edges. Nodes consist of enzymes (circles) and enzymatic reactions
    (rectangles), and the edges between the nodes represent reaction-enzyme association
    of each metabolic reaction. Size of enzyme nodes represents node degree, and the
    number of enzymatic reactions catalyzed by that enzyme. AMI - amino acid biosynthesis,
    CAL - Calvin cycle, CEL - cell wall biosynthesis, FAT - fatty acid biosynthesis,
    NUC - nucleotide biosynthesis, RES – respiration, STA - starch biosynthesis, and
    SUC - sucrose biosynthesis.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TCF15
  - MET
  - RLN3
  - SLC51A
  - RO0430
  - MET B
  - F01858
  - O0000
  - O000
  - AMI O CAL OCEL OFAT ONUC ORES OSTA Osuc O
  - O0000 00000 O000 AMI
genes:
- word: EC2
  symbol: EC2
  source: hgnc_alias_symbol
  hgnc_symbol: TCF15
  entrez: '6939'
- word: (MET
  symbol: MET
  source: hgnc_symbol
  hgnc_symbol: MET
  entrez: '4233'
- word: RXN3
  symbol: RXN3
  source: hgnc_alias_symbol
  hgnc_symbol: RLN3
  entrez: '117579'
- word: OSTA
  symbol: OSTalpha
  source: hgnc_alias_symbol
  hgnc_symbol: SLC51A
  entrez: '200931'
chemicals:
- word: RO0430
  source: MESH
  identifier: C410428
- word: MET B
  source: MESH
  identifier: C098756
- word: F01858
  source: MESH
  identifier: D005461
- word: O0000
  source: MESH
  identifier: C510817
- word: O000
  source: MESH
  identifier: D007479
- word: AMI O CAL OCEL OFAT ONUC ORES OSTA Osuc O
  source: MESH
  identifier: C090894
diseases:
- word: O0000 00000 O000 AMI
  source: ''
  identifier: ''
---
