---
figid: PMC6381062__fpls-10-00102-g002
figlink: /pmc/articles/PMC6381062/figure/F2/
number: FIGURE 2
caption: A working model of NPR1, NPR3, and NPR4. The left side of the diagram, partially
  separated by a dashed line, depicts the regulation of NPR1-dependant defense genes
  in SA naïve cells. At low SA concentrations, S-nitrosylation of NPR1Cys156 by GSNO
  encourages the existence of NPR1 in its oligomeric form. The oligomeric form of
  NPR1 is predominantly found within the cytoplasm. To prevent the uninduced expression
  of SAR-related genes, several mechanisms are in place to suppress NPR1-dependant
  defense gene expression. Phosphorylation of NPR1Ser55/59 suppresses defense gene
  expression and prevents sumoylation of NPR1 by SUMO3, an important aspect of NPR1
  activation. The N-terminal BTB/POZ domain of NPR1 interacts with and suppresses
  the function of the C-terminal transactivation domain of NPR1. Additionally, NIMIN
  proteins interact with NPR1 to suppress gene expression. Paralogs of NPR1, NPR3,
  and NPR4, interact with TGA2/TGA5/TGA6 to further suppress transcription. Certain
  WRKY transcription factors act as transcriptional repressors of a subset of SAR-related
  genes. Finally, NPR1 is degraded by the 26S proteasome following CUL3-mediated ubiquitinylation.
  However, NPR1 is unable to directly interact with CUL3 and E3 ligases, likely requiring
  a substrate adapter. The right side of the diagram depicts NPR1 regulation in SAR-induced
  cells where SA concentration is elevated either due to exogenous application of
  SA/one of its functional analogs or during biotrophic/hemibiotrophic pathogen challenge.
  Increased oxidative stress and subsequent increases in antioxidant production leads
  to the reduction of NPR1Cys156, specifically by thioredoxins, leading to the disassembly
  of the NPR1 oligomer. Within the cytoplasm, NPR1 antagonizes the JA-defense response
  pathway. Monomeric NPR1 is then translocated to the nucleus via the action of a
  bipartite nuclear localization signal. Within the nucleus, NPR1 suppresses the expression
  of ICS1 which is essential to SA synthesis in response to pathogenic stress, forming
  a negative feedback loop. Phosphorylation of NPR1Ser11/15 within the N-terminal
  IκB-like phosphodegron motif both enhances interaction with SUMO3 and targets NPR1
  for ubiquitinylation and degradation by the 26S proteasome. Sumoylation of NPR1
  by SUMO3 also increases and decreases association of NPR1 with TGA and WRKY transcription
  factors, respectively. SUMO3 is also required for phosphorylation of NPR1Ser11/15,
  creating an amplification loop which leads to the activation of more NPR1, increasing
  SAR-related gene expression. Interaction of SA and NPR1 requires a transition metal.
  Following binding of SA to NPR1 a conformational change of C-terminal transactivation
  domain of NPR1 decreases its affinity for the inhibitory N-terminal BTB/POZ domain.
  In turn, the BTB/POZ domain of NPR1 interacts with the N-terminal repression domain
  of TGA transcription factors, thereby activating transcription. Furthermore, binding
  of SA to NPR1 alters its interaction with NIMINs, relieving repression. Moreover,
  binding of SA to NPR3/NPR4 diminishes their ability to suppress SAR-related gene
  expression. Turnover of NPR1 through degradation by the 26S proteasome is essential
  to preserving peak gene expression and is required for the complete induction of
  SAR.
pmcid: PMC6381062
papertitle: 'The NONEXPRESSOR OF PATHOGENESIS-RELATED GENES 1 (NPR1) and Related Family:
  Mechanistic Insights in Plant Disease Resistance.'
reftext: Robert Backer, et al. Front Plant Sci. 2019;10:102.
pmc_ranked_result_index: '153447'
pathway_score: 0.9626378
filename: fpls-10-00102-g002.jpg
figtitle: A working model of NPR1, NPR3, and NPR4
year: '2019'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6381062__fpls-10-00102-g002.html
  '@type': Dataset
  description: A working model of NPR1, NPR3, and NPR4. The left side of the diagram,
    partially separated by a dashed line, depicts the regulation of NPR1-dependant
    defense genes in SA naïve cells. At low SA concentrations, S-nitrosylation of
    NPR1Cys156 by GSNO encourages the existence of NPR1 in its oligomeric form. The
    oligomeric form of NPR1 is predominantly found within the cytoplasm. To prevent
    the uninduced expression of SAR-related genes, several mechanisms are in place
    to suppress NPR1-dependant defense gene expression. Phosphorylation of NPR1Ser55/59
    suppresses defense gene expression and prevents sumoylation of NPR1 by SUMO3,
    an important aspect of NPR1 activation. The N-terminal BTB/POZ domain of NPR1
    interacts with and suppresses the function of the C-terminal transactivation domain
    of NPR1. Additionally, NIMIN proteins interact with NPR1 to suppress gene expression.
    Paralogs of NPR1, NPR3, and NPR4, interact with TGA2/TGA5/TGA6 to further suppress
    transcription. Certain WRKY transcription factors act as transcriptional repressors
    of a subset of SAR-related genes. Finally, NPR1 is degraded by the 26S proteasome
    following CUL3-mediated ubiquitinylation. However, NPR1 is unable to directly
    interact with CUL3 and E3 ligases, likely requiring a substrate adapter. The right
    side of the diagram depicts NPR1 regulation in SAR-induced cells where SA concentration
    is elevated either due to exogenous application of SA/one of its functional analogs
    or during biotrophic/hemibiotrophic pathogen challenge. Increased oxidative stress
    and subsequent increases in antioxidant production leads to the reduction of NPR1Cys156,
    specifically by thioredoxins, leading to the disassembly of the NPR1 oligomer.
    Within the cytoplasm, NPR1 antagonizes the JA-defense response pathway. Monomeric
    NPR1 is then translocated to the nucleus via the action of a bipartite nuclear
    localization signal. Within the nucleus, NPR1 suppresses the expression of ICS1
    which is essential to SA synthesis in response to pathogenic stress, forming a
    negative feedback loop. Phosphorylation of NPR1Ser11/15 within the N-terminal
    IκB-like phosphodegron motif both enhances interaction with SUMO3 and targets
    NPR1 for ubiquitinylation and degradation by the 26S proteasome. Sumoylation of
    NPR1 by SUMO3 also increases and decreases association of NPR1 with TGA and WRKY
    transcription factors, respectively. SUMO3 is also required for phosphorylation
    of NPR1Ser11/15, creating an amplification loop which leads to the activation
    of more NPR1, increasing SAR-related gene expression. Interaction of SA and NPR1
    requires a transition metal. Following binding of SA to NPR1 a conformational
    change of C-terminal transactivation domain of NPR1 decreases its affinity for
    the inhibitory N-terminal BTB/POZ domain. In turn, the BTB/POZ domain of NPR1
    interacts with the N-terminal repression domain of TGA transcription factors,
    thereby activating transcription. Furthermore, binding of SA to NPR1 alters its
    interaction with NIMINs, relieving repression. Moreover, binding of SA to NPR3/NPR4
    diminishes their ability to suppress SAR-related gene expression. Turnover of
    NPR1 through degradation by the 26S proteasome is essential to preserving peak
    gene expression and is required for the complete induction of SAR.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - NPR1
  - TXN
  - NPR3
  - CUL3
  - SUMO3
genes:
- word: NPR1
  symbol: NPR1
  source: hgnc_symbol
  hgnc_symbol: NPR1
  entrez: '4881'
- word: TRX
  symbol: TRX
  source: hgnc_alias_symbol
  hgnc_symbol: TXN
  entrez: '7295'
- word: NPR3/4
  symbol: NPR3
  source: hgnc_symbol
  hgnc_symbol: NPR3
  entrez: '4883'
- word: CUL3
  symbol: CUL3
  source: hgnc_symbol
  hgnc_symbol: CUL3
  entrez: '8452'
- word: SUMO3
  symbol: SUMO3
  source: hgnc_symbol
  hgnc_symbol: SUMO3
  entrez: '6612'
chemicals: []
diseases: []
---
