---
figid: PMC6064744__fnagi-10-00220-g002
figlink: /pmc/articles/PMC6064744/figure/F2/
number: FIGURE 2
caption: Schematic representation of cellular mechanisms possibly implicated in miRNA
  dysregulation in prion diseases. The canonical miRNA biogenesis and function pathway
  is indicated in green boxes. Mechanisms possibly contributing to miRNA dyregulation
  in prion diseases are indicated with dashed red lines. miRNA genes, residing in
  exonic, intronic or intergenic regions, are transcribed by RNA polymerase II or
  III (1). Altered miRNA gene transcription rate may impact miRNA levels. This can
  be associated to altered (≠) abundance of Transciption Factors (TF) inducing miRNA
  gene expression (e.g., STAT3), and/or to Single Nucleotide Variants (SNPs) residing
  within miRNA-gene promoter regions (e.g., rs57095329, illustrated as solid red line).
  Upon transcription, pri-miRNAs’ flanking sequences are removed by the microprossesor
  complex, containing Drosha and DGCR8 (2). This processing generates pre-miRNA stem-loop
  structures (3). Pre-miRNAs are exported into the cytoplasm through Exportin5 (EXPO5).
  Dicer removes the pre-miRNA loop, generating a miRNA duplex (4). SNPs in miRNA genes
  may interfere with pri- and/or pre-miRNAs processing by Drosha/DGCR8 (2) and Dicer
  (4) respectively. RNA editing, mediated by ADARs and/or APOBEC1, may occur in pri-
  and/or pre-miRNAs, introducing Single Nucleotide Variants (SNVs) with the potential
  to affect miRNA maturation (2, 4). Reduced Drosha, DGCR8 and Dicer levels, reported
  in human prion diseases, may impact miRNA maturation (2, 4). Following Dicer processing,
  the miRNA duplex is loaded from the RISC-loading complex (RLC) onto RISC. Ago proteins,
  are core components of RISC, forming the active functional unit of the miRNA-mediated
  gene regulation (5). Abnormal subcellular localization of Ago proteins, reported
  in sCJD, may affect miRNA machinery (5). PrPC-Ago interactions, which stabilize
  RISC, may be affected by PrPSc presence. Similarly, PrPSc could possibly impact
  PrPC-Dicer interactions and thus miRNA maturation and function. Incorporated into
  the RISC complex, miRNAs promote either translational repression or mRNA degradation,
  according to sequence complementary (5). SNPs in miRNA seed sequences and/or RNA
  editing events possibly occurring within miRNA seed sequences may impact miRNA-target
  mRNA interactions and even redirect miRNAs in new targets (5).
pmcid: PMC6064744
papertitle: 'MicroRNA Alterations in the Brain and Body Fluids of Humans and Animal
  Prion Disease Models: Current Status and Perspectives.'
reftext: Eirini Kanata, et al. Front Aging Neurosci. 2018;10:220.
pmc_ranked_result_index: '232852'
pathway_score: 0.8788845
filename: fnagi-10-00220-g002.jpg
figtitle: Schematic representation of cellular mechanisms possibly implicated in miRNA
  dysregulation in prion diseases
year: '2018'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6064744__fnagi-10-00220-g002.html
  '@type': Dataset
  description: Schematic representation of cellular mechanisms possibly implicated
    in miRNA dysregulation in prion diseases. The canonical miRNA biogenesis and function
    pathway is indicated in green boxes. Mechanisms possibly contributing to miRNA
    dyregulation in prion diseases are indicated with dashed red lines. miRNA genes,
    residing in exonic, intronic or intergenic regions, are transcribed by RNA polymerase
    II or III (1). Altered miRNA gene transcription rate may impact miRNA levels.
    This can be associated to altered (≠) abundance of Transciption Factors (TF) inducing
    miRNA gene expression (e.g., STAT3), and/or to Single Nucleotide Variants (SNPs)
    residing within miRNA-gene promoter regions (e.g., rs57095329, illustrated as
    solid red line). Upon transcription, pri-miRNAs’ flanking sequences are removed
    by the microprossesor complex, containing Drosha and DGCR8 (2). This processing
    generates pre-miRNA stem-loop structures (3). Pre-miRNAs are exported into the
    cytoplasm through Exportin5 (EXPO5). Dicer removes the pre-miRNA loop, generating
    a miRNA duplex (4). SNPs in miRNA genes may interfere with pri- and/or pre-miRNAs
    processing by Drosha/DGCR8 (2) and Dicer (4) respectively. RNA editing, mediated
    by ADARs and/or APOBEC1, may occur in pri- and/or pre-miRNAs, introducing Single
    Nucleotide Variants (SNVs) with the potential to affect miRNA maturation (2, 4).
    Reduced Drosha, DGCR8 and Dicer levels, reported in human prion diseases, may
    impact miRNA maturation (2, 4). Following Dicer processing, the miRNA duplex is
    loaded from the RISC-loading complex (RLC) onto RISC. Ago proteins, are core components
    of RISC, forming the active functional unit of the miRNA-mediated gene regulation
    (5). Abnormal subcellular localization of Ago proteins, reported in sCJD, may
    affect miRNA machinery (5). PrPC-Ago interactions, which stabilize RISC, may be
    affected by PrPSc presence. Similarly, PrPSc could possibly impact PrPC-Dicer
    interactions and thus miRNA maturation and function. Incorporated into the RISC
    complex, miRNAs promote either translational repression or mRNA degradation, according
    to sequence complementary (5). SNPs in miRNA seed sequences and/or RNA editing
    events possibly occurring within miRNA seed sequences may impact miRNA-target
    mRNA interactions and even redirect miRNAs in new targets (5).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TF
  - ADAR
  - ITGA9
  - FBXW7
  - DICER1
  - DROSHA
  - SCPEP1
  - MSMB
  - APOBEC1
  - Ti5
  - Ca
genes:
- word: TF
  symbol: TF
  source: hgnc_symbol
  hgnc_symbol: TF
  entrez: '7018'
- word: ADAR
  symbol: ADAR
  source: hgnc_symbol
  hgnc_symbol: ADAR
  entrez: '103'
- word: RLC
  symbol: RLC
  source: hgnc_alias_symbol
  hgnc_symbol: ITGA9
  entrez: '3680'
- word: Ago
  symbol: AGO
  source: hgnc_alias_symbol
  hgnc_symbol: FBXW7
  entrez: '55294'
- word: Dicer
  symbol: Dicer
  source: hgnc_alias_symbol
  hgnc_symbol: DICER1
  entrez: '23405'
- word: Drosha
  symbol: DROSHA
  source: hgnc_symbol
  hgnc_symbol: DROSHA
  entrez: '29102'
- word: RISC
  symbol: RISC
  source: hgnc_alias_symbol
  hgnc_symbol: SCPEP1
  entrez: '59342'
- word: Prps
  symbol: PRPS
  source: hgnc_alias_symbol
  hgnc_symbol: MSMB
  entrez: '4477'
- word: APOBEC1
  symbol: APOBEC1
  source: hgnc_symbol
  hgnc_symbol: APOBEC1
  entrez: '339'
chemicals:
- word: Ti5
  source: MESH
  identifier: D014025
- word: Ca
  source: MESH
  identifier: D002118
diseases: []
---
