---
figid: PMC6617847__12870_2019_1793_Fig5_HTML
figlink: /pmc/articles/PMC6617847/figure/Fig5/
number: F5
caption: Schematic outline of the salinity-induced response pathway based on proteome,
  gene expression, and physiological changes in chickpea leaves. Numbers in yellow
  circles are corresponding to the protein(s) mentioned in Table . Blue arrows indicate
  induction or increase; red arrows indicate repression or decrease, and dotted arrows
  indicate possible or reversible alterations. Salinity activates several signaling
  cascades, regulates gene expression, and promotes regulatory and functional protein
  biosynthesis, in which the main role of ROS, antioxidants, and chaperones under
  saline condition is presented. Tolerant seedlings inhibited or decreased major metabolic
  pathways (i.e., photosynthesis, photorespiration, TCA cycle, and amino acid metabolism)
  less than susceptible seedlings to provide more energy and other compatible metabolites.
  Importantly, salt tolerance relies on higher transporter activities to remove H2O2,
  Na+, etc., and enhance osmotic regulation, and membrane and cell wall re-modulation
  to cope with the stress
pmcid: PMC6617847
papertitle: Comparative proteomics and gene expression analyses revealed responsive
  proteins and mechanisms for salt tolerance in chickpea genotypes.
reftext: Mohammad Arefian, et al. BMC Plant Biol. 2019;19:300.
pmc_ranked_result_index: '3233'
pathway_score: 0.933715
filename: 12870_2019_1793_Fig5_HTML.jpg
figtitle: Schematic outline of the salinity-induced response pathway based on proteome,
  gene expression, and physiological changes in chickpea leaves
year: '2019'
organisms: Homo sapiens
ndex: 531d4c2c-df09-11ea-99da-0ac135e8bacf
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC6617847__12870_2019_1793_Fig5_HTML.html
  '@type': Dataset
  description: Schematic outline of the salinity-induced response pathway based on
    proteome, gene expression, and physiological changes in chickpea leaves. Numbers
    in yellow circles are corresponding to the protein(s) mentioned in Table . Blue
    arrows indicate induction or increase; red arrows indicate repression or decrease,
    and dotted arrows indicate possible or reversible alterations. Salinity activates
    several signaling cascades, regulates gene expression, and promotes regulatory
    and functional protein biosynthesis, in which the main role of ROS, antioxidants,
    and chaperones under saline condition is presented. Tolerant seedlings inhibited
    or decreased major metabolic pathways (i.e., photosynthesis, photorespiration,
    TCA cycle, and amino acid metabolism) less than susceptible seedlings to provide
    more energy and other compatible metabolites. Importantly, salt tolerance relies
    on higher transporter activities to remove H2O2, Na+, etc., and enhance osmotic
    regulation, and membrane and cell wall re-modulation to cope with the stress
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SOS2
  - YWHAQ
  - SHBG
  - AVP
  - TFAP2A
  - GNB2
  - SOS1
  - CA2
  - MAPK10
  - MAPK9
  - MAPK11
  - MAPK13
  - MAPK1
  - MAPK14
  - MAPK12
  - MAPK8
  - MYB
  - NLRP1
  - MYC
  - GNB4
  - TNFSF14
  - SLTM
  - GNB1
  - GNB3
  - GRP
  - GNB5
  - MAPK3
  - Na
  - Ca
  - CO
  - Sugar
  - Pentose Phosphate
  - TCA
  - NAC
  - NADPH
  - Polyols
  - sugars
  - Fatty acids
  - Met
  - Glu
  - polyamine
  - ethylene
  - Amino acid
  - callose
genes:
- word: (soS,
  symbol: SOS
  source: bioentities_symbol
  hgnc_symbol: SOS2
  entrez: '6655'
- word: 14-3-3
  symbol: 14-3-3
  source: hgnc_alias_symbol
  hgnc_symbol: YWHAQ
  entrez: '10971'
- word: ABP,
  symbol: ABP
  source: hgnc_alias_symbol
  hgnc_symbol: SHBG
  entrez: '6462'
- word: (soS,ATPase,AVP,NHX,HKT,PIP,etc.)
  symbol: AVP
  source: hgnc_symbol
  hgnc_symbol: AVP
  entrez: '551'
- word: factors(MYB,AP2,
  symbol: AP-2
  source: hgnc_alias_symbol
  hgnc_symbol: TFAP2A
  entrez: '7020'
- word: (GB,
  symbol: G_beta
  source: bioentities_symbol
  hgnc_symbol: GNB2
  entrez: '2783'
- word: (soS,
  symbol: SOS
  source: bioentities_symbol
  hgnc_symbol: SOS1
  entrez: '6654'
- word: Ca2+||ABA
  symbol: CA-II
  source: hgnc_alias_symbol
  hgnc_symbol: CA2
  entrez: '760'
- word: (MAPK,
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK10
  entrez: '5602'
- word: (MAPK,
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK9
  entrez: '5601'
- word: (MAPK,
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK11
  entrez: '5600'
- word: (MAPK,
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK13
  entrez: '5603'
- word: (MAPK,
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK1
  entrez: '5594'
- word: (MAPK,
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK14
  entrez: '1432'
- word: (MAPK,
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK12
  entrez: '6300'
- word: (MAPK,
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK8
  entrez: '5599'
- word: (MYB,
  symbol: MYB
  source: hgnc_symbol
  hgnc_symbol: MYB
  entrez: '4602'
- word: MYC,BZIP,NAC,
  symbol: NAC
  source: hgnc_alias_symbol
  hgnc_symbol: NLRP1
  entrez: '22861'
- word: MYC,BZIP,NAC,
  symbol: MYC
  source: hgnc_symbol
  hgnc_symbol: MYC
  entrez: '4609'
- word: (GB,
  symbol: G_beta
  source: bioentities_symbol
  hgnc_symbol: GNB4
  entrez: '59345'
- word: Light
  symbol: LIGHT
  source: hgnc_alias_symbol
  hgnc_symbol: TNFSF14
  entrez: '8740'
- word: Met,
  symbol: Met
  source: hgnc_alias_symbol
  hgnc_symbol: SLTM
  entrez: '79811'
- word: (GB,
  symbol: G_beta
  source: bioentities_symbol
  hgnc_symbol: GNB1
  entrez: '2782'
- word: (GB,
  symbol: G_beta
  source: bioentities_symbol
  hgnc_symbol: GNB3
  entrez: '2784'
- word: Structuraladaptation(Membranes,Cellwall,lignin,GRP,callose)
  symbol: GRP
  source: hgnc_symbol
  hgnc_symbol: GRP
  entrez: '2922'
- word: (GB,
  symbol: G_beta
  source: bioentities_symbol
  hgnc_symbol: GNB5
  entrez: '10681'
- word: (MAPK,
  symbol: MAPK
  source: bioentities_symbol
  hgnc_symbol: MAPK3
  entrez: '5595'
chemicals:
- word: Na
  source: MESH
  identifier: D012964
- word: Ca
  source: MESH
  identifier: D002118
- word: CO
  source: MESH
  identifier: D002245
- word: Sugar
  source: MESH
  identifier: D002241
- word: Pentose Phosphate
  source: MESH
  identifier: D010429
- word: TCA
  source: MESH
  identifier: C000589078
- word: NAC
  source: MESH
  identifier: C086501
- word: NADPH
  source: MESH
  identifier: D009249
- word: Polyols
  source: MESH
  identifier: C029873
- word: sugars
  source: MESH
  identifier: D002241
- word: Fatty acids
  source: MESH
  identifier: D005227
- word: Met
  source: MESH
  identifier: C034758
- word: Glu
  source: MESH
  identifier: C094686
- word: polyamine
  source: MESH
  identifier: D011073
- word: ethylene
  source: MESH
  identifier: D005030
- word: Amino acid
  source: MESH
  identifier: D000596
- word: callose
  source: MESH
  identifier: C048306
diseases: []
figid_alias: PMC6617847__F5
redirect_from: /figures/PMC6617847__F5
figtype: Figure
---
