---
figid: PMC7022656__antioxidants-09-00004-g001
figlink: pmc/articles/PMC7022656/figure/antioxidants-09-00004-f001/
number: F1
caption: Inhibition of Nrf1 N-glycosylation by tunicamycin (TU) in the endoplasmic
  reticulum (ER)- to-nuclear signaling responses. A topobiological model is herein
  proposed to explain the inhibition of N-glycosylation of Nrf1 by TU, serving as
  a classic ER stressor, which can also block the ensuing deglycosylation and proteolytic
  processing of this cap’n’collar (CNC)-basic-region leucine zipper (bZIP) protein
  to yield a mature transcription factor. The resulting non-glycosylated Nrf1 and
  others are subjected to the ER-associated degradation (ERAD) pathway mediated by
  proteasomes. To the contrary, the non-glycosylated and unfolded proteins are accumulated
  insomuch as to stimulate ER stress. Thereby, inhibition of Nrf1 by TU is accompanied
  by the activation of a canonic ER-to-nuclear response signaling mediated by PERK,
  IRE1 and ATF6, three key transducers tethered to ER membranes as reviewed by the
  authors [,,,]. Of note, PERK was identified as a direct upstream kinase of Nrf2,
  as well as elF2α [], but Nrf2 is neither translocated nor N-glycosylated in the
  lumen of ER. As such, Nrf2 is generally accepted as a master transcription factor
  to regulate cognate genes driven by antioxidant response elements (ARE) in the promoter
  regions. In such responses, ATF4 is selectively translated by the elF2α-controlled
  initiation machinery to regulate target genes (including Chop, Ero1, Gadd34) containing
  amino acid response elements (AARE) in their promoters. To get rid of ER stress,
  ATF6 is activated by its consecutive proteolytic processing by Site-1 and Site-2
  proteases (i.e., S1P and S2P, respectively) in the Golgi apparatus. The full-length
  mRNA of XBP is alternatively spliced to remove its 26 nucleotides at the nearly
  3′-end, such that the subsequent portion of its open reading frame is shifted so
  much as to yield another inducible protein isoform, called XBP-1s, that is longer
  in size than its original protein of XBP-1u. Consequently, distinct subsets of cognate
  genes containing cis-regulatory ESRE (ER stress response element) or UPRE (unfolded
  protein response element) are regulated transcriptionally by ATF6n (i.e., its active
  N-terminal portion) and XBP-1s, respectively. In addition, a putative micropeptide
  Nrf1u is likely generated by a translation of the upstream open reading frame existing
  in the full-length mRNA transcript of this CNC-bZIP factor, particularly under ER
  stress conditions. Certainly activated or blocked nodes in the some cascades are
  denoted by arrows (→) or bars (⊥), respectively. Scissors represent putative protein
  processing
pmcid: PMC7022656
papertitle: Unification of Opposites between Two Antioxidant Transcription Factors
  Nrf1 and Nrf2 in Mediating Distinct Cellular Responses to the Endoplasmic Reticulum
  Stressor Tunicamycin.
reftext: Yu-ping Zhu, et al. Antioxidants (Basel). 2020 Jan;9(1):4.
pmc_ranked_result_index: '92316'
pathway_score: 0.9398028
filename: antioxidants-09-00004-g001.jpg
figtitle: Inhibition of Nrf1 N-glycosylation by tunicamycin (TU) in the endoplasmic
  reticulum (ER)- to-nuclear signaling responses
year: '2020'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7022656__antioxidants-09-00004-g001.html
  '@type': Dataset
  description: Inhibition of Nrf1 N-glycosylation by tunicamycin (TU) in the endoplasmic
    reticulum (ER)- to-nuclear signaling responses. A topobiological model is herein
    proposed to explain the inhibition of N-glycosylation of Nrf1 by TU, serving as
    a classic ER stressor, which can also block the ensuing deglycosylation and proteolytic
    processing of this cap’n’collar (CNC)-basic-region leucine zipper (bZIP) protein
    to yield a mature transcription factor. The resulting non-glycosylated Nrf1 and
    others are subjected to the ER-associated degradation (ERAD) pathway mediated
    by proteasomes. To the contrary, the non-glycosylated and unfolded proteins are
    accumulated insomuch as to stimulate ER stress. Thereby, inhibition of Nrf1 by
    TU is accompanied by the activation of a canonic ER-to-nuclear response signaling
    mediated by PERK, IRE1 and ATF6, three key transducers tethered to ER membranes
    as reviewed by the authors [,,,]. Of note, PERK was identified as a direct upstream
    kinase of Nrf2, as well as elF2α [], but Nrf2 is neither translocated nor N-glycosylated
    in the lumen of ER. As such, Nrf2 is generally accepted as a master transcription
    factor to regulate cognate genes driven by antioxidant response elements (ARE)
    in the promoter regions. In such responses, ATF4 is selectively translated by
    the elF2α-controlled initiation machinery to regulate target genes (including
    Chop, Ero1, Gadd34) containing amino acid response elements (AARE) in their promoters.
    To get rid of ER stress, ATF6 is activated by its consecutive proteolytic processing
    by Site-1 and Site-2 proteases (i.e., S1P and S2P, respectively) in the Golgi
    apparatus. The full-length mRNA of XBP is alternatively spliced to remove its
    26 nucleotides at the nearly 3′-end, such that the subsequent portion of its open
    reading frame is shifted so much as to yield another inducible protein isoform,
    called XBP-1s, that is longer in size than its original protein of XBP-1u. Consequently,
    distinct subsets of cognate genes containing cis-regulatory ESRE (ER stress response
    element) or UPRE (unfolded protein response element) are regulated transcriptionally
    by ATF6n (i.e., its active N-terminal portion) and XBP-1s, respectively. In addition,
    a putative micropeptide Nrf1u is likely generated by a translation of the upstream
    open reading frame existing in the full-length mRNA transcript of this CNC-bZIP
    factor, particularly under ER stress conditions. Certainly activated or blocked
    nodes in the some cascades are denoted by arrows (→) or bars (⊥), respectively.
    Scissors represent putative protein processing
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - NKRF
  - EIF2AK3
  - NKD1
  - NKD2
  - EIF2A
  - ATF6
  - ATF4
  - NRF1
  - DDIT3
  - PPP1R15A
  - XBP1
  - STT3A
  - ERN1
genes:
- word: Nrf
  symbol: NRF
  source: hgnc_alias_symbol
  hgnc_symbol: NKRF
  entrez: '55922'
- word: PERK
  symbol: PERK
  source: hgnc_alias_symbol
  hgnc_symbol: EIF2AK3
  entrez: '9451'
- word: NKD
  symbol: NKD
  source: bioentities_symbol
  hgnc_symbol: NKD1
  entrez: '85407'
- word: NKD
  symbol: NKD
  source: bioentities_symbol
  hgnc_symbol: NKD2
  entrez: '85409'
- word: elF2a
  symbol: EIF2A
  source: hgnc_symbol
  hgnc_symbol: EIF2A
  entrez: '83939'
- word: ATF6
  symbol: ATF6
  source: hgnc_symbol
  hgnc_symbol: ATF6
  entrez: '22926'
- word: ATF4
  symbol: ATF4
  source: hgnc_symbol
  hgnc_symbol: ATF4
  entrez: '468'
- word: Nrf1
  symbol: NRF1
  source: hgnc_symbol
  hgnc_symbol: NRF1
  entrez: '4899'
- word: Nrf1
  symbol: NRF1
  source: hgnc_symbol
  hgnc_symbol: NRF1
  entrez: '4899'
- word: Chop,Ero1
  symbol: CHOP
  source: hgnc_alias_symbol
  hgnc_symbol: DDIT3
  entrez: '1649'
- word: Chop,
  symbol: CHOP
  source: hgnc_alias_symbol
  hgnc_symbol: DDIT3
  entrez: '1649'
- word: Gadd34
  symbol: GADD34
  source: hgnc_alias_symbol
  hgnc_symbol: PPP1R15A
  entrez: '23645'
- word: XBP1,BIP,Chop
  symbol: CHOP
  source: hgnc_alias_symbol
  hgnc_symbol: DDIT3
  entrez: '1649'
- word: XBP1,BIP,Chop
  symbol: XBP1
  source: hgnc_symbol
  hgnc_symbol: XBP1
  entrez: '7494'
- word: XBP1,
  symbol: XBP1
  source: hgnc_symbol
  hgnc_symbol: XBP1
  entrez: '7494'
- word: Chop
  symbol: CHOP
  source: hgnc_alias_symbol
  hgnc_symbol: DDIT3
  entrez: '1649'
- word: TMc
  symbol: TMC
  source: hgnc_alias_symbol
  hgnc_symbol: STT3A
  entrez: '3703'
- word: TMc
  symbol: TMC
  source: hgnc_alias_symbol
  hgnc_symbol: STT3A
  entrez: '3703'
- word: IRE1
  symbol: IRE1
  source: hgnc_alias_symbol
  hgnc_symbol: ERN1
  entrez: '2081'
- word: NKD
  symbol: NKD
  source: bioentities_symbol
  hgnc_symbol: NKD1
  entrez: '85407'
- word: NKD
  symbol: NKD
  source: bioentities_symbol
  hgnc_symbol: NKD2
  entrez: '85409'
- word: ATF4
  symbol: ATF4
  source: hgnc_symbol
  hgnc_symbol: ATF4
  entrez: '468'
chemicals: []
diseases: []
figid_alias: PMC7022656__F1
redirect_from: /figures/PMC7022656__F1
figtype: Figure
---
