---
figid: PMC7859603__mejdd-12-257-g001
figlink: pmc/articles/PMC7859603/figure/F1/
number: Fig.1
caption: DisGeNET contains a list of diseases associated genes that have been collected
  based on the presence of genetic overlaps between diseases. In this list, among
  the 4,753,986 potential disease associations, 13,064 diseases were found to share
  at least one gene with other diseases. Among these associations, CeD was significantly
  associated (combined score at p< 0.05) with atherosclerosis, multiple sclerosis,
  and hepatitis C, while in IBD, the significant associations were seen with stomach
  neoplasms, osteoporosis, postmenopausal syndrome, and sepsis. The combined score
  is calculated by taking the log of the p values generated by the Fisher exact test
  and multiplying that by the z-score of the deviation from the expected rank. Both
  CeD and IBD, with a partial similar genetic background, target the gut, which is
  usually followed up by chronic inflammation. Since the ubiquity of CeD in patients
  with IBD remains unclear, we, therefore, were interested in mining CeD-IBD correlations
  at the transcriptome level to better understand the underlying reasons for this
  overlap. Only CeD-IBD associated genes retrieved from DisGeNET database with at
  least one evidence from a database of protein families (Pfam) were chosen for further
  analysis. Out of 325 CeD and 880 IBD associated genes, 314 and 851 genes, respectively,
  were selected from which 149 genes were common between CeD and IBD. As mentioned
  earlier, GeneMANIA server was used to establish a common network of co-expressed
  genes and pathways in CeD-IBD by utilizing the common identified 149 genes. Afterward,
  the GeneMANIA network was further clustered into the highly interconnected gene
  clusters by MCODE, which totally resulted in the eight sub-networks (). The MCODE
  scores varied from 2.8-28.8. According to the previous studies, immune response
  associated transcription factors (NFKB1, IRF2, VDR), interleukins (IL10, IL6, IL2,
  and IL2RA), epidermal growth factor receptor (EGFR), and interferon-gamma (IFNG)
  were more interesting. The genes within the identified sub-networks were enriched
  in the inflammatory response (FAS), apoptotic pathway (CD14, IRF1), and signaling
  pathways (EFGR, CCL5, CCR5). In pathway analysis, signaling pathways like inflammation
  mediated by cytokine and chemokines, interleukin, toll receptor, and apoptosis seemed
  to be more biologically associated with CeD and IBD (). We additionally investigated
  the 149 common genes to discover disease-associated motifs and master regulators
  by iRegulon and TRRUST (). Among the eight identified sub-networks except for sub-network
  1 in which IRF2 was found as the most probable regulator and NFKB1 placed as the
  second regulator, NFKB1 was found to be the global regulator of 149 common genes
  in other sub-networks. We also examined if the gene-transcription factor links from
  iRegulon were also present as experimentally validated links in TRRUST. As mentioned
  in the method section, finally, the differential expressed genes from these 149
  common genes were determined using Gemma server. Only 39 genes were detected as
  differentially expressed in CeD-oriented experiments in Gemma in different cell
  types () while all of the 149 genes were identified as differentially expressed
  in IBD experiments under different time points, genetic modifications, phenotypes,
  cell types, and medical procedures. The expression of these genes did not change
  in different diets and clinical treatments of patients with IBD ().
pmcid: PMC7859603
papertitle: Network Analysis of Common Genes and Transcriptional Factors between Celiac
  Disease and Inflammatory Bowel Diseases.
reftext: Fereshteh Izadi, et al. Middle East J Dig Dis. 2020 Oct;12(4):257-264.
pmc_ranked_result_index: '122266'
pathway_score: 0.6656182
filename: mejdd-12-257-g001.jpg
figtitle: DisGeNET contains a list of diseases associated genes that have been collected
  based on the presence of genetic overlaps between diseases
year: '2020'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7859603__mejdd-12-257-g001.html
  '@type': Dataset
  description: DisGeNET contains a list of diseases associated genes that have been
    collected based on the presence of genetic overlaps between diseases. In this
    list, among the 4,753,986 potential disease associations, 13,064 diseases were
    found to share at least one gene with other diseases. Among these associations,
    CeD was significantly associated (combined score at p< 0.05) with atherosclerosis,
    multiple sclerosis, and hepatitis C, while in IBD, the significant associations
    were seen with stomach neoplasms, osteoporosis, postmenopausal syndrome, and sepsis.
    The combined score is calculated by taking the log of the p values generated by
    the Fisher exact test and multiplying that by the z-score of the deviation from
    the expected rank. Both CeD and IBD, with a partial similar genetic background,
    target the gut, which is usually followed up by chronic inflammation. Since the
    ubiquity of CeD in patients with IBD remains unclear, we, therefore, were interested
    in mining CeD-IBD correlations at the transcriptome level to better understand
    the underlying reasons for this overlap. Only CeD-IBD associated genes retrieved
    from DisGeNET database with at least one evidence from a database of protein families
    (Pfam) were chosen for further analysis. Out of 325 CeD and 880 IBD associated
    genes, 314 and 851 genes, respectively, were selected from which 149 genes were
    common between CeD and IBD. As mentioned earlier, GeneMANIA server was used to
    establish a common network of co-expressed genes and pathways in CeD-IBD by utilizing
    the common identified 149 genes. Afterward, the GeneMANIA network was further
    clustered into the highly interconnected gene clusters by MCODE, which totally
    resulted in the eight sub-networks (). The MCODE scores varied from 2.8-28.8.
    According to the previous studies, immune response associated transcription factors
    (NFKB1, IRF2, VDR), interleukins (IL10, IL6, IL2, and IL2RA), epidermal growth
    factor receptor (EGFR), and interferon-gamma (IFNG) were more interesting. The
    genes within the identified sub-networks were enriched in the inflammatory response
    (FAS), apoptotic pathway (CD14, IRF1), and signaling pathways (EFGR, CCL5, CCR5).
    In pathway analysis, signaling pathways like inflammation mediated by cytokine
    and chemokines, interleukin, toll receptor, and apoptosis seemed to be more biologically
    associated with CeD and IBD (). We additionally investigated the 149 common genes
    to discover disease-associated motifs and master regulators by iRegulon and TRRUST
    (). Among the eight identified sub-networks except for sub-network 1 in which
    IRF2 was found as the most probable regulator and NFKB1 placed as the second regulator,
    NFKB1 was found to be the global regulator of 149 common genes in other sub-networks.
    We also examined if the gene-transcription factor links from iRegulon were also
    present as experimentally validated links in TRRUST. As mentioned in the method
    section, finally, the differential expressed genes from these 149 common genes
    were determined using Gemma server. Only 39 genes were detected as differentially
    expressed in CeD-oriented experiments in Gemma in different cell types () while
    all of the 149 genes were identified as differentially expressed in IBD experiments
    under different time points, genetic modifications, phenotypes, cell types, and
    medical procedures. The expression of these genes did not change in different
    diets and clinical treatments of patients with IBD ().
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MMP3
  - IL3
  - GATA4
  - GATA5
  - GATA1
  - GATA2
  - GATA3
  - GATA6
  - CCL2
  - TGFBI
  - CD14
  - FAS
  - SGPL1
  - IL10
  - CTGF
  - IL6
  - IL2
  - RELA
  - NFKBIA
  - JUN
  - VDR
  - BCL2
  - TBX2
  - STAT3
  - TNF
  - CEBPB
  - TG
  - EGFR
  - IFNG
  - HLA-G
  - PPARG
  - IL2RA
  - APOB
  - IL17A
  - MYB
  - TNFRSF9
  - ATF2
  - CREB5
  - CREB3
  - CREB3L3
  - ATF1
  - CREB1
  - CREM
  - CREB3L2
  - CREB3L1
  - CREB3L4
  - SELE
  - NOS2
  - HLA-DRA
  - ABCB1
  - CIITA
  - FOXP3
  - APC
  - NLRP3
  - TBX2
  - APC
genes:
- word: (MMP3
  symbol: MMP3
  source: hgnc_symbol
  hgnc_symbol: MMP3
  entrez: '4314'
- word: IL3
  symbol: IL3
  source: hgnc_symbol
  hgnc_symbol: IL3
  entrez: '3562'
- word: GATA
  symbol: GATA
  source: bioentities_symbol
  hgnc_symbol: GATA4
  entrez: '2626'
- word: GATA
  symbol: GATA
  source: bioentities_symbol
  hgnc_symbol: GATA5
  entrez: '140628'
- word: GATA
  symbol: GATA
  source: bioentities_symbol
  hgnc_symbol: GATA1
  entrez: '2623'
- word: GATA
  symbol: GATA
  source: bioentities_symbol
  hgnc_symbol: GATA2
  entrez: '2624'
- word: GATA
  symbol: GATA
  source: bioentities_symbol
  hgnc_symbol: GATA3
  entrez: '2625'
- word: GATA
  symbol: GATA
  source: bioentities_symbol
  hgnc_symbol: GATA6
  entrez: '2627'
- word: CCL2
  symbol: CCL2
  source: hgnc_symbol
  hgnc_symbol: CCL2
  entrez: '6347'
- word: (TGFBI
  symbol: TGFBI
  source: hgnc_symbol
  hgnc_symbol: TGFBI
  entrez: '7045'
- word: (CD14
  symbol: CD14
  source: hgnc_symbol
  hgnc_symbol: CD14
  entrez: '929'
- word: FAS
  symbol: FAS
  source: hgnc_symbol
  hgnc_symbol: FAS
  entrez: '355'
- word: SPI
  symbol: SPL
  source: hgnc_alias_symbol
  hgnc_symbol: SGPL1
  entrez: '8879'
- word: IL10
  symbol: IL10
  source: hgnc_symbol
  hgnc_symbol: IL10
  entrez: '3586'
- word: CTGF
  symbol: CTGF
  source: hgnc_symbol
  hgnc_symbol: CTGF
  entrez: '1490'
- word: IL6
  symbol: IL6
  source: hgnc_symbol
  hgnc_symbol: IL6
  entrez: '3569'
- word: IL2
  symbol: IL2
  source: hgnc_symbol
  hgnc_symbol: IL2
  entrez: '3558'
- word: RELA
  symbol: RELA
  source: hgnc_symbol
  hgnc_symbol: RELA
  entrez: '5970'
- word: NFKBI
  symbol: NFKBI
  source: hgnc_prev_symbol
  hgnc_symbol: NFKBIA
  entrez: '4792'
- word: JUN
  symbol: JUN
  source: hgnc_symbol
  hgnc_symbol: JUN
  entrez: '3725'
- word: VDR
  symbol: VDR
  source: hgnc_symbol
  hgnc_symbol: VDR
  entrez: '7421'
- word: BCL2
  symbol: BCL2
  source: hgnc_symbol
  hgnc_symbol: BCL2
  entrez: '596'
- word: (TBX2)]
  symbol: TBX2
  source: hgnc_symbol
  hgnc_symbol: TBX2
  entrez: '6909'
- word: STAT3
  symbol: STAT3
  source: hgnc_symbol
  hgnc_symbol: STAT3
  entrez: '6774'
- word: TNF
  symbol: TNF
  source: hgnc_symbol
  hgnc_symbol: TNF
  entrez: '7124'
- word: CEBPB
  symbol: CEBPB
  source: hgnc_symbol
  hgnc_symbol: CEBPB
  entrez: '1051'
- word: NFKBIA
  symbol: NFKBIA
  source: hgnc_symbol
  hgnc_symbol: NFKBIA
  entrez: '4792'
- word: TG
  symbol: TG
  source: hgnc_symbol
  hgnc_symbol: TG
  entrez: '7038'
- word: EGFR
  symbol: EGFR
  source: hgnc_symbol
  hgnc_symbol: EGFR
  entrez: '1956'
- word: IFNG
  symbol: IFNG
  source: hgnc_symbol
  hgnc_symbol: IFNG
  entrez: '3458'
- word: HLA-G
  symbol: HLA-G
  source: hgnc_symbol
  hgnc_symbol: HLA-G
  entrez: '3135'
- word: PPARG
  symbol: PPARG
  source: hgnc_symbol
  hgnc_symbol: PPARG
  entrez: '5468'
- word: IL2RA
  symbol: IL2RA
  source: hgnc_symbol
  hgnc_symbol: IL2RA
  entrez: '3559'
- word: APOB
  symbol: APOB
  source: hgnc_symbol
  hgnc_symbol: APOB
  entrez: '338'
- word: (IL17A)
  symbol: IL17A
  source: hgnc_symbol
  hgnc_symbol: IL17A
  entrez: '3605'
- word: MYB
  symbol: MYB
  source: hgnc_symbol
  hgnc_symbol: MYB
  entrez: '4602'
- word: ILA
  symbol: ILA
  source: hgnc_prev_symbol
  hgnc_symbol: TNFRSF9
  entrez: '3604'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: ATF2
  entrez: '1386'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB5
  entrez: '9586'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3
  entrez: '10488'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L3
  entrez: '84699'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: ATF1
  entrez: '466'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB1
  entrez: '1385'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREM
  entrez: '1390'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L2
  entrez: '64764'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L1
  entrez: '90993'
- word: CREB
  symbol: CREB
  source: bioentities_symbol
  hgnc_symbol: CREB3L4
  entrez: '148327'
- word: SELE
  symbol: SELE
  source: hgnc_symbol
  hgnc_symbol: SELE
  entrez: '6401'
- word: NOS2
  symbol: NOS2
  source: hgnc_symbol
  hgnc_symbol: NOS2
  entrez: '4843'
- word: HLA-DRA
  symbol: HLA-DRA
  source: hgnc_symbol
  hgnc_symbol: HLA-DRA
  entrez: '3122'
- word: ABCB1
  symbol: ABCB1
  source: hgnc_symbol
  hgnc_symbol: ABCB1
  entrez: '5243'
- word: CIITA
  symbol: CIITA
  source: hgnc_symbol
  hgnc_symbol: CIITA
  entrez: '4261'
- word: FOXP3
  symbol: FOXP3
  source: hgnc_symbol
  hgnc_symbol: FOXP3
  entrez: '50943'
- word: APC
  symbol: APC
  source: hgnc_symbol
  hgnc_symbol: APC
  entrez: '324'
- word: NLRP3
  symbol: NLRP3
  source: hgnc_symbol
  hgnc_symbol: NLRP3
  entrez: '114548'
chemicals:
- word: TBX2
  source: MESH
  identifier: C091856
diseases:
- word: APC
  source: MESH
  identifier: D011125
---
