---
figid: PMC7758401__fpls-11-595603-g001
figlink: pmc/articles/PMC7758401/figure/F1/
number: F1
caption: 'The establishment, maintenance and erasing of 5mC in plants. (A) The diverse
  small RNAs-generating pathways involved in RNA-directed DNA methylation (RdDM).
  The small RNAs-generating pathways in the canonical Pol IV -RDR2-DCL3-dependent
  RdDM pathway, Pol II-DCL3 RdDM pathway, RDR6 RdDM pathway, RDR6-DCL3 RdDM pathway,
  Pol IV-NERD RdDM pathway, and dicer-independent RdDM, are briefly presented, respectively
  (). (B) The establishment of 5mC in plants. 21–24-nt siRNAs or miRNAs, which are
  loaded onto AGO2 or AGO4/6, base-pair with Pol V-generated nascent scaffold transcript
  of target loci, resulting in the DRM2 recruitment and DNA methylation at the source
  loci with the aid of RNA binding proteins, methylated DNA binding proteins, chromatin-remodeling
  complex and histone modification-associated proteins (; ). (C) The maintenance of
  5mC in different contexts. CG, CHG and asymmetric CHH methylation are maintained
  by MET1, CMT3/CMT2, DRM2/CMT2, respectively. It is needed to note that CMT3 and
  SUVH4/5/6 form a self-reinforcing feedback loop between mCHG and H3K9me (; ). (D)
  The erasing of 5mC through active and passive demethylation pathway. In the passive
  demethylation, 5mC is diluted during DNA replication. In the active demethylation
  pathway, four bifunctional 5mC DNA glycosylases-apurinic/apyrimidinic lyases, DME,
  ROS1 and its homologs DML2 and DML3, catalyze the active removal of 5-methylcytosine
  from all sequence contexts through the base excision repair pathway (). The proteins
  are represented in circles. The regulatory pathways are indicated with solid arrows.
  Abbreviations: mC, methylated cytosine; Pol IV, Polymerase IV; RDR, RNA-DEPENDENT
  RNA POLYMERASE; DCL, DICER-LIKE; NERD, NEEDED FOR RDR2-INDEPENDENT DNA METHYLATION;
  AGO, ARGONAUTE; MET1, METHYLTRANSFERASE 1; CMT, CHROMOMETHYLASE; DRM2, DOMAINS REARRANGED
  METHYLTRANSFERASE 2; SUVH4/5/6, SU(VAR)3-9 HOMOLOGUE; DME, DEMETER; ROS1, REPRESSOR
  OF SILENCING 1; DML, DME-LIKE 2.'
pmcid: PMC7758401
papertitle: Small DNA Methylation, Big Player in Plant Abiotic Stress Responses and
  Memory.
reftext: Junzhong Liu, et al. Front Plant Sci. 2020;11:595603.
pmc_ranked_result_index: '42432'
pathway_score: 0.9059808
filename: fpls-11-595603-g001.jpg
figtitle: Establishment, maintenance and erasing of 5mC in plants
year: '2020'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7758401__fpls-11-595603-g001.html
  '@type': Dataset
  description: 'The establishment, maintenance and erasing of 5mC in plants. (A) The
    diverse small RNAs-generating pathways involved in RNA-directed DNA methylation
    (RdDM). The small RNAs-generating pathways in the canonical Pol IV -RDR2-DCL3-dependent
    RdDM pathway, Pol II-DCL3 RdDM pathway, RDR6 RdDM pathway, RDR6-DCL3 RdDM pathway,
    Pol IV-NERD RdDM pathway, and dicer-independent RdDM, are briefly presented, respectively
    (). (B) The establishment of 5mC in plants. 21–24-nt siRNAs or miRNAs, which are
    loaded onto AGO2 or AGO4/6, base-pair with Pol V-generated nascent scaffold transcript
    of target loci, resulting in the DRM2 recruitment and DNA methylation at the source
    loci with the aid of RNA binding proteins, methylated DNA binding proteins, chromatin-remodeling
    complex and histone modification-associated proteins (; ). (C) The maintenance
    of 5mC in different contexts. CG, CHG and asymmetric CHH methylation are maintained
    by MET1, CMT3/CMT2, DRM2/CMT2, respectively. It is needed to note that CMT3 and
    SUVH4/5/6 form a self-reinforcing feedback loop between mCHG and H3K9me (; ).
    (D) The erasing of 5mC through active and passive demethylation pathway. In the
    passive demethylation, 5mC is diluted during DNA replication. In the active demethylation
    pathway, four bifunctional 5mC DNA glycosylases-apurinic/apyrimidinic lyases,
    DME, ROS1 and its homologs DML2 and DML3, catalyze the active removal of 5-methylcytosine
    from all sequence contexts through the base excision repair pathway (). The proteins
    are represented in circles. The regulatory pathways are indicated with solid arrows.
    Abbreviations: mC, methylated cytosine; Pol IV, Polymerase IV; RDR, RNA-DEPENDENT
    RNA POLYMERASE; DCL, DICER-LIKE; NERD, NEEDED FOR RDR2-INDEPENDENT DNA METHYLATION;
    AGO, ARGONAUTE; MET1, METHYLTRANSFERASE 1; CMT, CHROMOMETHYLASE; DRM2, DOMAINS
    REARRANGED METHYLTRANSFERASE 2; SUVH4/5/6, SU(VAR)3-9 HOMOLOGUE; DME, DEMETER;
    ROS1, REPRESSOR OF SILENCING 1; DML, DME-LIKE 2.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - DICER1
  - TES
  - AGO4
  - AGO2
  - MAD2L1BP
  - ATP5F1A
  - METTL2B
genes:
- word: Dicer-independent
  symbol: Dicer
  source: hgnc_alias_symbol
  hgnc_symbol: DICER1
  entrez: '23405'
- word: TEs
  symbol: TES
  source: hgnc_symbol
  hgnc_symbol: TES
  entrez: '26136'
- word: TES
  symbol: TES
  source: hgnc_symbol
  hgnc_symbol: TES
  entrez: '26136'
- word: TEs
  symbol: TES
  source: hgnc_symbol
  hgnc_symbol: TES
  entrez: '26136'
- word: AGO4
  symbol: AGO4
  source: hgnc_symbol
  hgnc_symbol: AGO4
  entrez: '192670'
- word: AGO2
  symbol: AGO2
  source: hgnc_symbol
  hgnc_symbol: AGO2
  entrez: '27161'
- word: AGO4
  symbol: AGO4
  source: hgnc_symbol
  hgnc_symbol: AGO4
  entrez: '192670'
- word: AGO2
  symbol: AGO2
  source: hgnc_symbol
  hgnc_symbol: AGO2
  entrez: '27161'
- word: CMT2
  symbol: CMT2
  source: hgnc_alias_symbol
  hgnc_symbol: MAD2L1BP
  entrez: '9587'
- word: ORM
  symbol: ORM
  source: hgnc_alias_symbol
  hgnc_symbol: ATP5F1A
  entrez: '498'
- word: METI
  symbol: METL
  source: hgnc_alias_symbol
  hgnc_symbol: METTL2B
  entrez: '55798'
- word: CMT2
  symbol: CMT2
  source: hgnc_alias_symbol
  hgnc_symbol: MAD2L1BP
  entrez: '9587'
chemicals: []
diseases: []
figid_alias: PMC7758401__F1
redirect_from: /figures/PMC7758401__F1
figtype: Figure
---
