---
figid: PMC7963035__nihms-1678290-f0001
figlink: pmc/articles/PMC7963035/figure/F1/
number: Fig. 1 |
caption: a | Nicotinamide adenine dinucleotide (NAD+) biosynthetic pathways. NAD+
  levels are maintained by three independent biosynthetic pathways. The kynurenine
  pathway (or de novo synthesis pathway) uses the dietary amino acid tryptophan to
  generate NAD+. Tryptophan enters the cell via the transporters SLC7A5 and SLC36A4.
  Within the cell, tryptophan is converted to N-formylkynurenine by the rate-limiting
  enzyme indoleamine 2,3-dioxygenase (IDO) or the rate-limiting enzyme tryptophan
  2,3-dioxygenase (TDO). N-Formylkynurenine is transformed into l-kynurenine, which
  is further converted to 3-hydroxykynurenine (3-HK) by kynurenine 3-monooxygenase
  (KMO) and to 3-hydroxyanthranilic acid (3-HAA) by tryptophan 2,3-dioxygenase (KYNU).
  The next step is performed by 3-hydroxyanthranilic acid oxygenase (3HAO) to generate
  α-amino-β-carboxymuconate ε-semialdehyde (ACMS). This compound can spontaneously
  condense and rearrange into quinolinic acid, which is transformed by quinolinate
  phosphoribosyltransferase (QPRT) into nicotinamide mononucleotide (NAMN), at which
  point it converges with the Preiss–Handler pathway. The Preiss–Handler pathway uses
  dietary nicotinic acid (NA), which enters the cell via SLC5A8 or SLC22A13 transporters,
  and the enzyme nicotinic acid phosphoribosyltransferase (NAPRT) to generate NAMN,
  which is then transformed into nicotinic acid adenine dinucleotide (NAAD) by nicotinamide
  mononucleotide adenylyltransferases (NMNAT1, NMNAT2 and NMNAT3). The process is
  completed by the transformation of NAAD into NAD+ by NAD+ synthetase (NADS). The
  NAD+ salvage pathway recycles the nicotinamide (NAM) generated as a by-product of
  the enzymatic activities of NAD+-consuming enzymes (sirtuins, poly(ADP-ribose) polymerases
  (PARPs) and the NAD+ glycohydrolase and cyclic ADP-ribose synthases CD38, CD157
  and SARM1). Initially, the intracellular nicotinamide phosphoribosyltransferase
  (iNAMPT) recycles NAM into nicotinamide mononucleotide (NMN), which is then converted
  into NAD+ via the different NMNATs. NAM can be alternatively methylated by the enzyme
  nicotinamide N-methyltransferase (NNMT) and secreted via the urine. In the extracellular
  space, NAM is generated as a by-product of the ectoenzymes CD38 and CD157 and can
  be converted to NMN by extracellular NAMPT (eNAMPT). NMN is then dephosphorylated
  by CD73 to nicotinamide riboside (NR), which is transported into the cell via an
  unknown nucleoside transporter (question mark). NMN can be imported into the cell
  via an NMN-specific transporter (SLC12A8 in the small intestine). Intracellularly,
  NR forms NMN via nicotinamide riboside kinases 1 and 2 (NRK1 and NRK2). NMN is then
  converted to NAD+ by NMNAT1, NMNAT2 and NMNAT3. b | NAD+ metabolism in different
  subcellular compartments. The NAD+ homeostasis is a balance of synthesis, consumption
  and regeneration in different subcellular compartments, which are regulated by subcellular-specific
  NAD+-consuming enzymes, subcellular transporters and redox reactions. NAD+ precursors
  enter the cell via the three biosynthetic pathways (part a). In the cytoplasm, NAM
  is converted to NMN by intracellular NAMPT (iNAMPT). NMN is then converted to NAD+
  by NMNAT2, which is the cytosol-specific isoform of this enzyme. NAD+ is utilized
  during glycolysis, generating NADH, which is transferred to the mitochondrial matrix
  via the malate/aspartate shuttle and the glyceraldehyde 3-phosphate shuttle. The
  mitochondrial NADH imported via the malate/aspartate shuttle is oxidized by complex
  I in the electron transport chain (ETC), whereas the resulting reduced flavin adenine
  dinucleotide (FADH2) from the glyceraldehyde 3-phosphate shuttle is oxidized by
  complex II. Recently the mammalian NAD+ mitochondrial transporter SLC25A51 was identified,
  and it has been demonstrated to be responsible for intact NAD+ uptake in the organelle.
  The salvage pathway for NAD+ in mitochondria has not been fully resolved, but the
  role of a specific NMNAT isoform (NMNAT3) has been proposed. In the mitochondria,
  NAD+ is consumed by NAD+-dependent mitochondrial SIRT3–SIRT5, generating NAM. It
  is still unknown whether NAM can be converted back to NMN or can be converted to
  NAD+ within the mitochondrion, or whether other precursors can be transported through
  the mitochondrial membrane to fuel NAD+synthesis. The nuclear NAD+ pool probably
  equilibrates with the cytosolic one by diffusion through the nuclear pore; however,
  the full dynamics are still largely unexplored. A nuclear-specific NMNAT isoform
  (NMNAT1) has been described and is part of the nuclear NAM salvage NAD+pathway.
  MNAM, N1-methylnicotinamide; TCA, tricarboxylic acid.
pmcid: PMC7963035
papertitle: NAD+ metabolism and its roles in cellular processes during ageing.
reftext: Anthony J. Covarrubias, et al. Nat Rev Mol Cell Biol. ;22(2):119-141.
pmc_ranked_result_index: '76527'
pathway_score: 0.9510007
filename: nihms-1678290-f0001.jpg
figtitle: NAD+ metabolism and its roles in cellular processes during ageing
year: ''
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7963035__nihms-1678290-f0001.html
  '@type': Dataset
  description: a | Nicotinamide adenine dinucleotide (NAD+) biosynthetic pathways.
    NAD+ levels are maintained by three independent biosynthetic pathways. The kynurenine
    pathway (or de novo synthesis pathway) uses the dietary amino acid tryptophan
    to generate NAD+. Tryptophan enters the cell via the transporters SLC7A5 and SLC36A4.
    Within the cell, tryptophan is converted to N-formylkynurenine by the rate-limiting
    enzyme indoleamine 2,3-dioxygenase (IDO) or the rate-limiting enzyme tryptophan
    2,3-dioxygenase (TDO). N-Formylkynurenine is transformed into l-kynurenine, which
    is further converted to 3-hydroxykynurenine (3-HK) by kynurenine 3-monooxygenase
    (KMO) and to 3-hydroxyanthranilic acid (3-HAA) by tryptophan 2,3-dioxygenase (KYNU).
    The next step is performed by 3-hydroxyanthranilic acid oxygenase (3HAO) to generate
    α-amino-β-carboxymuconate ε-semialdehyde (ACMS). This compound can spontaneously
    condense and rearrange into quinolinic acid, which is transformed by quinolinate
    phosphoribosyltransferase (QPRT) into nicotinamide mononucleotide (NAMN), at which
    point it converges with the Preiss–Handler pathway. The Preiss–Handler pathway
    uses dietary nicotinic acid (NA), which enters the cell via SLC5A8 or SLC22A13
    transporters, and the enzyme nicotinic acid phosphoribosyltransferase (NAPRT)
    to generate NAMN, which is then transformed into nicotinic acid adenine dinucleotide
    (NAAD) by nicotinamide mononucleotide adenylyltransferases (NMNAT1, NMNAT2 and
    NMNAT3). The process is completed by the transformation of NAAD into NAD+ by NAD+
    synthetase (NADS). The NAD+ salvage pathway recycles the nicotinamide (NAM) generated
    as a by-product of the enzymatic activities of NAD+-consuming enzymes (sirtuins,
    poly(ADP-ribose) polymerases (PARPs) and the NAD+ glycohydrolase and cyclic ADP-ribose
    synthases CD38, CD157 and SARM1). Initially, the intracellular nicotinamide phosphoribosyltransferase
    (iNAMPT) recycles NAM into nicotinamide mononucleotide (NMN), which is then converted
    into NAD+ via the different NMNATs. NAM can be alternatively methylated by the
    enzyme nicotinamide N-methyltransferase (NNMT) and secreted via the urine. In
    the extracellular space, NAM is generated as a by-product of the ectoenzymes CD38
    and CD157 and can be converted to NMN by extracellular NAMPT (eNAMPT). NMN is
    then dephosphorylated by CD73 to nicotinamide riboside (NR), which is transported
    into the cell via an unknown nucleoside transporter (question mark). NMN can be
    imported into the cell via an NMN-specific transporter (SLC12A8 in the small intestine).
    Intracellularly, NR forms NMN via nicotinamide riboside kinases 1 and 2 (NRK1
    and NRK2). NMN is then converted to NAD+ by NMNAT1, NMNAT2 and NMNAT3. b | NAD+
    metabolism in different subcellular compartments. The NAD+ homeostasis is a balance
    of synthesis, consumption and regeneration in different subcellular compartments,
    which are regulated by subcellular-specific NAD+-consuming enzymes, subcellular
    transporters and redox reactions. NAD+ precursors enter the cell via the three
    biosynthetic pathways (part a). In the cytoplasm, NAM is converted to NMN by intracellular
    NAMPT (iNAMPT). NMN is then converted to NAD+ by NMNAT2, which is the cytosol-specific
    isoform of this enzyme. NAD+ is utilized during glycolysis, generating NADH, which
    is transferred to the mitochondrial matrix via the malate/aspartate shuttle and
    the glyceraldehyde 3-phosphate shuttle. The mitochondrial NADH imported via the
    malate/aspartate shuttle is oxidized by complex I in the electron transport chain
    (ETC), whereas the resulting reduced flavin adenine dinucleotide (FADH2) from
    the glyceraldehyde 3-phosphate shuttle is oxidized by complex II. Recently the
    mammalian NAD+ mitochondrial transporter SLC25A51 was identified, and it has been
    demonstrated to be responsible for intact NAD+ uptake in the organelle. The salvage
    pathway for NAD+ in mitochondria has not been fully resolved, but the role of
    a specific NMNAT isoform (NMNAT3) has been proposed. In the mitochondria, NAD+
    is consumed by NAD+-dependent mitochondrial SIRT3–SIRT5, generating NAM. It is
    still unknown whether NAM can be converted back to NMN or can be converted to
    NAD+ within the mitochondrion, or whether other precursors can be transported
    through the mitochondrial membrane to fuel NAD+synthesis. The nuclear NAD+ pool
    probably equilibrates with the cytosolic one by diffusion through the nuclear
    pore; however, the full dynamics are still largely unexplored. A nuclear-specific
    NMNAT isoform (NMNAT1) has been described and is part of the nuclear NAM salvage
    NAD+pathway. MNAM, N1-methylnicotinamide; TCA, tricarboxylic acid.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SLC5A8
  - SLC22A13
  - NT5E
  - BST1
  - CD38
  - SLC12A8
  - SLC7A5
  - SLC36A4
  - NAPRT
  - NMRK1
  - NMNAT3
  - NMNAT1
  - NMNAT2
  - KMO
  - KYNU
  - NADK
  - QPRT
  - SLC25A51
  - SIRT2
  - SARM1
  - SIRT1
  - SIRT6
  - SIRT7
  - SIRT3
  - SIRT4
  - SIRT5
  - NAMPT
  - Kynurenine
  - NAD
  - Tryptophan
  - N-Formylkynurenine
  - NAMN
  - NADS
  - KYNU
  - NADH S NAD
  - 3-NAA
  - 3-NAO
  - NADK
  - NADP
  - Quinolinic acid
  - NADPH
  - NADH
  - Malate
  - aspartate
  - TSA
  - FADH
  - Glyceraldehyde
  - phosphate
  - NAMPT
genes:
- word: SLC5A8
  symbol: SLC5A8
  source: hgnc_symbol
  hgnc_symbol: SLC5A8
  entrez: '160728'
- word: SLC22A13-
  symbol: SLC22A13
  source: hgnc_symbol
  hgnc_symbol: SLC22A13
  entrez: '9390'
- word: CD73
  symbol: CD73
  source: hgnc_alias_symbol
  hgnc_symbol: NT5E
  entrez: '4907'
- word: CD157
  symbol: CD157
  source: hgnc_alias_symbol
  hgnc_symbol: BST1
  entrez: '683'
- word: CD38
  symbol: CD38
  source: hgnc_symbol
  hgnc_symbol: CD38
  entrez: '952'
- word: SLC12A8
  symbol: SLC12A8
  source: hgnc_symbol
  hgnc_symbol: SLC12A8
  entrez: '84561'
- word: SLC7A5
  symbol: SLC7A5
  source: hgnc_symbol
  hgnc_symbol: SLC7A5
  entrez: '8140'
- word: SLC36A4
  symbol: SLC36A4
  source: hgnc_symbol
  hgnc_symbol: SLC36A4
  entrez: '120103'
- word: NAPRT
  symbol: NAPRT
  source: hgnc_symbol
  hgnc_symbol: NAPRT
  entrez: '93100'
- word: NRK1
  symbol: NRK1
  source: hgnc_alias_symbol
  hgnc_symbol: NMRK1
  entrez: '54981'
- word: NMNAT1–3
  symbol: NMNAT3
  source: hgnc_symbol
  hgnc_symbol: NMNAT3
  entrez: '349565'
- word: NMNAT1–3
  symbol: NMNAT1
  source: hgnc_symbol
  hgnc_symbol: NMNAT1
  entrez: '64802'
- word: NMNAT1–3
  symbol: NMNAT2
  source: hgnc_symbol
  hgnc_symbol: NMNAT2
  entrez: '23057'
- word: КМО
  symbol: KMO
  source: hgnc_symbol
  hgnc_symbol: KMO
  entrez: '8564'
- word: NMNAT1-3
  symbol: NMNAT3
  source: hgnc_symbol
  hgnc_symbol: NMNAT3
  entrez: '349565'
- word: NMNAT1-3
  symbol: NMNAT1
  source: hgnc_symbol
  hgnc_symbol: NMNAT1
  entrez: '64802'
- word: NMNAT1-3
  symbol: NMNAT2
  source: hgnc_symbol
  hgnc_symbol: NMNAT2
  entrez: '23057'
- word: ΚYNU
  symbol: KYNU
  source: hgnc_symbol
  hgnc_symbol: KYNU
  entrez: '8942'
- word: NADK
  symbol: NADK
  source: hgnc_symbol
  hgnc_symbol: NADK
  entrez: '65220'
- word: QPRT
  symbol: QPRT
  source: hgnc_symbol
  hgnc_symbol: QPRT
  entrez: '23475'
- word: SLC25A51
  symbol: SLC25A51
  source: hgnc_symbol
  hgnc_symbol: SLC25A51
  entrez: '92014'
- word: SIRT2
  symbol: SIRT2
  source: hgnc_symbol
  hgnc_symbol: SIRT2
  entrez: '22933'
- word: CD38
  symbol: CD38
  source: hgnc_symbol
  hgnc_symbol: CD38
  entrez: '952'
- word: •CD38
  symbol: CD38
  source: hgnc_symbol
  hgnc_symbol: CD38
  entrez: '952'
- word: •SARM1?
  symbol: SARM1
  source: hgnc_symbol
  hgnc_symbol: SARM1
  entrez: '23098'
- word: SARM1?
  symbol: SARM1
  source: hgnc_symbol
  hgnc_symbol: SARM1
  entrez: '23098'
- word: SIRT1
  symbol: SIRT1
  source: hgnc_symbol
  hgnc_symbol: SIRT1
  entrez: '23411'
- word: SIRT6
  symbol: SIRT6
  source: hgnc_symbol
  hgnc_symbol: SIRT6
  entrez: '51548'
- word: •SIRT7
  symbol: SIRT7
  source: hgnc_symbol
  hgnc_symbol: SIRT7
  entrez: '51547'
- word: SIRT7
  symbol: SIRT7
  source: hgnc_symbol
  hgnc_symbol: SIRT7
  entrez: '51547'
- word: CD38?
  symbol: CD38
  source: hgnc_symbol
  hgnc_symbol: CD38
  entrez: '952'
- word: •CD38?
  symbol: CD38
  source: hgnc_symbol
  hgnc_symbol: CD38
  entrez: '952'
- word: SARM1?
  symbol: SARM1
  source: hgnc_symbol
  hgnc_symbol: SARM1
  entrez: '23098'
- word: •SIRT3
  symbol: SIRT3
  source: hgnc_symbol
  hgnc_symbol: SIRT3
  entrez: '23410'
- word: SIRT3
  symbol: SIRT3
  source: hgnc_symbol
  hgnc_symbol: SIRT3
  entrez: '23410'
- word: SIRT4
  symbol: SIRT4
  source: hgnc_symbol
  hgnc_symbol: SIRT4
  entrez: '23409'
- word: SIRT5
  symbol: SIRT5
  source: hgnc_symbol
  hgnc_symbol: SIRT5
  entrez: '23408'
- word: NMNAT3?
  symbol: NMNAT3
  source: hgnc_symbol
  hgnc_symbol: NMNAT3
  entrez: '349565'
- word: NMNAT2
  symbol: NMNAT2
  source: hgnc_symbol
  hgnc_symbol: NMNAT2
  entrez: '23057'
- word: NMNAT1
  symbol: NMNAT1
  source: hgnc_symbol
  hgnc_symbol: NMNAT1
  entrez: '64802'
- word: NAMPT?
  symbol: NAMPT
  source: hgnc_symbol
  hgnc_symbol: NAMPT
  entrez: '10135'
chemicals:
- word: Kynurenine
  source: MESH
  identifier: C029366
- word: NAD
  source: MESH
  identifier: D009243
- word: Tryptophan
  source: ''
  identifier: ''
- word: N-Formylkynurenine
  source: MESH
  identifier: D009584
- word: NAMN
  source: ''
  identifier: ''
- word: NADS
  source: MESH
  identifier: C055737
- word: KYNU
  source: ''
  identifier: ''
- word: NADH S NAD
  source: MESH
  identifier: C041808
- word: 3-NAA
  source: MESH
  identifier: D009536
- word: 3-NAO
  source: MESH
  identifier: C041691
- word: NADK
  source: ''
  identifier: ''
- word: NADP
  source: MESH
  identifier: D009249
- word: Quinolinic acid
  source: MESH
  identifier: D017378
- word: NADPH
  source: MESH
  identifier: D009249
- word: NADH
  source: MESH
  identifier: D009243
- word: Malate
  source: MESH
  identifier: D008293
- word: aspartate
  source: ''
  identifier: ''
- word: TSA
  source: MESH
  identifier: C070842
- word: FADH
  source: MESH
  identifier: C058805
- word: Glyceraldehyde
  source: MESH
  identifier: D005985
- word: phosphate
  source: MESH
  identifier: D010710
- word: NAMPT
  source: MESH
  identifier: C085455
diseases: []
---
