---
figid: PMC7942223__nihms-1675777-f0002
figlink: pmc/articles/PMC7942223/figure/F2/
number: F2
caption: One-carbon metabolism, transmethylation reactions, folate cycle, and DNA
  demethylation. DNA methylation on specific cytosine moieties is catalyzed by DNA-methyltransferases
  (DNMTs). This reaction is dependent on the levels of S-adenosyl-methionine (SAM),
  which is synthesized in the brain by the methionineadenosyl-transferase (MAT) 2.
  After the DNA methyl transfer reaction, S-adenosylhomocysteine (SAH) is formed as
  a by-product. SAH exerts a feedback inhibitory activity on DNMTs and is used as
  a substrate for the adenosyl homocysteine hydrolase (AHCY) leading to the synthesis
  of homocysteine. The remethylation of homocysteine to form methionine is catalyzed
  via the folate cycle, that is, the conversion of folic acid into tetrahydrofolate
  (THF) and 5,10-methylenete-trahydrofolate (5,10-MTHF). The methylene tetrahydrofolate
  reductase (MTHFR) will produce 5-methyltetrahydrofolate (5-MTHF) that is used as
  a substrate by methionine synthase (MTR) to form methionine. The synthesis of methionine
  may also involve the activation of the betaine homocysteine methyltransferase (BHMT)
  pathway. DNA demethylation is initiated by ten-eleven-translocase (TET) enzymes
  that hydroxylate 5-methylcytosine (5mC) forming 5-hydroxymethylcytosine (5hmC).
  In the end, through a GADD45B coordinated process, 5hmC is first converted to 5-formylcytosine
  (5fC) and then to 5-carboxylcytosine (5caC) by TET-mediated iterative processing.
  Thymidine deglycosylase (TDG) and members of the base excision repair (BER) pathway
  remove the modified residue, substituting it with cytosine. Stressful stimuli are
  known to activate neuronal pathways triggering DNA methylation or hydroxymethylation
  and demethylation, altering gene expression. Plus signs indicate positive regulation.
pmcid: PMC7942223
papertitle: 'Epigenetic landscape of stress surfeit disorders: Key role for DNA methylation
  dynamics.'
reftext: Eleonora Gatta, et al. Int Rev Neurobiol. ;156:127-183.
pmc_ranked_result_index: '39872'
pathway_score: 0.940859
filename: nihms-1675777-f0002.jpg
figtitle: One-carbon metabolism, transmethylation reactions, folate cycle, and DNA
  demethylation
year: ''
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7942223__nihms-1675777-f0002.html
  '@type': Dataset
  description: One-carbon metabolism, transmethylation reactions, folate cycle, and
    DNA demethylation. DNA methylation on specific cytosine moieties is catalyzed
    by DNA-methyltransferases (DNMTs). This reaction is dependent on the levels of
    S-adenosyl-methionine (SAM), which is synthesized in the brain by the methionineadenosyl-transferase
    (MAT) 2. After the DNA methyl transfer reaction, S-adenosylhomocysteine (SAH)
    is formed as a by-product. SAH exerts a feedback inhibitory activity on DNMTs
    and is used as a substrate for the adenosyl homocysteine hydrolase (AHCY) leading
    to the synthesis of homocysteine. The remethylation of homocysteine to form methionine
    is catalyzed via the folate cycle, that is, the conversion of folic acid into
    tetrahydrofolate (THF) and 5,10-methylenete-trahydrofolate (5,10-MTHF). The methylene
    tetrahydrofolate reductase (MTHFR) will produce 5-methyltetrahydrofolate (5-MTHF)
    that is used as a substrate by methionine synthase (MTR) to form methionine. The
    synthesis of methionine may also involve the activation of the betaine homocysteine
    methyltransferase (BHMT) pathway. DNA demethylation is initiated by ten-eleven-translocase
    (TET) enzymes that hydroxylate 5-methylcytosine (5mC) forming 5-hydroxymethylcytosine
    (5hmC). In the end, through a GADD45B coordinated process, 5hmC is first converted
    to 5-formylcytosine (5fC) and then to 5-carboxylcytosine (5caC) by TET-mediated
    iterative processing. Thymidine deglycosylase (TDG) and members of the base excision
    repair (BER) pathway remove the modified residue, substituting it with cytosine.
    Stressful stimuli are known to activate neuronal pathways triggering DNA methylation
    or hydroxymethylation and demethylation, altering gene expression. Plus signs
    indicate positive regulation.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - MTHFR
  - MTR
  - BHMT
  - MAT2B
  - MAT2A
  - AHCY
  - ACSM3
  - TDG
  - Folic Acid
  - 5,10-MTHF
  - 5-MTHF
  - Methionine
  - Homocysteine
  - S-adenosylmethionine
  - S-adenosylhomocysteine
  - Cytosine
  - S-adenosylhomocysteine
  - SAH
genes:
- word: MTHFR
  symbol: MTHFR
  source: hgnc_symbol
  hgnc_symbol: MTHFR
  entrez: '4524'
- word: MTR
  symbol: MTR
  source: hgnc_symbol
  hgnc_symbol: MTR
  entrez: '4548'
- word: BHMT
  symbol: BHMT
  source: hgnc_symbol
  hgnc_symbol: BHMT
  entrez: '635'
- word: MAT2A/b
  symbol: MAT2B
  source: hgnc_symbol
  hgnc_symbol: MAT2B
  entrez: '27430'
- word: MAT2A/b
  symbol: MAT2A
  source: hgnc_symbol
  hgnc_symbol: MAT2A
  entrez: '4144'
- word: АНСY
  symbol: AHCY
  source: hgnc_symbol
  hgnc_symbol: AHCY
  entrez: '191'
- word: '[SAH]'
  symbol: SAH
  source: hgnc_prev_symbol
  hgnc_symbol: ACSM3
  entrez: '6296'
- word: TDG/BER
  symbol: TDG
  source: hgnc_symbol
  hgnc_symbol: TDG
  entrez: '6996'
chemicals:
- word: Folic Acid
  source: MESH
  identifier: D005492
- word: 5,10-MTHF
  source: MESH
  identifier: C032418
- word: 5-MTHF
  source: MESH
  identifier: C032418
- word: Methionine
  source: MESH
  identifier: D008715
- word: Homocysteine
  source: MESH
  identifier: D006710
- word: S-adenosylmethionine
  source: MESH
  identifier: D012436
- word: S-adenosylhomocysteine
  source: MESH
  identifier: D012435
- word: Cytosine
  source: MESH
  identifier: D003596
diseases:
- word: S-adenosylhomocysteine
  source: MESH
  identifier: C564683
- word: SAH
  source: MESH
  identifier: D013345
figid_alias: PMC7942223__F2
redirect_from: /figures/PMC7942223__F2
figtype: Figure
---
