---
figid: PMC7563533__cancers-12-02694-g002
figlink: pmc/articles/PMC7563533/figure/cancers-12-02694-f002/
number: F2
caption: 'Key databases as primary resources for drug repositioning. Databases (DBs)
  can be divided into five classes, including Drug Design DB, Specific DB, Target-based
  DB, Tool-based DB, and Raw DB. The content of each DB is shown with an edge to the
  subcategories. Refer to  for a full description. DBs with omics and high-throughput
  datasets are colored in yellow. Subcategories: 3D: three-dimensional information;
  CLI: clinical; CUA: curation; DI: disease; ENZ: enzyme; GE: gene target; GO: geographic;
  INT: integration; Lit: literature; MAU: manual; MO: molecular information; MT: multidrug;
  NB: net-base; PR: protein target; PTH: pathway; SDE: side effect; ST: simple tool;
  TR: software. Databases: ACHI: Achilles; ARREXP: ArrayExpress; BLGe: BaderLabGenes;
  BIOM: Biomodels; BMRB: Biological Magnetic Resonance Data Bank; CCOM: Cancer Common;
  CCLF: cancer cell line factory; CCLE: Cancer Cell Line Encyclopedia; CFBs: Clearity
  Foundation Biomarkers; CFCT: Clearity Foundation Clinical Trial; CGI: Cancer Genome
  Interpreter; ChBLI: The ChEMBL Bioactivity Database; CIViC: Clinical Interpretations
  of Variants in Cancer; CKB: The Jackson Laboratory Clinical Knowledgebase; CTDB:
  Cheminformatic Tools and Databases; CMI: Caris Molecular Intelligence; CORUM: comprehensive
  resource of mammalian protein complexes; CTR: Cancer Therapeutics Response Portal;
  CTD: The Comparative Toxicogenomics Database; CTRP: Clinical Trials Reporting Program;
  dGene: The Druggable Gene List; DGIdb: drug–gene interaction database; DMAP: Dependency
  Map; DMC: Drug Map Central; DNET: Drug-Disease Network database; DoCM: Database
  of Curated Mutations; DPTH: Drug Pathway database; DRUGB: DrugBank; DRHUB: drug
  repurposing hub; DRAR: Drug Repurposing Adverse Reaction; DSDB: Drug Signatures
  Database; DSIG: DrugSig; DTW: Drug Target Web; ENCODE: Encyclopedia of DNA Elements,
  Entrez, Ensembl; FDA: FDA Pharmacogenomic Biomarkers; FOGe: Foundation One Gene;
  GEO: Gene Expression Omnibus; GPIs: Guide To Pharmacology Interactions; GPCR: G-protein-coupled
  receptors assay bank; GDSC: Genomics of Drug Sensitivity in Cancer; GO: The Gene
  Ontology; GRNAi: GenomeRNAi; GSEA(MSigDB): Gene Set Enrichment Analysis; GSDB: Gene
  Set Database; GTPGe: Guide To Pharmacology Genes; HCa: Hingorani Casas; HIVRT: HIV
  Drug Resistance Database; HkG: Hopkins Groom; HTRCP: host transcriptional response
  connectivity map; L1000: Library of Integrated Network-based Cellular Signatures
  (LINCS) L1000; MCGCT: My Cancer Genome Clinical Trial; METN: Metlin; META: Metabolic
  Atlas; MskIm: Memorial Sloan Kettering IMPACT; NCI: Cancer Gene Index; NCATS: National
  Center for Advancing Translational Sciences; NNFIN: network-based similarity finder;
  ODB: Ontario database; OGEE: Online GEne Essentiality; OMDI: Omics Discovery Index
  (omicsdi); OncoKB: A Precision Oncology Knowledge Base; PACO: pathway commons; HPA:
  Protein Atlas; PBL: ProBiS-ligands; PDTD: Potential Drug-Target Database; PhGKB:
  The Pharmacogenomics Knowledgebase; PRIDE: PRoteomicsIDEntifications (PRIDE) database;
  PROM: Promiscuous; RepDB: Repurpose DB; RLa: RussLampel; SAINT: Significance Analysis
  of INTeractome; SBIOS: Swiss BIOisostere; SCYP: Super Cytochrome P450; SIDER: Side
  Effect Resource; SMPDB: Small Molecule Pathway Database; TALC: Targeted Agents in
  Lung Cancer; TCSBN: tissue- and cancer-specific biological networks; TDR: Tropical
  Diseases Research; TDGCT: The Druggable Genome (TDG) Clinical Trial; TEND: trends
  in the exploitation of novel drug targets; TTD: Therapeutic Target Database.'
pmcid: PMC7563533
papertitle: Applications of Genome-Wide Screening and Systems Biology Approaches in
  Drug Repositioning.
reftext: Elyas Mohammadi, et al. Cancers (Basel). 2020 Sep;12(9):2694.
pmc_ranked_result_index: '37816'
pathway_score: 0.9461834
filename: cancers-12-02694-g002.jpg
figtitle: Key databases as primary resources for drug repositioning
year: '2020'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7563533__cancers-12-02694-g002.html
  '@type': Dataset
  description: 'Key databases as primary resources for drug repositioning. Databases
    (DBs) can be divided into five classes, including Drug Design DB, Specific DB,
    Target-based DB, Tool-based DB, and Raw DB. The content of each DB is shown with
    an edge to the subcategories. Refer to  for a full description. DBs with omics
    and high-throughput datasets are colored in yellow. Subcategories: 3D: three-dimensional
    information; CLI: clinical; CUA: curation; DI: disease; ENZ: enzyme; GE: gene
    target; GO: geographic; INT: integration; Lit: literature; MAU: manual; MO: molecular
    information; MT: multidrug; NB: net-base; PR: protein target; PTH: pathway; SDE:
    side effect; ST: simple tool; TR: software. Databases: ACHI: Achilles; ARREXP:
    ArrayExpress; BLGe: BaderLabGenes; BIOM: Biomodels; BMRB: Biological Magnetic
    Resonance Data Bank; CCOM: Cancer Common; CCLF: cancer cell line factory; CCLE:
    Cancer Cell Line Encyclopedia; CFBs: Clearity Foundation Biomarkers; CFCT: Clearity
    Foundation Clinical Trial; CGI: Cancer Genome Interpreter; ChBLI: The ChEMBL Bioactivity
    Database; CIViC: Clinical Interpretations of Variants in Cancer; CKB: The Jackson
    Laboratory Clinical Knowledgebase; CTDB: Cheminformatic Tools and Databases; CMI:
    Caris Molecular Intelligence; CORUM: comprehensive resource of mammalian protein
    complexes; CTR: Cancer Therapeutics Response Portal; CTD: The Comparative Toxicogenomics
    Database; CTRP: Clinical Trials Reporting Program; dGene: The Druggable Gene List;
    DGIdb: drug–gene interaction database; DMAP: Dependency Map; DMC: Drug Map Central;
    DNET: Drug-Disease Network database; DoCM: Database of Curated Mutations; DPTH:
    Drug Pathway database; DRUGB: DrugBank; DRHUB: drug repurposing hub; DRAR: Drug
    Repurposing Adverse Reaction; DSDB: Drug Signatures Database; DSIG: DrugSig; DTW:
    Drug Target Web; ENCODE: Encyclopedia of DNA Elements, Entrez, Ensembl; FDA: FDA
    Pharmacogenomic Biomarkers; FOGe: Foundation One Gene; GEO: Gene Expression Omnibus;
    GPIs: Guide To Pharmacology Interactions; GPCR: G-protein-coupled receptors assay
    bank; GDSC: Genomics of Drug Sensitivity in Cancer; GO: The Gene Ontology; GRNAi:
    GenomeRNAi; GSEA(MSigDB): Gene Set Enrichment Analysis; GSDB: Gene Set Database;
    GTPGe: Guide To Pharmacology Genes; HCa: Hingorani Casas; HIVRT: HIV Drug Resistance
    Database; HkG: Hopkins Groom; HTRCP: host transcriptional response connectivity
    map; L1000: Library of Integrated Network-based Cellular Signatures (LINCS) L1000;
    MCGCT: My Cancer Genome Clinical Trial; METN: Metlin; META: Metabolic Atlas; MskIm:
    Memorial Sloan Kettering IMPACT; NCI: Cancer Gene Index; NCATS: National Center
    for Advancing Translational Sciences; NNFIN: network-based similarity finder;
    ODB: Ontario database; OGEE: Online GEne Essentiality; OMDI: Omics Discovery Index
    (omicsdi); OncoKB: A Precision Oncology Knowledge Base; PACO: pathway commons;
    HPA: Protein Atlas; PBL: ProBiS-ligands; PDTD: Potential Drug-Target Database;
    PhGKB: The Pharmacogenomics Knowledgebase; PRIDE: PRoteomicsIDEntifications (PRIDE)
    database; PROM: Promiscuous; RepDB: Repurpose DB; RLa: RussLampel; SAINT: Significance
    Analysis of INTeractome; SBIOS: Swiss BIOisostere; SCYP: Super Cytochrome P450;
    SIDER: Side Effect Resource; SMPDB: Small Molecule Pathway Database; TALC: Targeted
    Agents in Lung Cancer; TCSBN: tissue- and cancer-specific biological networks;
    TDR: Tropical Diseases Research; TDGCT: The Druggable Genome (TDG) Clinical Trial;
    TEND: trends in the exploitation of novel drug targets; TTD: Therapeutic Target
    Database.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - HPSE
  - PTH
  - BCR
  - CLU
  - CALCR
  - CKB
  - RTRAF
  - NCL
  - ZBTB8OS
  - GTF2H5
  - ROR2
  - DMAP
  - ARCH
genes:
- word: НРА
  symbol: HPA
  source: hgnc_alias_symbol
  hgnc_symbol: HPSE
  entrez: '10855'
- word: PTH
  symbol: PTH
  source: hgnc_symbol
  hgnc_symbol: PTH
  entrez: '5741'
- word: CMI
  symbol: CML
  source: hgnc_alias_symbol
  hgnc_symbol: BCR
  entrez: '613'
- word: CLI
  symbol: CLI
  source: hgnc_prev_symbol
  hgnc_symbol: CLU
  entrez: '1191'
- word: CTR
  symbol: CTR
  source: hgnc_alias_symbol
  hgnc_symbol: CALCR
  entrez: '799'
- word: CKB
  symbol: CKB
  source: hgnc_symbol
  hgnc_symbol: CKB
  entrez: '1152'
- word: CLE
  symbol: CLE
  source: hgnc_alias_symbol
  hgnc_symbol: RTRAF
  entrez: '51637'
- word: NCI
  symbol: NCL
  source: hgnc_symbol
  hgnc_symbol: NCL
  entrez: '4691'
- word: ARCH
  symbol: ARCH
  source: hgnc_alias_symbol
  hgnc_symbol: ZBTB8OS
  entrez: '339487'
- word: TTD
  symbol: TTD
  source: hgnc_prev_symbol
  hgnc_symbol: GTF2H5
  entrez: '404672'
- word: BDB
  symbol: BDB
  source: hgnc_prev_symbol
  hgnc_symbol: ROR2
  entrez: '4920'
chemicals:
- word: DMAP
  source: MESH
  identifier: C003885
diseases:
- word: ARCH
  source: MESH
  identifier: D004310
figid_alias: PMC7563533__F2
redirect_from: /figures/PMC7563533__F2
figtype: Figure
---
