---
figid: PMC7479627__gr2
figlink: pmc/articles/PMC7479627/figure/fig2/
number: F2
caption: Potential CRISPR/Cas Applications to Increase Pathogen Resistance(A–E) Depending
  on the repair mechanism the CRISPR/Cas-induced double-strand breaks (DSBs) can lead
  to simple mutation of a certain sequence (via the error-prone non-homologous end
  joining [NHEJ] pathway), resulting often in small insertions or deletions (InDels)
  causing frameshift mutations and thereby loss of function (A). Alternatively, in
  the presence of a repair template, the DSB can be repaired via homology-directed
  repair (HDR) allowing the insertion of larger foreign DNA in a genomic locus of
  choice (B) or modification of a target region by introducing mutations of choice
  (C). Single-nucleotide mutations can further be induced by deploying cytosine base
  editors (CBEs) with cytidine deaminase or adenine base editors (ABEs) with adenine
  deaminase activity (D). Larger defined mutations can be induced by prime editing,
  which is based on nCas9 fused with reverse transcriptase depending on prime editing
  gRNA instead of normal sgRNAs (E). For details see text.
pmcid: PMC7479627
papertitle: 'Applications of CRISPR/Cas to Improve Crop Disease Resistance: Beyond
  Inactivation of Susceptibility Factors.'
reftext: Dirk Schenke, et al. iScience. 2020 Sep 25;23(9):101478.
pmc_ranked_result_index: '105725'
pathway_score: 0.5657157
filename: gr2.jpg
figtitle: 'Applications of CRISPR/Cas to Improve Crop Disease Resistance: Beyond Inactivation
  of Susceptibility Factors'
year: '2020'
organisms:
- Verticillium longisporum
- Tomato yellow leaf curl virus
- Wheat dwarf virus
- Nilaparvata lugens
- Papaya ringspot virus
- Potato virus Y
- Soybean mosaic virus
- Turnip mosaic virus
- Zucchini yellow mosaic virus
- Streptococcus pyogenes
- Peronospora farinosa
- Cucumber vein yellowing virus
- Phytophthora tropicalis
- Cassava brown streak virus
- Chilo suppressalis
- Erysiphe neolycopersici
- Cotton leaf curl Multan virus
- Pseudomonas syringae pv. tomato str. DC3000
- Citrullus lanatus subsp. vulgaris
- Citrus sinensis
- Verticillium dahliae
- Chlamydomonas reinhardtii
- Pseudomonas syringae
- Pyricularia oryzae
- Pseudomonas syringae pv. tomato
- Xanthomonas campestris
- Blumeria graminis
- Heterodera avenae
- Xanthomonas citri
- Xanthomonas oryzae
- Rice tungro spherical virus
- Agrobacterium tumefaciens
- Gossypium hirsutum
- Cucumis sativus
- Arabidopsis thaliana
- Brassica napus
- Brassica rapa
- Brassica oleracea
- Fragaria x ananassa
- Malus domestica
- Citrus x paradisi
- Glycine max
- Manihot esculenta
- Botrytis cinerea
- Solanum lycopersicum
- Nicotiana tabacum
- Nicotiana benthamiana
- Petunia x hybrida
- Solanum tuberosum
- Liliopsida
- Hordeum vulgare
- Oryza sativa
- Saccharum officinarum
- Sorghum bicolor
- Triticum aestivum
- Zea mays
- Musa acuminata
- Musa sp.
- Phytophthora capsici
- Phytophthora infestans
- Phytophthora parasitica
- Sclerotinia sclerotiorum
- Fusarium oxysporum
- Fusarium graminearum
- Erwinia amylovora
- Blumeria graminis f. sp. tritici
- Xanthomonas oryzae pv. oryzae
- Banana streak virus
- eudicotyledons
- Exserohilum turcicum
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7479627__gr2.html
  '@type': Dataset
  description: Potential CRISPR/Cas Applications to Increase Pathogen Resistance(A–E)
    Depending on the repair mechanism the CRISPR/Cas-induced double-strand breaks
    (DSBs) can lead to simple mutation of a certain sequence (via the error-prone
    non-homologous end joining [NHEJ] pathway), resulting often in small insertions
    or deletions (InDels) causing frameshift mutations and thereby loss of function
    (A). Alternatively, in the presence of a repair template, the DSB can be repaired
    via homology-directed repair (HDR) allowing the insertion of larger foreign DNA
    in a genomic locus of choice (B) or modification of a target region by introducing
    mutations of choice (C). Single-nucleotide mutations can further be induced by
    deploying cytosine base editors (CBEs) with cytidine deaminase or adenine base
    editors (ABEs) with adenine deaminase activity (D). Larger defined mutations can
    be induced by prime editing, which is based on nCas9 fused with reverse transcriptase
    depending on prime editing gRNA instead of normal sgRNAs (E). For details see
    text.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - CEACAM6
  - GATA3
  - CST8
  - HAS1
  - cytidine
  - adenine
  - HIGS
genes:
- word: CRISPR/nCas
  symbol: NCA
  source: hgnc_prev_symbol
  hgnc_symbol: CEACAM6
  entrez: '4680'
- word: HDR
  symbol: HDR
  source: hgnc_alias_symbol
  hgnc_symbol: GATA3
  entrez: '2625'
- word: CRES
  symbol: CRES
  source: hgnc_alias_symbol
  hgnc_symbol: CST8
  entrez: '10047'
- word: CRES
  symbol: CRES
  source: hgnc_alias_symbol
  hgnc_symbol: CST8
  entrez: '10047'
- word: has
  symbol: HAS
  source: hgnc_prev_symbol
  hgnc_symbol: HAS1
  entrez: '3036'
chemicals:
- word: cytidine
  source: MESH
  identifier: D003562
- word: adenine
  source: MESH
  identifier: D000225
diseases:
- word: HIGS
  source: ''
  identifier: ''
figid_alias: PMC7479627__F2
redirect_from: /figures/PMC7479627__F2
figtype: Figure
---
