---
figid: PMC7396666__fonc-10-01100-g0002
figlink: pmc/articles/PMC7396666/figure/F2/
number: Figure 2
caption: 'Overview of the mammalian UPR. The three ER resident sensors (IRE1α, PERK,
  and ATF6) transduce information about the protein folding status of the ER to the
  cytosol and nucleus to restore the protein folding capacity. Under normal conditions,
  the sensors are inactivated by binding to the chaperone BiP. Under ER stress conditions,
  the sensors are activated by BiP dissociation and/or direct misfolded protein binding.
  Each pathway uses a different mechanism for signal transduction upon activation.
  IRE1α dimerizes, autophosphorylates, and triggers its RNase activity. This leads
  to the splicing of the XBP1 mRNA to produce an active transcription factor, spliced
  XBP1 (XBP1s). XBP1s induces the transcription of the genes encoding protein chaperones,
  ERAD, and phospholipid synthesis. The RNase activity of IRE1α also degrades certain
  mRNAs through RIDD. Activated IRE1α can activate the JNK, p38, ERK, and NF-kB pathways,
  thus playing an XBP1-independent role to modulate diverse cellular responses. Upon
  activation, PERK phosphorylates eIF2α, leading to global translational attenuation
  while selectively mediating translation of ATF4. In turn, ATF4 induces the expression
  of genes involved in amino acid metabolism, proapoptotic factor DDIT3/CHOP, and
  antioxidant responses (HO1). PERK also phosphorylates and stabilizes NRF2, a transcription
  factor involved in redox metabolism. ATF6 is transported to the Golgi apparatus
  under ER stress, where it is processed by S1P and S2P, releasing its cytosolic domain
  fragment as a transcription factor. ATF6 activates genes encoding protein chaperones,
  ERAD components, and XBP1. Abbreviations: ATF, activating transcription factor;
  BiP, binding immunoglobulin protein; DDIT3, DNA damage inducible transcript 3; eIF2α,
  eukaryotic translation initiation factor 2 subunit 1; ER, endoplasmic reticulum;
  ERAD, ER-associated protein degradation; HO1, heme oxygenase 1; IRE1α, inositol-requiring
  enzyme 1α; JNK, c-Jun N-terminal kinase; NF-kB, nuclear factor kappa light-chain
  enhancer of activated B cells; NRF2, NF-E2-related factor 2; PERK, PKR-like ER kinase;
  RIDD, regulated IRE1α dependent decay of mRNA; S1P and S2P, site 1 and site 2 proteases;
  UPR, unfolded protein response.'
pmcid: PMC7396666
papertitle: Endoplasmic Reticulum Stress in Bone Metastases.
reftext: Longyong Xu, et al. Front Oncol. 2020;10:1100.
pmc_ranked_result_index: '106872'
pathway_score: 0.965945
filename: fonc-10-01100-g0002.jpg
figtitle: Mammalian UPR
year: '2020'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7396666__fonc-10-01100-g0002.html
  '@type': Dataset
  description: 'Overview of the mammalian UPR. The three ER resident sensors (IRE1α,
    PERK, and ATF6) transduce information about the protein folding status of the
    ER to the cytosol and nucleus to restore the protein folding capacity. Under normal
    conditions, the sensors are inactivated by binding to the chaperone BiP. Under
    ER stress conditions, the sensors are activated by BiP dissociation and/or direct
    misfolded protein binding. Each pathway uses a different mechanism for signal
    transduction upon activation. IRE1α dimerizes, autophosphorylates, and triggers
    its RNase activity. This leads to the splicing of the XBP1 mRNA to produce an
    active transcription factor, spliced XBP1 (XBP1s). XBP1s induces the transcription
    of the genes encoding protein chaperones, ERAD, and phospholipid synthesis. The
    RNase activity of IRE1α also degrades certain mRNAs through RIDD. Activated IRE1α
    can activate the JNK, p38, ERK, and NF-kB pathways, thus playing an XBP1-independent
    role to modulate diverse cellular responses. Upon activation, PERK phosphorylates
    eIF2α, leading to global translational attenuation while selectively mediating
    translation of ATF4. In turn, ATF4 induces the expression of genes involved in
    amino acid metabolism, proapoptotic factor DDIT3/CHOP, and antioxidant responses
    (HO1). PERK also phosphorylates and stabilizes NRF2, a transcription factor involved
    in redox metabolism. ATF6 is transported to the Golgi apparatus under ER stress,
    where it is processed by S1P and S2P, releasing its cytosolic domain fragment
    as a transcription factor. ATF6 activates genes encoding protein chaperones, ERAD
    components, and XBP1. Abbreviations: ATF, activating transcription factor; BiP,
    binding immunoglobulin protein; DDIT3, DNA damage inducible transcript 3; eIF2α,
    eukaryotic translation initiation factor 2 subunit 1; ER, endoplasmic reticulum;
    ERAD, ER-associated protein degradation; HO1, heme oxygenase 1; IRE1α, inositol-requiring
    enzyme 1α; JNK, c-Jun N-terminal kinase; NF-kB, nuclear factor kappa light-chain
    enhancer of activated B cells; NRF2, NF-E2-related factor 2; PERK, PKR-like ER
    kinase; RIDD, regulated IRE1α dependent decay of mRNA; S1P and S2P, site 1 and
    site 2 proteases; UPR, unfolded protein response.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - EIF2AK3
  - ATF6
  - MAPK14
  - MAPK11
  - MAPK13
  - MAPK12
  - MAPK8
  - MAPK9
  - MAPK10
  - XBP1
  - EIF2A
  - MBTPS2
  - ATF4
  - DDIT3
  - Amino acid
  - RIDD
  - SIP
genes:
- word: PERK
  symbol: PERK
  source: hgnc_alias_symbol
  hgnc_symbol: EIF2AK3
  entrez: '9451'
- word: ATF6
  symbol: ATF6
  source: hgnc_symbol
  hgnc_symbol: ATF6
  entrez: '22926'
- word: p38
  symbol: p38
  source: bioentities_symbol
  hgnc_symbol: MAPK14
  entrez: '1432'
- word: p38
  symbol: p38
  source: bioentities_symbol
  hgnc_symbol: MAPK11
  entrez: '5600'
- word: p38
  symbol: p38
  source: bioentities_symbol
  hgnc_symbol: MAPK13
  entrez: '5603'
- word: p38
  symbol: p38
  source: bioentities_symbol
  hgnc_symbol: MAPK12
  entrez: '6300'
- word: JNK
  symbol: JNK
  source: bioentities_symbol
  hgnc_symbol: MAPK8
  entrez: '5599'
- word: JNK
  symbol: JNK
  source: bioentities_symbol
  hgnc_symbol: MAPK9
  entrez: '5601'
- word: JNK
  symbol: JNK
  source: bioentities_symbol
  hgnc_symbol: MAPK10
  entrez: '5602'
- word: XBP1
  symbol: XBP1
  source: hgnc_symbol
  hgnc_symbol: XBP1
  entrez: '7494'
- word: elF2a
  symbol: EIF2A
  source: hgnc_symbol
  hgnc_symbol: EIF2A
  entrez: '83939'
- word: S2P
  symbol: S2P
  source: hgnc_alias_symbol
  hgnc_symbol: MBTPS2
  entrez: '51360'
- word: ATF4
  symbol: ATF4
  source: hgnc_symbol
  hgnc_symbol: ATF4
  entrez: '468'
- word: ATF6
  symbol: ATF6
  source: hgnc_symbol
  hgnc_symbol: ATF6
  entrez: '22926'
- word: ATF4
  symbol: ATF4
  source: hgnc_symbol
  hgnc_symbol: ATF4
  entrez: '468'
- word: ATF6
  symbol: ATF6
  source: hgnc_symbol
  hgnc_symbol: ATF6
  entrez: '22926'
- word: CHOP
  symbol: CHOP
  source: hgnc_alias_symbol
  hgnc_symbol: DDIT3
  entrez: '1649'
- word: XBP1
  symbol: XBP1
  source: hgnc_symbol
  hgnc_symbol: XBP1
  entrez: '7494'
chemicals:
- word: Amino acid
  source: MESH
  identifier: D000596
diseases:
- word: RIDD
  source: ''
  identifier: ''
- word: SIP
  source: ''
  identifier: ''
---
