---
figid: PMC7454314__bbaa173f1
figlink: pmc/articles/PMC7454314/figure/f1/
number: F1
caption: The overall experimental workflow applied in this study. Following establishing
  SARS-CoV as the closest relative of 2019-nCoV through whole-genome alignment (1000
  bootstrap tests), the microarray datasets related to SARS-CoV infection and four
  COVID-19 complications (i.e. SARS, PNA, SHOB and DRA) were retrieved and analyzed
  to identify differentially expressed genes (DEGs). Herein, Limma R package was employed
  where logFC≥|1| was used to distinguish differential expression, and adjusted P-value
  ≤0.05 was set as cutoff for statistical significance. Then, the DEGs from SARS-CoV
  infection were subjected to validation with gold-standard databases (i.e. OMIM disease,
  OMIM expanded and dbGaP) for comorbidity and risk factor assessment followed by
  comparative analysis with the disease datasets. Finally, we used the shared DEGs
  for network-based interactome analysis and functional enrichments in terms of protein–protein
  interactions (PPI confidence cutoff = 980), gene–biomarker associations (DC≥5 and
  BC≥35) and protein–chemical interactions (DC≥20 and BC≥20). For pathway analysis,
  we considered BioCarta (2016), Reactome (2016), KEGG (human; 2019) and WikiPathways
  (human; 2019) databases, while Biological Process (2018), Molecular Function (2018)
  and Human Phenotype Ontology databases were used for gene ontological evaluation.
  In both cases, adjusted P-value ≤0.05 was considered as significant.
pmcid: PMC7454314
papertitle: Pathogenetic profiling of COVID-19 and SARS-like viruses.
reftext: Zulkar Nain, et al. Brief Bioinform. 2020 Aug 11 :bbaa173.
pmc_ranked_result_index: '85608'
pathway_score: 0.515482
filename: bbaa173f1.jpg
figtitle: Overall experimental workflow applied in this study
year: '2020'
organisms:
- Vaccinia virus
- H5N1 subtype
- Suid alphaherpesvirus 1
- Coronaviridae
- Human coronavirus 229E
- Murine hepatitis virus strain 1
- Vesicular stomatitis virus
- Orthomyxoviridae
- Influenza A virus
- H1N1 subtype
- H3N2 subtype
- Human immunodeficiency virus
- Respiratory syncytial virus
- Middle East respiratory syndrome-related coronavirus
- Macropodid alphaherpesvirus 1
- Bat coronavirus
- Bat SARS-like coronavirus
- Severe acute respiratory syndrome coronavirus 2
- Sar
- Porcine reproductive and respiratory syndrome virus
- Human coronavirus HKU1
- Human coronavirus OC43
- H7N9 subtype
- Saccharomyces cerevisiae
- Betacoronavirus
- Severe acute respiratory syndrome-related coronavirus
- unidentified human coronavirus
- Homo sapiens
- Canis lupus familiaris
- Mustela putorius furo
- Felis catus
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7454314__bbaa173f1.html
  '@type': Dataset
  description: The overall experimental workflow applied in this study. Following
    establishing SARS-CoV as the closest relative of 2019-nCoV through whole-genome
    alignment (1000 bootstrap tests), the microarray datasets related to SARS-CoV
    infection and four COVID-19 complications (i.e. SARS, PNA, SHOB and DRA) were
    retrieved and analyzed to identify differentially expressed genes (DEGs). Herein,
    Limma R package was employed where logFC≥|1| was used to distinguish differential
    expression, and adjusted P-value ≤0.05 was set as cutoff for statistical significance.
    Then, the DEGs from SARS-CoV infection were subjected to validation with gold-standard
    databases (i.e. OMIM disease, OMIM expanded and dbGaP) for comorbidity and risk
    factor assessment followed by comparative analysis with the disease datasets.
    Finally, we used the shared DEGs for network-based interactome analysis and functional
    enrichments in terms of protein–protein interactions (PPI confidence cutoff =
    980), gene–biomarker associations (DC≥5 and BC≥35) and protein–chemical interactions
    (DC≥20 and BC≥20). For pathway analysis, we considered BioCarta (2016), Reactome
    (2016), KEGG (human; 2019) and WikiPathways (human; 2019) databases, while Biological
    Process (2018), Molecular Function (2018) and Human Phenotype Ontology databases
    were used for gene ontological evaluation. In both cases, adjusted P-value ≤0.05
    was considered as significant.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - C1QB
  - C1R
  - C1QA
  - C1QC
  - C1S
  - C2
  - C3
  - PPL
  - 2019-nCov
  - COVID-19
  - CoV
genes:
- word: C1
  symbol: C1
  source: bioentities_symbol
  hgnc_symbol: C1QB
  entrez: '713'
- word: C1
  symbol: C1
  source: bioentities_symbol
  hgnc_symbol: C1R
  entrez: '715'
- word: C1
  symbol: C1
  source: bioentities_symbol
  hgnc_symbol: C1QA
  entrez: '712'
- word: C1
  symbol: C1
  source: bioentities_symbol
  hgnc_symbol: C1QC
  entrez: '714'
- word: C1
  symbol: C1
  source: bioentities_symbol
  hgnc_symbol: C1S
  entrez: '716'
- word: C2
  symbol: C2
  source: hgnc_symbol
  hgnc_symbol: C2
  entrez: '717'
- word: C3
  symbol: C3
  source: hgnc_symbol
  hgnc_symbol: C3
  entrez: '718'
- word: PPI
  symbol: PPL
  source: hgnc_symbol
  hgnc_symbol: PPL
  entrez: '5493'
chemicals: []
diseases:
- word: 2019-nCov
  source: MESH
  identifier: C000657245
- word: COVID-19
  source: MESH
  identifier: C000657245
- word: CoV
  source: MESH
  identifier: C000657245
figid_alias: PMC7454314__F1
redirect_from: /figures/PMC7454314__F1
figtype: Figure
---
