---
figid: PMC7284592__nutrients-12-01420-g001
figlink: pmc/articles/PMC7284592/figure/nutrients-12-01420-f001/
number: F1
caption: 'Schematic representation of two-sample Mendelian randomization analysis
  (2SMR) using Bonder microbiome and Dubois celiac disease (CeD) genome-wide association
  study (GWAS) as exposure and outcome datasets, respectively. (a) Flowchart of the
  step-by-step analysis: after preparing exposure data (Step 1), outcome data is extracted
  (Step 2), both datasets are harmonized (Step 3), and 2SMR analysis is performed
  (Step 4); finally, significant hits are selected based on their false discovery
  rate (FDR) (Step 5); (b) Diagram, representing the number of single nucleotide polymorphisms
  (SNPs) selected in each category (taxa, pathway, gene ontology (GO)) after performing
  each step of the analysis.'
pmcid: PMC7284592
papertitle: A Two-Sample Mendelian Randomization Analysis Investigates Associations
  Between Gut Microbiota and Celiac Disease.
reftext: Iraia Garc√≠a-Santisteban, et al. Nutrients. 2020 May;12(5):1420.
pmc_ranked_result_index: '52585'
pathway_score: 0.9090961
filename: nutrients-12-01420-g001.jpg
figtitle: Two-sample Mendelian randomization analysis (2SMR) using Bonder microbiome
  and Dubois celiac disease (CeD) genome-wide association study (GWAS) as exposure
  and outcome datasets, respectively
year: '2020'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7284592__nutrients-12-01420-g001.html
  '@type': Dataset
  description: 'Schematic representation of two-sample Mendelian randomization analysis
    (2SMR) using Bonder microbiome and Dubois celiac disease (CeD) genome-wide association
    study (GWAS) as exposure and outcome datasets, respectively. (a) Flowchart of
    the step-by-step analysis: after preparing exposure data (Step 1), outcome data
    is extracted (Step 2), both datasets are harmonized (Step 3), and 2SMR analysis
    is performed (Step 4); finally, significant hits are selected based on their false
    discovery rate (FDR) (Step 5); (b) Diagram, representing the number of single
    nucleotide polymorphisms (SNPs) selected in each category (taxa, pathway, gene
    ontology (GO)) after performing each step of the analysis.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - TGFB1
  - RPL39
  - RPL37
genes:
- word: (CeD,
  symbol: CED
  source: hgnc_alias_symbol
  hgnc_symbol: TGFB1
  entrez: '7040'
- word: '139'
  symbol: L39
  source: hgnc_alias_symbol
  hgnc_symbol: RPL39
  entrez: '6170'
- word: '137'
  symbol: L37
  source: hgnc_alias_symbol
  hgnc_symbol: RPL37
  entrez: '6167'
- word: CeD)
  symbol: CED
  source: hgnc_alias_symbol
  hgnc_symbol: TGFB1
  entrez: '7040'
- word: CeD
  symbol: CED
  source: hgnc_alias_symbol
  hgnc_symbol: TGFB1
  entrez: '7040'
chemicals: []
diseases: []
figid_alias: PMC7284592__F1
redirect_from: /figures/PMC7284592__F1
figtype: Figure
---
