---
figid: PMC7803631__gr3a
figlink: .na.character
number: F3a
caption: 'Cell free biosynthesis for erythromycin A. The erythromycin A biosynthetic
  pathway (a) color-coded for steps leading to complete compound formation. Cell free
  biosynthesis for TDP-4′-keto-6′-deoxy glucose using plasmids pGJZ10 (indicated [])
  and pGro7 (providing the GroEL/ES chaperonin genes), using a plasmid expressing
  eGFP as a negative control and compared to samples with pGJZ10 and pGJZ10+pGro7
  as analyzed by LC-MS (b). Cell free production for TDP-l-mycarose and TDP-d-desosamine
  using pTailoring previously described [] (c). Cell free biosynthesis of 6-deoxyerythronolide
  B (6dEB) using plasmids pDEBS [] assessed by LC-MS (d). Combining the pTailoring
  plasmid with the 6dEB molecule indicated erythromycin A formation by LC-MS (e).
  Cell free extract was prepared as follows: 10 mL of overnight-cultured E. coli BL21
  Star (DE3) was inoculated in 1 L of 2 × YTPG medium (10 g/L yeast extract, 16 g/L
  tryptone, 3 g/L K2HPO4, 7 g/L KH2PO4, 5 g/L NaCl, and 18 g/L glucose) and grown
  at 37 °C with 250 rpm shaking. The culture was induced with 0.5 mM IPTG at an OD600nm
  of 0.6–0.8, and the cells were grown until OD600nm 3.5. The cells were harvested
  by centrifuge (8000 rpm, 10 min) and washed thrice with S30 Buffer (10 mM Tris-acetate,
  14 mM Mg(OAc)2, 60 mM KOAc, 5 mM DTT) and stored at −80 °C after flash-freeze using
  liquid nitrogen. The cells were resuspended in S30 Buffer (1.2 g/mL) and aliquoted
  to 1.5 mL. Aliquots were sonicated with a FB50 (Thermo Fisher) sonifier with a 3 mm
  microtip (10 s on/10 s off) 5 times. The extract was prepared by collecting supernatant
  after 12,000 rpm of centrifugation. Cell free natural product biosynthesis was performed
  by using eGFP plasmid DNA as a control and a reaction buffer with the following
  contents: 1.2 mM ATP; 0.85 mM each of GTP, UTP and CTP; 34 μg/mL folinic acid; 171 μg/mL
  T7 RNA polymerase; 2 mM each of the 20 translatable amino acids, 0.33 mM nicotinamide
  adenine dinucleotide (NAD), 0.26 mM coenzyme A (CoA), 33 mM PEP, 130 mM potassium
  glutamate, 10 mM ammonium glutamate, 12 mM magnesium glutamate, 1.5 mM spermidine,
  1 mM putrescine, 57 mM HEPES, 4 mM sodium oxalate, and 1 μL of 20 mg/mL Sfp [].
  To the reaction buffer, 0.25 volume of cell extract (described above) and 20 μg/mL
  plasmid DNA were added to reach a final volume of 25 μL. The reaction was held for
  20 h at 30 °C and the required natural product metabolic substrates were added:
  gluocse-1-phosphate (10 mM) and dTTP (10 mM) for tailoring sugar biosynthesis, and
  propionyl-CoA (1 mM) and methyl malonyl-CoA (6 mM) for 6dEB biosynthesis. After
  20 h, the samples were moved to −20 °C prior to SDS-PAGE and LC-MS analysis. Samples
  were prepared for analysis by diluting three times in LC-MS grade methanol and centrifuging
  at 13,000 rpm for 15 min with supernatant used for analysis. Erythromycin and 6dEB
  samples were applied to a Thermo Scientific Orbitrap XL with a C-18 analytical column.
  TDP-deoxysugar samples (mycarose, desosamine) were measured by Agilent G6545A quadrupole-time-of-flight
  (Q-TOF) using an XBridge Shield RP18 3.5 μm, 3.0 mm × 150 mm column from Waters.
  All MS analyses were conducted in positive ion mode. A linear gradient of 80% buffer
  A (95% water/5% acetonitrile/0.1% formic acid) to 100% buffer B (5% water/95% acetonitrile/0.1%
  formic acid) was used at a flow rate of 10 μL/min for the LC.'
pmcid: PMC7803631
papertitle: Complex natural product production methods and options.
reftext: Dongwon Park, et al. Synth Syst Biotechnol. 2021 Mar;6(1):1-11.
pmc_ranked_result_index: '124515'
pathway_score: 0.6484323
filename: gr3a.jpg
figtitle: Complex natural product production methods and options
year: '2021'
organisms: Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7803631__gr3a.html
  '@type': Dataset
  description: 'Cell free biosynthesis for erythromycin A. The erythromycin A biosynthetic
    pathway (a) color-coded for steps leading to complete compound formation. Cell
    free biosynthesis for TDP-4′-keto-6′-deoxy glucose using plasmids pGJZ10 (indicated
    []) and pGro7 (providing the GroEL/ES chaperonin genes), using a plasmid expressing
    eGFP as a negative control and compared to samples with pGJZ10 and pGJZ10+pGro7
    as analyzed by LC-MS (b). Cell free production for TDP-l-mycarose and TDP-d-desosamine
    using pTailoring previously described [] (c). Cell free biosynthesis of 6-deoxyerythronolide
    B (6dEB) using plasmids pDEBS [] assessed by LC-MS (d). Combining the pTailoring
    plasmid with the 6dEB molecule indicated erythromycin A formation by LC-MS (e).
    Cell free extract was prepared as follows: 10 mL of overnight-cultured E. coli
    BL21 Star (DE3) was inoculated in 1 L of 2 × YTPG medium (10 g/L yeast extract,
    16 g/L tryptone, 3 g/L K2HPO4, 7 g/L KH2PO4, 5 g/L NaCl, and 18 g/L glucose) and
    grown at 37 °C with 250 rpm shaking. The culture was induced with 0.5 mM IPTG
    at an OD600nm of 0.6–0.8, and the cells were grown until OD600nm 3.5. The cells
    were harvested by centrifuge (8000 rpm, 10 min) and washed thrice with S30 Buffer
    (10 mM Tris-acetate, 14 mM Mg(OAc)2, 60 mM KOAc, 5 mM DTT) and stored at −80 °C
    after flash-freeze using liquid nitrogen. The cells were resuspended in S30 Buffer
    (1.2 g/mL) and aliquoted to 1.5 mL. Aliquots were sonicated with a FB50 (Thermo
    Fisher) sonifier with a 3 mm microtip (10 s on/10 s off) 5 times. The extract
    was prepared by collecting supernatant after 12,000 rpm of centrifugation. Cell
    free natural product biosynthesis was performed by using eGFP plasmid DNA as a
    control and a reaction buffer with the following contents: 1.2 mM ATP; 0.85 mM
    each of GTP, UTP and CTP; 34 μg/mL folinic acid; 171 μg/mL T7 RNA polymerase;
    2 mM each of the 20 translatable amino acids, 0.33 mM nicotinamide adenine dinucleotide
    (NAD), 0.26 mM coenzyme A (CoA), 33 mM PEP, 130 mM potassium glutamate, 10 mM
    ammonium glutamate, 12 mM magnesium glutamate, 1.5 mM spermidine, 1 mM putrescine,
    57 mM HEPES, 4 mM sodium oxalate, and 1 μL of 20 mg/mL Sfp []. To the reaction
    buffer, 0.25 volume of cell extract (described above) and 20 μg/mL plasmid DNA
    were added to reach a final volume of 25 μL. The reaction was held for 20 h at
    30 °C and the required natural product metabolic substrates were added: gluocse-1-phosphate
    (10 mM) and dTTP (10 mM) for tailoring sugar biosynthesis, and propionyl-CoA (1 mM)
    and methyl malonyl-CoA (6 mM) for 6dEB biosynthesis. After 20 h, the samples were
    moved to −20 °C prior to SDS-PAGE and LC-MS analysis. Samples were prepared for
    analysis by diluting three times in LC-MS grade methanol and centrifuging at 13,000 rpm
    for 15 min with supernatant used for analysis. Erythromycin and 6dEB samples were
    applied to a Thermo Scientific Orbitrap XL with a C-18 analytical column. TDP-deoxysugar
    samples (mycarose, desosamine) were measured by Agilent G6545A quadrupole-time-of-flight
    (Q-TOF) using an XBridge Shield RP18 3.5 μm, 3.0 mm × 150 mm column from Waters.
    All MS analyses were conducted in positive ion mode. A linear gradient of 80%
    buffer A (95% water/5% acetonitrile/0.1% formic acid) to 100% buffer B (5% water/95%
    acetonitrile/0.1% formic acid) was used at a flow rate of 10 μL/min for the LC.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - BCR
  - ETV5
  - CACNA1E
  - NDUFB9
  - C2
  - CLU
  - ADAM8
  - Propionyl
  - Methylmalonyl
  - Al
  - Deaxyerythronolide B
  - Glucose
  - erythronolide B
  - TDP-4'-keto-6'-deoxy-glucose
  - TDP-D-Desosamine
  - 3-marcarosyl erythronolide B
  - Erythromycin A
  - PGJZ10
  - D-Desosamine
  - Glucose-1'-P
genes:
- word: All
  symbol: ALL
  source: hgnc_alias_symbol
  hgnc_symbol: BCR
  entrez: '613'
- word: Erm
  symbol: ERM
  source: hgnc_alias_symbol
  hgnc_symbol: ETV5
  entrez: '2119'
- word: BII
  symbol: BII
  source: hgnc_alias_symbol
  hgnc_symbol: CACNA1E
  entrez: '777'
- word: BIIII,
  symbol: B22
  source: hgnc_alias_symbol
  hgnc_symbol: NDUFB9
  entrez: '4715'
- word: CII,
  symbol: C2
  source: hgnc_symbol
  hgnc_symbol: C2
  entrez: '717'
- word: CII
  symbol: CLI
  source: hgnc_prev_symbol
  hgnc_symbol: CLU
  entrez: '1191'
- word: CII
  symbol: C2
  source: hgnc_symbol
  hgnc_symbol: C2
  entrez: '717'
- word: MS2
  symbol: MS2
  source: hgnc_alias_symbol
  hgnc_symbol: ADAM8
  entrez: '101'
- word: MS2
  symbol: MS2
  source: hgnc_alias_symbol
  hgnc_symbol: ADAM8
  entrez: '101'
chemicals:
- word: Propionyl
  source: MESH
  identifier: C009061
- word: Methylmalonyl
  source: MESH
  identifier: C015357
- word: Al
  source: MESH
  identifier: D000535
- word: Deaxyerythronolide B
  source: MESH
  identifier: D001895
- word: Glucose
  source: MESH
  identifier: D005947
- word: erythronolide B
  source: MESH
  identifier: C038315
- word: TDP-4'-keto-6'-deoxy-glucose
  source: MESH
  identifier: C112447
- word: TDP-D-Desosamine
  source: MESH
  identifier: C513911
- word: 3-marcarosyl erythronolide B
  source: MESH
  identifier: C038315
- word: Erythromycin A
  source: MESH
  identifier: D004917
- word: PGJZ10
  source: MESH
  identifier: C052091
- word: D-Desosamine
  source: MESH
  identifier: C110125
diseases:
- word: Glucose-1'-P
  source: MESH
  identifier: C566748
figid_alias: PMC7803631__F3a
redirect_from: /figures/PMC7803631__F3a
figtype: Figure
---
