---
figid: PMC7140926__PP_201901118DR1_f6
figlink: pmc/articles/PMC7140926/figure/fig6/
number: F6
caption: Metabolic regulatory network of Osgs1;1 in rice roots by GAM visualization.
  Integrated metabolomic and transcriptomic network analysis of metabolite amounts
  and transcript abundance between Osgs1;1 mutant roots and wild type at the third-leaf
  stage. The enriched metabolomic/transcriptomic network highlights the most regulated
  subnetwork in the mutants in comparison with wild type, based on the GAM web application
  (; https://artyomovlab.wustl.edu/shiny/gam/). Metabolites and mRNAs whose levels
  were increased/upregulated (log2 FC > 0) or decreased/downregulated (log2 FC < 0)
  in the mutants versus wild type are represented as red (log2 FC > 0) or green (log2
  FC < 0) nodes (metabolites) and links (reactions), respectively. Metabolite names
  with italic characters represent metabolites that were undetectable by gas chromatography-mass
  spectrometry-based metabolite profiling. Larger nodes and wider links indicate lower
  FDR values. Solid lines represent direct enzyme reactions between metabolites, whereas
  dashed lines indicate “main” reaction formula in one-to-one relationships of substrate-product
  pairs in the KEGG pathway maps for Arabidopsis. Gene names and metabolite names
  are listed in Supplemental Tables S8 and S9. We used transcript profiles of mutant
  NC2373 to represent Osgs1;1 mutant for the network analysis.
pmcid: PMC7140926
papertitle: Cytosolic GLUTAMINE SYNTHETASE1;1 Modulates Metabolism and Chloroplast
  Development in Roots.
reftext: Miyako Kusano, et al. Plant Physiol. 2020 Apr;182(4):1894-1909.
pmc_ranked_result_index: '27810'
pathway_score: 0.6672875
filename: PP_201901118DR1_f6.jpg
figtitle: Metabolic regulatory network of Osgs1;1 in rice roots by GAM visualization
year: '2020'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7140926__PP_201901118DR1_f6.html
  '@type': Dataset
  description: Metabolic regulatory network of Osgs1;1 in rice roots by GAM visualization.
    Integrated metabolomic and transcriptomic network analysis of metabolite amounts
    and transcript abundance between Osgs1;1 mutant roots and wild type at the third-leaf
    stage. The enriched metabolomic/transcriptomic network highlights the most regulated
    subnetwork in the mutants in comparison with wild type, based on the GAM web application
    (; https://artyomovlab.wustl.edu/shiny/gam/). Metabolites and mRNAs whose levels
    were increased/upregulated (log2 FC > 0) or decreased/downregulated (log2 FC <
    0) in the mutants versus wild type are represented as red (log2 FC > 0) or green
    (log2 FC < 0) nodes (metabolites) and links (reactions), respectively. Metabolite
    names with italic characters represent metabolites that were undetectable by gas
    chromatography-mass spectrometry-based metabolite profiling. Larger nodes and
    wider links indicate lower FDR values. Solid lines represent direct enzyme reactions
    between metabolites, whereas dashed lines indicate “main” reaction formula in
    one-to-one relationships of substrate-product pairs in the KEGG pathway maps for
    Arabidopsis. Gene names and metabolite names are listed in Supplemental Tables
    S8 and S9. We used transcript profiles of mutant NC2373 to represent Osgs1;1 mutant
    for the network analysis.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - PLP1
  - SERPINA1
  - TULP1
  - TYRP1
  - ACO1
  - MAL
  - KLK15
  - IDH1
  - AGXT2
  - TREH
  - Asn
  - Glu
  - Asp
  - Trp
  - Ser
  - TPP
genes:
- word: PLP
  symbol: PLP
  source: hgnc_prev_symbol
  hgnc_symbol: PLP1
  entrez: '5354'
- word: AAT
  symbol: AAT
  source: hgnc_alias_symbol
  hgnc_symbol: SERPINA1
  entrez: '5265'
- word: PLP
  symbol: PLP
  source: hgnc_prev_symbol
  hgnc_symbol: PLP1
  entrez: '5354'
- word: RP14
  symbol: RP14
  source: hgnc_prev_symbol
  hgnc_symbol: TULP1
  entrez: '7287'
- word: AAT
  symbol: AAT
  source: hgnc_alias_symbol
  hgnc_symbol: SERPINA1
  entrez: '5265'
- word: Trp
  symbol: TRP
  source: hgnc_alias_symbol
  hgnc_symbol: TYRP1
  entrez: '7306'
- word: ACO1
  symbol: ACO1
  source: hgnc_symbol
  hgnc_symbol: ACO1
  entrez: '48'
- word: Mal
  symbol: MAL
  source: hgnc_symbol
  hgnc_symbol: MAL
  entrez: '4118'
- word: Асо
  symbol: ACO
  source: hgnc_alias_symbol
  hgnc_symbol: KLK15
  entrez: '55554'
- word: AAT
  symbol: AAT
  source: hgnc_alias_symbol
  hgnc_symbol: SERPINA1
  entrez: '5265'
- word: IDH1
  symbol: IDH1
  source: hgnc_symbol
  hgnc_symbol: IDH1
  entrez: '3417'
- word: AAT
  symbol: AAT
  source: hgnc_alias_symbol
  hgnc_symbol: SERPINA1
  entrez: '5265'
- word: AGT2
  symbol: AGT2
  source: hgnc_alias_symbol
  hgnc_symbol: AGXT2
  entrez: '64902'
- word: Tre
  symbol: TRE
  source: hgnc_alias_symbol
  hgnc_symbol: TREH
  entrez: '11181'
chemicals:
- word: Asn
  source: MESH
  identifier: C528802
- word: Glu
  source: MESH
  identifier: C094686
- word: Asp
  source: MESH
  identifier: C108952
- word: Trp
  source: MESH
  identifier: C509690
- word: Ser
  source: MESH
  identifier: C530429
- word: TPP
  source: MESH
  identifier: D013835
diseases: []
figid_alias: PMC7140926__F6
redirect_from: /figures/PMC7140926__F6
figtype: Figure
---
