---
figid: PMC7045056__fcimb-10-00042-g0007
figlink: pmc/articles/PMC7045056/figure/F7/
number: Figure 7
caption: 'Interaction networks of top score canonical pathways and transcripts affected
  in FFPE tissue samples from meningococcal septic shock patients. The interaction
  networks in lungs (A), heart (B), kidneys (C), liver (D), and spleen (E) are generated
  through the use of IPA ≪core analysis≫. In addition, ≪Canonical Pathways Analysis≫
  was performed to determine genes that were involved in well-documented canonical
  signal transduction or metabolic pathways, from the library of canonical pathways
  in IPA. Benjamini–Hochberg procedure for multiple testing corrections was performed.
  Significantly enriched genes in the dataset were overlayed into the specific canonical
  pathway. The analysis was performed in silico using Molecular Activity Predictor
  analysis of IPA. The shapes represent the molecular classes of the ≪gene≫. In the
  figures, red represents upregulation and green down-regulation, and color intensity
  represents the relative magnitude of change in gene expression. Gray color indicates
  no activity pattern available. Direct and indirect interactions are indicated by
  solid and dashed lines, respectively. The network diagram shows the biological relationship
  between the indicated genes lines: — represents direct physical interactions; —–
  represents indirect functional interactions; → represents activation; ⊣ represents
  inhibition. The blue lines indicate that the direction of regulation is consistent
  with IPA prediction. In contrast, yellow lines indicate that the regulation observed
  is inconsistent with expectations, while gray lines indicate lack of pre-existing
  data to formulate expectations. Nodes are displayed using various shapes that represent
  the functional class of the genes.'
pmcid: PMC7045056
papertitle: Extensive Changes in Transcriptomic “Fingerprints” and Immunological Cells
  in the Large Organs of Patients Dying of Acute Septic Shock and Multiple Organ Failure
  Caused by Neisseria meningitidis.
reftext: Berit Sletbakk Brusletto, et al. Front Cell Infect Microbiol. 2020;10:42.
pmc_ranked_result_index: '111556'
pathway_score: 0.8548791
filename: fcimb-10-00042-g0007.jpg
figtitle: Interaction networks of top score canonical pathways and transcripts affected
  in FFPE tissue samples from meningococcal septic shock patients
year: '2020'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7045056__fcimb-10-00042-g0007.html
  '@type': Dataset
  description: 'Interaction networks of top score canonical pathways and transcripts
    affected in FFPE tissue samples from meningococcal septic shock patients. The
    interaction networks in lungs (A), heart (B), kidneys (C), liver (D), and spleen
    (E) are generated through the use of IPA ≪core analysis≫. In addition, ≪Canonical
    Pathways Analysis≫ was performed to determine genes that were involved in well-documented
    canonical signal transduction or metabolic pathways, from the library of canonical
    pathways in IPA. Benjamini–Hochberg procedure for multiple testing corrections
    was performed. Significantly enriched genes in the dataset were overlayed into
    the specific canonical pathway. The analysis was performed in silico using Molecular
    Activity Predictor analysis of IPA. The shapes represent the molecular classes
    of the ≪gene≫. In the figures, red represents upregulation and green down-regulation,
    and color intensity represents the relative magnitude of change in gene expression.
    Gray color indicates no activity pattern available. Direct and indirect interactions
    are indicated by solid and dashed lines, respectively. The network diagram shows
    the biological relationship between the indicated genes lines: — represents direct
    physical interactions; —– represents indirect functional interactions; → represents
    activation; ⊣ represents inhibition. The blue lines indicate that the direction
    of regulation is consistent with IPA prediction. In contrast, yellow lines indicate
    that the regulation observed is inconsistent with expectations, while gray lines
    indicate lack of pre-existing data to formulate expectations. Nodes are displayed
    using various shapes that represent the functional class of the genes.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - APOA2
  - APOA4
  - APOA5
  - APOA1
  - O Cr
  - O Eyne
  - CO
  - Re
genes:
- word: APOA
  symbol: APOA
  source: bioentities_symbol
  hgnc_symbol: APOA2
  entrez: '336'
- word: APOA
  symbol: APOA
  source: bioentities_symbol
  hgnc_symbol: APOA4
  entrez: '337'
- word: APOA
  symbol: APOA
  source: bioentities_symbol
  hgnc_symbol: APOA5
  entrez: '116519'
- word: APOA
  symbol: APOA
  source: bioentities_symbol
  hgnc_symbol: APOA1
  entrez: '335'
chemicals:
- word: O Cr
  source: MESH
  identifier: D002857
- word: O Eyne
  source: MESH
  identifier: D013481
- word: CO
  source: MESH
  identifier: D002245
- word: Re
  source: MESH
  identifier: D012211
diseases: []
---
