---
figid: PMC7878030__41437_2021_403_Fig1_HTML
figlink: pmc/articles/PMC7878030/figure/Fig1/
number: Fig. 1
caption: In each example, multiple independent variants can be combined to constitute
  the population scale loss-of-function allele state. a Loss of function in SLC30A8
  is associated with reduced risk of type 2 diabetes in humans (Flannick et al. ;
  Dwivedi et al. ). b Experimental evolution in Pseudomonas aeruginosa resulted repeatedly
  in loss-of-function mutations in nfxB, conferring antibiotic resistance (Wong et
  al. ). c Experimental evolution in yeast led to consistent disruption of specific
  signaling pathway genes including MTH1 during adaptation to stable environments
  (Kvitek and Sherlock ). d Populations of Plasmodium falciparum repeatedly evolved
  loss-of-function alleles in Epac during adaptation to lab culture environments (Claessens
  et al. ). e Natural RDO5 loss-of-function variants in Arabidopsis thaliana occurred
  at high frequency in northwest Europe and caused reduced seed dormancy, a trait
  under strong locally adaptive selection (Xiang et al. ). f Adaptation to agricultural
  intensification led to selection for semi-dwarf rice, which is caused by loss-of-function
  variants in GA20ox2 (Spielmeyer et al. ; Sasaki et al. ).
pmcid: PMC7878030
papertitle: The population genomics of adaptive loss of function.
reftext: J. Grey Monroe, et al. Heredity (Edinb). 2021 Feb 11 :1-13.
pmc_ranked_result_index: '101873'
pathway_score: 0.6885645
filename: 41437_2021_403_Fig1_HTML.jpg
figtitle: Population genomics of adaptive loss of function
year: '2021'
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7878030__41437_2021_403_Fig1_HTML.html
  '@type': Dataset
  description: In each example, multiple independent variants can be combined to constitute
    the population scale loss-of-function allele state. a Loss of function in SLC30A8
    is associated with reduced risk of type 2 diabetes in humans (Flannick et al.
    ; Dwivedi et al. ). b Experimental evolution in Pseudomonas aeruginosa resulted
    repeatedly in loss-of-function mutations in nfxB, conferring antibiotic resistance
    (Wong et al. ). c Experimental evolution in yeast led to consistent disruption
    of specific signaling pathway genes including MTH1 during adaptation to stable
    environments (Kvitek and Sherlock ). d Populations of Plasmodium falciparum repeatedly
    evolved loss-of-function alleles in Epac during adaptation to lab culture environments
    (Claessens et al. ). e Natural RDO5 loss-of-function variants in Arabidopsis thaliana
    occurred at high frequency in northwest Europe and caused reduced seed dormancy,
    a trait under strong locally adaptive selection (Xiang et al. ). f Adaptation
    to agricultural intensification led to selection for semi-dwarf rice, which is
    caused by loss-of-function variants in GA20ox2 (Spielmeyer et al. ; Sasaki et
    al. ).
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - SLC30A8
  - NUDT1
  - MAP6
  - HTRA2
  - RAPGEF3
  - LAT2
  - O
  - diabetes
genes:
- word: SLC30A8
  symbol: SLC30A8
  source: hgnc_symbol
  hgnc_symbol: SLC30A8
  entrez: '169026'
- word: '*SLC30A8'
  symbol: SLC30A8
  source: hgnc_symbol
  hgnc_symbol: SLC30A8
  entrez: '169026'
- word: MTH1
  symbol: MTH1
  source: hgnc_prev_symbol
  hgnc_symbol: NUDT1
  entrez: '4521'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: OMI
  symbol: OMI
  source: hgnc_alias_symbol
  hgnc_symbol: HTRA2
  entrez: '27429'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: Epac
  symbol: EPAC
  source: hgnc_alias_symbol
  hgnc_symbol: RAPGEF3
  entrez: '10411'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: stop
  symbol: STOP
  source: hgnc_alias_symbol
  hgnc_symbol: MAP6
  entrez: '4135'
- word: lab
  symbol: LAB
  source: hgnc_alias_symbol
  hgnc_symbol: LAT2
  entrez: '7462'
chemicals:
- word: O
  source: MESH
  identifier: D013481
diseases:
- word: diabetes
  source: MESH
  identifier: D003920
---
