---
figid: PMC7681076__thnov11p0222g004
figlink: pmc/articles/PMC7681076/figure/F4/
number: F4
caption: 'Proteaphagy: selective autophagy of inactive proteasomes. (A) Selective
  pathway: upon proteasome inactivation by MG132, 26S proteasomes are ubiquitinated
  in an Hsp42-dependent manner and shepherded to expanding autophagosome membranes
  by the selective proteaphagy receptors Cue5 in yeast or RPN10 in Arabidopsis. RPN10
  recognizes Atg8, an ubiquitin-like modifier that decorates the autophagosome membrane,
  via the C-terminal ubiquitin-interacting motifs (UIMs) of RPN10. Cue5 can simultaneously
  bind ubiquitin and Atg8. In mammals, the proteasome subunits RPN1, RPN10, and RPN13
  are poly-ubiquitinated upon amino acid starvation, which facilitates their recognition
  by P62. By simultaneous interaction with LC3, P62 delivers inactive 26S proteasomes
  to the expanding phagophore for eventual turnover by autophagy, a process that requires
  the TOR kinase. (B) Nonselective pathway: nitrogen starvation induces proteaphagy
  in yeast in a way that does not depend on RUNP10. During this process, 26S proteasomes
  dissociate into core particle (CP) and regulatory particle (RP) sub-complexes. The
  CP and RP then coalesce into cytoplasmic foci in a Snx4/41/42-dependent manner.
  Autophagy of CP (but not RP) also depends on the deubiquitinating enzyme Ubp3/Bre5.
  In contrast, during carbon starvation, proteasomes are reversibly sorted to avoid
  autophagic degradation. This process requires Blm10 for the CP, Spg5 and the C-terminus
  of Rpn11 for the RP, and the Nat3/Mdm20 complex for both.'
pmcid: PMC7681076
papertitle: 'Selective autophagy of intracellular organelles: recent research advances.'
reftext: Wen Li, et al. Theranostics. 2021;11(1):222-256.
pmc_ranked_result_index: '63400'
pathway_score: 0.8641169
filename: thnov11p0222g004.jpg
figtitle: 'Proteaphagy: selective autophagy of inactive proteasomes'
year: '2021'
organisms:
- Mus musculus
- Rattus norvegicus
- Human alphaherpesvirus 1
- Human immunodeficiency virus 1
- Dengue virus
- Ebola virus
- Listeria monocytogenes
- Arabidopsis thaliana
- Komagataella pastoris
- Saccharomyces cerevisiae
- Caenorhabditis elegans
- Zika virus
- Drosophila melanogaster
- Danio rerio
- Salmonella enterica subsp. enterica serovar Typhimurium
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7681076__thnov11p0222g004.html
  '@type': Dataset
  description: 'Proteaphagy: selective autophagy of inactive proteasomes. (A) Selective
    pathway: upon proteasome inactivation by MG132, 26S proteasomes are ubiquitinated
    in an Hsp42-dependent manner and shepherded to expanding autophagosome membranes
    by the selective proteaphagy receptors Cue5 in yeast or RPN10 in Arabidopsis.
    RPN10 recognizes Atg8, an ubiquitin-like modifier that decorates the autophagosome
    membrane, via the C-terminal ubiquitin-interacting motifs (UIMs) of RPN10. Cue5
    can simultaneously bind ubiquitin and Atg8. In mammals, the proteasome subunits
    RPN1, RPN10, and RPN13 are poly-ubiquitinated upon amino acid starvation, which
    facilitates their recognition by P62. By simultaneous interaction with LC3, P62
    delivers inactive 26S proteasomes to the expanding phagophore for eventual turnover
    by autophagy, a process that requires the TOR kinase. (B) Nonselective pathway:
    nitrogen starvation induces proteaphagy in yeast in a way that does not depend
    on RUNP10. During this process, 26S proteasomes dissociate into core particle
    (CP) and regulatory particle (RP) sub-complexes. The CP and RP then coalesce into
    cytoplasmic foci in a Snx4/41/42-dependent manner. Autophagy of CP (but not RP)
    also depends on the deubiquitinating enzyme Ubp3/Bre5. In contrast, during carbon
    starvation, proteasomes are reversibly sorted to avoid autophagic degradation.
    This process requires Blm10 for the CP, Spg5 and the C-terminus of Rpn11 for the
    RP, and the Nat3/Mdm20 complex for both.'
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - ULK1
  - GABARAPL2
  - CP
  - RPN1
  - MAP1LC3A
  - RORC
  - PSMD14
  - SNX4
  - carbon
  - nitrogen
genes:
- word: Atg1
  symbol: ATG1
  source: hgnc_alias_symbol
  hgnc_symbol: ULK1
  entrez: '8408'
- word: Atg8
  symbol: ATG8
  source: hgnc_alias_symbol
  hgnc_symbol: GABARAPL2
  entrez: '11345'
- word: CP
  symbol: CP
  source: hgnc_symbol
  hgnc_symbol: CP
  entrez: '1356'
- word: Atg8
  symbol: ATG8
  source: hgnc_alias_symbol
  hgnc_symbol: GABARAPL2
  entrez: '11345'
- word: RPN1
  symbol: RPN1
  source: hgnc_symbol
  hgnc_symbol: RPN1
  entrez: '6184'
- word: LC3
  symbol: LC3
  source: hgnc_alias_symbol
  hgnc_symbol: MAP1LC3A
  entrez: '84557'
- word: LC3
  symbol: LC3
  source: hgnc_alias_symbol
  hgnc_symbol: MAP1LC3A
  entrez: '84557'
- word: TOR
  symbol: TOR
  source: hgnc_alias_symbol
  hgnc_symbol: RORC
  entrez: '6097'
- word: Rpn11
  symbol: Rpn11
  source: hgnc_alias_symbol
  hgnc_symbol: PSMD14
  entrez: '10213'
- word: Atg8
  symbol: ATG8
  source: hgnc_alias_symbol
  hgnc_symbol: GABARAPL2
  entrez: '11345'
- word: СР
  symbol: CP
  source: hgnc_symbol
  hgnc_symbol: CP
  entrez: '1356'
- word: Atg8
  symbol: ATG8
  source: hgnc_alias_symbol
  hgnc_symbol: GABARAPL2
  entrez: '11345'
- word: CP
  symbol: CP
  source: hgnc_symbol
  hgnc_symbol: CP
  entrez: '1356'
- word: Snx4/41
  symbol: SNX4
  source: hgnc_symbol
  hgnc_symbol: SNX4
  entrez: '8723'
- word: Snx4/42
  symbol: SNX4
  source: hgnc_symbol
  hgnc_symbol: SNX4
  entrez: '8723'
chemicals:
- word: carbon
  source: MESH
  identifier: D002244
- word: nitrogen
  source: MESH
  identifier: D009584
diseases: []
figid_alias: PMC7681076__F4
redirect_from: /figures/PMC7681076__F4
figtype: Figure
---
