---
figid: PMC7083089__nihms-1573695-f0004
figlink: pmc/articles/PMC7083089/figure/F4/
number: Figure 4
caption: Multiple secondary structures at CFSs may progressively impede replication.
  Replication impairment can potentially be resolved by helicases such as WRN and
  RecQ1 (, ). TLS synthesis could also facilitate replication, or gap filling if structures
  are bypassed. If the replisome becomes uncoupled from the replicative helicase,
  ssDNA stretches can result in RPA binding, activation of stalled fork sensors, and
  ATR signaling. A stalled fork (top left side) can result in the approach of a converging
  fork from the opposite direction (top right side) that in turn could stall at other
  DNA structures or due to a transcription-replication collision, leading to an unreplicated
  area. Impaired replication can be resolved by structure-specific endonuclease (SSE)
  cleavage at secondary structures or at stalled and reversed forks caused by secondary
  structures. Cleavage of a stalled fork by MUS81-EME1/Mus81-Mms4 would result in
  a one-ended break that could engage in break-induced replication (BIR) using another
  chromosome as a template for repair (left pathway). Alternatively, broken fork repair
  (BFR) using the other sister chromatid as a template for repair or gap filling will
  result in conservative repair without loss of sequence (middle pathway). Cleavage
  of forks by MUS81/Mus81 at two stalled forks (blue arrows), or across the base of
  a 4-way junction (not shown), will result in a two-ended break, which can heal with
  fidelity (for example using homologous recombination (HR); middle pathway) or in
  a manner that results in deletions (for example by single strand annealing (SSA),
  non-homologous end joining (NHEJ), or microhomology-mediated end joining MMEJ; right
  pathway). Other SSEs (e.g. XPF-ERCC1/Rad1-Rad10 (pink arrows), or SLX1-SLX4/Slx1-Slx4
  (not shown)) may also be involved in cleavage. Gap filling or HR-dependent mechanisms
  require DNA synthesis, which can manifest as MiDAS if it occurs in M phase. Human
  proteins are listed first, with S. cerevisiae homologs after the slash.
pmcid: PMC7083089
papertitle: The role of fork stalling and DNA structures in causing chromosome fragility.
reftext: Simran Kaushal, et al. Genes Chromosomes Cancer. ;58(5):270-283.
pmc_ranked_result_index: '39044'
pathway_score: 0.9187307
filename: nihms-1573695-f0004.jpg
figtitle: Secondary structures at CFSs can impede replication, resulting in fork stalling
  and cleavage to cause fragility
year: ''
organisms:
- Homo sapiens
ndex: ''
annotations: []
seo: CreativeWork
schema-jsonld:
  '@context': https://schema.org/
  '@id': https://pfocr.wikipathways.org/figures/PMC7083089__nihms-1573695-f0004.html
  '@type': Dataset
  description: Multiple secondary structures at CFSs may progressively impede replication.
    Replication impairment can potentially be resolved by helicases such as WRN and
    RecQ1 (, ). TLS synthesis could also facilitate replication, or gap filling if
    structures are bypassed. If the replisome becomes uncoupled from the replicative
    helicase, ssDNA stretches can result in RPA binding, activation of stalled fork
    sensors, and ATR signaling. A stalled fork (top left side) can result in the approach
    of a converging fork from the opposite direction (top right side) that in turn
    could stall at other DNA structures or due to a transcription-replication collision,
    leading to an unreplicated area. Impaired replication can be resolved by structure-specific
    endonuclease (SSE) cleavage at secondary structures or at stalled and reversed
    forks caused by secondary structures. Cleavage of a stalled fork by MUS81-EME1/Mus81-Mms4
    would result in a one-ended break that could engage in break-induced replication
    (BIR) using another chromosome as a template for repair (left pathway). Alternatively,
    broken fork repair (BFR) using the other sister chromatid as a template for repair
    or gap filling will result in conservative repair without loss of sequence (middle
    pathway). Cleavage of forks by MUS81/Mus81 at two stalled forks (blue arrows),
    or across the base of a 4-way junction (not shown), will result in a two-ended
    break, which can heal with fidelity (for example using homologous recombination
    (HR); middle pathway) or in a manner that results in deletions (for example by
    single strand annealing (SSA), non-homologous end joining (NHEJ), or microhomology-mediated
    end joining MMEJ; right pathway). Other SSEs (e.g. XPF-ERCC1/Rad1-Rad10 (pink
    arrows), or SLX1-SLX4/Slx1-Slx4 (not shown)) may also be involved in cleavage.
    Gap filling or HR-dependent mechanisms require DNA synthesis, which can manifest
    as MiDAS if it occurs in M phase. Human proteins are listed first, with S. cerevisiae
    homologs after the slash.
  license: CC0
  name: CreativeWork
  creator:
    '@type': Organization
    name: WikiPathways
  keywords:
  - RPA1
  - RPA2
  - RPA3
  - ATR
  - SLX4
  - MUS81
  - FUS
  - RASA1
  - GOLPH3
  - CALR
  - KCNJ11
  - ENG
genes:
- word: RPA
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA1
  entrez: '6117'
- word: RPA
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA2
  entrez: '6118'
- word: RPA
  symbol: RPA
  source: bioentities_symbol
  hgnc_symbol: RPA3
  entrez: '6119'
- word: ATR
  symbol: ATR
  source: hgnc_symbol
  hgnc_symbol: ATR
  entrez: '545'
- word: SLX4/
  symbol: SLX4
  source: hgnc_symbol
  hgnc_symbol: SLX4
  entrez: '84464'
- word: MUS81
  symbol: MUS81
  source: hgnc_symbol
  hgnc_symbol: MUS81
  entrez: '80198'
- word: TLS,RecQhelicases
  symbol: TLS
  source: hgnc_alias_symbol
  hgnc_symbol: FUS
  entrez: '2521'
- word: TLS,
  symbol: TLS
  source: hgnc_alias_symbol
  hgnc_symbol: FUS
  entrez: '2521'
- word: gap
  symbol: GAP
  source: hgnc_alias_symbol
  hgnc_symbol: RASA1
  entrez: '5921'
- word: MÄ°DAS
  symbol: MIDAS
  source: hgnc_alias_symbol
  hgnc_symbol: GOLPH3
  entrez: '64083'
- word: SSA,
  symbol: SSA
  source: hgnc_alias_symbol
  hgnc_symbol: CALR
  entrez: '811'
- word: SSA,NHEJ,MMEJ
  symbol: SSA
  source: hgnc_alias_symbol
  hgnc_symbol: CALR
  entrez: '811'
- word: BIR/BFR
  symbol: BIR
  source: hgnc_alias_symbol
  hgnc_symbol: KCNJ11
  entrez: '3767'
- word: End
  symbol: END
  source: hgnc_alias_symbol
  hgnc_symbol: ENG
  entrez: '2022'
chemicals: []
diseases: []
---
